
<!DOCTYPE html>

<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"> <!--<![endif]-->
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>Preprocessing — The COBRA Toolbox</title>
<link href="../../../../_static/img/apple-touch-icon.png" rel="apple-touch-icon" sizes="180x180"/>
<link href="../../../../_static/img/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png"/>
<link href="../../../../_static/img/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png"/>
<link color="#5bbad5" href="../../../../_static/img/safari-pinned-tab.svg" rel="mask-icon"/>
<link href="../../../../_static/json/manifest.json" rel="manifest"/>
<meta content="../../../../_static/browserconfig.xml" name="msapplication-config"/>
<meta content="#ffffff" name="theme-color"/>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
<script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
<script src="../../../../_static/js/jquery.easy-autocomplete.min.js" type="text/javascript"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet"/>
<link href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" rel="stylesheet" type="text/css"/>
<link href="../../../../_static/css/easy-autocomplete.min.css" rel="stylesheet" type="text/css"/>
<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"/>
<link href="../../../../_static/css/cobra.css" rel="stylesheet" type="text/css">
<link href="../../../../genindex.html" rel="index" title="Index">
<link href="../../../../search.html" rel="search" title="Search">
<link href="../../../../index.html" rel="top" title="The COBRA Toolbox">
<link href="../index.html" rel="up" title="Transcriptomics">
<link href="../../../design/index.html" rel="next" title="Design">
<link href="../iMAT/index.html" rel="prev" title="iMAT">
<script src="../../../../_static/js/modernizr.min.js"></script>
</link></link></link></link></link></link></link></head>
<!-- Google tag (gtag.js) -->
<script async="" src="https://www.googletagmanager.com/gtag/js?id=G-KFJTC6J15L"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-KFJTC6J15L');
</script>

<body class="wy-body-for-nav" role="document">
<div class="wy-grid-for-nav">
<nav class="wy-nav-side" data-toggle="wy-nav-shift">
<div class="wy-side-scroll">
<div class="wy-side-nav-search">
<a href="../../../../index.html">
<img alt="logo" class="logo" src="http://gibbs.unal.edu.co/cobradoc/img/logos/logo.png"/>
</a>
<h1>The COBRA Toolbox</h1>
<div role="search">
<form action="../../../../search.html" class="wy-form" id="rtd-search-form" method="get">
<input id="simple" name="q" placeholder="Search docs" type="text"/>
<input name="check_keywords" type="hidden" value="yes"/>
<input name="area" type="hidden" value="default"/>
<script>
    	var options = {

		url: "../../../../_static/json/functions.json",

		getValue: "name",

		list: {
            onKeyEnterEvent: function(item) {
                    return "_static/../" + item.website_url
            },
            maxNumberOfElements: 25,
			match: {
					enabled: true,
			},
            sort: {
			    enabled: true
		    }
		},

    	template: {
	    	type: "custom",
		    method: function(value, item) {
                return "<span><a href='../../../../search.html?q=" + item.name + "&check_keywords=yes&area=default'>" + value + "</a>" + "<a href='../../../../_static/../" + item.website_url + "' ><img src='../../../../_static/link.svg'/></a></span>";
		    }
	    },

		theme: "square"

	};

	$("#simple").easyAutocomplete(options);


    </script>
</form>
</div>
</div>
<div aria-label="main navigation" class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation">
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../../../../index.html">Home</a></li>
</ul>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../../../../installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="../../../index.html">Functions</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="../../../analysis/index.html">Analysis</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../../base/index.html">Base</a></li>
<li class="toctree-l2 current"><a class="reference internal" href="../../index.html">Data integration</a><ul class="current">
<li class="toctree-l3"><a class="reference internal" href="../../chemoInformatics/index.html">ChemoInformatics</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../fluxomics/index.html">Fluxomics</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../metabotools/index.html">Metabotools</a></li>
<li class="toctree-l3 current"><a class="reference internal" href="../index.html">Transcriptomics</a><ul class="current">
<li class="toctree-l4"><a class="reference internal" href="../FASTCORE/index.html">FASTCORE</a></li>
<li class="toctree-l4"><a class="reference internal" href="../MBA/index.html">MBA</a></li>
<li class="toctree-l4"><a class="reference internal" href="../GIMME/index.html">GIMME</a></li>
<li class="toctree-l4"><a class="reference internal" href="../INIT/index.html">INIT</a></li>
<li class="toctree-l4"><a class="reference internal" href="../iMAT/index.html">iMAT</a></li>
<li class="toctree-l4 current"><a class="current reference internal" href="#">Preprocessing</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../../design/index.html">Design</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../../reconstruction/index.html">Reconstruction</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../../visualization/index.html">Visualization</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../../../tutorials/index.html">Tutorials</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../contributing.html">How to contribute</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../cite.html">How to cite</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../support.html">Support</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../faq.html">FAQ</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../contributors.html">Contributors</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../funding.html">Funding</a></li>
</ul>
</div>
</div>
</nav>
<section class="wy-nav-content-wrap" data-toggle="wy-nav-shift">
<nav aria-label="top navigation" class="wy-nav-top" role="navigation">
<i class="fa fa-bars" data-toggle="wy-nav-top"></i>
<a href="../../../../index.html">The COBRA Toolbox</a>
</nav>
<div class="wy-nav-content">
<div class="rst-content">
<div aria-label="breadcrumbs navigation" role="navigation">
<ul class="wy-breadcrumbs">
<li><a href="../../../../contents.html">Docs</a> »</li>
<li><a href="../../../index.html">Functions</a> »</li>
<li><a href="../../index.html">Data integration</a> »</li>
<li><a href="../index.html">Transcriptomics</a> »</li>
<li>Preprocessing</li>
<li class="wy-breadcrumbs-aside">
</li>
</ul>
<hr/>
</div>
<div class="document" itemscope="itemscope" itemtype="https://schema.org/Article" role="main">
<div itemprop="articleBody">
<div class="section" id="preprocessing">
<span id="id1"></span><h1>Preprocessing<a class="headerlink" href="#preprocessing" title="Permalink to this headline">¶</a></h1>
<span class="target" id="module-src.dataIntegration.transcriptomics.preprocessing"></span><dl class="function">
<dt id="src.dataIntegration.transcriptomics.preprocessing.GPRparser">
<code class="descname">GPRparser</code><span class="sig-paren">(</span><em>model</em>, <em>getCNFSets</em><span class="sig-paren">)</span><a class="reference external" href="https://github.com/opencobra/cobratoolbox/blob/f3fe20df5c977cf0d212e12b1763bd96d15c8760/src/dataIntegration/transcriptomics/preprocessing/GPRparser.m"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#src.dataIntegration.transcriptomics.preprocessing.GPRparser" title="Permalink to this definition">¶</a></dt>
<dd><p>Maps the GPR rules of the model to a specified format that is used by
the model extraction methods</p>
<p><strong>Usage</strong></p>
<blockquote>
<div>parsedGPR = GPRparser(model)</div></blockquote>
<p><strong>Input</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>model</strong> –
cobra model structure</li>
</ul>
</div></blockquote>
<p><strong>Optional input</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>getCNFSets</strong> –
whether to get the CNF sets (true) or DNF sets (false).
DNF sets represent functional enzyme complexes, while
CNF sets represent the possible subunits of a complex.
(default: false , i.e. DNF sets)</li>
</ul>
</div></blockquote>
<p><strong>Output</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>parsedGPR</strong> –
cell matrix containing parsed GPR rule</li>
</ul>
</div></blockquote>
<p>AUTHORS: Thomas Pfau &amp; Anne Richelle, May 2017</p>
</dd></dl>
<dl class="function">
<dt id="src.dataIntegration.transcriptomics.preprocessing.findUsedGenesLevels">
<code class="descname">findUsedGenesLevels</code><span class="sig-paren">(</span><em>model</em>, <em>exprData</em>, <em>printLevel</em><span class="sig-paren">)</span><a class="reference external" href="https://github.com/opencobra/cobratoolbox/blob/f3fe20df5c977cf0d212e12b1763bd96d15c8760/src/dataIntegration/transcriptomics/preprocessing/findUsedGenesLevels.m"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#src.dataIntegration.transcriptomics.preprocessing.findUsedGenesLevels" title="Permalink to this definition">¶</a></dt>
<dd><p>Returns vectors of gene identifiers and corresponding gene expression
levels for each gene present in the model (‘model.genes’).</p>
<p><strong>Usage</strong></p>
<blockquote>
<div>[gene_id, gene_expr] = findUsedGenesLevels(model, exprData)</div></blockquote>
<p><strong>Inputs</strong></p>
<blockquote>
<div><ul>
<li><p class="first"><strong>model</strong> –
input model (COBRA model structure)</p>
</li>
<li><p class="first"><strong>exprData</strong> –
mRNA expression data structure
.gene                cell array containing GeneIDs in the same</p>
<blockquote>
<div><p>format as model.genes</p>
</div></blockquote>
<p>.value               Vector containing corresponding expression value (FPKM)</p>
</li>
</ul>
</div></blockquote>
<p><strong>Optional input</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>printLevel</strong> –
Printlevel for output (default 0);</li>
</ul>
</div></blockquote>
<p><strong>Outputs</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>gene_id</strong> –
vector of gene identifiers present in the model
that are associated with expression data</li>
<li><strong>gene_expr</strong> –
vector of expression values associated to each
‘gened_id’</li>
</ul>
</div></blockquote>
<p>Authors: - S. Opdam &amp; A. Richelle May 2017</p>
</dd></dl>
<dl class="function">
<dt id="src.dataIntegration.transcriptomics.preprocessing.mapExpressionToReactions">
<code class="descname">mapExpressionToReactions</code><span class="sig-paren">(</span><em>model</em>, <em>expressionData</em>, <em>minSum</em><span class="sig-paren">)</span><a class="reference external" href="https://github.com/opencobra/cobratoolbox/blob/f3fe20df5c977cf0d212e12b1763bd96d15c8760/src/dataIntegration/transcriptomics/preprocessing/mapExpressionToReactions.m"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#src.dataIntegration.transcriptomics.preprocessing.mapExpressionToReactions" title="Permalink to this definition">¶</a></dt>
<dd><p>Determines the expression data associated to each reaction present in
the model</p>
<p><strong>Usage</strong></p>
<blockquote>
<div>[expressionRxns parsedGPR] = mapExpressionToReactions(model, expressionData)</div></blockquote>
<p><strong>Inputs</strong></p>
<blockquote>
<div><blockquote>
<div><ul class="simple">
<li><strong>model                   model strusture</strong></li>
<li><strong>expressionData          mRNA expression data structure</strong></li>
</ul>
</div></blockquote>
<dl class="docutils">
<dt>.gene                    cell array containing GeneIDs in the same</dt>
<dd>format as model.genes</dd>
<dt>.value                  Vector containing corresponding expression</dt>
<dd>value (FPKM/RPKM)</dd>
</dl>
</div></blockquote>
<p><strong>Optional input</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>minSum</strong> –
instead of using min and max, use min for AND and Sum
for OR (default: false, i.e. use min)</li>
</ul>
</div></blockquote>
<p><strong>Outputs</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>expressionRxns</strong> –
reaction expression, corresponding to model.rxns.</li>
<li><strong>parsedGPR</strong> –
cell matrix containing parsed GPR rule</li>
</ul>
</div></blockquote>
</dd></dl>
<dl class="function">
<dt id="src.dataIntegration.transcriptomics.preprocessing.selectGeneFromGPR">
<code class="descname">selectGeneFromGPR</code><span class="sig-paren">(</span><em>model</em>, <em>gene_names</em>, <em>gene_exp</em>, <em>parsedGPR</em>, <em>minSum</em><span class="sig-paren">)</span><a class="reference external" href="https://github.com/opencobra/cobratoolbox/blob/f3fe20df5c977cf0d212e12b1763bd96d15c8760/src/dataIntegration/transcriptomics/preprocessing/selectGeneFromGPR.m"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#src.dataIntegration.transcriptomics.preprocessing.selectGeneFromGPR" title="Permalink to this definition">¶</a></dt>
<dd><p>Map gene expression to reaction expression using the GPR rules. An AND
will be replaced by MIN and an OR will be replaced by MAX.</p>
<p><strong>Usage</strong></p>
<blockquote>
<div>expressionCol = selectGeneFromGPR(model, gene_names, gene_exp, parsedGPR, minMax)</div></blockquote>
<p><strong>Inputs</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>model</strong> –
COBRA model struct</li>
<li><strong>gene_names</strong> –
gene identifiers corresponding to gene_exp. Names must
be in the same format as model.genes (column vector)
(as returned by “findUsedGeneLevels.m”)</li>
<li><strong>gene_exp</strong> –
gene FPKM/expression values, corresponding to names (column vector)
(as returned by “findUsedGeneLevels.m”)</li>
<li><strong>parsedGPR</strong> –
GPR matrix as returned by “GPRparser.m”</li>
</ul>
</div></blockquote>
<p><strong>Optional input</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>minSum</strong> –
instead of using min and max, use min for AND and Sum
for OR</li>
</ul>
</div></blockquote>
<p><strong>Output</strong></p>
<blockquote>
<div><ul class="simple">
<li><strong>expressionCol</strong> –
reaction expression, corresponding to model.rxns.
No gene-expression data and orphan reactions will
be given a value of -1.</li>
</ul>
</div></blockquote>
<p>AUTHOR: Anne Richelle, May 2017</p>
</dd></dl>
</div>
</div>
<div class="articleComments">
</div>
</div>
<footer>
<div aria-label="footer navigation" class="rst-footer-buttons" role="navigation">
<a accesskey="n" class="btn btn-default float-right" href="../../../design/index.html" rel="next" title="Design">Next <span class="fa fa-arrow-circle-right"></span></a>
<a accesskey="p" class="btn btn-default" href="../iMAT/index.html" rel="prev" title="iMAT"><span class="fa fa-arrow-circle-left"></span> Previous</a>
</div>
<hr/>
<div role="contentinfo">
<p>
        © Copyright 2017-2018, The COBRA Toolbox developers.

    </p>
</div>
  Built with Sphinx using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>
</div>
</div>
</section>
</div>
<script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'../../../../',
            VERSION:'2.0.0',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true,
            SOURCELINK_SUFFIX: '.txt'
        };
    </script>
<script src="../../../../_static/jquery.js" type="text/javascript"></script>
<script src="../../../../_static/underscore.js" type="text/javascript"></script>
<script src="../../../../_static/doctools.js" type="text/javascript"></script>
<script src="https://cdn.jsdelivr.net/gh/mathjax/MathJax@2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="../../../../_static/js/theme.js" type="text/javascript"></script>
<script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.StickyNav.enable();
      });
  </script>
</body>
</html>
