

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Citations &mdash; The COBRA Toolbox</title>
  

  
  
  
    <link rel="apple-touch-icon" sizes="180x180" href="_static/img/apple-touch-icon.png">
  
  
    <link rel="icon" type="image/png" sizes="32x32" href="_static/img/favicon-32x32.png">
  
  
    <link rel="icon" type="image/png" sizes="16x16" href="_static/img/favicon-16x16.png">
  
  
    <link rel="mask-icon" href="_static/img/safari-pinned-tab.svg" color="#5bbad5">
  
  
    <link rel="manifest" href="_static/json/manifest.json">
  
  
    <meta name="msapplication-config" content="_static/browserconfig.xml">
  
  <meta name="theme-color" content="#ffffff">


  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
  <script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
  <script src="_static/js/jquery.easy-autocomplete.min.js" type="text/javascript" ></script>

    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <link href='https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700' rel='stylesheet' type='text/css'>
  <link href="_static/css/easy-autocomplete.min.css" rel="stylesheet" type="text/css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

  
  
    

  

  
  
    <link rel="stylesheet" href="_static/css/cobra.css" type="text/css" />
  

  
    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
  
    <link rel="stylesheet" href="_static/css/cobra.css" type="text/css" />
  
    <link rel="stylesheet" href="_static/tabs.css" type="text/css" />
  

  
        <link rel="index" title="Index"
              href="genindex.html"/>
        <link rel="search" title="Search" href="search.html"/>
    <link rel="top" title="The COBRA Toolbox" href="index.html"/>
        <link rel="prev" title="Contact" href="contact.html"/> 

  
  <script src="_static/js/modernizr.min.js"></script>
</head>

<body class="wy-body-for-nav" role="document">

   
  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search">
          

          
            <a href="index.html" class="icon icon-home"> The COBRA Toolbox
          

          
            <img src="https://king.nuigalway.ie/cobratoolbox/img/cobratoolbox_logo_transparent.png" class="logo" alt="cobratoolbox logo" />
          
          </a>

              
          <h1>The COBRA Toolbox</h1>

          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" id="simple"/>
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
    <script>
    	var options = {

		url: "_static/json/functions.json",

		getValue: "name",

		list: {
            onKeyEnterEvent: function(item) {
                    return "_static/../" + item.website_url
            },
            maxNumberOfElements: 25,
			match: {
					enabled: true,
			},
            sort: {
			    enabled: true
		    }
		},

    	template: {
	    	type: "custom",
		    method: function(value, item) {
                return "<span><a href='search.html?q=" + item.name + "&check_keywords=yes&area=default'>" + value + "</a>" + "<a href='_static/../" + item.website_url + "' ><img src='_static/link.svg'/></a></span>";
		    }
	    },

		theme: "square"

	};

	$("#simple").easyAutocomplete(options);


    </script>
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
              <ul class="current">
                  <li class="toctree-l1"><a class="reference internal" href="index.html">Home</a></li>
              </ul>
              <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="modules/index.html">Functions</a></li>
<li class="toctree-l1"><a class="reference internal" href="tutorials/index.html">Tutorials</a></li>
<li class="toctree-l1"><a class="reference internal" href="contributing.html">How to contribute</a></li>
<li class="toctree-l1"><a class="reference internal" href="cite.html">How to cite</a></li>
<li class="toctree-l1"><a class="reference internal" href="support.html">Support</a></li>
<li class="toctree-l1"><a class="reference internal" href="faq.html">FAQ</a></li>
<li class="toctree-l1"><a class="reference internal" href="contributors.html">Contributors</a></li>
<li class="toctree-l1"><a class="reference internal" href="funding.html">Funding</a></li>
<li class="toctree-l1"><a class="reference internal" href="plan.html">Development Plan</a></li>
<li class="toctree-l1"><a class="reference internal" href="contact.html">Contact</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Citations</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#publications-that-cited-cobra-toolbox">Publications that cited COBRA Toolbox</a></li>
</ul>
</li>
</ul>

          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">The COBRA Toolbox</a>
        
      </nav>


      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="contents.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">Citations</li>
      <li class="wy-breadcrumbs-aside">
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="https://schema.org/Article">
           <div itemprop="articleBody">
            
  <section id="citations">
<h1>Citations<a class="headerlink" href="#citations" title="Permalink to this heading">¶</a></h1>
<br><section id="publications-that-cited-cobra-toolbox">
<h2>Publications that cited COBRA Toolbox<a class="headerlink" href="#publications-that-cited-cobra-toolbox" title="Permalink to this heading">¶</a></h2>
<div class="sphinx-tabs docutils container">
<div aria-label="Tabbed content" class="closeable" role="tablist"><button aria-controls="panel-0-0-0" aria-selected="true" class="sphinx-tabs-tab" id="tab-0-0-0" name="0-0" role="tab" tabindex="0">2024</button><button aria-controls="panel-0-0-1" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-1" name="0-1" role="tab" tabindex="-1">2023</button><button aria-controls="panel-0-0-2" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-2" name="0-2" role="tab" tabindex="-1">2022</button><button aria-controls="panel-0-0-3" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-3" name="0-3" role="tab" tabindex="-1">2021</button><button aria-controls="panel-0-0-4" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-4" name="0-4" role="tab" tabindex="-1">2020</button><button aria-controls="panel-0-0-5" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-5" name="0-5" role="tab" tabindex="-1">2019</button><button aria-controls="panel-0-0-6" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-6" name="0-6" role="tab" tabindex="-1">2018</button><button aria-controls="panel-0-0-7" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-7" name="0-7" role="tab" tabindex="-1">2017</button><button aria-controls="panel-0-0-8" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-8" name="0-8" role="tab" tabindex="-1">2016</button><button aria-controls="panel-0-0-9" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-9" name="0-9" role="tab" tabindex="-1">2015</button><button aria-controls="panel-0-0-10" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-10" name="0-10" role="tab" tabindex="-1">2014</button><button aria-controls="panel-0-0-11" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-11" name="0-11" role="tab" tabindex="-1">2013</button><button aria-controls="panel-0-0-12" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-12" name="0-12" role="tab" tabindex="-1">2012</button><button aria-controls="panel-0-0-13" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-13" name="0-13" role="tab" tabindex="-1">2011</button><button aria-controls="panel-0-0-14" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-14" name="0-14" role="tab" tabindex="-1">2010</button><button aria-controls="panel-0-0-15" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-15" name="0-15" role="tab" tabindex="-1">2009</button><button aria-controls="panel-0-0-16" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-16" name="0-16" role="tab" tabindex="-1">2008</button><button aria-controls="panel-0-0-17" aria-selected="false" class="sphinx-tabs-tab" id="tab-0-0-17" name="0-17" role="tab" tabindex="-1">2007</button></div><div aria-labelledby="tab-0-0-0" class="sphinx-tabs-panel" id="panel-0-0-0" name="0-0" role="tabpanel" tabindex="0"><div class="docutils container" id="id1">
<ol class="arabic simple" start="1">
<li id="id2"><p>Jinlin Zhu, Jialin Yin, Jing Chen, Mingyi Hu, Wenwei Lu, Hongchao Wang, Hao Zhang, and Wei Chen. Integrative analysis with microbial modelling and machine learning uncovers potential alleviators for ulcerative colitis. <em>GUT MICROBES</em>, DEC 31 2024. <a class="reference external" href="https://doi.org/10.1080/19490976.2024.2336877">[DOI]</a>.</p></li>
<li id="id3"><p>Seo-Young Park, Jinsung Song, Dong-Hyuk Choi, Uiseon Park, Hyeran Cho, Bee Hak Hong, Yaron R. Silberberg, and Dong-Yup Lee. Exploring metabolic effects of dipeptide feed media on cho cell cultures by in silico model-guided flux analysis. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 108(1):16, DEC 2024. <a class="reference external" href="https://doi.org/10.1007/s00253-023-12997-0">[DOI]</a>.</p></li>
<li id="id4"><p>Ehsan Motamedian, Matiss Ricards Baumanis, Janis Edmunds Daugavietis, Ieva Berzina, Karlis Shvirksts, Elina Dace, Janis Liepins, and Egils Stalidzans. A hybrid strategy for growth-associated production of sophorolipid using a reconstructed genome-scale metabolic model for the fructophilic yeast starmerella bombicola. <em>PROCESS SAFETY AND ENVIRONMENTAL PROTECTION</em>, 189:541–548, SEP 2024. <a class="reference external" href="https://doi.org/10.1016/j.psep.2024.06.094">[DOI]</a>.</p></li>
<li id="id5"><p>Xia Yang, Huiying Wang, Dongqin Ding, Huan Fang, Huina Dong, and Dawei Zhang. A hybrid rna-protein biosensor for high-throughput screening of adenosylcobalamin biosynthesis. <em>SYNTHETIC AND SYSTEMS BIOTECHNOLOGY</em>, 9(3):513–521, SEP 2024. <a class="reference external" href="https://doi.org/10.1016/j.synbio.2024.04.008">[DOI]</a>.</p></li>
<li id="id6"><p>Aparajitha Srinivasan, Kevin Chen-Xiao, Deepanwita Banerjee, Asun Oka, Venkataramana R. Pidatala, Aymerick Eudes, Blake A. Simmons, Thomas Eng, and Aindrila Mukhopadhyay. Sustainable production of 2,3,5,6-tetramethylpyrazine at high titer in engineered corynebacterium glutamicum. <em>JOURNAL OF INDUSTRIAL MICROBIOLOGY &amp; BIOTECHNOLOGY</em>, AUG 6 2024. <a class="reference external" href="https://doi.org/10.1093/jimb/kuae026">[DOI]</a>.</p></li>
<li id="id7"><p>Elaine Zaunseder, Ulrike Muetze, Jueurorgen G. Okun, Georg F. Hoffmann, Stefan Koelker, Vincent Heuveline, and Ines Thiele. Personalized metabolic whole-body models for newborns and infants predict growth and biomarkers of inherited metabolic diseases. <em>CELL METABOLISM</em>, AUG 6 2024. <a class="reference external" href="https://doi.org/10.1016/j.cmet.2024.05.006">[DOI]</a>.</p></li>
<li id="id8"><p>Anna Kattel, Valter Aro, Petri-Jaan Lahtvee, Jekaterina Kazantseva, Arvi Joers, Ranno Nahku, and Isma Belouah. Exploring the resilience and stability of a defined human gut microbiota consortium: an isothermal microcalorimetric study. <em>MICROBIOLOGYOPEN</em>, AUG 2024. <a class="reference external" href="https://doi.org/10.1002/mbo3.1430">[DOI]</a>.</p></li>
<li id="id9"><p>Jungyeon Kim, Gun-Hwi Yeon, Mi-Jin Kim, Jung-Hoon Bae, Jung-Hoon Sohn, and Bong Hyun Sung. Systems metabolic engineering to elucidate and enhance intestinal metabolic activities of &lt;i&gt;escherichia coli &lt;/i&gt;nissle 1917. <em>JOURNAL OF AGRICULTURAL AND FOOD CHEMISTRY</em>, 72(32):18234–18246, AUG 1 2024. <a class="reference external" href="https://doi.org/10.1021/acs.jafc.4c00182">[DOI]</a>.</p></li>
<li id="id10"><p>Hatice Busra Luleci, Attila Jones, and Tunahan Cakir. Multi-omics analyses highlight molecular differences between clinical and neuropathological diagnoses in alzheimer's disease. <em>EUROPEAN JOURNAL OF NEUROSCIENCE</em>, 2024 JUL 28 2024. <a class="reference external" href="https://doi.org/10.1111/ejn.16482">[DOI]</a>.</p></li>
<li id="id11"><p>Hongzhong Lu, Luchi Xiao, Wenbin Liao, Xuefeng Yan, and Jens Nielsen. Cell factory design with advanced metabolic modelling empowered by artificial intelligence. <em>METABOLIC ENGINEERING</em>, 85:61–72, SEP 2024. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2024.07.003">[DOI]</a>.</p></li>
<li id="id12"><p>N. Suhas Jagannathan, Javier Yu Peng Koh, Younghwan Lee, Radoslaw Mikolaj Sobota, Aaron T. Irving, Lin-fa Wang, Yoko Itahana, Koji Itahana, and Lisa Tucker-Kellogg. Multi-omic analysis of bat versus human fibroblasts reveals altered central metabolism. <em>ELIFE</em>, JUL 22 2024. <a class="reference external" href="https://doi.org/10.7554/eLife.94007; 10.7554/eLife.94007.sa1; 10.7554/eLife.94007.sa2">[DOI]</a>.</p></li>
<li id="id13"><p>Marina Maurizio, Maria Masid, Kerry Woods, Reto Caldelari, John G. Doench, Arunasalam Naguleswaran, Denis Joly, Martin Gonzalez-Fernandez, Jonas Zemp, Melanie Borteele, Vassily Hatzimanikatis, Volker Heussler, Sven Rottenberg, and Philipp Olias. Host cell crispr genomics and modelling reveal shared metabolic vulnerabilities in the intracellular development of &lt;i&gt;plasmodium falciparum&lt;/i&gt; and related hemoparasites. <em>NATURE COMMUNICATIONS</em>, JUL 21 2024. <a class="reference external" href="https://doi.org/10.1038/s41467-024-50405-x">[DOI]</a>.</p></li>
<li id="id14"><p>Indumathi Palanikumar, Himanshu Sinha, and Karthik Raman. Panera: an innovative framework for surmounting uncertainty in microbial community modeling using pan-genera metabolic models. <em>ISCIENCE</em>, JUL 19 2024. <a class="reference external" href="https://doi.org/10.1016/j.isci.2024.110358">[DOI]</a>.</p></li>
<li id="id15"><p>Manish Kumar, Juan D. Tibocha-Bonilla, Zoltan Fussy, Chloe Lieng, Sarah M. Schwenck, Alice V. Levesque, Mahmoud M. Al-Bassam, Anurag Passi, Maxwell Neal, Cristal Zuniga, Farrah Kaiyom, Josh L. Espinoza, Hyungyu Lim, Shawn W. Polson, Lisa Zeigler Allen, and Karsten Zengler. Mixotrophic growth of a ubiquitous marine diatom. <em>SCIENCE ADVANCES</em>, JUL 17 2024. <a class="reference external" href="https://doi.org/10.1126/sciadv.ado2623">[DOI]</a>.</p></li>
<li id="id16"><p>Richard D. Paul, Johann F. Jadebeck, Anton Stratmann, Wolfgang Wiechert, and Katharina Noeh. Hopsy - a methods marketplace for convex polytope sampling in python. <em>BIOINFORMATICS</em>, JUL 12 2024. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btae430">[DOI]</a>.</p></li>
<li id="id17"><p>Pritam Kundu, Satyajit Beura, Suman Mondal, Amit Kumar Das, and Amit Ghosh. Machine learning for the advancement of genome-scale metabolic modeling. <em>BIOTECHNOLOGY ADVANCES</em>, SEP 2024. <a class="reference external" href="https://doi.org/10.1016/j.biotechadv.2024.108400">[DOI]</a>.</p></li>
<li id="id18"><p>Cecilia Noecker and Peter J. Turnbaugh. Emerging tools and best practices for studying gut microbial community metabolism. <em>NATURE METABOLISM</em>, 6(7):1225–1236, JUL 2024. <a class="reference external" href="https://doi.org/10.1038/s42255-024-01074-z">[DOI]</a>.</p></li>
<li id="id19"><p>Jesse Bucksot, Katherine Ritchie, Matthew Biancalana, John A. Cole, and Daniel Cook. Pan-cancer, genome-scale metabolic network analysis of over 10,000 patients elucidates relationship between metabolism and survival. <em>CANCERS</em>, JUL 2024. <a class="reference external" href="https://doi.org/10.3390/cancers16132302">[DOI]</a>.</p></li>
<li id="id20"><p>Nayeon Kim, Junyeong Ma, Wonjong Kim, Jungyeon Kim, Peter Belenky, and Insuk Lee. Genome-resolved metagenomics: a game changer for microbiome medicine. <em>EXPERIMENTAL AND MOLECULAR MEDICINE</em>, 56(7):1501–1512, JUL 2024. <a class="reference external" href="https://doi.org/10.1038/s12276-024-01262-7">[DOI]</a>.</p></li>
<li id="id21"><p>Wheaton L. Schroeder, Patrick F. Suthers, Thomas C. Willis, Eric J. Mooney, and Costas D. Maranas. Current state, challenges, and opportunities in genome-scale resource allocation models: a mathematical perspective. <em>METABOLITES</em>, JUL 2024. <a class="reference external" href="https://doi.org/10.3390/metabo14070365">[DOI]</a>.</p></li>
<li id="id22"><p>Xi Luo, Yanjun Liu, Alexander Balck, Christine Klein, and Ronan M. T. Fleming. Identification of metabolites reproducibly associated with parkinson's disease via meta-analysis and computational modelling. <em>NPJ PARKINSONS DISEASE</em>, JUN 29 2024. <a class="reference external" href="https://doi.org/10.1038/s41531-024-00732-z">[DOI]</a>.</p></li>
<li id="id23"><p>Lu Liu, Weihe Rong, Xiang Du, Qianqian Yuan, Zhaoyu Xu, Chang Yu, Hongzhong Lu, Yanfei Wang, Yan Zhu, Zhijia Liu, and Guokun Wang. Integrating experimental and computational analyses of yeast protein profiles for optimizing the production of high-quality microbial proteins. <em>APPLIED BIOCHEMISTRY AND BIOTECHNOLOGY</em>, 2024 JUN 26 2024. <a class="reference external" href="https://doi.org/10.1007/s12010-024-04995-3">[DOI]</a>.</p></li>
<li id="id24"><p>Carlos J. Rodriguez-Flores, Naroa Barrena, Danel Olaverri-Mendizabal, Idoia Ochoa, Luis V. Valcarcel, and Francisco J. Planes. Gmcspy: efficient and accurate computation of genetic minimal cut sets in python. <em>BIOINFORMATICS</em>, JUN 25 2024. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btae318">[DOI]</a>.</p></li>
<li id="id25"><p>Magdalena Wutkowska, Vojtech Tlaskal, Sergio Bordel, Lisa Y. Stein, Justus Amuche Nweze, and Anne Daebeler. Leveraging genome-scale metabolic models to understand aerobic methanotrophs. <em>ISME JOURNAL</em>, JUN 24 2024. <a class="reference external" href="https://doi.org/10.1093/ismejo/wrae102">[DOI]</a>.</p></li>
<li id="id26"><p>Lizhan Zhang, Jian-Wen Ye, Gang Li, Helen Park, Hao Luo, Yina Lin, Shaowei Li, Weinan Yang, Yuying Guan, Fuqing Wu, Wuzhe Huang, Qiong Wu, Nigel S. Scrutton, Jens Nielsen, and Guo-Qiang Chen. A long-term growth stable halomonas sp. deleted with multiple transposases guided by its metabolic network model halo-ecgem. <em>METABOLIC ENGINEERING</em>, 84:95–108, JUL 2024. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2024.06.004">[DOI]</a>.</p></li>
<li id="id27"><p>Fatima Zulqarnain, Xueheng Zhao, Kenneth D. R. Setchell, Yash Sharma, Phillip Fernandes, Sanjana Srivastava, Aman Shrivastava, Lubaina Ehsan, Varun Jain, Shyam Raghavan, Christopher Moskaluk, Yael Haberman, Lee A. Denson, Khyati Mehta, Najeeha T. Iqbal, Najeeb Rahman, Kamran Sadiq, Zubair Ahmad, Romana Idress, Junaid Iqbal, Sheraz Ahmed, Aneeta Hotwani, Fayyaz Umrani, Beatrice Amadi, Paul Kelly, Donald E. Brown, Sean R. Moore, Syed Asad Ali, and Sana Syed. Machine-learning-based integrative -'omics analyses reveal immunologic and metabolic dysregulation in environmental enteric dysfunction. <em>ISCIENCE</em>, JUN 21 2024. <a class="reference external" href="https://doi.org/10.1016/j.isci.2024.110013">[DOI]</a>.</p></li>
<li id="id28"><p>Emanuel Lange, Lena Kranert, Jacob Kruger, Dirk Benndorf, and Robert Heyer. Microbiome modeling: a beginner's guide. <em>FRONTIERS IN MICROBIOLOGY</em>, JUN 19 2024. <a class="reference external" href="https://doi.org/10.3389/fmicb.2024.1368377">[DOI]</a>.</p></li>
<li id="id29"><p>Zixin Yang, Jinlin Zhu, Wenwei Lu, Fengwei Tian, Hao Zhang, and Wei Chen. Integrating artificial intelligence in exploring multiscale gut microbiota and diet relations for health promotion: a comprehensive review. <em>FOOD BIOSCIENCE</em>, OCT 2024. <a class="reference external" href="https://doi.org/10.1016/j.fbio.2024.104541">[DOI]</a>.</p></li>
<li id="id30"><p>Chaimaa Tarzi, Guido Zampieri, Neil Sullivan, and Claudio Angione. Emerging methods for genome-scale metabolic modeling of microbial communities. <em>TRENDS IN ENDOCRINOLOGY AND METABOLISM</em>, 35(6):533–548, JUN 2024. <a class="reference external" href="https://doi.org/10.1016/j.tem.2024.02.018">[DOI]</a>.</p></li>
<li id="id31"><p>Soraya Mirzaei and Mojtaba Tefagh. Gem-based computational modeling for exploring metabolic interactions in a microbial community. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUN 2024. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1012233">[DOI]</a>.</p></li>
<li id="id32"><p>Lorenzo Pasotti, Ilaria Massaiu, Paolo Magni, and Cinzia Calvio. Metabolic engineering of &lt;i&gt;bacillus subtilis&lt;/i&gt; for the production of poly-γ-glutamic acid from glycerol feedstock. <em>FERMENTATION-BASEL</em>, JUN 2024. <a class="reference external" href="https://doi.org/10.3390/fermentation10060319">[DOI]</a>.</p></li>
<li id="id33"><p>Zhepu Ruan, Kai Chen, Weimiao Cao, Lei Meng, Bingang Yang, Mengjun Xu, Youwen Xing, Pengfa Li, Shiri Freilich, Chen Chen, Yanzheng Gao, Jiandong Jiang, and Xihui Xu. Engineering natural microbiomes toward enhanced bioremediation by microbiome modeling. <em>NATURE COMMUNICATIONS</em>, JUN 1 2024. <a class="reference external" href="https://doi.org/10.1038/s41467-024-49098-z">[DOI]</a>.</p></li>
<li id="id34"><p>Josh A. Taylor, Alain Rapaport, and Denis Dochain. Convex representation of metabolic networks with michaelis-menten kinetics. <em>BULLETIN OF MATHEMATICAL BIOLOGY</em>, JUN 2024. <a class="reference external" href="https://doi.org/10.1007/s11538-024-01293-1">[DOI]</a>.</p></li>
<li id="id35"><p>Priyanka Choudhury, Sanjukta Dasgupta, Parthasarathi Bhattacharyya, Sushmita Roychowdhury, and Koel Chaudhury. Understanding pulmonary hypertension: the need for an integrative metabolomics and transcriptomics approach. <em>MOLECULAR OMICS</em>, 20(6):366–389, JUL 8 2024. <a class="reference external" href="https://doi.org/10.1039/d3mo00266g">[DOI]</a>.</p></li>
<li id="id36"><p>Shengbo Wu, Zheping Qu, Danlei Chen, Hao Wu, Qinggele Caiyin, and Jianjun Qiao. Deciphering and designing microbial communities by genome-scale metabolic modelling. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 23:1990–2000, DEC 2024. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2024.04.055">[DOI]</a>.</p></li>
<li id="id37"><p>Aswathy K. Raghu, Indumathi Palanikumar, and Karthik Raman. Designing function-specific minimal microbiomes from large microbial communities. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, MAY 3 2024. <a class="reference external" href="https://doi.org/10.1038/s41540-024-00373-1">[DOI]</a>.</p></li>
<li id="id38"><p>Telmo Blasco, Francesco Balzerani, Luis V. Valcarcel, Pedro Larranaga, Concha Bielza, Maria Pilar Francino, Jose Angel Rufian-Henares, Francisco J. Planes, and Sergio Perez-Burillo. Bn-bacarena: bayesian network extension of bacarena for the dynamic simulation of microbial communities. <em>BIOINFORMATICS</em>, MAY 2 2024. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btae266">[DOI]</a>.</p></li>
<li id="id39"><p>Ali Kishk, Maria Pires Pacheco, Tony Heurtaux, and Thomas Sauter. Metabolic models predict fotemustine and the combination of eflornithine/rifamycin and adapalene/cannabidiol for the treatment of gliomas. <em>BRIEFINGS IN BIOINFORMATICS</em>, MAY 2 2024. <a class="reference external" href="https://doi.org/10.1093/bib/bbae199">[DOI]</a>.</p></li>
<li id="id40"><p>Fernando Silva-Lance, Isabel Montejano-Montelongo, Eric Bautista, Lars K. Nielsen, Par I. Johansson, and Igor Marin de Mas. Integrating genome-scale metabolic models with patient plasma metabolome to study endothelial metabolism in situ. <em>INTERNATIONAL JOURNAL OF MOLECULAR SCIENCES</em>, MAY 2024. <a class="reference external" href="https://doi.org/10.3390/ijms25105406">[DOI]</a>.</p></li>
<li id="id41"><p>Merve Atasoy, William T. Scott Jr, Alberte Regueira, Miguel Mauricio-Iglesias, Peter J. Schaap, and Hauke Smidt. Biobased short chain fatty acid production - exploring microbial community dynamics and metabolic networks through kinetic and microbial modeling approaches. <em>BIOTECHNOLOGY ADVANCES</em>, JUL-AUG 2024. <a class="reference external" href="https://doi.org/10.1016/j.biotechadv.2024.108363">[DOI]</a>.</p></li>
<li id="id42"><p>Tong Hao, Zhentao Song, Mingzhi Zhang, Lingrui Zhang, Jiarui Yang, Jingjing Li, and Jinsheng Sun. Reconstruction of metabolic-protein interaction integrated network of &lt;i&gt;eriocheir sinensis&lt;/i&gt; and analysis of ecdysone synthesis. <em>GENES</em>, APR 2024. <a class="reference external" href="https://doi.org/10.3390/genes15040410">[DOI]</a>.</p></li>
<li id="id43"><p>Danel Olaverri-Mendizabal, Luis V. Valcarcel, Naroa Barrena, Carlos J. Rodriguez, and Francisco J. Planes. Review and meta-analysis of the genetic minimal cut set approach for gene essentiality prediction in cancer metabolism. <em>BRIEFINGS IN BIOINFORMATICS</em>, MAR 27 2024. <a class="reference external" href="https://doi.org/10.1093/bib/bbae115">[DOI]</a>.</p></li>
<li id="id44"><p>Pavel Dvorak, Barbora Buryskova, Barbora Popelarova, Birgitta E. Ebert, Tibor Botka, Dalimil Bujdos, Alberto Sanchez-Pascuala, Hannah Schoettler, Heiko Hayen, Victor de Lorenzo, Lars M. Blank, and Martin Benesik. Synthetically-primed adaptation of &lt;i&gt;pseudomonas putida&lt;/i&gt; to a non-native substrate d-xylose. <em>NATURE COMMUNICATIONS</em>, MAR 26 2024. <a class="reference external" href="https://doi.org/10.1038/s41467-024-46812-9">[DOI]</a>.</p></li>
<li id="id45"><p>Zhijin Gong, Jiayao Chen, Xinyu Jiao, Hao Gong, Danzi Pan, Lingli Liu, Yang Zhang, and Tianwei Tan. Genome-scale metabolic network models for industrial microorganisms metabolic engineering: current advances and future prospects. <em>BIOTECHNOLOGY ADVANCES</em>, MAY-JUN 2024. <a class="reference external" href="https://doi.org/10.1016/j.biotechadv.2024.108319">[DOI]</a>.</p></li>
<li id="id46"><p>Anqiang Ye, Jian-Ning Shen, Yong Li, Xiang Lian, Bin-Guang Ma, and Feng-Biao Guo. Reconstruction of the genome-scale metabolic network model of &lt;i&gt;sinorhizobium fredii&lt;/i&gt; ccbau45436 for free-living and symbiotic states. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, MAR 25 2024. <a class="reference external" href="https://doi.org/10.3389/fbioe.2024.1377334">[DOI]</a>.</p></li>
<li id="id47"><p>Maria-Tsampika Manoli, Alvaro Gargantilla-Becerra, Carlos del Cerro Sanchez, Virginia Rivero-Buceta, M. Auxiliadora Prieto, and Juan Nogales. A model-driven approach to upcycling recalcitrant feedstocks in pseudomonas putida by decoupling pha production from nutrient limitation. <em>CELL REPORTS</em>, APR 23 2024. <a class="reference external" href="https://doi.org/10.1016/j.celrep.2024.113979">[DOI]</a>.</p></li>
<li id="id48"><p>Amandine Paulay, Ghjuvan M. Grimaud, Raphael Caballero, Beatrice Laroche, Marion Leclerc, Simon Labarthe, and Emmanuelle Maguin. Design of a proteolytic module for improved metabolic modeling of &lt;i&gt;bacteroides caccae&lt;/i&gt;. <em>MSYSTEMS</em>, 2024 MAR 22 2024. <a class="reference external" href="https://doi.org/10.1128/msystems.00153-24">[DOI]</a>.</p></li>
<li id="id49"><p>Mustafa Sertbas and Kutlu O. Ulgen. Uncovering the effect of sars-cov-2 on liver metabolism via genome-scale metabolic modeling for reprogramming and therapeutic strategies. <em>ACS OMEGA</em>, 9(13):15535–15546, MAR 22 2024. <a class="reference external" href="https://doi.org/10.1021/acsomega.4c00392">[DOI]</a>.</p></li>
<li id="id50"><p>Abhirami Thumsi, Diego Martinez, Srivatsan J. Swaminathan, Arezoo Esrafili, Abhirami P. Suresh, Madhan Mohan Chandrasekhar Jaggarapu, Kelly Lintecum, Michelle Halim, Shivani V. Mantri, Yasmine Sleiman, Nicole Appel, Haiwei Gu, Marion Curtis, Cristal Zuniga, and Abhinav P. Acharya. Inverse-vaccines for rheumatoid arthritis re-establish metabolic and immunological homeostasis in joint tissues. <em>ADVANCED HEALTHCARE MATERIALS</em>, 2024 MAR 19 2024. <a class="reference external" href="https://doi.org/10.1002/adhm.202303995">[DOI]</a>.</p></li>
<li id="id51"><p>Carolina Shene, Allison Leyton, Liset Flores, Daniela Chavez, Juan A. Asenjo, and Yusuf Chisti. Genome-scale metabolic modeling of &lt;i&gt;thraustochytrium&lt;/i&gt; sp. rt2316-16: effects of nutrients on metabolism. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 121(6):1986–2001, JUN 2024. <a class="reference external" href="https://doi.org/10.1002/bit.28689">[DOI]</a>.</p></li>
<li id="id52"><p>Anita L. Ziegler, Lena Ullmann, Manuel Bossmann, Karla L. Stein, Ulf W. Liebal, Alexander Mitsos, and Lars M. Blank. Itaconic acid production by co-feeding of &lt;i&gt;ustilago maydis&lt;/i&gt;: a combined approach of experimental data, design of experiments, and metabolic modeling. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 121(6):1846–1858, JUN 2024. <a class="reference external" href="https://doi.org/10.1002/bit.28693">[DOI]</a>.</p></li>
<li id="id53"><p>Marcia R. Couto, Joana L. Rodrigues, Adelaide Braga, Oscar Dias, and Ligia R. Rodrigues. Optimization of chondroitin production in &lt;i&gt;e. coli&lt;/i&gt; using genome scale models. <em>MOLECULAR SYSTEMS DESIGN &amp; ENGINEERING</em>, 9(6):597–611, JUN 4 2024. <a class="reference external" href="https://doi.org/10.1039/d3me00199g">[DOI]</a>.</p></li>
<li id="id54"><p>Filippo Martinelli, Almut Heinken, Ann-Kristin Henning, Maria A. Ulmer, Tim Hensen, Antonio Gonzalez, Matthias Arnold, Sanjay Asthana, Kathrin Budde, Corinne D. Engelman, Mehrbod Estaki, Hans-Joergen Grabe, Margo B. Heston, Sterling Johnson, Gabi Kastenmueller, Cameron Martino, Daniel Mcdonald, Federico E. Rey, Ingo Kilimann, Olive Peters, Xiao Wang, Eike Jakob Spruth, Anja Schneider, Klaus Fliessbach, Jens Wiltfang, Niels Hansen, Wenzel Glanz, Katharina Buerger, Daniel Janowitz, Christoph Laske, Matthias H. Munk, Annika Spottke, Nina Roy, Matthias Nauck, Stefan Teipel, Rob Knight, Rima F. Kaddurah-Daouk, Barbara B. Bendlin, Johannes Hertel, and Ines Thiele. Whole-body metabolic modelling reveals microbiome and genomic interactions on reduced urine formate levels in alzheimer's disease. <em>SCIENTIFIC REPORTS</em>, MAR 13 2024. <a class="reference external" href="https://doi.org/10.1038/s41598-024-55960-3">[DOI]</a>.</p></li>
<li id="id55"><p>Alvaro R. Lara, Flavio Kunert, Vincent Vandenbroucke, Hilal Taymaz-Nikerel, Luz Maria Martinez, Juan-Carlos Sigala, Frank Delvigne, Guillermo Gosset, and Jochen Buechs. Transport-controlled growth decoupling for self-induced protein expression with a glycerol-repressible genetic circuit. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 121(6):1789–1802, JUN 2024. <a class="reference external" href="https://doi.org/10.1002/bit.28697">[DOI]</a>.</p></li>
<li id="id56"><p>Fernando Cruz, Joao Capela, Eugenio Campos Ferreira, Miguel Rocha, and Oscar Dias. Bioiso: an objective-oriented application for assisting the curation of genome-scale metabolic models. <em>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS</em>, 21(2):215–226, MAR-APR 2024. <a class="reference external" href="https://doi.org/10.1109/TCBB.2023.3339972">[DOI]</a>.</p></li>
<li id="id57"><p>Quinn S. Solfisburg, Federico Baldini, Brittany Baldwin-Hunter, George I. Austin, Harry H. Lee, Heekuk Park, Daniel E. Freedberg, Charles J. Lightdale, Tal Korem, and Julian A. Abrams. The salivary microbiome and predicted metabolite production are associated with barrett's esophagus and high-grade dysplasia or adenocarcinoma. <em>CANCER EPIDEMIOLOGY BIOMARKERS &amp; PREVENTION</em>, 33(3):371–380, MAR 1 2024. <a class="reference external" href="https://doi.org/10.1158/1055-9965.EPI-23-0652">[DOI]</a>.</p></li>
<li id="id58"><p>Panyawarin Soommat, Nachon Raethong, Ratchaprapa Ruengsang, Roypim Thananusak, Teeraphan Laomettachit, Kobkul Laoteng, Treenut Saithong, and Wanwipa Vongsangnak. Light-exposed metabolic responses of &lt;i&gt;cordyceps militaris&lt;/i&gt; through transcriptome-integrated genome-scale modeling. <em>BIOLOGY-BASEL</em>, MAR 2024. <a class="reference external" href="https://doi.org/10.3390/biology13030139">[DOI]</a>.</p></li>
<li id="id59"><p>Debiao Wu, Feng Xu, Yaying Xu, Mingzhi Huang, Zhimin Li, and Ju Chu. Towards a hybrid model-driven platform based on flux balance analysis and a machine learning pipeline for biosystem design. <em>SYNTHETIC AND SYSTEMS BIOTECHNOLOGY</em>, 9(1):33–42, MAR 2024. <a class="reference external" href="https://doi.org/10.1016/j.synbio.2023.12.004">[DOI]</a>.</p></li>
<li id="id60"><p>Meng Luo, Jinlin Zhu, Jiajia Jia, Hao Zhang, and Jianxin Zhao. Progress on network modeling and analysis of gut microecology: a review. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, MAR 20 2024. <a class="reference external" href="https://doi.org/10.1128/aem.00092-24">[DOI]</a>.</p></li>
<li id="id61"><p>Deepanwita Banerjee, Ian S. Yunus, Xi Wang, Jinho Kim, Aparajitha Srinivasan, Russel Menchavez, Yan Chen, Jennifer W. Gin, Christopher J. Petzold, Hector Garcia Martin, Jon K. Magnuson, Paul D. Adams, Blake A. Simmons, Aindrila Mukhopadhyay, Joonhoon Kim, and Taek Soon Lee. Genome-scale and pathway engineering for the sustainable aviation fuel precursor isoprenol production in&lt;i&gt; pseudomonas&lt;/i&gt;&lt;i&gt; putida&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 82:157–170, MAR 2024. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2024.02.004">[DOI]</a>.</p></li>
<li id="id62"><p>Xavier Marbehan, Magali Roger, Frantz Fournier, Pascale Infossi, Emmanuel Guedon, Louis Delecourt, Regine Lebrun, Marie-Therese Giudici-Orticoni, and Stephane Delaunay. Combining metabolic flux analysis with proteomics to shed light on the metabolic flexibility: the case of &lt;i&gt;desulfovibrio vulgaris&lt;/i&gt; hildenborough. <em>FRONTIERS IN MICROBIOLOGY</em>, FEB 23 2024. <a class="reference external" href="https://doi.org/10.3389/fmicb.2024.1336360">[DOI]</a>.</p></li>
<li id="id63"><p>Ning Qin, Lingyun Li, Xiaozhen Wan, Xu Ji, Yu Chen, Chaokun Li, Ping Liu, Yijie Zhang, Weijie Yang, Junfeng Jiang, Jianye Xia, Shuobo Shi, Tianwei Tan, Jens Nielsen, Yun Chen, and Zihe Liu. Increased co&lt;sub&gt;2&lt;/sub&gt; fixation enables high carbon-yield production of 3-hydroxypropionic acid in yeast. <em>NATURE COMMUNICATIONS</em>, FEB 21 2024. <a class="reference external" href="https://doi.org/10.1038/s41467-024-45557-9">[DOI]</a>.</p></li>
<li id="id64"><p>Loreto Roman, Felipe Melis-Arcos, Tomas Proschle, Pedro A. Saa, and Daniel Garrido. Genome-scale metabolic modeling of the human milk oligosaccharide utilization by &lt;i&gt;bifidobacterium longum&lt;/i&gt; subsp. &lt;i&gt;infantis&lt;/i&gt;. <em>MSYSTEMS</em>, MAR 19 2024. <a class="reference external" href="https://doi.org/10.1128/msystems.00715-23">[DOI]</a>.</p></li>
<li id="id65"><p>Yu Chen, Johan Gustafsson, Jingyu Yang, Jens Nielsen, and Eduard J. Kerkhoven. Single-cell omics analysis with genome-scale metabolic modeling. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, APR 2024. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2024.103078">[DOI]</a>.</p></li>
<li id="id66"><p>David M. Versluis, Ruud Schoemaker, Ellen Looijesteijn, Jan M. W. Geurts, and Roeland M. H. Merks. 2'-fucosyllactose helps butyrate producers outgrow competitors in infant gut microbiota simulations. <em>ISCIENCE</em>, MAR 15 2024. <a class="reference external" href="https://doi.org/10.1016/j.isci.2024.109085">[DOI]</a>.</p></li>
<li id="id67"><p>Eduardo Luis Menezes de Almeida, Eduard J. Kerkhoven, and Wendel Batista da Silveira. Reconstruction of genome-scale metabolic models of non-conventional yeasts: current state, challenges, and perspectives. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 29(1):177–183, FEB 2024. <a class="reference external" href="https://doi.org/10.1007/s12257-024-00009-5">[DOI]</a>.</p></li>
<li id="id68"><p>Chenguang Liang, Sue Murray, Yang Li, Richard Lee, Audrey Low, Shruti Sasaki, Austin W. T. Chiang, Wen -Jen Lin, Joel Mathews, Will Barnes, and Nathan E. Lewis. Lipidsim: inferring mechanistic lipid biosynthesis perturbations from lipidomics with a flexible, low-parameter, markov modeling framework. <em>METABOLIC ENGINEERING</em>, 82:110–122, MAR 2024. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2024.01.004">[DOI]</a>.</p></li>
<li id="id69"><p>Zachary M. Burcham, Aeriel D. Belk, Bridget B. Mcgivern, Amina Bouslimani, Parsa Ghadermazi, Cameron Martino, Liat Shenhav, Anru R. Zhang, Pixu Shi, Alexandra Emmons, Heather L. Deel, Zhenjiang Zech Xu, Victoria Nieciecki, Qiyun Zhu, Michael Shaffer, Morgan Panitchpakdi, Kelly C. Weldon, Kalen Cantrell, Asa Ben-Hur, Sasha C. Reed, Greg C. Humphry, Gail Ackermann, Daniel Mcdonald, Siu Hung Joshua Chan, Melissa Connor, Derek Boyd, Jake Smith, Jenna M. S. Watson, Giovanna Vidoli, Dawnie Steadman, Aaron M. Lynne, Sibyl Bucheli, Pieter C. Dorrestein, Kelly C. Wrighton, David O. Carter, Rob Knight, and Jessica L. Metcalf. A conserved interdomain microbial network underpins cadaver decomposition despite environmental variables. <em>NATURE MICROBIOLOGY</em>, MAR 2024. <a class="reference external" href="https://doi.org/10.1038/s41564-023-01580-y">[DOI]</a>.</p></li>
<li id="id70"><p>Aruldoss Immanuel, Ragothaman M. Yennamalli, and Venkatasubramanian Ulaganathan. Targeting the bottlenecks in levan biosynthesis pathway in &lt;i&gt;bacillus subtilis&lt;/i&gt; and strain optimization by computational modeling and omics integration. <em>OMICS-A JOURNAL OF INTEGRATIVE BIOLOGY</em>, 28(2):49–58, FEB 1 2024. <a class="reference external" href="https://doi.org/10.1089/omi.2023.0277">[DOI]</a>.</p></li>
<li id="id71"><p>Gustav Sjoberg, Alina Rekena, Matilda Fornstad, Petri-Jaan Lahtvee, and Antonius J. A. van Maris. Evaluation of enzyme-constrained genome-scale model through metabolic engineering of anaerobic co-production of 2,3-butanediol and glycerol by &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 82:49–59, MAR 2024. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2024.01.007">[DOI]</a>.</p></li>
<li id="id72"><p>Egeria Scoditti, Silvia Sabatini, Fabrizia Carli, and Amalia Gastaldelli. Hepatic glucose metabolism in the steatotic liver. <em>NATURE REVIEWS GASTROENTEROLOGY &amp; HEPATOLOGY</em>, 21(5):319–334, MAY 2024. <a class="reference external" href="https://doi.org/10.1038/s41575-023-00888-8">[DOI]</a>.</p></li>
<li id="id73"><p>Naveed Ziari and Marc K. Hellerstein. Pymida: a graphical user interface for mass isotopomer distribution analysis. <em>ANALYTICAL CHEMISTRY</em>, 96(6):2303–2308, FEB 2 2024. <a class="reference external" href="https://doi.org/10.1021/acs.analchem.3c02211">[DOI]</a>.</p></li>
<li id="id74"><p>Freyr Johannsson, James T. Yurkovich, Steinn Guomundsson, Olafur E. Sigurjonsson, and Ottar Rolfsson. Temperature dependence of platelet metabolism. <em>METABOLITES</em>, FEB 2024. <a class="reference external" href="https://doi.org/10.3390/metabo14020091">[DOI]</a>.</p></li>
<li id="id75"><p>Bruno G. Galuzzi, Luca Milazzo, and Chiara Damiani. Adjusting for false discoveries in constraint-based differential metabolic flux analysis. <em>JOURNAL OF BIOMEDICAL INFORMATICS</em>, FEB 2024. <a class="reference external" href="https://doi.org/10.1016/j.jbi.2024.104597">[DOI]</a>.</p></li>
<li id="id76"><p>Leila Hassani, Mohammad R. Moosavi, Payam Setoodeh, and Habil Zare. Fastknock: an efficient next-generation approach to identify all knockout strategies for strain optimization. <em>MICROBIAL CELL FACTORIES</em>, JAN 29 2024. <a class="reference external" href="https://doi.org/10.1186/s12934-023-02277-x">[DOI]</a>.</p></li>
<li id="id77"><p>Sonia Alcala, Lara Villarino, Laura Ruiz-Canas, Jose R. Couceiro, Miguel Martinez-Calvo, Adrian Palencia-Campos, Diego Navarro, Pablo Cabezas-Sainz, Iker Rodriguez-Arabaolaza, Alfonso Cordero-Barreal, Lucia Trilla-Fuertes, Juan A. Rubiolo, Sandra Batres-Ramos, Mireia Vallespinos, Cristina Gonzalez-Paramos, Jessica Rodriguez, Angelo Gamez-Pozo, Juan angel Fresno Vara, Sara Fra Fernandez, Amparo Benito Berlinches, Nicolas Moreno-Mata, Ana Maria Torres Redondo, Alfredo Carrato, Patrick C. Hermann, Laura Sanchez, Susana Torrente, Miguel Angel Fernandez-Moreno, Jose L. Mascarenas, and Bruno Sainz Jr. Targeting cancer stem cell oxphos with tailored ruthenium complexes as a new anti-cancer strategy. <em>JOURNAL OF EXPERIMENTAL &amp; CLINICAL CANCER RESEARCH</em>, JAN 27 2024. <a class="reference external" href="https://doi.org/10.1186/s13046-023-02931-7">[DOI]</a>.</p></li>
<li id="id78"><p>Eliott Jacopin, Yuki Sakamoto, Kozo Nishida, Kazunari Kaizu, and Koichi Takahashi. An architecture for collaboration in systems biology at the age of the metaverse. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, JAN 27 2024. <a class="reference external" href="https://doi.org/10.1038/s41540-024-00334-8">[DOI]</a>.</p></li>
<li id="id80"><p>Mahnoor Zulfiqar, Vinay Singh, Christoph Steinbeck, and Maria Sorokina. Review on computer-assisted biosynthetic capacities elucidation to assess metabolic interactions and communication within microbial communities. <em>CRITICAL REVIEWS IN MICROBIOLOGY</em>, 2024 JAN 24 2024. <a class="reference external" href="https://doi.org/10.1080/1040841X.2024.2306465">[DOI]</a>.</p></li>
<li id="id81"><p>Clemence Joseph, Haris Zafeiropoulos, Kristel Bernaerts, and Karoline Faust. Predicting microbial interactions with approaches based on flux balance analysis: an evaluation. <em>BMC BIOINFORMATICS</em>, JAN 23 2024. <a class="reference external" href="https://doi.org/10.1186/s12859-024-05651-7">[DOI]</a>.</p></li>
<li id="id82"><p>Klara Cerk, Pablo Ugalde-Salas, Chabname Ghassemi Nedjad, Maxime Lecomte, Coralie Muller, David J. Sherman, Falk Hildebrand, Simon Labarthe, and Clemence Frioux. Community-scale models of microbiomes: articulating metabolic modelling and metagenome sequencing. <em>MICROBIAL BIOTECHNOLOGY</em>, 2024. <a class="reference external" href="https://doi.org/10.1111/1751-7915.14396">[DOI]</a>.</p></li>
<li id="id83"><p>Yu Chen, Johan Gustafsson, Albert Tafur Rangel, Mihail Anton, Ivan Domenzain, Cheewin Kittikunapong, Feiran Li, Le Yuan, Jens Nielsen, and Eduard J. Kerkhoven. Reconstruction, simulation and analysis of enzyme-constrained metabolic models using gecko toolbox 3.0. <em>NATURE PROTOCOLS</em>, 19(3):629–667, MAR 2024. <a class="reference external" href="https://doi.org/10.1038/s41596-023-00931-7">[DOI]</a>.</p></li>
<li id="id84"><p>Himel Nahreen Khaleque, Hadi Nazem-Bokaee, Yosephine Gumulya, Ross P. Carlson, and Anna H. Kaksonen. Simulating compatible solute biosynthesis using a metabolic flux model of the biomining acidophile,&lt;i&gt; acidithiobacillus&lt;/i&gt;&lt;i&gt; ferrooxidans&lt;/i&gt; atcc 23270. <em>RESEARCH IN MICROBIOLOGY</em>, JAN-FEB 2024. <a class="reference external" href="https://doi.org/10.1016/j.resmic.2023.104115">[DOI]</a>.</p></li>
<li id="id85"><p>Sol Han, Dohyeon Kim, Youngshin Kim, and Sung Ho Yoon. Genome-scale metabolic network model and phenome of solvent-tolerant &lt;i&gt;pseudomonas putida&lt;/i&gt; s12. <em>BMC GENOMICS</em>, JAN 16 2024. <a class="reference external" href="https://doi.org/10.1186/s12864-023-09940-y">[DOI]</a>.</p></li>
<li id="id86"><p>Pedro A. Saa, Sebastian Zapararte, Christopher C. Drovandi, and Lars K. Nielsen. Looplessfluxsampler: an efficient toolbox for sampling the loopless flux solution space of metabolic models. <em>BMC BIOINFORMATICS</em>, JAN 2 2024. <a class="reference external" href="https://doi.org/10.1186/s12859-023-05616-2">[DOI]</a>.</p></li>
<li id="id87"><p>Andrea Angarita-Rodriguez, Yeimy Gonzalez-Giraldo, Juan J. Rubio-Mesa, Andres Felipe Aristizabal, Andres Pinzon, Janneth Gonzalez, and King-Ho Cheung. Control theory and systems biology: potential applications in neurodegeneration and search for therapeutic targets. <em>INTERNATIONAL JOURNAL OF MOLECULAR SCIENCES</em>, JAN 2024. <a class="reference external" href="https://doi.org/10.3390/ijms25010365">[DOI]</a>.</p></li>
<li id="id88"><p>Elena Lucy Carter, Chrystala Constantinidou, and Mohammad Tauqeer Alam. Applications of genome-scale metabolic models to investigate microbial metabolic adaptations in response to genetic or environmental perturbations. <em>BRIEFINGS IN BIOINFORMATICS</em>, JAN 2024. <a class="reference external" href="https://doi.org/10.1093/bib/bbad439">[DOI]</a>.</p></li>
<li id="id89"><p>Mitzi de la Cruz, Flavio Kunert, Hilal Taymaz-Nikerel, Juan-Carlos Sigala, Guillermo Gosset, Jochen Buchs, and Alvaro R. Lara. Increasing the pentose phosphate pathway flux to improve plasmid dna production in engineered &lt;i&gt;e. coli&lt;/i&gt;. <em>MICROORGANISMS</em>, JAN 2024. <a class="reference external" href="https://doi.org/10.3390/microorganisms12010150">[DOI]</a>.</p></li>
<li id="id90"><p>Isaac Klapper, Daniel B. Szyld, Xinli Yu, Karsten Zengler, Tianyu Zhang, and Cristal Zuniga. A domain decomposition method for solution of a pde-constrained generalized nash equilibrium model of biofilm community metabolism. <em>SIAM JOURNAL ON APPLIED MATHEMATICS</em>, 84(1):97–113, 2024. <a class="reference external" href="https://doi.org/10.1137/22M1511023">[DOI]</a>.</p></li>
<li id="id92"><p>Weiren Liu, Huqiang Wang, Qianfu Zhao, Chenyang Tao, Weifeng Qu, Yushan Hou, Run Huang, Zimei Sun, Guiqi Zhu, Xifei Jiang, Yuan Fang, Jun Gao, Xiaoling Wu, Zhixiang Yang, Rongyu Ping, Jiafeng Chen, Rui Yang, Tianhao Chu, Jian Zhou, Jia Fan, Zheng Tang, Dong Yang, and Yinghong Shi. Multiomics analysis reveals metabolic subtypes and identifies diacylglycerol kinase α (dgka) as a potential therapeutic target for intrahepatic cholangiocarcinoma. <em>CANCER COMMUNICATIONS</em>, 44(2):226–250, FEB 2024. <a class="reference external" href="https://doi.org/10.1002/cac2.12513">[DOI]</a>.</p></li>
<li id="id94"><p>Maximilian Luginsland, Cleo Kontoravdi, Andy Racher, Colin Jaques, and Alexandros Kiparissides. Elucidating lactate metabolism in industrial cho cultures through the combined use of flux balance and principal component analyses. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, FEB 2024. <a class="reference external" href="https://doi.org/10.1016/j.bej.2023.109184">[DOI]</a>.</p></li>
<li id="id107"><p>Joshua A. M. Kaste and Yair Shachar-Hill. Model validation and selection in metabolic flux analysis and flux balance analysis. <em>BIOTECHNOLOGY PROGRESS</em>, JAN 2024. <a class="reference external" href="https://doi.org/10.1002/btpr.3413">[DOI]</a>.</p></li>
<li id="id119"><p>Maria Teresita Castaneda, Sebastian Nunez, Martin Jamilis, and Hernan De Battista. Computational assessment of lipid production in &lt;i&gt;rhodosporidium toruloides&lt;/i&gt; in two-stage and one-stage batch bioprocesses. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 121(1):238–249, JAN 2024. <a class="reference external" href="https://doi.org/10.1002/bit.28579">[DOI]</a>.</p></li>
<li id="id123"><p>Chengjie Hou, Xin Song, Zhiqiang Xiong, Guangqiang Wang, Yongjun Xia, and Lianzhong Ai. Genome-scale reconstruction of the metabolic network in &lt;i&gt;streptococcus thermophilus&lt;/i&gt; s-3 and assess urea metabolism. <em>JOURNAL OF THE SCIENCE OF FOOD AND AGRICULTURE</em>, 104(3):1458–1469, FEB 2024. <a class="reference external" href="https://doi.org/10.1002/jsfa.13026">[DOI]</a>.</p></li>
<li id="id154"><p>Dario Neves, Daniel Meinen, Tobias B. B. Alter, Lars M. M. Blank, and Birgitta E. E. Ebert. Expanding &lt;i&gt;pseudomonas taiwanensis&lt;/i&gt; vlb120's acyl-coa portfolio: propionate production in mineral salt medium. <em>MICROBIAL BIOTECHNOLOGY</em>, 2024. <a class="reference external" href="https://doi.org/10.1111/1751-7915.14309">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-1" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-1" name="0-1" role="tabpanel" tabindex="0"><div class="docutils container" id="id1821">
<ol class="arabic simple" start="1">
<li id="id1899"><p>R. Munoz-Tamayo, M. Davoudkhani, I. Fakih, C. E. Robles-Rodriguez, F. Rubino, C. J. Creevey, and E. Forano. Review: towards the next-generation models of the rumen microbiome for enhancing predictive power and guiding sustainable production strategies. <em>ANIMAL</em>, DEC 2023. <a class="reference external" href="https://doi.org/10.1016/j.animal.2023.100984">[DOI]</a>.</p></li>
<li id="id1911"><p>Arianna Basile, Almut Heinken, Johannes Hertel, Larry Smarr, Weizhong Li, Laura Treu, Giorgio Valle, Stefano Campanaro, and Ines Thiele. Longitudinal flux balance analyses of a patient with episodic colonic inflammation reveals microbiome metabolic dynamics. <em>GUT MICROBES</em>, DEC 31 2023. <a class="reference external" href="https://doi.org/10.1080/19490976.2023.2226921">[DOI]</a>.</p></li>
<li id="id1913"><p>Peter J. Gwynne, Kee-Lee K. Stocks, Elysse S. Karozichian, Aarya Pandit, and Linden T. Hu. Metabolic modeling predicts unique drug targets in &lt;i&gt;borrelia burgdorferi&lt;/i&gt;. <em>MSYSTEMS</em>, DEC 21 2023. <a class="reference external" href="https://doi.org/10.1128/msystems.00835-23">[DOI]</a>.</p></li>
<li id="id1915"><p>Thummarat Paklao, Apichat Suratanee, and Kitiporn Plaimas. Icon-gems: integration of co-expression network in genome-scale metabolic models, shedding light through systems biology. <em>BMC BIOINFORMATICS</em>, DEC 21 2023. <a class="reference external" href="https://doi.org/10.1186/s12859-023-05599-0">[DOI]</a>.</p></li>
<li id="id1916"><p>Isabel Rosety, Alise Zagare, Claudia Saraiva, Sarah Nickels, Paul Antony, Catarina Almeida, Enrico Glaab, Rashi Halder, Sergiy Velychko, Thomas Rauen, Hans R. Schoeler, Silvia Bolognin, Thomas Sauter, Javier Jarazo, Rejko Kruger, and Jens C. Schwamborn. Impaired neuron differentiation in gba-associated parkinson's disease is linked to cell cycle defects in organoids. <em>NPJ PARKINSONS DISEASE</em>, DEC 18 2023. <a class="reference external" href="https://doi.org/10.1038/s41531-023-00616-8">[DOI]</a>.</p></li>
<li id="id1917"><p>Dae-Hee Lee, Haseong Kim, Bong-Hyun Sung, Byung Kwan Cho, and Seung-Goo Lee. Biofoundries: bridging automation and biomanufacturing in synthetic biology. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 28(6, SI):892–904, DEC 2023. <a class="reference external" href="https://doi.org/10.1007/s12257-023-0226-x">[DOI]</a>.</p></li>
<li id="id1918"><p>Mikhail A. Kulyashov, Semyon K. Kolmykov, Tamara M. Khlebodarova, and Ilya R. Akberdin. State-of the-art constraint-based modeling of microbial metabolism: from basics to context-specific models with a focus on methanotrophs. <em>MICROORGANISMS</em>, DEC 2023. <a class="reference external" href="https://doi.org/10.3390/microorganisms11122987">[DOI]</a>.</p></li>
<li id="id1919"><p>Aliya Lakhani, Da Hyun Kang, Yea Eun Kang, and Junyoung O. Park. Toward systems-level metabolic analysis in endocrine disorders and cancer. <em>ENDOCRINOLOGY AND METABOLISM</em>, 38(6):619–630, DEC 2023. <a class="reference external" href="https://doi.org/10.3803/EnM.2023.1814">[DOI]</a>.</p></li>
<li id="id1920"><p>Zeqian Li, Ahmed Selim, and Seppe Kuehn. Statistical prediction of microbial metabolic traits from genomes. <em>PLOS COMPUTATIONAL BIOLOGY</em>, DEC 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1011705">[DOI]</a>.</p></li>
<li id="id1921"><p>Jorge Carrasco Muriel, Christopher Long, and Nikolaus Sonnenschein. Simultaneous application of enzyme and thermodynamic constraints to metabolic models using an updated python implementation of gecko. <em>MICROBIOLOGY SPECTRUM</em>, DEC 2023. <a class="reference external" href="https://doi.org/10.1128/spectrum.01705-23">[DOI]</a>.</p></li>
<li id="id1922"><p>Karel Olavarria, Marco V. Becker, Diana Z. Sousa, Mark C. M. van Loosdrecht, and S. Aljoscha Wahl. Design and thermodynamic analysis of a pathway enabling anaerobic production of poly-3-hydroxybutyrate in escherichia coli. <em>SYNTHETIC AND SYSTEMS BIOTECHNOLOGY</em>, 8(4):629–639, DEC 2023. <a class="reference external" href="https://doi.org/10.1016/j.synbio.2023.09.005">[DOI]</a>.</p></li>
<li id="id1923"><p>Egils Stalidzans, Reinis Muiznieks, Konstantins Dubencovs, Elina Sile, Kristaps Berzins, Arturs Suleiko, and Juris Vanags. A fermentation state marker rule design task in metabolic engineering. <em>BIOENGINEERING-BASEL</em>, DEC 2023. <a class="reference external" href="https://doi.org/10.3390/bioengineering10121427">[DOI]</a>.</p></li>
<li id="id1924"><p>Qianqian Yuan, Fan Wei, Xiaogui Deng, Aonan Li, Zhenkun Shi, Zhitao Mao, Feiran Li, and Hongwu Ma. Reconstruction and metabolic profiling of the genome-scale metabolic network model of pseudomonas stutzeri a1501. <em>SYNTHETIC AND SYSTEMS BIOTECHNOLOGY</em>, 8(4):688–696, DEC 2023. <a class="reference external" href="https://doi.org/10.1016/j.synbio.2023.10.001">[DOI]</a>.</p></li>
<li id="id1925"><p>Koray Malci, Rodrigo Santibanez, Nestor Jonguitud-Borrego, Jorge H. Santoyo-Garcia, Eduard J. Kerkhoven, and Leonardo Rios-Solis. Improved production of taxol® precursors in s. cerevisiae using combinatorial in silico design and metabolic engineering. <em>MICROBIAL CELL FACTORIES</em>, NOV 29 2023. <a class="reference external" href="https://doi.org/10.1186/s12934-023-02251-7">[DOI]</a>.</p></li>
<li id="id1926"><p>Hananeh Ahmadpanah, Ehsan Motamedian, and Mohammad Mahdi Mardanpour. Metabolic regulation boosts bioelectricity generation in &lt;i&gt;zymomonas mobilis&lt;/i&gt; microbial fuel cell, surpassing ethanol production. <em>SCIENTIFIC REPORTS</em>, NOV 24 2023. <a class="reference external" href="https://doi.org/10.1038/s41598-023-47846-7">[DOI]</a>.</p></li>
<li id="id1928"><p>Laura Judith Marcos-Zambrano, Victor Manuel Lopez-Molina, Burcu Bakir-Gungor, Marcus Frohme, Kanita Karaduzovic-Hadziabdic, Thomas Klammsteiner, Eliana Ibrahimi, Leo Lahti, Tatjana Loncar-Turukalo, Xhilda Dhamo, Andrea Simeon, Alina Nechyporenko, Gianvito Pio, Piotr Przymus, Alexia Sampri, Vladimir Trajkovik, Blanca Lacruz-Pleguezuelos, Oliver Aasmets, Ricardo Araujo, Ioannis Anagnostopoulos, Onder Aydemir, Magali Berland, M. Luz Calle, Michelangelo Ceci, Hatice Duman, Aycan Gundogdu, Aki S. Havulinna, Kardokh Hama Najib Kaka Bra, Eglantina Kalluci, Sercan Karav, Daniel Lode, Marta B. Lopes, Patrick May, Bram Nap, Miroslava Nedyalkova, Ines Paciencia, Lejla Pasic, Meritxell Pujolassos, Rajesh Shigdel, Antonio Susin, Ines Thiele, Ciprian-Octavian Truica, Paul Wilmes, Ercument Yilmaz, Malik Yousef, Marcus Joakim Claesson, Jaak Truu, Enrique Carrillo de Santa Pau, and ML4Microbiome. A toolbox of machine learning software to support microbiome analysis. <em>FRONTIERS IN MICROBIOLOGY</em>, NOV 22 2023. <a class="reference external" href="https://doi.org/10.3389/fmicb.2023.1250806">[DOI]</a>.</p></li>
<li id="id1929"><p>Mirco Plante. Epistemology of synthetic biology: a new theoretical framework based on its potential objects and objectives. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, NOV 20 2023. <a class="reference external" href="https://doi.org/10.3389/fbioe.2023.1266298">[DOI]</a>.</p></li>
<li id="id1930"><p>Alise Zagare, German Preciat, Sarah. L. Nickels, Xi Luo, Anna S. Monzel, Gemma Gomez-Giro, Graham Robertson, Christian Jaeger, Jafar Sharif, Haruhiko Koseki, Nico J. Diederich, Enrico Glaab, Ronan M. T. Fleming, and Jens C. Schwamborn. Omics data integration suggests a potential idiopathic parkinson's disease signature. <em>COMMUNICATIONS BIOLOGY</em>, NOV 20 2023. <a class="reference external" href="https://doi.org/10.1038/s42003-023-05548-w">[DOI]</a>.</p></li>
<li id="id1931"><p>Silvia Avila-Cabre, Miriam Perez-Trujillo, Joan Albiol, and Pau Ferrer. Engineering the synthetic β-alanine pathway in komagataella phaffii for conversion of methanol into 3-hydroxypropionic acid. <em>MICROBIAL CELL FACTORIES</em>, NOV 17 2023. <a class="reference external" href="https://doi.org/10.1186/s12934-023-02241-9">[DOI]</a>.</p></li>
<li id="id1932"><p>Le Gao, Jiao Meng, Wuling Dai, Zhaokun Zhang, Haofan Dong, Qianqian Yuan, Wuyuan Zhang, Shuguang Liu, and Xin Wu. Deciphering cell wall sensors enabling the construction of robust &lt;i&gt;p. pastoris&lt;/i&gt; for single-cell protein production. <em>BIOTECHNOLOGY FOR BIOFUELS AND BIOPRODUCTS</em>, NOV 17 2023. <a class="reference external" href="https://doi.org/10.1186/s13068-023-02428-7">[DOI]</a>.</p></li>
<li id="id1933"><p>Ehsan Motamedian, Kristaps Berzins, Reinis Muiznieks, and Egils Stalidzans. Optenvelope: a target point guided method for growth-coupled production using knockouts. <em>PLOS ONE</em>, NOV 16 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0294313">[DOI]</a>.</p></li>
<li id="id1934"><p>Won Jun Kim, Youngjoon Lee, Hyun Uk Kim, Jae Yong Ryu, Jung Eun Yang, and Sang Yup Lee. Genome-wide identification of overexpression and downregulation gene targets based on the sum of covariances of the outgoing reaction fluxes. <em>CELL SYSTEMS</em>, 14(11):990+, NOV 15 2023. <a class="reference external" href="https://doi.org/10.1016/j.cels.2023.10.005">[DOI]</a>.</p></li>
<li id="id1935"><p>Ya-Wen Li, Jin-Yi Qian, Jia-Cong Huang, Dong-Sheng Guo, Zhi-Kui Nie, Chao Ye, and Tian-Qiong Shi. Improving gibberellin ga&lt;sub&gt;3&lt;/sub&gt; production with the construction of a genome-scale metabolic model of fusarium fujikuroi. <em>JOURNAL OF AGRICULTURAL AND FOOD CHEMISTRY</em>, 71(48):18890–18897, NOV 6 2023. <a class="reference external" href="https://doi.org/10.1021/acs.jafc.3c05309">[DOI]</a>.</p></li>
<li id="id1936"><p>Jing Yu, Xiaowei Wang, Qianqian Yuan, Jiaxin Shi, Jingyi Cai, Zhichao Li, and Hongwu Ma. Elucidating the impact of &lt;i&gt;in vitro&lt;/i&gt; cultivation on &lt;i&gt;nicotiana tabacum&lt;/i&gt; metabolism through combined &lt;i&gt;in silico&lt;/i&gt; modeling and multiomics analysis. <em>FRONTIERS IN PLANT SCIENCE</em>, NOV 3 2023. <a class="reference external" href="https://doi.org/10.3389/fpls.2023.1281348">[DOI]</a>.</p></li>
<li id="id1937"><p>Takeyuki Tamura. Metnetcomp: database for minimal and maximal gene-deletion strategies for growth-coupled production of genome-scale metabolic networks. <em>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS</em>, 20(6):3748–3758, NOV 2023. <a class="reference external" href="https://doi.org/10.1109/TCBB.2023.3317837">[DOI]</a>.</p></li>
<li id="id1938"><p>Omer F. Bay, Kelly S. Hayes, Jean-Marc Schwartz, Richard K. Grencis, and Ian S. Roberts. A genome-scale metabolic model of parasitic whipworm. <em>NATURE COMMUNICATIONS</em>, OCT 31 2023. <a class="reference external" href="https://doi.org/10.1038/s41467-023-42552-4">[DOI]</a>.</p></li>
<li id="id1940"><p>Miguel Ponce-de-Leon, Arnau Montagud, Vincent Noel, Annika Meert, Gerard Pradas, Emmanuel Barillot, Laurence Calzone, and Alfonso Valencia. Physiboss 2.0: a sustainable integration of stochastic boolean and agent-based modelling frameworks. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, OCT 30 2023. <a class="reference external" href="https://doi.org/10.1038/s41540-023-00314-4">[DOI]</a>.</p></li>
<li id="id1941"><p>Jean-Pierre Mazat. The metabolic control theory: its development and its application to mitochondrial oxidative phosphorylation. <em>BIOSYSTEMS</em>, DEC 2023. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2023.105038">[DOI]</a>.</p></li>
<li id="id1942"><p>Yanhua Liu and Hans V. Westerhoff. `social' versus `asocial' cells-dynamic competition flux balance analysis. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, OCT 28 2023. <a class="reference external" href="https://doi.org/10.1038/s41540-023-00313-5">[DOI]</a>.</p></li>
<li id="id1944"><p>Taghreed Elkasaby, Dao Duy Hanh, Hideo Kawaguchi, Masakazu Toyoshima, Akihiko Kondo, and Chiaki Ogino. Co-utilization of maltose and sodium acetate via engineered &lt;i&gt;corynebacterium glutamicum&lt;/i&gt; for improved itaconic acid production. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 28(5):790–803, OCT 2023. <a class="reference external" href="https://doi.org/10.1007/s12257-023-0091-7">[DOI]</a>.</p></li>
<li id="id1945"><p>Garhima Arora, Mallar Banerjee, Jimpi Langthasa, Ramray Bhat, and Samrat Chatterjee. Article targeting metabolic fluxes reverts metastatic transitions in ovarian cancer. <em>ISCIENCE</em>, NOV 17 2023. <a class="reference external" href="https://doi.org/10.1016/j.isci.2023.108081">[DOI]</a>.</p></li>
<li id="id1946"><p>Isabella Casini, Tim Mccubbin, Sofia Esquivel-Elizondo, Guillermo G. Luque, Daria Evseeva, Christian Fink, Sebastian Beblawy, Nicholas D. Youngblut, Ludmilla Aristilde, Daniel H. Huson, Andreas Draeger, Ruth E. Ley, Esteban Marcellin, Largus T. Angenent, and Bastian Molitor. An integrated systems biology approach reveals differences in formate metabolism in the genus &lt;i&gt;methanothermobacter&lt;/i&gt;. <em>ISCIENCE</em>, OCT 20 2023. <a class="reference external" href="https://doi.org/10.1016/j.isci.2023.108016">[DOI]</a>.</p></li>
<li id="id1947"><p>Qusheng Jin. Building microbial kinetic models for environmental application: a theoretical perspective. <em>APPLIED GEOCHEMISTRY</em>, NOV 2023. <a class="reference external" href="https://doi.org/10.1016/j.apgeochem.2023.105782">[DOI]</a>.</p></li>
<li id="id1948"><p>Mauricio Alexander de Moura Ferreira, Wendel Batista da Silveira, and Zoran Nikoloski. Parrot: prediction of enzyme abundances using protein-constrained metabolic models. <em>PLOS COMPUTATIONAL BIOLOGY</em>, OCT 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1011549">[DOI]</a>.</p></li>
<li id="id1949"><p>Maxwell Neal, Deepan Thiruppathy, and Karsten Zengler. Genome-scale metabolic modeling of the human gut bacterium &lt;i&gt;bacteroides fragilis&lt;/i&gt; strain 638r. <em>PLOS COMPUTATIONAL BIOLOGY</em>, OCT 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1011594">[DOI]</a>.</p></li>
<li id="id1950"><p>Ke-Lin Chen and Feng-Sheng Wang. Cell-specific genome-scale metabolic modeling of sars-cov-2-infected lung to identify antiviral enzymes. <em>FEBS OPEN BIO</em>, 13(12):2172–2186, DEC 2023. <a class="reference external" href="https://doi.org/10.1002/2211-5463.13710">[DOI]</a>.</p></li>
<li id="id1951"><p>Xia Wang, Ying Teng, Xiaomi Wang, Yongfeng Xu, Ran Li, Yi Sun, Shixiang Dai, Wenbo Hu, Hongzhe Wang, Yanning Li, Yan Fang, and Yongming Luo. Nitrogen transfer and cross-feeding between &lt;i&gt;azotobacter chroococcum&lt;/i&gt; and &lt;i&gt;paracoccus aminovorans&lt;/i&gt; promotes pyrene degradation. <em>ISME JOURNAL</em>, 17(12):2169–2181, DEC 2023. <a class="reference external" href="https://doi.org/10.1038/s41396-023-01522-w">[DOI]</a>.</p></li>
<li id="id1952"><p>Jingwei Cai, Alexis Auster, Sungjoon Cho, and Zijuan Lai. Dissecting the human gut microbiome to better decipher drug liability: a once-forgotten organ takes center stage. <em>JOURNAL OF ADVANCED RESEARCH</em>, 52:171–201, OCT 2023. <a class="reference external" href="https://doi.org/10.1016/j.jare.2023.07.002">[DOI]</a>.</p></li>
<li id="id1953"><p>Brandt Bessell, Josh Loecker, Zhongyuan Zhao, Sara Sadat Aghamiri, Sabyasachi Mohanty, Rada Amin, Tomas Helikar, and Bhanwar Lal Puniya. Como: a pipeline for multi-omics data integration in metabolic modeling and drug discovery. <em>BRIEFINGS IN BIOINFORMATICS</em>, SEP 22 2023. <a class="reference external" href="https://doi.org/10.1093/bib/bbad387">[DOI]</a>.</p></li>
<li id="id1954"><p>Agris Pentjuss, Emils Bolmanis, Anastasija Suleiko, Elina Didrihsone, Arturs Suleiko, Konstantins Dubencovs, Janis Liepins, Andris Kazaks, and Juris Vanags. &lt;i&gt;pichia pastoris&lt;/i&gt; growth-coupled heme biosynthesis analysis using metabolic modelling. <em>SCIENTIFIC REPORTS</em>, SEP 22 2023. <a class="reference external" href="https://doi.org/10.1038/s41598-023-42865-w">[DOI]</a>.</p></li>
<li id="id1955"><p>Ehsan Zangene, Sayed-Amir Marashi, and Hesam Montazeri. Sl-scan identifies synthetic lethal interactions in cancer using metabolic networks. <em>SCIENTIFIC REPORTS</em>, SEP 22 2023. <a class="reference external" href="https://doi.org/10.1038/s41598-023-42992-4">[DOI]</a>.</p></li>
<li id="id1956"><p>Subasree Sridhar, Prerna Bhalla, Justin Kullu, Sriya Veerapaneni, Swagatika Sahoo, Nirav Bhatt, and G. K. Suraishkumar. A reactive species reactions module for integration into genome-scale metabolic models for improved insights: application to cancer. <em>METABOLIC ENGINEERING</em>, 80:78–93, NOV 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2023.08.006">[DOI]</a>.</p></li>
<li id="id1957"><p>Bo Wang, Cristal Zuniga, Michael T. Guarnieri, Karsten Zengler, Michael Betenbaugh, and Jamey D. Young. Metabolic engineering of&lt;i&gt; synechococcus&lt;/i&gt;&lt;i&gt; elongatus&lt;/i&gt; 7942 for enhanced sucrose biosynthesis. <em>METABOLIC ENGINEERING</em>, 80:12–24, NOV 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2023.09.002">[DOI]</a>.</p></li>
<li id="id1958"><p>Thomas Eng, Deepanwita Banerjee, Javier Menasalvas, Yan Chen, Jennifer Gin, Hemant Choudhary, Edward Baidoo, Jian Hua Chen, Axel Ekman, Ramu Kakumanu, Yuzhong Liu Diercks, Alex Codik, Carolyn Larabell, John Gladden, Blake A. Simmons, Jay D. Keasling, Christopher J. Petzold, and Aindrila Mukhopadhyay. Maximizing microbial bioproduction from sustainable carbon sources using iterative systems engineering. <em>CELL REPORTS</em>, SEP 26 2023. <a class="reference external" href="https://doi.org/10.1016/j.celrep.2023.113087">[DOI]</a>.</p></li>
<li id="id1959"><p>Ronan M. T. Fleming, Hulda S. Haraldsdottir, Le Hoai Minh, Phan Tu Vuong, Thomas Hankemeier, and Ines Thiele. Cardinality optimization in constraint-based modelling: application to human metabolism. <em>BIOINFORMATICS</em>, SEP 2 2023. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btad450">[DOI]</a>.</p></li>
<li id="id1960"><p>Marc Griesemer and Ali Navid. Uses of multi-objective flux analysis for optimization of microbial production of secondary metabolites. <em>MICROORGANISMS</em>, SEP 2023. <a class="reference external" href="https://doi.org/10.3390/microorganisms11092149">[DOI]</a>.</p></li>
<li id="id1961"><p>David Healy, Shuo Wang, Ghjuvan Grimaud, Alicja Katarzyna Warda, Paul Ross, Catherine Stanton, and Eugene M. Dempsey. Longitudinal observational study protocol - preterm infants: microbiome establishment, neuro-crosstalk and origins (pimento). <em>BMJ OPEN</em>, SEP 2023. <a class="reference external" href="https://doi.org/10.1136/bmjopen-2023-075060">[DOI]</a>.</p></li>
<li id="id1962"><p>Nela Pragathi Sneha, S. Akila Parvathy Dharshini, Y. -H Taguchi, and M. Michael Gromiha. Investigating neuron degeneration in huntington's disease using rna-seq based transcriptome study. <em>GENES</em>, SEP 2023. <a class="reference external" href="https://doi.org/10.3390/genes14091801">[DOI]</a>.</p></li>
<li id="id1963"><p>Elias Vera-Siguenza, Cristina Escribano-Gonzalez, Irene Serrano-Gonzalo, Kattri-Liis Eskla, Fabian Spill, and Daniel Tennant. Mathematical reconstruction of the metabolic network in an in-vitro multiple myeloma model. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1011374">[DOI]</a>.</p></li>
<li id="id1964"><p>Bethany L. Holland, Megan L. Matthews, Pedro Bota, Lee J. Sweetlove, Stephen P. Long, and George C. Dicenzo. A genome-scale metabolic reconstruction of soybean and bradyrhizobium diazoefficiens reveals the cost-benefit of nitrogen fixation. <em>NEW PHYTOLOGIST</em>, 240(2):744–756, OCT 2023. <a class="reference external" href="https://doi.org/10.1111/nph.19203">[DOI]</a>.</p></li>
<li id="id1965"><p>Kyoshiro Nonaka, Tatsuya Osamura, and Fumikazu Takahashi. A 4-hydroxybenzoate 3-hydroxylase mutant enables 4-amino-3-hydroxybenzoic acid production from glucose in &lt;i&gt;corynebacterium glutamicum&lt;/i&gt;. <em>MICROBIAL CELL FACTORIES</em>, AUG 29 2023. <a class="reference external" href="https://doi.org/10.1186/s12934-023-02179-y">[DOI]</a>.</p></li>
<li id="id1966"><p>Seo-Young Park, Dong-Hyuk Choi, Jinsung Song, Uiseon Park, Hyeran Cho, Bee Hak Hong, Yaron R. Silberberg, and Dong-Yup Lee. Debottlenecking and reformulating feed media for improved cho cell growth and titer by data-driven and model-guided analyses. <em>BIOTECHNOLOGY JOURNAL</em>, DEC 2023. <a class="reference external" href="https://doi.org/10.1002/biot.202300126">[DOI]</a>.</p></li>
<li id="id1967"><p>Francesc Puig-Castellvi, Romina Pacheco-Tapia, Maxime Deslande, Manyi Jia, Petros Andrikopoulos, Kanta Chechi, Amelie Bonnefond, Philippe Froguel, and Marc-Emmanuel Dumas. Advances in the integration of metabolomics and metagenomics for human gut microbiome and their clinical applications. <em>TRAC-TRENDS IN ANALYTICAL CHEMISTRY</em>, OCT 2023. <a class="reference external" href="https://doi.org/10.1016/j.trac.2023.117248">[DOI]</a>.</p></li>
<li id="id1968"><p>Patrick E. Gelbach and Stacey D. Finley. Genome-scale modeling predicts metabolic differences between macrophage subtypes in colorectal cancer. <em>ISCIENCE</em>, SEP 15 2023. <a class="reference external" href="https://doi.org/10.1016/j.isci.2023.107569">[DOI]</a>.</p></li>
<li id="id1969"><p>Adrian Povo-Retana, Marco Farinas, Rodrigo Landauro-Vera, Marina Mojena, Carlota Alvarez-Lucena, Miguel A. Fernandez-Moreno, Antonio Castrillo, Juan Vladimir de la Rosa Medina, Sergio Sanchez-Garcia, Carles Foguet, Francesc Mas, Silvia Marin, Marta Cascante, and Lisardo Bosca. Immunometabolic actions of trabectedin and lurbinectedin on human macrophages: relevance for their anti-tumor activity. <em>FRONTIERS IN IMMUNOLOGY</em>, AUG 22 2023. <a class="reference external" href="https://doi.org/10.3389/fimmu.2023.1211068">[DOI]</a>.</p></li>
<li id="id1970"><p>Juan Ricardo Velasco-Alvarez, Nimbe Torres y Torres, Isaac Chairez, and Jose Luis Castrejon-Flores. Microbiome distribution modeling using gradient descent strategies for mock, &lt;i&gt;in vitro&lt;/i&gt; and clinical community distributions. <em>PLOS ONE</em>, AUG 21 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0290082">[DOI]</a>.</p></li>
<li id="id1971"><p>Sandra Correa Cordoba, Hao Tong, Asdrubal Burgos, Feng Zhu, Saleh Alseekh, Alisdair R. Fernie, and Zoran Nikoloski. Identification of gene function based on models capturing natural variability of &lt;i&gt;arabidopsis thaliana&lt;/i&gt; lipid metabolism. <em>NATURE COMMUNICATIONS</em>, AUG 14 2023. <a class="reference external" href="https://doi.org/10.1038/s41467-023-40644-9">[DOI]</a>.</p></li>
<li id="id1972"><p>Marius Arend, David Zimmer, Rudan Xu, Frederik Sommer, Timo Muehlhaus, and Zoran Nikoloski. Proteomics and constraint-based modelling reveal enzyme kinetic properties of &lt;i&gt;chlamydomonas reinhardtii&lt;/i&gt; on a genome scale. <em>NATURE COMMUNICATIONS</em>, AUG 8 2023. <a class="reference external" href="https://doi.org/10.1038/s41467-023-40498-1">[DOI]</a>.</p></li>
<li id="id1973"><p>Pedro de Atauri, Carles Foguet, and Marta Cascante. Control analysis in the identification of key enzymes driving metabolic adaptations: towards drug target discovery. <em>BIOSYSTEMS</em>, SEP 2023. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2023.104984">[DOI]</a>.</p></li>
<li id="id1975"><p>Hiroki Suyama, Laurence Don Wai Luu, Ling Zhong, Mark J. Raftery, and Ruiting Lan. Integrating proteomic data with metabolic modeling provides insight into key pathways of &lt;i&gt;bordetella pertussis&lt;/i&gt; biofilms. <em>FRONTIERS IN MICROBIOLOGY</em>, AUG 3 2023. <a class="reference external" href="https://doi.org/10.3389/fmicb.2023.1169870">[DOI]</a>.</p></li>
<li id="id1976"><p>Sarayu Murali, Maziya Ibrahim, Hemalatha Rajendran, Shagun Shagun, Shyam Kumar Masakapalli, Karthik Raman, and Smita Srivastava. Genome-scale metabolic model led engineering of &lt;i&gt;nothapodytes nimmoniana&lt;/i&gt; plant cells for high camptothecin production. <em>FRONTIERS IN PLANT SCIENCE</em>, AUG 2 2023. <a class="reference external" href="https://doi.org/10.3389/fpls.2023.1207218">[DOI]</a>.</p></li>
<li id="id1977"><p>Mueberra Fatma Cesur, Arianna Basile, Kiran Raosaheb Patil, and Tunahan Cakir. A new metabolic model of&lt;i&gt; drosophila&lt;/i&gt;&lt;i&gt; melanogaster&lt;/i&gt; and the integrative analysis of parkinson's disease. <em>LIFE SCIENCE ALLIANCE</em>, AUG 2023. <a class="reference external" href="https://doi.org/10.26508/lsa.202201695">[DOI]</a>.</p></li>
<li id="id1978"><p>Emanuel Cunha, Davide Lagoa, Jose P. Faria, Filipe Liu, Christopher S. Henry, and Oscar Dias. Tran&lt;i&gt;syt&lt;/i&gt;, an innovative framework for identifying transport systems. <em>BIOINFORMATICS</em>, AUG 1 2023. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btad466">[DOI]</a>.</p></li>
<li id="id1979"><p>Johann. F. F. Jadebeck, Wolfgang Wiechert, and Katharina Noeh. Practical sampling of constraint-based models: optimized thinning boosts chrr performance. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1011378">[DOI]</a>.</p></li>
<li id="id1980"><p>X. Luo, Y. J. Liu, A. Balck, C. Klein, and R. Fleming. Identification of reproducible metabolites for parkinson's disease diagnosis using clinical data and computational modelling. <em>MOVEMENT DISORDERS</em>, 38(1):S647–S648, AUG 2023. International Congress of Parkinson's Disease and Movement Disorders, Copenhagen, DENMARK, AUG 27-31, 2023.</p></li>
<li id="id1981"><p>Sizhe Qiu, Aidong Yang, and Hong Zeng. Flux balance analysis-based metabolic modeling of microbial secondary metabolism: current status and outlook. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1011391">[DOI]</a>.</p></li>
<li id="id1982"><p>William T. R. Scott Jr, Sara Benito-Vaquerizo, Johannes R. Zimmermann, Djordje Bajic, Almut R. Heinken, Maria Suarez-Diez, and Peter R. Schaap. A structured evaluation of genome-scale constraint-based modeling tools for microbial consortia. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1011363">[DOI]</a>.</p></li>
<li id="id1983"><p>Diego Tec-Campos, Camila Posadas, Juan D. Tibocha-Bonilla, Deepan Thiruppathy, Nathan Glonek, Cristal Zuniga, Alejandro Zepeda, and Karsten Zengler. The genome-scale metabolic model for the purple non-sulfur bacterium &lt;i&gt;rhodopseudomonas palustris&lt;/i&gt; bis a53 accurately predicts phenotypes under chemoheterotrophic, chemoautotrophic, photoheterotrophic, and photoautotrophic growth conditions. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1011371">[DOI]</a>.</p></li>
<li id="id1984"><p>Yoon-Mi Choi, Dong-Hyuk Choi, Yi Qing Lee, Lokanand Koduru, Nathan E. Lewis, Meiyappan Lakshmanan, and Dong-Yup Lee. Mitigating biomass composition uncertainties in flux balance analysis using ensemble representations. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 21:3736–3745, 2023. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2023.07.025">[DOI]</a>.</p></li>
<li id="id1985"><p>Hao Tong, Roosa A. E. Laitinen, and Zoran Nikoloski. Predicting plasticity of rosette growth and metabolic fluxes in &lt;i&gt;arabidopsis thaliana&lt;/i&gt;. <em>NEW PHYTOLOGIST</em>, 240(1):426–438, OCT 2023. <a class="reference external" href="https://doi.org/10.1111/nph.19154">[DOI]</a>.</p></li>
<li id="id1986"><p>Xiaoxiang Gao, Jianxin Zhao, Wei Chen, and Qixiao Zhai. Food and drug design for gut microbiota-directed regulation: current experimental landscape and future innovation. <em>PHARMACOLOGICAL RESEARCH</em>, AUG 2023. <a class="reference external" href="https://doi.org/10.1016/j.phrs.2023.106867">[DOI]</a>.</p></li>
<li id="id1987"><p>Tamara Bintener, Maria Pires Pacheco, Demetra Philippidou, Christiane Margue, Ali Kishk, Greta Del Mistro, Luca Di Leo, Maria Moscardo Garcia, Rashi Halder, Lasse Sinkkonen, Daniela De Zio, Stephanie Kreis, Dagmar Kulms, and Thomas Sauter. Metabolic modelling-based in silico drug target prediction identifies six novel repurposable drugs for melanoma. <em>CELL DEATH &amp; DISEASE</em>, JUL 26 2023. <a class="reference external" href="https://doi.org/10.1038/s41419-023-05955-1">[DOI]</a>.</p></li>
<li id="id1988"><p>Zohreh Mirveis, Orla Howe, Paul Cahill, Nitin Patil, and Hugh J. J. Byrne. Monitoring and modelling the glutamine metabolic pathway: a review and future perspectives. <em>METABOLOMICS</em>, JUL 23 2023. <a class="reference external" href="https://doi.org/10.1007/s11306-023-02031-9">[DOI]</a>.</p></li>
<li id="id1989"><p>Kengo Watanabe, Tomasz Wilmanski, Priyanka Baloni, Max Robinson, Gonzalo G. Garcia, Michael R. Hoopmann, Mukul K. Midha, David H. Baxter, Michal Maes, Seamus R. Morrone, Kelly M. Crebs, Charu Kapil, Ulrike Kusebauch, Jack Wiedrick, Jodi Lapidus, Lance Pflieger, Christopher Lausted, Jared C. Roach, Gwenlyn Glusman, Steven R. Cummings, Nicholas J. Schork, Nathan D. Price, Leroy Hood, Richard A. Miller, Robert L. Moritz, and Noa Rappaport. Lifespan-extending interventions induce consistent patterns of fatty acid oxidation in mouse livers. <em>COMMUNICATIONS BIOLOGY</em>, JUL 22 2023. <a class="reference external" href="https://doi.org/10.1038/s42003-023-05128-y">[DOI]</a>.</p></li>
<li id="id1990"><p>Jingning Chen, Yiran Huang, and Cheng Zhong. Minimizing enzyme mass to decompose flux distribution for identifying biologically relevant elementary flux modes. <em>BIOSYSTEMS</em>, SEP 2023. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2023.104981">[DOI]</a>.</p></li>
<li id="id1991"><p>Naroa Barrena, Luis V. V. Valcarcel, Danel Olaverri-Mendizabal, Inigo Apaolaza, and Francisco J. J. Planes. Synthetic lethality in large-scale integrated metabolic and regulatory network models of human cells. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, JUL 15 2023. <a class="reference external" href="https://doi.org/10.1038/s41540-023-00296-3">[DOI]</a>.</p></li>
<li id="id1992"><p>Xingcun Fan, Lingfeng Cao, and Xuefeng Yan. Sensitivity analysis and adaptive mutation strategy differential evolution algorithm for optimizing enzymes' turnover numbers in metabolic models. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 120(8):2301–2313, AUG 2023. <a class="reference external" href="https://doi.org/10.1002/bit.28493">[DOI]</a>.</p></li>
<li id="id1993"><p>Su-Kyung Kim, Minouk Lee, Yi Qing Lee, Hyun Jun Lee, Mina Rho, Yunkwan Kim, Jung Yeon Seo, Sung Hun Youn, Seung Jin Hwang, Nae Gyu Kang, Choong-Hwan Lee, Seo-Young Park, and Dong-Yup Lee. Genome-scale metabolic modeling and &lt;i&gt;in silico&lt;/i&gt; analysis of opportunistic skin pathogen &lt;i&gt;cutibacterium acnes&lt;/i&gt;. <em>FRONTIERS IN CELLULAR AND INFECTION MICROBIOLOGY</em>, JUL 13 2023. <a class="reference external" href="https://doi.org/10.3389/fcimb.2023.1099314">[DOI]</a>.</p></li>
<li id="id1994"><p>Martin Schafer, Alan R. Pacheco, Rahel Kunzler, Miriam Bortfeld-Miller, Christopher M. Field, Evangelia Vayena, Vassily Hatzimanikatis, and Julia A. Vorholt. Metabolic interaction models recapitulate leaf microbiota ecology. <em>SCIENCE</em>, 381(6653):42+, JUL 7 2023. <a class="reference external" href="https://doi.org/10.1126/science.adf5121">[DOI]</a>.</p></li>
<li id="id1995"><p>Peishun Li, Stefan Roos, Hao Luo, Boyang Ji, and Jens Nielsen. Metabolic engineering of human gut microbiome: recent developments and future perspectives. <em>METABOLIC ENGINEERING</em>, 79:1–13, SEP 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2023.06.006">[DOI]</a>.</p></li>
<li id="id1996"><p>Signe T. Karlsen, Martin H. Rau, Benjamin J. Sanchez, Kristian Jensen, and Ahmad A. Zeidan. From genotype to phenotype: computational approaches for inferring microbial traits relevant to the food industry. <em>FEMS MICROBIOLOGY REVIEWS</em>, JUL 5 2023. <a class="reference external" href="https://doi.org/10.1093/femsre/fuad030">[DOI]</a>.</p></li>
<li id="id1997"><p>Yang Zhang, Menglei Yang, Yangyang Bao, Weihua Tao, Jinyou Tuo, Boya Liu, Luxi Gan, Shuilin Fu, and Heng Gong. A genome-scale metabolic model of the effect of dissolved oxygen on 1,3-propanediol fermentation by &lt;i&gt;klebsiella pneumoniae&lt;/i&gt;. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 46(9):1319–1330, SEP 2023. <a class="reference external" href="https://doi.org/10.1007/s00449-023-02899-w">[DOI]</a>.</p></li>
<li id="id1998"><p>Marta Balazs, Hunor Bartos, Szabolcs Lanyi, Zsolt Bodor, and Ildiko Miklossy. Substrate type and co&lt;sub&gt;2&lt;/sub&gt; addition significantly influence succinic acid production of &lt;i&gt;basfia succiniciproducens&lt;/i&gt;. <em>BIOTECHNOLOGY LETTERS</em>, 45(9):1133–1145, SEP 2023. <a class="reference external" href="https://doi.org/10.1007/s10529-023-03406-7">[DOI]</a>.</p></li>
<li id="id1999"><p>Fidias D. Gonzalez Camargo, Mary Santamaria-Torres, Monica P. Cala, Marcela Guevara-Suarez, Silvia Restrepo Restrepo, Andrea Sanchez-Camargo, Miguel Fernandez-Nino, Maria Corujo, Ada Carolina Gallo Molina, Javier Cifuentes, Julian A. Serna, Juan C. Cruz, Carolina Munoz-Camargo, and Andres F. Gonzalez Barrios. Genome-scale metabolic reconstruction, non-targeted lc-qtof-ms based metabolomics data, and evaluation of anticancer activity of &lt;i&gt;cannabis sativa&lt;/i&gt; leaf extracts. <em>METABOLITES</em>, JUL 2023. <a class="reference external" href="https://doi.org/10.3390/metabo13070788">[DOI]</a>.</p></li>
<li id="id2000"><p>Partho Sen and Matej Oresic. Integrating omics data in genome-scale metabolic modeling: a methodological perspective for precision medicine. <em>METABOLITES</em>, JUL 2023. <a class="reference external" href="https://doi.org/10.3390/metabo13070855">[DOI]</a>.</p></li>
<li id="id2001"><p>Mohammadreza Yasemi and Mario Jolicoeur. A genome-scale dynamic constraint-based modelling (gdcbm) framework predicts growth dynamics, medium composition and intracellular flux distributions in cho clonal variations. <em>METABOLIC ENGINEERING</em>, 78:209–222, JUL 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2023.06.005">[DOI]</a>.</p></li>
<li id="id2002"><p>Albert Fina, Pierre Millard, Joan Albiol, Pau Ferrer, and Stephanie Heux. High throughput &lt;sup&gt;13&lt;/sup&gt;c-metabolic flux analysis of 3-hydroxypropionic acid producing &lt;i&gt;pichia pastoris&lt;/i&gt; reveals limited availability of acetyl-coa and atp due to tight control of the glycolytic flux. <em>MICROBIAL CELL FACTORIES</em>, JUN 29 2023. <a class="reference external" href="https://doi.org/10.1186/s12934-023-02123-0">[DOI]</a>.</p></li>
<li id="id2003"><p>Dipanka Tanu Sarmah, Abhijit Paul, Shivam Kumar, Nandadulal Bairagi, and Samrat Chatterjee. A data-driven multilayer approach for the identification of potential therapeutic targets in non-alcoholic steatohepatitis. <em>PHYSICA A-STATISTICAL MECHANICS AND ITS APPLICATIONS</em>, AUG 15 2023. <a class="reference external" href="https://doi.org/10.1016/j.physa.2023.128955">[DOI]</a>.</p></li>
<li id="id2004"><p>Zhongcai Li, Qian Liu, Jiahui Sun, Jianjian Sun, Mingjie Li, Yun Zhang, Aihua Deng, Shuwen Liu, and Tingyi Wen. Multivariate modular metabolic engineering for enhanced &lt;sup&gt;l&lt;/sup&gt;-methionine biosynthesis in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOTECHNOLOGY FOR BIOFUELS AND BIOPRODUCTS</em>, JUN 13 2023. <a class="reference external" href="https://doi.org/10.1186/s13068-023-02347-7">[DOI]</a>.</p></li>
<li id="id2005"><p>Sandra Gomez Romero and Nanette Boyle. Systems biology and metabolic modeling for cultivated meat: a promising approach for cell culture media optimization and cost reduction. <em>COMPREHENSIVE REVIEWS IN FOOD SCIENCE AND FOOD SAFETY</em>, 22(4):3422–3443, JUL 2023. <a class="reference external" href="https://doi.org/10.1111/1541-4337.13193">[DOI]</a>.</p></li>
<li id="id2006"><p>Sourav Chowdhury, Daniel C. Zielinski, Christopher Dalldorf, Joao V. Rodrigues, Bernhard O. Palsson, and Eugene I. Shakhnovich. Empowering drug off-target discovery with metabolic and structural analysis. <em>NATURE COMMUNICATIONS</em>, JUN 9 2023. <a class="reference external" href="https://doi.org/10.1038/s41467-023-38859-x">[DOI]</a>.</p></li>
<li id="id2007"><p>Ibrahim Fakih, Jeanne Got, Carlos Eduardo Robles-Rodriguez, Anne Siegel, Evelyne Forano, and Rafael Munoz-Tamayo. Dynamic genome-based metabolic modeling of the predominant cellulolytic rumen bacterium &lt;i&gt;fibrobacter succinogenes&lt;/i&gt; s85. <em>MSYSTEMS</em>, JUN 29 2023. <a class="reference external" href="https://doi.org/10.1128/msystems.01027-22">[DOI]</a>.</p></li>
<li id="id2008"><p>Matti Gralka. Searching for principles of microbial ecology across levels of biological organization. <em>INTEGRATIVE AND COMPARATIVE BIOLOGY</em>, 63(6):1520–1531, DEC 29 2023. <a class="reference external" href="https://doi.org/10.1093/icb/icad060">[DOI]</a>.</p></li>
<li id="id2009"><p>Apostolos Chalkis, Vissarion Fisikopoulos, Marios Papachristou, and Elias Tsigaridas. Truncated log-concave sampling for convex bodies with reflective hamiltonian monte carlo. <em>ACM TRANSACTIONS ON MATHEMATICAL SOFTWARE</em>, JUN 2023. <a class="reference external" href="https://doi.org/10.1145/3589505">[DOI]</a>.</p></li>
<li id="id2010"><p>Kristina Grausa, Shahida A. Siddiqui, Norbert Lameyer, Karin Wiesotzki, Sergiy Smetana, and Agris Pentjuss. Metabolic modeling of &lt;i&gt;hermetia illucens&lt;/i&gt; larvae resource allocation for high-value fatty acid production. <em>METABOLITES</em>, JUN 2023. <a class="reference external" href="https://doi.org/10.3390/metabo13060724">[DOI]</a>.</p></li>
<li id="id2011"><p>Reinis Muiznieks, Elina Dace, and Egils Stalidzans. Integrated sustainability score implementation as an objective function in sustainable metabolic engineering. <em>FERMENTATION-BASEL</em>, JUN 2023. <a class="reference external" href="https://doi.org/10.3390/fermentation9060548">[DOI]</a>.</p></li>
<li id="id2012"><p>Anna Stikane, Matiss Ricards Baumanis, Reinis Muiznieks, and Egils Stalidzans. Impact of waste as a substrate on biomass formation, and optimization of spent microbial biomass re-use by sustainable metabolic engineering. <em>FERMENTATION-BASEL</em>, JUN 2023. <a class="reference external" href="https://doi.org/10.3390/fermentation9060531">[DOI]</a>.</p></li>
<li id="id2013"><p>Xueting Wang, Ali Mohsin, Yifei Sun, Chao Li, Yingping Zhuang, and Guan Wang. From spatial-temporal multiscale modeling to application: bridging the valley of death in industrial biotechnology. <em>BIOENGINEERING-BASEL</em>, JUN 2023. <a class="reference external" href="https://doi.org/10.3390/bioengineering10060744">[DOI]</a>.</p></li>
<li id="id2014"><p>Anna T. Germann, Andreas Nakielski, Maximilian Dietsch, Tim Petzel, Daniel Moser, Sebastian Triesch, Philipp Westhoff, and Ilka M. Axmann. A systematic overexpression approach reveals native targets to increase squalene production in &lt;i&gt;synechocystis&lt;/i&gt; sp. pcc 6803. <em>FRONTIERS IN PLANT SCIENCE</em>, MAY 30 2023. <a class="reference external" href="https://doi.org/10.3389/fpls.2023.1024981">[DOI]</a>.</p></li>
<li id="id2015"><p>Havard Molversmyr, Ove Oyas, Filip Rotnes, and Jon Olav Vik. Extracting functionally accurate context-specific models of atlantic salmon metabolism. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, MAY 27 2023. <a class="reference external" href="https://doi.org/10.1038/s41540-023-00280-x">[DOI]</a>.</p></li>
<li id="id2016"><p>Chi Hung Vo, Nishu Goyal, Markus Kraft, and Iftekhar A. Karimi. Carbon conversion by&lt;i&gt; methanococcus&lt;/i&gt;&lt;i&gt; maripaludis&lt;/i&gt; s2 under diazotrophy and a revised genome-scale metabolic model. <em>CHEMICAL ENGINEERING SCIENCE</em>, AUG 15 2023. <a class="reference external" href="https://doi.org/10.1016/j.ces.2023.118910">[DOI]</a>.</p></li>
<li id="id2017"><p>David Henriques, Romain Minebois, David dos Santos, Eladio Barrio, Amparo Querol, and Eva Balsa-Canto. A dynamic genome-scale model identifies metabolic pathways associated with cold tolerance in saccharomyces kudriavzevii. <em>MICROBIOLOGY SPECTRUM</em>, JUN 2023. <a class="reference external" href="https://doi.org/10.1128/spectrum.03519-22">[DOI]</a>.</p></li>
<li id="id2018"><p>Lvjing Wang, Xiaoyu Wang, Hao Wu, Haixia Wang, Yihan Wang, and Zhenmei Lu. Metabolic modeling of synthetic microbial communities for bioremediation. <em>CRITICAL REVIEWS IN ENVIRONMENTAL SCIENCE AND TECHNOLOGY</em>, 53(24):2092–2111, DEC 17 2023. <a class="reference external" href="https://doi.org/10.1080/10643389.2023.2212569">[DOI]</a>.</p></li>
<li id="id2019"><p>Igor Marin de Mas, Helena Herand, Jorge Carrasco, Lars K. Nielsen, and Par I. Johansson. A protocol for the automatic construction of highly curated genome-scale models of human metabolism. <em>BIOENGINEERING-BASEL</em>, MAY 10 2023. <a class="reference external" href="https://doi.org/10.3390/bioengineering10050576">[DOI]</a>.</p></li>
<li id="id2020"><p>Laura M. Helleckes, Johannes Hemmerich, Wolfgang Wiechert, Eric von Lieres, and Alexander Gruenberger. Machine learning in bioprocess development: from promise to practice. <em>TRENDS IN BIOTECHNOLOGY</em>, 41(6):817–835, JUN 2023. <a class="reference external" href="https://doi.org/10.1016/j.tibtech.2022.10.010">[DOI]</a>.</p></li>
<li id="id2021"><p>Zhitao Mao, Qianqian Yuan, Haoran Li, Yue Zhang, Yuanyuan Huang, Chunhe Yang, Ruoyu Wang, Yongfu Yang, Yalun Wu, Shihui Yang, Xiaoping Liao, and Hongwu Ma. Cave: a cloud-based platform for analysis and visualization of metabolic pathways. <em>NUCLEIC ACIDS RESEARCH</em>, 51(W1):W70–W77, JUL 5 2023. <a class="reference external" href="https://doi.org/10.1093/nar/gkad360">[DOI]</a>.</p></li>
<li id="id2022"><p>Nattharat Punyasu, Saowalak Kalapanulak, and Treenut Saithong. Co&lt;sub&gt;2&lt;/sub&gt; recycling by phospho&lt;i&gt;enol&lt;/i&gt;pyruvate carboxylase enables cassava leaf metabolism to tolerate low water availability. <em>FRONTIERS IN PLANT SCIENCE</em>, MAY 9 2023. <a class="reference external" href="https://doi.org/10.3389/fpls.2023.1159247">[DOI]</a>.</p></li>
<li id="id2023"><p>William T. Scott, David Henriques, Eddy J. Smid, Richard A. Notebaart, and Eva Balsa-Canto. Dynamic genome-scale modeling of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; unravels mechanisms for ester formation during alcoholic fermentation. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 120(7):1998–2012, JUL 2023. <a class="reference external" href="https://doi.org/10.1002/bit.28421">[DOI]</a>.</p></li>
<li id="id2024"><p>Joshua A. M. Kaste and Yair Shachar-Hill. Accurate flux predictions using tissue-specific gene expression in plant metabolic modeling. <em>BIOINFORMATICS</em>, MAY 4 2023. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btad186">[DOI]</a>.</p></li>
<li id="id2025"><p>Cecilia Noecker, Juan Sanchez, Jordan E. Bisanz, Veronica Escalante, Margaret Alexander, Kai Trepka, Almut Heinken, Yuanyuan Liu, Dylan Dodd, Ines Thiele, Brian C. DeFelice, and Peter J. Turnbaugh. Systems biology elucidates the distinctive metabolic niche filled by the human gut microbe &lt;i&gt;eggerthella lenta&lt;/i&gt;. <em>PLOS BIOLOGY</em>, MAY 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pbio.3002125">[DOI]</a>.</p></li>
<li id="id2026"><p>GaRyoung Lee, Sang Mi Lee, and Hyun Uk Kim. A contribution of metabolic engineering to addressing medical problems: metabolic flux analysis. <em>METABOLIC ENGINEERING</em>, 77:283–293, MAY 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2023.04.008">[DOI]</a>.</p></li>
<li id="id2027"><p>Andrew Walakira, Cene Skubic, Nejc Nadizar, Damjana Rozman, Tadeja Rezen, Miha Mraz, and Miha Moskon. Integrative computational modeling to unravel novel potential biomarkers in hepatocellular carcinoma. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, JUN 2023. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2023.106957">[DOI]</a>.</p></li>
<li id="id2028"><p>Eric Libby, Christopher P. Kempes, and Jordan G. Okie. Metabolic compatibility and the rarity of prokaryote endosymbioses. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, APR 25 2023. <a class="reference external" href="https://doi.org/10.1073/pnas.2206527120">[DOI]</a>.</p></li>
<li id="id2029"><p>Darwin Carranza-Saavedra, Jesus Torres-Bacete, Blas Blazquez, Claudia Patricia Sanchez Henao, Jose Edgar Zapata Montoya, and Juan Nogales. System metabolic engineering of &lt;i&gt;escherichia coli&lt;/i&gt; w for the production of 2-ketoisovalerate using unconventional feedstock. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, APR 20 2023. <a class="reference external" href="https://doi.org/10.3389/fbioe.2023.1176445">[DOI]</a>.</p></li>
<li id="id2030"><p>Mona Synnove Bjune, Laurence Lawrence-Archer, Johnny Laupsa-Borge, Cathrine Horn Sommersten, Adrian McCann, Robert Clay Glastad, Iain George Johnston, Matthias Kern, Matthias Blueher, Gunnar Mellgren, and Simon N. Dankel. Metabolic role of the hepatic valine/3-hydroxyisobutyrate (3-hib) pathway in fatty liver disease. <em>EBIOMEDICINE</em>, MAY 2023. <a class="reference external" href="https://doi.org/10.1016/j.ebiom.2023.104569">[DOI]</a>.</p></li>
<li id="id2031"><p>Yiming Zhang, Mo Su, Yu Chen, Zheng Wang, Jens Nielsen, and Zihe Liu. Engineering yeast mitochondrial metabolism for 3-hydroxypropionate production. <em>BIOTECHNOLOGY FOR BIOFUELS AND BIOPRODUCTS</em>, APR 8 2023. <a class="reference external" href="https://doi.org/10.1186/s13068-023-02309-z">[DOI]</a>.</p></li>
<li id="id2032"><p>Yanhua Liu and Hans V. Westerhoff. Competitive, multi-objective, and compartmented flux balance analysis for addressing tissue-specific inborn errors of metabolism. <em>JOURNAL OF INHERITED METABOLIC DISEASE</em>, 46(4):573–585, JUL 2023. <a class="reference external" href="https://doi.org/10.1002/jimd.12603">[DOI]</a>.</p></li>
<li id="id2033"><p>Jinyi Qian, Yuzhou Wang, Xiner Liu, Zijian Hu, Nan Xu, Yuetong Wang, Tianqiong Shi, and Chao Ye. Improving acetoin production through construction of a genome-scale metabolic model. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, MAY 2023. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2023.106833">[DOI]</a>.</p></li>
<li id="id2034"><p>Blas Blazquez, David San Leon, Antonia Rojas, Marta Tortajada, and Juan Nogales. New insights on metabolic features of &lt;i&gt;bacillus subtilis&lt;/i&gt; based on multistrain genome-scale metabolic modeling. <em>INTERNATIONAL JOURNAL OF MOLECULAR SCIENCES</em>, APR 2023. <a class="reference external" href="https://doi.org/10.3390/ijms24087091">[DOI]</a>.</p></li>
<li id="id2035"><p>Heewon Jung, Hyun-Seob Song, and Christof Meile. Complab v1.0: a scalable pore-scale model for flow, biogeochemistry, microbial metabolism, and biofilm dynamics. <em>GEOSCIENTIFIC MODEL DEVELOPMENT</em>, 16(6):1683–1696, MAR 27 2023. <a class="reference external" href="https://doi.org/10.5194/gmd-16-1683-2023">[DOI]</a>.</p></li>
<li id="id2036"><p>Jonathan Lo, Chao Wu, Jonathan R. Humphreys, Bin Yang, Zhenxiong Jiang, Xin Wang, PinChing Maness, Nicolas Tsesmetzis, and Wei Xiong. Thermodynamic and kinetic modeling directs pathway optimization for isopropanol production in a gas-fermenting bacterium. <em>MSYSTEMS</em>, APR 27 2023. <a class="reference external" href="https://doi.org/10.1128/msystems.01274-22">[DOI]</a>.</p></li>
<li id="id2037"><p>Charlotte Ramon and Jorg Stelling. Functional comparison of metabolic networks across species. <em>NATURE COMMUNICATIONS</em>, MAR 27 2023. <a class="reference external" href="https://doi.org/10.1038/s41467-023-37429-5">[DOI]</a>.</p></li>
<li id="id2038"><p>Kenta Miyoshi, Ryutaro Kawai, Teppei Niide, Yoshihiro Toya, and Hiroshi Shimizu. Functional evaluation of non-oxidative glycolysis in&lt;i&gt; escherichia&lt;/i&gt;&lt;i&gt; coli&lt;/i&gt; in the stationary phase under microaerobic conditions. <em>JOURNAL OF BIOSCIENCE AND BIOENGINEERING</em>, 135(4):291–297, APR 2023. <a class="reference external" href="https://doi.org/10.1016/j.jbiosc.2023.01.002">[DOI]</a>.</p></li>
<li id="id2039"><p>Amirhossein Shahbazinia, Saber Salehkaleybar, and Matin Hashemi. Paralingam: parallel causal structure learning for linear non-gaussian acyclic models. <em>JOURNAL OF PARALLEL AND DISTRIBUTED COMPUTING</em>, 176:114–127, JUN 2023. <a class="reference external" href="https://doi.org/10.1016/j.jpdc.2023.01.007">[DOI]</a>.</p></li>
<li id="id2040"><p>Benjamin Strain, James Morrissey, Athanasios Antonakoudis, and Cleo Kontoravdi. How reliable are chinese hamster ovary (cho) cell genome-scale metabolic models? <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 120(9, SI):2460–2478, SEP 2023. <a class="reference external" href="https://doi.org/10.1002/bit.28366">[DOI]</a>.</p></li>
<li id="id2041"><p>Philipp Wendering, Marius Arend, Zahra Razaghi-Moghadam, and Zoran Nikoloski. Data integration across conditions improves turnover number estimates and metabolic predictions. <em>NATURE COMMUNICATIONS</em>, MAR 17 2023. <a class="reference external" href="https://doi.org/10.1038/s41467-023-37151-2">[DOI]</a>.</p></li>
<li id="id2042"><p>Goncalo D. S. Correia, Julian R. Marchesi, and David A. MacIntyre. Moving beyond dna: towards functional analysis of the vaginal microbiome by non-sequencing-based methods. <em>CURRENT OPINION IN MICROBIOLOGY</em>, JUN 2023. <a class="reference external" href="https://doi.org/10.1016/j.mib.2023.102292">[DOI]</a>.</p></li>
<li id="id2043"><p>Maria Santos-Merino, Alvaro Gargantilla-Becerra, Fernando de la Cruz, and Juan Nogales. Highlighting the potential of &lt;i&gt;synechococcus elongatus&lt;/i&gt; pcc 7942 as platform to produce α-linolenic acid through an updated genome-scale metabolic modeling. <em>FRONTIERS IN MICROBIOLOGY</em>, MAR 14 2023. <a class="reference external" href="https://doi.org/10.3389/fmicb.2023.1126030">[DOI]</a>.</p></li>
<li id="id2044"><p>Neema Jamshidi, Kabir B. Nigam, and Sanjay K. Nigam. Loss of the kidney urate transporter, urat1, leads to disrupted redox homeostasis in mice. <em>ANTIOXIDANTS</em>, MAR 2023. <a class="reference external" href="https://doi.org/10.3390/antiox12030780">[DOI]</a>.</p></li>
<li id="id2045"><p>Nantia Leonidou, Alina Renz, Reihaneh Mostolizadeh, and Andreas Draeger. New workflow predicts drug targets against sars-cov-2 via metabolic changes in infected cells. <em>PLOS COMPUTATIONAL BIOLOGY</em>, MAR 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1010903">[DOI]</a>.</p></li>
<li id="id2046"><p>Takeyuki Tamura. Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: trimgdel. <em>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS</em>, 20(2):1540–1549, MAR-APR 2023. <a class="reference external" href="https://doi.org/10.1109/TCBB.2022.3185221">[DOI]</a>.</p></li>
<li id="id2047"><p>Christopher Riccardi, Marzia Calvanese, Veronica Ghini, Tania Alonso-Vasquez, Elena Perrin, Paola Turano, Giorgio Giurato, Alessandro Weisz, Ermenegilda Parrilli, Maria Luisa Tutino, and Marco Fondi. Metabolic robustness to growth temperature of a cold- adapted marine bacterium. <em>MSYSTEMS</em>, APR 27 2023. <a class="reference external" href="https://doi.org/10.1128/msystems.01124-22">[DOI]</a>.</p></li>
<li id="id2048"><p>Takeyuki Tamura, Ai Muto-fujita, Yukako Tohsato, and Tomoyuki Kosaka. Gene deletion algorithms for minimum reaction network design by mixed-integer linear programming for metabolite production in constraint-based models: gdel_minrn. <em>JOURNAL OF COMPUTATIONAL BIOLOGY</em>, 30(5):553–568, MAY 1 2023. <a class="reference external" href="https://doi.org/10.1089/cmb.2022.0352">[DOI]</a>.</p></li>
<li id="id2049"><p>Zhidong Zhang, Qi Guo, Jinyi Qian, Chao Ye, and He Huang. Construction and application of the genome-scale metabolic model of &lt;i&gt;streptomyces radiopugnans&lt;/i&gt;. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, FEB 17 2023. <a class="reference external" href="https://doi.org/10.3389/fbioe.2023.1108412">[DOI]</a>.</p></li>
<li id="id2050"><p>Ning Qin, Lingyun Li, Xu Ji, Rui Pereira, Yu Chen, Shile Yin, Chaokun Li, Xiaozhen Wan, Danye Qiu, Junfeng Jiang, Hao Luo, Yueping Zhang, Genlai Dong, Yiming Zhang, Shuobo Shi, Henning J. Jessen, Jianye Xia, Yun Chen, Christer Larsson, Tianwei Tan, Zihe Liu, and Jens Nielsen. Flux regulation through glycolysis and respiration is balanced by inositol pyrophosphates in yeast. <em>CELL</em>, 186(4):748+, FEB 16 2023. <a class="reference external" href="https://doi.org/10.1016/j.cell.2023.01.014">[DOI]</a>.</p></li>
<li id="id2051"><p>Bronson R. Weston and Ines Thiele. A nutrition algorithm to optimize feed and medium composition using genome-scale metabolic models. <em>METABOLIC ENGINEERING</em>, 76:167–178, MAR 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2023.01.010">[DOI]</a>.</p></li>
<li id="id2052"><p>Benedict Borer and Stefania Magnusdottir. The media composition as a crucial element in high-throughput metabolic network reconstruction. <em>INTERFACE FOCUS</em>, FEB 10 2023. <a class="reference external" href="https://doi.org/10.1098/rsfs.2022.0070">[DOI]</a>.</p></li>
<li id="id2053"><p>Frederick Clasen, Patricia M. Nunes, Gholamreza Bidkhori, Nourdine Bah, Stefan Boeing, Saeed Shoaie, and Dimitrios Anastasiou. Systematic diet composition swap in a mouse genome-scale metabolic model reveals determinants of diet metabolism in liver cancer. <em>ISCIENCE</em>, FEB 17 2023. <a class="reference external" href="https://doi.org/10.1016/j.isci.2023.106040">[DOI]</a>.</p></li>
<li id="id2054"><p>Guido Zampieri, Georgios Efthimiou, and Claudio Angione. Multi-dimensional experimental and computational exploration of metabolism pinpoints complex probiotic interactions. <em>METABOLIC ENGINEERING</em>, 76:120–132, MAR 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2023.01.008">[DOI]</a>.</p></li>
<li id="id2055"><p>Nicolas Gurdo, Daniel C. Volke, Douglas McCloskey, and Pablo Ivan Nikel. Automating the design-build-test-learn cycle towards next-generation bacterial cell factories. <em>NEW BIOTECHNOLOGY</em>, 74:1–15, MAY 25 2023. <a class="reference external" href="https://doi.org/10.1016/j.nbt.2023.01.002">[DOI]</a>.</p></li>
<li id="id2056"><p>Elif Esvap and Kutlu O. Ulgen. Neuroinflammation, energy and sphingolipid metabolism biomarkers are revealed by metabolic modeling of autistic brains. <em>BIOMEDICINES</em>, FEB 2023. <a class="reference external" href="https://doi.org/10.3390/biomedicines11020583">[DOI]</a>.</p></li>
<li id="id2057"><p>Qiu Shen, Hua Yang, Qing-Peng Kong, Gong-Hua Li, and Li Li. Metabolic modeling identifies a novel molecular type of glioblastoma associated with good prognosis. <em>METABOLITES</em>, FEB 2023. <a class="reference external" href="https://doi.org/10.3390/metabo13020172">[DOI]</a>.</p></li>
<li id="id2058"><p>Thomas Sevrin, Lisa Strasser, Camille Ternet, Philipp Junk, Miriam Caffarini, Stella Prins, Cian D'Arcy, Simona Catozzi, Giorgio Oliviero, Kieran Wynne, Christina Kiel, and Philip J. Luthert. Whole-cell energy modeling reveals quantitative changes of predicted energy flows in ras mutant cancer cell lines. <em>ISCIENCE</em>, FEB 17 2023. <a class="reference external" href="https://doi.org/10.1016/j.isci.2023.105931">[DOI]</a>.</p></li>
<li id="id2059"><p>Almut Heinken, Johannes Hertel, Geeta Acharya, Dmitry A. A. Ravcheev, Malgorzata Nyga, Onyedika Emmanuel Okpala, Marcus Hogan, Stefania Magnusdottir, Filippo Martinelli, Bram Nap, German Preciat, Janaka N. N. Edirisinghe, Christopher S. S. Henry, Ronan M. T. Fleming, and Ines Thiele. Genome-scale metabolic reconstruction of 7,302 human microorganisms for personalized medicine. <em>NATURE BIOTECHNOLOGY</em>, 41(9):1320+, SEP 2023. <a class="reference external" href="https://doi.org/10.1038/s41587-022-01628-0">[DOI]</a>.</p></li>
<li id="id2060"><p>Haoyang Yao, Sanjeev Dahal, and Laurence Yang. Novel context-specific genome-scale modelling explores the potential of triacylglycerol production by &lt;i&gt;chlamydomonas reinhardtii&lt;/i&gt;. <em>MICROBIAL CELL FACTORIES</em>, JAN 17 2023. <a class="reference external" href="https://doi.org/10.1186/s12934-022-02004-y">[DOI]</a>.</p></li>
<li id="id2061"><p>Yunpeng Zhao, Yiming Feng, Jianhang Zhou, Kuo Zhang, Jingqi Sun, Lina Wang, and Sitong Liu. Potential bacterial isolation by dosing metabolites in cross-feedings. <em>WATER RESEARCH</em>, MAR 1 2023. <a class="reference external" href="https://doi.org/10.1016/j.watres.2023.119589">[DOI]</a>.</p></li>
<li id="id2062"><p>M. Vidya Muthulakshmi, Aparajitha Srinivasan, and Smita Srivastava. Antioxidant green factories: toward sustainable production of vitamin e in plant&lt;i&gt; in&lt;/i&gt;&lt;i&gt; vitro&lt;/i&gt; cultures. <em>ACS OMEGA</em>, 2023 JAN 13 2023. <a class="reference external" href="https://doi.org/10.1021/acsomega.2c05819">[DOI]</a>.</p></li>
<li id="id2063"><p>William F. F. Kindschuh, Federico Baldini, Martin C. C. Liu, Jingqiu Liao, Yoli Meydan, Harry H. H. Lee, Almut Heinken, Ines Thiele, Christoph A. A. Thaiss, Maayan Levy, and Tal Korem. Preterm birth is associated with xenobiotics and predicted by the vaginal metabolome. <em>NATURE MICROBIOLOGY</em>, 8(2):246+, FEB 2023. <a class="reference external" href="https://doi.org/10.1038/s41564-022-01293-8">[DOI]</a>.</p></li>
<li id="id2064"><p>Nunthaphan Vikromvarasiri, Shuhei Noda, Tomokazu Shirai, and Akihiko Kondo. Investigation of two metabolic engineering approaches for (&lt;i&gt;r,r&lt;/i&gt;)-2,3-butanediol production from glycerol in &lt;i&gt;bacillus subtilis&lt;/i&gt;. <em>JOURNAL OF BIOLOGICAL ENGINEERING</em>, JAN 10 2023. <a class="reference external" href="https://doi.org/10.1186/s13036-022-00320-w">[DOI]</a>.</p></li>
<li id="id2065"><p>Emil Karlsen, Marianne Gylseth, Christian Schulz, and Eivind Almaas. A study of a diauxic growth experiment using an expanded dynamic flux balance framework. <em>PLOS ONE</em>, JAN 6 2023. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0280077">[DOI]</a>.</p></li>
<li id="id2066"><p>Clara Correia-Melo, Stephan Kamrad, Roland Tengoelics, Christoph B. Messner, Pauline Trebulle, StJohn Townsend, Sreejith Jayasree Varma, Anja Freiwald, Benjamin M. Heineike, Kate Campbell, Lucia Herrera-Dominguez, Simran Kaur Aulakh, Lukasz Szyrwiel, Jason S. L. Yu, Aleksej Zelezniak, Vadim Demichev, Michael Muelleder, Balazs Papp, Mohammad Tauqeer Alam, and Markus Rasler. Cell-cell metabolite exchange creates a pro-survival metabolic environment that extends lifespan. <em>CELL</em>, 186(1):63+, JAN 5 2023. <a class="reference external" href="https://doi.org/10.1016/j.cell.2022.12.007">[DOI]</a>.</p></li>
<li id="id2067"><p>Philip Fernandes, Yash Sharma, Fatima Zulqarnain, Brooklyn McGrew, Aman Shrivastava, Lubaina Ehsan, Dawson Payne, Lillian Dillard, Deborah Powers, Isabelle Aldridge, Jason Matthews, Subra Kugathasan, Facundo M. Fernandez, David Gaul, Jason A. Papin, and Sana Syed. Identifying metabolic shifts in crohn's disease using' omics-driven contextualized computational metabolic network models. <em>SCIENTIFIC REPORTS</em>, JAN 5 2023. <a class="reference external" href="https://doi.org/10.1038/s41598-022-26816-5">[DOI]</a>.</p></li>
<li id="id2068"><p>Romeu Viana, Tiago Carreiro, Diogo Couceiro, Oscar Dias, Isabel Rocha, and Miguel Cacho Teixeira. Metabolic reconstruction of the human pathogen candida auris: using a cross-species approach for drug target prediction. <em>FEMS YEAST RESEARCH</em>, JAN 4 2023. <a class="reference external" href="https://doi.org/10.1093/femsyr/foad045">[DOI]</a>.</p></li>
<li id="id2069"><p>Miha Moskon and Tadeja Rezen. Context-specific genome-scale metabolic modelling and its application to the analysis of covid-19 metabolic signatures. <em>METABOLITES</em>, JAN 2023. <a class="reference external" href="https://doi.org/10.3390/metabo13010126">[DOI]</a>.</p></li>
<li id="id2070"><p>Saratram Gopalakrishnan, Chintan J. Joshi, Miguel A. Valderrama-Gomez, Elcin Icten, Pablo Rolandi, William Johnson, Cleo Kontoravdi, and Nathan E. Lewis. Guidelines for extracting biologically relevant context-specific metabolic models using gene expression data. <em>METABOLIC ENGINEERING</em>, 75:181–191, JAN 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2022.12.003">[DOI]</a>.</p></li>
<li id="id2080"><p>Dalimil Bujdos, Barbora Popelarova, Daniel C. Volke, Pablo I. Nikel, Nikolaus Sonnenschein, and Pavel Dvorak. Engineering of&lt;i&gt; pseudomonas&lt;/i&gt;&lt;i&gt; putida&lt;/i&gt; for accelerated co-utilization of glucose and cellobiose yields aerobic overproduction of pyruvate explained by an upgraded metabolic model. <em>METABOLIC ENGINEERING</em>, 75:29–46, JAN 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2022.10.011">[DOI]</a>.</p></li>
<li id="id2082"><p>Xuan Cao, Wei Yu, Yu Chen, Shan Yang, Zongbao K. Zhao, Jens Nielsen, Hongwei Luan, and Yongjin J. Zhou. Engineering yeast for high-level production of diterpenoid sclareol. <em>METABOLIC ENGINEERING</em>, 75:19–28, JAN 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2022.11.002">[DOI]</a>.</p></li>
<li id="id2083"><p>Zhendong Li, Cong Gao, Chao Ye, Liang Guo, Jia Liu, Xiulai Chen, Wei Song, Jing Wu, and Liming Liu. Systems engineering of escherichia coli for high-level shikimate production. <em>METABOLIC ENGINEERING</em>, 75:1–11, JAN 2023. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2022.10.010">[DOI]</a>.</p></li>
<li id="id2093"><p>Mahsa Mekanik, Reza Fotovat, Ehsan Motamedian, and Vahab Jafarian. Improvement of lutein production in &lt;i&gt;auxenochlorella protothecoides&lt;/i&gt; using its genome-scale metabolic model and a system-oriented approach. <em>APPLIED BIOCHEMISTRY AND BIOTECHNOLOGY</em>, 195(2):889–904, FEB 2023. <a class="reference external" href="https://doi.org/10.1007/s12010-022-04186-y">[DOI]</a>.</p></li>
<li id="id2153"><p>Tobias Mueller, Simon Schick, Jonathan Beck, Georg Sprenger, and Ralf Takors. Synthetic mutualism in engineered &lt;i&gt;e. coli&lt;/i&gt; mutant strains as functional basis for microbial production consortia. <em>ENGINEERING IN LIFE SCIENCES</em>, JAN 2023. <a class="reference external" href="https://doi.org/10.1002/elsc.202100158">[DOI]</a>.</p></li>
<li id="id2242"><p>Manvel Gasparyan, Arnout Van Messem, and Shodhan Rao. Parameter estimation for models of chemical reaction networks from experimental data of reaction rates. <em>INTERNATIONAL JOURNAL OF CONTROL</em>, 96(2):392–407, FEB 1 2023. <a class="reference external" href="https://doi.org/10.1080/00207179.2021.1998636">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-2" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-2" name="0-2" role="tabpanel" tabindex="0"><div class="docutils container" id="id3641">
<ol class="arabic simple" start="1">
<li id="id3891"><p>Juan D. Tibocha-Bonilla, Cristal Zuniga, Asama Lekbua, Colton Lloyd, Kevin Rychel, Katie Short, and Karsten Zengler. Predicting stress response and improved protein overproduction in &lt;i&gt;bacillus subtilis&lt;/i&gt;. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, DEC 27 2022. <a class="reference external" href="https://doi.org/10.1038/s41540-022-00259-0">[DOI]</a>.</p></li>
<li id="id3892"><p>Gabriela Diaz-Tang, Estefania Marin Meneses, Kavish Patel, Sophia Mirkin, Laura Garcia-Dieguez, Camryn Pajon, Ivana Barraza, Vijay Patel, Helana Ghali, Angelica P. Tracey, Christopher A. Blanar, Allison J. Lopatkin, and Robert P. Smith. Growth productivity as a determinant of the inoculum effect for bactericidal antibiotics. <em>SCIENCE ADVANCES</em>, DEC 14 2022. <a class="reference external" href="https://doi.org/10.1126/sciadv.add0924">[DOI]</a>.</p></li>
<li id="id3893"><p>Lokanand Koduru, Meiyappan Lakshmanan, Yi Qing Lee, Pooi-Leng Ho, Pei-Yu Lim, Wei Xuan Ler, Say Kong Ng, Dongseok Kim, Doo-Sang Park, Mazlina Banu, Dave Siak Wei Ow, and Dong-Yup Lee. Systematic evaluation of genome-wide metabolic landscapes in lactic acid bacteria reveals diet- and strain-specific probiotic idiosyncrasies. <em>CELL REPORTS</em>, DEC 6 2022. <a class="reference external" href="https://doi.org/10.1016/j.celrep.2022.111735">[DOI]</a>.</p></li>
<li id="id3894"><p>Freddy Castillo-Alfonso, Alejandro Quintana-Menendez, Gabriel Vigueras-Ramirez, Alfonso Mauricio Sales-Cruz, Luis Manuel Rosales-Colunga, and Roberto Olivares-Hernandez. Analysis of the propionate metabolism in &lt;i&gt;bacillus subtilis&lt;/i&gt; during 3-indolacetic production. <em>MICROORGANISMS</em>, DEC 2022. <a class="reference external" href="https://doi.org/10.3390/microorganisms10122352">[DOI]</a>.</p></li>
<li id="id3895"><p>Piubeli Francine. Systems biology: new insight into antibiotic resistance. <em>MICROORGANISMS</em>, DEC 2022. <a class="reference external" href="https://doi.org/10.3390/microorganisms10122362">[DOI]</a>.</p></li>
<li id="id3896"><p>Maria Pires Pacheco, Jimmy Ji, Tessy Prohaska, Maria Moscardo Garcia, and Thomas Sauter. Scfastcormics: a contextualization algorithm to reconstruct metabolic multi-cell population models from single-cell rnaseq data. <em>METABOLITES</em>, DEC 2022. <a class="reference external" href="https://doi.org/10.3390/metabo12121211">[DOI]</a>.</p></li>
<li id="id3897"><p>Carles Foguet, Yu Xu, Scott C. Ritchie, Samuel A. Lambert, Elodie Persyn, Artika P. Nath, Emma E. Davenport, David J. Roberts, Dirk S. Paul, Emanuele Di Angelantonio, John Danesh, Adam S. Butterworth, Christopher Yau, and Michael Inouye. Genetically personalised organ-specific metabolic models in health and disease. <em>NATURE COMMUNICATIONS</em>, NOV 29 2022. <a class="reference external" href="https://doi.org/10.1038/s41467-022-35017-7">[DOI]</a>.</p></li>
<li id="id3898"><p>Bin Han, Zeyu Dai, and Zhimin Li. Computer-based design of a cell factory for high-yield cytidine production. <em>ACS SYNTHETIC BIOLOGY</em>, 11(12):4123–4133, DEC 16 2022. <a class="reference external" href="https://doi.org/10.1021/acssynbio.2c00431">[DOI]</a>.</p></li>
<li id="id3899"><p>Tamizhini Loganathan and George Priya Doss C. The influence of machine learning technologies in gut microbiome research and cancer studies - a review. <em>LIFE SCIENCES</em>, DEC 15 2022. <a class="reference external" href="https://doi.org/10.1016/j.lfs.2022.121118">[DOI]</a>.</p></li>
<li id="id3901"><p>Evangelia Vayena, Anush Chiappino-Pepe, Homa MohammadiPeyhani, Yannick Francioli, Noushin Hadadi, Meric Ataman, Jasmin Hafner, Stavros Pavlou, and Vassily Hatzimanikatis. A workflow for annotating the knowledge gaps in metabolic reconstructions using known and hypothetical reactions. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, NOV 15 2022. <a class="reference external" href="https://doi.org/10.1073/pnas.2211197119">[DOI]</a>.</p></li>
<li id="id3904"><p>Giuseppe Magazzu, Guido Zampieri, and Claudio Angione. Clinical stratification improves the diagnostic accuracy of small omics datasets within machine learning and genome-scale metabolic modelling methods? <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, DEC 2022. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2022.106244">[DOI]</a>.</p></li>
<li id="id3905"><p>Kaustubh R. Sawant, Aditya P. Sarnaik, Prashant Savvashe, Nima Hajinajaf, Parker Poole, Arul M. Varman, Arvind Lali, and Reena Pandit. One cell-two wells bio-refinery: demonstrating cyanobacterial chassis for co-production of heterologous and natural hydrocarbons. <em>BIORESOURCE TECHNOLOGY</em>, NOV 2022. <a class="reference external" href="https://doi.org/10.1016/j.biortech.2022.127921">[DOI]</a>.</p></li>
<li id="id3906"><p>Alicja K. Warda, Eugene M. Dempsey, Sofia D. Forssten, C. Anthony Ryan, John F. Cryan, Elaine Patterson, Mairead N. O'Riordan, Carol-Anne O'Shea, Finola Keohane, Grainne Meehan, Orlagh O'Connor, R. Paul Ross, and Catherine Stanton. Cross-sectional observational study protocol: missing microbes in infants born by caesarean section (mimic): antenatal antibiotics and mode of delivery. <em>BMJ OPEN</em>, NOV 2022. <a class="reference external" href="https://doi.org/10.1136/bmjopen-2022-064398">[DOI]</a>.</p></li>
<li id="id3907"><p>Neema Jamshidi and Sanjay K. Nigam. Drug transporters oat1 and oat3 have specific effects on multiple organs and gut microbiome as revealed by contextualized metabolic network reconstructions. <em>SCIENTIFIC REPORTS</em>, OCT 31 2022. <a class="reference external" href="https://doi.org/10.1038/s41598-022-21091-w">[DOI]</a>.</p></li>
<li id="id3908"><p>Kathryne C. Ford, Joshua A. M. Kaste, Yair Shachar-Hill, and Michaela A. TerAvest. Flux-balance analysis and mobile crispri-guided deletion of a conditionally essential gene in&lt;i&gt; shewanella&lt;/i&gt;&lt;i&gt; oneidensis&lt;/i&gt; mr-1. <em>ACS SYNTHETIC BIOLOGY</em>, 11(10):3405–3413, OCT 21 2022. <a class="reference external" href="https://doi.org/10.1021/acssynbio.2c00323">[DOI]</a>.</p></li>
<li id="id3909"><p>Carles Ciurans, Josep M. Guerrero, Ivan Martinez-Mongue, Claude G. Dussap, Igor Marin de Mas, and Francesc Godia. Enhancing control systems of higher plant culture chambers via multilevel structural mechanistic modelling. <em>FRONTIERS IN PLANT SCIENCE</em>, OCT 20 2022. <a class="reference external" href="https://doi.org/10.3389/fpls.2022.970410">[DOI]</a>.</p></li>
<li id="id3910"><p>Joao R. C. Ramos, Gil P. Oliveira, Patrick Dumas, and Rui Oliveira. Genome-scale modeling of chinese hamster ovary cells by hybrid semi-parametric flux balance analysis. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 45(11):1889–1904, NOV 2022. <a class="reference external" href="https://doi.org/10.1007/s00449-022-02795-9">[DOI]</a>.</p></li>
<li id="id3911"><p>Hamza Umut Karakurt and Pinar Pir. In silico analysis of metabolic effects of bipolar disorder on prefrontal cortex identified altered gaba, glutamate-glutamine cycle, energy metabolism and amino acid synthesis pathways. <em>INTEGRATIVE BIOLOGY</em>, 14(6):127–136, OCT 21 2022. <a class="reference external" href="https://doi.org/10.1093/intbio/zyac012">[DOI]</a>.</p></li>
<li id="id3912"><p>Sankalpa Venkatraghavan, Sathvik Anantakrishnan, and Karthik Raman. Probing patterning in microbial consortia with a cellular automaton for spatial organisation. <em>SCIENTIFIC REPORTS</em>, OCT 13 2022. <a class="reference external" href="https://doi.org/10.1038/s41598-022-20705-7">[DOI]</a>.</p></li>
<li id="id3914"><p>Kathryne C. Ford, Joshua A. M. Kaste, Yair Shachar-Hill, and Michaela A. TerAvest. Flux-balance analysis and mobile crispri-guided deletion of a conditionally essential gene in&lt;i&gt; shewanella&lt;/i&gt;&lt;i&gt; oneidensis&lt;/i&gt; mr-1. <em>ACS SYNTHETIC BIOLOGY</em>, 2022 OCT 11 2022. <a class="reference external" href="https://doi.org/10.1021/acssynbio.2c00323">[DOI]</a>.</p></li>
<li id="id3915"><p>Priyanka Baloni, Matthias Arnold, Luna Buitrago, Kwangsik Nho, Herman Moreno, Kevin Huynh, Barbara Brauner, Gregory Louie, Alexandra Kueider-Paisley, Karsten Suhre, Andrew J. Saykin, Kim Ekroos, Peter J. Meikle, Leroy Hood, Nathan D. Price, P. Murali Doraiswamy, Cory C. Funk, A. Ivan Hernandez, Gabi Kastenmueller, Rebecca Baillie, Xianlin Han, Rima Kaddurah-Daouk, and Alzheimer's Dis Metabolomics Conso. Multi-omic analyses characterize the ceramide/sphingomyelin pathway as a therapeutic target in alzheimer's disease. <em>COMMUNICATIONS BIOLOGY</em>, OCT 8 2022. <a class="reference external" href="https://doi.org/10.1038/s42003-022-04011-6">[DOI]</a>.</p></li>
<li id="id3916"><p>Jungyeon Kim, Yu Eun Cheong, Sora Yu, Yong-Su Jin, and Kyoung Heon Kim. Strain engineering and metabolic flux analysis of a probiotic yeast &lt;i&gt;saccharomyces boulardii&lt;/i&gt; for metabolizing l-fucose, a mammalian mucin component. <em>MICROBIAL CELL FACTORIES</em>, OCT 7 2022. <a class="reference external" href="https://doi.org/10.1186/s12934-022-01926-x">[DOI]</a>.</p></li>
<li id="id3917"><p>Jeffry C. Granados, Kian Falah, Imhoi Koo, Ethan W. Morgan, Gary H. Perdew, Andrew D. Patterson, Neema Jamshidi, and Sanjay K. Nigam. Ahr is a master regulator of diverse pathways in endogenous metabolism. <em>SCIENTIFIC REPORTS</em>, OCT 5 2022. <a class="reference external" href="https://doi.org/10.1038/s41598-022-20572-2">[DOI]</a>.</p></li>
<li id="id3918"><p>Prabir Kumar Das, Ansuman Sahoo, and Venkata Dasu Veeranki. Current status, and the developments of hosts and expression systems for the production of recombinant human cytokines. <em>BIOTECHNOLOGY ADVANCES</em>, OCT 2022. <a class="reference external" href="https://doi.org/10.1016/j.biotechadv.2022.107969">[DOI]</a>.</p></li>
<li id="id3919"><p>Elif Gencturk and Kutlu O. Ulgen. Understanding hmf inhibition on yeast growth coupled with ethanol production for the improvement of bio-based industrial processes. <em>PROCESS BIOCHEMISTRY</em>, 121:425–438, OCT 2022. <a class="reference external" href="https://doi.org/10.1016/j.procbio.2022.07.015">[DOI]</a>.</p></li>
<li id="id3920"><p>Paula Jouhten, Dimitrios Konstantinidis, Filipa Pereira, Sergej Andrejev, Kristina Grkovska, Sandra Castillo, Payam Ghiachi, Gemma Beltran, Eivind Almaas, Albert Mas, Jonas Warringer, Ramon Gonzalez, Pilar Morales, and Kiran R. Patil. Predictive evolution of metabolic phenotypes using model-designed environments. <em>MOLECULAR SYSTEMS BIOLOGY</em>, OCT 2022. <a class="reference external" href="https://doi.org/10.15252/msb.202210980">[DOI]</a>.</p></li>
<li id="id3921"><p>Maxime R. F. Gosselin, Virginie Mournetas, Malgorzata Borczyk, Suraj Verma, Annalisa Occhipinti, Justyna Rog, Lukasz Bozycki, Michal Korostynski, Samuel C. Robson, Claudio Angione, Christian Pinset, and Dariusz C. Gorecki. Loss of full-length dystrophin expression results in major cell-autonomous abnormalities in proliferating myoblasts. <em>ELIFE</em>, SEP 27 2022. <a class="reference external" href="https://doi.org/10.7554/eLife.75521">[DOI]</a>.</p></li>
<li id="id3922"><p>Abhishek Subramanian, Pooya Zakeri, Mira Mousa, Halima Alnaqbi, Fatima Yousif Alshamsi, Leo Bettoni, Ernesto Damiani, Habiba Alsafar, Yvan Saeys, and Peter Carmeliet. Angiogenesis goes computational - the future way forward to discover new angiogenic targets? <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 20:5235–5255, 2022. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2022.09.019">[DOI]</a>.</p></li>
<li id="id3923"><p>Abhinav Achreja, Tao Yu, Anjali Mittal, Srinadh Choppara, Olamide Animasahun, Minal Nenwani, Fulei Wuchu, Noah Meurs, Aradhana Mohan, Jin Heon Jeon, Itisam Sarangi, Anusha Jayaraman, Sarah Owen, Reva Kulkarni, Michele Cusato, Frank Weinberg, Hye Kyong Kweon, Chitra Subramanian, Max S. Wicha, Sofia D. Merajver, Sunitha Nagrath, Kathleen R. Cho, Analisa DiFeo, Xiongbin Lu, and Deepak Nagrath. Metabolic collateral lethal target identification reveals mthfd2 paralogue dependency in ovarian cancer. <em>NATURE METABOLISM</em>, 4(9):1119+, SEP 2022. <a class="reference external" href="https://doi.org/10.1038/s42255-022-00636-3">[DOI]</a>.</p></li>
<li id="id3924"><p>Zhepu Ruan, Mengjun Xu, Youwen Xing, Qi Jiang, Bingang Yang, Jiandong Jiang, and Xihui Xu. Interspecies metabolic interactions in a synergistic consortium drive efficient degradation of the herbicide bromoxynil octanoate. <em>JOURNAL OF AGRICULTURAL AND FOOD CHEMISTRY</em>, 70(37):11613–11622, SEP 21 2022. <a class="reference external" href="https://doi.org/10.1021/acs.jafc.2c03057">[DOI]</a>.</p></li>
<li id="id3925"><p>Philipp Schneider, Pavlos Stephanos Bekiaris, Axel von Kamp, and Steffen Klamt. Straindesign: a comprehensive python package for computational design of metabolic networks. <em>BIOINFORMATICS</em>, 38(21):4981–4983, OCT 31 2022. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btac632">[DOI]</a>.</p></li>
<li id="id3926"><p>Aidin Behravan, Atieh Hashemi, Sayed-Amir Marashi, and Hamideh Fouladiha. Genome-scale metabolic model-based engineering of &lt;i&gt;escherichia coli&lt;/i&gt; enhances recombinant single-chain antibody fragment production. <em>BIOTECHNOLOGY LETTERS</em>, 44(10):1231–1242, OCT 2022. <a class="reference external" href="https://doi.org/10.1007/s10529-022-03301-7">[DOI]</a>.</p></li>
<li id="id3927"><p>Philipp Keller, Michael A. Reiter, Patrick Kiefer, Thomas Gassler, Lucas Hemmerle, Philipp Christen, Elad Noor, and Julia A. Vorholt. Generation of an &lt;i&gt;escherichia coli&lt;/i&gt; strain growing on methanol via the ribulose monophosphate cycle. <em>NATURE COMMUNICATIONS</em>, SEP 6 2022. <a class="reference external" href="https://doi.org/10.1038/s41467-022-32744-9">[DOI]</a>.</p></li>
<li id="id3928"><p>Yuan-Hang Du, Min-Yu Wang, Lin-Hui Yang, Ling-Ling Tong, Dong-Sheng Guo, and Xiao-Jun Ji. Optimization and scale-up of fermentation processes driven by models. <em>BIOENGINEERING-BASEL</em>, SEP 2022. <a class="reference external" href="https://doi.org/10.3390/bioengineering9090473">[DOI]</a>.</p></li>
<li id="id3929"><p>Nirvana Nursimulu, Alan M. Moses, and John Parkinson. Architect: a tool for aiding the reconstruction of high-quality metabolic models through improved enzyme annotation. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2022. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1010452">[DOI]</a>.</p></li>
<li id="id3930"><p>Simone Schito, Rico Zuchowski, Daniel Bergen, Daniel Strohmeier, Bastian Wollenhaupt, Philipp Menke, Johannes Seiffarth, Katharina Noeh, Dietrich Kohlheyer, Michael Bott, Wolfgang Wiechert, Meike Baumgart, and Stephan Noack. Communities of niche-optimized strains (conos) - design and creation of stable, genome-reduced co-cultures. <em>METABOLIC ENGINEERING</em>, 73:91–103, SEP 2022. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2022.06.004">[DOI]</a>.</p></li>
<li id="id3931"><p>Ines Thiele, German Preciat, and Ronan M. T. Fleming. Metaboannotator: an efficient toolbox to annotate metabolites in genome-scale metabolic reconstructions. <em>BIOINFORMATICS</em>, 38(20):4831–4832, OCT 14 2022. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btac596">[DOI]</a>.</p></li>
<li id="id3932"><p>Satyajit Beura, Pritam Kundu, Amit Kumar Das, and Amit Ghosh. Metagenome-scale community metabolic modelling for understanding the role of gut microbiota in human health. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, OCT 2022. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2022.105997">[DOI]</a>.</p></li>
<li id="id3933"><p>Belen Hirmas, Naschla Gasaly, Guillermo Orellana, Marco Vega-Sagardia, Pedro Saa, Martin Gotteland, and Daniel Garrido. Metabolic modeling and bidirectional culturing of two gut microbes reveal cross-feeding interactions and protective effects on intestinal cells. <em>MSYSTEMS</em>, OCT 26 2022. <a class="reference external" href="https://doi.org/10.1128/msystems.00646-22">[DOI]</a>.</p></li>
<li id="id3934"><p>Alan R. Pacheco, Charlie Pauvert, Dileep Kishore, and Daniel Segre. Toward fair representations of microbial interactions. <em>MSYSTEMS</em>, OCT 26 2022. <a class="reference external" href="https://doi.org/10.1128/msystems.00659-22">[DOI]</a>.</p></li>
<li id="id3935"><p>Anoop T. Ambikan, Hong Yang, Shuba Krishnan, Sara Svensson Akusjarvi, Soham Gupta, Magda Lourda, Maike Sperk, Muhammad Arif, Chenq Zhang, Hampus Nordqvist, Sivasankaran Munusamy Ponnan, Anders Sonnerborg, Carl Johan Treutiger, Liam O'Mahony, Adil Mardinoglu, Rui Benfeitas, and Ujjwal Neogi. Multi-omics personalized network analyses highlight progressive disruption of central metabolism associated with covid-19 severity. <em>CELL SYSTEMS</em>, 13(8):665+, AUG 17 2022. <a class="reference external" href="https://doi.org/10.1016/j.cels.2022.06.006">[DOI]</a>.</p></li>
<li id="id3936"><p>J. Ashwini John, Melvin S. Samuel, Muthusamy Govarthanan, and Ethiraj Selvarajan. A comprehensive review on strategic study of cellulase producing marine actinobacteria for biofuel applications. <em>ENVIRONMENTAL RESEARCH</em>, NOV 2022. <a class="reference external" href="https://doi.org/10.1016/j.envres.2022.114018">[DOI]</a>.</p></li>
<li id="id3937"><p>Raja Murugan, Archanaa Sundararaghavan, Navdeep K. Dhami, Abhijit Mukherjee, and G. K. Suraishkumar. Importance of carbon to nitrogen ratio in microbial cement production: insights through experiments and genome-scale metabolic modelling. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, AUG 2022. <a class="reference external" href="https://doi.org/10.1016/j.bej.2022.108573">[DOI]</a>.</p></li>
<li id="id3938"><p>Ryutaro Kawai, Yoshihiro Toya, and Hiroshi Shimizu. Metabolic pathway design for growth-associated phenylalanine production using synthetically designed mutualism. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 45(9):1539–1546, SEP 2022. <a class="reference external" href="https://doi.org/10.1007/s00449-022-02762-4">[DOI]</a>.</p></li>
<li id="id3939"><p>J. Carlsen, H. H. Henriksen, I. Marin de Mas, and P. Johansson, I. An explorative metabolomic analysis of the endothelium in pulmonary hypertension. <em>SCIENTIFIC REPORTS</em>, AUG 2 2022. <a class="reference external" href="https://doi.org/10.1038/s41598-022-17374-x">[DOI]</a>.</p></li>
<li id="id3940"><p>Handan Cetin, Zeynep Petek Cakar, and Kutlu O. Ulgen. Understanding the adaptive laboratory evolution of multiple stress-resistant yeast strains by genome-scale modeling. <em>YEAST</em>, 39(8):449–465, AUG 2022. <a class="reference external" href="https://doi.org/10.1002/yea.3806">[DOI]</a>.</p></li>
<li id="id3941"><p>Pattsarun Cheawchanlertfa, Suwalak Chitcharoen, Nachon Raethong, Qing Liu, Pramote Chumnanpuen, Panyawarin Soommat, Yuanda Song, Mattheos Koffas, Kobkul Laoteng, and Wanwipa Vongsangnak. Enhancing genome-scale model by integrative exometabolome and transcriptome: unveiling carbon assimilation towards sphingolipid biosynthetic capability of &lt;i&gt;cordyceps militaris&lt;/i&gt;. <em>JOURNAL OF FUNGI</em>, AUG 2022. <a class="reference external" href="https://doi.org/10.3390/jof8080887">[DOI]</a>.</p></li>
<li id="id3942"><p>Subham Choudhury, Michael Moret, Pierre Salvy, Daniel Weilandt, Vassily Hatzimanikatis, and Ljubisa Miskovic. Reconstructing kinetic models for dynamical studies of metabolism using generative adversarial networks. <em>NATURE MACHINE INTELLIGENCE</em>, 4(8):710+, AUG 2022. <a class="reference external" href="https://doi.org/10.1038/s42256-022-00519-y">[DOI]</a>.</p></li>
<li id="id3943"><p>D. S. Dvoretsky, M. S. Temnov, I. Markin, V, Ya Ustinskaya, V, and M. A. Es'kova. Problems in the development of efficient biotechnology for the synthesis of valuable components from microalgae biomass. <em>THEORETICAL FOUNDATIONS OF CHEMICAL ENGINEERING</em>, 56(4):425–439, AUG 2022. <a class="reference external" href="https://doi.org/10.1134/S0040579522040224">[DOI]</a>.</p></li>
<li id="id3944"><p>Ali Kishk, Maria Pires Pacheco, Tony Heurtaux, Lasse Sinkkonen, Jun Pang, Sabrina Fritah, Simone P. Niclou, and Thomas Sauter. Review of current human genome-scale metabolic models for brain cancer and neurodegenerative diseases. <em>CELLS</em>, AUG 2022. <a class="reference external" href="https://doi.org/10.3390/cells11162486">[DOI]</a>.</p></li>
<li id="id3945"><p>Sebastian Huss, Rika Siedah Judd, Kaan Koper, Hiroshi A. Maeda, and Zoran Nikoloski. An automated workflow that generates atom mappings for large-scale metabolic models and its application to &lt;i&gt;arabidopsis thaliana&lt;/i&gt;. <em>PLANT JOURNAL</em>, 111(5):1486–1500, SEP 2022. <a class="reference external" href="https://doi.org/10.1111/tpj.15903">[DOI]</a>.</p></li>
<li id="id3946"><p>Chao Ye, Xinyu Wei, Tianqiong Shi, Xiaoman Sun, Nan Xu, Cong Gao, and Wei Zou. Genome-scale metabolic network models: from first-generation to next-generation. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 106(13-16):4907–4920, AUG 2022. <a class="reference external" href="https://doi.org/10.1007/s00253-022-12066-y">[DOI]</a>.</p></li>
<li id="id3947"><p>Francesco Balzerani, Daniel Hinojosa-Nogueira, Xabier Cendoya, Telmo Blasco, Sergio Perez-Burillo, Inigo Apaolaza, M. Pilar Francino, Jose Angel Rufian-Henares, and Francisco J. Planes. Prediction of degradation pathways of phenolic compounds in the human gut microbiota through enzyme promiscuity methods. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, JUL 12 2022. <a class="reference external" href="https://doi.org/10.1038/s41540-022-00234-9">[DOI]</a>.</p></li>
<li id="id3948"><p>Jong Kwang Hong, Dong-Hyuk Choi, Seo-Young Park, Yaron R. Silberberg, Fumi Shozui, Eiji Nakamura, Takashi Kayahara, and Dong-Yup Lee. Data-driven and model-guided systematic framework for media development in cho cell culture. <em>METABOLIC ENGINEERING</em>, 73:114–123, SEP 2022. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2022.07.003">[DOI]</a>.</p></li>
<li id="id3949"><p>Rachel H. Ng, Jihoon W. Lee, Priyanka Baloni, Christian Diener, James R. Heath, and Yapeng Su. Constraint-based reconstruction and analyses of metabolic models: open-source python tools and applications to cancer. <em>FRONTIERS IN ONCOLOGY</em>, JUL 7 2022. <a class="reference external" href="https://doi.org/10.3389/fonc.2022.914594">[DOI]</a>.</p></li>
<li id="id3950"><p>Antonino Baez, Ashish K. Sharma, Andrey Bryukhanov, Eric D. Anderson, Leba Rudack, Roberto Olivares-Hernandez, David Quan, and Joseph Shiloach. Iron availability enhances the cellular energetics of aerobic escherichia coli cultures while upregulating anaerobic respiratory chains. <em>NEW BIOTECHNOLOGY</em>, 71:11–20, NOV 25 2022. <a class="reference external" href="https://doi.org/10.1016/j.nbt.2022.06.004">[DOI]</a>.</p></li>
<li id="id3951"><p>Claudio Tomi-Andrino, Alina Pandele, Klaus Winzer, John King, Ruman Rahman, and Dong-Hyun Kim. Metabolic modeling-based drug repurposing in glioblastoma. <em>SCIENTIFIC REPORTS</em>, JUL 1 2022. <a class="reference external" href="https://doi.org/10.1038/s41598-022-14721-w">[DOI]</a>.</p></li>
<li id="id3952"><p>Ivan Domenzain, Benjamin Sanchez, Mihail Anton, Eduard J. Kerkhoven, Aaron Millan-Oropeza, Celine Henry, Verena Siewers, John P. Morrissey, Nikolaus Sonnenschein, and Jens Nielsen. Reconstruction of a catalogue of genome-scale metabolic models with enzymatic constraints using gecko 2.0. <em>NATURE COMMUNICATIONS</em>, JUN 30 2022. <a class="reference external" href="https://doi.org/10.1038/s41467-022-31421-1">[DOI]</a>.</p></li>
<li id="id3953"><p>Andreas Wagner. Competition for nutrients increases invasion resistance during assembly of microbial communities. <em>MOLECULAR ECOLOGY</em>, 31(15):4188–4203, AUG 2022. <a class="reference external" href="https://doi.org/10.1111/mec.16565">[DOI]</a>.</p></li>
<li id="id3954"><p>Muberra Fatma Cesur, Tunahan Cakir, and Pinar Pir. Genome-wide analysis of yeast metabolic cycle through metabolic network models reveals superiority of integrated atac-seq data over rna-seq data. <em>MSYSTEMS</em>, JUN 28 2022. <a class="reference external" href="https://doi.org/10.1128/msystems.01347-21">[DOI]</a>.</p></li>
<li id="id3955"><p>Jingjing Li, Yifei Gou, Jiarui Yang, Lingxuan Zhao, Bin Wang, Tong Hao, and Jinsheng Sun. Genome-scale metabolic network model of &lt;i&gt;eriocheir sinensis i&lt;/i&gt;crab4665 and nutritional requirement analysis. <em>BMC GENOMICS</em>, JUN 28 2022. <a class="reference external" href="https://doi.org/10.1186/s12864-022-08698-z">[DOI]</a>.</p></li>
<li id="id3956"><p>Patrick F. Suthers and Costas D. Maranas. Examining organic acid production potential and growth-coupled strategies in &lt;i&gt;issatchenkia orientalis&lt;/i&gt; using constraint-based modeling. <em>BIOTECHNOLOGY PROGRESS</em>, SEP 2022. <a class="reference external" href="https://doi.org/10.1002/btpr.3276">[DOI]</a>.</p></li>
<li id="id3957"><p>Menglei Xia, Di Wang, Yiming Xia, Haijiao Shi, Zhongyu Tian, Yu Zheng, and Min Wang. Oxidoreduction potential controlling for increasing the fermentability of enzymatically hydrolyzed steam-exploded corn stover for butanol production. <em>MICROBIAL CELL FACTORIES</em>, JUN 27 2022. <a class="reference external" href="https://doi.org/10.1186/s12934-022-01824-2">[DOI]</a>.</p></li>
<li id="id3958"><p>Jun Feng, Xiaolong Guo, Feifei Cai, Hongxin Fu, and Jufang Wang. Model-based driving mechanism analysis for butyric acid production in &lt;i&gt;clostridium tyrobutyricum&lt;/i&gt;. <em>BIOTECHNOLOGY FOR BIOFUELS AND BIOPRODUCTS</em>, JUN 25 2022. <a class="reference external" href="https://doi.org/10.1186/s13068-022-02169-z">[DOI]</a>.</p></li>
<li id="id3959"><p>Feiran Li, Le Yuan, Hongzhong Lu, Gang Li, Yu Chen, Martin K. M. Engqvist, Eduard J. Kerkhoven, and Jens Nielsen. Deep learning-based &lt;i&gt;k&lt;/i&gt;&lt;sub&gt;cat&lt;/sub&gt; prediction enables improved enzyme-constrained model reconstruction. <em>NATURE CATALYSIS</em>, 5(8):662+, AUG 2022. <a class="reference external" href="https://doi.org/10.1038/s41929-022-00798-z">[DOI]</a>.</p></li>
<li id="id3960"><p>Ceri Proffitt, Gholamreza Bidkhori, Sunjae Lee, Abdellah Tebani, Adil Mardinoglu, Mathias Uhlen, David L. Moyes, and Saeed Shoaie. Genome-scale metabolic modelling of the human gut microbiome reveals changes in the glyoxylate and dicarboxylate metabolism in metabolic disorders. <em>ISCIENCE</em>, JUL 15 2022. <a class="reference external" href="https://doi.org/10.1016/j.isci.2022.104513">[DOI]</a>.</p></li>
<li id="id3961"><p>Kulwadee Thanamit, Franziska Hoerhold, Marcus Oswald, and Rainer Koenig. Linear programming based gene expression model (lpm-gem) predicts the carbon source for &lt;i&gt;bacillus subtilis&lt;/i&gt;. <em>BMC BIOINFORMATICS</em>, JUN 10 2022. <a class="reference external" href="https://doi.org/10.1186/s12859-022-04742-7">[DOI]</a>.</p></li>
<li id="id3962"><p>Mohammad Mazharul Islam, Andrea Goertzen, Pankaj K. Singh, and Rajib Saha. Exploring the metabolic landscape of pancreatic ductal adenocarcinoma cells using genome-scale metabolic modeling. <em>ISCIENCE</em>, JUN 17 2022. <a class="reference external" href="https://doi.org/10.1016/j.isci.2022.104483">[DOI]</a>.</p></li>
<li id="id3963"><p>Shengbo Wu, Jie Feng, Chunjiang Liu, Hao Wu, Zekai Qiu, Jianjun Ge, Shuyang Sun, Xia Hong, Yukun Li, Xiaona Wang, Aidong Yang, Fei Guo, and Jianjun Qiao. Machine learning aided construction of the quorum sensing communication network for human gut microbiota. <em>NATURE COMMUNICATIONS</em>, JUN 2 2022. <a class="reference external" href="https://doi.org/10.1038/s41467-022-30741-6">[DOI]</a>.</p></li>
<li id="id3964"><p>Richard C. Law, Aliya Lakhani, Samantha O'Keeffe, Sevcan Ersan, and Junyoung O. Park. Integrative metabolic flux analysis reveals an indispensable dimension of phenotypes. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, JUN 2022. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2022.102701">[DOI]</a>.</p></li>
<li id="id3965"><p>Laurence Legon, Christophe Corre, Declan G. Bates, and Ahmad A. Mannan. Gcfront: a tool for determining a pareto front of growth-coupled cell factory designs. <em>BIOINFORMATICS</em>, 2022 JUN 1 2022. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btac376">[DOI]</a>.</p></li>
<li id="id3966"><p>Xiaoping Liao, Hongwu Ma, and Yinjie J. Tang. Artificial intelligence: a solution to involution of design-build-test-learn cycle. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, JUN 2022. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2022.102712">[DOI]</a>.</p></li>
<li id="id3967"><p>Feng Xu, Ju Lu, Xiang Ke, Minghao Shao, Mingzhi Huang, and Ju Chu. Reconstruction of the genome-scale metabolic model of &lt;i&gt;saccharopolyspora erythraea&lt;/i&gt; and its application in the overproduction of erythromycin. <em>METABOLITES</em>, JUN 2022. <a class="reference external" href="https://doi.org/10.3390/metabo12060509">[DOI]</a>.</p></li>
<li id="id3968"><p>Yameng Xu, Xinglong Wang, Chenyang Zhang, Xuan Zhou, Xianhao Xu, Luyao Han, Xueqin Lv, Yanfeng Liu, Song Liu, Jianghua Li, Guocheng Du, Jian Chen, Rodrigo Ledesma-Amaro, and Long Liu. De novo biosynthesis of rubusoside and rebaudiosides in engineered yeasts. <em>NATURE COMMUNICATIONS</em>, JUN 1 2022. <a class="reference external" href="https://doi.org/10.1038/s41467-022-30826-2">[DOI]</a>.</p></li>
<li id="id3969"><p>Feiran Li, Yu Chen, Qi Qi, Yanyan Wang, Le Yuan, Mingtao Huang, Ibrahim E. Elsemman, Amir Feizi, Eduard J. Kerkhoven, and Jens Nielsen. Improving recombinant protein production by yeast through genome-scale modeling using proteome constraints. <em>NATURE COMMUNICATIONS</em>, MAY 27 2022. <a class="reference external" href="https://doi.org/10.1038/s41467-022-30689-7">[DOI]</a>.</p></li>
<li id="id3970"><p>Andisheh Dadashi and Derek Martinez. Flux balance network expansion predicts stage-specific human peri_implantation embryo metabolism. <em>JOURNAL OF BIOINFORMATICS AND COMPUTATIONAL BIOLOGY</em>, AUG 2022. <a class="reference external" href="https://doi.org/10.1142/S021972002250010X">[DOI]</a>.</p></li>
<li id="id3971"><p>Kadir Kocabas, Alina Arif, Reaz Uddin, and Tunahan Cakir. Dual transcriptome based reconstruction of salmonella-human integrated metabolic network to screen potential drug targets. <em>PLOS ONE</em>, MAY 24 2022. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0268889">[DOI]</a>.</p></li>
<li id="id3972"><p>Barbara Bourgade, Christopher M. Humphreys, James Millard, Nigel P. Minton, and M. Ahsanul Islam. Design, analysis, and implementation of a novel biochemical pathway for ethylene glycol production in &lt;i&gt;clostridium autoethanogenum&lt;/i&gt;. <em>ACS SYNTHETIC BIOLOGY</em>, 11(5):1790–1800, MAY 20 2022. <a class="reference external" href="https://doi.org/10.1021/acssynbio.1c00624">[DOI]</a>.</p></li>
<li id="id3974"><p>Benjamin P. Kellman, Anne Richelle, Jeong-Yeh Yang, Digantkumar Chapla, Austin W. T. Chiang, Julia Najera, Bokan Bao, Natalia Koga, Mahmoud A. Mohammad, Anders Bech Bruntse, Morey W. Haymond, Kelley W. Moremen, Lars Bode, Nathan E. Lewis, Chenguang Liang, and Annalee Furst. Elucidating human milk oligosaccharide biosynthetic genes through network-based multi-omics integration. <em>NATURE COMMUNICATIONS</em>, MAY 4 2022. <a class="reference external" href="https://doi.org/10.1038/s41467-022-29867-4">[DOI]</a>.</p></li>
<li id="id3975"><p>Ruibing Chen, Jiaoqi Gao, Wei Yu, Xianghui Chen, Xiaoxin Zhai, Yu Chen, Lei Zhang, and Yongjin J. Zhou. Engineering cofactor supply and recycling to drive phenolic acid biosynthesis in yeast. <em>NATURE CHEMICAL BIOLOGY</em>, 18(5):520+, MAY 2022. <a class="reference external" href="https://doi.org/10.1038/s41589-022-01014-6">[DOI]</a>.</p></li>
<li id="id3976"><p>Seungyoon Nam and Yongmin Lee. Genome-scale metabolic model analysis of metabolic differences between lauren diffuse and intestinal subtypes in gastric cancer. <em>CANCERS</em>, MAY 2022. <a class="reference external" href="https://doi.org/10.3390/cancers14092340">[DOI]</a>.</p></li>
<li id="id3977"><p>Nicole Pearcy, Marco Garavaglia, Thomas Millat, James P. Gilbert, Yoseb A. Song, Hassan Hartman, Craig R. Woods, Claudio Tomi-Andrino, Rajesh Reddy Bommareddy, Byung-Kwan P. Cho, David A. A. Fell, Mark D. Poolman, John R. A. King, Klaus D. Winzer, Jamie A. Twycross, and Nigel P. D. Minton. A genome-scale metabolic model of &lt;i&gt;cupriavidus necator&lt;/i&gt; h16 integrated with tradis and transcriptomic data reveals metabolic insights for biotechnological applications. <em>PLOS COMPUTATIONAL BIOLOGY</em>, MAY 2022. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1010106">[DOI]</a>.</p></li>
<li id="id3978"><p>Tomokazu Shirai and Akihiko Kondo. In silico design strategies for the production of target chemical compounds using iterative single-level linear programming problems. <em>BIOMOLECULES</em>, MAY 2022. <a class="reference external" href="https://doi.org/10.3390/biom12050620">[DOI]</a>.</p></li>
<li id="id3979"><p>Parsa Ghadermazi, Angela Re, Luca Ricci, and Siu Hung Joshua Chan. Metabolic engineering interventions for sustainable 2,3-butanediol production in gas-fermenting &lt;i&gt;clostridium autoethanogenum&lt;/i&gt;. <em>MSYSTEMS</em>, APR 26 2022. <a class="reference external" href="https://doi.org/10.1128/msystems.01111-21">[DOI]</a>.</p></li>
<li id="id3980"><p>Nahikari Lopez-Lopez, David San Leon, Sonia de Castro, Roberto Diez-Martinez, Manuel Iglesias-Bexiga, Maria Jose Camarasa, Margarita Menendez, Juan Nogales, and Junkal Garmendia. Interrogation of essentiality in the reconstructed &lt;i&gt;haemophilus influenzae&lt;/i&gt; metabolic network identifies lipid metabolism antimicrobial targets: preclinical evaluation of a fabh β-ketoacyl-acp synthase inhibitor. <em>MSYSTEMS</em>, APR 26 2022. <a class="reference external" href="https://doi.org/10.1128/msystems.01459-21">[DOI]</a>.</p></li>
<li id="id3981"><p>Juan C. Villada, Maria F. Duran, Chee Kent Lim, Lisa Y. Stein, and Patrick K. H. Lee. Integrative genome-scale metabolic modeling reveals versatile metabolic strategies for methane utilization in &lt;i&gt;methylomicrobium album&lt;/i&gt; bg8. <em>MSYSTEMS</em>, APR 26 2022. <a class="reference external" href="https://doi.org/10.1128/msystems.00073-22">[DOI]</a>.</p></li>
<li id="id3982"><p>Hock Chuan Yeo, Seo-Young Park, Tessa Tan, Say K. Ng, Meiyappan Lakshmanan, and Dong-Yup Lee. Combined multivariate statistical and flux balance analyses uncover media bottlenecks to the growth and productivity of chinese hamster ovary cell cultures. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 119(7):1740–1754, JUL 2022. <a class="reference external" href="https://doi.org/10.1002/bit.28104">[DOI]</a>.</p></li>
<li id="id3983"><p>Dominik Ternes, Mina Tsenkova, Vitaly Igorevich Pozdeev, Marianne Meyers, Eric Koncina, Sura Atatri, Martine Schmitz, Jessica Karta, Maryse Schmoetten, Almut Heinken, Fabien Rodriguez, Catherine Delbrouck, Anthoula Gaigneaux, Aurelien Ginolhac, Tam Thuy Dan Nguyen, Lea Grandmougin, Audrey Frachet-Bour, Camille Martin-Gallausiaux, Maria Pacheco, Lorie Neuberger-Castillo, Paulo Miranda, Nikolaus Zuegel, Jean-Yves Ferrand, Manon Gantenbein, Thomas Sauter, Daniel Joseph Slade, Ines Thiele, Johannes Meiser, Serge Haan, Paul Wilmes, and Elisabeth Letellier. The gut microbial metabolite formate exacerbates colorectal cancer progression. <em>NATURE METABOLISM</em>, 4(4):458+, APR 2022. <a class="reference external" href="https://doi.org/10.1038/s42255-022-00558-0">[DOI]</a>.</p></li>
<li id="id3984"><p>Padhmanand Sudhakar, Tahila Andrighetti, Sare Verstockt, Clara Caenepeel, Marc Ferrante, Joao Sabino, Bram Verstockt, and Severine Vermeire. Integrated analysis of microbe-host interactions in crohn's disease reveals potential mechanisms of microbial proteins on host gene expression. <em>ISCIENCE</em>, MAY 20 2022. <a class="reference external" href="https://doi.org/10.1016/j.isci.2022.103963">[DOI]</a>.</p></li>
<li id="id3985"><p>Shouyong Jiang, Irene Otero-Muras, Julio R. Banga, Yong Wang, Marcus Kaiser, and Natalio Krasnogor. Optdesign: identifying optimum design strategies in strain engineering for biochemical production. <em>ACS SYNTHETIC BIOLOGY</em>, 11(4):1531–1541, APR 15 2022. <a class="reference external" href="https://doi.org/10.1021/acssynbio.1c00610">[DOI]</a>.</p></li>
<li id="id3986"><p>Lorena Azevedo de Lima, Henri Ingelman, Kush Brahmbhatt, Kristina Reinmets, Craig Barry, Audrey Harris, Esteban Marcellin, Michael Kopke, and Kaspar Valgepea. Faster growth enhances low carbon fuel and chemical production through gas fermentation. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, APR 12 2022. <a class="reference external" href="https://doi.org/10.3389/fbioe.2022.879578">[DOI]</a>.</p></li>
<li id="id3987"><p>Craig Woods, Christopher M. Humphreys, Claudio Tomi-Andrino, Anne M. Henstra, Michael Kopke, Sean D. Simpson, Klaus Winzer, and Nigel P. Minton. Required gene set for autotrophic growth of &lt;i&gt;clostridium autoethanogenum&lt;/i&gt;. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, APR 12 2022. <a class="reference external" href="https://doi.org/10.1128/aem.02479-21">[DOI]</a>.</p></li>
<li id="id3988"><p>Kalaivani Paramasivan, Aneesha Abdulla, Nabarupa Gupta, and Sarma Mutturi. &lt;i&gt;in silico&lt;/i&gt; target-based strain engineering of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; for terpene precursor improvement. <em>INTEGRATIVE BIOLOGY</em>, 14(2):25–36, APR 8 2022. <a class="reference external" href="https://doi.org/10.1093/intbio/zyac003">[DOI]</a>.</p></li>
<li id="id3989"><p>Merve Kutay, Devrim Gozuacik, and Tunahan Cakir. Cancer recurrence and omics: metabolic signatures of cancer dormancy revealed by transcriptome mapping of genome-scale networks. <em>OMICS-A JOURNAL OF INTEGRATIVE BIOLOGY</em>, 26(5):270–279, MAY 1 2022. <a class="reference external" href="https://doi.org/10.1089/omi.2022.0008">[DOI]</a>.</p></li>
<li id="id3990"><p>Kaspar Valgepea, Gert Talbo, Nobuaki Takemori, Ayako Takemori, Christina Ludwig, Vishnuvardhan Mahamkali, Alexander P. Mueller, Ryan Tappel, Michael Kopke, Sean Dennis Simpson, Lars Keld Nielsen, and Esteban Marcellin. Absolute proteome quantification in the gas-fermenting acetogen &lt;i&gt;clostridium autoethanogenum&lt;/i&gt;. <em>MSYSTEMS</em>, APR 26 2022. <a class="reference external" href="https://doi.org/10.1128/msystems.00026-22">[DOI]</a>.</p></li>
<li id="id3991"><p>Joshua E. Goldford, Ashish B. George, Avi I. Flamholz, and Daniel Segre. Protein cost minimization promotes the emergence of coenzyme redundancy. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, APR 5 2022. <a class="reference external" href="https://doi.org/10.1073/pnas.2110787119">[DOI]</a>.</p></li>
<li id="id3992"><p>Antoine Regimbeau, Marko Budinich, Abdelhalim Larhlimi, Juan Jose Pierella Karlusich, Olivier Aumont, Laurent Memery, Chris Bowler, and Damien Eveillard. Contribution of genome-scale metabolic modelling to niche theory. <em>ECOLOGY LETTERS</em>, 25(6):1352–1364, JUN 2022. <a class="reference external" href="https://doi.org/10.1111/ele.13954">[DOI]</a>.</p></li>
<li id="id3993"><p>Kristina Grausa, Ivars Mozga, Karlis Pleiko, and Agris Pentjuss. Integrative gene expression and metabolic analysis tool &lt;i&gt;igemrna&lt;/i&gt;. <em>BIOMOLECULES</em>, APR 2022. <a class="reference external" href="https://doi.org/10.3390/biom12040586">[DOI]</a>.</p></li>
<li id="id3994"><p>Bingqing He, Chen Cai, Tim McCubbin, Jorge Carrasco Muriel, Nikolaus Sonnenschein, Shihu Hu, Zhiguo Yuan, and Esteban Marcellin. A genome-scale metabolic model of &lt;i&gt;methanoperedens nitroreducens&lt;/i&gt;: assessing bioenergetics and thermodynamic feasibility. <em>METABOLITES</em>, APR 2022. <a class="reference external" href="https://doi.org/10.3390/metabo12040314">[DOI]</a>.</p></li>
<li id="id3995"><p>Qusheng Jin, Qiong Wu, Benjamin M. Shapiro, and Shannon E. McKernan. Limited mechanistic link between the monod equation and methanogen growth: a perspective from metabolic modeling. <em>MICROBIOLOGY SPECTRUM</em>, APR 2022. <a class="reference external" href="https://doi.org/10.1128/spectrum.02259-21">[DOI]</a>.</p></li>
<li id="id3996"><p>Joao B. Xavier, Jonathan M. Monk, Saugat Poudel, Charles J. Norsigian, Anand Sastry, V, Chen Liao, Jose Bento, Marc A. Suchard, Mario L. Arrieta-Ortiz, Eliza J. R. Peterson, Nitin S. Baliga, Thomas Stoeger, Felicia Ruffin, Reese A. K. Richardson, Catherine A. Gao, Thomas D. Horvath, Anthony M. Haag, Qinglong Wu, Tor Savidge, and Michael R. Yeaman. Mathematical models to study the biology of pathogens and the infectious diseases they cause. <em>ISCIENCE</em>, APR 15 2022. <a class="reference external" href="https://doi.org/10.1016/j.isci.2022.104079">[DOI]</a>.</p></li>
<li id="id3997"><p>Gong-Hua Li, Feifei Han, Fu-Hui Xiao, Kang-Su-Yun Gu, Qiu Shen, Weihong Xu, Wen-Xing Li, Yan-Li Wang, Bin Liang, Jing-Fei Huang, Wenzhong Xiao, and Qing-Peng Kong. System-level metabolic modeling facilitates unveiling metabolic signature in exceptional longevity. <em>AGING CELL</em>, APR 2022. <a class="reference external" href="https://doi.org/10.1111/acel.13595">[DOI]</a>.</p></li>
<li id="id3998"><p>Defei Liu, Zixiang Xu, Jingen Li, Qian Liu, Qianqian Yuan, Yanmei Guo, Hongwu Ma, and Chaoguang Tian. Reconstruction and analysis of genome-scale metabolic model for thermophilic fungus &lt;i&gt;myceliophthora thermophila&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 119(7):1926–1937, JUL 2022. <a class="reference external" href="https://doi.org/10.1002/bit.28080">[DOI]</a>.</p></li>
<li id="id3999"><p>Tadeja Rezen, Alexandre Martins, Miha Mraz, Nikolaj Zimic, Damjana Rozman, and Miha Moskon. Integration of omics data to generate and analyse covid-19 specific genome-scale metabolic models. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, JUN 2022. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2022.105428">[DOI]</a>.</p></li>
<li id="id4000"><p>Xingcun Fan, Jingru Zhou, Jianye Xia, and Xuefeng Yan. Genome-scale metabolic model's multi-objective solving algorithm based on the inflexion point of pareto front including maximum energy utilization and its application in &lt;i&gt;aspergillus niger&lt;/i&gt; ds03043. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 119(6):1539–1555, JUN 2022. <a class="reference external" href="https://doi.org/10.1002/bit.28078">[DOI]</a>.</p></li>
<li id="id4001"><p>Ehsan Salehabadi, Ehsan Motamedian, and Seyed Abbas Shojaosadati. Reconstruction of a generic genome-scale metabolic network for chicken: investigating network connectivity and finding potential biomarkers. <em>PLOS ONE</em>, MAR 22 2022. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0254270">[DOI]</a>.</p></li>
<li id="id4002"><p>Jason S. L. Yu, Clara Correia-Melo, Francisco Zorrilla, Lucia Herrera-Dominguez, Mary Y. Wu, Johannes Hartl, Kate Campbell, Sonja Blasche, Marco Kreidl, Anna-Sophia Egger, Christoph B. Messner, Vadim Demichev, Anja Freiwald, Michael Mulleder, Michael Howell, Judith Berman, Kiran R. Patil, Mohammad Tauqeer Alam, and Markus Ralser. Microbial communities form rich extracellular metabolomes that foster metabolic interactions and promote drug tolerance. <em>NATURE MICROBIOLOGY</em>, 7(4):542+, APR 2022. <a class="reference external" href="https://doi.org/10.1038/s41564-022-01072-5">[DOI]</a>.</p></li>
<li id="id4003"><p>Kunle Odunsi, Feng Qian, Amit A. Lugade, Han Yu, Melissa A. Geller, Steven P. Fling, Judith C. Kaiser, Andreanne M. Lacroix, Leonard D'Amico, Nirasha Ramchurren, Chihiro Morishima, Mary L. Disis, Lucas Dennis, Patrick Danaher, Sarah Warren, Van Anh Nguyen, Sudharshan Ravi, Takemasa Tsuji, Spencer Rosario, Wenjuan Zha, Alan Hutson, Song Liu, Shashikant Lele, Emese Zsiros, A. J. Robert McGray, Jessie Chiello, Richard Koya, Thinle Chodon, Carl D. Morrison, Vasanta Putluri, Nagireddy Putluri, Donald E. Mager, Rudiyanto Gunawan, Martin A. Cheever, Sebastiano Battaglia, and Junko Matsuzaki. Metabolic adaptation of ovarian tumors in patients treated with an ido1 inhibitor constrains antitumor immune responses. <em>SCIENCE TRANSLATIONAL MEDICINE</em>, MAR 16 2022. <a class="reference external" href="https://doi.org/10.1126/scitranslmed.abg8402">[DOI]</a>.</p></li>
<li id="id4004"><p>Adam Amara, Clement Frainay, Fabien Jourdan, Thomas Naake, Steffen Neumann, Elva Maria Novoa-del-Toro, Reza M. Salek, Liesa Salzer, Sarah Scharfenberg, and Michael Witting. Networks and graphs discovery in metabolomics data analysis and interpretation. <em>FRONTIERS IN MOLECULAR BIOSCIENCES</em>, MAR 8 2022. <a class="reference external" href="https://doi.org/10.3389/fmolb.2022.841373">[DOI]</a>.</p></li>
<li id="id4005"><p>Abhijit Paul, Salman Azhar, Phonindra Nath Das, Nandadulal Bairagi, and Samrat Chatterjee. Elucidating the metabolic characteristics of pancreatic β-cells from patients with type 2 diabetes (t2d) using a genome-scale metabolic modeling. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, MAY 2022. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2022.105365">[DOI]</a>.</p></li>
<li id="id4006"><p>Faiz Khan Mohammad, Meghana Venkata Palukuri, Shruti Shivakumar, Raghunathan Rengaswamy, and Swagatika Sahoo. A computational framework for studying gut-brain axis in autism spectrum disorder. <em>FRONTIERS IN PHYSIOLOGY</em>, MAR 7 2022. <a class="reference external" href="https://doi.org/10.3389/fphys.2022.760753">[DOI]</a>.</p></li>
<li id="id4007"><p>Lokanand Koduru, Meiyappan Lakshmanan, Shawn Hoon, Dong-Yup Lee, Yuan Kun Lee, and Dave Siak-Wei Ow. Systems biology of gut microbiota-human receptor interactions: toward anti-inflammatory probiotics. <em>FRONTIERS IN MICROBIOLOGY</em>, MAR 3 2022. <a class="reference external" href="https://doi.org/10.3389/fmicb.2022.846555">[DOI]</a>.</p></li>
<li id="id4008"><p>Philipp Wendering and Zoran Nikoloski. Commit: consideration of metabolite leakage and community composition improves microbial community reconstructions. <em>PLOS COMPUTATIONAL BIOLOGY</em>, MAR 2022. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1009906">[DOI]</a>.</p></li>
<li id="id4009"><p>Feng-Sheng Wang, Ke-Lin Chen, and Sz-Wei Chu. Human/sars-cov-2 genome-scale metabolic modeling to discover potential antiviral targets for covid-19. <em>JOURNAL OF THE TAIWAN INSTITUTE OF CHEMICAL ENGINEERS</em>, APR 2022. <a class="reference external" href="https://doi.org/10.1016/j.jtice.2022.104273">[DOI]</a>.</p></li>
<li id="id4010"><p>Stefano Camborda, Jan-Niklas Weder, and Nadine Toepfer. Cobramod: a pathway-centric curation tool for constraint-based metabolic models. <em>BIOINFORMATICS</em>, 38(9):2654–2656, APR 28 2022. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btac119">[DOI]</a>.</p></li>
<li id="id4011"><p>Teresa J. Clark and Jorg Schwender. Elucidation of triacylglycerol overproduction in the c&lt;sub&gt;4&lt;/sub&gt; bioenergy crop &lt;i&gt;sorghum bicolor&lt;/i&gt; by constraint-based analysis. <em>FRONTIERS IN PLANT SCIENCE</em>, FEB 17 2022. <a class="reference external" href="https://doi.org/10.3389/fpls.2022.787265">[DOI]</a>.</p></li>
<li id="id4012"><p>Almut Heinken and Ines Thiele. Microbiome modelling toolbox 2.0: efficient, tractable modelling of microbiome communities. <em>BIOINFORMATICS</em>, 38(8):2367–2368, APR 12 2022. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btac082">[DOI]</a>.</p></li>
<li id="id4013"><p>Lisha Qu, Xiang Xiu, Guoyun Sun, Chenyang Zhang, Haiquan Yang, Yanfeng Liu, Jianghua Li, Guocheng Du, Xueqin Lv, and Long Liu. Engineered yeast for efficient de novo synthesis of 7-dehydrocholesterol. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 119(5):1278–1289, MAY 2022. <a class="reference external" href="https://doi.org/10.1002/bit.28055">[DOI]</a>.</p></li>
<li id="id4014"><p>Nam Kyu Kang, Minsik Kim, Kwangryul R. Baek, Yong Keun Chang, Donald Ort, and Yong-Su Jin. Photoautotrophic organic acid production: glycolic acid production by microalgal cultivation. <em>CHEMICAL ENGINEERING JOURNAL</em>, APR 1 2022. <a class="reference external" href="https://doi.org/10.1016/j.cej.2021.133636">[DOI]</a>.</p></li>
<li id="id4015"><p>Sven Thiele, Axel von Kamp, Pavlos Stephanos Bekiaris, Philipp Schneider, and Steffen Klamt. Cnapy: a cellnetanalyzer gui in python for analyzing and designing metabolic networks. <em>BIOINFORMATICS</em>, 38(5):1467–1469, FEB 7 2022. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btab828">[DOI]</a>.</p></li>
<li id="id4016"><p>Elle M. Barnes and Susannah G. Tringe. Exploring the roles of microbes in facilitating plant adaptation to climate change. <em>BIOCHEMICAL JOURNAL</em>, 479(3):327–335, FEB 2022. <a class="reference external" href="https://doi.org/10.1042/BCJ20210793">[DOI]</a>.</p></li>
<li id="id4017"><p>Kristaps Berzins, Reinis Muiznieks, Matiss R. Baumanis, Inese Strazdina, Karlis Shvirksts, Santa Prikule, Vytautas Galvanauskas, Daniel Pleissner, Agris Pentjuss, Mara Grube, Uldis Kalnenieks, and Egils Stalidzans. Kinetic and stoichiometric modeling-based analysis of docosahexaenoic acid (dha) production potential by &lt;i&gt;crypthecodinium cohnii&lt;/i&gt; from glycerol, glucose and ethanol. <em>MARINE DRUGS</em>, FEB 2022. <a class="reference external" href="https://doi.org/10.3390/md20020115">[DOI]</a>.</p></li>
<li id="id4018"><p>Gabriela Canto-Encalada, Diego Tec-Campos, Juan D. Tibocha-Bonilla, Karsten Zengler, Alejandro Zepeda, and Cristal Zuniga. Flux balance analysis of the ammonia-oxidizing bacterium &lt;i&gt;nitrosomonas europaea&lt;/i&gt; atcc19718 unravels specific metabolic activities while degrading toxic compounds. <em>PLOS COMPUTATIONAL BIOLOGY</em>, FEB 2022. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1009828">[DOI]</a>.</p></li>
<li id="id4019"><p>Claudia Cipriani, Maria Pires Pacheco, Ali Kishk, Maryem Wachich, Daniel Abankwa, Elisabeth Schaffner-Reckinger, and Thomas Sauter. Bruceine d identified as a drug candidate against breast cancer by a novel drug selection pipeline and cell viability assay. <em>PHARMACEUTICALS</em>, FEB 2022. <a class="reference external" href="https://doi.org/10.3390/ph15020179">[DOI]</a>.</p></li>
<li id="id4020"><p>Catherine Bjerre Collin, Tom Gebhardt, Martin Golebiewski, Tugce Karaderi, Maximilian Hillemanns, Faiz Muhammad Khan, Ali Salehzadeh-Yazdi, Marc Kirschner, Sylvia Krobitsch, and Lars Kuepfer. Computational models for clinical applications in personalized medicine-guidelines and recommendations for data integration and model validation. <em>JOURNAL OF PERSONALIZED MEDICINE</em>, FEB 2022. <a class="reference external" href="https://doi.org/10.3390/jpm12020166">[DOI]</a>.</p></li>
<li id="id4021"><p>Zihe Liu, Junyang Wang, and Jens Nielsen. Yeast synthetic biology advances biofuel production. <em>CURRENT OPINION IN MICROBIOLOGY</em>, 65:33–39, FEB 2022. <a class="reference external" href="https://doi.org/10.1016/j.mib.2021.10.010">[DOI]</a>.</p></li>
<li id="id4022"><p>Romeu Viana, Diogo Couceiro, Tiago Carreiro, Oscar Dias, Isabel Rocha, and Miguel Cacho Teixeira. A genome-scale metabolic model for the human pathogen &lt;i&gt;candida parapsilosis&lt;/i&gt; and early identification of putative novel antifungal drug targets. <em>GENES</em>, FEB 2022. <a class="reference external" href="https://doi.org/10.3390/genes13020303">[DOI]</a>.</p></li>
<li id="id4023"><p>Vetle Simensen, Christian Schulz, Emil Karlsen, Signe Bratelund, Idun Burgos, Lilja Brekke Thorfinnsdottir, Laura Garcia-Calvo, Per Bruheim, and Eivind Almaas. Experimental determination of &lt;i&gt;escherichia coli&lt;/i&gt; biomass composition for constraint-based metabolic modeling. <em>PLOS ONE</em>, JAN 27 2022. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0262450">[DOI]</a>.</p></li>
<li id="id4024"><p>Nasser Delangiz, Sajad Aliyar, Neda Pashapoor, Khatereh Nobaharan, Behnam Asgari Lajayer, and Susana Rodriguez-Couto. Can polymer-degrading microorganisms solve the bottleneck of plastics' environmental challenges? <em>CHEMOSPHERE</em>, MAY 2022. <a class="reference external" href="https://doi.org/10.1016/j.chemosphere.2022.133709">[DOI]</a>.</p></li>
<li id="id4025"><p>Shivendra G. Tewari, Bobby Kwan, Rubayet Elahi, Krithika Rajaram, Jaques Reifman, Sean T. Prigge, Akhil B. Vaidya, and Anders Wallqvist. Metabolic adjustments of blood-stage &lt;i&gt;plasmodium falciparum&lt;/i&gt; in response to sublethal pyrazoleamide exposure. <em>SCIENTIFIC REPORTS</em>, JAN 21 2022. <a class="reference external" href="https://doi.org/10.1038/s41598-022-04985-7">[DOI]</a>.</p></li>
<li id="id4026"><p>Prerna Bhalla, Raghunathan Rengaswamy, Devarajan Karunagaran, G. K. Suraishkumar, and Swagatika Sahoo. Metabolic modeling of host-microbe interactions for therapeutics in colorectal cancer. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, JAN 19 2022. <a class="reference external" href="https://doi.org/10.1038/s41540-021-00210-9">[DOI]</a>.</p></li>
<li id="id4027"><p>Iman Shahidi Pour Savizi, Nader Maghsoudi, Ehsan Motamedian, Nathan E. Lewis, and Seyed Abbas Shojaosadati. Valine feeding reduces ammonia production through rearrangement of metabolic fluxes in central carbon metabolism of cho cells. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 106(3):1113–1126, FEB 2022. <a class="reference external" href="https://doi.org/10.1007/s00253-021-11755-4">[DOI]</a>.</p></li>
<li id="id4028"><p>Kirk Smith, Fangzhou Shen, Ho Joon Lee, and Sriram Chandrasekaran. Metabolic signatures of regulation by phosphorylation and acetylation. <em>ISCIENCE</em>, JAN 21 2022. <a class="reference external" href="https://doi.org/10.1016/j.isci.2021.103730">[DOI]</a>.</p></li>
<li id="id4029"><p>Aidin Behravan, Atieh Hashemi, and Sayed-Amir Marashi. A constraint-based modeling approach to reach an improved chemically defined minimal medium for recombinant antiepex-scfv production by &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, FEB 2022. <a class="reference external" href="https://doi.org/10.1016/j.bej.2022.108339">[DOI]</a>.</p></li>
<li id="id4030"><p>Minsuk Kim, Jaeyun Sung, and Nicholas Chia. Resource-allocation constraint governs structure and function of microbial communities in metabolic modeling. <em>METABOLIC ENGINEERING</em>, 70:12–22, MAR 2022. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.12.011">[DOI]</a>.</p></li>
<li id="id4031"><p>Eunice A. Ferreira, Catarina C. Pacheco, Joao S. Rodrigues, Filipe Pinto, Pedro Lamosa, David Fuente, Javier Urchueguia, and Paula Tamagnini. Heterologous production of glycine betaine using &lt;i&gt;synechocystis&lt;/i&gt; sp. pcc 6803-based chassis lacking native compatible solutes. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, JAN 7 2022. <a class="reference external" href="https://doi.org/10.3389/fbioe.2021.821075">[DOI]</a>.</p></li>
<li id="id4032"><p>Lavanya Raajaraam and Karthik Raman. A computational framework to identify metabolic engineering strategies for the co-production of metabolites. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, JAN 7 2022. <a class="reference external" href="https://doi.org/10.3389/fbioe.2021.779405">[DOI]</a>.</p></li>
<li id="id4033"><p>John R. Casey, Rene M. Boiteau, Martin K. M. Engqvist, Zoe Finkel, V, Gang Li, Justin Liefer, Christian L. Muller, Nathalie Munoz, and Michael J. Follows. Basin-scale biogeography of marine phytoplankton reflects cellular-scale optimization of metabolism and physiology. <em>SCIENCE ADVANCES</em>, JAN 2022. <a class="reference external" href="https://doi.org/10.1126/sciadv.abl4930">[DOI]</a>.</p></li>
<li id="id4034"><p>Lady Johanna Forero-Rodriguez, Jonathan Josephs-Spaulding, Stefano Flor, Andres Pinzon, and Christoph Kaleta. Parkinson's disease and the metal-microbiome-gut-brain axis: a systems toxicology approach. <em>ANTIOXIDANTS</em>, JAN 2022. <a class="reference external" href="https://doi.org/10.3390/antiox11010071">[DOI]</a>.</p></li>
<li id="id4035"><p>Natalia I. Garcia-Tomsig, Marta Robledo, George C. diCenzo, Alessio Mengoni, Vicenta Millan, Alexandra Peregrina, Alejandro Uceta, and Jose I. Jimenez-Zurdo. Pervasive rna regulation of metabolism enhances the root colonization ability of nitrogen-fixing symbiotic α-rhizobia. <em>MBIO</em>, JAN-FEB 2022. <a class="reference external" href="https://doi.org/10.1128/mbio.03576-21">[DOI]</a>.</p></li>
<li id="id4036"><p>Sang Mi Lee, GaRyoung Lee, and Hyun Uk Kim. Machine learning-guided evaluation of extraction and simulation methods for cancer patient-specific metabolic models. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 20:3041–3052, 2022. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2022.06.027">[DOI]</a>.</p></li>
<li id="id4037"><p>Seyed Babak Loghmani, Nadine Veith, Sven Sahle, Frank T. Bergmann, Brett G. Olivier, and Ursula Kummer. Inspecting the solution space of genome-scale metabolic models. <em>METABOLITES</em>, JAN 2022. <a class="reference external" href="https://doi.org/10.3390/metabo12010043">[DOI]</a>.</p></li>
<li id="id4038"><p>Carlos E. Mejia-Gomez, Rigoberto Rios-Estepa, Luis A. Gonzalez-Lopez, and Norman Balcazar-Morales. An experimental and &lt;i&gt;in silico&lt;/i&gt; analysis of &lt;i&gt;lacticaseibacillus&lt;/i&gt; &lt;i&gt;paracasei&lt;/i&gt; isolated from whey shows an association between lactate production and amino acid catabolism. <em>ANAIS DA ACADEMIA BRASILEIRA DE CIENCIAS</em>, 2022. <a class="reference external" href="https://doi.org/10.1590/0001-3765202220211071">[DOI]</a>.</p></li>
<li id="id4039"><p>Anurag Passi, Juan D. Tibocha-Bonilla, Manish Kumar, Diego Tec-Campos, Karsten Zengler, and Cristal Zuniga. Genome-scale metabolic modeling enables in-depth understanding of big data. <em>METABOLITES</em>, JAN 2022. <a class="reference external" href="https://doi.org/10.3390/metabo12010014">[DOI]</a>.</p></li>
<li id="id4040"><p>Albert E. Tafur Rangel, Abel Garcia Oviedo, Freddy Cabrera Mojica, Jorge M. Gomez, and Andres Fernando Gonzalez Barrios. Development of an integrating systems metabolic engineering and bioprocess modeling approach for rational strain improvement. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, JAN 2022. <a class="reference external" href="https://doi.org/10.1016/j.bej.2021.108268">[DOI]</a>.</p></li>
<li id="id4041"><p>Thomas Sauter, Tamara Bintener, Ali Kishk, Luana Presta, Tessy Prohaska, Daniel Guignard, Ni Zeng, Claudia Cipriani, Sundas Arshad, Thomas Pfau, Patricia Martins Conde, and Maria Pires Pacheco. Project-based learning course on metabolic network modelling in computational systems biology. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JAN 2022. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1009711">[DOI]</a>.</p></li>
<li id="id4042"><p>Carolin C. M. Schulte, Vinoy K. Ramachandran, Antonis Papachristodoulou, and Philip S. Poole. Genome-scale metabolic modelling of lifestyle changes in &lt;i&gt;rhizobium leguminosarum&lt;/i&gt;. <em>MSYSTEMS</em>, JAN-FEB 2022.</p></li>
<li id="id4043"><p>Carolin C. M. Schulte, Vinoy K. Ramachandran, Antonis Papachristodoulou, and Philip S. Poole. Genome-scale metabolic modelling of lifestyle changes in &lt;i&gt;rhizobium leguminosarum&lt;/i&gt;. <em>MSYSTEMS</em>, JAN-FEB 2022.</p></li>
<li id="id4044"><p>Hilal Taymaz-Nikerel and Alvaro R. Lara. &lt;i&gt;vitreoscilla&lt;/i&gt; haemoglobin: a tool to reduce overflow metabolism. <em>MICROORGANISMS</em>, JAN 2022. <a class="reference external" href="https://doi.org/10.3390/microorganisms10010043">[DOI]</a>.</p></li>
<li id="id4045"><p>Ines Thiele and Ronan M. T. Fleming. Whole-body metabolic modelling predicts isoleucine dependency of sars-cov-2 replication. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 20:4098–4109, 2022. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2022.07.019">[DOI]</a>.</p></li>
<li id="id4046"><p>Philipp Wendering and Zoran Nikoloski. Genome-scale modeling specifies the metabolic capabilities of &lt;i&gt;rhizophagus irregularis&lt;/i&gt;. <em>MSYSTEMS</em>, JAN-FEB 2022.</p></li>
<li id="id4047"><p>Pedro Saa, Arles Urrutia, Claudia Silva-Andrade, Alberto J. Martin, and Daniel Garrido. Modeling approaches for probing cross-feeding interactions in the human gut microbiome. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 20:79–89, 2022. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2021.12.006">[DOI]</a>.</p></li>
<li id="id4049"><p>Junmin Wang, Alireza Delfarah, Patrick E. Gelbach, Emma Fong, Paul Macklin, Shannon M. Mumenthaler, Nicholas A. Graham, and Stacey D. Finley. Elucidating tumor-stromal metabolic crosstalk in colorectal cancer through integration of constraint-based models and lc-ms metabolomics. <em>METABOLIC ENGINEERING</em>, 69:175–187, JAN 2022. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.11.006">[DOI]</a>.</p></li>
<li id="id4059"><p>Miroslav Kratochvil, Laurent Heirendt, St Elmo Wilken, Taneli Pusa, Sylvain Arreckx, Alberto Noronha, Marvin van Aalst, Venkata P. Satagopam, Oliver Ebenhoh, Reinhard Schneider, Christophe Trefois, and Wei Gu. Cobrexa.jl: constraint-based reconstruction and exascale analysis. <em>BIOINFORMATICS</em>, 38(4):1171–1172, FEB 15 2022. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btab782">[DOI]</a>.</p></li>
<li id="id4063"><p>Behnaz Nowrouzi and Leonardo Rios-Solis. Redox metabolism for improving whole-cell p450-catalysed terpenoid biosynthesis. <em>CRITICAL REVIEWS IN BIOTECHNOLOGY</em>, 42(8):1213–1237, NOV 17 2022. <a class="reference external" href="https://doi.org/10.1080/07388551.2021.1990210">[DOI]</a>.</p></li>
<li id="id4071"><p>James M. Clomburg, Angela Cintolesi, and Ramon Gonzalez. &lt;i&gt;in silico&lt;/i&gt; and &lt;i&gt;in vivo&lt;/i&gt; analyses reveal key metabolic pathways enabling the fermentative utilization of glycerol in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>MICROBIAL BIOTECHNOLOGY</em>, 15(1):289–304, JAN 2022. <a class="reference external" href="https://doi.org/10.1111/1751-7915.13938">[DOI]</a>.</p></li>
<li id="id4081"><p>Khushboo, Punit Kumar, Kashyap K. Dubey, Zeba Usmani, Minaxi Sharma, and Vijai Kumar Gupta. Biotechnological and industrial applications of &lt;i&gt;streptomyces&lt;/i&gt; metabolites. <em>BIOFUELS BIOPRODUCTS &amp; BIOREFINING-BIOFPR</em>, 16(1):244–264, JAN 2022. <a class="reference external" href="https://doi.org/10.1002/bbb.2294">[DOI]</a>.</p></li>
<li id="id4090"><p>Gianvito Pio, Paolo Mignone, Giuseppe Magazzu, Guido Zampieri, Michelangelo Ceci, and Claudio Angione. Integrating genome-scale metabolic modelling and transfer learning for human gene regulatory network reconstruction. <em>BIOINFORMATICS</em>, 38(2):487–493, JAN 15 2022. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btab647">[DOI]</a>.</p></li>
<li id="id4091"><p>Jihoon W. Lee, Yapeng Su, Priyanka Baloni, Daniel Chen, Ana Jimena Pavlovitch-Bedzyk, Dan Yuan, Venkata R. Duvvuri, Rachel H. Ng, Jongchan Choi, Jingyi Xie, Rongyu Zhang, Kim Murray, Sergey Kornilov, Brett Smith, Andrew T. Magis, Dave S. B. Hoon, Jennifer J. Hadlock, Jason D. Goldman, Nathan D. Price, Raphael Gottardo, Mark M. Davis, Leroy Hood, Philip D. Greenberg, and James R. Heath. Integrated analysis of plasma and single immune cells uncovers metabolic changes in individuals with covid-19. <em>NATURE BIOTECHNOLOGY</em>, 40(1):110+, JAN 2022. <a class="reference external" href="https://doi.org/10.1038/s41587-021-01020-4">[DOI]</a>.</p></li>
<li id="id4109"><p>Axel Theorell, Johann F. Jadebeck, Katharina Noeh, and Joerg Stelling. Polyround: polytope rounding for random sampling in metabolic networks. <em>BIOINFORMATICS</em>, 38(2):566–567, JAN 15 2022. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btab552">[DOI]</a>.</p></li>
<li id="id4130"><p>Kyoichi Ebata, Sawa Yamashiro, Keita Iida, and Mariko Okada. Building patient-specific models for receptor tyrosine kinase signaling networks. <em>FEBS JOURNAL</em>, 289(1):90–101, JAN 2022. <a class="reference external" href="https://doi.org/10.1111/febs.15831">[DOI]</a>.</p></li>
<li id="id4363"><p>Masoud Ahookhosh, Ronan M. T. Fleming, and Phan T. Vuong. Finding zeros of holder metrically subregular mappings via globally convergent levenberg-marquardt methods. <em>OPTIMIZATION METHODS &amp; SOFTWARE</em>, 37(1):113–149, JAN 2 2022. <a class="reference external" href="https://doi.org/10.1080/10556788.2020.1712602">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-3" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-3" name="0-3" role="tabpanel" tabindex="0"><div class="docutils container" id="id5461">
<ol class="arabic simple" start="1">
<li id="id5868"><p>Marie Schopping, Paula Gaspar, Ana Rute Neves, Carl Johan Franzen, and Ahmad A. Zeidan. Identifying the essential nutritional requirements of the probiotic bacteria &lt;i&gt;bifidobacterium animalis&lt;/i&gt; and &lt;i&gt;bifidobacterium longum&lt;/i&gt; through genome-scale modeling. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, DEC 9 2021. <a class="reference external" href="https://doi.org/10.1038/s41540-021-00207-4">[DOI]</a>.</p></li>
<li id="id5870"><p>Selva Rupa Christinal Immanuel, Mario L. Arrieta-Ortiz, Rene A. Ruiz, Min Pan, Adrian Lopez Garcia de Lomana, Eliza J. R. Peterson, and Nitin S. Baliga. Quantitative prediction of conditional vulnerabilities in regulatory and metabolic networks using prime. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, DEC 6 2021. <a class="reference external" href="https://doi.org/10.1038/s41540-021-00205-6">[DOI]</a>.</p></li>
<li id="id5871"><p>Amnah Alzahmi, Sarah Daakour, Diana Charles El Assal, Bushra S. Dohai, Amphun Chaiboonchoe, Weiqi Fu, David R. Nelson, Alexandra Mystikou, and Kourosh Salehi-Ashtiani. High-throughput metabolic profiling for model refinements of microalgae. <em>JOVE-JOURNAL OF VISUALIZED EXPERIMENTS</em>, DEC 2021. <a class="reference external" href="https://doi.org/10.3791/61913">[DOI]</a>.</p></li>
<li id="id5872"><p>Joao Pedro Saraiva, Alexandre Bartholomaeus, Rene Kallies, Marta Gomes, Marcos Bicalho, Jonas Coelho Kasmanas, Carsten Vogt, Antonis Chatzinotas, Peter Stadler, Oscar Dias, and Ulisses Nunes da Rocha. Ortsuite: from genomes to prediction of microbial interactions within targeted ecosystem processes. <em>LIFE SCIENCE ALLIANCE</em>, DEC 2021. <a class="reference external" href="https://doi.org/10.26508/lsa.202101167">[DOI]</a>.</p></li>
<li id="id5873"><p>Baoyu Xiang, Liping Zhao, and Menghui Zhang. Metagenome-scale metabolic network suggests folate produced by &lt;i&gt;bifidobacterium longum&lt;/i&gt; might contribute to high-fiber-diet-induced weight loss in a prader-willi syndrome child. <em>MICROORGANISMS</em>, DEC 2021. <a class="reference external" href="https://doi.org/10.3390/microorganisms9122493">[DOI]</a>.</p></li>
<li id="id5874"><p>Neeraj Sinha, Evert M. van Schothorst, Guido J. E. J. Hooiveld, Jaap Keijer, Vitor A. P. Martins dos Santos, and Maria Suarez-Diez. Exploring the associations between transcript levels and fluxes in constraint-based models of metabolism. <em>BMC BIOINFORMATICS</em>, NOV 29 2021. <a class="reference external" href="https://doi.org/10.1186/s12859-021-04488-8">[DOI]</a>.</p></li>
<li id="id5875"><p>Ali Kishk, Maria Pires Pacheco, and Thomas Sauter. Dccov: repositioning of drugs and drug combinations for sars-cov-2 infected lung through constraint-based modeling. <em>ISCIENCE</em>, NOV 19 2021. <a class="reference external" href="https://doi.org/10.1016/j.isci.2021.103331">[DOI]</a>.</p></li>
<li id="id5876"><p>Carl Malina, Francesca Di Bartolomeo, Eduard J. Kerkhoven, and Jens Nielsen. Constraint-based modeling of yeast mitochondria reveals the dynamics of protein import and iron-sulfur cluster biogenesis. <em>ISCIENCE</em>, NOV 19 2021. <a class="reference external" href="https://doi.org/10.1016/j.isci.2021.103294">[DOI]</a>.</p></li>
<li id="id5877"><p>Puhua Niu, Maria J. Soto, Byung-Jun Yoon, Edward R. Dougherty, Francis J. Alexander, Ian Blaby, and Xiaoning Qian. Trimer: transcription regulation integrated with metabolic regulation. <em>ISCIENCE</em>, NOV 19 2021. <a class="reference external" href="https://doi.org/10.1016/j.isci.2021.103218">[DOI]</a>.</p></li>
<li id="id5878"><p>Maziya Ibrahim and Karthik Raman. Two-species community design of lactic acid bacteria for optimal production of lactate. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 19:6039–6049, 2021. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2021.11.009">[DOI]</a>.</p></li>
<li id="id5880"><p>Anja Karlstaedt. Stable isotopes for tracing cardiac metabolism in diseases. <em>FRONTIERS IN CARDIOVASCULAR MEDICINE</em>, NOV 11 2021. <a class="reference external" href="https://doi.org/10.3389/fcvm.2021.734364">[DOI]</a>.</p></li>
<li id="id5881"><p>Mario L. Arrieta-Ortiz, Selva Rupa Christinal Immanuel, Serdar Turkarslan, Wei-Ju Wu, Brintha P. Girinathan, Jay N. Worley, Nicholas DiBenedetto, Olga Soutourina, Johann Peltier, Bruno Dupuy, Lynn Bry, and Nitin S. Baliga. Predictive regulatory and metabolic network models for systems analysis of &lt;i&gt;clostridioides difficile&lt;/i&gt;. <em>CELL HOST &amp; MICROBE</em>, 29(11):1709+, NOV 10 2021. <a class="reference external" href="https://doi.org/10.1016/j.chom.2021.09.008">[DOI]</a>.</p></li>
<li id="id5884"><p>Mihir Shah, V, Hadi Nazem-Bokaee, James Antoney, Suk Woo Kang, Colin J. Jackson, and Colin Scott. Improved production of the non-native cofactor f&lt;sub&gt;420&lt;/sub&gt; in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>SCIENTIFIC REPORTS</em>, NOV 5 2021. <a class="reference external" href="https://doi.org/10.1038/s41598-021-01224-3">[DOI]</a>.</p></li>
<li id="id5885"><p>Chao-Ting Cheng, Tsun-Yu Wang, Pei-Rong Chen, Wu-Hsiung Wu, Jin-Mei Lai, Peter Mu-Hsin Chang, Yi-Ren Hong, Chi-Ying F. Huang, and Feng-Sheng Wang. Computer-aided design for identifying anticancer targets in genome-scale metabolic models of colon cancer. <em>BIOLOGY-BASEL</em>, NOV 2021. <a class="reference external" href="https://doi.org/10.3390/biology10111115">[DOI]</a>.</p></li>
<li id="id5886"><p>Dafni Giannari, Cleo Hanchen Ho, and Radhakrishnan Mahadevan. A gap-filling algorithm for prediction of metabolic interactions in microbial communities. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1009060">[DOI]</a>.</p></li>
<li id="id5887"><p>Nicolai S. Panikov. Genome-scale reconstruction of microbial dynamic phenotype: successes and challenges. <em>MICROORGANISMS</em>, NOV 2021. <a class="reference external" href="https://doi.org/10.3390/microorganisms9112352">[DOI]</a>.</p></li>
<li id="id5888"><p>Sudharshan A. Ravi and Rudiyanto A. Gunawan. Δfba-predicting metabolic flux alterations using genome-scale metabolic models and differential transcriptomic data. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1009589">[DOI]</a>.</p></li>
<li id="id5889"><p>Chaitra Sarathy, Marian Breuer, Martina Kutmon, Michiel E. Adriaens, Chris T. Evelo, and Ilja C. W. Arts. Comparison of metabolic states using genome-scale metabolic models. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1009522">[DOI]</a>.</p></li>
<li id="id5890"><p>Theresa L. Whiteside. The role of tumor-derived exosomes (tex) in shaping anti-tumor immune competence. <em>CELLS</em>, NOV 2021. <a class="reference external" href="https://doi.org/10.3390/cells10113054">[DOI]</a>.</p></li>
<li id="id5892"><p>Nicholas S. Kruyer, Matthew J. Realff, Wenting Sun, Caroline L. Genzale, and Pamela Peralta-Yahya. Designing the bioproduction of martian rocket propellant via a biotechnology-enabled in situ resource utilization strategy. <em>NATURE COMMUNICATIONS</em>, OCT 25 2021. <a class="reference external" href="https://doi.org/10.1038/s41467-021-26393-7">[DOI]</a>.</p></li>
<li id="id5893"><p>Maria Moscardo Garcia, Maria Pacheco, Tamara Bintener, Luana Presta, and Thomas Sauter. Importance of the biomass formulation for cancer metabolic modeling and drug prediction. <em>ISCIENCE</em>, OCT 22 2021. <a class="reference external" href="https://doi.org/10.1016/j.isci.2021.103110">[DOI]</a>.</p></li>
<li id="id5894"><p>William T. Scott, Jr., Eddy J. Smid, David E. Block, and Richard A. Notebaart. Metabolic flux sampling predicts strain-dependent differences related to aroma production among commercial wine yeasts. <em>MICROBIAL CELL FACTORIES</em>, OCT 21 2021. <a class="reference external" href="https://doi.org/10.1186/s12934-021-01694-0">[DOI]</a>.</p></li>
<li id="id5895"><p>John P. Thomas, Dezso Modos, Tamas Korcsmaros, and Johanne Brooks-Warburton. Network biology approaches to achieve precision medicine in inflammatory bowel disease. <em>FRONTIERS IN GENETICS</em>, OCT 21 2021. <a class="reference external" href="https://doi.org/10.3389/fgene.2021.760501">[DOI]</a>.</p></li>
<li id="id5896"><p>Kangsan Kim, Donghui Choe, Yoseb Song, Minjeong Kang, Seung-Goo Lee, Dae-Hee Lee, and Byung-Kwan Cho. Engineering&lt;i&gt; bacteroides&lt;/i&gt;&lt;i&gt; thetaiotaomicron&lt;/i&gt; to produce non-native butyrate based on a genome-scale metabolic model-guided design. <em>METABOLIC ENGINEERING</em>, 68:174–186, NOV 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.10.005">[DOI]</a>.</p></li>
<li id="id5897"><p>Ilija Dukovski, Djordje Bajic, Jeremy M. Chacon, Michael Quintin, Jean C. C. Vila, Snorre Sulheim, Alan R. Pacheco, David B. Bernstein, William J. Riehl, Kirill S. Korolev, Alvaro Sanchez, William R. Harcombe, and Daniel Segre. A metabolic modeling platform for the computation of microbial ecosystems in time and space (comets). <em>NATURE PROTOCOLS</em>, 16(11):5030+, NOV 2021. <a class="reference external" href="https://doi.org/10.1038/s41596-021-00593-3">[DOI]</a>.</p></li>
<li id="id5898"><p>Nunthaphan Vikromvarasiri, Tomokazu Shirai, and Akihiko Kondo. Metabolic engineering design to enhance (r,r)-2,3-butanediol production from glycerol in &lt;i&gt;bacillus subtilis&lt;/i&gt; based on flux balance analysis. <em>MICROBIAL CELL FACTORIES</em>, OCT 9 2021. <a class="reference external" href="https://doi.org/10.1186/s12934-021-01688-y">[DOI]</a>.</p></li>
<li id="id5899"><p>Nadeem Joudeh, Athanasios Saragliadis, Christian Schulz, Andre Voigt, Eivind Almaas, and Dirk Linke. Transcriptomic response analysis of &lt;i&gt;escherichia coli&lt;/i&gt; to palladium stress. <em>FRONTIERS IN MICROBIOLOGY</em>, OCT 8 2021. <a class="reference external" href="https://doi.org/10.3389/fmicb.2021.741836">[DOI]</a>.</p></li>
<li id="id5900"><p>Yuan Zhu, Ying Li, Ya Xu, Jian Zhang, Linlin Ma, Qingsheng Qi, and Qian Wang. Development of bifunctional biosensors for sensing and dynamic control of glycolysis flux in metabolic engineering. <em>METABOLIC ENGINEERING</em>, 68:142–151, NOV 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.09.011">[DOI]</a>.</p></li>
<li id="id5902"><p>Albert A. Antolin and Marta Cascante. Ai delivers michaelis constants as fuel for genome-scale metabolic models. <em>PLOS BIOLOGY</em>, OCT 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pbio.3001415">[DOI]</a>.</p></li>
<li id="id5903"><p>Hongzhong Lu, Feiran Li, Le Yuan, Ivan Domenzain, Rosemary Yu, Hao Wang, Gang Li, Yu Chen, Boyang Ji, Eduard J. Kerkhoven, and Jens Nielsen. Yeast metabolic innovations emerged via expanded metabolic network and gene positive selection. <em>MOLECULAR SYSTEMS BIOLOGY</em>, OCT 2021. <a class="reference external" href="https://doi.org/10.15252/msb.202110427">[DOI]</a>.</p></li>
<li id="id5904"><p>Lachlan J. Munro and Douglas B. Kell. Intelligent host engineering for metabolic flux optimisation in biotechnology. <em>BIOCHEMICAL JOURNAL</em>, 478(20):3685–3721, OCT 2021. <a class="reference external" href="https://doi.org/10.1042/BCJ20210535">[DOI]</a>.</p></li>
<li id="id5905"><p>Yuki Kuriya, Mai Inoue, Masaki Yamamoto, Masahiro Murata, and Michihiro Araki. Knowledge extraction from literature and enzyme sequences complements fba analysis in metabolic engineering. <em>BIOTECHNOLOGY JOURNAL</em>, DEC 2021. <a class="reference external" href="https://doi.org/10.1002/biot.202000443">[DOI]</a>.</p></li>
<li id="id5906"><p>Indu Jatain, Kashyap Kumar Dubey, Manisha Sharma, Zeba Usmani, Minaxi Sharma, and Vijai Kumar Gupta. Synthetic biology potential for carbon sequestration into biocommodities. <em>JOURNAL OF CLEANER PRODUCTION</em>, NOV 10 2021. <a class="reference external" href="https://doi.org/10.1016/j.jclepro.2021.129176">[DOI]</a>.</p></li>
<li id="id5907"><p>Yara Seif and Bernhard Orn Palsson. Path to improving the life cycle and quality of genome-scale models of metabolism. <em>CELL SYSTEMS</em>, 12(9):842–859, SEP 22 2021. <a class="reference external" href="https://doi.org/10.1016/j.cels.2021.06.005">[DOI]</a>.</p></li>
<li id="id5908"><p>Fernando Medeiros Filho, Ana Paula Barbosa do Nascimento, Maiana de Oliveira Cerqueira e Costa, Thiago Castanheira Merigueti, Marcio Argollo de Menezes, Marisa Fabiana Nicolas, Marcelo Trindade dos Santos, Ana Paula D'Alincourt Carvalho-Assef, and Fabricio Alves Barbosa da Silva. A systematic strategy to find potential therapeutic targets for &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt; using integrated computational models. <em>FRONTIERS IN MOLECULAR BIOSCIENCES</em>, SEP 20 2021. <a class="reference external" href="https://doi.org/10.3389/fmolb.2021.728129">[DOI]</a>.</p></li>
<li id="id5909"><p>Sigurdur Trausti Karvelsson, Qiong Wang, Bylgja Hilmarsdottir, Arnar Sigurdsson, Siver Andreas Moestue, Gunhild Mari Maelandsmo, Skarphedinn Halldorsson, Steinn Gudmundsson, and Ottar Rolfsson. Argininosuccinate lyase is a metabolic vulnerability in breast development and cancer. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, SEP 17 2021. <a class="reference external" href="https://doi.org/10.1038/s41540-021-00195-5">[DOI]</a>.</p></li>
<li id="id5912"><p>Elisabeth Yaneske, Guido Zampieri, Loris Bertoldi, Giuseppe Benvenuto, and Claudio Angione. Genome-scale metabolic modelling of sars-cov-2 in cancer cells reveals an increased shift to glycolytic energy production. <em>FEBS LETTERS</em>, 595(18):2350–2365, SEP 2021. <a class="reference external" href="https://doi.org/10.1002/1873-3468.14180">[DOI]</a>.</p></li>
<li id="id5913"><p>Dung Hoang Anh Mai, Thu Thi Nguyen, and Eun Yeol Lee. The ethylmalonyl-coa pathway for methane-based biorefineries: a case study of using &lt;i&gt;methylosinus trichosporium&lt;/i&gt; ob3b, an alpha-proteobacterial methanotroph, for producing 2-hydroxyisobutyric acid and 1,3-butanediol from methane. <em>GREEN CHEMISTRY</em>, 23(19):7712–7723, OCT 4 2021. <a class="reference external" href="https://doi.org/10.1039/d1gc02866a">[DOI]</a>.</p></li>
<li id="id5914"><p>Almut Heinken, Stefania Magnusdottir, Ronan M. T. Fleming, and Ines Thiele. Demeter: efficient simultaneous curation of genome-scale reconstructions guided by experimental data and refined gene annotations. <em>BIOINFORMATICS</em>, 37(21):3974–3975, NOV 1 2021. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btab622">[DOI]</a>.</p></li>
<li id="id5915"><p>Mohammad Karim Khaleghi, Iman Shahidi Pour Savizi, Nathan E. Lewis, and Seyed Abbas Shojaosadati. Synergisms of machine learning and constraint-based modeling of metabolism for analysis and optimization of fermentation parameters. <em>BIOTECHNOLOGY JOURNAL</em>, NOV 2021. <a class="reference external" href="https://doi.org/10.1002/biot.202100212">[DOI]</a>.</p></li>
<li id="id5916"><p>Philippe Bogaerts and Alain Vande Wouwer. How to tackle underdeterminacy in metabolic flux analysis? a tutorial and critical review. <em>PROCESSES</em>, SEP 2021. <a class="reference external" href="https://doi.org/10.3390/pr9091577">[DOI]</a>.</p></li>
<li id="id5917"><p>Itziar Frades, Carles Foguet, Marta Cascante, and Marcos J. Arauzo-Bravo. Genome scale modeling to study the metabolic competition between cells in the tumor microenvironment. <em>CANCERS</em>, SEP 2021. <a class="reference external" href="https://doi.org/10.3390/cancers13184609">[DOI]</a>.</p></li>
<li id="id5918"><p>Matthew L. Jenior, Jhansi L. Leslie, Deborah A. Powers, Elizabeth M. Garrett, Kimberly A. Walker, Mary E. Dickenson, William A. Petri, Jr., Rita Tamayo, and Jason A. Papin. Novel drivers of virulence in &lt;i&gt;clostridioides difficile&lt;/i&gt; identified via context-specific metabolic network analysis. <em>MSYSTEMS</em>, SEP-OCT 2021. <a class="reference external" href="https://doi.org/10.1128/mSystems.00919-21">[DOI]</a>.</p></li>
<li id="id5919"><p>William T. Scott, Jr., Oscar van Mastrigt, David E. Block, Richard A. Notebaart, and Eddy J. Smid. Nitrogenous compound utilization and production of volatile organic compounds among commercial wine yeasts highlight strain-specific metabolic diversity. <em>MICROBIOLOGY SPECTRUM</em>, SEP 2021. <a class="reference external" href="https://doi.org/10.1128/Spectrum.00485-21">[DOI]</a>.</p></li>
<li id="id5920"><p>Egils Stalidzans and Elina Dace. Sustainable metabolic engineering for sustainability optimisation of industrial biotechnology. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 19:4770–4776, 2021. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2021.08.034">[DOI]</a>.</p></li>
<li id="id5921"><p>Rudolfs Petrovs, Egils Stalidzans, and Agris Pentjuss. Imfler: a web application for interactive metabolic flux analysis and visualization. <em>JOURNAL OF COMPUTATIONAL BIOLOGY</em>, 28(10):1021–1032, OCT 1 2021. <a class="reference external" href="https://doi.org/10.1089/cmb.2021.0056">[DOI]</a>.</p></li>
<li id="id5922"><p>Elif Esvap and Kutlu O. Ulgen. Advances in genome-scale metabolic modeling toward microbial community analysis of the human microbiome. <em>ACS SYNTHETIC BIOLOGY</em>, 10(9):2121–2137, SEP 17 2021. <a class="reference external" href="https://doi.org/10.1021/acssynbio.1c00140">[DOI]</a>.</p></li>
<li id="id5923"><p>Omid Oftadeh, Pierre Salvy, Maria Masid, Maxime Curvat, Ljubisa Miskovic, and Vassily Hatzimanikatis. A genome-scale metabolic model of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; that integrates expression constraints and reaction thermodynamics. <em>NATURE COMMUNICATIONS</em>, AUG 9 2021. <a class="reference external" href="https://doi.org/10.1038/s41467-021-25158-6">[DOI]</a>.</p></li>
<li id="id5924"><p>Allon Wagner, Chao Wang, Johannes Fessler, David DeTomaso, Julian Avila-Pacheco, James Kaminski, Sarah Zaghouani, Elena Christian, Pratiksha Thakore, Brandon Schellhaass, Elliot Akama-Garren, Kerry Pierce, Vasundhara Singh, Noga Ron-Harel, Vivian Paraskevi Douglas, Lloyd Bod, Alexandra Schnell, Daniel Puleston, Raymond A. Sobel, Marcia Haigis, Erika L. Pearce, Manoocher Soleimani, Clary Clish, Aviv Regev, Vijay K. Kuchroo, and Nir Yosef. Metabolic modeling of single th17 cells reveals regulators of autoimmunity. <em>CELL</em>, 184(16):4168+, AUG 5 2021. <a class="reference external" href="https://doi.org/10.1016/j.cell.2021.05.045">[DOI]</a>.</p></li>
<li id="id5925"><p>Priyanka Baloni, Cory C. Funk, Ben Readhead, and Nathan D. Price. Systems modeling of metabolic dysregulation in neurodegenerative diseases. <em>CURRENT OPINION IN PHARMACOLOGY</em>, 60:59–65, OCT 2021. <a class="reference external" href="https://doi.org/10.1016/j.coph.2021.06.012">[DOI]</a>.</p></li>
<li id="id5926"><p>Marco Fondi, Stefano Gonzi, Mikolaj Dziurzynski, Paola Turano, Veronica Ghini, Marzia Calvanese, Andrea Colarusso, Concetta Lauro, Ermenegilda Parrilli, and Maria Luisa Tutino. Modelling hcdkl5 heterologous expression in bacteria. <em>METABOLITES</em>, AUG 2021. <a class="reference external" href="https://doi.org/10.3390/metabo11080491">[DOI]</a>.</p></li>
<li id="id5927"><p>Howard Ramirez-Malule, Victor A. Lopez-Agudelo, David Gomez-Rios, Silvia Ochoa, Rigoberto Rios-Estepa, Stefan Junne, and Peter Neubauer. Tca cycle and its relationship with clavulanic acid production: a further interpretation by using a reduced genome-scale metabolic model of &lt;i&gt;streptomyces clavuligerus&lt;/i&gt;. <em>BIOENGINEERING-BASEL</em>, AUG 2021. <a class="reference external" href="https://doi.org/10.3390/bioengineering8080103">[DOI]</a>.</p></li>
<li id="id5928"><p>St Elmo Wilken, Victor Vera Frazao, Nima P. Saadat, and Oliver Ebenhoeh. The view of microbes as energy converters illustrates the trade-off between growth rate and yield. <em>BIOCHEMICAL SOCIETY TRANSACTIONS</em>, 49(4):1663–1674, AUG 2021. <a class="reference external" href="https://doi.org/10.1042/BST20200977">[DOI]</a>.</p></li>
<li id="id5930"><p>Zongjie Dai, Yan Zhu, Hongjun Dong, Chunhua Zhao, Yanping Zhang, and Yin Li. Enforcing atp hydrolysis enhanced anaerobic glycolysis and promoted solvent production in &lt;i&gt;clostridium acetobutylicum&lt;/i&gt;. <em>MICROBIAL CELL FACTORIES</em>, JUL 29 2021. <a class="reference external" href="https://doi.org/10.1186/s12934-021-01639-7">[DOI]</a>.</p></li>
<li id="id5931"><p>Olga Y. Echeverri-Pena, Diego A. Salazar-Barreto, Alexander Rodriguez-Lopez, Janneth Gonzalez, Carlos J. Almeciga-Diaz, Cristian H. Verano-Guevara, and Luis A. Barrera. Use of a neuron-glia genome-scale metabolic reconstruction to model the metabolic consequences of the arylsulphatase a deficiency through a systems biology approach. <em>HELIYON</em>, JUL 2021. <a class="reference external" href="https://doi.org/10.1016/j.heliyon.2021.e07671">[DOI]</a>.</p></li>
<li id="id5932"><p>Alvar J. Alonso-Lavin, Djordje Bajic, and Juan F. Poyatos. Tolerance to nadh/nad&lt;sup&gt;+&lt;/sup&gt; imbalance anticipates aging and anti-aging interventions. <em>ISCIENCE</em>, JUL 23 2021. <a class="reference external" href="https://doi.org/10.1016/j.isci.2021.102697">[DOI]</a>.</p></li>
<li id="id5933"><p>You-Tyun Wang, Min-Ru Lin, Wei-Chen Chen, Wu-Hsiung Wu, and Feng-Sheng Wang. Optimization of a modeling platform to predict oncogenes from genome-scale metabolic networks of non-small-cell lung cancers. <em>FEBS OPEN BIO</em>, 11(8):2078–2094, AUG 2021. <a class="reference external" href="https://doi.org/10.1002/2211-5463.13231">[DOI]</a>.</p></li>
<li id="id5934"><p>Devlin Moyer, Alan R. Pacheco, David B. Bernstein, and Daniel Segre. Stoichiometric modeling of artificial string chemistries reveals constraints on metabolic network structure. <em>JOURNAL OF MOLECULAR EVOLUTION</em>, 89(7):472–483, AUG 2021. <a class="reference external" href="https://doi.org/10.1007/s00239-021-10018-0">[DOI]</a>.</p></li>
<li id="id5935"><p>David Henriques, Romain Minebois, Sebastian N. Mendoza, Laura G. Macias, Roberto Perez-Torrado, Eladio Barrio, Bas Teusink, Amparo Querol, and Eva Balsa-Canto. A multiphase multiobjective dynamic genome-scale model shows different redox balancing among yeast species of the &lt;i&gt;saccharomyces&lt;/i&gt; genus in fermentation. <em>MSYSTEMS</em>, JUL-AUG 2021. <a class="reference external" href="https://doi.org/10.1128/mSystems.00260-21">[DOI]</a>.</p></li>
<li id="id5936"><p>Furkan Ozden, Metin Can Siper, Necmi Acarsoy, Tugrulcan Elmas, Bryan Marty, Xinjian Qi, and A. Ercument Cicek. Dorman: database of reconstructed metabolic networks. <em>IEEE-ACM TRANSACTIONS ON COMPUTATIONAL BIOLOGY AND BIOINFORMATICS</em>, 18(4):1474–1480, JUL-AUG 2021. <a class="reference external" href="https://doi.org/10.1109/TCBB.2019.2944905">[DOI]</a>.</p></li>
<li id="id5937"><p>Pejman Salahshouri, Modjtaba Emadi-Baygi, Mahdi Jalili, Faiz M. Khan, Olaf Wolkenhauer, and Ali Salehzadeh-Yazdi. A metabolic model of intestinal secretions: the link between human microbiota and colorectal cancer progression. <em>METABOLITES</em>, JUL 2021. <a class="reference external" href="https://doi.org/10.3390/metabo11070456">[DOI]</a>.</p></li>
<li id="id5938"><p>Carolin C. M. Schulte, Khushboo Borah, Rachel M. Wheatley, Jason J. Terpolilli, Gerhard Saalbach, Nick Crang, Daan H. de Groot, R. George Ratcliffe, Nicholas J. Kruger, Antonis Papachristodoulou, and Philip S. Poole. Metabolic control of nitrogen fixation in rhizobium-legume symbioses. <em>SCIENCE ADVANCES</em>, JUL 2021. <a class="reference external" href="https://doi.org/10.1126/sciadv.abh2433">[DOI]</a>.</p></li>
<li id="id5939"><p>Jingru Zhou, Yingping Zhuang, and Jianye Xia. Integration of enzyme constraints in a genome-scale metabolic model of &lt;i&gt;aspergillus niger&lt;/i&gt; improves phenotype predictions. <em>MICROBIAL CELL FACTORIES</em>, JUN 30 2021. <a class="reference external" href="https://doi.org/10.1186/s12934-021-01614-2">[DOI]</a>.</p></li>
<li id="id5940"><p>Alan R. Pacheco and Daniel Segre. An evolutionary algorithm for designing microbial communities via environmental modification. <em>JOURNAL OF THE ROYAL SOCIETY INTERFACE</em>, JUN 23 2021. <a class="reference external" href="https://doi.org/10.1098/rsif.2021.0348">[DOI]</a>.</p></li>
<li id="id5941"><p>Jiufu Qin, Anastasia Krivoruchko, Boyang Ji, Yu Chen, Mette Kristensen, Emre Ozdemir, Jay D. Keasling, Michael Krogh Jensen, and Jens Nielsen. Engineering yeast metabolism for the discovery and production of polyamines and polyamine analogues. <em>NATURE CATALYSIS</em>, 4(6):498+, JUN 2021. <a class="reference external" href="https://doi.org/10.1038/s41929-021-00631-z">[DOI]</a>.</p></li>
<li id="id5942"><p>Johann F. Jadebeck, Axel Theorell, Samuel Leweke, and Katharina Noeh. Hops: high-performance library for (non-)uniform sampling of convex-constrained models. <em>BIOINFORMATICS</em>, 37(12):1776–1777, JUN 15 2021. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btaa872">[DOI]</a>.</p></li>
<li id="id5943"><p>Hossein Firoozabadi, Mohammad Mahdi Mardanpour, and Ehsan Motamedian. A system-oriented strategy to enhance electron production of &lt;i&gt;synechocystis&lt;/i&gt; sp. pcc6803 in bio-photovoltaic devices: experimental and modeling insights. <em>SCIENTIFIC REPORTS</em>, JUN 10 2021. <a class="reference external" href="https://doi.org/10.1038/s41598-021-91906-9">[DOI]</a>.</p></li>
<li id="id5944"><p>Michael A. Henson. Interrogation of the perturbed gut microbiota in gouty arthritis patients through in silico metabolic modeling. <em>ENGINEERING IN LIFE SCIENCES</em>, 21(7):489–501, JUL 2021. <a class="reference external" href="https://doi.org/10.1002/elsc.202100003">[DOI]</a>.</p></li>
<li id="id5945"><p>David Hala, Patricia Faulkner, Kai He, Manoj Kamalanathan, Mikeelee Brink, Kristina Simons, Meltem Apaydin, Beatrice Hernout, Lene H. Petersen, Ivan Ivanov, and Xiaoning Qian. An integrated in vivo and in silico analysis of the metabolism disrupting effects of cpi-613 on embryo-larval zebrafish (danio rerio). <em>COMPARATIVE BIOCHEMISTRY AND PHYSIOLOGY C-TOXICOLOGY &amp; PHARMACOLOGY</em>, OCT 2021. <a class="reference external" href="https://doi.org/10.1016/j.cbpc.2021.109084">[DOI]</a>.</p></li>
<li id="id5946"><p>Thora Bjorg Sigmarsdottir, Sarah McGarrity, James T. Yurkovich, Ottar Rolfsson, and Olafur Eysteinn Sigurjonsson. Analyzing metabolic states of adipogenic and osteogenic differentiation in human mesenchymal stem cells via genome scale metabolic model reconstruction. <em>FRONTIERS IN CELL AND DEVELOPMENTAL BIOLOGY</em>, JUN 4 2021. <a class="reference external" href="https://doi.org/10.3389/fcell.2021.642681">[DOI]</a>.</p></li>
<li id="id5947"><p>Kristopher A. Hunt, Natasha D. Mallette, Brent M. Peyton, and Ross P. Carlson. In silico analysis of functionalized hydrocarbon production using ehrlich pathway and fatty acid derivatives in an endophytic fungus. <em>JOURNAL OF FUNGI</em>, JUN 2021. <a class="reference external" href="https://doi.org/10.3390/jof7060435">[DOI]</a>.</p></li>
<li id="id5948"><p>Mahdi Jalili, Martin Scharm, Olaf Wolkenhauer, Mehdi Damaghi, and Ali Salehzadeh-Yazdi. Exploring the metabolic heterogeneity of cancers: a benchmark study of context-specific models. <em>JOURNAL OF PERSONALIZED MEDICINE</em>, JUN 2021. <a class="reference external" href="https://doi.org/10.3390/jpm11060496">[DOI]</a>.</p></li>
<li id="id5949"><p>Felipe Scott, Luz Yanez, Raul Conejeros, Blanca Araya, and Alberto Vergara-Fernandez. Two internal bottlenecks cause the overflow metabolism leading to poly (3-hydroxybutyrate) production in &lt;i&gt;azohydromonas lata&lt;/i&gt; dsm1123. <em>JOURNAL OF ENVIRONMENTAL CHEMICAL ENGINEERING</em>, AUG 2021. <a class="reference external" href="https://doi.org/10.1016/j.jece.2021.105665">[DOI]</a>.</p></li>
<li id="id5951"><p>Chenchen Gao, Jiarui Yang, Tong Hao, Jingjing Li, and Jinsheng Sun. Reconstruction of &lt;i&gt;litopenaeus vannamei&lt;/i&gt; genome-scale metabolic network model and nutritional requirements analysis of different shrimp commercial varieties. <em>FRONTIERS IN GENETICS</em>, MAY 12 2021. <a class="reference external" href="https://doi.org/10.3389/fgene.2021.658109">[DOI]</a>.</p></li>
<li id="id5952"><p>Thomas Eng, Deepanwita Banerjee, Andrew K. Lau, Emily Bowden, Robin A. Herbert, Jessica Trinh, Jan-Philip Prahl, Adam Deutschbauer, Deepti Tanjore, and Aindrila Mukhopadhyay. Engineering &lt;i&gt;pseudomonas putida&lt;/i&gt; for efficient aromatic conversion to bioproduct using high throughput screening in a bioreactor. <em>METABOLIC ENGINEERING</em>, 66:229–238, JUL 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.04.015">[DOI]</a>.</p></li>
<li id="id5953"><p>Giuseppe Magazzu, Guido Zampieri, and Claudio Angione. Multimodal regularized linear models with flux balance analysis for mechanistic integration of omics data. <em>BIOINFORMATICS</em>, 37(20):3546–3552, OCT 15 2021. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btab324">[DOI]</a>.</p></li>
<li id="id5954"><p>Padhmanand Sudhakar, Kathleen Machiels, Bram Verstockt, Tamas Korcsmaros, and Severine Vermeire. Computational biology and machine learning approaches to understand mechanistic microbiome-host interactions. <em>FRONTIERS IN MICROBIOLOGY</em>, MAY 11 2021. <a class="reference external" href="https://doi.org/10.3389/fmicb.2021.618856">[DOI]</a>.</p></li>
<li id="id5955"><p>Almut Heinken, Johannes Hertel, and Ines Thiele. Metabolic modelling reveals broad changes in gut microbial metabolism in inflammatory bowel disease patients with dysbiosis. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, MAY 6 2021. <a class="reference external" href="https://doi.org/10.1038/s41540-021-00178-6">[DOI]</a>.</p></li>
<li id="id5956"><p>Lekha T. Pazhamala, Himabindu Kudapa, Wolfram Weckwerth, A. Harvey Millar, and Rajeev K. Varshney. Systems biology for crop improvement. <em>PLANT GENOME</em>, JUL 2021. <a class="reference external" href="https://doi.org/10.1002/tpg2.20098">[DOI]</a>.</p></li>
<li id="id5957"><p>Wen-Tao Tang, Tian-Wei Hao, and Guang-Hao Chen. Comparative metabolic modeling of multiple sulfate-reducing prokaryotes reveals versatile energy conservation mechanisms. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 118(7):2676–2693, JUL 2021. <a class="reference external" href="https://doi.org/10.1002/bit.27787">[DOI]</a>.</p></li>
<li id="id5958"><p>Nana Y. D. Ankrah, Brandon E. Barker, Joan Song, Cindy Wu, John G. McMullen, II, and Angela E. Douglas. Predicted metabolic function of the gut microbiota of &lt;i&gt;drosophila melanogaster&lt;/i&gt;. <em>MSYSTEMS</em>, MAY-JUN 2021. <a class="reference external" href="https://doi.org/10.1128/mSystems.01369-20">[DOI]</a>.</p></li>
<li id="id5959"><p>Despoina Kerselidou, Bushra Saeed Dohai, David R. Nelson, Sarah Daakour, Nicolas De Cock, Zahra Al Oula Hassoun, Dae-Kyum Kim, Julien Olivet, Diana C. El Assal, Ashish Jaiswal, Amnah Alzahmi, Deeya Saha, Charlotte Pain, Filip Matthijssens, Pierre Lemaitre, Michael Herfs, Julien Chapuis, Bart Ghesquiere, Didier Vertommen, Verena Kriechbaumer, Kevin Knoops, Carmen Lopez-Iglesias, Marc van Zandvoort, Jean-Charles Lambert, Julien Hanson, Christophe Desmet, Marc Thiry, Kyle J. Lauersen, Marc Vidal, Pieter Van Vlierberghe, Franck Dequiedt, Kourosh Salehi-Ashtiani, and Jean-Claude Twizere. Alternative glycosylation controls endoplasmic reticulum dynamics and tubular extension in mammalian cells. <em>SCIENCE ADVANCES</em>, MAY 2021. <a class="reference external" href="https://doi.org/10.1126/sciadv.abe8349">[DOI]</a>.</p></li>
<li id="id5960"><p>Cheewin Kittikunapong, Suhui Ye, Patricia Magadan-Corpas, Alvaro Perez-Valero, Claudio J. Villar, Felipe Lombo, and Eduard J. Kerkhoven. Reconstruction of a genome-scale metabolic model of &lt;i&gt;streptomyces albus&lt;/i&gt; j1074: improved engineering strategies in natural product synthesis. <em>METABOLITES</em>, MAY 2021. <a class="reference external" href="https://doi.org/10.3390/metabo11050304">[DOI]</a>.</p></li>
<li id="id5961"><p>Veronica Porubsky, Lucian Smith, and Herbert M. Sauro. Publishing reproducible dynamic kinetic models. <em>BRIEFINGS IN BIOINFORMATICS</em>, MAY 2021. <a class="reference external" href="https://doi.org/10.1093/bib/bbaa152">[DOI]</a>.</p></li>
<li id="id5962"><p>David J. Sanders, Saskia Inniss, Gregory Sebepos-Rogers, Farooq Z. Rahman, and Andrew M. Smith. The role of the microbiome in gastrointestinal inflammation. <em>BIOSCIENCE REPORTS</em>, MAY 2021. <a class="reference external" href="https://doi.org/10.1042/BSR20203850">[DOI]</a>.</p></li>
<li id="id5963"><p>Christian Schulz, Tjasa Kumelj, Emil Karlsen, and Eivind Almaas. Genome-scale metabolic modelling when changes in environmental conditions affect biomass composition. <em>PLOS COMPUTATIONAL BIOLOGY</em>, MAY 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1008528">[DOI]</a>.</p></li>
<li id="id5964"><p>Takeyuki Tamura. L1 norm minimal mode-based methods for listing reaction network designs for metabolite production. <em>IEICE TRANSACTIONS ON INFORMATION AND SYSTEMS</em>, E104D(5):679–687, MAY 2021. <a class="reference external" href="https://doi.org/10.1587/transinf.2020EDP7247">[DOI]</a>.</p></li>
<li id="id5965"><p>A. S. Trigos, B. W. Goudey, J. Bedo, T. C. Conway, N. G. Faux, and K. L. Wyres. Collateral sensitivity to β-lactam drugs in drug-resistant tuberculosis is driven by the transcriptional wiring of blai operon genes. <em>MSPHERE</em>, MAY-JUN 2021. <a class="reference external" href="https://doi.org/10.1128/mSphere.00245-21">[DOI]</a>.</p></li>
<li id="id5966"><p>Eunice van Pelt-KleinJan, Daan H. de Groot, and Bas Teusink. Understanding fba solutions under multiple nutrient limitations. <em>METABOLITES</em>, MAY 2021. <a class="reference external" href="https://doi.org/10.3390/metabo11050257">[DOI]</a>.</p></li>
<li id="id5967"><p>Freddy Castillo Alfonso, Gabriel Vigueras-Ramirez, Luis Manuel Rosales-Colunga, Alberto del Monte-Martinez, and Roberto Olivares Hernandez. Propionate as the preferred carbon source to produce 3-indoleacetic acid in &lt;i&gt;b. subtilis&lt;/i&gt;: comparative flux analysis using five carbon sources. <em>MOLECULAR OMICS</em>, 17(4):554–564, AUG 1 2021. <a class="reference external" href="https://doi.org/10.1039/d1mo00039j">[DOI]</a>.</p></li>
<li id="id5968"><p>Till Tiso, Tanja Narancic, Ren Wei, Eric Pollet, Niall Beagan, Katja Schroeder, Annett Honak, Mengying Jiang, Shane T. Kenny, Nick Wierckx, Remi Perrin, Luc Averous, Wolfgang Zimmermann, Kevin O'Connor, and Lars M. Blank. Towards bio-upcycling of polyethylene terephthalate. <em>METABOLIC ENGINEERING</em>, 66:167–178, JUL 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.03.011">[DOI]</a>.</p></li>
<li id="id5969"><p>Ratchaprapa Kamsen, Saowalak Kalapanulak, Porntip Chiewchankaset, and Treenut Saithong. Transcriptome integrated metabolic modeling of carbon assimilation underlying storage root development in cassava. <em>SCIENTIFIC REPORTS</em>, APR 22 2021. <a class="reference external" href="https://doi.org/10.1038/s41598-021-88129-3">[DOI]</a>.</p></li>
<li id="id5970"><p>Ruiwen Hu, Haiming Zhao, Xihui Xu, Zhigang Wang, Ke Yu, Longfei Shu, Qingyun Yan, Bo Wu, Cehui Mo, Zhili He, and Cheng Wang. Bacteria-driven phthalic acid ester biodegradation: current status and emerging opportunities. <em>ENVIRONMENT INTERNATIONAL</em>, SEP 2021. <a class="reference external" href="https://doi.org/10.1016/j.envint.2021.106560">[DOI]</a>.</p></li>
<li id="id5971"><p>Daniel A. Norena-Caro, Cristal Zuniga, Amber J. Pete, Sven A. Saemundsson, Morgan R. Donaldson, Alexandria J. Adams, Kerry M. Dooley, Karsten Zengler, and Michael G. Benton. Analysis of the cyanobacterial amino acid metabolism with a precise genome-scale metabolic reconstruction of &lt;i&gt;anabaena&lt;/i&gt; sp. utex 2576. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, JUL 2021. <a class="reference external" href="https://doi.org/10.1016/j.bej.2021.108008">[DOI]</a>.</p></li>
<li id="id5972"><p>Daniel J. Lugar, Sean G. Mack, and Ganesh Sriram. Netred, an algorithm to reduce genome-scale metabolic networks and facilitate the analysis of flux predictions. <em>METABOLIC ENGINEERING</em>, 65:207–222, MAY 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2020.11.003">[DOI]</a>.</p></li>
<li id="id5973"><p>Song-Min Schinn, Carly Morrison, Wei Wei, Lin Zhang, and Nathan E. Lewis. Systematic evaluation of parameters for genome-scale metabolic models of cultured mammalian cells. <em>METABOLIC ENGINEERING</em>, 66:21–30, JUL 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.03.013">[DOI]</a>.</p></li>
<li id="id5974"><p>Ozlem Altay, Cheng Zhang, Hasan Turkez, Jens Nielsen, Mathias Uhlen, and Adil Mardinoglu. Revealing the metabolic alterations during biofilm development of &lt;i&gt;burkholderia cenocepacia&lt;/i&gt; based on genome-scale metabolic modeling. <em>METABOLITES</em>, APR 2021. <a class="reference external" href="https://doi.org/10.3390/metabo11040221">[DOI]</a>.</p></li>
<li id="id5975"><p>Tobias B. Alter, Lars M. Blank, and Birgitta E. Ebert. Proteome regulation patterns determine &lt;i&gt;escherichia coli&lt;/i&gt; wild-type and mutant phenotypes. <em>MSYSTEMS</em>, APR 2021. <a class="reference external" href="https://doi.org/10.1128/mSystems.00625-20">[DOI]</a>.</p></li>
<li id="id5976"><p>Yu Chen, Eunice van Pelt-KleinJan, Berdien van Olst, Sieze Douwenga, Sjef Boeren, Herwig Bachmann, Douwe Molenaar, Jens Nielsen, and Bas Teusink. Proteome constraints reveal targets for improving microbial fitness in nutrient-rich environments. <em>MOLECULAR SYSTEMS BIOLOGY</em>, APR 2021. <a class="reference external" href="https://doi.org/10.15252/msb.202010093">[DOI]</a>.</p></li>
<li id="id5977"><p>Ankur Gupta, Ajay Kumar, Rajat Anand, Nandadulal Bairagi, and Samrat Chatterjee. Genome scale metabolic model driven strategy to delineate host response to &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; infection. <em>MOLECULAR OMICS</em>, 17(2):296–306, APR 1 2021. <a class="reference external" href="https://doi.org/10.1039/d0mo00138d">[DOI]</a>.</p></li>
<li id="id5978"><p>Karl Kochanowski, Hiroyuki Okano, Vadim Patsalo, James Williamson, Uwe Sauer, and Terence Hwa. Global coordination of metabolic pathways in &lt;i&gt;escherichia coli&lt;/i&gt; by active and passive regulation. <em>MOLECULAR SYSTEMS BIOLOGY</em>, APR 2021. <a class="reference external" href="https://doi.org/10.15252/msb.202010064">[DOI]</a>.</p></li>
<li id="id5979"><p>Peng Liu, Shuai Wang, Chao Li, Yingping Zhuang, Jianye Xia, and Henk Noorman. Dynamic response of &lt;i&gt;aspergillus niger&lt;/i&gt; to periodical glucose pulse stimuli in chemostat cultures. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 118(6):2265–2282, JUN 2021. <a class="reference external" href="https://doi.org/10.1002/bit.27739">[DOI]</a>.</p></li>
<li id="id5980"><p>Piyush Nanda and Amit Ghosh. Genome scale-differential flux analysis reveals deregulation of lung cell metabolism on sars-cov-2 infection. <em>PLOS COMPUTATIONAL BIOLOGY</em>, APR 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1008860">[DOI]</a>.</p></li>
<li id="id5981"><p>Gurneet S. Sangha, Craig J. Goergen, Steven J. Prior, Sushant M. Ranadive, and Alisa M. Clyne. Preclinical techniques to investigate exercise training in vascular pathophysiology. <em>AMERICAN JOURNAL OF PHYSIOLOGY-HEART AND CIRCULATORY PHYSIOLOGY</em>, 320(4):H1566–H1600, APR 2021. <a class="reference external" href="https://doi.org/10.1152/ajpheart.00719.2020">[DOI]</a>.</p></li>
<li id="id5982"><p>Ewelina Weglarz-Tomczak, Thierry D. G. A. Mondeel, Diewertje G. E. Piebes, and Hans V. Westerhoff. Simultaneous integration of gene expression and nutrient availability for studying the metabolism of hepatocellular carcinoma cell lines. <em>BIOMOLECULES</em>, APR 2021. <a class="reference external" href="https://doi.org/10.3390/biom11040490">[DOI]</a>.</p></li>
<li id="id5983"><p>Vivienne Mol, Martyn Bennett, Benjamin J. Sanchez, Beata K. Lisowska, Markus J. Herrgard, Alex Toftgaard Nielsen, David J. Leak, and Nikolaus Sonnenschein. Genome-scale metabolic modeling of&lt;i&gt; p.&lt;/i&gt;&lt;i&gt; thermoglucosidasius&lt;/i&gt; ncimb 11955 reveals metabolic bottlenecks in anaerobic metabolism. <em>METABOLIC ENGINEERING</em>, 65:123–134, MAY 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.03.002">[DOI]</a>.</p></li>
<li id="id5984"><p>Albert Enrique Tafur Rangel, Wendy Rios, Daisy Mejia, Carmen Ojeda, Ross Carlson, Jorge Mario Gomez Ramirez, and Andres Fernando Gonzalez Barrios. &lt;i&gt;in silico&lt;/i&gt; design for systems-based metabolic engineering for the bioconversion of valuable compounds from industrial by-products. <em>FRONTIERS IN GENETICS</em>, MAR 26 2021. <a class="reference external" href="https://doi.org/10.3389/fgene.2021.633073">[DOI]</a>.</p></li>
<li id="id5985"><p>Karim Azer, Chanchala D. Kaddi, Jeffrey S. Barrett, Jane P. F. Bai, Sean T. McQuade, Nathaniel J. Merrill, Benedetto Piccoli, Susana Neves-Zaph, Luca Marchetti, Rosario Lombardo, Silvia Parolo, Selva Rupa Christinal Immanuel, and Nitin S. Baliga. History and future perspectives on the discipline of quantitative systems pharmacology modeling and its applications. <em>FRONTIERS IN PHYSIOLOGY</em>, MAR 25 2021. <a class="reference external" href="https://doi.org/10.3389/fphys.2021.637999">[DOI]</a>.</p></li>
<li id="id5986"><p>Ana Vila-Santa, M. Ahsanul Islam, Frederico C. Ferreira, Kristala L. J. Prather, and Nuno P. Mira. Prospecting biochemical pathways to implement microbe-based production of the new-to-nature platform chemical levulinic acid. <em>ACS SYNTHETIC BIOLOGY</em>, 10(4):724–736, APR 16 2021. <a class="reference external" href="https://doi.org/10.1021/acssynbio.0c00518">[DOI]</a>.</p></li>
<li id="id5987"><p>Yu Chen, Feiran Li, Jiwei Mao, Yun Chen, and Jens Nielsen. Yeast optimizes metal utilization based on metabolic network and enzyme kinetics. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, MAR 23 2021. <a class="reference external" href="https://doi.org/10.1073/pnas.2020154118">[DOI]</a>.</p></li>
<li id="id5988"><p>Mattia G. Gollub, Hans-Michael Kaltenbach, and Joerg Stelling. Probabilistic thermodynamic analysis of metabolic networks. <em>BIOINFORMATICS</em>, 37(18):2938–2945, SEP 15 2021. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btab194">[DOI]</a>.</p></li>
<li id="id5989"><p>Tuure Hameri, Georgios Fengos, and Vassily Hatzimanikatis. The effects of model complexity and size on metabolic flux distribution and control: case study in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BMC BIOINFORMATICS</em>, MAR 20 2021. <a class="reference external" href="https://doi.org/10.1186/s12859-021-04066-y">[DOI]</a>.</p></li>
<li id="id5990"><p>E. Zelle, N. Pfelzer, M. Oldiges, A. Koch-Koerfges, M. Bott, K. Noeh, and W. Wiechert. An energetic profile of&lt;i&gt; corynebacterium&lt;/i&gt;&lt;i&gt; glutamicum&lt;/i&gt; underpinned by measured biomass yield on atp. <em>METABOLIC ENGINEERING</em>, 65:66–78, MAY 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.03.006">[DOI]</a>.</p></li>
<li id="id5991"><p>Thiago J. B. Mesquita, Gilson Campani, Roberto C. Giordano, Teresa C. Zangirolami, and Antonio C. L. Horta. Machine learning applied for metabolic flux-based control of micro-aerated fermentations in bioreactors. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 118(5):2076–2091, MAY 2021. <a class="reference external" href="https://doi.org/10.1002/bit.27721">[DOI]</a>.</p></li>
<li id="id5992"><p>Vetle Simensen, Andre Voigt, and Eivind Almaas. High-quality genome-scale metabolic model of &lt;i&gt;aurantiochytrium&lt;/i&gt; sp. t66. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 118(5):2105–2117, MAY 2021. <a class="reference external" href="https://doi.org/10.1002/bit.27726">[DOI]</a>.</p></li>
<li id="id5993"><p>H. Kamp, J. Wahrheit, S. Stinchcombe, T. Walk, F. Stauber, and B. von Ravenzwaay. Succinate dehydrogenase inhibitors: &lt;i&gt;in silico&lt;/i&gt; flux analysis and &lt;i&gt;in vivo&lt;/i&gt; metabolomics investigations show no severe metabolic consequences for rats and humans. <em>FOOD AND CHEMICAL TOXICOLOGY</em>, APR 2021. <a class="reference external" href="https://doi.org/10.1016/j.fct.2021.112085">[DOI]</a>.</p></li>
<li id="id5994"><p>Ecehan Abdik and Tunahan Cakir. Systematic investigation of mouse models of parkinson's disease by transcriptome mapping on a brain-specific genome-scale metabolic network. <em>MOLECULAR OMICS</em>, 17(4):492+, AUG 1 2021. <a class="reference external" href="https://doi.org/10.1039/d0mo00135j">[DOI]</a>.</p></li>
<li id="id5995"><p>Bonnie Dougherty, V, Kristopher D. Rawls, Glynis L. Kolling, Kalyan C. Vinnakota, Anders Wallqvist, and Jason A. Papin. Identifying functional metabolic shifts in heart failure with the integration of omics data and a heart-specific, genome-scale model. <em>CELL REPORTS</em>, MAR 9 2021. <a class="reference external" href="https://doi.org/10.1016/j.celrep.2021.108836">[DOI]</a>.</p></li>
<li id="id5996"><p>Albert Enrique Tafur Rangel, Luis H. Reyes, Jorge Mario Gomez Ramirez, and Andres Fernando Gonzalez Barrios. Optimization of glycerol consumption in wild-type &lt;i&gt;escherichia coli&lt;/i&gt; using central carbon modeling as an alternative approach. <em>BIOFUELS BIOPRODUCTS &amp; BIOREFINING-BIOFPR</em>, 15(3):825–839, MAY 2021. <a class="reference external" href="https://doi.org/10.1002/bbb.2205">[DOI]</a>.</p></li>
<li id="id5997"><p>Dorines Rosario, Gholamreza Bidkhori, Sunjae Lee, Janis Bedarf, Falk Hildebrand, Emmanuelle Le Chatelier, Mathias Uhlen, Stanislav Dusko Ehrlich, Gordon Proctor, Ullrich Wuellner, Adil Mardinoglu, and Saeed Shoaie. Systematic analysis of gut microbiome reveals the role of bacterial folate and homocysteine metabolism in parkinson's disease. <em>CELL REPORTS</em>, MAR 2 2021. <a class="reference external" href="https://doi.org/10.1016/j.celrep.2021.108807">[DOI]</a>.</p></li>
<li id="id5998"><p>Hunor Bartos, Marta Balazs, Ildiko Hajnalka Kuzman, Szabolcs Lanyi, and Ildiko Miklossy. Production of high added-value chemicals in &lt;i&gt;basfia succiniciproducens&lt;/i&gt;: role of medium composition. <em>SUSTAINABILITY</em>, MAR 2021. <a class="reference external" href="https://doi.org/10.3390/su13063513">[DOI]</a>.</p></li>
<li id="id5999"><p>John Hendry, I, Hoang Dinh, V, Debolina Sarkar, Lin Wang, Anindita Bandyopadhyay, Himadri B. Pakrasi, and Costas D. Maranas. A genome-scale metabolic model of &lt;i&gt;anabaena&lt;/i&gt; 33047 to guide genetic modifications to overproduce nylon monomers. <em>METABOLITES</em>, MAR 2021. <a class="reference external" href="https://doi.org/10.3390/metabo11030168">[DOI]</a>.</p></li>
<li id="id6000"><p>MacLean G. Kohlmeier, Ben A. Bailey-Elkin, Brian L. Mark, and Ivan J. Oresnik. Characterization of the sorbitol dehydrogenase smos from &lt;i&gt;sinorhizobium meliloti&lt;/i&gt; 1021. <em>ACTA CRYSTALLOGRAPHICA SECTION D-STRUCTURAL BIOLOGY</em>, 77(3):380–390, MAR 1 2021. <a class="reference external" href="https://doi.org/10.1107/S2059798321001017">[DOI]</a>.</p></li>
<li id="id6001"><p>Masakazu Toyoshima, Chiaki Yamamoto, Yoshifumi Ueno, Yoshihiro Toya, Seiji Akimoto, and Hiroshi Shimizu. Role of type i nadh dehydrogenase in synechocystis sp. pcc 6803 under phycobilisome excited red light. <em>PLANT SCIENCE</em>, MAR 2021. <a class="reference external" href="https://doi.org/10.1016/j.plantsci.2020.110798">[DOI]</a>.</p></li>
<li id="id6002"><p>Akshit Goyal, Tong Wang, Veronika Dubinkina, and Sergei Maslov. Ecology-guided prediction of cross-feeding interactions in the human gut microbiome. <em>NATURE COMMUNICATIONS</em>, FEB 26 2021. <a class="reference external" href="https://doi.org/10.1038/s41467-021-21586-6">[DOI]</a>.</p></li>
<li id="id6003"><p>Snorre Sulheim, Fredrik A. Fossheim, Alexander Wentzel, and Eivind Almaas. Automatic reconstruction of metabolic pathways from identified biosynthetic gene clusters. <em>BMC BIOINFORMATICS</em>, FEB 23 2021. <a class="reference external" href="https://doi.org/10.1186/s12859-021-03985-0">[DOI]</a>.</p></li>
<li id="id6004"><p>Laura Judith Marcos-Zambrano, Kanita Karaduzovic-Hadziabdic, Tatjana Loncar Turukalo, Piotr Przymus, Vladimir Trajkovik, Oliver Aasmets, Magali Berland, Aleksandra Gruca, Jasminka Hasic, Karel Hron, Thomas Klammsteiner, Mikhail Kolev, Leo Lahti, Marta B. Lopes, Victor Moreno, Irina Naskinova, Elin Org, Ines Paciencia, Georgios Papoutsoglou, Rajesh Shigdel, Blaz Stres, Baiba Vilne, Malik Yousef, Eftim Zdravevski, Ioannis Tsamardinos, Enrique Carrillo de Santa Pau, Marcus J. Claesson, Isabel Moreno-Indias, Jaak Truu, and ML4Microbiome. Applications of machine learning in human microbiome studies: a review on feature selection, biomarker identification, disease prediction and treatment. <em>FRONTIERS IN MICROBIOLOGY</em>, FEB 19 2021. <a class="reference external" href="https://doi.org/10.3389/fmicb.2021.634511">[DOI]</a>.</p></li>
<li id="id6005"><p>Song-Min Schinn, Carly Morrison, Wei Wei, Lin Zhang, and Nathan E. Lewis. A genome-scale metabolic network model and machine learning predict amino acid concentrations in chinese hamster ovary cell cultures. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 118(5):2118–2123, MAY 2021. <a class="reference external" href="https://doi.org/10.1002/bit.27714">[DOI]</a>.</p></li>
<li id="id6006"><p>David B. Bernstein, Snorre Sulheim, Eivind Almaas, and Daniel Segre. Addressing uncertainty in genome-scale metabolic model reconstruction and analysis. <em>GENOME BIOLOGY</em>, FEB 18 2021. <a class="reference external" href="https://doi.org/10.1186/s13059-021-02289-z">[DOI]</a>.</p></li>
<li id="id6007"><p>Barbara Bourgade, Nigel P. Minton, and M. Ahsanul Islam. Genetic and metabolic engineering challenges of c1-gas fermenting acetogenic chassis organisms. <em>FEMS MICROBIOLOGY REVIEWS</em>, MAR 2021. <a class="reference external" href="https://doi.org/10.1093/femsre/fuab008">[DOI]</a>.</p></li>
<li id="id6008"><p>Thea Os Andersen, Benoit J. Kunath, Live H. Hagen, Magnus O. Arntzen, and Phillip B. Pope. Rumen metaproteomics: closer to linking rumen microbial function to animal productivity traits. <em>METHODS</em>, 186(SI):42–51, FEB 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymeth.2020.07.011">[DOI]</a>.</p></li>
<li id="id6009"><p>Shany Ofaim, Snorre Sulheim, Eivind Almaas, Daniel Sher, and Daniel Segre. Dynamic allocation of carbon storage and nutrient-dependent exudation in a revised genome-scale model of &lt;i&gt;prochlorococcus&lt;/i&gt;. <em>FRONTIERS IN GENETICS</em>, FEB 9 2021. <a class="reference external" href="https://doi.org/10.3389/fgene.2021.586293">[DOI]</a>.</p></li>
<li id="id6010"><p>Hui-Yin Chang, Sean M. Colby, Xiuxia Du, Javier D. Gomez, Maximilian J. Helf, Katerina Kechris, Christine R. Kirkpatrick, Shuzhao Li, Gary J. Patti, Ryan S. Renslow, Shankar Subramaniam, Mukesh Verma, Jianguo Xia, and Jamey D. Young. A practical guide to metabolomics software development. <em>ANALYTICAL CHEMISTRY</em>, 93(4):1912–1923, FEB 2 2021. <a class="reference external" href="https://doi.org/10.1021/acs.analchem.0c03581">[DOI]</a>.</p></li>
<li id="id6011"><p>Ying Li, He Xian, Ya Xu, Yuan Zhu, Zhijie Sun, Qian Wang, and Qingsheng Qi. Fine tuning the glycolytic flux ratio of ep-bifido pathway for mevalonate production by enhancing glucose-6-phosphate dehydrogenase (zwf) and crispri suppressing 6-phosphofructose kinase (pfka) in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>MICROBIAL CELL FACTORIES</em>, FEB 2 2021. <a class="reference external" href="https://doi.org/10.1186/s12934-021-01526-1">[DOI]</a>.</p></li>
<li id="id6012"><p>Michael A. Henson. Computational modeling of the gut microbiota reveals putative metabolic mechanisms of recurrent &lt;i&gt;clostridioides difficile&lt;/i&gt; infection. <em>PLOS COMPUTATIONAL BIOLOGY</em>, FEB 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1008782">[DOI]</a>.</p></li>
<li id="id6013"><p>Dohyeon Kim, Youngshin Kim, and Sung Ho Yoon. Development of a genome-scale metabolic model and phenome analysis of the probiotic &lt;i&gt;escherichia coli&lt;/i&gt; strain nissle 1917. <em>INTERNATIONAL JOURNAL OF MOLECULAR SCIENCES</em>, FEB 2021. <a class="reference external" href="https://doi.org/10.3390/ijms22042122">[DOI]</a>.</p></li>
<li id="id6014"><p>Julia Koblitz, Sabine Eva Will, S. Alexander Riemer, Thomas Ulas, Meina Neumann-Schaal, and Dietmar Schomburg. The metano modeling toolbox mmtb: an intuitive, web-based toolbox introduced by two use cases. <em>METABOLITES</em>, FEB 2021. <a class="reference external" href="https://doi.org/10.3390/metabo11020113">[DOI]</a>.</p></li>
<li id="id6015"><p>Pablo Rodriguez-Mier, Nathalie Poupin, Carlo de Blasio, Laurent Le Cam, and Fabien Jourdan. Dexom: diversity-based enumeration of optimal context-specific metabolic networks. <em>PLOS COMPUTATIONAL BIOLOGY</em>, FEB 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1008730">[DOI]</a>.</p></li>
<li id="id6016"><p>Woosub Shin and Joseph L. Hellerstein. Isolating structural errors in reaction networks in systems biology. <em>BIOINFORMATICS</em>, 37(3):388–395, FEB 1 2021. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btaa720">[DOI]</a>.</p></li>
<li id="id6017"><p>Aitor Villate, Markel San Nicolas, Mara Gallastegi, Pierre-Antoine Aulas, Maitane Olivares, Aresatz Usobiaga, Nestor Etxebarria, and Oier Aizpurua-Olaizola. Review: metabolomics as a prediction tool for plants performance under environmental stress. <em>PLANT SCIENCE</em>, FEB 2021. <a class="reference external" href="https://doi.org/10.1016/j.plantsci.2020.110789">[DOI]</a>.</p></li>
<li id="id6018"><p>Ewelina Weglarz-Tomczak, Demi J. Rijlaarsdam, Jakub M. Tomczak, and Stanley Brul. Gem-based metabolic profiling for human bone osteosarcoma under different glucose and glutamine availability. <em>INTERNATIONAL JOURNAL OF MOLECULAR SCIENCES</em>, FEB 2021. <a class="reference external" href="https://doi.org/10.3390/ijms22031470">[DOI]</a>.</p></li>
<li id="id6019"><p>R. P. van Rosmalen, R. W. Smith, V. A. P. Martins dos Santos, C. Fleck, and M. Suarez-Diez. Model reduction of genome-scale metabolic models as a basis for targeted kinetic models. <em>METABOLIC ENGINEERING</em>, 64:74–84, MAR 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2021.01.008">[DOI]</a>.</p></li>
<li id="id6020"><p>Varshit Dusad, Denise Thiel, Mauricio Barahona, Hector C. Keun, and Diego A. Oyarzun. Opportunities at the interface of network science and metabolic modeling. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, JAN 25 2021. <a class="reference external" href="https://doi.org/10.3389/fbioe.2020.591049">[DOI]</a>.</p></li>
<li id="id6021"><p>Patricia Martins Conde, Thomas Pfau, Maria Pires Pacheco, and Thomas Sauter. A dynamic multi-tissue model to study human metabolism. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, JAN 22 2021. <a class="reference external" href="https://doi.org/10.1038/s41540-020-00159-1">[DOI]</a>.</p></li>
<li id="id6022"><p>Maciek R. Antoniewicz. A guide to metabolic flux analysis in metabolic engineering: methods, tools and applications. <em>METABOLIC ENGINEERING</em>, 63(SI):2–12, JAN 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2020.11.002">[DOI]</a>.</p></li>
<li id="id6023"><p>Irene Otero-Muras and Pablo Carbonell. Automated engineering of synthetic metabolic pathways for efficient biomanufacturing. <em>METABOLIC ENGINEERING</em>, 63(SI):61–80, JAN 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2020.11.012">[DOI]</a>.</p></li>
<li id="id6024"><p>Patrick F. Suthers, Charles J. Foster, Debolina Sarkar, Lin Wang, and Costas D. Maranas. Recent advances in constraint and machine learning-based metabolic modeling by leveraging stoichiometric balances, thermodynamic feasibility and kinetic law formalisms. <em>METABOLIC ENGINEERING</em>, 63(SI):13–33, JAN 2021. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2020.11.013">[DOI]</a>.</p></li>
<li id="id6025"><p>Sathyanarayanan Ranganayaki, Neema Jamshidi, Mohamad Aiyaz, Santhosh-Kumar Rashmi, Narayanappa Gayathri, Pulleri Kandi Harsha, Balasundaram Padmanabhan, and Muchukunte Mukunda Srinivas Bharath. Inhibition of mitochondrial complex ii in neuronal cells triggers unique pathways culminating in autophagy with implications for neurodegeneration. <em>SCIENTIFIC REPORTS</em>, JAN 15 2021. <a class="reference external" href="https://doi.org/10.1038/s41598-020-79339-2">[DOI]</a>.</p></li>
<li id="id6026"><p>Jibin Liu, Anchun Cheng, Mingshu Wang, Mafeng Liu, Dekang Zhu, Qiao Yang, Ying Wu, Renyong Jia, Shun Chen, Xinxin Zhao, Shaqiu Zhang, Juan Huang, Xumin Ou, Sai Mao, Qun Gao, Xingjian Wen, Ling Zhang, Yunya Liu, Yanling Yu, Bin Tian, Leichang Pan, Mujeeb Ur Rehman, and Xiaoyue Chen. Comparative genomics and metabolomics analysis of &lt;i&gt;riemerella anatipestifer&lt;/i&gt; strain ch-1 and ch-2. <em>SCIENTIFIC REPORTS</em>, JAN 12 2021. <a class="reference external" href="https://doi.org/10.1038/s41598-020-79733-w">[DOI]</a>.</p></li>
<li id="id6027"><p>Nan Xu, Qiyuan Yang, Xiaojing Yang, Mingqi Wang, and Minliang Guo. Reconstruction and analysis of a genome-scale metabolic model for &lt;i&gt;agrobacterium tumefaciens&lt;/i&gt;. <em>MOLECULAR PLANT PATHOLOGY</em>, 22(3):348–360, MAR 2021. <a class="reference external" href="https://doi.org/10.1111/mpp.13032">[DOI]</a>.</p></li>
<li id="id6028"><p>Ivan Domenzain, Feiran Li, Eduard J. Kerkhoven, and Verena Siewers. Evaluating accessibility, usability and interoperability of genome-scale metabolic models for diverse yeasts species. <em>FEMS YEAST RESEARCH</em>, FEB 2021. <a class="reference external" href="https://doi.org/10.1093/femsyr/foab002">[DOI]</a>.</p></li>
<li id="id6029"><p>Krzysztof Ciomek and Milosz Kadzinski. Polyrun: a java library for sampling from the bounded convex polytopes. <em>SOFTWAREX</em>, JAN 2021. <a class="reference external" href="https://doi.org/10.1016/j.softx.2021.100659">[DOI]</a>.</p></li>
<li id="id6030"><p>Sebastien Moretti, Van Du T. Tran, Florence Mehl, Mark Ibberson, and Marco Pagni. Metanetx/mnxref: unified namespace for metabolites and biochemical reactions in the context of metabolic models. <em>NUCLEIC ACIDS RESEARCH</em>, 49(D1):D570–D574, JAN 8 2021. <a class="reference external" href="https://doi.org/10.1093/nar/gkaa992">[DOI]</a>.</p></li>
<li id="id6031"><p>Abhijit Paul, Rajat Anand, Sonali Porey Karmakar, Surender Rawat, Nandadulal Bairagi, and Samrat Chatterjee. Exploring gene knockout strategies to identify potential drug targets using genome-scale metabolic models. <em>SCIENTIFIC REPORTS</em>, JAN 8 2021. <a class="reference external" href="https://doi.org/10.1038/s41598-020-80561-1">[DOI]</a>.</p></li>
<li id="id6032"><p>Kristopher D. Rawls, Bonnie Dougherty, V, Kalyan C. Vinnakota, Venkat R. Pannala, Anders Wallqvist, Glynis L. Kolling, and Jason A. Papin. Predicting changes in renal metabolism after compound exposure with a genome-scale metabolic model. <em>TOXICOLOGY AND APPLIED PHARMACOLOGY</em>, FEB 1 2021. <a class="reference external" href="https://doi.org/10.1016/j.taap.2020.115390">[DOI]</a>.</p></li>
<li id="id6033"><p>Nhung Pham, Maarten Reijnders, Maria Suarez-Diez, Bart Nijsse, Jan Springer, Gerrit Eggink, and Peter J. Schaap. Genome-scale metabolic modeling underscores the potential of &lt;i&gt;cutaneotrichosporon oleaginosus&lt;/i&gt; atcc 20509 as a cell factory for biofuel production. <em>BIOTECHNOLOGY FOR BIOFUELS</em>, JAN 6 2021. <a class="reference external" href="https://doi.org/10.1186/s13068-020-01838-1">[DOI]</a>.</p></li>
<li id="id6034"><p>Maria A. Woerheide, Jan Krumsiek, Gabi Kastenmueller, and Matthias Arnold. Multi-omics integration in biomedical research - a metabolomics-centric review. <em>ANALYTICA CHIMICA ACTA</em>, 1141:144–162, JAN 2 2021. <a class="reference external" href="https://doi.org/10.1016/j.aca.2020.10.038">[DOI]</a>.</p></li>
<li id="id6035"><p>Priyanka Baloni, Wikum Dinalankara, John C. Earls, Theo A. Knijnenburg, Donald Geman, Luigi Marchionni, and Nathan D. Price. Identifying personalized metabolic signatures in breast cancer. <em>METABOLITES</em>, JAN 2021. <a class="reference external" href="https://doi.org/10.3390/metabo11010020">[DOI]</a>.</p></li>
<li id="id6036"><p>Sandrien Desmet, Marlies Brouckaert, Wout Boerjan, and Kris Morreel. Seeing the forest for the trees: retrieving plant secondary biochemical pathways from metabolome networks. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 19:72–85, 2021. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2020.11.050">[DOI]</a>.</p></li>
<li id="id6037"><p>Ilaria Granata, Mario Manzo, Ari Kusumastuti, and Mario R. Guarracino. Learning from metabolic networks: current trends and future directions for precision medicine. <em>CURRENT MEDICINAL CHEMISTRY</em>, 28(32):6619–6653, 2021. <a class="reference external" href="https://doi.org/10.2174/0929867328666201217103148">[DOI]</a>.</p></li>
<li id="id6038"><p>Zachary B. Haiman, Daniel C. Zielinski, Yuko Koike, James T. Yurkovich, and Bernhard O. Palsson. Masspy: building, simulating, and visualizing dynamic biological models in python using mass action kinetics. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JAN 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1008208">[DOI]</a>.</p></li>
<li id="id6039"><p>Johannes Hertel, Almut Heinken, Filippo Martinelli, and Ines Thiele. Integration of constraint-based modeling with fecal metabolomics reveals large deleterious effects of &lt;i&gt;fusobacterium&lt;/i&gt; spp. on community butyrate production. <em>GUT MICROBES</em>, JAN 1 2021. <a class="reference external" href="https://doi.org/10.1080/19490976.2021.1915673">[DOI]</a>.</p></li>
<li id="id6040"><p>F. Hosseinkhani, A. Heinken, I Thiele, P. W. Lindenburg, A. C. Harms, and T. Hankemeier. The contribution of gut bacterial metabolites in the human immune signaling pathway of non-communicable diseases. <em>GUT MICROBES</em>, 13(1):1–22, JAN 1 2021. <a class="reference external" href="https://doi.org/10.1080/19490976.2021.1882927">[DOI]</a>.</p></li>
<li id="id6041"><p>Zachary J. Johnson, Dennis D. Krutkin, Pavlo Bohutskyi, and Marina G. Kalyuzhnaya. Metals and methylotrophy: via global gene expression studies. In JA Cotruvo, editor, <em>RARE-EARTH ELEMENT BIOCHEMISTRY: METHANOL DEHYDROGENASES AND LANTHANIDE BIOLOGY</em>, volume 650 of Methods in Enzymology, pages 185–213. 2021. <a class="reference external" href="https://doi.org/10.1016/bs.mie.2021.01.046">[DOI]</a>.</p></li>
<li id="id6042"><p>Santosh Lamichhane, Partho Sen, Marina Amaral Alves, Henrique C. Ribeiro, Peppi Raunioniemi, Tuulia Hyotylainen, and Matej Oresic. Linking gut microbiome and lipid metabolism: moving beyond associations. <em>METABOLITES</em>, JAN 2021. <a class="reference external" href="https://doi.org/10.3390/metabo11010055">[DOI]</a>.</p></li>
<li id="id6043"><p>Steven Le Feunteun, Ahmed Al-Razaz, Matthijs Dekker, Erwin George, Beatrice Laroche, and George van Aken. Physiologically based modeling of food digestion and intestinal microbiota: state of the art and future challenges. an infogest review. In M Doyle and DJ McClements, editors, <em>ANNUAL REVIEW OF FOOD SCIENCE AND TECHNOLOGY, VOL 12, 2021</em>, volume 12 of Annual Review of Food Science and Technology, pages 149–167. 2021. <a class="reference external" href="https://doi.org/10.1146/annurev-food-070620-124140">[DOI]</a>.</p></li>
<li id="id6044"><p>Juan P. Molina Ortiz, Dale D. McClure, Erin R. Shanahan, Fariba Dehghani, Andrew J. Holmes, and Mark N. Read. Enabling rational gut microbiome manipulations by understanding gut ecology through experimentally-evidenced in silico models. <em>GUT MICROBES</em>, JAN 1 2021. <a class="reference external" href="https://doi.org/10.1080/19490976.2021.1965698">[DOI]</a>.</p></li>
<li id="id6045"><p>Claudio Tomi-Andrino, Rupert Norman, Thomas Millat, Philippe Soucaille, Klaus Winzer, David A. Barrett, John King, and Dong-Hyun Kim. Physicochemical and metabolic constraints for thermodynamics-based stoichiometric modelling under mesophilic growth conditions. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JAN 2021. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1007694">[DOI]</a>.</p></li>
<li id="id6046"><p>Andrew Walakira, Damjana Rozman, Tadeja Rezen, Miha Mraz, and Miha Moskon. Guided extraction of genome-scale metabolic models for the integration and analysis of omics data. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 19:3521–3530, 2021. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2021.06.0092001-0370/">[DOI]</a>.</p></li>
<li id="id6047"><p>St. Elmo Wilken, Jonathan M. Monk, Patrick A. Leggieri, Christopher E. Lawson, Thomas S. Lankiewicz, Susanna Seppala, Chris G. Daum, Jerry Jenkins, Anna M. Lipzen, Stephen J. Mondo, Kerrie W. Barry, Igor V. Grigoriev, John K. Henske, Michael K. Theodorou, Bernhard O. Palsson, Linda R. Petzold, and Michelle A. O'Malley. Experimentally validated reconstruction and analysis of a genome-scale metabolic model of an anaerobic neocallimastigomycota fungus. <em>MSYSTEMS</em>, JAN-FEB 2021. <a class="reference external" href="https://doi.org/10.1128/mSystems.00002-21">[DOI]</a>.</p></li>
<li id="id6062"><p>Jyotshana Gautam and Zhaohui Xu. Construction and validation of a genome-scale metabolic network of &lt;i&gt;thermotoga&lt;/i&gt; sp. strain rq7. <em>APPLIED BIOCHEMISTRY AND BIOTECHNOLOGY</em>, 193(3):896–911, MAR 2021. <a class="reference external" href="https://doi.org/10.1007/s12010-020-03470-z">[DOI]</a>.</p></li>
<li id="id6063"><p>Piao Zou, Yunze Zhang, Jean Bosco Nshimiyimana, Qianwen Cao, Yang Yang, Hui Geng, and Li Xiong. Reconstruction of a context-specific model based on genome-scale metabolic simulation for identification of prochloraz resistance mechanisms in &lt;i&gt;penicillium digitatum&lt;/i&gt;. <em>MICROBIAL DRUG RESISTANCE</em>, 27(6):776–785, JUN 1 2021. <a class="reference external" href="https://doi.org/10.1089/mdr.2020.0018">[DOI]</a>.</p></li>
<li id="id6064"><p>Jose Manuel Borrero-de Acuna, Izabook Gutierrez-Urrutia, Cristian Hidalgo-Dumont, Carla Aravena-Carrasco, Matias Orellana-Saez, Nestor Palominos-Gonzalez, Jozef B. J. H. van Duuren, Viktoria Wagner, Lars Glaser, Judith Becker, Michael Kohlstedt, Flavia C. Zacconi, Christoph Wittmann, and Ignacio Poblete-Castro. Channelling carbon flux through the &lt;i&gt;meta&lt;/i&gt;-cleavage route for improved poly(3-hydroxyalkanoate) production from benzoate and lignin-based aromatics in &lt;i&gt;pseudomonas putida&lt;/i&gt; h. <em>MICROBIAL BIOTECHNOLOGY</em>, 14(6):2385–2402, NOV 2021. <a class="reference external" href="https://doi.org/10.1111/1751-7915.13705">[DOI]</a>.</p></li>
<li id="id6070"><p>Hamideh Fouladiha, Sayed-Amir Marashi, Shangzhong Li, Zerong Li, Helen O. Masson, Behrouz Vaziri, and Nathan E. Lewis. Systematically gap-filling the genome-scale metabolic model of cho cells. <em>BIOTECHNOLOGY LETTERS</em>, 43(1):73–87, JAN 2021. <a class="reference external" href="https://doi.org/10.1007/s10529-020-03021-w">[DOI]</a>.</p></li>
<li id="id6079"><p>Emrah Ozcan, Merve Seven, Burcu Sirin, Tunahan Cakir, Emrah Nikerel, Bas Teusink, and Ebru Toksoy Oner. Dynamic co-culture metabolic models reveal the fermentation dynamics, metabolic capacities and interplays of cheese starter cultures. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 118(1):223–237, JAN 2021. <a class="reference external" href="https://doi.org/10.1002/bit.27565">[DOI]</a>.</p></li>
<li id="id6123"><p>Jagir R. Hussan and Peter J. Hunter. Our natural “makeup” reveals more than it hides: modeling the skin and its microbiome. <em>WIRES MECHANISMS OF DISEASE</em>, JAN 2021. <a class="reference external" href="https://doi.org/10.1002/wsbm.1497">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-4" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-4" name="0-4" role="tabpanel" tabindex="0"><div class="docutils container" id="id7281">
<ol class="arabic simple" start="1">
<li id="id7868"><p>Arnaud Belcour, Clemence Frioux, Meziane Aite, Anthony Bretaudeau, Falk Hildebrand, and Anne Siegel. Metage2metabo, microbiota-scale metabolic complementarity for the identification of key species. <em>ELIFE</em>, DEC 29 2020. <a class="reference external" href="https://doi.org/10.7554/eLife.61968">[DOI]</a>.</p></li>
<li id="id7869"><p>Christian Schulz and Eivind Almaas. Genome-scale reconstructions to assess metabolic phylogeny and organism clustering. <em>PLOS ONE</em>, DEC 29 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0240953">[DOI]</a>.</p></li>
<li id="id7870"><p>Renliang Sun, Yizhou Xu, Hang Zhang, Qiangzhen Yang, Ke Wang, Yongyong Shi, and Zhuo Wang. Mechanistic modeling of gene regulation and metabolism identifies potential targets for hepatocellular carcinoma. <em>FRONTIERS IN GENETICS</em>, DEC 23 2020. <a class="reference external" href="https://doi.org/10.3389/fgene.2020.595242">[DOI]</a>.</p></li>
<li id="id7871"><p>Supreeta Vijayakumar, Pattanathu K. S. M. Rahman, and Claudio Angione. A hybrid flux balance analysis and machine learning pipeline elucidates metabolic adaptation in cyanobacteria. <em>ISCIENCE</em>, DEC 18 2020. <a class="reference external" href="https://doi.org/10.1016/j.isci.2020.101818">[DOI]</a>.</p></li>
<li id="id7872"><p>Joana P. Bernardes, Neha Mishra, Florian Tran, Thomas Bahmer, Lena Best, Johanna Blase, I, Dora Bordoni, Jeanette Franzenburg, Ulf Geisen, Jonathan Josephs-Spaulding, Philipp Koehler, Axel Kuenstner, Elisa Rosati, Anna C. Aschenbrenner, Petra Bacher, Nathan Baran, Teide Boysen, Burkhard Brandt, Niklas Bruse, Jonathan Doerr, Andreas Draeger, Gunnar Elke, David Ellinghaus, Julia Fischer, Michael Forster, Andre Franke, Soeren Franzenburg, Norbert Frey, Anette Friedrichs, Janina Fuss, Andreas Glueck, Jacob Hamm, Finn Hinrichsen, Marc P. Hoeppner, Simon Imm, Ralf Junker, Sina Kaiser, Ying H. Kan, Rainer Knoll, Christoph Lange, Georg Laue, Clemens Lier, Matthias Lindner, Georgios Marinos, Robert Markewitz, Jacob Nattermann, Rainer Noth, Peter Pickkers, Klaus F. Rabe, Alina Renz, Christoph Roecken, Jan Rupp, Annika Schaffarzyk, Alexander Scheffold, Jonas Schulte-Schrepping, Domagoj Schunk, Dirk Skowasch, Thomas Ulas, Klaus-Peter Wandinger, Michael Wittig, Johannes Zimmermann, Hauke Busch, Bimba F. Hoyer, Christoph Kaleta, Jan Heyckendorf, Matthijs Kox, Jan Rybniker, Stefan Schreiber, Joachim L. Schultze, Philip Rosenstiel, HCA Lung Biol Network, and Deutsch COVID-19 Omics Initiative. Longitudinal multi-omics analyses identify responses of megakaryocytes, erythroid cells, and plasmablasts as hallmarks of severe covid-19. <em>IMMUNITY</em>, 53(6):1296+, DEC 15 2020. <a class="reference external" href="https://doi.org/10.1016/j.immuni.2020.11.017">[DOI]</a>.</p></li>
<li id="id7873"><p>Ushashi Banerjee, Santhosh Sankar, Amit Singh, and Nagasuma Chandra. A multi-pronged computational pipeline for prioritizing drug target strategies for latent tuberculosis. <em>FRONTIERS IN CHEMISTRY</em>, DEC 14 2020. <a class="reference external" href="https://doi.org/10.3389/fchem.2020.593497">[DOI]</a>.</p></li>
<li id="id7874"><p>Christopher A. Voigt. Synthetic biology 2020-2030: six commercially-available products that are changing our world. <em>NATURE COMMUNICATIONS</em>, DEC 11 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-20122-2">[DOI]</a>.</p></li>
<li id="id7875"><p>Hanne H. Henriksen, Sarah McGarrity, Rosa S. Siguroardottir, Travis Nemkov, Angelo D'Alessandro, Bernhard O. Palsson, Jakob Stensballe, Charles E. Wade, Ottar Rolfsson, and Par I. Johansson. Metabolic systems analysis of shock-induced endotheliopathy (shine) in trauma a new research paradigm. <em>ANNALS OF SURGERY</em>, 272(6):1140–1148, DEC 2020. <a class="reference external" href="https://doi.org/10.1097/SLA.0000000000003307">[DOI]</a>.</p></li>
<li id="id7876"><p>Gian Marco Messa, Francesco Napolitano, Sarah H. Elsea, Diego di Bernardo, and Xin Gao. A siamese neural network model for the prioritization of metabolic disorders by integrating real and simulated data. <em>BIOINFORMATICS</em>, 36(2):I787–I794, DEC 2020. 19th European Conference on Computational Biology (ECCB), ELECTR NETWORK, SEP 07-08, 2020. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btaa841">[DOI]</a>.</p></li>
<li id="id7877"><p>S. Ranjbar, M. Shahmansouri, P. Attri, and A. Bogaerts. Effect of plasma-induced oxidative stress on the glycolysis pathway of &lt;i&gt;escherichia coli&lt;/i&gt;. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, DEC 2020. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2020.104064">[DOI]</a>.</p></li>
<li id="id7878"><p>Kent M. Rapp, Jackson P. Jenkins, and Michael J. Betenbaugh. Partners for life: building microbial consortia for the future. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 66:292–300, DEC 2020. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2020.10.001">[DOI]</a>.</p></li>
<li id="id7879"><p>Daniel Craig Zielinski, Arjun Patel, and Bernhard O. Palsson. The expanding computational toolbox for engineering microbial phenotypes at the genome scale. <em>MICROORGANISMS</em>, DEC 2020. <a class="reference external" href="https://doi.org/10.3390/microorganisms8122050">[DOI]</a>.</p></li>
<li id="id7880"><p>Eileen Marie Hanna, Xiaokang Zhang, Marta Eide, Shirin Fallahi, Tomasz Furmanek, Fekadu Yadetie, Daniel Craig Zielinski, Anders Goksoyr, and Inge Jonassen. Recodliver0.9: overcoming challenges in genome-scale metabolic reconstruction of a non-model species. <em>FRONTIERS IN MOLECULAR BIOSCIENCES</em>, NOV 26 2020. <a class="reference external" href="https://doi.org/10.3389/fmolb.2020.591406">[DOI]</a>.</p></li>
<li id="id7881"><p>Priyanka Baloni, Cory C. Funk, Jingwen Yan, James T. Yurkovich, Alexandra Kueider-Paisley, Kwangsik Nho, Almut Heinken, Wei Jia, Siamak Mahmoudiandehkordi, Gregory Louie, Andrew J. Saykin, Matthias Arnold, Gabi Kastenmueller, William J. Griffiths, Ines Thiele, Rima Kaddurah-Daouk, Nathan D. Price, and Alzheimers Dis Metabolomics Consor. Metabolic network analysis reveals altered bile acid synthesis and metabolism in alzheimer's disease. <em>CELL REPORTS MEDICINE</em>, NOV 17 2020. <a class="reference external" href="https://doi.org/10.1016/j.xcrm.2020.100138">[DOI]</a>.</p></li>
<li id="id7885"><p>Arianna Basile, Stefano Campanaro, Adam Kovalovszki, Guido Zampieri, Alessandro Rossi, Irini Angelidaki, Giorgio Valle, and Laura Treu. Revealing metabolic mechanisms of interaction in the anaerobic digestion microbiome by flux balance analysis. <em>METABOLIC ENGINEERING</em>, 62:138–149, NOV 2020. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2020.08.013">[DOI]</a>.</p></li>
<li id="id7886"><p>Jinxin Zhao, Yan Zhu, Jiru Han, Yu-Wei Lin, Michael Aichem, Jiping Wang, Ke Chen, Tony Velkov, Falk Schreiber, and Jian Li. Genome-scale metabolic modeling reveals metabolic alterations of multidrug-resistant &lt;i&gt;acinetobacter baumannii&lt;/i&gt; in a murine bloodstream infection model. <em>MICROORGANISMS</em>, NOV 2020. <a class="reference external" href="https://doi.org/10.3390/microorganisms8111793">[DOI]</a>.</p></li>
<li id="id7887"><p>Philipp Keller, Elad Noor, Fabian Meyer, Michael A. Reiter, Stanislav Anastassov, Patrick Kiefer, and Julia A. Vorholt. Methanol-dependent &lt;i&gt;escherichia coli&lt;/i&gt; strains with a complete ribulose monophosphate cycle. <em>NATURE COMMUNICATIONS</em>, OCT 26 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-19235-5">[DOI]</a>.</p></li>
<li id="id7888"><p>Deepanwita Banerjee, Thomas Eng, Andrew K. Lau, Yusuke Sasaki, Brenda Wang, Yan Chen, Jan-Philip Prahl, Vasanth R. Singan, Robin A. Herbert, Yuzhong Liu, Deepti Tanjore, Christopher J. Petzold, Jay D. Keasling, and Aindrila Mukhopadhyay. Genome-scale metabolic rewiring improves titers rates and yields of the non-native product indigoidine at scale. <em>NATURE COMMUNICATIONS</em>, OCT 23 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-19171-4">[DOI]</a>.</p></li>
<li id="id7889"><p>Nikolaos Tsiantis and Julio R. Banga. Using optimal control to understand complex metabolic pathways. <em>BMC BIOINFORMATICS</em>, OCT 21 2020. <a class="reference external" href="https://doi.org/10.1186/s12859-020-03808-8">[DOI]</a>.</p></li>
<li id="id7891"><p>Amin Espah Borujeni, Jing Zhang, Hamid Doosthosseini, Alec A. K. Nielsen, and Christopher A. Voigt. Genetic circuit characterization by inferring rna polymerase movement and ribosome usage. <em>NATURE COMMUNICATIONS</em>, OCT 5 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-18630-2">[DOI]</a>.</p></li>
<li id="id7892"><p>Fernando Cruz, Jose P. Faria, Miguel Rocha, Isabel Rocha, and Oscar Dias. A review of methods for the reconstruction and analysis of integrated genome-scale models of metabolism and regulation. <em>BIOCHEMICAL SOCIETY TRANSACTIONS</em>, 48(5):1889–1903, OCT 2020. <a class="reference external" href="https://doi.org/10.1042/BST20190840">[DOI]</a>.</p></li>
<li id="id7893"><p>S. T. R. Moolamalla and P. K. Vinod. Genome-scale metabolic modelling predicts biomarkers and therapeutic targets for neuropsychiatric disorders. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, OCT 2020. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2020.103994">[DOI]</a>.</p></li>
<li id="id7894"><p>Lutfu Safak Yilmaz, Xuhang Li, Shivani Nanda, Bennett Fox, Frank Schroeder, and Albertha J. M. Walhout. Modeling tissue-relevant &lt;i&gt;caenorhabditis elegans&lt;/i&gt; metabolism at network, pathway, reaction, and metabolite levels. <em>MOLECULAR SYSTEMS BIOLOGY</em>, OCT 2020. <a class="reference external" href="https://doi.org/10.15252/msb.20209649">[DOI]</a>.</p></li>
<li id="id7895"><p>Ailin Liu, Yee-Shan Ku, Carolina A. Contador, and Hon-Ming Lam. The impacts of domestication and agricultural practices on legume nutrient acquisition through symbiosis with rhizobia and arbuscular mycorrhizal fungi. <em>FRONTIERS IN GENETICS</em>, SEP 30 2020. <a class="reference external" href="https://doi.org/10.3389/fgene.2020.583954">[DOI]</a>.</p></li>
<li id="id7896"><p>Javiera Lopez, Diego Bustos, Conrado Camilo, Natalia Arenas, Pedro A. Saa, and Eduardo Agosin. Engineering &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; for the overproduction of β-ionone and its precursor β-carotene. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, SEP 30 2020. <a class="reference external" href="https://doi.org/10.3389/fbioe.2020.578793">[DOI]</a>.</p></li>
<li id="id7897"><p>Marouen Ben Guebila. Vffva: dynamic load balancing enables large-scale flux variability analysis. <em>BMC BIOINFORMATICS</em>, SEP 29 2020. <a class="reference external" href="https://doi.org/10.1186/s12859-020-03711-2">[DOI]</a>.</p></li>
<li id="id7898"><p>Fabian Moreno-Avitia, Jose Utrilla, Francisco Bolivar, Juan Nogales, and Adelfo Escalante. Metabolic reconstruction of&lt;i&gt;pseudomonas chlororaphis&lt;/i&gt;atcc 9446 to understand its metabolic potential as a phenazine-1-carboxamide-producing strain. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 104(23):10119–10132, DEC 2020. <a class="reference external" href="https://doi.org/10.1007/s00253-020-10913-4">[DOI]</a>.</p></li>
<li id="id7900"><p>Bin Wang, Jiarui Yang, Chenchen Gao, Tong Hao, Jingjing Li, and Jinsheng Sun. Reconstruction of&lt;i&gt;eriocheir sinensis&lt;/i&gt;y-organ genome-scale metabolic network and differential analysis after eyestalk ablation. <em>FRONTIERS IN GENETICS</em>, SEP 25 2020. <a class="reference external" href="https://doi.org/10.3389/fgene.2020.532492">[DOI]</a>.</p></li>
<li id="id7901"><p>Jie Zhang, Soren D. Petersen, Tijana Radivojevic, Andres Ramirez, Andres Perez-Manriquez, Eduardo Abeliuk, Benjamin J. Sanchez, Zak Costello, Yu Chen, Michael J. Fero, Hector Garcia Martin, Jens Nielsen, Jay D. Keasling, and Michael K. Jensen. Combining mechanistic and machine learning models for predictive engineering and optimization of tryptophan metabolism. <em>NATURE COMMUNICATIONS</em>, SEP 25 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-17910-1">[DOI]</a>.</p></li>
<li id="id7902"><p>Xin Fang, Colton J. Lloyd, and Bernhard O. Palsson. Reconstructing organisms in silico: genome-scale models and their emerging applications. <em>NATURE REVIEWS MICROBIOLOGY</em>, 18(12):731–743, DEC 2020. <a class="reference external" href="https://doi.org/10.1038/s41579-020-00440-4">[DOI]</a>.</p></li>
<li id="id7903"><p>Kyle Stone, Matthew Hilliard, Kiumars Badr, Alisabeth Bradford, Q. Peter He, and Jin Wang. Comparative study of oxygen-limited and methane-limited growth phenotypes of &lt;i&gt;methylomicrobium buryatense&lt;/i&gt; 5gb1. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, SEP 15 2020. <a class="reference external" href="https://doi.org/10.1016/j.bej.2020.107707">[DOI]</a>.</p></li>
<li id="id7904"><p>Anukriti Verma, Shivani Sharda, Bhawna Rathi, Pallavi Somvanshi, and Bimlesh Dhar Pandey. Elucidating potential molecular signatures through host-microbe interactions for reactive arthritis and inflammatory bowel disease using combinatorial approach. <em>SCIENTIFIC REPORTS</em>, SEP 15 2020. <a class="reference external" href="https://doi.org/10.1038/s41598-020-71674-8">[DOI]</a>.</p></li>
<li id="id7905"><p>Tobias Ploch, Eric von Lieres, Wolfgang Wiechert, Alexander Mitsos, and Ralf Hannemann-Tamas. Simulation of differential-algebraic equation systems with optimization criteria embedded in modelica. <em>COMPUTERS &amp; CHEMICAL ENGINEERING</em>, SEP 2 2020. <a class="reference external" href="https://doi.org/10.1016/j.compchemeng.2020.106920">[DOI]</a>.</p></li>
<li id="id7906"><p>Alyaa M. Abdel-Haleem, Vaishnavi Ravikumar, Boyang Ji, Katsuhiko Mineta, Xin Gao, Jens Nielsen, Takashi Gojobori, and Ivan Mijakovic. Integrated metabolic modeling, culturing, and transcriptomics explain enhanced virulence of &lt;i&gt;vibrio cholerae&lt;/i&gt; during coinfection with enterotoxigenic &lt;i&gt;escherichia coli&lt;/i&gt;. <em>MSYSTEMS</em>, SEP-OCT 2020. <a class="reference external" href="https://doi.org/10.1128/mSystems.00491-20">[DOI]</a>.</p></li>
<li id="id7907"><p>Sanjeev Dahal, James T. Yurkovich, Hao Xu, Bernhard O. Palsson, and Laurence Yang. Synthesizing systems biology knowledge from omics using genome-scale models. <em>PROTEOMICS</em>, SEP 2020. <a class="reference external" href="https://doi.org/10.1002/pmic.201900282">[DOI]</a>.</p></li>
<li id="id7908"><p>Cristina Herencias, Sergio Salgado-Briegas, Maria A. Prieto, and Juan Nogales. Providing new insights on the biphasic lifestyle of the predatory bacterium &lt;i&gt;bdellovibrio bacteriovorus&lt;/i&gt; through genome-scale metabolic modeling. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1007646">[DOI]</a>.</p></li>
<li id="id7909"><p>Juan F. Poyatos. Genetic buffering and potentiation in metabolism. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1008185">[DOI]</a>.</p></li>
<li id="id7910"><p>William T. Scott, Jr., Eddy J. Smid, Richard A. Notebaart, and David E. Block. Curation and analysis of a &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; genome-scale metabolic model for predicting production of sensory impact molecules under enological conditions. <em>PROCESSES</em>, SEP 2020. <a class="reference external" href="https://doi.org/10.3390/pr8091195">[DOI]</a>.</p></li>
<li id="id7911"><p>Romeu Viana, Oscar Dias, Davide Lagoa, Monica Galocha, Isabel Rocha, and Miguel Cacho Teixeira. Genome-scale metabolic model of the human pathogen&lt;i&gt;candida albicans&lt;/i&gt;: a promising platform for drug target prediction. <em>JOURNAL OF FUNGI</em>, SEP 2020. <a class="reference external" href="https://doi.org/10.3390/jof6030171">[DOI]</a>.</p></li>
<li id="id7912"><p>Maryam Khoshnejat, Kaveh Kavousi, Ali Mohammad Banaei-Moghaddam, and Ali Akbar Moosavi-Movahedi. Unraveling the molecular heterogeneity in type 2 diabetes: a potential subtype discovery followed by metabolic modeling. <em>BMC MEDICAL GENOMICS</em>, AUG 24 2020. <a class="reference external" href="https://doi.org/10.1186/s12920-020-00767-0">[DOI]</a>.</p></li>
<li id="id7913"><p>Zhuangrong Huang, Jianlin Xu, Andrew Yongky, Caitlin S. Morris, Ashli L. Polanco, Michael Reily, Michael C. Borys, Zheng Jian Li, and Seongkyu Yoon. Cho cell productivity improvement by genome-scale modeling and pathway analysis: application to feed supplements. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, AUG 15 2020. <a class="reference external" href="https://doi.org/10.1016/j.bej.2020.107638">[DOI]</a>.</p></li>
<li id="id7914"><p>Georgios Marinos, Christoph Kaleta, and Silvio Waschina. Defining the nutritional input for genome-scale metabolic models: a roadmap. <em>PLOS ONE</em>, AUG 14 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0236890">[DOI]</a>.</p></li>
<li id="id7915"><p>David M. Curran, Alexandra Grote, Nirvana Nursimulu, Adam Geber, Dennis Voronin, Drew R. Jones, Elodie Ghedin, and John Parkinson. Modeling the metabolic interplay between a parasitic worm and its bacterial endosymbiont allows the identification of novel drug targets. <em>ELIFE</em>, AUG 11 2020. <a class="reference external" href="https://doi.org/10.7554/eLife.51850">[DOI]</a>.</p></li>
<li id="id7916"><p>Christopher Culley, Supreeta Vijayakumar, Guido Zampieri, and Claudio Angione. A mechanism-aware and multiomic machine-learning pipeline characterizes yeast cell growth. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 117(31):18869–18879, AUG 4 2020. <a class="reference external" href="https://doi.org/10.1073/pnas.2002959117">[DOI]</a>.</p></li>
<li id="id7917"><p>Cristal Zuniga, Beth Peacock, Bo Liang, Greg McCollum, Sonia C. Irigoyen, Diego Tec-Campos, Clarisse Marotz, Nien-Chen Weng, Alejandro Zepeda, Georgios Vidalakis, Kranthi K. Mandadi, James Borneman, and Karsten Zengler. Linking metabolic phenotypes to pathogenic traits among “&lt;i&gt;candidatus&lt;/i&gt;liberibacter asiaticus” and its hosts. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, AUG 4 2020. <a class="reference external" href="https://doi.org/10.1038/s41540-020-00142-w">[DOI]</a>.</p></li>
<li id="id7918"><p>Maureen A. Carey, Andreas Draeger, Moritz E. Beber, Jason A. Papin, and James T. Yurkovich. Community standards to facilitate development and address challenges in metabolic modeling. <em>MOLECULAR SYSTEMS BIOLOGY</em>, AUG 2020. <a class="reference external" href="https://doi.org/10.15252/msb.20199235">[DOI]</a>.</p></li>
<li id="id7919"><p>John R. Casey and Michael J. Follows. A steady-state model of microbial acclimation to substrate limitation. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1008140">[DOI]</a>.</p></li>
<li id="id7920"><p>Yan Chen, Deepanwita Banerjee, Aindrila Mukhopadhyay, and Christopher J. Petzold. Systems and synthetic biology tools for advanced bioproduction hosts. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 64(SI):101–109, AUG 2020. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2019.12.007">[DOI]</a>.</p></li>
<li id="id7921"><p>Yoon-Mi Choi, Yi Qing Lee, Hyun-Seob Song, and Dong-Yup Lee. Genome scale metabolic models and analysis for evaluating probiotic potentials. <em>BIOCHEMICAL SOCIETY TRANSACTIONS</em>, 48(4):1309–1321, AUG 2020. <a class="reference external" href="https://doi.org/10.1042/BST20190668">[DOI]</a>.</p></li>
<li id="id7922"><p>Bianca M. Esch, Sergej Limar, Andre Bogdanowski, Christos Gournas, Tushar More, Celine Sundag, Stefan Walter, Juergen J. Heinisch, Christer S. Ejsing, Bruno Andre, and Florian Froehlich. Uptake of exogenous serine is important to maintain sphingolipid homeostasis in &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;. <em>PLOS GENETICS</em>, AUG 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pgen.1008745">[DOI]</a>.</p></li>
<li id="id7923"><p>Sarah M. Keating, Dagmar Waltemath, Matthias Koenig, Fengkai Zhang, Andreas Draeger, Claudine Chaouiya, Frank T. Bergmann, Andrew Finney, Colin S. Gillespie, Tomas Helikar, Stefan Hoops, Rahuman S. Malik-Sheriff, Stuart L. Moodie, Ion I. Moraru, Chris J. Myers, Aurelien Naldi, Brett G. Olivier, Sven Sahle, James C. Schaff, Lucian P. Smith, Maciej J. Swat, Denis Thieffry, Leandro Watanabe, Darren J. Wilkinson, Michael L. Blinov, Kimberly Begley, James R. Faeder, Harold F. Gomez, Thomas M. Hamm, Yuichiro Inagaki, Wolfram Liebermeister, Allyson L. Lister, Daniel Lucio, Eric Mjolsness, Carole J. Proctor, Karthik Raman, Nicolas Rodriguez, Clifford A. Shaffer, Bruce E. Shapiro, Joerg Stelling, Neil Swainston, Naoki Tanimura, John Wagner, Martin Meier-Schellersheim, Herbert M. Sauro, Bernhard Palsson, Hamid Bolouri, Hiroaki Kitano, Akira Funahashi, Henning Hermjakob, John C. Doyle, Michael Hucka, and SBML Level 3 Community Members. Sbml level 3: an extensible format for the exchange and reuse of biological models. <em>MOLECULAR SYSTEMS BIOLOGY</em>, AUG 2020. <a class="reference external" href="https://doi.org/10.15252/msb.20199110">[DOI]</a>.</p></li>
<li id="id7924"><p>Narges Pedram, Hamid Rashedi, and Ehsan Motamedian. A systematic strategy using a reconstructed genome-scale metabolic network for pathogen &lt;i&gt;streptococcus pneumoniae&lt;/i&gt; d39 to find novel potential drug targets. <em>PATHOGENS AND DISEASE</em>, AUG 2020. <a class="reference external" href="https://doi.org/10.1093/femspd/ftaa051">[DOI]</a>.</p></li>
<li id="id7925"><p>Gayathri Sambamoorthy and Karthik Raman. Minreact: a systematic approach for identifying minimal metabolic networks. <em>BIOINFORMATICS</em>, 36(15):4309–4315, AUG 1 2020. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btaa497">[DOI]</a>.</p></li>
<li id="id7926"><p>Svetlana Volkova, Marta R. A. Matos, Matthias Mattanovich, and Igor Marin de Mas. Metabolic modelling as a framework for metabolomics data integration and analysis. <em>METABOLITES</em>, AUG 2020. <a class="reference external" href="https://doi.org/10.3390/metabo10080303">[DOI]</a>.</p></li>
<li id="id7927"><p>Cristal Zuniga, Tingting Li, Michael T. Guarnieri, Jackson P. Jenkins, Chien-Ting Li, Kerem Bingol, Young-Mo Kim, Michael J. Betenbaugh, and Karsten Zengler. Synthetic microbial communities of heterotrophs and phototrophs facilitate sustainable growth. <em>NATURE COMMUNICATIONS</em>, JUL 30 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-17612-8">[DOI]</a>.</p></li>
<li id="id7928"><p>Yu Chen, Yan Sun, Zhihao Liu, Fengqing Dong, Yuanyuan Li, and Yonghong Wang. Genome-scale modeling for&lt;i&gt;bacillus coagulans&lt;/i&gt;to understand the metabolic characteristics. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 117(11):3545–3558, NOV 2020. <a class="reference external" href="https://doi.org/10.1002/bit.27488">[DOI]</a>.</p></li>
<li id="id7929"><p>Sandra M. Correa, Saleh Alseekh, Lucia Atehortua, Yariv Brotman, Rigoberto Rios-Estepa, Alisdair R. Fernie, and Zoran Nikoloski. Model-assisted identification of metabolic engineering strategies for&lt;i&gt;jatropha curcas&lt;/i&gt;lipid pathways. <em>PLANT JOURNAL</em>, 104(1):76–95, SEP 2020. <a class="reference external" href="https://doi.org/10.1111/tpj.14906">[DOI]</a>.</p></li>
<li id="id7930"><p>Sergio Garcia and Cong T. Trinh. Harnessing natural modularity of metabolism with goal attainment optimization to design a modular chassis cell for production of diverse chemicals. <em>ACS SYNTHETIC BIOLOGY</em>, 9(7):1665–1681, JUL 17 2020. <a class="reference external" href="https://doi.org/10.1021/acssynbio.9b00518">[DOI]</a>.</p></li>
<li id="id7931"><p>Zhuangrong Huang and Seongkyu Yoon. Identifying metabolic features and engineering targets for productivity improvement in cho cells by integrated transcriptomics and genome-scale metabolic model. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, JUL 15 2020. <a class="reference external" href="https://doi.org/10.1016/j.bej.2020.107624">[DOI]</a>.</p></li>
<li id="id7932"><p>Carolina Shene, Paris Paredes, Liset Flores, Allison Leyton, Juan A. Asenjo, and Yusuf Chisti. Dynamic flux balance analysis of biomass and lipid production by antarctic thraustochytrid &lt;i&gt;oblongichytrium&lt;/i&gt; sp. rt2316-13. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 117(10):3006–3017, OCT 2020. <a class="reference external" href="https://doi.org/10.1002/bit.27463">[DOI]</a>.</p></li>
<li id="id7933"><p>Shaozhen Ding, Yu Tian, Pengli Cai, Dachuan Zhang, Xingxiang Cheng, Dandan Sun, Le Yuan, Junni Chen, Weizhong Tu, Dong-Qing Wei, and Qian-Nan Hu. Novopathfinder: a webserver of designing novel-pathway with integrating gem-model. <em>NUCLEIC ACIDS RESEARCH</em>, 48(W1):W477–W487, JUL 2 2020. <a class="reference external" href="https://doi.org/10.1093/nar/gkaa230">[DOI]</a>.</p></li>
<li id="id7934"><p>Frances Blow, Nana Y. D. Ankrah, Noah Clark, Imhoi Koo, Erik L. Allman, Qing Liu, Mallappa Anitha, Andrew D. Patterson, and Angela E. Douglas. Impact of facultative bacteria on the metabolic function of an obligate insect-bacterial symbiosis. <em>MBIO</em>, JUL-AUG 2020. <a class="reference external" href="https://doi.org/10.1128/mBio.00402-20">[DOI]</a>.</p></li>
<li id="id7935"><p>Shirin Fallahi, Hans J. Skaug, and Guttorm Alendal. A comparison of monte carlo sampling methods for metabolic network models. <em>PLOS ONE</em>, JUL 1 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0235393">[DOI]</a>.</p></li>
<li id="id7936"><p>David Hoksza, Piotr Gawron, Marek Ostaszewski, Jan Hasenauer, and Reinhard Schneider. Closing the gap between formats for storing layout information in systems biology. <em>BRIEFINGS IN BIOINFORMATICS</em>, 21(4):1249–1260, JUL 2020. <a class="reference external" href="https://doi.org/10.1093/bib/bbz067">[DOI]</a>.</p></li>
<li id="id7937"><p>Mikhail Kulyashov, Sergey E. Peltek, and Ilya R. Akberdin. A genome-scale metabolic model of 2,3-butanediol production by thermophilic bacteria&lt;i&gt;geobacillus icigianus&lt;/i&gt;. <em>MICROORGANISMS</em>, JUL 2020. <a class="reference external" href="https://doi.org/10.3390/microorganisms8071002">[DOI]</a>.</p></li>
<li id="id7938"><p>J. Kyle Medley, Joseph Hellerstein, and Herbert M. Sauro. Libsbmljs-enabling web-based sbml tools. <em>BIOSYSTEMS</em>, JUL 2020. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2020.104150">[DOI]</a>.</p></li>
<li id="id7939"><p>Hardy Rolletschek, Joerg Schwender, Christina Koenig, Kent D. Chapman, Trevor Romsdahl, Christin Lorenz, Hans-Peter Braun, Peter Denolf, Katrien Van Audenhove, Eberhard Munz, Nicolas Heinzel, Stefan Ortleb, Twan Rutten, Sean McCorkle, Taras Borysyuk, Andre Guendel, Hai Shi, Michiel Vander Auwermeulen, Stephane Bourot, and Ljudmilla Borisjuk. Cellular plasticity in response to suppression of storage proteins in the &lt;i&gt;brassica napus&lt;/i&gt; embryo. <em>PLANT CELL</em>, 32(7):2383–2401, JUL 2020. <a class="reference external" href="https://doi.org/10.1105/tpc.19.00879">[DOI]</a>.</p></li>
<li id="id7940"><p>Hock Chuan Yeo, Jongkwang Hong, Meiyappan Lakshmanan, and Dong-Yup Lee. Enzyme capacity-based genome scale modelling of cho cells. <em>METABOLIC ENGINEERING</em>, 60:138–147, JUL 2020. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2020.04.005">[DOI]</a>.</p></li>
<li id="id7941"><p>Francesca Antonaros, Veronica Ghini, Francesca Pulina, Giuseppe Ramacieri, Elena Cicchini, Elisa Mannini, Anna Martelli, Agnese Feliciello, Silvia Lanfranchi, Sara Onnivello, Renzo Vianello, Chiara Locatelli, Guido Cocchi, Maria Chiara Pelleri, Lorenza Vitale, Pierluigi Strippoli, Claudio Luchinat, Paola Turano, Allison Piovesan, and Maria Caracausi. Plasma metabolome and cognitive skills in down syndrome. <em>SCIENTIFIC REPORTS</em>, JUN 26 2020. <a class="reference external" href="https://doi.org/10.1038/s41598-020-67195-z">[DOI]</a>.</p></li>
<li id="id7942"><p>Huseyin Tas, Adam Amara, Miguel E. Cueva, Nadine Bongaerts, Alicia Calvo-Villamanan, Samir Hamadache, and Konstantinos Vavitsas. The synthetic microbiology caucus: are synthetic biology standards applicable in everyday research practice? <em>MICROBIAL BIOTECHNOLOGY</em>, 13(5):1304–1308, SEP 2020. <a class="reference external" href="https://doi.org/10.1111/1751-7915.13612">[DOI]</a>.</p></li>
<li id="id7944"><p>Denny Popp and Florian Centler. Μbialsim: constraint-based dynamic simulation of complex microbiomes. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, JUN 10 2020. <a class="reference external" href="https://doi.org/10.3389/fbioe.2020.00574">[DOI]</a>.</p></li>
<li id="id7945"><p>Federico Baldini, Johannes Hertel, Estelle Sandt, Cyrille C. Thinnes, Lorieza Neuberger-Castillo, Lukas Pavelka, Fay Betsou, Rejko Krueger, Ines Thiele, and NCER-PD Consortium. Parkinson's disease-associated alterations of the gut microbiome predict disease-relevant changes in metabolic functions. <em>BMC BIOLOGY</em>, JUN 9 2020. <a class="reference external" href="https://doi.org/10.1186/s12915-020-00775-7">[DOI]</a>.</p></li>
<li id="id7946"><p>Tamara Bintener, Maria Pires Pacheco, and Thomas Sauter. Towards the routine use of &lt;i&gt;in silico&lt;/i&gt; screenings for drug discovery using metabolic modelling. <em>BIOCHEMICAL SOCIETY TRANSACTIONS</em>, 48(3):955–969, JUN 2020. <a class="reference external" href="https://doi.org/10.1042/BST20190867">[DOI]</a>.</p></li>
<li id="id7947"><p>Pritam Chanda, Eduardo Costa, Jie Hu, Shravan Sukumar, John Van Hemert, and Rasna Walia. Information theory in computational biology: where we stand today. <em>ENTROPY</em>, JUN 2020. <a class="reference external" href="https://doi.org/10.3390/e22060627">[DOI]</a>.</p></li>
<li id="id7948"><p>Clemence Frioux, Simon M. Dittami, and Anne Siegel. Using automated reasoning to explore the metabolism of unconventional organisms: a first step to explore host-microbial interactions. <em>BIOCHEMICAL SOCIETY TRANSACTIONS</em>, 48(3):901–913, JUN 2020. <a class="reference external" href="https://doi.org/10.1042/BST20190667">[DOI]</a>.</p></li>
<li id="id7949"><p>Shouyong Jiang, Yong Wang, Marcus Kaiser, and Natalio Krasnogor. Nihba: a network interdiction approach for metabolic engineering design. <em>BIOINFORMATICS</em>, 36(11):3482–3492, JUN 2020. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btaa163">[DOI]</a>.</p></li>
<li id="id7950"><p>Victor A. Lopez-Agudelo, Tom A. Mendum, Emma Laing, HuiHai Wu, Andres Baena, Luis F. Barrera, Dany J. V. Beste, and Rigoberto Rios-Estepa. A systematic evaluation of &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; genome-scale metabolic networks. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUN 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1007533">[DOI]</a>.</p></li>
<li id="id7951"><p>Naoya Nochino, Yoshihiro Toya, and Hiroshi Shimizu. Transcription factor arca is a flux sensor for the oxygen consumption rate in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOTECHNOLOGY JOURNAL</em>, JUN 2020. <a class="reference external" href="https://doi.org/10.1002/biot.201900353">[DOI]</a>.</p></li>
<li id="id7952"><p>Eliana B. Souto, Elena Sanchez-Lopez, Joana R. Campos, Raquel da Ana, Marta Espina, Maria L. Garcia, Patricia Severino, Fernando Batain, Thais F. R. Alves, Kessi M. M. Crescencio, Selma B. Souto, and Marco V. Chaud. Retinal drug delivery: rethinking outcomes for the efficient replication of retinal behavior. <em>APPLIED SCIENCES-BASEL</em>, JUN 2020. <a class="reference external" href="https://doi.org/10.3390/app10124258">[DOI]</a>.</p></li>
<li id="id7953"><p>Rashi Verma, Dibyabhaba Pradhan, Mohsin Maseet, Harpreet Singh, Arun Kumar Jain, and Luqman Ahmad Khan. Genome-wide screening and &lt;i&gt;in silico&lt;/i&gt; gene knockout to predict potential candidates for drug designing against &lt;i&gt;candida albicans&lt;/i&gt;. <em>INFECTION GENETICS AND EVOLUTION</em>, JUN 2020. <a class="reference external" href="https://doi.org/10.1016/j.meegid.2020.104196">[DOI]</a>.</p></li>
<li id="id7954"><p>Inmaculada Garcia-Romero, Juan Nogales, Eduardo Diaz, Eduardo Santero, and Belen Floriano. Understanding the metabolism of the tetralin degrader &lt;i&gt;sphingopyxis granuli&lt;/i&gt; strain tfa through genome-scale metabolic modelling. <em>SCIENTIFIC REPORTS</em>, MAY 26 2020. <a class="reference external" href="https://doi.org/10.1038/s41598-020-65258-9">[DOI]</a>.</p></li>
<li id="id7955"><p>Emadoddin Karimian and Ehsan Motamedian. Acbm: an integrated agent and constraint based modeling framework for simulation of microbial communities. <em>SCIENTIFIC REPORTS</em>, MAY 26 2020. <a class="reference external" href="https://doi.org/10.1038/s41598-020-65659-w">[DOI]</a>.</p></li>
<li id="id7956"><p>George C. diCenzo, Michelangelo Tesi, Thomas Pfau, Alessio Mengoni, and Marco Fondi. Genome-scale metabolic reconstruction of the symbiosis between a leguminous plant and a nitrogen-fixing bacterium. <em>NATURE COMMUNICATIONS</em>, MAY 22 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-16484-2">[DOI]</a>.</p></li>
<li id="id7957"><p>Juan D. Tibocha-Bonilla, Manish Kumar, Anne Richelle, Ruben D. Godoy-Silva, Karsten Zengler, and Cristal Zuniga. Dynamic resource allocation drives growth under nitrogen starvation in eukaryotes. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, MAY 15 2020. <a class="reference external" href="https://doi.org/10.1038/s41540-020-0135-y">[DOI]</a>.</p></li>
<li id="id7958"><p>Hao Tong, Anika Kueken, and Zoran Nikoloski. Integrating molecular markers into metabolic models improves genomic selection for arabidopsis growth. <em>NATURE COMMUNICATIONS</em>, MAY 15 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-16279-5">[DOI]</a>.</p></li>
<li id="id7959"><p>Hoda Nouri, Hamideh Fouladiha, Hamid Moghimi, and Sayed-Amir Marashi. A reconciliation of genome-scale metabolic network model of zymomonas mobilis zm4. <em>SCIENTIFIC REPORTS</em>, MAY 8 2020. <a class="reference external" href="https://doi.org/10.1038/s41598-020-64721-x">[DOI]</a>.</p></li>
<li id="id7960"><p>Ahmad Ahmad, Ruchi Pathania, and Shireesh Srivastava. Biochemical characteristics and a genome-scale metabolic model of an indian euryhaline cyanobacterium with high polyglucan content. <em>METABOLITES</em>, MAY 2020. <a class="reference external" href="https://doi.org/10.3390/metabo10050177">[DOI]</a>.</p></li>
<li id="id7961"><p>Tahila Andrighetti, Balazs Bohar, Ney Lemke, Padhmanand Sudhakar, and Tamas Korcsmaros. Microbiolink: an integrated computational pipeline to infer functional effects of microbiome-host interactions. <em>CELLS</em>, MAY 2020. <a class="reference external" href="https://doi.org/10.3390/cells9051278">[DOI]</a>.</p></li>
<li id="id7962"><p>Zsolt Bodor, Andrea (Iuhasz) Fazakas, Katalin Bodor, Erika Kovacs, Ildiko Miklossy, and Beata Albert. Using genome-scale model to predict the metabolic engineering impact on escherichia coli metabolism during succinic acid production optimization. <em>ROMANIAN BIOTECHNOLOGICAL LETTERS</em>, 25(3):1666–1676, MAY-JUN 2020. <a class="reference external" href="https://doi.org/10.25083/rbl/25.3/1666.1676">[DOI]</a>.</p></li>
<li id="id7963"><p>Jan Geryk, Daniel Krsicka, Marketa Vlckova, Marketa Havlovicova, Milan Macek, Jr., and Radka Kremlikova Pourova. The key role of purine metabolism in the folate-dependent phenotype of autism spectrum disorders: an in silico analysis. <em>METABOLITES</em>, MAY 2020. <a class="reference external" href="https://doi.org/10.3390/metabo10050184">[DOI]</a>.</p></li>
<li id="id7964"><p>Yuki Kuriya and Michihiro Araki. Dynamic flux balance analysis to evaluate the strain production performance on shikimic acid production in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>METABOLITES</em>, MAY 2020. <a class="reference external" href="https://doi.org/10.3390/metabo10050198">[DOI]</a>.</p></li>
<li id="id7965"><p>Ines Thiele, Swagatika Sahoo, Almut Heinken, Johannes Hertel, Laurent Heirendt, Maike K. Aurich, and Ronan M. T. Fleming. Personalized whole-body models integrate metabolism, physiology, and the gut microbiome. <em>MOLECULAR SYSTEMS BIOLOGY</em>, MAY 2020. <a class="reference external" href="https://doi.org/10.15252/msb.20198982">[DOI]</a>.</p></li>
<li id="id7966"><p>Nana Y. D. Ankrah, Rebecca A. Wilkes, Freya Q. Zhang, Dantong Zhu, Tadeo Kaweesi, Ludmilla Aristilde, and Angela E. Douglas. Syntrophic splitting of central carbon metabolism in host cells bearing functionally different symbiotic bacteria. <em>ISME JOURNAL</em>, 14(8):1982–1993, AUG 2020. <a class="reference external" href="https://doi.org/10.1038/s41396-020-0661-z">[DOI]</a>.</p></li>
<li id="id7967"><p>Rosemary Yu, Kate Campbell, Rui Pereira, Johan Bjorkeroth, Qi Qi, Egor Vorontsov, Carina Sihlbom, and Jens Nielsen. Nitrogen limitation reveals large reserves in metabolic and translational capacities of yeast. <em>NATURE COMMUNICATIONS</em>, APR 20 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-15749-0">[DOI]</a>.</p></li>
<li id="id7968"><p>Thomas C. Keaty and Paul A. Jensen. Gapsplit: efficient random sampling for non-convex constraint-based models. <em>BIOINFORMATICS</em>, 36(8):2623–2625, APR 15 2020. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btz971">[DOI]</a>.</p></li>
<li id="id7969"><p>Andre Schultz and Rehan Akbani. Sammi: a semi-automated tool for the visualization of metabolic networks. <em>BIOINFORMATICS</em>, 36(8):2616–2617, APR 15 2020. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btz927">[DOI]</a>.</p></li>
<li id="id7970"><p>Lucia Trilla-Fuertes, Angelo Gamez-Pozo, Elena Lopez-Camacho, Guillermo Prado-Vazquez, Andrea Zapater-Moros, Rocio Lopez-Vacas, Jorge M. Arevalillo, Mariana Diaz-Almiron, Hilario Navarro, Paloma Main, Enrique Espinosa, Pilar Zamora, and Juan Angel Fresno Vara. Computational models applied to metabolomics data hints at the relevance of glutamine metabolism in breast cancer. <em>BMC CANCER</em>, APR 15 2020. <a class="reference external" href="https://doi.org/10.1186/s12885-020-06764-x">[DOI]</a>.</p></li>
<li id="id7971"><p>Saeideh Khodaee, Yazdan Asgari, Mehdi Totonchi, and Mohammad Hossein Karimi-Jafari. Imm1865: a new reconstruction of mouse genome-scale metabolic model. <em>SCIENTIFIC REPORTS</em>, APR 10 2020. <a class="reference external" href="https://doi.org/10.1038/s41598-020-63235-w">[DOI]</a>.</p></li>
<li id="id7972"><p>Katerina Rohlenova, Jermaine Goveia, Melissa Garcia-Caballero, Abhishek Subramanian, Joanna Kalucka, Lucas Treps, Kim D. Falkenberg, Laura P. M. H. de Rooij, Yingfeng Zheng, Lin Lin, Liliana Sokol, Laure-Anne Teuwen, Vincent Geldhof, Federico Taverna, Andreas Pircher, Lena-Christin Conradi, Shawez Khan, Steve Stegen, Dena Panovska, Frederik De Smet, Frank J. T. Staal, Rene J. Mclaughlin, Stefan Vinckier, Tine Van Bergen, Nadine Ectors, Patrik De Haes, Jian Wang, Lars Bolund, Luc Schoonjans, Tobias K. Karakach, Huanming Yang, Geert Carmeliet, Yizhi Liu, Bernard Thienpont, Mieke Dewerchin, Guy Eelen, Xuri Li, Yonglun Luo, and Peter Carmeliet. Single-cell rna sequencing maps endothelial metabolic plasticity in pathological angiogenesis. <em>CELL METABOLISM</em>, 31(4):862+, APR 7 2020. <a class="reference external" href="https://doi.org/10.1016/j.cmet.2020.03.009">[DOI]</a>.</p></li>
<li id="id7973"><p>Archanaa Sundararaghavan, Amitava Mukherjee, Swagatika Sahoo, and G. K. Suraishkumar. Mechanism of the oxidative stress-mediated increase in lipid accumulation by the bacterium, &lt;i&gt;r. opacus&lt;/i&gt; pd630: experimental analysis and genome-scale metabolic modeling. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 117(6):1779–1788, JUN 2020. <a class="reference external" href="https://doi.org/10.1002/bit.27330">[DOI]</a>.</p></li>
<li id="id7974"><p>Bing Hu, Miaoxiao Wang, Shuang Geng, Liqun Wen, Mengdi Wu, Yong Nie, Yue-Qin Tang, and Xiao-Lei Wu. Metabolic exchange with non-alkane-consuming &lt;i&gt;pseudomonas stutzeri&lt;/i&gt; slg510a3-8 improves &lt;i&gt;n&lt;/i&gt;-alkane biodegradation by the alkane degrader &lt;i&gt;dietzia&lt;/i&gt; sp. strain dq12-45-1b. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, APR 2020. <a class="reference external" href="https://doi.org/10.1128/AEM.02931-19">[DOI]</a>.</p></li>
<li id="id7975"><p>Na-Rae Lee, Choong Hwan Lee, Dong-Yup Lee, and Jin-Byung Park. Genome-scale metabolic network reconstruction and in silico analysis of hexanoic acid producing &lt;i&gt;megasphaera elsdenii&lt;/i&gt;. <em>MICROORGANISMS</em>, APR 2020. <a class="reference external" href="https://doi.org/10.3390/microorganisms8040539">[DOI]</a>.</p></li>
<li id="id7976"><p>Ratklao Siriwach, Fumio Matsuda, Kentaro Yano, and Masami Yokota Hirai. Drought stress responses in context-specific genome-scale metabolic models of &lt;i&gt;arabidopsis thaliana&lt;/i&gt;. <em>METABOLITES</em>, APR 2020. <a class="reference external" href="https://doi.org/10.3390/metabo10040159">[DOI]</a>.</p></li>
<li id="id7977"><p>Lucia Trilla-Fuertes, Ismael Ghanem, Angelo Gamez-Pozo, Joan Maurel, Laura G-Pastrian, Marta Mendiola, Cristina Pena, Rocio Lopez-Vacas, Guillermo Prado-Vazquez, Elena Lopez-Camacho, Andrea Zapater-Moros, Victoria Heredia, Miriam Cuatrecasas, Pilar Garcia-Alfonso, Jaume Capdevila, Carles Conill, Rocio Garcia-Carbonero, Ricardo Ramos-Ruiz, Claudia Fortes, Carlos Llorens, Paolo Nanni, Juan Angel Fresno Vara, and Jaime Feliu. Genetic profile and functional proteomics of anal squamous cell carcinoma: proposal for a molecular classification. <em>MOLECULAR &amp; CELLULAR PROTEOMICS</em>, 19(4):690–700, APR 2020. <a class="reference external" href="https://doi.org/10.1074/mcp.RA120.001954">[DOI]</a>.</p></li>
<li id="id7978"><p>Yan Zhu, Jing Lu, Jinxin Zhao, Xinru Zhang, Heidi H. Yu, Tony Velkov, and Jian Li. Complete genome sequence and genome-scale metabolic modelling of &lt;i&gt;acinetobacter baumannii&lt;/i&gt; type strain atcc 19606. <em>INTERNATIONAL JOURNAL OF MEDICAL MICROBIOLOGY</em>, APR 2020. <a class="reference external" href="https://doi.org/10.1016/j.ijmm.2020.151412">[DOI]</a>.</p></li>
<li id="id7979"><p>Yoseb Song, Jin Soo Lee, Jongoh Shin, Gyu Min Lee, Sangrak Jin, Seulgi Kang, Jung-Kul Lee, Dong Rip Kim, Eun Yeol Lee, Sun Chang Kim, Suhyung Cho, Donghyuk Kim, and Byung-Kwan Cho. Functional cooperation of the glycine synthase-reductase and wood-ljungdahl pathways for autotrophic growth of &lt;i&gt;clostridium drakei&lt;/i&gt;. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 117(13):7516–7523, MAR 31 2020. <a class="reference external" href="https://doi.org/10.1073/pnas.1912289117">[DOI]</a>.</p></li>
<li id="id7980"><p>Agnieszka B. Wegrzyn, Katharina Herzog, Albert Gerding, Marcel Kwiatkowski, Justina C. Wolters, Amalia M. Dolga, Alida E. M. van Lint, Ronald J. A. Wanders, Hans R. Waterham, and Barbara M. Bakker. Fibroblast-specific genome-scale modelling predicts an imbalance in amino acid metabolism in refsum disease. <em>FEBS JOURNAL</em>, 287(23):5096–5113, DEC 2020. <a class="reference external" href="https://doi.org/10.1111/febs.15292">[DOI]</a>.</p></li>
<li id="id7981"><p>Dinka Mandakovic, Angela Cintolesi, Jonathan Maldonado, Sebastian N. Mendoza, Meziane Aite, Alexis Gaete, Francisco Saitua, Miguel Allende, Veronica Cambiazo, Anne Siegel, Alejandro Maass, Mauricio Gonzalez, and Mauricio Latorre. Genome-scale metabolic models of &lt;i&gt;microbacterium&lt;/i&gt; species isolated from a high altitude desert environment. <em>SCIENTIFIC REPORTS</em>, MAR 27 2020. <a class="reference external" href="https://doi.org/10.1038/s41598-020-62130-8">[DOI]</a>.</p></li>
<li id="id7982"><p>Hamideh Fouladiha, Sayed-Amir Marashi, Fatemeh Torkashvand, Fereidoun Mahboudi, Nathan E. Lewis, and Behrouz Vaziri. A metabolic network-based approach for developing feeding strategies for cho cells to increase monoclonal antibody production. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 43(8):1381–1389, AUG 2020. <a class="reference external" href="https://doi.org/10.1007/s00449-020-02332-6">[DOI]</a>.</p></li>
<li id="id7983"><p>Jonathan L. Robinson, Pinar Kocabas, Hao Wang, Pierre-Etienne Cholley, Daniel Cook, Avlant Nilsson, Mihail Anton, Raphael Ferreira, Ivan Domenzain, Virinchi Billa, Angelo Limeta, Alex Hedin, Johan Gustafsson, Eduard J. Kerkhoven, L. Thomas Svensson, Bernhard O. Palsson, Adil Mardinoglu, Lena Hansson, Mathias Uhlen, and Jens Nielsen. An atlas of human metabolism. <em>SCIENCE SIGNALING</em>, MAR 24 2020. <a class="reference external" href="https://doi.org/10.1126/scisignal.aaz1482">[DOI]</a>.</p></li>
<li id="id7984"><p>Axel von Kamp and Steffen Klamt. Memo: a method for computing metabolic modules for cell-free production systems. <em>ACS SYNTHETIC BIOLOGY</em>, 9(3):556–566, MAR 20 2020. <a class="reference external" href="https://doi.org/10.1021/acssynbio.9b00434">[DOI]</a>.</p></li>
<li id="id7985"><p>Shan Zhang, Boris Reljic, Chao Liang, Baptiste Kerouanton, Joel Celio Francisco, Jih Hou Peh, Camille Mary, Narendra Suhas Jagannathan, Volodimir Olexiouk, Claire Tang, Gio Fidelito, Srikanth Nama, Ruey-Kuang Cheng, Caroline Lei Wee, Loo Chien Wang, Paula Duek Roggli, Prabha Sampath, Lydie Lane, Enrico Petretto, Radoslaw M. Sobota, Suresh Jesuthasan, Lisa Tucker-Kellogg, Bruno Reversade, Gerben Menschaert, Lei Sun, David A. Stroud, and Lena Ho. Mitochondrial peptide brawnin is essential for vertebrate respiratory complex iii assembly. <em>NATURE COMMUNICATIONS</em>, MAR 11 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-020-14999-2">[DOI]</a>.</p></li>
<li id="id7986"><p>Hong Zeng and Aidong Yang. Bridging substrate intake kinetics and bacterial growth phenotypes with flux balance analysis incorporating proteome allocation. <em>SCIENTIFIC REPORTS</em>, MAR 9 2020. <a class="reference external" href="https://doi.org/10.1038/s41598-020-61174-0">[DOI]</a>.</p></li>
<li id="id7987"><p>Christian Lieven, Moritz E. Beber, Brett G. Olivier, Frank T. Bergmann, Meric Ataman, Parizad Babaei, Jennifer A. Bartell, Lars M. Blank, Siddharth Chauhan, Kevin Correia, Christian Diener, Andreas Draeger, Birgitta E. Ebert, Janaka N. Edirisinghe, Jose P. Faria, Adam M. Feist, Georgios Fengos, Ronan M. T. Fleming, Beatriz Garcia-Jimenez, Vassily Hatzimanikatis, Wout van Helvoirt, Christopher S. Henry, Henning Hermjakob, Markus J. Herrgard, Ali Kaafarani, Hyun Uk Kim, Zachary King, Steffen Klamt, Edda Klipp, Jasper J. Koehorst, Matthias Koenig, Meiyappan Lakshmanan, Dong-Yup Lee, Sang Yup Lee, Sunjae Lee, Nathan E. Lewis, Filipe Liu, Hongwu Ma, Daniel Machado, Radhakrishnan Mahadevan, Paulo Maia, Adil Mardinoglu, Gregory L. Medlock, Jonathan M. Monk, Jens Nielsen, Lars Keld Nielsen, Juan Nogales, Intawat Nookaew, Bernhard O. Palsson, Jason A. Papin, Kiran R. Patil, Mark Poolman, Nathan D. Price, Osbaldo Resendis-Antonio, Anne Richelle, Isabel Rocha, Benjamin J. Sanchez, Peter J. Schaap, Rahuman S. Malik Sheriff, Saeed Shoaie, Nikolaus Sonnenschein, Bas Teusink, Paulo Vilaca, Jon Olav Vik, Judith A. H. Wodke, Joana C. Xavier, Qianqian Yuan, Maksim Zakhartsev, and Cheng Zhang. Memote for standardized genome-scale metabolic model testing. <em>NATURE BIOTECHNOLOGY</em>, 38(3):272–276, MAR 2020. <a class="reference external" href="https://doi.org/10.1038/s41587-020-0446-y">[DOI]</a>.</p></li>
<li id="id7988"><p>Manali Das, Pradipta Patra, and Amit Ghosh. Metabolic engineering for enhancing microbial biosynthesis of advanced biofuels. <em>RENEWABLE &amp; SUSTAINABLE ENERGY REVIEWS</em>, MAR 2020. <a class="reference external" href="https://doi.org/10.1016/j.rser.2019.109562">[DOI]</a>.</p></li>
<li id="id7989"><p>Zhuangrong Huang and Seongkyu Yoon. Integration of time-series transcriptomic data with genome-scale cho metabolic models for mab engineering. <em>PROCESSES</em>, MAR 2020. <a class="reference external" href="https://doi.org/10.3390/pr8030331">[DOI]</a>.</p></li>
<li id="id7990"><p>Marius Tomas-Gamisans, Cristiane Conte Paim Andrade, Francisco Maresca, Sergi Monforte, Pau Ferrer, and Joan Albiol. Redox engineering by ectopic overexpression of nadh kinase in recombinant &lt;i&gt;pichia pastoris&lt;/i&gt; (&lt;i&gt;komagataella phaffii&lt;/i&gt;): impact on cell physiology and recombinant production of secreted proteins. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, MAR 2020. <a class="reference external" href="https://doi.org/10.1128/AEM.02038-19">[DOI]</a>.</p></li>
<li id="id7991"><p>Yuliang Wang, Shuyi Ma, and Walter L. Ruzzo. Spatial modeling of prostate cancer metabolic gene expression reveals extensive heterogeneity and selective vulnerabilities. <em>SCIENTIFIC REPORTS</em>, FEB 26 2020. <a class="reference external" href="https://doi.org/10.1038/s41598-020-60384-w">[DOI]</a>.</p></li>
<li id="id7992"><p>Gunvor Bjerkelund Rokke, Martin Frank Hohmann-Marriott, and Eivind Almaas. An adjustable algal chloroplast plug-and-play model for genome-scale metabolic models. <em>PLOS ONE</em>, FEB 24 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0229408">[DOI]</a>.</p></li>
<li id="id7993"><p>Gong-Hua Li, Shaoxing Dai, Feifei Han, Wenxin Li, Jingfei Huang, and Wenzhong Xiao. Fastmm: an efficient toolbox for personalized constraint-based metabolic modeling. <em>BMC BIOINFORMATICS</em>, FEB 21 2020. <a class="reference external" href="https://doi.org/10.1186/s12859-020-3410-4">[DOI]</a>.</p></li>
<li id="id7994"><p>Lokanand Koduru, Hyang Yeon Kim, Meiyappan Lakshmanan, Bijayalaxmi Mohanty, Yi Qing Lee, Choong Hwan Lee, and Dong-Yup Lee. Genome-scale metabolic reconstruction and in silico analysis of the rice leaf blight pathogen, &lt;i&gt;xanthomonas oryzae&lt;/i&gt;. <em>MOLECULAR PLANT PATHOLOGY</em>, 21(4):527–540, APR 2020. <a class="reference external" href="https://doi.org/10.1111/mpp.12914">[DOI]</a>.</p></li>
<li id="id7995"><p>Wentao Dong, Sun Jin Moon, Joanne K. Kelleher, and Gregory Stephanopoulos. Dissecting mammalian cell metabolism through &lt;sup&gt;13&lt;/sup&gt;c- and &lt;sup&gt;2&lt;/sup&gt;h-isotope tracing: interpretations at the molecular and systems levels. <em>INDUSTRIAL &amp; ENGINEERING CHEMISTRY RESEARCH</em>, 59(6):2593–2610, FEB 12 2020. <a class="reference external" href="https://doi.org/10.1021/acs.iecr.9b05154">[DOI]</a>.</p></li>
<li id="id7996"><p>Aarti Krishnan, Joachim Kloehn, Matteo Lunghi, Anush Chiappino-Pepe, Benjamin S. Waldman, Damien Nicolas, Emmanuel Varesio, Adrian Hehl, Sebastian Lourido, Vassily Hatzimanikatis, and Dominique Soldati-Favre. Functional and computational genomics reveal unprecedented flexibility in stage-specific toxoplasma metabolism. <em>CELL HOST &amp; MICROBE</em>, 27(2):290+, FEB 12 2020. <a class="reference external" href="https://doi.org/10.1016/j.chom.2020.01.002">[DOI]</a>.</p></li>
<li id="id7997"><p>Michel Lavoie, Blanche Saint-Beat, Jan Strauss, Sebastien Guerin, Antoine Allard, Simon V. Hardy, Angela Falciatore, and Johann Lavaud. Genome-scale metabolic reconstruction and &lt;i&gt;in silico&lt;/i&gt; perturbation analysis of the polar diatom &lt;i&gt;fragilariopsis cylindrus&lt;/i&gt; predicts high metabolic robustness. <em>BIOLOGY-BASEL</em>, FEB 2020. <a class="reference external" href="https://doi.org/10.3390/biology9020030">[DOI]</a>.</p></li>
<li id="id7998"><p>Tal Pecht, Anna C. Aschenbrenner, Thomas Ulas, and Antonella Succurro. Modeling population heterogeneity from microbial communities to immune response in cells. <em>CELLULAR AND MOLECULAR LIFE SCIENCES</em>, 77(3):415–432, FEB 2020. <a class="reference external" href="https://doi.org/10.1007/s00018-019-03378-w">[DOI]</a>.</p></li>
<li id="id7999"><p>Mohammad Rezazadeh, Valiollah Babaeipour, and Ehsan Motamedian. Reconstruction, verification and in-silico analysis of a genome-scale metabolic model of bacterial cellulose producing &lt;i&gt;komagataeibacter xylinus&lt;/i&gt;. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 43(6):1017–1026, JUN 2020. <a class="reference external" href="https://doi.org/10.1007/s00449-020-02299-4">[DOI]</a>.</p></li>
<li id="id8000"><p>Yu Xu, Roman Holic, and Qiang Hua. Comparison and analysis of published genome-scale metabolic models of &lt;i&gt;yarrowia lipolytica&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 25(1):53–61, FEB 2020. <a class="reference external" href="https://doi.org/10.1007/s12257-019-0208-1">[DOI]</a>.</p></li>
<li id="id8001"><p>M. Fata Moradali and Bernd H. A. Rehm. Bacterial biopolymers: from pathogenesis to advanced materials. <em>NATURE REVIEWS MICROBIOLOGY</em>, 18(4):195–210, APR 2020. <a class="reference external" href="https://doi.org/10.1038/s41579-019-0313-3">[DOI]</a>.</p></li>
<li id="id8002"><p>Ali Malek Shahkouhi and Ehsan Motamedian. Reconstruction of a regulated two-cell metabolic model to study biohydrogen production in a diazotrophic cyanobacterium &lt;i&gt;anabaena variabilis&lt;/i&gt; atcc 29413. <em>PLOS ONE</em>, JAN 24 2020. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0227977">[DOI]</a>.</p></li>
<li id="id8004"><p>Birthe Halmschlag, Kyra Hoffmann, Rene Hanke, Sastia P. Putri, Eiichiro Fukusaki, Jochen Buechs, and Lars M. Blank. Comparison of isomerase and weimberg pathway for γ-pga production from xylose by engineered &lt;i&gt;bacillus subtilis&lt;/i&gt;. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, JAN 21 2020. <a class="reference external" href="https://doi.org/10.3389/fbioe.2019.00476">[DOI]</a>.</p></li>
<li id="id8005"><p>Elham Iranmanesh, Mohammad Ali Asadollahi, and Davoud Biria. Improving l-phenylacetylcarbinol production in &lt;i&gt;saccharomyces&lt;/i&gt; &lt;i&gt;cerevisiae&lt;/i&gt; by &lt;i&gt;in silico&lt;/i&gt; aided metabolic engineering. <em>JOURNAL OF BIOTECHNOLOGY</em>, 308:27–34, JAN 20 2020. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2019.11.008">[DOI]</a>.</p></li>
<li id="id8006"><p>Isabel Bator, Andreas Wittgens, Frank Rosenau, Till Tiso, and Lars M. Blank. Comparison of three xylose pathways in &lt;i&gt;pseudomonas putida&lt;/i&gt; kt2440 for the synthesis of valuable products. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, JAN 17 2020. <a class="reference external" href="https://doi.org/10.3389/fbioe.2019.00480">[DOI]</a>.</p></li>
<li id="id8007"><p>Taneli Pusa, Mariana Galvao Ferrarini, Ricardo Andrade, Arnaud Mary, Alberto Marchetti-Spaccamela, Leen Stougie, and Marie-France Sagot. Moomin - mathematical exploration of `omics data on a metabolic network. <em>BIOINFORMATICS</em>, 36(2):514–523, JAN 15 2020. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btz584">[DOI]</a>.</p></li>
<li id="id8008"><p>Pavlos Stephanos Bekiaris and Steffen Klamt. Automatic construction of metabolic models with enzyme constraints. <em>BMC BIOINFORMATICS</em>, JAN 14 2020. <a class="reference external" href="https://doi.org/10.1186/s12859-019-3329-9">[DOI]</a>.</p></li>
<li id="id8009"><p>Pierre Salvy and Vassily Hatzimanikatis. The etfl formulation allows multi-omics integration in thermodynamics-compliant metabolism and expression models. <em>NATURE COMMUNICATIONS</em>, JAN 13 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-019-13818-7">[DOI]</a>.</p></li>
<li id="id8010"><p>Julia Troendle, Kristin Schoppel, Arne Bleidt, Natalia Trachtmann, Georg A. Sprenger, and Dirk Weuster-Botz. Metabolic control analysis of l-tryptophan production with &lt;i&gt;escherichia coli&lt;/i&gt; based on data from short-term perturbation experiments. <em>JOURNAL OF BIOTECHNOLOGY</em>, 307:15–28, JAN 10 2020. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2019.10.009">[DOI]</a>.</p></li>
<li id="id8011"><p>Jahir M. Gutierrez, Amir Feizi, Shangzhong Li, Thomas B. Kallehauge, Hooman Hefzi, Lise M. Grav, Daniel Ley, Deniz Baycin Hizal, Michael J. Betenbaugh, Bjorn Voldborg, Helene Faustrup Kildegaard, Gyun Min Lee, Bernhard O. Palsson, Jens Nielsen, and Nathan E. Lewis. Genome-scale reconstructions of the mammalian secretory pathway predict metabolic costs and limitations of protein secretion. <em>NATURE COMMUNICATIONS</em>, JAN 2 2020. <a class="reference external" href="https://doi.org/10.1038/s41467-019-13867-y">[DOI]</a>.</p></li>
<li id="id8012"><p>Shreya Anand, Koel Mukherjee, and Padmini Padmanabhan. An insight to flux-balance analysis for biochemical networks. In <em>BIOTECHNOLOGY AND GENETIC ENGINEERING REVIEWS, VOL 36, NO 1</em>, volume 36 of Biotechnology &amp; Genetic Engineering Reviews, pages 32–55. 2020. <a class="reference external" href="https://doi.org/10.1080/02648725.2020.1847440">[DOI]</a>.</p></li>
<li id="id8013"><p>Teresa J. Clark, Longyun Guo, John Morgan, and Jorg Schwender. Modeling plant metabolism: from network reconstruction to mechanistic models. In SS Merchant, editor, <em>ANNUAL REVIEW OF PLANT BIOLOGY, VOL 71, 2020</em>, volume 71 of Annual Review of Plant Biology, pages 303–326. 2020. <a class="reference external" href="https://doi.org/10.1146/annurev-arplant-050718-100221">[DOI]</a>.</p></li>
<li id="id8014"><p>Carolina A. Contador, Siu-Kit Lo, Siu H. J. Chan, and Hon-Ming Lam. Metabolic analyses of nitrogen fixation in the soybean microsymbiont &lt;i&gt;sinorhizobium fredii&lt;/i&gt; using constraint-based modeling. <em>MSYSTEMS</em>, JAN-FEB 2020. <a class="reference external" href="https://doi.org/10.1128/mSystems.00516-19">[DOI]</a>.</p></li>
<li id="id8015"><p>Alex Graudenzi, Davide Maspero, and Chiara Damiani. Fbca, a multiscale modeling framework combining cellular automata and flux balance analysis. <em>JOURNAL OF CELLULAR AUTOMATA</em>, 15(1-2):75–95, 2020.</p></li>
<li id="id8016"><p>Davide Maspero, Chiara Damiani, Marco Antoniotti, Alex Graudenzi, Marzia Di Filippo, Marco Vanoni, Giulio Caravagna, Riccardo Colombo, Daniele Ramazzotti, and Dario Pescini. The influence of nutrients diffusion on a metabolism-driven model of a multi-cellular system. <em>FUNDAMENTA INFORMATICAE</em>, 171(1-4, SI):279–295, 2020. <a class="reference external" href="https://doi.org/10.3233/FI-2020-1883">[DOI]</a>.</p></li>
<li id="id8017"><p>Charles J. Norsigian, Xin Fang, Yara Seif, Jonathan M. Monk, and Bernhard O. Palsson. A workflow for generating multi-strain genome-scale metabolic models of prokaryotes. <em>NATURE PROTOCOLS</em>, 15(1):1–14, JAN 2020. <a class="reference external" href="https://doi.org/10.1038/s41596-019-0254-3">[DOI]</a>.</p></li>
<li id="id8018"><p>Rosina Pryor, Daniel Martinez-Martinez, Leonor Quintaneiro, and Filipe Cabreiro. The role of the microbiome in drug response. In PA Insel, editor, <em>ANNUAL REVIEW OF PHARMACOLOGY AND TOXICOLOGY, VOL 60</em>, volume 60 of Annual Review of Pharmacology and Toxicology, pages 417–435. 2020. <a class="reference external" href="https://doi.org/10.1146/annurev-pharmtox-010919-023612">[DOI]</a>.</p></li>
<li id="id8019"><p>Kwanjeera Wanichthanarak, Chuthamas Boonchai, Thammaporn Kojonna, Supachitra Chadchawan, Wichian Sangwongchai, and Maysaya Thitisaksakul. Deciphering rice metabolic flux reprograming under salinity stress via &lt;i&gt;in silico&lt;/i&gt; metabolic modeling. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 18:3555–3566, 2020. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2020.11.023">[DOI]</a>.</p></li>
<li id="id8021"><p>Guan Wang, Cees Haringa, Wenjun Tang, Henk Noorman, Ju Chu, Yingping Zhuang, and Siliang Zhang. Coupled metabolic-hydrodynamic modeling enabling rational scale-up of industrial bioprocesses. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 117(3):844–867, MAR 2020. <a class="reference external" href="https://doi.org/10.1002/bit.27243">[DOI]</a>.</p></li>
<li id="id8023"><p>Freyr Johannsson, Niels A. Arnason, Ragna Landroe, Sveinn Gudmundsson, Olafur E. Sigurjonsson, and Ottar Rolfsson. Metabolomics study of platelet concentrates photochemically treated with amotosalen and uva light for pathogen inactivation. <em>TRANSFUSION</em>, 60(2):367–377, FEB 2020. <a class="reference external" href="https://doi.org/10.1111/trf.15610">[DOI]</a>.</p></li>
<li id="id8035"><p>Mochamad Nurcholis, Noppon Lertwattanasakul, Nadchanok Rodrussamee, Tomoyuki Kosaka, Masayuki Murata, and Mamoru Yamada. Integration of comprehensive data and biotechnological tools for industrial applications of &lt;i&gt;kluyveromyces marxianus&lt;/i&gt;. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 104(2):475–488, JAN 2020. <a class="reference external" href="https://doi.org/10.1007/s00253-019-10224-3">[DOI]</a>.</p></li>
<li id="id8041"><p>Juan Nogales, Joshua Mueller, Steinn Gudmundsson, Francisco J. Canalejo, Estrella Duque, Jonathan Monk, Adam M. Feist, Juan Luis Ramos, Wei Niu, and Bernhard O. Palsson. High-quality genome-scale metabolic modelling of &lt;i&gt;pseudomonas putida&lt;/i&gt; highlights its broad metabolic capabilities. <em>ENVIRONMENTAL MICROBIOLOGY</em>, 22(1):255–269, JAN 2020. <a class="reference external" href="https://doi.org/10.1111/1462-2920.14843">[DOI]</a>.</p></li>
<li id="id8048"><p>Masakazu Toyoshima, Yoshihiro Toya, and Hiroshi Shimizu. Flux balance analysis of cyanobacteria reveals selective use of photosynthetic electron transport components under different spectral light conditions. <em>PHOTOSYNTHESIS RESEARCH</em>, 143(1):31–43, JAN 2020. <a class="reference external" href="https://doi.org/10.1007/s11120-019-00678-x">[DOI]</a>.</p></li>
<li id="id8050"><p>Krishna Kanhaiya and Dwitiya Tiwari. Identification of drug targets in breast cancer metabolic network. <em>JOURNAL OF COMPUTATIONAL BIOLOGY</em>, 27(6):975–986, JUN 1 2020. <a class="reference external" href="https://doi.org/10.1089/cmb.2019.0258">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-5" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-5" name="0-5" role="tabpanel" tabindex="0"><div class="docutils container" id="id9101">
<ol class="arabic simple" start="1">
<li id="id9840"><p>D. A. Mugahid, T. G. Sengul, X. You, Y. Wang, L. Steil, N. Bergmann, M. H. Radke, A. Ofenbauer, M. Gesell-Salazar, A. Balogh, S. Kempa, B. Tursun, C. T. Robbins, U. Voelker, W. Chen, L. Nelson, and M. Gotthardt. Proteomic and transcriptomic changes in hibernating grizzly bears reveal metabolic and signaling pathways that protect against muscle atrophy. <em>SCIENTIFIC REPORTS</em>, DEC 27 2019. <a class="reference external" href="https://doi.org/10.1038/s41598-019-56007-8">[DOI]</a>.</p></li>
<li id="id9842"><p>Sucheta Gokhale and Anirban Bhaduri. Provitamin d&lt;sub&gt;3&lt;/sub&gt; modulation through prebiotics supplementation: simulation based assessment. <em>SCIENTIFIC REPORTS</em>, DEC 17 2019. <a class="reference external" href="https://doi.org/10.1038/s41598-019-55699-2">[DOI]</a>.</p></li>
<li id="id9844"><p>N. T. Devika and Karthik Raman. Deciphering the metabolic capabilities of bifidobacteria using genome-scale metabolic models. <em>SCIENTIFIC REPORTS</em>, DEC 3 2019. <a class="reference external" href="https://doi.org/10.1038/s41598-019-54696-9">[DOI]</a>.</p></li>
<li id="id9845"><p>Masoud Ahookhosh, Francisco J. Aragon Artacho, Ronan M. T. Fleming, and Phan T. Vuong. Local convergence of the levenberg-marquardt method under holder metric subregularity. <em>ADVANCES IN COMPUTATIONAL MATHEMATICS</em>, 45(5-6):2771–2806, DEC 2019. <a class="reference external" href="https://doi.org/10.1007/s10444-019-09708-7">[DOI]</a>.</p></li>
<li id="id9846"><p>Joshua E. Goldford, Hyman Hartman, Robert Marsland, III, and Daniel Segre. Environmental boundary conditions for the origin of life converge to an organo-sulfur metabolism. <em>NATURE ECOLOGY &amp; EVOLUTION</em>, 3(12):1715+, DEC 2019. <a class="reference external" href="https://doi.org/10.1038/s41559-019-1018-8">[DOI]</a>.</p></li>
<li id="id9847"><p>Allison J. Lopatkin, Jonathan M. Stokes, Erica J. Zheng, Jason H. Yang, Melissa K. Takahashi, Lingchong You, and James J. Collins. Bacterial metabolic state more accurately predicts antibiotic lethality than growth rate. <em>NATURE MICROBIOLOGY</em>, 4(12):2109–2117, DEC 2019. <a class="reference external" href="https://doi.org/10.1038/s41564-019-0536-0">[DOI]</a>.</p></li>
<li id="id9848"><p>Adam Meyer, Ishtiaq Saaem, Adam Silverman, Vanessa A. Varaljay, Rebecca Mickol, Steven Blum, Alexander V. Tobias, Nathan D. Schwalm, III, Wais Mojadedi, Elizabeth Onderko, Cassandra Bristol, Shangtao Liu, Katelin Pratt, Arturo Casini, Raissa Eluere, Felix Moser, Carrie Drake, Maneesh Gupta, Nancy Kelley-Loughnane, Julius P. Lucks, Katherine L. Akingbade, Matthew P. Lux, Sarah Glaven, Wendy Crookes-Goodson, Michael C. Jewett, D. Benjamin Gordon, and Christopher A. Voigt. Organism engineering for the bioproduction of the triaminotrinitrobenzene (tatb) precursor phloroglucinol (pg). <em>ACS SYNTHETIC BIOLOGY</em>, 8(12):2746–2755, DEC 2019. <a class="reference external" href="https://doi.org/10.1021/acssynbio.9b00393">[DOI]</a>.</p></li>
<li id="id9849"><p>Rui M. C. Portela, Anne Richelle, Patrick Dumas, and Moritz von Stosch. Time integrated flux analysis: exploiting the concentration measurements directly for cost-effective metabolic network flux analysis. <em>MICROORGANISMS</em>, DEC 2019. <a class="reference external" href="https://doi.org/10.3390/microorganisms7120620">[DOI]</a>.</p></li>
<li id="id9850"><p>Kristopher D. Rawls, Edik M. Blais, Bonnie Dougherty, V, Kalyan C. Vinnakota, Venkat R. Pannala, Anders Wallqvist, Glynis L. Kolling, and Jason A. Papin. Genome-scale characterization of toxicity-induced metabolic alterations in primary hepatocytes. <em>TOXICOLOGICAL SCIENCES</em>, 172(2):279–291, DEC 2019. <a class="reference external" href="https://doi.org/10.1093/toxsci/kfz197">[DOI]</a>.</p></li>
<li id="id9851"><p>Nathan D. Schwalm, III, Wais Mojadedi, Elliot S. Gerlach, Marcus Benyamin, Matthew A. Perisin, and Katherine L. Akingbade. Developing a microbial consortium for enhanced metabolite production from simulated food waste. <em>FERMENTATION-BASEL</em>, DEC 2019. <a class="reference external" href="https://doi.org/10.3390/fermentation5040098">[DOI]</a>.</p></li>
<li id="id9852"><p>Livia S. Zaramela, Cameron Martino, Frederico Alisson-Silva, Steven D. Rees, Sandra L. Diaz, Lea Chuzel, Mehul B. Ganatra, Christopher H. Taron, Patrick Secrest, Cristal Zuniga, Jianbo Huang, Dionicio Siegel, Geoffrey Chang, Ajit Varki, and Karsten Zengler. Gut bacteria responding to dietary change encode sialidases that exhibit preference for red meat-associated carbohydrates. <em>NATURE MICROBIOLOGY</em>, 4(12):2082–2089, DEC 2019. <a class="reference external" href="https://doi.org/10.1038/s41564-019-0564-9">[DOI]</a>.</p></li>
<li id="id9853"><p>Cristal Zuniga, Chien-Ting Li, Geng Yu, Mahmoud M. Al-Bassam, Tingting Li, Liqun Jiang, Livia S. Zaramela, Michael Guarnieri, Michael J. Betenbaugh, and Karsten Zengler. Environmental stimuli drive a transition from cooperation to competition in synthetic phototrophic communities. <em>NATURE MICROBIOLOGY</em>, 4(12):2184–2191, DEC 2019. <a class="reference external" href="https://doi.org/10.1038/s41564-019-0567-6">[DOI]</a>.</p></li>
<li id="id9854"><p>Rotem Katzir, Ibrahim H. Polat, Michal Harel, Shir Katz, Carles Foguet, Vitaly A. Selivanov, Philippe Sabatier, Marta Cascante, Tamar Geiger, and Eytan Ruppin. The landscape of tiered regulation of breast cancer cell metabolism. <em>SCIENTIFIC REPORTS</em>, NOV 28 2019. <a class="reference external" href="https://doi.org/10.1038/s41598-019-54221-y">[DOI]</a>.</p></li>
<li id="id9856"><p>Nutan Chauhan and Shailza Singh. Integrative computational framework for understanding metabolic modulation in leishmania. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, NOV 19 2019. <a class="reference external" href="https://doi.org/10.3389/fbioe.2019.00336">[DOI]</a>.</p></li>
<li id="id9857"><p>Aparajitha Srinivasan, S. Vijayakumar, Karthik Raman, and Smita Srivastava. Rational metabolic engineering for enhanced alpha-tocopherol production in &lt;i&gt;helianthus annuus&lt;/i&gt; cell culture. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, NOV 15 2019. <a class="reference external" href="https://doi.org/10.1016/j.bej.2019.107256">[DOI]</a>.</p></li>
<li id="id9858"><p>Uldis Kalnenieks, Elina Balodite, Steffi Straehler, Inese Strazdina, Julia Rex, Agris Pentjuss, Katsuya Fuchino, Per Bruheim, Reinis Rutkis, Katherine M. Pappas, Robert K. Poole, Oliver Sawodny, and Katja Bettenbrock. Improvement of acetaldehyde production in &lt;i&gt;zymomonas mobilis&lt;/i&gt; by engineering of its aerobic metabolism. <em>FRONTIERS IN MICROBIOLOGY</em>, NOV 14 2019. <a class="reference external" href="https://doi.org/10.3389/fmicb.2019.02533">[DOI]</a>.</p></li>
<li id="id9859"><p>Rebecca R. Stanway, Ellen Bushell, Anush Chiappino-Pepe, Magali Roques, Theo Sanderson, Blandine Franke-Fayard, Reto Caldelari, Murielle Golomingi, Mary Nyonda, Vikash Pandey, Frank Schwach, Severine Chevalley, Jai Ramesar, Tom Metcalf, Colin Herd, Paul-Christian Burda, Julian C. Rayner, Dominique Soldati-Favre, Chris J. Janse, Vassily Hatzimanikatis, Oliver Billker, and Volker T. Heussler. Genome-scale identification of essential metabolic processes for targeting the &lt;i&gt;plasmodium&lt;/i&gt; liver stage. <em>CELL</em>, 179(5):1112+, NOV 14 2019. <a class="reference external" href="https://doi.org/10.1016/j.cell.2019.10.030">[DOI]</a>.</p></li>
<li id="id9860"><p>Johannes Hertel, Amy C. Harms, Almut Heinken, Federico Baldini, Cyrille C. Thinnes, Enrico Glaab, Daniel A. Vasco, Maik Pietzner, Isobel D. Stewart, Nicholas J. Wareham, Claudia Langenberg, Claudia Trenkwalder, Rejko Krueger, Thomas Hankemeier, Ronan M. T. Fleming, Brit Mollenhauer, and Ines Thiele. Integrated analyses of microbiome and longitudinal metabolome data reveal microbial-host interactions on sulfur metabolism in parkinson's disease. <em>CELL REPORTS</em>, 29(7):1767+, NOV 12 2019. <a class="reference external" href="https://doi.org/10.1016/j.celrep.2019.10.035">[DOI]</a>.</p></li>
<li id="id9862"><p>Alessandra Biz, Scott Proulx, Zhiqing Xu, Kavya Siddartha, Alex Mulet Indrayanti, and Radhakrishnan Mahadevan. Systems biology based metabolic engineering for non-natural chemicals. <em>BIOTECHNOLOGY ADVANCES</em>, NOV 1 2019. <a class="reference external" href="https://doi.org/10.1016/j.biotechadv.2019.04.001">[DOI]</a>.</p></li>
<li id="id9863"><p>Zsolt Bodor, Szabolcs Lanyi, Beata Albert, Katalin Bodor, Aurelia Cristina Nechifor, and Ildiko Miklossy. Model driven analysis of the biosynthesis of 1,4-butanediol from renewable feedstocks in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>REVISTA DE CHIMIE</em>, 70(11):3808–3817, NOV 2019.</p></li>
<li id="id9864"><p>Cecilia Noecker, Hsuan-Chao Chiu, Colin P. McNally, and Elhanan Borenstein. Defining and evaluating microbial contributions to metabolite variation in microbiome-metabolome association studies. <em>MSYSTEMS</em>, NOV-DEC 2019. <a class="reference external" href="https://doi.org/10.1128/mSystems.00579-19">[DOI]</a>.</p></li>
<li id="id9865"><p>W. Pinzon, H. Vega, J. Gonzalez, and A. Pinzon. Mathematical framework behind the reconstruction and analysis of genome scale metabolic models. <em>ARCHIVES OF COMPUTATIONAL METHODS IN ENGINEERING</em>, 26(5):1593–1606, NOV 2019. <a class="reference external" href="https://doi.org/10.1007/s11831-018-9290-3">[DOI]</a>.</p></li>
<li id="id9866"><p>Quanli Liu, Tao Yu, Xiaowei Li, Yu Chen, Kate Campbell, Jens Nielsen, and Yun Chen. Rewiring carbon metabolism in yeast for high level production of aromatic chemicals. <em>NATURE COMMUNICATIONS</em>, OCT 31 2019. <a class="reference external" href="https://doi.org/10.1038/s41467-019-12961-5">[DOI]</a>.</p></li>
<li id="id9867"><p>Lazaro Molina, Ruggero La Rosa, Juan Nogales, and Fernando Rojo. Influence of the crc global regulator on substrate uptake rates and the distribution of metabolic fluxes in &lt;i&gt;pseudomonas putida&lt;/i&gt; kt2440 growing in a complete medium. <em>ENVIRONMENTAL MICROBIOLOGY</em>, 21(11):4446–4459, NOV 2019. <a class="reference external" href="https://doi.org/10.1111/1462-2920.14812">[DOI]</a>.</p></li>
<li id="id9869"><p>Emelyne Teo, Sudharshan Ravi, Diogo Barardo, Hyung-Seok Kim, Sheng Fong, Amaury Cazenave-Gassiot, Tsze Yin Tan, Jianhong Ching, Jean-Paul Kovalik, Markus R. Wenk, Rudiyanto Gunawan, Philip K. Moore, Barry Halliwell, Nicholas Tolwinski, and Jan Gruber. Metabolic stress is a primary pathogenic event in transgenic &lt;i&gt;caenorhabditis&lt;/i&gt; &lt;i&gt;elegans&lt;/i&gt; expressing pan-neuronal human amyloid beta. <em>ELIFE</em>, OCT 15 2019. <a class="reference external" href="https://doi.org/10.7554/eLife.50069">[DOI]</a>.</p></li>
<li id="id9871"><p>Maciek R. Antoniewicz. Synthetic methylotrophy: strategies to assimilate methanol for growth and chemicals production. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 59:165–174, OCT 2019. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2019.07.001">[DOI]</a>.</p></li>
<li id="id9872"><p>Zsolt Bodor, Lehel Tompos, Aurelia Cristina Nechifor, and Katalin Bodor. &lt;i&gt;in silico&lt;/i&gt; analysis of 1,4-butanediol heterologous pathway impact on &lt;i&gt;escherichia coli&lt;/i&gt; metabolism. <em>REVISTA DE CHIMIE</em>, 70(10):3448–3455, OCT 2019.</p></li>
<li id="id9873"><p>Josi Buerger, Luisa S. Gronenberg, Hans Jasper Genee, and Morten O. A. Sommer. Wiring cell growth to product formation. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 59:85–92, OCT 2019. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2019.02.014">[DOI]</a>.</p></li>
<li id="id9874"><p>Shaozhen Ding, Pengli Cai, Le Yuan, Yu Tian, Weizhong Tu, Dachuan Zhang, Xingxiang Cheng, Dandan Sun, Junni Chen, and Qian-Nan Hu. Cf-targeter: a rational biological cell factory targeting platform for biosynthetic target chemicals. <em>ACS SYNTHETIC BIOLOGY</em>, 8(10):2280–2286, OCT 2019. <a class="reference external" href="https://doi.org/10.1021/acssynbio.9b00070">[DOI]</a>.</p></li>
<li id="id9875"><p>Christopher P. Long and Maciek R. Antoniewicz. High-resolution&lt;sup&gt; 13&lt;/sup&gt;c metabolic flux analysis. <em>NATURE PROTOCOLS</em>, 14(10):2856–2877, OCT 2019. <a class="reference external" href="https://doi.org/10.1038/s41596-019-0204-0">[DOI]</a>.</p></li>
<li id="id9876"><p>Jelili Oyelade, Itunuoluwa Isewon, Olufemi Aromolaran, Efosa Uwoghiren, Titilope Dokunmu, Solomon Rotimi, Oluwadurotimi Aworunse, Olawole Obembe, and Ezekiel Adebiyi. Computational identification of metabolic pathways of &lt;i&gt;plasmodium falciparum&lt;/i&gt; using the k-shortest path algorithm. <em>INTERNATIONAL JOURNAL OF GENOMICS</em>, OCT 1 2019. <a class="reference external" href="https://doi.org/10.1155/2019/1750291">[DOI]</a>.</p></li>
<li id="id9877"><p>Lucia Trilla-Fuertes, Angelo Gamez-Pozo, Mariana Diaz-Almiron, Guillermo Prado-Vazquez, Andrea Zapater-Moros, Rocio Lopez-Vacas, Paolo Nanni, Pilar Zamora, Enrique Espinosa, and Juan Angel Fresno Vara. Computational metabolism modeling predicts risk of distant relapse-free survival in breast cancer patients. <em>FUTURE ONCOLOGY</em>, 15(30):3483–3490, OCT 2019. <a class="reference external" href="https://doi.org/10.2217/fon-2018-0698">[DOI]</a>.</p></li>
<li id="id9878"><p>Chien-Ting Li, Jacob Yelsky, Yiqun Chen, Cristal Zuniga, Richard Eng, Liqun Jiang, Alison Shapiro, Kai-Wen Huang, Karsten Zengler, and Michael J. Betenbaugh. Utilizing genome-scale models to optimize nutrient supply for sustained algal growth and lipid productivity. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, SEP 24 2019. <a class="reference external" href="https://doi.org/10.1038/s41540-019-0110-7">[DOI]</a>.</p></li>
<li id="id9879"><p>James Gilbert, Nicole Pearcy, Rupert Norman, Thomas Millat, Klaus Winzer, John King, Charlie Hodgman, Nigel Minton, and Jamie Twycross. Gsmodutils: a python based framework for test-driven genome scale metabolic model development. <em>BIOINFORMATICS</em>, 35(18):3397–3403, SEP 15 2019. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btz088">[DOI]</a>.</p></li>
<li id="id9880"><p>Helena A. Herrmann, Beth C. Dyson, Lucy Vass, Giles N. Johnson, and Jean-Marc Schwartz. Flux sampling is a powerful tool to study metabolism under changing environmental conditions. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, SEP 2 2019. <a class="reference external" href="https://doi.org/10.1038/s41540-019-0109-0">[DOI]</a>.</p></li>
<li id="id9881"><p>Christopher P. Long and Maciek R. Antoniewicz. Metabolic flux responses to deletion of 20 core enzymes reveal flexibility and limits of &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; metabolism. <em>METABOLIC ENGINEERING</em>, 55:249–257, SEP 2019. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2019.08.003">[DOI]</a>.</p></li>
<li id="id9882"><p>Maximilian Lularevic, Andrew J. Racher, Colin Jaques, and Alexandros Kiparissides. Improving the accuracy of flux balance analysis through the implementation of carbon availability constraints for intracellular reactions. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 116(9):2339–2352, SEP 2019. <a class="reference external" href="https://doi.org/10.1002/bit.27025">[DOI]</a>.</p></li>
<li id="id9883"><p>Kirubhakaran Puvendran and Guhan Jayaraman. Enhancement of acetyl-coa by acetate co-utilization in recombinant lactococcus lactis cultures enables the production of high molecular weight hyaluronic acid. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 103(17):6989–7001, SEP 2019. <a class="reference external" href="https://doi.org/10.1007/s00253-019-09987-6">[DOI]</a>.</p></li>
<li id="id9884"><p>Nathalie Poupin, Marie Tremblay-Franco, Aurelien Amiel, Cecile Canletz, Didier Remond, Laurent Debrauwer, Dominique Dardevet, Ines Thiele, Maike K. Aurich, Fabien Jourdan, Isabelle Savary-Auzeloux, and Sergio Polakof. Arterio-venous metabolomics exploration reveals major changes across liver and intestine in the obese yucatan minipig. <em>SCIENTIFIC REPORTS</em>, AUG 29 2019. <a class="reference external" href="https://doi.org/10.1038/s41598-019-48997-2">[DOI]</a>.</p></li>
<li id="id9885"><p>Meisam Yousefi, Sayed-Amir Marashi, Ali Sharifi-Zarchi, and Sara Taleahmad. The metabolic network model of primed/naive human embryonic stem cells underlines the importance of oxidation-reduction potential and tryptophan metabolism in primed pluripotency. <em>CELL AND BIOSCIENCE</em>, AUG 29 2019. <a class="reference external" href="https://doi.org/10.1186/s13578-019-0334-7">[DOI]</a>.</p></li>
<li id="id9886"><p>Yasir Suhail, Margo P. Cain, Kiran Vanaja, Paul A. Kurywchak, Andre Levchenko, Raghu Kalluri, and Kshitiz. Systems biology of cancer metastasis. <em>CELL SYSTEMS</em>, 9(2):109–127, AUG 28 2019. <a class="reference external" href="https://doi.org/10.1016/j.cels.2019.07.003">[DOI]</a>.</p></li>
<li id="id9887"><p>Yu Chen and Jens Nielsen. Energy metabolism controls phenotypes by protein efficiency and allocation. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 116(35):17592–17597, AUG 27 2019. <a class="reference external" href="https://doi.org/10.1073/pnas.1906569116">[DOI]</a>.</p></li>
<li id="id9888"><p>Jennifer Greene, James Daniell, Michael Kopke, Linda Broadbelt, and Keith E. J. Tyo. Kinetic ensemble model of gas fermenting &lt;i&gt;clostridium autoethanogenum&lt;/i&gt; for improved ethanol production. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, 148:46–56, AUG 15 2019. <a class="reference external" href="https://doi.org/10.1016/j.bej.2019.04.021">[DOI]</a>.</p></li>
<li id="id9889"><p>Kalaivani Paramasivan, Punil H. N. Kumar, and Sarma Mutturi. Systems-based saccharomyces cerevisiae strain design for improved squalene synthesis. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, 148:37–45, AUG 15 2019. <a class="reference external" href="https://doi.org/10.1016/j.bej.2019.04.025">[DOI]</a>.</p></li>
<li id="id9890"><p>Hongzhong Lu, Feiran Li, Benjamin J. Sanchez, Zhengming Zhu, Gang Li, Ivan Domenzain, Simonas Marcisauskas, Petre Mihail Anton, Dimitra Lappa, Christian Lieven, Moritz Emanuel Beber, Nikolaus Sonnenschein, Eduard J. Kerkhoven, and Jens Nielsen. A consensus &lt;i&gt;s&lt;/i&gt;. &lt;i&gt;cerevisiae&lt;/i&gt; metabolic model yeast8 and its ecosystem for comprehensively probing cellular metabolism. <em>NATURE COMMUNICATIONS</em>, AUG 8 2019. <a class="reference external" href="https://doi.org/10.1038/s41467-019-11581-3">[DOI]</a>.</p></li>
<li id="id9891"><p>Viridiana Olin-Sandoval, Jason Shu Lim Yu, Leonor Miller-Fleming, Mohammad Tauqeer Alam, Stephan Kamrad, Clara Correia-Melo, Robert Haas, Joanna Segal, David Alejandro Pena Navarro, Lucia Herrera-Dominguez, Oscar Mendez-Lucio, Jakob Vowinckel, Michael Muelleder, and Markus Ralser. Lysine harvesting is an antioxidant strategy and triggers underground polyamine metabolism. <em>NATURE</em>, 572(7768):249+, AUG 8 2019. <a class="reference external" href="https://doi.org/10.1038/s41586-019-1442-6">[DOI]</a>.</p></li>
<li id="id9892"><p>Mustafa Saifuddin, Jennifer M. Bhatnagar, Daniel Segre, and Adrien C. Finzi. Microbial carbon use efficiency predicted from genome-scale metabolic models. <em>NATURE COMMUNICATIONS</em>, AUG 8 2019. <a class="reference external" href="https://doi.org/10.1038/s41467-019-11488-z">[DOI]</a>.</p></li>
<li id="id9893"><p>Sebastian N. Mendoza, Brett G. Olivier, Douwe Molenaar, and Bas Teusink. A systematic assessment of current genome-scale metabolic reconstruction tools. <em>GENOME BIOLOGY</em>, AUG 7 2019. <a class="reference external" href="https://doi.org/10.1186/s13059-019-1769-1">[DOI]</a>.</p></li>
<li id="id9894"><p>Manish Kumar, Boyang Ji, Karsten Zengler, and Jens Nielsen. Modelling approaches for studying the microbiome. <em>NATURE MICROBIOLOGY</em>, 4(8):1253–1267, AUG 2019. <a class="reference external" href="https://doi.org/10.1038/s41564-019-0491-9">[DOI]</a>.</p></li>
<li id="id9895"><p>Romina Lasry Testa, Claudio Delpino, Vanina Estrada, and Soledad M. Diaz. In silico strategies to couple production of bioethanol with growth in cyanobacteria. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 116(8):2061–2073, AUG 2019. <a class="reference external" href="https://doi.org/10.1002/bit.26998">[DOI]</a>.</p></li>
<li id="id9896"><p>Karel Olavarria, Albert Fina, Mariana I. Velasco, Mark C. M. van Loosdrecht, and Sebastian Aljoscha Wahl. Metabolism of sucrose in a non-fermentative escherichia coli under oxygen limitation. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 103(15):6245–6256, AUG 2019. <a class="reference external" href="https://doi.org/10.1007/s00253-019-09909-6">[DOI]</a>.</p></li>
<li id="id9897"><p>Joel A. Paulson, Marc Martin-Casas, and Ali Mesbah. Fast uncertainty quantification for dynamic flux balance analysis using non-smooth polynomial chaos expansions. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2019. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1007308">[DOI]</a>.</p></li>
<li id="id9898"><p>Minsuk Kim, Beom Gi Park, Eun-Jung Kim, Joonwon Kim, and Byung-Gee Kim. In silico identification of metabolic engineering strategies for improved lipid production in yarrowia lipolytica by genome-scale metabolic modeling. <em>BIOTECHNOLOGY FOR BIOFUELS</em>, JUL 24 2019. <a class="reference external" href="https://doi.org/10.1186/s13068-019-1518-4">[DOI]</a>.</p></li>
<li id="id9899"><p>Markus Heinonen, Maria Osmala, Henrik Mannerstrom, Janne Wallenius, Samuel Kaski, Juho Rousu, and Harri Lahdesmaki. Bayesian metabolic flux analysis reveals intracellular flux couplings. <em>BIOINFORMATICS</em>, 35(14):I548–I557, JUL 15 2019. Biennial Joint Meeting of the 27th Annual Conference on Intelligent Systems for Molecular Biology (ISMB) / 18th European Conference on Computational Biology (ECCB), Basel, SWITZERLAND, JUL 21-25, 2019. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btz315">[DOI]</a>.</p></li>
<li id="id9900"><p>Han Yu and Rachael Hageman Blair. Integration of probabilistic regulatory networks into constraint-based models of metabolism with applications to alzheimer's disease. <em>BMC BIOINFORMATICS</em>, JUL 10 2019. <a class="reference external" href="https://doi.org/10.1186/s12859-019-2872-8">[DOI]</a>.</p></li>
<li id="id9901"><p>Priyanka Baloni, Vineet Sangar, James T. Yurkovich, Max Robinson, Scott Taylor, Christine M. Karbowski, Hisham K. Hamadeh, Yudong D. He, and Nathan D. Price. Genome-scale metabolic model of the rat liver predicts effects of diet restriction. <em>SCIENTIFIC REPORTS</em>, JUL 8 2019. <a class="reference external" href="https://doi.org/10.1038/s41598-019-46245-1">[DOI]</a>.</p></li>
<li id="id9902"><p>Brendan Fu-Long Sieow, Toni Juhani Nurminen, Hua Ling, and Matthew Wook Chang. Meta-omics- and metabolic modeling-assisted deciphering of human microbiota metabolism. <em>BIOTECHNOLOGY JOURNAL</em>, SEP 2019. <a class="reference external" href="https://doi.org/10.1002/biot.201800445">[DOI]</a>.</p></li>
<li id="id9903"><p>Shoval Lagziel, Won Dong Lee, and Tomer Shlomi. Studying metabolic flux adaptations in cancer through integrated experimental-computational approaches. <em>BMC BIOLOGY</em>, JUL 4 2019. <a class="reference external" href="https://doi.org/10.1186/s12915-019-0669-x">[DOI]</a>.</p></li>
<li id="id9904"><p>Gaoyang Li, Huansheng Cao, and Ying Xu. Structural and functional analyses of microbial metabolic networks reveal novel insights into genome-scale metabolic fluxes. <em>BRIEFINGS IN BIOINFORMATICS</em>, 20(4):1590–1603, JUL 2019. <a class="reference external" href="https://doi.org/10.1093/bib/bby022">[DOI]</a>.</p></li>
<li id="id9905"><p>Yusuke Minato, Daryl M. Gohl, Joshua M. Thiede, Jeremy M. Chacon, William R. Harcombe, Fumito Maruyama, and Anthony D. Baughn. Genomewide assessment of &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; conditionally essential metabolic pathways. <em>MSYSTEMS</em>, JUL-AUG 2019. <a class="reference external" href="https://doi.org/10.1128/mSystems.00070-19">[DOI]</a>.</p></li>
<li id="id9906"><p>Lazaro Molina, Ruggero La Rosa, Juan Nogales, and Fernando Rojo. &lt;i&gt;pseudomonas putida&lt;/i&gt; kt2440 metabolism undergoes sequential modifications during exponential growth in a complete medium as compounds are gradually consumed. <em>ENVIRONMENTAL MICROBIOLOGY</em>, 21(7):2375–2390, JUL 2019. <a class="reference external" href="https://doi.org/10.1111/1462-2920.14622">[DOI]</a>.</p></li>
<li id="id9907"><p>Stephane Pinhal, Delphine Ropers, Johannes Geiselmann, and Hidde de Jong. Acetate metabolism and the inhibition of bacterial growth by acetate. <em>JOURNAL OF BACTERIOLOGY</em>, JUL 2019. <a class="reference external" href="https://doi.org/10.1128/JB.00147-19">[DOI]</a>.</p></li>
<li id="id9908"><p>Guido Zampieri, Supreeta Vijayakumar, Elisabeth Yaneske, and Claudio Angione. Machine and deep learning meet genome-scale metabolic modeling. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUL 2019. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1007084">[DOI]</a>.</p></li>
<li id="id9909"><p>Lucia Trilla-Fuertes, Angelo Gamez-Pozo, Guillermo Prado-Vazquez, Andrea Zapater-Moros, Mariana Diaz-Almiron, Jorge M. Arevalillo, Maria Ferrer-Gomez, Hilario Navarro, Paloma Main, Enrique Espinosa, Alvaro Pinto, and Juan Angel Fresno Vara. Biological molecular layer classification of muscle-invasive bladder cancer opens new treatment opportunities. <em>BMC CANCER</em>, JUN 28 2019. <a class="reference external" href="https://doi.org/10.1186/s12885-019-5858-z">[DOI]</a>.</p></li>
<li id="id9910"><p>Ankit Gupta, Ahmad Ahmad, Dipesh Chothwe, Midhun K. Madhu, Shireesh Srivastava, and Vineet K. Sharma. Genome-scale metabolic reconstruction and metabolic versatility of an obligate methanotroph &lt;i&gt;methylococcus capsulatus&lt;/i&gt; str. bath. <em>PEERJ</em>, JUN 14 2019. <a class="reference external" href="https://doi.org/10.7717/peerj.6685">[DOI]</a>.</p></li>
<li id="id9911"><p>David B. Bernstein, Floyd E. Dewhirst, and Daniel Segre. Metabolic network percolation quantifies biosynthetic capabilities across the human oral microbiome. <em>ELIFE</em>, JUN 13 2019. <a class="reference external" href="https://doi.org/10.7554/eLife.39733">[DOI]</a>.</p></li>
<li id="id9912"><p>Eun-Youn Kim, Daniel Ashlock, and Sung Ho Yoon. Identification of critical connectors in the directed reaction-centric graphs of microbial metabolic networks. <em>BMC BIOINFORMATICS</em>, JUN 13 2019. <a class="reference external" href="https://doi.org/10.1186/s12859-019-2897-z">[DOI]</a>.</p></li>
<li id="id9913"><p>Beatriz Penalver Bernabe, Ines Thiele, Eugene Galdones, Anaar Siletz, Sriram Chandrasekaran, Teresa K. Woodruff, Linda J. Broadbelt, and Lonnie D. Shea. Dynamic genome-scale cell-specific metabolic models reveal novel inter-cellular and intra-cellular metabolic communications during ovarian follicle development. <em>BMC BIOINFORMATICS</em>, JUN 10 2019. <a class="reference external" href="https://doi.org/10.1186/s12859-019-2825-2">[DOI]</a>.</p></li>
<li id="id9914"><p>Boaz Styr, Nir Gonen, Daniel Zarhin, Antonella Ruggiero, Refaela Atsmon, Neta Gazit, Gabriella Braun, Samuel Frere, Irena Vertkin, Ilana Shapira, Michal Harel, Leore R. Heim, Maxim Katsenelson, Ohad Rechnitz, Saja Fadila, Dori Derdikman, Moran Rubinstein, Tamar Geiger, Eytan Ruppin, and Inna Slutsky. Mitochondrial regulation of the hippocampal firing rate set point and seizure susceptibility. <em>NEURON</em>, 102(5):1009+, JUN 5 2019. <a class="reference external" href="https://doi.org/10.1016/j.neuron.2019.03.045">[DOI]</a>.</p></li>
<li id="id9915"><p>Abinaya Badri, Karthik Raman, and Guhan Jayaraman. Uncovering novel pathways for enhancing hyaluronan synthesis in recombinant lactococcus lactis: genome-scale metabolic modeling and experimental validation. <em>PROCESSES</em>, JUN 2019. <a class="reference external" href="https://doi.org/10.3390/pr7060343">[DOI]</a>.</p></li>
<li id="id9916"><p>Su H. Chu, Mengna Huang, Rachel S. Kelly, Elisa Benedetti, Jalal K. Siddiqui, Oana A. Zeleznik, Alexandre Pereira, David Herrington, Craig E. Wheelock, Jan Krumsiek, Michael McGeachie, Steven C. Moore, Peter Kraft, Ewy Mathe, Jessica Lasky-Su, and Consortium Metab Studies Stat Work. Integration of metabolomic and other omics data in population-based study designs: an epidemiological perspective. <em>METABOLITES</em>, JUN 2019. <a class="reference external" href="https://doi.org/10.3390/metabo9060117">[DOI]</a>.</p></li>
<li id="id9917"><p>Leah Guthrie and Libusha Kelly. Bringing microbiome-drug interaction research into the clinic. <em>EBIOMEDICINE</em>, 44:708–715, JUN 2019. <a class="reference external" href="https://doi.org/10.1016/j.ebiom.2019.05.009">[DOI]</a>.</p></li>
<li id="id9918"><p>Reza Mohammadi, Javad Zahiri, and Mohammad Javad Niroomand. Imet: a graphical user interface software tool to merge metabolic networks. <em>HELIYON</em>, JUN 2019. <a class="reference external" href="https://doi.org/10.1016/j.heliyon.2019.e01766">[DOI]</a>.</p></li>
<li id="id9919"><p>Junyoung O. Park, Nian Liu, Kara M. Holinski, David F. Emerson, Kangjian Qiao, Benjamin M. Woolston, Jingyang Xu, Zbigniew Lazar, M. Ahsanul Islam, Charles Vidoudez, Peter R. Girguis, and Gregory Stephanopoulos. Synergistic substrate cofeeding stimulates reductive metabolism. <em>NATURE METABOLISM</em>, 1(6):643–651, JUN 2019. <a class="reference external" href="https://doi.org/10.1038/s42255-019-0077-0">[DOI]</a>.</p></li>
<li id="id9920"><p>Hong Zeng and Aidong Yang. Quantification of proteomic and metabolic burdens predicts growth retardation and overflow metabolism in recombinant &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 116(6):1484–1495, JUN 2019. <a class="reference external" href="https://doi.org/10.1002/bit.26943">[DOI]</a>.</p></li>
<li id="id9921"><p>Jason H. Yang, Sarah N. Wright, Meagan Hamblin, Douglas McCloskey, Miguel A. Alcantar, Lars Schrubbers, Allison J. Lopatkin, Sangeeta Satish, Amir Nili, Bernhard O. Palsson, Graham C. Walker, and James J. Collins. A white-box machine learning approach for revealing antibiotic mechanisms of action. <em>CELL</em>, 177(6):1649+, MAY 30 2019. <a class="reference external" href="https://doi.org/10.1016/j.cell.2019.04.016">[DOI]</a>.</p></li>
<li id="id9922"><p>Matias Orellana-Saez, Nicolas Pacheco, Jose Costa, I, Katterinne N. Mendez, Matthieu J. Miossec, Claudio Meneses, Eduardo Castro-Nallar, Andres E. Marcoleta, and Ignacio Poblete-Castro. In-depth genomic and phenotypic characterization of the antarctic psychrotolerant strain &lt;i&gt;pseudomonas&lt;/i&gt; sp. mpc6 reveals unique metabolic features, plasticity, and biotechnological potential. <em>FRONTIERS IN MICROBIOLOGY</em>, MAY 24 2019. <a class="reference external" href="https://doi.org/10.3389/fmicb.2019.01154">[DOI]</a>.</p></li>
<li id="id9923"><p>Almut Heinken, Dmitry A. Raycheev, Federico Baldini, Laurent Heirendt, Ronan M. T. Fleming, and Ines Thiele. Systematic assessment of secondary bile acid metabolism in gut microbes reveals distinct metabolic capabilities in inflammatory bowel disease. <em>MICROBIOME</em>, MAY 15 2019. <a class="reference external" href="https://doi.org/10.1186/s40168-019-0689-3">[DOI]</a>.</p></li>
<li id="id9924"><p>Lucia Trilla-Fuertes, Angelo Gamez-Pozo, Guillermo Prado-Vazquez, Andrea Zapater-Moros, Mariana Diaz-Almiron, Claudia Fortes, Maria Ferrer-Gomez, Rocio Lopez-Vacas, Veronica Parra Blanco, Ivan Marquez-Rodas, Ainara Soria, Juan Angel Fresno Vara, and Enrique Espinosa. Melanoma proteomics suggests functional differences related to mutational status. <em>SCIENTIFIC REPORTS</em>, MAY 10 2019. <a class="reference external" href="https://doi.org/10.1038/s41598-019-43512-z">[DOI]</a>.</p></li>
<li id="id9925"><p>Aitor Blanco-Miguez, Florentino Fdez-Riverola, Borja Sanchez, and Analia Lourenco. Resources and tools for the high-throughput, multi-omic study of intestinal microbiota. <em>BRIEFINGS IN BIOINFORMATICS</em>, 20(3):1032–1056, MAY 2019. <a class="reference external" href="https://doi.org/10.1093/bib/bbx156">[DOI]</a>.</p></li>
<li id="id9926"><p>Porntip Chiewchankaset, Saowalak Kalapanulak, and Treenut Saithong. Extended utilization of constraint-based metabolic model in a long-growing crop. <em>PROCESSES</em>, MAY 2019. <a class="reference external" href="https://doi.org/10.3390/pr7050259">[DOI]</a>.</p></li>
<li id="id9927"><p>Kentaro Kamata, Yoshihiro Toya, and Hiroshi Shimizu. Effect of precise control of flux ratio between the glycolytic pathways on mevalonate production in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 116(5):1080–1088, MAY 2019. <a class="reference external" href="https://doi.org/10.1002/bit.26923">[DOI]</a>.</p></li>
<li id="id9928"><p>Rui Pereira, Paulo Vilaca, Paulo Maia, Jens Nielsen, and Isabel Rocha. Turnover dependent phenotypic simulation: a quantitative constraint-based simulation method that accommodates all main strain design strategies. <em>ACS SYNTHETIC BIOLOGY</em>, 8(5):976–988, MAY 2019. <a class="reference external" href="https://doi.org/10.1021/acssynbio.8b00248">[DOI]</a>.</p></li>
<li id="id9929"><p>Kacy Greenhalgh, Javier Ramiro-Garcia, Almut Heinken, Pit Ullmann, Tamara Bintener, Maria Pires Pacheco, Joanna Baginska, Pranjul Shah, Audrey Frachet, Rashi Halder, Joelle V. Fritz, Thomas Sauter, Ines Thiele, Serge Haan, Elisabeth Letellier, and Paul Wilmes. Integrated &lt;i&gt;in vitro&lt;/i&gt; and &lt;i&gt;in silico&lt;/i&gt; modeling delineates the molecular effects of a synbiotic regimen on colorectal-cancer-derived cells. <em>CELL REPORTS</em>, 27(5):1621+, APR 30 2019. <a class="reference external" href="https://doi.org/10.1016/j.celrep.2019.04.001">[DOI]</a>.</p></li>
<li id="id9930"><p>Nicolas Sompairac, Jennifer Modamio, Emmanuel Barillot, Ronan M. T. Fleming, Andrei Zinovyev, and Inna Kuperstein. Metabolic and signalling network maps integration: application to cross-talk studies and omics data analysis in cancer. <em>BMC BIOINFORMATICS</em>, APR 18 2019. 17th International Workshop on Network Tools and Applications in Biology (NETTAB) - Methods, Tools and Platforms for Personalized Medicine in the Big Data Era, Palermo, ITALY, OCT 16-18, 2017. <a class="reference external" href="https://doi.org/10.1186/s12859-019-2682-z">[DOI]</a>.</p></li>
<li id="id9931"><p>Ana M. Untaroiu, Maureen A. Carey, Jennifer L. Guler, and Jason A. Papin. Leveraging the effects of chloroquine on resistant malaria parasites for combination therapies. <em>BMC BIOINFORMATICS</em>, APR 15 2019. <a class="reference external" href="https://doi.org/10.1186/s12859-019-2756-y">[DOI]</a>.</p></li>
<li id="id9932"><p>Anna S. Blazier and Jason A. Papin. Reconciling high-throughput gene essentiality data with metabolic network reconstructions. <em>PLOS COMPUTATIONAL BIOLOGY</em>, APR 2019. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1006507">[DOI]</a>.</p></li>
<li id="id9933"><p>Oveis Jamialahmadi, Sameereh Hashemi-Najafabadi, Ehsan Motamedian, Stefano Romeo, and Fatemeh Bagheri. A benchmark-driven approach to reconstruct metabolic networks for studying cancer metabolism. <em>PLOS COMPUTATIONAL BIOLOGY</em>, APR 2019. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1006936">[DOI]</a>.</p></li>
<li id="id9934"><p>Emrah Ozcan, S. Selvin Selvi, Emrah Nikerel, Bas Teusink, Ebru Toksoy Oner, and Tunahan Cakir. A genome-scale metabolic network of the aroma bacterium leuconostoc mesenteroides subsp. cremoris. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 103(7):3153–3165, APR 2019. <a class="reference external" href="https://doi.org/10.1007/s00253-019-09630-4">[DOI]</a>.</p></li>
<li id="id9935"><p>Sabina Zoledowska, Luana Presta, Marco Fondi, Francesca Decorosi, Luciana Giovannetti, Alessio Mengoni, and Ewa Lojkowska. Metabolic modeling of pectobacterium parmentieri scc3193 provides insights into metabolic pathways of plant pathogenic bacteria. <em>MICROORGANISMS</em>, APR 2019. <a class="reference external" href="https://doi.org/10.3390/microorganisms7040101">[DOI]</a>.</p></li>
<li id="id9936"><p>Songul Yasar Yildiz, Emrah Nikerel, and Ebru Toksoy Oner. Genome-scale metabolic model of a microbial cell factory (&lt;i&gt;brevibacillus thermoruber&lt;/i&gt; 423) with multi-industry potentials for exopolysaccharide production. <em>OMICS-A JOURNAL OF INTEGRATIVE BIOLOGY</em>, 23(4):237–246, APR 1 2019. <a class="reference external" href="https://doi.org/10.1089/omi.2019.0028">[DOI]</a>.</p></li>
<li id="id9937"><p>Andreas Schwentner, Andre Feith, Eugenia Muench, Judith Stiefelmaier, Ira Lauer, Lorenzo Favilli, Christoph Massner, Johannes Oehrlein, Bastian Grund, Andrea Hueser, Ralf Takors, and Bastian Blombach. Modular systems metabolic engineering enables balancing of relevant pathways for l-histidine production with &lt;i&gt;corynebacterium glutamicum&lt;/i&gt;. <em>BIOTECHNOLOGY FOR BIOFUELS</em>, MAR 25 2019. <a class="reference external" href="https://doi.org/10.1186/s13068-019-1410-2">[DOI]</a>.</p></li>
<li id="id9938"><p>Yanfen Fu, Lian He, Jennifer Reeve, David A. C. Beck, and Mary E. Lidstrom. Core metabolism shifts during growth on methanol versus methane in the methanotroph &lt;i&gt;methylomicrobium buryatense&lt;/i&gt; 5gb1. <em>MBIO</em>, MAR-APR 2019. <a class="reference external" href="https://doi.org/10.1128/mBio.00406-19">[DOI]</a>.</p></li>
<li id="id9939"><p>Tuure Hameri, Georgios Fengos, Meric Ataman, Ljubisa Miskovic, and Vassily Hatzimanikatis. Kinetic models of metabolism that consider alternative steady-state solutions of intracellular fluxes and concentrations. <em>METABOLIC ENGINEERING</em>, 52:29–41, MAR 2019. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2018.10.005">[DOI]</a>.</p></li>
<li id="id9940"><p>Laurent Heirendt, Sylvain Arreckx, Thomas Pfau, Sebastian N. Mendoza, Anne Richelle, Almut Heinken, Hulda S. Haraldsdottir, Jacek Wachowiak, Sarah M. Keating, Vanja Vlasov, Stefania Magnusdottir, Chiam Yu Ng, German Preciat, Alise Zagare, Siu H. J. Chan, Maike K. Aurich, Catherine M. Clancy, Jennifer Modamio, John T. Sauls, Alberto Noronha, Aarash Bordbar, Benjamin Cousins, Diana C. El Assal, Luis V. Valcarcel, Inigo Apaolaza, Susan Ghaderi, Masoud Ahookhosh, Marouen Ben Guebila, Andrejs Kostromins, Nicolas Sompairac, Hoai M. Le, Ding Ma, Yuekai Sun, Lin Wang, James T. Yurkovich, Miguel A. P. Oliveira, Phan T. Vuong, Lemmer P. El Assal, Inna Kuperstein, Andrei Zinovyev, H. Scott Hinton, William A. Bryant, Francisco J. Aragon Artacho, Francisco J. Planes, Egils Stalidzans, Alejandro Maass, Santosh Vempala, Michael Hucka, Michael A. Saunders, Costas D. Maranas, Nathan E. Lewis, Thomas Sauter, Bernhard O. Palsson, Ines Thiele, and Ronan M. T. Fleming. Creation and analysis of biochemical constraint-based models using the cobra toolbox v.3.0. <em>NATURE PROTOCOLS</em>, 14(3):639–702, MAR 2019. <a class="reference external" href="https://doi.org/10.1038/s41596-018-0098-2">[DOI]</a>.</p></li>
<li id="id9941"><p>Ann Marie Reinhold, Geoffrey C. Poole, Clemente Izurieta, Ashley M. Helton, and Emily S. Bernhardt. Constraint-based simulation of multiple interactive elemental cycles in biogeochemical systems. <em>ECOLOGICAL INFORMATICS</em>, 50:102–121, MAR 2019. <a class="reference external" href="https://doi.org/10.1016/j.ecoinf.2018.12.008">[DOI]</a>.</p></li>
<li id="id9942"><p>Meghan Thommes, Taiyao Wang, Qi Zhao, Ioannis C. Paschalidis, and Daniel Segre. Designing metabolic division of labor in microbial communities. <em>MSYSTEMS</em>, MAR-APR 2019. <a class="reference external" href="https://doi.org/10.1128/mSystems.00263-18">[DOI]</a>.</p></li>
<li id="id9943"><p>Porntip Chiewchankaset, Wanatsanan Siriwat, Malinee Suksangpanomrung, Opas Boonseng, Asawin Meechai, Morakot Tanticharoen, Saowalak Kalapanulak, and Treenut Saithong. Understanding carbon utilization routes between high and low starch-producing cultivars of cassava through flux balance analysis. <em>SCIENTIFIC REPORTS</em>, FEB 27 2019. <a class="reference external" href="https://doi.org/10.1038/s41598-019-39920-w">[DOI]</a>.</p></li>
<li id="id9944"><p>Miha Kastelic, Drejc Kopac, Uros Novak, and Blaz Likozar. Dynamic metabolic network modeling of mammalian chinese hamster ovary (cho) cell cultures with continuous phase kinetics transitions. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, 142:124–134, FEB 15 2019. <a class="reference external" href="https://doi.org/10.1016/j.bej.2018.11.015">[DOI]</a>.</p></li>
<li id="id9945"><p>Inigo Apaolaza, Luis Vitores Valcarcel, and Francisco J. Planes. &lt;i&gt;gmcs&lt;/i&gt;: fast computation of genetic minimal cut sets in large networks. <em>BIOINFORMATICS</em>, 35(3):535–537, FEB 1 2019. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/bty656">[DOI]</a>.</p></li>
<li id="id9946"><p>Chiara Damiani, Davide Maspero, Marzia Di Filippo, Riccardo Colombo, Dario Pescini, Alex Graudenzi, Hans Victor Westerhoff, Lilia Alberghina, Marco Vanoni, and Giancarlo Mauri. Integration of single-cell rna-seq data into population models to characterize cancer metabolism. <em>PLOS COMPUTATIONAL BIOLOGY</em>, FEB 2019. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1006733">[DOI]</a>.</p></li>
<li id="id9947"><p>Kristopher D. Rawls, Bonnie Dougherty, V, Edik M. Blais, Ethan Stancliffe, Glynis L. Kolling, Kalyan Vinnakota, Venkat R. Pannala, Anders Wallqvist, and Jason A. Papin. A simplified metabolic network reconstruction to promote understanding and development of flux balance analysis tools. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, 105:64–71, FEB 2019. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2018.12.010">[DOI]</a>.</p></li>
<li id="id9948"><p>Partho Sen and Matej Oresic. Metabolic modeling of human gut microbiota on a genome scale: an overview. <em>METABOLITES</em>, FEB 2019. <a class="reference external" href="https://doi.org/10.3390/metabo9020022">[DOI]</a>.</p></li>
<li id="id9949"><p>Agnieszka B. Wegrzyn, Sarah Stolle, Rienk A. Rienksma, Vitor A. P. Martins dos Santos, Barbara M. Bakker, and Maria Suarez-Diez. Cofactors revisited - predicting the impact of flavoprotein-related diseases on a genome scale. <em>BIOCHIMICA ET BIOPHYSICA ACTA-MOLECULAR BASIS OF DISEASE</em>, 1865(2):360–370, FEB 1 2019. <a class="reference external" href="https://doi.org/10.1016/j.bbadis.2018.10.021">[DOI]</a>.</p></li>
<li id="id9950"><p>Xihui Xu, Raphy Zarecki, Shlomit Medina, Shany Ofaim, Xiaowei Liu, Chen Chen, Shunli Hu, Dan Brom, Daniella Gat, Seema Porob, Hanan Eizenberg, Zeev Ronen, Jiandong Jiang, and Shiri Freilich. Modeling microbial communities from atrazine contaminated soils promotes the development of biostimulation solutions. <em>ISME JOURNAL</em>, 13(2):494–508, FEB 2019. <a class="reference external" href="https://doi.org/10.1038/s41396-018-0288-5">[DOI]</a>.</p></li>
<li id="id9951"><p>Markus Janasch, Johannes Asplund-Samuelsson, Ralf Steuer, and Elton P. Hudson. Kinetic modeling of the calvin cycle identifies flux control and stable metabolomes in &lt;i&gt;synechocystis&lt;/i&gt; carbon fixation. <em>JOURNAL OF EXPERIMENTAL BOTANY</em>, 70(3):973–983, JAN 30 2019. <a class="reference external" href="https://doi.org/10.1093/jxb/ery382">[DOI]</a>.</p></li>
<li id="id9952"><p>Nikolay Martyushenko and Eivind Almaas. Modelexplorer - software for visual inspection and inconsistency correction of genome-scale metabolic reconstructions. <em>BMC BIOINFORMATICS</em>, JAN 28 2019. <a class="reference external" href="https://doi.org/10.1186/s12859-019-2615-x">[DOI]</a>.</p></li>
<li id="id9953"><p>Mahsa Mekanik, Ehsan Motamedian, Reza Fotovat, and Vahab Jafarian. Reconstruction of a genome-scale metabolic model for &lt;i&gt;auxenochlorella protothecoides&lt;/i&gt; to study hydrogen production under anaerobiosis using multiple optimal solutions. <em>INTERNATIONAL JOURNAL OF HYDROGEN ENERGY</em>, 44(5):2580–2591, JAN 28 2019. <a class="reference external" href="https://doi.org/10.1016/j.ijhydene.2018.12.049">[DOI]</a>.</p></li>
<li id="id9954"><p>Laura J. Dunphy, Phillip Yen, and Jason A. Papin. Integrated experimental and computational analyses reveal differential metabolic functionality in antibiotic-resistant &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt;. <em>CELL SYSTEMS</em>, 8(1):3+, JAN 23 2019. <a class="reference external" href="https://doi.org/10.1016/j.cels.2018.12.002">[DOI]</a>.</p></li>
<li id="id9955"><p>Carolina A. Contador, Vida Rodriguez, Barbara A. Andrews, and Juan A. Asenjo. Use of genome-scale models to get new insights into the marine actinomycete genus &lt;i&gt;salinispora&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, JAN 21 2019. <a class="reference external" href="https://doi.org/10.1186/s12918-019-0683-1">[DOI]</a>.</p></li>
<li id="id9956"><p>Hong Zeng and Aidong Yang. Modelling overflow metabolism in &lt;i&gt;escherichia coli&lt;/i&gt; with flux balance analysis incorporating differential proteomic efficiencies of energy pathways. <em>BMC SYSTEMS BIOLOGY</em>, JAN 10 2019. <a class="reference external" href="https://doi.org/10.1186/s12918-018-0677-4">[DOI]</a>.</p></li>
<li id="id9957"><p>Ilaria Massaiu, Lorenzo Pasotti, Nikolaus Sonnenschein, Erlinda Rama, Matteo Cavaletti, Paolo Magni, Cinzia Calvio, and Markus J. Herrgard. Integration of enzymatic data in &lt;i&gt;bacillus subtilis&lt;/i&gt; genome-scale metabolic model improves phenotype predictions and enables in silico design of poly–glutamic acid production strains. <em>MICROBIAL CELL FACTORIES</em>, JAN 9 2019. <a class="reference external" href="https://doi.org/10.1186/s12934-018-1052-2">[DOI]</a>.</p></li>
<li id="id9958"><p>Alan R. Pacheco, Mauricio Moel, and Daniel Segre. Costless metabolic secretions as drivers of interspecies interactions in microbial ecosystems. <em>NATURE COMMUNICATIONS</em>, JAN 9 2019. <a class="reference external" href="https://doi.org/10.1038/s41467-018-07946-9">[DOI]</a>.</p></li>
<li id="id9959"><p>Deepanwita Banerjee and Anu Raghunathan. Constraints-based analysis identifies nad&lt;sup&gt;+&lt;/sup&gt; recycling through metabolic reprogramming in antibiotic resistant &lt;i&gt;chromobacterium violaceum&lt;/i&gt;. <em>PLOS ONE</em>, JAN 4 2019. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0210008">[DOI]</a>.</p></li>
<li id="id9960"><p>Claudio Angione. Human systems biology and metabolic modelling: a reviewfrom disease metabolism to precision medicine. <em>BIOMED RESEARCH INTERNATIONAL</em>, 2019. <a class="reference external" href="https://doi.org/10.1155/2019/8304260">[DOI]</a>.</p></li>
<li id="id9961"><p>George C. diCenzo, Alessio Mengoni, and Marco Fondi. Tn-core: a toolbox for integrating tn-seq gene essentiality data and constraint-based metabolic modeling. <em>ACS SYNTHETIC BIOLOGY</em>, 8(1):158–169, JAN 2019. <a class="reference external" href="https://doi.org/10.1021/acssynbio.8b00432">[DOI]</a>.</p></li>
<li id="id9962"><p>George C. diCenzo, Maryam Zamani, Alice Checcucci, Marco Fondi, Joel S. Griffitts, Turlough M. Finan, and Alessio Mengoni. Multidisciplinary approaches for studying rhizobium-legume symbioses. <em>CANADIAN JOURNAL OF MICROBIOLOGY</em>, 65(1):1–33, JAN 2019. <a class="reference external" href="https://doi.org/10.1139/cjm-2018-0377">[DOI]</a>.</p></li>
<li id="id9963"><p>Clemence Frioux, Torsten Schaub, Sebastian Schellhorn, Anne Siegel, and Philipp Wanko. Technical note hybrid metabolic network completion. <em>THEORY AND PRACTICE OF LOGIC PROGRAMMING</em>, 19(1):83–108, JAN 2019. <a class="reference external" href="https://doi.org/10.1017/S1471068418000455">[DOI]</a>.</p></li>
<li id="id9964"><p>Sergio Garcia and Cong T. Trinh. Multiobjective strain design: a framework for modular cell engineering. <em>METABOLIC ENGINEERING</em>, 51:110–120, JAN 2019. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2018.09.003">[DOI]</a>.</p></li>
<li id="id9965"><p>Anika Kueken, Jeanne M. O. Eloundou-Mbebi, Georg Basler, and Zoran Nikoloski. Cellular determinants of metabolite concentration ranges. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JAN 2019. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1006687">[DOI]</a>.</p></li>
<li id="id9966"><p>Swagatika Sahoo, Ranjith Kumar Ravi Kumar, Brandon Nicolay, Omkar Mohite, Karthikeyan Sivaraman, Vikas Khetan, Pukhraj Rishi, Suganeswari Ganesan, Krishnakumar Subramanyan, Karthik Raman, Wayne Miles, and Sailaja V. Elchuri. Metabolite systems profiling identifies exploitable weaknesses in retinoblastoma. <em>FEBS LETTERS</em>, 593(1):23–41, JAN 2019. <a class="reference external" href="https://doi.org/10.1002/1873-3468.13294">[DOI]</a>.</p></li>
<li id="id9967"><p>Pierre Salvy, Georgios Fengos, Meric Ataman, Thomas Pathier, Keng C. Soh, and Vassily Hatzimanikatis. Pytfa and mattfa: a python package and a matlab toolbox for thermodynamics-based flux analysis. <em>BIOINFORMATICS</em>, 35(1):167–169, JAN 1 2019. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/bty499">[DOI]</a>.</p></li>
<li id="id9968"><p>Yu Zhang, Yun Zhang, Xiuling Shang, Bo Wang, Qitiao Hu, Shuwen Liu, and Tingyi Wen. Reconstruction of tricarboxylic acid cycle in &lt;i&gt;corynebacterium glutamicum&lt;/i&gt; with a genome-scale metabolic network model for &lt;i&gt;trans&lt;/i&gt;-4-hydroxyproline production. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 116(1):99–109, JAN 2019. <a class="reference external" href="https://doi.org/10.1002/bit.26818">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-6" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-6" name="0-6" role="tabpanel" tabindex="0"><div class="docutils container" id="id10921">
<ol class="arabic simple" start="1">
<li id="id11789"><p>Ailin Liu, Carolina A. Contador, Kejing Fan, and Hon-Ming Lam. Interaction and regulation of carbon, nitrogen, and phosphorus metabolisms in root nodules of legumes. <em>FRONTIERS IN PLANT SCIENCE</em>, DEC 18 2018. <a class="reference external" href="https://doi.org/10.3389/fpls.2018.01860">[DOI]</a>.</p></li>
<li id="id11790"><p>Annalisa Occhipinti, Filmon Eyassu, Thahira J. Rahman, Pattanathu K. S. M. Rahman, and Claudio Angione. In silico engineering of &lt;i&gt;pseudomonas&lt;/i&gt; metabolism reveals new biomarkers for increased biosurfactant production. <em>PEERJ</em>, DEC 17 2018. <a class="reference external" href="https://doi.org/10.7717/peerj.6046">[DOI]</a>.</p></li>
<li id="id11791"><p>Naveen Venayak, Axel von Kamp, Steffen Klamt, and Radhakrishnan Mahadevan. Move identifies metabolic valves to switch between phenotypic states. <em>NATURE COMMUNICATIONS</em>, DEC 14 2018. <a class="reference external" href="https://doi.org/10.1038/s41467-018-07719-4">[DOI]</a>.</p></li>
<li id="id11792"><p>Kelly Botero, Silvia Restrepo, and Andres Pinzon. A genome-scale metabolic model of potato late blight suggests a photosynthesis suppression mechanism. <em>BMC GENOMICS</em>, DEC 11 2018. 4th Colombian Congress on Bioinformatics and Computational Biology / 8th SolBio International Conference on Bioinformatics, Santiago de Cali, COLOMBIA, SEP 13-15, 2017. <a class="reference external" href="https://doi.org/10.1186/s12864-018-5192-x">[DOI]</a>.</p></li>
<li id="id11793"><p>Zhongbao Ma, Chao Ye, Weiwei Deng, Mengmeng Xu, Qiong Wang, Gaoqiang Liu, Feng Wang, Liming Liu, Zhenghong Xu, Guiyang Shi, and Zhongyang Ding. Reconstruction and analysis of a genome-scale metabolic model of &lt;i&gt;ganoderma lucidum&lt;/i&gt; for improved extracellular polysaccharide production. <em>FRONTIERS IN MICROBIOLOGY</em>, DEC 11 2018. <a class="reference external" href="https://doi.org/10.3389/fmicb.2018.03076">[DOI]</a>.</p></li>
<li id="id11794"><p>Emil Karlsen, Christian Schulz, and Eivind Almaas. Automated generation of genome-scale metabolic draft reconstructions based on kegg. <em>BMC BIOINFORMATICS</em>, DEC 4 2018. <a class="reference external" href="https://doi.org/10.1186/s12859-018-2472-z">[DOI]</a>.</p></li>
<li id="id11795"><p>Ayca Cankorur-Cetinkaya, Nathalie Narraidoo, Ceyda Kasavi, Nigel K. H. Slater, David B. Archer, and Stephen G. Oliver. Process development for the continuous production of heterologous proteins by the industrial yeast, &lt;i&gt;komagataella phaffii&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 115(12):2962–2973, DEC 2018. <a class="reference external" href="https://doi.org/10.1002/bit.26846">[DOI]</a>.</p></li>
<li id="id11796"><p>Amornpan Klanchui, Sudarat Dulsawat, Kullapat Chaloemngam, Supapon Cheevadhanarak, Peerada Prommeenate, and Asawin Meechai. An improved genome-scale metabolic model of &lt;i&gt;arthrospira platensis&lt;/i&gt; c1 (&lt;i&gt;i&lt;/i&gt;ak888) and its application in glycogen overproduction. <em>METABOLITES</em>, DEC 2018. <a class="reference external" href="https://doi.org/10.3390/metabo8040084">[DOI]</a>.</p></li>
<li id="id11797"><p>Benjamin Shapiro, Tori M. Hoehler, and Qusheng Jin. Integrating genome-scale metabolic models into the prediction of microbial kinetics in natural environments. <em>GEOCHIMICA ET COSMOCHIMICA ACTA</em>, 242:102–122, DEC 1 2018. <a class="reference external" href="https://doi.org/10.1016/j.gca.2018.08.047">[DOI]</a>.</p></li>
<li id="id11798"><p>Carolina Shene, Juan A. Asenjo, and Yusuf Chisti. Metabolic modelling and simulation of the light and dark metabolism of &lt;i&gt;chlamydomonas reinhardtii&lt;/i&gt;. <em>PLANT JOURNAL</em>, 96(5):1076–1088, DEC 2018. <a class="reference external" href="https://doi.org/10.1111/tpj.14078">[DOI]</a>.</p></li>
<li id="id11799"><p>Julia Troendle, Natalia Trachtmann, Georg A. Sprenger, and Dirk Weuster-Botz. Fed-batch production of l-tryptophan from glycerol using recombinant &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 115(12):2881–2892, DEC 2018. <a class="reference external" href="https://doi.org/10.1002/bit.26834">[DOI]</a>.</p></li>
<li id="id11800"><p>Alessio Mancini, Filmon Eyassu, Maxwell Conway, Annalisa Occhipinti, Pietro Lio, Claudio Angione, and Sandra Pucciarelli. Ciliategem: an open-project and a tool for predictions of ciliate metabolic variations and experimental condition design. <em>BMC BIOINFORMATICS</em>, NOV 30 2018. 12th International BBCC Conference, Naples, ITALY, DEC 18-20, 2017. <a class="reference external" href="https://doi.org/10.1186/s12859-018-2422-9">[DOI]</a>.</p></li>
<li id="id11801"><p>Felipe Scott, Pamela Wilson, Raul Conejeros, and Vassilios S. Vassiliadis. Simulation and optimization of dynamic flux balance analysis models using an interior point method reformulation. <em>COMPUTERS &amp; CHEMICAL ENGINEERING</em>, 119:152–170, NOV 2 2018. <a class="reference external" href="https://doi.org/10.1016/j.compchemeng.2018.08.041">[DOI]</a>.</p></li>
<li id="id11802"><p>Jose Aguilar-Rodriguez and Andreas Wagner. Metabolic determinants of enzyme evolution in a genome-scale bacterial metabolic network. <em>GENOME BIOLOGY AND EVOLUTION</em>, 10(11):3076–3088, NOV 2018. <a class="reference external" href="https://doi.org/10.1093/gbe/evy234">[DOI]</a>.</p></li>
<li id="id11803"><p>Muhammad Azharuddin Arif, Mohd Saberi Mohamad, Muhammad Shafie Abd Latif, Safaai Deris, Muhammad Akmal Remli, Kauthar Mohd Daud, Zuwairie Ibrahim, Sigeru Omatu, and Juan Manuel Corchado. A hybrid of cuckoo search and minimization of metabolic adjustment to optimize metabolites production in genome-scale models. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, 102:112–119, NOV 1 2018. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2018.09.015">[DOI]</a>.</p></li>
<li id="id11804"><p>Rafael S. Costa and Susana Vinga. Assessing &lt;i&gt;escherichia coli&lt;/i&gt; metabolism models and simulation approaches in phenotype predictions: validation against experimental data. <em>BIOTECHNOLOGY PROGRESS</em>, 34(6):1344–1354, NOV-DEC 2018. <a class="reference external" href="https://doi.org/10.1002/btpr.2700">[DOI]</a>.</p></li>
<li id="id11805"><p>Alex Graudenzi, Davide Maspero, Marzia Di Filippo, Marco Gnugnoli, Claudio Isella, Giancarlo Mauri, Enzo Medico, Marco Antoniotti, and Chiara Damiani. Integration of transcriptomic data and metabolic networks in cancer samples reveals highly significant prognostic power. <em>JOURNAL OF BIOMEDICAL INFORMATICS</em>, 87:37–49, NOV 2018. <a class="reference external" href="https://doi.org/10.1016/j.jbi.2018.09.010">[DOI]</a>.</p></li>
<li id="id11806"><p>Nam-Hoon Kim, Murukarthick Jayakodi, Sang-Choon Lee, Beom-Soon Choi, Woojong Jang, Junki Lee, Hyun Hee Kim, Nomar E. Waminal, Meiyappan Lakshmanan, Binh Van Nguyen, Yun Sun Lee, Hyun-Seung Park, Hyun Jo Koo, Jee Young Park, Sampath Perumal, Ho Jun Joh, Hana Lee, Jinkyung Kim, In Seo Kim, Kyunghee Kim, Lokanand Koduru, Kyo Bin Kang, Sang Hyun Sung, Yeisoo Yu, Daniel S. Park, Doil Choi, Eunyoung Seo, Seungill Kim, Young-Chang Kim, Dong Yun Hyun, Youn-Il Park, Changsoo Kim, Tae-Ho Lee, Hyun Uk Kim, Moon Soo Soh, Yi Lee, Jun Gyo In, Heui-Soo Kim, Yong-Min Kim, Deok-Chun Yang, Rod A. Wing, Dong-Yup Lee, Andrew H. Paterson, and Tae-Jin Yang. Genome and evolution of the shade-requiring medicinal herb &lt;i&gt;panax ginseng&lt;/i&gt;. <em>PLANT BIOTECHNOLOGY JOURNAL</em>, 16(11):1904–1917, NOV 2018. <a class="reference external" href="https://doi.org/10.1111/pbi.12926">[DOI]</a>.</p></li>
<li id="id11807"><p>Majid Latif, Jr. and Elebeoba E. May. A multiscale agent-based model for the investigation of &lt;i&gt;e-coli&lt;/i&gt; k12 metabolic response during biofilm formation. <em>BULLETIN OF MATHEMATICAL BIOLOGY</em>, 80(11):2917–2956, NOV 2018. <a class="reference external" href="https://doi.org/10.1007/s11538-018-0494-3">[DOI]</a>.</p></li>
<li id="id11808"><p>G. Mitsue Leon-Saiki, Narcis Ferrer Ledo, David Lao-Martil, Douwe van der Veen, Rene H. Wijffels, and Dirk E. Martens. Metabolic modelling and energy parameter estimation of &lt;i&gt;tetradesmus obliquus&lt;/i&gt;. <em>ALGAL RESEARCH-BIOMASS BIOFUELS AND BIOPRODUCTS</em>, 35:378–387, NOV 2018. <a class="reference external" href="https://doi.org/10.1016/j.algal.2018.09.008">[DOI]</a>.</p></li>
<li id="id11809"><p>Felix Moser, Amin Espah Borujeni, Amar N. Ghodasara, Ewen Cameron, Yongjin Park, and Christopher A. Voigt. Dynamic control of endogenous metabolism with combinatorial logic circuits. <em>MOLECULAR SYSTEMS BIOLOGY</em>, NOV 2018. <a class="reference external" href="https://doi.org/10.15252/msb.20188605">[DOI]</a>.</p></li>
<li id="id11810"><p>Lokanand Koduru, Meiyappan Lakshmanan, and Dong-Yup Lee. In silico model-guided identification of transcriptional regulator targets for efficient strain design. <em>MICROBIAL CELL FACTORIES</em>, OCT 25 2018. <a class="reference external" href="https://doi.org/10.1186/s12934-018-1015-7">[DOI]</a>.</p></li>
<li id="id11811"><p>Elizabeth Brunk, Roger L. Chang, Jing Xia, Hooman Hefzi, James T. Yurkovich, Donghyuk Kim, Evan Buckmiller, Harris H. Wang, Byung-Kwan Cho, Chen Yang, Bernhard O. Palsson, George M. Church, and Nathan E. Lewis. Characterizing posttranslational modifications in prokaryotic metabolism using a multiscale workflow. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 115(43):11096–11101, OCT 23 2018. <a class="reference external" href="https://doi.org/10.1073/pnas.1811971115">[DOI]</a>.</p></li>
<li id="id11812"><p>Matthew A. Perisin and Christian J. Sund. Human gut microbe co-cultures have greater potential than monocultures for food waste remediation to commodity chemicals. <em>SCIENTIFIC REPORTS</em>, OCT 22 2018. <a class="reference external" href="https://doi.org/10.1038/s41598-018-33733-z">[DOI]</a>.</p></li>
<li id="id11813"><p>D. Hala, J. A. Cullen, B. Hernout, and I. Ivanov. &lt;i&gt;in silico&lt;/i&gt; predicted transcriptional regulatory control of steroidogenesis in spawning female fathead minnows (&lt;i&gt;pimephales promelas&lt;/i&gt;). <em>JOURNAL OF THEORETICAL BIOLOGY</em>, 455:179–190, OCT 14 2018. <a class="reference external" href="https://doi.org/10.1016/j.jtbi.2018.07.020">[DOI]</a>.</p></li>
<li id="id11814"><p>Anika Kueken, Frederik Sommer, Liliya Yaneva-Roder, Luke C. M. Mackinder, Melanie Hoehne, Stefan Geimer, Martin C. Jonikas, Michael Schroda, Mark Stitt, Zoran Nikoloski, and Tabea Mettler-Altmann. Effects of microcompartmentation on flux distribution and metabolic pools in &lt;i&gt;chlamydomonas reinhardtii&lt;/i&gt; chloroplasts. <em>ELIFE</em>, OCT 11 2018. <a class="reference external" href="https://doi.org/10.7554/eLife.37960">[DOI]</a>.</p></li>
<li id="id11815"><p>Santhust Kumar, Saurabh Mahajan, and Sanjay Jain. Feedbacks from the metabolic network to the genetic network reveal regulatory modules in &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; and &lt;i&gt;b&lt;/i&gt;. &lt;i&gt;subtilis&lt;/i&gt;. <em>PLOS ONE</em>, OCT 4 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0203311">[DOI]</a>.</p></li>
<li id="id11816"><p>Veronika Boczonadi, Martin S. King, Anthony C. Smith, Monika Olahova, Boglarka Bansagi, Andreas Roos, Filmon Eyassu, Christoph Borchers, Venkateswaran Ramesh, Hanns Lochmueller, Tuomo Polvikoski, Roger G. Whittaker, Angela Pyle, Helen Griffin, Robert W. Taylor, Patrick F. Chinnery, Alan J. Robinson, Edmund R. S. Kunji, and Rita Horvath. Mitochondrial oxodicarboxylate carrier deficiency is associated with mitochondrial dna depletion and spinal muscular atrophy-like disease. <em>GENETICS IN MEDICINE</em>, 20(10):1224–1235, OCT 2018. <a class="reference external" href="https://doi.org/10.1038/gim.2017.251">[DOI]</a>.</p></li>
<li id="id11817"><p>Ilyas Kabimoldayev, Anh Duc Nguyen, Laurence Yang, Sunghoon Park, Eun Yeol Lee, and Donghyuk Kim. Basics of genome-scale metabolic modeling and applications on c1-utilization. <em>FEMS MICROBIOLOGY LETTERS</em>, OCT 2018. <a class="reference external" href="https://doi.org/10.1093/femsle/fny241">[DOI]</a>.</p></li>
<li id="id11818"><p>Hao Wang, Simonas Marcisauskas, Benjamin J. Sanchez, Ivan Domenzain, Daniel Hermansson, Rasmus Agren, Jens Nielsen, and Eduard J. Kerkhoven. Raven 2.0: a versatile toolbox for metabolic network reconstruction and a case study on &lt;i&gt;streptomyces&lt;/i&gt; &lt;i&gt;coelicolor&lt;/i&gt;. <em>PLOS COMPUTATIONAL BIOLOGY</em>, OCT 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1006541">[DOI]</a>.</p></li>
<li id="id11819"><p>Wei Zou, Xianghua Xiong, Jing Zhang, Kaizheng Zhang, Xingxiu Zhao, and Changqing Zhao. Reconstruction and analysis of a genome-scale metabolic model of &lt;i&gt;methylovorus&lt;/i&gt; sp mp688, a high-level pyrroloquinolone quinone producer. <em>BIOSYSTEMS</em>, 172:37–42, OCT 2018. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2018.07.009">[DOI]</a>.</p></li>
<li id="id11820"><p>Cheng Zhang, Gholamreza Bidkhori, Rui Benfeitas, Sunjae Lee, Muhammad Arif, Mathias Uhlen, and Adil Mardinoglu. Ess: a tool for genome-scale quantification of essentiality score for reaction/genes in constraint-based modeling. <em>FRONTIERS IN PHYSIOLOGY</em>, SEP 28 2018. <a class="reference external" href="https://doi.org/10.3389/fphys.2018.01355">[DOI]</a>.</p></li>
<li id="id11821"><p>Hongzhong Lu, Weiqiang Cao, Xiaoyun Liu, Yufei Sui, Liming Ouyang, Jianye Xia, Mingzhi Huang, Yingping Zhuang, Siliang Zhang, Henk Noorman, and Ju Chu. Multi-omics integrative analysis with genome-scale metabolic model simulation reveals global cellular adaptation of &lt;i&gt;aspergillus niger&lt;/i&gt; under industrial enzyme production condition. <em>SCIENTIFIC REPORTS</em>, SEP 26 2018. <a class="reference external" href="https://doi.org/10.1038/s41598-018-32341-1">[DOI]</a>.</p></li>
<li id="id11822"><p>Elliot Rowe, Bernhard O. Palsson, and Zachary A. King. Escher-fba: a web application for interactive flux balance analysis. <em>BMC SYSTEMS BIOLOGY</em>, SEP 26 2018. <a class="reference external" href="https://doi.org/10.1186/s12918-018-0607-5">[DOI]</a>.</p></li>
<li id="id11823"><p>Hanseol Kim, Sinyeon Kim, and Sung Ho Yoon. Metabolic network reconstruction and phenome analysis of the industrial microbe, &lt;i&gt;escherichia coli&lt;/i&gt; bl21 (de3). <em>PLOS ONE</em>, SEP 21 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0204375">[DOI]</a>.</p></li>
<li id="id11824"><p>Takeyuki Tamura. Grid-based computational methods for the design of constraint-based parsimonious chemical reaction networks to simulate metabolite production: gridprod. <em>BMC BIOINFORMATICS</em>, SEP 14 2018. <a class="reference external" href="https://doi.org/10.1186/s12859-018-2352-6">[DOI]</a>.</p></li>
<li id="id11825"><p>Hilal Taymaz-Nikerel, Muhammed Erkan Karabekmez, Serpil Eraslan, and Betul Kirdar. Doxorubicin induces an extensive transcriptional and metabolic rewiring in yeast cells. <em>SCIENTIFIC REPORTS</em>, SEP 12 2018. <a class="reference external" href="https://doi.org/10.1038/s41598-018-31939-9">[DOI]</a>.</p></li>
<li id="id11826"><p>Nana Y. D. Ankrah, Bessem Chouaia, and Angela E. Douglas. The cost of metabolic interactions in symbioses between insects and bacteria with reduced genomes. <em>MBIO</em>, SEP-OCT 2018. <a class="reference external" href="https://doi.org/10.1128/mBio.01433-18">[DOI]</a>.</p></li>
<li id="id11827"><p>Elias Englund, Kiyan Shabestary, Elton P. Hudson, and Pia Lindberg. Systematic overexpression study to find target enzymes enhancing production of terpenes in &lt;i&gt;synechocystis&lt;/i&gt; pcc 6803, using isoprene as a model compound. <em>METABOLIC ENGINEERING</em>, 49:164–177, SEP 2018. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2018.07.004">[DOI]</a>.</p></li>
<li id="id11828"><p>Beatriz Garcia-Jimenez, Jose Luis Garcia, and Juan Nogales. Flycop: metabolic modeling-based analysis and engineering microbial communities. <em>BIOINFORMATICS</em>, 34(17):954–963, SEP 1 2018. 17th European Conference on Computational Biology (ECCB), Athens, GREECE, SEP 08-12, 2018. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/bty561">[DOI]</a>.</p></li>
<li id="id11829"><p>Kristopher A. Hunt, Ryan M. Jennings, William P. Inskeep, and Ross P. Carlson. Multiscale analysis of autotroph-heterotroph interactions in a high-temperature microbial community. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1006431">[DOI]</a>.</p></li>
<li id="id11830"><p>Selva Rupa Christinal Immanuel, Deepanwita Banerjee, Mayooreshwar P. Rajankar, and Anu Raghunathan. Integrated constraints based analysis of an engineered violacein pathway in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOSYSTEMS</em>, 171:10–19, SEP 2018. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2018.06.002">[DOI]</a>.</p></li>
<li id="id11831"><p>Ben M. Maoz, Anna Herland, Edward A. FitzGerald, Thomas Grevesse, Charles Vidoudez, Alan R. Pacheco, Sean P. Sheehy, Tae-Eun Park, Stephanie Dauth, Robert Mannix, Nikita Budnik, Kevin Shores, Alexander Cho, Janna C. Nawroth, Daniel Segre, Bogdan Budnik, Donald E. Ingber, and Kevin Kit Parker. A linked organ-on-chip model of the human neurovascular unit reveals the metabolic coupling of endothelial and neuronal cells. <em>NATURE BIOTECHNOLOGY</em>, 36(9):865+, SEP 2018. <a class="reference external" href="https://doi.org/10.1038/nbt.4226">[DOI]</a>.</p></li>
<li id="id11832"><p>Ioannis Papapetridis, Maarten D. Verhoeven, Sanne J. Wiersma, Maaike Goudriaan, Antonius J. A. van Maris, and Jack T. Pronk. Laboratory evolution for forced glucose-xylose co-consumption enables identification of mutations that improve mixed-sugar fermentation by xylose-fermenting &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;. <em>FEMS YEAST RESEARCH</em>, SEP 2018. <a class="reference external" href="https://doi.org/10.1093/femsyr/foy056">[DOI]</a>.</p></li>
<li id="id11833"><p>Zixiang Xu, Jing Guo, Yunxia Yue, Jing Meng, and Xiao Sun. &lt;i&gt;in silico&lt;/i&gt; genome-scale reconstruction and analysis of the &lt;i&gt;shewanella loihica&lt;/i&gt; pv-4 metabolic network. <em>JOURNAL OF BIOLOGICAL SYSTEMS</em>, 26(3):373–397, SEP 2018. <a class="reference external" href="https://doi.org/10.1142/S0218339018500171">[DOI]</a>.</p></li>
<li id="id11834"><p>Ignace L. M. M. Tack, Philippe Nimmegeers, Simen Akkermans, Filip Logist, and Jan F. M. Van Impe. A low-complexity metabolic network model for the respiratory and fermentative metabolism of &lt;i&gt;escherichia coli&lt;/i&gt;. <em>PLOS ONE</em>, AUG 29 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0202565">[DOI]</a>.</p></li>
<li id="id11835"><p>L. Kyselova, D. Kreitmayer, A. Kremling, and K. Bettenbrock. Type and capacity of glucose transport influences succinate yield in two-stage cultivations. <em>MICROBIAL CELL FACTORIES</em>, AUG 28 2018. <a class="reference external" href="https://doi.org/10.1186/s12934-018-0980-1">[DOI]</a>.</p></li>
<li id="id11836"><p>Hadi Nazem-Bokaee and Costas D. Maranas. A prospective study on the fermentation landscape of gaseous substrates to biorenewables using &lt;i&gt;methanosarcina acetivorans &lt;/i&gt;metabolic model. <em>FRONTIERS IN MICROBIOLOGY</em>, AUG 24 2018. <a class="reference external" href="https://doi.org/10.3389/fmicb.2018.01855">[DOI]</a>.</p></li>
<li id="id11837"><p>Jose P. Faria, Miguel Rocha, Isabel Rocha, and Christopher S. Henry. Methods for automated genome-scale metabolic model reconstruction. <em>BIOCHEMICAL SOCIETY TRANSACTIONS</em>, 46(4):931–936, AUG 20 2018. <a class="reference external" href="https://doi.org/10.1042/BST20170246">[DOI]</a>.</p></li>
<li id="id11838"><p>Maria Teresita Castaneda, Sebastian Nunez, Fabricio Garelli, Claudio Voget, and Hernan De Battista. Comprehensive analysis of a metabolic model for lipid production in &lt;i&gt;rhodosporidium toruloides&lt;/i&gt;. <em>JOURNAL OF BIOTECHNOLOGY</em>, 280:11–18, AUG 20 2018. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2018.05.010">[DOI]</a>.</p></li>
<li id="id11839"><p>Rienk A. Rienksma, Peter J. Schaap, Vitor A. P. Martins dos Santos, and Maria Suarez-Diez. Modeling the metabolic state of &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; upon infection. <em>FRONTIERS IN CELLULAR AND INFECTION MICROBIOLOGY</em>, AUG 3 2018. <a class="reference external" href="https://doi.org/10.3389/fcimb.2018.00264">[DOI]</a>.</p></li>
<li id="id11840"><p>George C. diCenzo, Klaudia Debiec, Jan Krzysztoforski, Witold Uhrynowski, Alessio Mengoni, Camilla Fagorzi, Adrian Gorecki, Lukasz Dziewit, Tomasz Bajda, Grzegorz Rzepa, and Lukasz Drewniak. Genomic and biotechnological characterization of the heavy-metal resistant, arsenic-oxidizing bacterium &lt;i&gt;ensifer&lt;/i&gt; sp m14. <em>GENES</em>, AUG 2018. <a class="reference external" href="https://doi.org/10.3390/genes9080379">[DOI]</a>.</p></li>
<li id="id11841"><p>Wei Zou, Guangbin Ye, Jing Zhang, Changqing Zhao, Xingxiu Zhao, and Kaizheng Zhang. Genome-scale metabolic reconstruction and analysis for &lt;i&gt;clostridium kluyveri&lt;/i&gt;. <em>GENOME</em>, 61(8):605–613, AUG 2018. <a class="reference external" href="https://doi.org/10.1139/gen-2017-0177">[DOI]</a>.</p></li>
<li id="id11842"><p>Osvaldo D. Kim, Miguel Rocha, and Paulo Maia. A review of dynamic modeling approaches and their application in computational strain optimization for metabolic engineering. <em>FRONTIERS IN MICROBIOLOGY</em>, JUL 31 2018. <a class="reference external" href="https://doi.org/10.3389/fmicb.2018.01690">[DOI]</a>.</p></li>
<li id="id11843"><p>Freyr Johannsson, Steinn Gudmundsson, Giuseppe Paglia, Sveinn Gudmundsson, Bernhard Palsson, Olafur E. Sigurjonsson, and Ottar Rolfsson. Systems analysis of metabolism in platelet concentrates during storage in platelet additive solution. <em>BIOCHEMICAL JOURNAL</em>, 475(13):2225–2240, JUL 16 2018. <a class="reference external" href="https://doi.org/10.1042/BCJ20170921">[DOI]</a>.</p></li>
<li id="id11844"><p>David Botero, Ivan Valdes, Maria-Juliana Rodriguez, Diana Henao, Giovanna Danies, Andres F. Gonzalez, and Silvia Restrepo. A genome-scale metabolic reconstruction of &lt;i&gt;phytophthora infestans&lt;/i&gt; with the integration of transcriptional data reveals the key metabolic patterns involved in the interaction of its host. <em>FRONTIERS IN GENETICS</em>, JUL 10 2018. <a class="reference external" href="https://doi.org/10.3389/fgene.2018.00244">[DOI]</a>.</p></li>
<li id="id11845"><p>Adam Amara, Eriko Takano, and Rainer Breitling. Development and validation of an updated computational model of &lt;i&gt;streptomyces&lt;/i&gt; &lt;i&gt;coelicolor&lt;/i&gt; primary and secondary metabolism. <em>BMC GENOMICS</em>, JUL 4 2018. <a class="reference external" href="https://doi.org/10.1186/s12864-018-4905-5">[DOI]</a>.</p></li>
<li id="id11846"><p>Ludovic Cottret, Clement Frainay, Maxime Chazalviel, Floreal Cabanettes, Yoann Gloaguen, Etienne Camenen, Benjamin Merlet, Stephanie Heux, Jean-Charles Portais, Nathalie Poupin, Florence Vinson, and Fabien Jourdan. Metexplore: collaborative edition and exploration of metabolic networks. <em>NUCLEIC ACIDS RESEARCH</em>, 46(W1):W495–W502, JUL 2 2018. <a class="reference external" href="https://doi.org/10.1093/nar/gky301">[DOI]</a>.</p></li>
<li id="id11847"><p>Samara Silva de Souza, Julia de Vasconcellos Castro, and Luismar Marques Porto. Modeling the core metabolism of &lt;i&gt;komagataeibacter hansenii&lt;/i&gt; atcc 23769 to evaluate nanocellulose biosynthesis. <em>BRAZILIAN JOURNAL OF CHEMICAL ENGINEERING</em>, 35(3):869–885, JUL-SEP 2018. <a class="reference external" href="https://doi.org/10.1590/0104-6632.20180353s20170327">[DOI]</a>.</p></li>
<li id="id11848"><p>Colton J. Lloyd, Ali Ebrahim, Laurence Yang, Zachary A. King, Edward Catoiu, Edward J. O'Brien, Joanne K. Liu, and Bernhard O. Polsson. Cobrame: a computational framework for genome-scale models of metabolism and gene expression. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUL 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1006302">[DOI]</a>.</p></li>
<li id="id11849"><p>Kurt Patterson, James Yu, Jenny Landberg, Ivan Chang, Farbod Shavarebi, Virginia Bilanchone, and Suzanne Sandmeyer. Functional genomics for the oleaginous yeast &lt;i&gt;yarrowia lipolytica&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 48:184–196, JUL 2018. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2018.05.008">[DOI]</a>.</p></li>
<li id="id11850"><p>Mustafa Sertbas and Kutlu O. Ulgen. Unlocking human brain metabolism by genome-scale and multiomics metabolic models: relevance for neurology research, health, and disease. <em>OMICS-A JOURNAL OF INTEGRATIVE BIOLOGY</em>, 22(7):455–467, JUL 2018. <a class="reference external" href="https://doi.org/10.1089/omi.2018.0088">[DOI]</a>.</p></li>
<li id="id11851"><p>Kiyan Shabestary, Josefine Anfelt, Emil Ljungqvist, Michael Jahn, Lun Yao, and Elton P. Hudson. Targeted repression of essential genes to arrest growth and increase carbon partitioning and biofuel titers in cyanobacteria. <em>ACS SYNTHETIC BIOLOGY</em>, 7(7):1669–1675, JUL 2018. <a class="reference external" href="https://doi.org/10.1021/acssynbio.8b00056">[DOI]</a>.</p></li>
<li id="id11852"><p>Feiran Li, Wei Xie, Qianqian Yuan, Hao Luo, Peishun Li, Tao Chen, Xueming Zhao, Zhiwen Wang, and Hongwu Ma. Genome-scale metabolic model analysis indicates low energy production efficiency in marine ammonia-oxidizing archaea. <em>AMB EXPRESS</em>, JUN 27 2018. <a class="reference external" href="https://doi.org/10.1186/s13568-018-0635-y">[DOI]</a>.</p></li>
<li id="id11853"><p>Dorines Rosario, Rui Benfeitas, Gholamreza Bidkhori, Cheng Zhang, Mathias Uhlen, Saeed Shoaie, and Adil Mardinoglu. Understanding the representative gut microbiota dysbiosis in metformin-treated type 2 diabetes patients using genome-scale metabolic modeling. <em>FRONTIERS IN PHYSIOLOGY</em>, JUN 25 2018. <a class="reference external" href="https://doi.org/10.3389/fphys.2018.00775">[DOI]</a>.</p></li>
<li id="id11854"><p>Neeraj Sinha, Maria Suarez-Diez, Guido J. E. J. Hooiveld, Jaap Keijer, Vitor Martin dos Santos, and Evert M. van Schothorst. A constraint-based model analysis of enterocyte mitochondrial adaptation to dietary interventions of lipid type and lipid load. <em>FRONTIERS IN PHYSIOLOGY</em>, JUN 15 2018. <a class="reference external" href="https://doi.org/10.3389/fphys.2018.00749">[DOI]</a>.</p></li>
<li id="id11855"><p>Jose R. Valverde, Sonia Gullon, and Rafael P. Mellado. Modelling the metabolism of protein secretion through the tat route in &lt;i&gt;streptomyces&lt;/i&gt; &lt;i&gt;lividans&lt;/i&gt;. <em>BMC MICROBIOLOGY</em>, JUN 14 2018. <a class="reference external" href="https://doi.org/10.1186/s12866-018-1199-3">[DOI]</a>.</p></li>
<li id="id11856"><p>Zachary P. Metz, Tong Ding, and David J. Baumler. Using genome-scale metabolic models to compare serovars of the foodborne pathogen &lt;i&gt;listeria monocytogenes&lt;/i&gt;. <em>PLOS ONE</em>, JUN 7 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0198584">[DOI]</a>.</p></li>
<li id="id11857"><p>Tahereh Ghasemi-Kahrizsangi, Sayed-Amir Marashi, and Zhaleh Hosseini. Genome-scale metabolic network models of bacillus species suggest that model improvement is necessary for biotechnological applications. <em>IRANIAN JOURNAL OF BIOTECHNOLOGY</em>, 16(3):164–172, SUM 2018. <a class="reference external" href="https://doi.org/10.21859/ijb.1684">[DOI]</a>.</p></li>
<li id="id11858"><p>Hai He, Christian Edlich-Muth, Steffen N. Lindner, and Arren Bar-Even. Ribulose monophosphate shunt provides nearly all biomass and energy required for growth of &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt;. <em>ACS SYNTHETIC BIOLOGY</em>, 7(6):1601–1611, JUN 2018. <a class="reference external" href="https://doi.org/10.1021/acssynbio.8b00093">[DOI]</a>.</p></li>
<li id="id11859"><p>Sander Y. A. Rodenburg, Michael F. Seidl, Dick De Ridder, and Francine Govers. Genome-wide characterization of &lt;i&gt;phytophthora infestans&lt;/i&gt; metabolism: a systems biology approach. <em>MOLECULAR PLANT PATHOLOGY</em>, 19(6):1403–1413, JUN 2018. <a class="reference external" href="https://doi.org/10.1111/mpp.12623">[DOI]</a>.</p></li>
<li id="id11860"><p>Kento Tokuyama, Yoshihiro Toya, Takaaki Horinouchi, Chikara Furusawa, Fumio Matsuda, and Hiroshi Shimizu. Application of adaptive laboratory evolution to overcome a flux limitation in an &lt;i&gt;escherichia coli&lt;/i&gt; production strain. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 115(6):1542–1551, JUN 2018. <a class="reference external" href="https://doi.org/10.1002/bit.26568">[DOI]</a>.</p></li>
<li id="id11861"><p>Michael Scheunemann, Siobhan M. Brady, and Zoran Nikoloski. Integration of large-scale data for extraction of integrated &lt;i&gt;arabidopsis&lt;/i&gt; root cell-type specific models. <em>SCIENTIFIC REPORTS</em>, MAY 21 2018. <a class="reference external" href="https://doi.org/10.1038/s41598-018-26232-8">[DOI]</a>.</p></li>
<li id="id11862"><p>Sailaja V. Elchuri, Swetha Rajasekaran, and Wayne O. Miles. Rna-sequencing of primary retinoblastoma tumors provides new insights and challenges into tumor development. <em>FRONTIERS IN GENETICS</em>, MAY 17 2018. <a class="reference external" href="https://doi.org/10.3389/fgene.2018.00170">[DOI]</a>.</p></li>
<li id="id11863"><p>Meaziane Aite, Marie Chevallier, Cleamence Frioux, Camille Trottier, Jeanne Got, Maria Paz Cortes, Sebastian N. Mendoza, Gregory Carrier, Olivier Dameron, Nicolas Guillaudeux, Mauricio Latorre, Nicolas Loira, Gabriel V. Markov, Alejandro Maass, and Anne Siegel. Traceability, reproducibility and wiki-exploration for “a-la-carte” reconstructions of genome-scale metabolic models. <em>PLOS COMPUTATIONAL BIOLOGY</em>, MAY 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1006146">[DOI]</a>.</p></li>
<li id="id11864"><p>Brendan King, Terry Farrah, Matthew A. Richards, Michael Mundy, Evangelos Simeonidis, and Nathan D. Price. Probannoweb and probannopy: probabilistic annotation and gap-filling of metabolic reconstructions. <em>BIOINFORMATICS</em>, 34(9):1594–1596, MAY 1 2018. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btx796">[DOI]</a>.</p></li>
<li id="id11865"><p>Sarah McGarrity, Osk Anuforo, Haraldur Halldorsson, Andreas Bergmann, Skarpheoinn Halldorsson, Sirus Palsson, Hanne H. Henriksen, Par Ingemar Johansson, and Ottar Rolfsson. Metabolic systems analysis of lps induced endothelial dysfunction applied to sepsis patient stratification. <em>SCIENTIFIC REPORTS</em>, MAY 1 2018. <a class="reference external" href="https://doi.org/10.1038/s41598-018-25015-5">[DOI]</a>.</p></li>
<li id="id11866"><p>Howard Ramirez-Malule, Stefan Junne, Mariano Nicolas Cruz-Bournazou, Peter Neubauer, and Rigoberto Rios-Estepa. &lt;i&gt;streptomyces clavuligerus&lt;/i&gt; shows a strong association between tca cycle intermediate accumulation and clavulanic acid biosynthesis. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 102(9):4009–4023, MAY 2018. <a class="reference external" href="https://doi.org/10.1007/s00253-018-8841-8">[DOI]</a>.</p></li>
<li id="id11867"><p>Andreas Schwentner, Andre Feith, Eugenia Muench, Tobias Busche, Christian Rueckert, Joern Kalinowski, Ralf Takors, and Bastian Blombach. Metabolic engineering to guide evolution - creating a novel mode for l-valine production with &lt;i&gt;corynebacterium glutamicum&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 47:31–41, MAY 2018. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2018.02.015">[DOI]</a>.</p></li>
<li id="id11868"><p>Leon Toro, Laura Pinilla, Claudio Avignone-Rossa, and Rigoberto Rios-Estepa. An enhanced genome-scale metabolic reconstruction of &lt;i&gt;streptomyces clavuligerus&lt;/i&gt; identifies novel strain improvement strategies. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 41(5):657–669, MAY 2018. <a class="reference external" href="https://doi.org/10.1007/s00449-018-1900-9">[DOI]</a>.</p></li>
<li id="id11869"><p>Julia Troendle, Christoph Albermann, Michael Weiner, Georg A. Sprenger, and Dirk Weuster-Botz. Phosphoenolpyruvate transporter enables targeted perturbation during metabolic analysis of l-phenylalanine production with &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOTECHNOLOGY JOURNAL</em>, MAY 2018. <a class="reference external" href="https://doi.org/10.1002/biot.201700611">[DOI]</a>.</p></li>
<li id="id11870"><p>Wei Zou, Guangbin Ye, and Kaizheng Zhang. Diversity, function, and application of &lt;i&gt;clostridium&lt;/i&gt; in chinese strong flavor baijiu ecosystem: a review. <em>JOURNAL OF FOOD SCIENCE</em>, 83(5):1193–1199, MAY 2018. <a class="reference external" href="https://doi.org/10.1111/1750-3841.14134">[DOI]</a>.</p></li>
<li id="id11871"><p>Erick Armingol, Eduardo Tobar, and Ricardo Cabrera. Understanding the impact of the cofactor swapping of isocitrate dehydrogenase over the growth phenotype of &lt;i&gt;escherichia coli&lt;/i&gt; on acetate by using constraint-based modeling. <em>PLOS ONE</em>, APR 20 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0196182">[DOI]</a>.</p></li>
<li id="id11872"><p>Nicholas A. Jose, Rebecca Lau, Tami L. Swenson, Niels Klitgord, Ferran Garcia-Pichel, Benjamin P. Bowen, Richard Baran, and Trent R. Northen. Flux balance modeling to predict bacterial survival during pulsed-activity events. <em>BIOGEOSCIENCES</em>, 15(7):2219–2229, APR 16 2018. <a class="reference external" href="https://doi.org/10.5194/bg-15-2219-2018">[DOI]</a>.</p></li>
<li id="id11873"><p>Donghyuk Kim, Sang Woo Seo, Ye Gao, Hojung Nam, Gabriela I. Guzman, Byung-Kwan Cho, and Bernhard O. Palsson. Systems assessment of transcriptional regulation on central carbon metabolism by cra and crp. <em>NUCLEIC ACIDS RESEARCH</em>, 46(6):2901–2917, APR 6 2018. <a class="reference external" href="https://doi.org/10.1093/nar/gky069">[DOI]</a>.</p></li>
<li id="id11874"><p>Jun Feng, Jing Yang, Wenwen Yang, Jie Chen, Min Jiang, and Xiang Zou. Metabolome- and genome-scale model analyses for engineering of &lt;i&gt;aureobasidium pullulans&lt;/i&gt; to enhance polymalic acid and malic acid production from sugarcane molasses. <em>BIOTECHNOLOGY FOR BIOFUELS</em>, APR 4 2018. <a class="reference external" href="https://doi.org/10.1186/s13068-018-1099-7">[DOI]</a>.</p></li>
<li id="id11875"><p>George C. Dicenzo, Alex B. Benedict, Marco Fondi, Graham C. Walker, Turlough M. Finan, Alessio Mengoni, and Joel S. Griffitts. Robustness encoded across essential and accessory replicons of the ecologically versatile bacterium &lt;i&gt;sinorhizobium meliloti&lt;/i&gt;. <em>PLOS GENETICS</em>, APR 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pgen.1007357">[DOI]</a>.</p></li>
<li id="id11876"><p>Jason J. Flowers, Matthew A. Richards, Nitin Baliga, Birte Meyer, and David A. Stahl. Constraint-based modelling captures the metabolic versatility of &lt;i&gt;desulfovibrio vulgaris&lt;/i&gt;. <em>ENVIRONMENTAL MICROBIOLOGY REPORTS</em>, 10(2):190–201, APR 2018. <a class="reference external" href="https://doi.org/10.1111/1758-2229.12619">[DOI]</a>.</p></li>
<li id="id11877"><p>Nan Xu, Chao Ye, and Liming Liu. Genome-scale biological models for industrial microbial systems. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 102(8):3439–3451, APR 2018. <a class="reference external" href="https://doi.org/10.1007/s00253-018-8803-1">[DOI]</a>.</p></li>
<li id="id11878"><p>Arturo Casini, Fang-Yuan Chang, Raissa Eluere, Andrew M. King, Eric M. Young, Quentin M. Dudley, Ashty Karim, Katelin Pratt, Cassandra Bristol, Anthony Forget, Amar Ghodasara, Robert Warden-Rothman, Rui Gan, Alexander Cristofaro, Amin Espah Borujeni, Min-Hyung Ryu, Jian Li, Yong-Chan Kwon, He Wang, Evangelos Tatsis, Carlos Rodriguez-Lopez, Sarah O'Connor, Marnix H. Medema, Michael A. Fischbach, Michael C. Jewett, Christopher Voigt, and D. Benjamin Gordon. A pressure test to make 10 molecules in 90 days: external evaluation of methods to engineer biology. <em>JOURNAL OF THE AMERICAN CHEMICAL SOCIETY</em>, 140(12):4302–4316, MAR 28 2018. <a class="reference external" href="https://doi.org/10.1021/jacs.7b13292">[DOI]</a>.</p></li>
<li id="id11879"><p>Pranjul Mishra, Na-Rae Lee, Meiyappan Lakshmanan, Minsuk Kim, Byung-Gee Kim, and Dong-Yup Lee. Genome-scale model-driven strain design for dicarboxylic acid production in &lt;i&gt;yarrowia lipolytica&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, MAR 19 2018. 28th International Conference on Genome Informatics - Systems Biology, Seoul, SOUTH KOREA, OCT 31-NOV 03, 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-018-0542-5">[DOI]</a>.</p></li>
<li id="id11880"><p>Yan Zhu, Tobias Czauderna, Jinxin Zhao, Matthias Klapperstueck, Mohd Hafidz Mahamad Maifiah, Mei-Ling Han, Jing Lu, Bjoern Sommer, Tony Velkov, Trevor Lithgow, Jiangning Song, Falk Schreiber, and Jian Li. Genome-scale metabolic modeling of responses to polymyxins in &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt;. <em>GIGASCIENCE</em>, MAR 13 2018. <a class="reference external" href="https://doi.org/10.1093/gigascience/giy021">[DOI]</a>.</p></li>
<li id="id11881"><p>Miroslava Cuperlovic-Culf. Machine learning methods for analysis of metabolic data and metabolic pathway modeling. <em>METABOLITES</em>, MAR 2018. <a class="reference external" href="https://doi.org/10.3390/metabo8010004">[DOI]</a>.</p></li>
<li id="id11882"><p>William Kelly, Sorelle Veigne, Xianhua Li, Shyam Sundar Subramanian, Zuyi Huang, and Eugene Schaefer. Optimizing performance of semi-continuous cell culture in an ambr15™ microbioreactor using dynamic flux balance modeling. <em>BIOTECHNOLOGY PROGRESS</em>, 34(2):420–431, MAR-APR 2018. <a class="reference external" href="https://doi.org/10.1002/btpr.2585">[DOI]</a>.</p></li>
<li id="id11883"><p>Bashir Sajo Mienda, Rabiu Salihu, Aliyu Adamu, and Shehu Idris. Genome-scale metabolic models as platforms for identification of novel genes as antimicrobial drug targets. <em>FUTURE MICROBIOLOGY</em>, 13(4):455–467, MAR 2018. <a class="reference external" href="https://doi.org/10.2217/fmb-2017-0195">[DOI]</a>.</p></li>
<li id="id11884"><p>John E. Pearl, Mrinal Das, and Andrea M. Cooper. Immunological roulette: luck or something more? considering the connections between host and environment in tb. <em>CELLULAR &amp; MOLECULAR IMMUNOLOGY</em>, 15(3, SI):226–232, MAR 2018. <a class="reference external" href="https://doi.org/10.1038/cmi.2017.145">[DOI]</a>.</p></li>
<li id="id11885"><p>Mohammad Tajparast and Dominic Frigon. Predicting the accumulation of storage compounds by &lt;i&gt;rhodococcus jostii&lt;/i&gt; rha1 in the feast-famine growth cycles using genome-scale flux balance analysis. <em>PLOS ONE</em>, MAR 1 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0191835">[DOI]</a>.</p></li>
<li id="id11886"><p>Yunze Zhang, Huxing Chen, Jiamei Liu, Ge Geng, Deli Liu, Hui Geng, and Li Xiong. Genome sequencing and biodegradation characteristics of the &lt;i&gt;n&lt;/i&gt;-butyl benzyl phthalate degrading &lt;i&gt;bacterium-rhodococcus&lt;/i&gt; sp hs-d2. <em>INTERNATIONAL BIODETERIORATION &amp; BIODEGRADATION</em>, 128(SI):56–62, MAR 2018. 1st International Symposium and the 4th Sino-Hungarian Workshop on Remediation and Restoration of Polluted Mining Areas, Wuhan, PEOPLES R CHINA, MAY 22-24, 2015. <a class="reference external" href="https://doi.org/10.1016/j.ibiod.2016.08.024">[DOI]</a>.</p></li>
<li id="id11887"><p>Henrik Cordes, Christoph Thiel, Vanessa Baier, Lars M. Blank, and Lars Kuepfer. Integration of genome-scale metabolic networks into whole-body pbpk models shows phenotype-specific cases of drug-induced metabolic perturbation. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, FEB 26 2018. <a class="reference external" href="https://doi.org/10.1038/s41540-018-0048-1">[DOI]</a>.</p></li>
<li id="id11888"><p>Miha Moskon, Nikolaj Zimic, and Miha Mraz. Grohar: automated visualization of genome-scale metabolic models and their pathways. <em>JOURNAL OF COMPUTATIONAL BIOLOGY</em>, 25(5):505–508, MAY 2018. <a class="reference external" href="https://doi.org/10.1089/cmb.2017.0209">[DOI]</a>.</p></li>
<li id="id11889"><p>Ran Yu, Octavio Perez-Garcia, Huijie Lu, and Kartik Chandran. &lt;i&gt;nitrosomonas europaea&lt;/i&gt; adaptation to anoxic-oxic cycling: insights from transcription analysis, proteomics and metabolic network modeling. <em>SCIENCE OF THE TOTAL ENVIRONMENT</em>, 615:1566–1573, FEB 15 2018. <a class="reference external" href="https://doi.org/10.1016/j.scitotenv.2017.09.142">[DOI]</a>.</p></li>
<li id="id11890"><p>Maria Caracausi, Veronica Ghini, Chiara Locatelli, Martina Mericio, Allison Piovesan, Francesca Antonaros, Maria Chiara Pelleri, Lorenza Vitale, Rosa Anna Vacca, Federica Bedetti, Maria Chiara Mimmi, Claudio Luchinat, Paola Turano, Pierluigi Strippoli, and Guido Cocchi. Plasma and urinary metabolomic profiles of down syndrome correlate with alteration of mitochondrial metabolism. <em>SCIENTIFIC REPORTS</em>, FEB 14 2018. <a class="reference external" href="https://doi.org/10.1038/s41598-018-20834-y">[DOI]</a>.</p></li>
<li id="id11891"><p>Ilya R. Akberdin, Merlin Thompson, Richard Hamilton, Nalini Desai, Danny Alexander, Calvin A. Henard, Michael T. Guarnieri, and Marina G. Kalyuzhnaya. Methane utilization in &lt;i&gt;methylomicrobium alcaliphilum&lt;/i&gt; 20z&lt;sup&gt;r&lt;/sup&gt;: a systems approach. <em>SCIENTIFIC REPORTS</em>, FEB 6 2018. <a class="reference external" href="https://doi.org/10.1038/s41598-018-20574-z">[DOI]</a>.</p></li>
<li id="id11892"><p>Claudio Angione. Integrating splice-isoform expression into genome-scale models characterizes breast cancer metabolism. <em>BIOINFORMATICS</em>, 34(3):494–501, FEB 1 2018. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btx562">[DOI]</a>.</p></li>
<li id="id11893"><p>Yazdan Asgari, Pegah Khosravi, Zahra Zabihinpour, and Mahnaz Habibi. Exploring candidate biomarkers for lung and prostate cancers using gene expression and flux variability analysis. <em>INTEGRATIVE BIOLOGY</em>, 10(2):113–120, FEB 2018. <a class="reference external" href="https://doi.org/10.1039/c7ib00135e">[DOI]</a>.</p></li>
<li id="id11894"><p>Hamideh Fouladiha, Sayed-Amir Marashi, Mohammad Ali Shokrgozar, Mehdi Farokhi, and Amir Atashi. Applications of a metabolic network model of mesenchymal stem cells for controlling cell proliferation and differentiation. <em>CYTOTECHNOLOGY</em>, 70(1):331–338, FEB 2018. <a class="reference external" href="https://doi.org/10.1007/s10616-017-0148-6">[DOI]</a>.</p></li>
<li id="id11895"><p>Yun Zhang, Xiuling Shang, Shujuan Lai, Yu Zhang, Qitiao Hu, Xin Chai, Bo Wang, Shuwen Liu, and Tingyi We. Reprogramming one-carbon metabolic pathways to decouple l-serine catabolism from cell growth in &lt;i&gt;corynebacterium glutamicum&lt;/i&gt;. <em>ACS SYNTHETIC BIOLOGY</em>, 7(2):635–646, FEB 2018. <a class="reference external" href="https://doi.org/10.1021/acssynbio.7b00373">[DOI]</a>.</p></li>
<li id="id11896"><p>Ambarish Nag, Peter C. St. John, Michael F. Crowley, and Yannick J. Bomble. Prediction of reaction knockouts to maximize succinate production by &lt;i&gt;actinobacillus succinogenes&lt;/i&gt;. <em>PLOS ONE</em>, JAN 30 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0189144">[DOI]</a>.</p></li>
<li id="id11897"><p>Wei-Chin Ho and Jianzhi Zhang. Evolutionary adaptations to new environments generally reverse plastic phenotypic changes. <em>NATURE COMMUNICATIONS</em>, JAN 24 2018. <a class="reference external" href="https://doi.org/10.1038/s41467-017-02724-5">[DOI]</a>.</p></li>
<li id="id11898"><p>Xiulai Chen, Cong Gao, Liang Guo, Guipeng Hu, Qiuling Luo, Jia Liu, Jens Nielsen, Jian Chen, and Liming Liu. Dceo biotechnology: tools to design, construct, evaluate, and optimize the metabolic pathway for biosynthesis of chemicals. <em>CHEMICAL REVIEWS</em>, 118(1):4–72, JAN 10 2018. <a class="reference external" href="https://doi.org/10.1021/acs.chemrev.6b00804">[DOI]</a>.</p></li>
<li id="id11899"><p>Francine Piubeli, Manuel Salvador, Montserrat Argandona, Joaquin J. Nieto, Vicente Bernal, Jose M. Pastor, Manuel Canovas, and Carmen Vargas. Insights into metabolic osmoadaptation of the ectoines-producer bacterium &lt;i&gt;chromohalobacter salexigens&lt;/i&gt; through a high-quality genome scale metabolic model. <em>MICROBIAL CELL FACTORIES</em>, JAN 9 2018. <a class="reference external" href="https://doi.org/10.1186/s12934-017-0852-0">[DOI]</a>.</p></li>
<li id="id11900"><p>Partho Sen, Esko Kemppainen, and Matej Oresic. Perspectives on systems modeling of human peripheral blood mononuclear cells. <em>FRONTIERS IN MOLECULAR BIOSCIENCES</em>, JAN 9 2018. <a class="reference external" href="https://doi.org/10.3389/fmolb.2017.00096">[DOI]</a>.</p></li>
<li id="id11901"><p>Alyaa M. Abdel-Haleem, Hooman Hefzi, Katsuhiko Mineta, Xin Gao, Takashi Gojobori, Bernhard O. Palsson, Nathan E. Lewis, and Neema Jamshidi. Functional interrogation of &lt;i&gt;plasmodium&lt;/i&gt; genus metabolism identifies species- and stage-specific differences in nutrient essentiality and drug targeting. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JAN 2018. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1005895">[DOI]</a>.</p></li>
<li id="id11902"><p>Kok Siong Ang, Meiyappan Lakshmanan, Na-Rae Lee, and Dong-Yup Lee. Metabolic modeling of microbial community interactions for health, environmental and biotechnological applications. <em>CURRENT GENOMICS</em>, 19(8):712–722, 2018. <a class="reference external" href="https://doi.org/10.2174/1389202919666180911144055">[DOI]</a>.</p></li>
<li id="id11903"><p>Yazdan Asgari, Zahra Zabihinpour, and Ali Masoudi-Nejad. Scan-toolbox: structural cobra add-on (scan) for analysing large metabolic networks. <em>CURRENT BIOINFORMATICS</em>, 13(1):100–107, 2018. <a class="reference external" href="https://doi.org/10.2174/1574893611666161123123729">[DOI]</a>.</p></li>
<li id="id11904"><p>Shashi Kant Bhatia, Ravi Kant Bhatia, Yong-Keun Choi, Eunsung Kan, Yun-Gon Kim, and Yung-Hun Yang. Biotechnological potential of microbial consortia and future perspectives. <em>CRITICAL REVIEWS IN BIOTECHNOLOGY</em>, 38(8):1209–1229, 2018. <a class="reference external" href="https://doi.org/10.1080/07388551.2018.1471445">[DOI]</a>.</p></li>
<li id="id11905"><p>David A. Collins and Marina G. Kalyuzhnaya. Navigating methane metabolism: enzymes, compartments, and networks. In F Armstrong, editor, <em>ENZYMES OF ENERGY TECHNOLOGY</em>, volume 613 of Methods in Enzymology, pages 349–383. 2018. <a class="reference external" href="https://doi.org/10.1016/bs.mie.2018.10.010">[DOI]</a>.</p></li>
<li id="id11906"><p>Tong Hao, Bin Wang, Lingxuan Zhao, Xin Feng, and Jinsheng Sun. Reconstruction and analysis of a genome-scale metabolic network for &lt;i&gt;eriocheir&lt;/i&gt; &lt;i&gt;sinensis&lt;/i&gt; hepatopancreas. <em>IEEE ACCESS</em>, 6:79235–79244, 2018. <a class="reference external" href="https://doi.org/10.1109/ACCESS.2018.2885005">[DOI]</a>.</p></li>
<li id="id11907"><p>Homa Majd, Martin S. King, Anthony C. Smith, and Edmund R. S. Kunji. Pathogenic mutations of the human mitochondrial citrate carrier slc25a1 lead to impaired citrate export required for lipid, dolichol, ubiquinone and sterol synthesis. <em>BIOCHIMICA ET BIOPHYSICA ACTA-BIOENERGETICS</em>, 1859(1):1–7, JAN 2018. <a class="reference external" href="https://doi.org/10.1016/j.bbabio.2017.10.002">[DOI]</a>.</p></li>
<li id="id11908"><p>Marius Tomas-Gamisans, Pau Ferrer, and Joan Albiol. Fine-tuning the &lt;i&gt;p-pastoris&lt;/i&gt; imt1026 genome-scale metabolic model for improved prediction of growth on methanol or glycerol as sole carbon sources. <em>MICROBIAL BIOTECHNOLOGY</em>, 11(1, SI):224–237, JAN 2018. <a class="reference external" href="https://doi.org/10.1111/1751-7915.12871">[DOI]</a>.</p></li>
<li id="id11909"><p>Junhua Wang, Cheng Wang, Huanhuan Liu, Haishan Qi, Hong Chen, and Jianping Wen. Metabolomics assisted metabolic network modeling and network wide analysis of metabolites in microbiology. <em>CRITICAL REVIEWS IN BIOTECHNOLOGY</em>, 38(7):1106–1120, 2018. <a class="reference external" href="https://doi.org/10.1080/07388551.2018.1462141">[DOI]</a>.</p></li>
<li id="id11910"><p>Zhendong Zhuang, Mingzhi Huang, and Ju Chu. In silico reconstruction and experimental validation of &lt;i&gt;saccharopolyspora erythraea&lt;/i&gt; genome-scale metabolic model izz1342 that accounts for 1685 orfs. <em>BIORESOURCES AND BIOPROCESSING</em>, 2018. <a class="reference external" href="https://doi.org/10.1186/s40643-018-0212-x">[DOI]</a>.</p></li>
<li id="id11911"><p>Cristal Zuniga, Jennifer Levering, Maciek R. Antoniewicz, Michael T. Guarnieri, Michael J. Betenbaugh, and Karsten Zengler. Predicting dynamic metabolic demands in the photosynthetic eukaryote &lt;i&gt;chlorella vulgaris&lt;/i&gt;. <em>PLANT PHYSIOLOGY</em>, 176(1):450–462, JAN 2018. <a class="reference external" href="https://doi.org/10.1104/pp.17.00605">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-7" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-7" name="0-7" role="tabpanel" tabindex="0"><div class="docutils container" id="id12741">
<ol class="arabic simple" start="1">
<li id="id13732"><p>Santosh Pandit, Vaishnavi Ravikumar, Alyaa M. Abdel-Haleem, Abderahmane Derouiche, V. R. S. S. Mokkapati, Carina Sihlbom, Katsuhiko Mineta, Takashi Gojobori, Xin Gao, Fredrik Westerlund, and Ivan Mijakovic. Low concentrations of vitamin c reduce the synthesis of extracellular polymers and destabilize bacterial biofilms. <em>FRONTIERS IN MICROBIOLOGY</em>, DEC 22 2017. <a class="reference external" href="https://doi.org/10.3389/fmicb.2017.02599">[DOI]</a>.</p></li>
<li id="id13733"><p>Andras Hartmann, Ana Vila-Santa, Nicolai Kallscheuer, Michael Vogt, Alice Julien-Laferriere, Marie-France Sagot, Jan Marienhagen, and Susana Vinga. Optpipe - a pipeline for optimizing metabolic engineering targets. <em>BMC SYSTEMS BIOLOGY</em>, DEC 21 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-017-0515-0">[DOI]</a>.</p></li>
<li id="id13734"><p>Octavio Perez-Garcia, Cody Mankelow, Kartik Chandran, Silas G. Villas-Boas, and Naresh Singhal. Modulation of nitrous oxide (n&lt;sub&gt;2&lt;/sub&gt;o) accumulation by primary metabolites in denitrifying cultures adapting to changes in environmental c and n. <em>ENVIRONMENTAL SCIENCE &amp; TECHNOLOGY</em>, 51(23):13678–13688, DEC 5 2017. <a class="reference external" href="https://doi.org/10.1021/acs.est.7b03345">[DOI]</a>.</p></li>
<li id="id13735"><p>Andreas Wagner. Information theory, evolutionary innovations and evolvability. <em>PHILOSOPHICAL TRANSACTIONS OF THE ROYAL SOCIETY B-BIOLOGICAL SCIENCES</em>, DEC 5 2017. <a class="reference external" href="https://doi.org/10.1098/rstb.2016.0416">[DOI]</a>.</p></li>
<li id="id13736"><p>Stephen P. Chapman, Marcelo Trindade dos Santos, Giles N. Johnson, Mauricio Vieira Kritz, and Jean-Marc Schwartz. Cyclic decomposition explains a photosynthetic down regulation for &lt;i&gt;chlamydomonas reinhardtii&lt;/i&gt;. <em>BIOSYSTEMS</em>, 162:119–127, DEC 2017. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2017.09.014">[DOI]</a>.</p></li>
<li id="id13737"><p>Robert Haesler, Raheleh Sheibani-Tezerji, Anupam Sinha, Matthias Barann, Ateequr Rehman, Daniela Esser, Konrad Aden, Carolin Knecht, Berenice Brandt, Susanna Nikolaus, Sascha Schaeuble, Christoph Kaleta, Andre Franke, Christoph Fretter, Werner Mueller, Marc-Thorsten Huett, Michael Krawczak, Stefan Schreiber, and Philip Rosenstiel. Uncoupling of mucosal gene regulation, mrna splicing and adherent microbiota signatures in inflammatory bowel disease. <em>GUT</em>, 66(12):2087–2097, DEC 2017. <a class="reference external" href="https://doi.org/10.1136/gutjnl-2016-311651">[DOI]</a>.</p></li>
<li id="id13738"><p>William C. Morell, Garrett W. Birkel, Mark Forrer, Teresa Lopez, Tyler W. H. Backman, Michael Dussault, Christpoher J. Petzold, Edward E. K. Baidoo, Zak Costello, David Ando, Jorge Alonso-Gutierrez, Kevin W. George, Aindrila Mukhopadhyay, Ian Vaino, Jay D. Keasling, Paul D. Adams, Nathan J. Hillson, and Hector Garcia Martin. The experiment data depot: a web-based software tool for biological experimental data storage, sharing, and visualization. <em>ACS SYNTHETIC BIOLOGY</em>, 6(12):2248–2259, DEC 2017. <a class="reference external" href="https://doi.org/10.1021/acssynbio.7b00204">[DOI]</a>.</p></li>
<li id="id13739"><p>E. Motamedian, E. Taheri, and F. Bagheri. Proliferation inhibition of cisplatin-resistant ovarian cancer cells using drugs screened by integrating a metabolic model and transcriptomic data. <em>CELL PROLIFERATION</em>, DEC 2017. <a class="reference external" href="https://doi.org/10.1111/cpr.12370">[DOI]</a>.</p></li>
<li id="id13740"><p>Yasutaka Hirokawa, Shingo Matsuo, Hiroyuki Hamada, Fumio Matsuda, and Taizo Hanai. Metabolic engineering of &lt;i&gt;synechococcus elongatus&lt;/i&gt; pcc 7942 for improvement of 1,3-propanediol and glycerol production based on in silico simulation of metabolic flux distribution. <em>MICROBIAL CELL FACTORIES</em>, NOV 25 2017. <a class="reference external" href="https://doi.org/10.1186/s12934-017-0824-4">[DOI]</a>.</p></li>
<li id="id13741"><p>Anthony C. Smith, Filmon Eyassu, Jean-Pierre Mazat, and Alan J. Robinson. Mitocore: a curated constraint-based model for simulating human central metabolism. <em>BMC SYSTEMS BIOLOGY</em>, NOV 25 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-017-0500-7">[DOI]</a>.</p></li>
<li id="id13742"><p>James T. Yurkovich, Benjamin J. Yurkovich, Andreas Draeger, Bernhard O. Palsson, and Zachary A. King. A padawan programmer's guide to developing software libraries. <em>CELL SYSTEMS</em>, 5(5):431–437, NOV 22 2017. <a class="reference external" href="https://doi.org/10.1016/j.cels.2017.08.003">[DOI]</a>.</p></li>
<li id="id13743"><p>Lokanand Koduru, Yujin Kim, Jeongsu Bang, Meiyappan Lakshmanan, Nam Soo Han, and Dong-Yup Lee. Genome-scale modeling and transcriptome analysis of &lt;i&gt;leuconostoc mesenteroides&lt;/i&gt; unravel the redox governed metabolic states in obligate heterofermentative lactic acid bacteria. <em>SCIENTIFIC REPORTS</em>, NOV 16 2017. <a class="reference external" href="https://doi.org/10.1038/s41598-017-16026-9">[DOI]</a>.</p></li>
<li id="id13744"><p>Ali R. Zomorrodi and Daniel Segre. Genome-driven evolutionary game theory helps understand the rise of metabolic interdependencies in microbial communities. <em>NATURE COMMUNICATIONS</em>, NOV 16 2017. <a class="reference external" href="https://doi.org/10.1038/s41467-017-01407-5">[DOI]</a>.</p></li>
<li id="id13745"><p>Siu H. J. Chan, Jingyi Cai, Lin Wang, Margaret N. Simons-Senftle, and Costas D. Maranas. Standardizing biomass reactions and ensuring complete mass balance in genome-scale metabolic models. <em>BIOINFORMATICS</em>, 33(22):3603–3609, NOV 15 2017. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btx453">[DOI]</a>.</p></li>
<li id="id13746"><p>Pavel Dvorak, Pablo I. Nikel, Jiri Damborsky, and Victor de Lorenzo. &lt;i&gt;bioremediation 3.0&lt;/i&gt;: engineering pollutant-removing bacteria in the times of systemic biology. <em>BIOTECHNOLOGY ADVANCES</em>, 35(7):845–866, NOV 15 2017. <a class="reference external" href="https://doi.org/10.1016/j.biotechadv.2017.08.001">[DOI]</a>.</p></li>
<li id="id13747"><p>Axel von Kamp, Sven Thiele, Oliver Haedicke, and Steffen Klamt. Use of &lt;i&gt;cellnetanalyzer&lt;/i&gt; in biotechnology and metabolic engineering. <em>JOURNAL OF BIOTECHNOLOGY</em>, 261(SI):221–228, NOV 10 2017. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2017.05.001">[DOI]</a>.</p></li>
<li id="id13748"><p>Ayca Cankorur-Cetinkaya, Duygu Dikicioglu, and Stephen G. Oliver. Metabolic modeling to identify engineering targets for &lt;i&gt;komagataella phaffii&lt;/i&gt;: the effect of biomass composition on gene target identification. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 114(11):2605–2615, NOV 2017. <a class="reference external" href="https://doi.org/10.1002/bit.26380">[DOI]</a>.</p></li>
<li id="id13749"><p>Lauren T. Cordova, Robert M. Cipolla, Adti Swarup, Christopher P. Long, and Maciek R. Antoniewicz. &lt;sup&gt;13&lt;/sup&gt;c metabolic flux analysis of three divergent extremely thermophilic bacteria: &lt;i&gt;geobacillus&lt;/i&gt; sp lc300, &lt;i&gt;thermus thermophilus&lt;/i&gt; hb8, and &lt;i&gt;rhodothermus marinus&lt;/i&gt; dsm 4252. <em>METABOLIC ENGINEERING</em>, 44:182–190, NOV 2017. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2017.10.007">[DOI]</a>.</p></li>
<li id="id13750"><p>Jorge Fernandez-de-Cossio-Diaz, Kalet Leon, and Roberto Mulet. Characterizing steady states of genome-scale metabolic networks in continuous cell cultures. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2017. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1005835">[DOI]</a>.</p></li>
<li id="id13751"><p>Kiyotaka Y. Hara, Jyumpei Kobayashi, Ryosuke Yamada, Daisuke Sasaki, Yuki Kuriya, Yoko Hirono-Hara, Jun Ishii, Michihiro Araki, and Akihiko Kondo. Transporter engineering in biomass utilization by yeast. <em>FEMS YEAST RESEARCH</em>, NOV 2017. <a class="reference external" href="https://doi.org/10.1093/femsyr/fox061">[DOI]</a>.</p></li>
<li id="id13752"><p>Chintan J. Joshi, Christie A. M. Peebles, and Ashok Prasad. Modeling and analysis of flux distribution and bioproduct formation in &lt;i&gt;synechocystis&lt;/i&gt; sp pcc 6803 using a new genome-scale metabolic reconstruction. <em>ALGAL RESEARCH-BIOMASS BIOFUELS AND BIOPRODUCTS</em>, 27:295–310, NOV 2017. <a class="reference external" href="https://doi.org/10.1016/j.algal.2017.09.013">[DOI]</a>.</p></li>
<li id="id13753"><p>Christopher P. Long, Jacqueline E. Gonzalez, Adam M. Feist, Bernhard O. Palsson, and Maciek R. Antoniewicz. Fast growth phenotype of &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; k-12 from adaptive laboratory evolution does not require intracellular flux rewiring. <em>METABOLIC ENGINEERING</em>, 44:100–107, NOV 2017. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2017.09.012">[DOI]</a>.</p></li>
<li id="id13754"><p>Christopher P. Long, Jacqueline E. Gonzalez, Robert M. Cipolla, and Maciek R. Antoniewicz. Metabolism of the fast-growing bacterium &lt;i&gt;vibrio natriegens&lt;/i&gt; elucidated by &lt;sup&gt;13&lt;/sup&gt;c metabolic flux analysis. <em>METABOLIC ENGINEERING</em>, 44:191–197, NOV 2017. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2017.10.008">[DOI]</a>.</p></li>
<li id="id13755"><p>Xiao Pan, Ciska Dalm, Rene H. Wijffels, and Dirk E. Martens. Metabolic characterization of a cho cell size increase phase in fed-batch cultures. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 101(22):8101–8113, NOV 2017. <a class="reference external" href="https://doi.org/10.1007/s00253-017-8531-y">[DOI]</a>.</p></li>
<li id="id13756"><p>Jan-Philipp Schwarzhans, Tobias Luttermann, Martina Geier, Joern Kalinowski, and Karl Friehs. Towards systems metabolic engineering in &lt;i&gt;pichia pastoris&lt;/i&gt;. <em>BIOTECHNOLOGY ADVANCES</em>, 35(6):681–710, NOV 1 2017. <a class="reference external" href="https://doi.org/10.1016/j.biotechadv.2017.07.009">[DOI]</a>.</p></li>
<li id="id13757"><p>R. Adam Thompson and Cong T. Trinh. Overflow metabolism and growth cessation in &lt;i&gt;clostridium thermocellum&lt;/i&gt; dsm1313 during high cellulose loading fermentations. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 114(11):2592–2604, NOV 2017. <a class="reference external" href="https://doi.org/10.1002/bit.26374">[DOI]</a>.</p></li>
<li id="id13758"><p>Yangyang Zheng, Qianqian Yuan, Xiaoyan Yang, and Hongwu Ma. Engineering &lt;i&gt;escherichia coli&lt;/i&gt; for poly-(3-hydroxybutyrate) production guided by genome-scale metabolic network analysis. <em>ENZYME AND MICROBIAL TECHNOLOGY</em>, 106:60–66, NOV 2017. <a class="reference external" href="https://doi.org/10.1016/j.enzmictec.2017.07.003">[DOI]</a>.</p></li>
<li id="id13759"><p>Cristal Zuniga, Livia Zaramela, and Karsten Zengler. Elucidation of complexity and prediction of interactions in microbial communities. <em>MICROBIAL BIOTECHNOLOGY</em>, 10(6, SI):1500–1522, NOV 2017. <a class="reference external" href="https://doi.org/10.1111/1751-7915.12855">[DOI]</a>.</p></li>
<li id="id13760"><p>Alyaa M. Abdel-Haleem, Nathan E. Lewis, Neema Jamshidi, Katsuhiko Mineta, Xin Gao, and Takashi Gojobori. The emerging facets of non-cancerous warburg effect. <em>FRONTIERS IN ENDOCRINOLOGY</em>, OCT 23 2017. <a class="reference external" href="https://doi.org/10.3389/fendo.2017.00279">[DOI]</a>.</p></li>
<li id="id13761"><p>Michael Weiner, Julia Troendle, Christoph Albermann, Georg A. Sprenger, and Dirk Weuster-Botz. Metabolic control analysis of l-phenylalanine production from glycerol with engineered &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; using data from short-term steady-state perturbation experiments. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, 126:86–100, OCT 15 2017. <a class="reference external" href="https://doi.org/10.1016/j.bej.2017.06.016">[DOI]</a>.</p></li>
<li id="id13762"><p>Junhua Wang, Cheng Wang, Kejing Song, and Jianping Wen. Metabolic network model guided engineering ethylmalonyl-coa pathway to improve ascomycin production in &lt;i&gt;streptomyces&lt;/i&gt; &lt;i&gt;hygroscopicus&lt;/i&gt; var. &lt;i&gt;ascomyceticus&lt;/i&gt;. <em>MICROBIAL CELL FACTORIES</em>, OCT 3 2017. <a class="reference external" href="https://doi.org/10.1186/s12934-017-0787-5">[DOI]</a>.</p></li>
<li id="id13763"><p>Tanja Cvitanovic, Matthias C. Reichert, Miha Moskon, Miha Mraz, Frank Lammert, and Damjana Rozman. Large-scale computational models of liver metabolism: how far from the clinics? <em>HEPATOLOGY</em>, 66(4):1323–1334, OCT 2017. <a class="reference external" href="https://doi.org/10.1002/hep.29268">[DOI]</a>.</p></li>
<li id="id13764"><p>Filmon Eyassu and Claudio Angione. Modelling pyruvate dehydrogenase under hypoxia and its role in cancer metabolism. <em>ROYAL SOCIETY OPEN SCIENCE</em>, OCT 2017. <a class="reference external" href="https://doi.org/10.1098/rsos.170360">[DOI]</a>.</p></li>
<li id="id13765"><p>Xingxing Jian, Ningchuan Li, Qian Chen, and Qiang Hua. Model-guided identification of novel gene amplification targets for improving succinate production in &lt;i&gt;escherichia coli&lt;/i&gt; nzn111. <em>INTEGRATIVE BIOLOGY</em>, 9(10):830–835, OCT 1 2017. <a class="reference external" href="https://doi.org/10.1039/c7ib00077d">[DOI]</a>.</p></li>
<li id="id13766"><p>Tjerko Kamminga, Simen-Jan Slagman, Jetta J. E. Bijlsma, Vitor A. P. Martins dos Santos, Maria Suarez-Diez, and Peter J. Schaap. Metabolic modeling of energy balances in &lt;i&gt;mycoplasma hyopneumoniae&lt;/i&gt; shows that pyruvate addition increases growth rate. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 114(10):2339–2347, OCT 2017. <a class="reference external" href="https://doi.org/10.1002/bit.26347">[DOI]</a>.</p></li>
<li id="id13767"><p>Jonathan M. Monk, Colton J. Lloyd, Elizabeth Brunk, Nathan Mih, Anand Sastry, Zachary King, Rikiya Takeuchi, Wataru Nomura, Zhen Zhang, Hirotada Mori, Adam M. Feist, and Bernhard O. Palsson. &lt;i&gt;i&lt;/i&gt;ml1515, a knowledgebase that computes &lt;i&gt;escherichia coli&lt;/i&gt; traits. <em>NATURE BIOTECHNOLOGY</em>, 35(10):904–908, OCT 2017. <a class="reference external" href="https://doi.org/10.1038/nbt.3956">[DOI]</a>.</p></li>
<li id="id13768"><p>Jennifer L. Greene, Andreas Waechter, Keith E. J. Tyo, and Linda J. Broadbelt. Acceleration strategies to enhance metabolic ensemble modeling performance. <em>BIOPHYSICAL JOURNAL</em>, 113(5):1150–1162, SEP 5 2017. <a class="reference external" href="https://doi.org/10.1016/j.bpj.2017.07.018">[DOI]</a>.</p></li>
<li id="id13769"><p>Chiara Damiani, Riccardo Colombo, Daniela Gaglio, Fabrizia Mastroianni, Dario Pescini, Hans Victor Westerhoff, Giancarlo Mauri, Marco Vanoni, and Lilia Alberghina. A metabolic core model elucidates how enhanced utilization of glucose and glutamine, with enhanced glutamine-dependent lactate production, promotes cancer cell growth: the warburq effect. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2017. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1005758">[DOI]</a>.</p></li>
<li id="id13770"><p>Mehrdad Ghadiri, Mahshid Heidari, Sayed-Amir Marashi, and Seyed Hasan Mousavi. A multiscale agent-based framework integrated with a constraint-based metabolic network model of cancer for simulating avascular tumor growth. <em>MOLECULAR BIOSYSTEMS</em>, 13(9):1888–1897, SEP 1 2017. <a class="reference external" href="https://doi.org/10.1039/c7mb00050b">[DOI]</a>.</p></li>
<li id="id13771"><p>Piyush Labhsetwar, Marcelo C. R. Melo, John A. Cole, and Zaida Luthey-Schulten. Population fba predicts metabolic phenotypes in yeast. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2017. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1005728">[DOI]</a>.</p></li>
<li id="id13772"><p>Manon Morin, Delphine Ropers, Eugenio Cinquemani, Jean-Charles Portais, Brice Enjalbert, and Muriel Cocaign-Bousquet. The csr system regulates &lt;i&gt;escherichia coli&lt;/i&gt; fitness by controlling glycogen accumulation and energy levels. <em>MBIO</em>, SEP-OCT 2017. <a class="reference external" href="https://doi.org/10.1128/mBio.01628-17">[DOI]</a>.</p></li>
<li id="id13773"><p>Noora Ottman, Mark Davids, Maria Suarez-Diez, Sjef Boeren, Peter J. Schaap, Vitor A. P. Martins dos Santos, Hauke Smidt, Clara Belzer, and Willem M. de Vos. Genome-scale model and omics analysis of metabolic capacities of &lt;i&gt;akkermansia muciniphila&lt;/i&gt; reveal a preferential mucin-degrading lifestyle. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, SEP 2017. <a class="reference external" href="https://doi.org/10.1128/AEM.01014-17">[DOI]</a>.</p></li>
<li id="id13774"><p>Farhana R. Pinu and Silas G. Villas-Boas. Extracellular microbial metabolomics: the state of the art. <em>METABOLITES</em>, SEP 2017. <a class="reference external" href="https://doi.org/10.3390/metabo7030043">[DOI]</a>.</p></li>
<li id="id13775"><p>Ziva Rejc, Lidija Magdevska, Tilen Trselic, Timotej Osolin, Rok Vodopivec, Jakob Mraz, Eva Pavliha, Nikolaj Zimic, Tanja Cvitanovic, Damjana Rozman, Miha Moskon, and Miha Mraz. Computational modelling of genome-scale metabolic networks and its application to cho cell cultures. <em>COMPUTERS IN BIOLOGY AND MEDICINE</em>, 88:150–160, SEP 1 2017. <a class="reference external" href="https://doi.org/10.1016/j.compbiomed.2017.07.005">[DOI]</a>.</p></li>
<li id="id13776"><p>Ab Rauf Shah, Ahmad Ahmad, Shireesh Srivastava, and B. M. Jaffar Ali. Reconstruction and analysis of a genome-scale metabolic model of &lt;i&gt;nannochloropsis gaditana&lt;/i&gt;. <em>ALGAL RESEARCH-BIOMASS BIOFUELS AND BIOPRODUCTS</em>, 26:354–364, SEP 2017. <a class="reference external" href="https://doi.org/10.1016/j.algal.2017.08.014">[DOI]</a>.</p></li>
<li id="id13777"><p>Angelo Gamez-Pozo, Lucia Trilla-Fuertes, Julia Berges-Soria, Nathalie Selevsek, Rocio Lopez-Vacas, Mariana Diaz-Almiron, Paolo Nanni, Jorge M. Arevalillo, Hilario Navarro, Jonas Grossmann, Francisco Gaya Moreno, Ruben Gomez Rioja, Guillermo Prado-Vazquez, Andrea Zapater-Moros, Paloma Main, Jaime Feliu, Purificacion Martinez del Prado, Pilar Zamora, Eva Ciruelos, Enrique Espinosa, and Juan Angel Fresno Vara. Functional proteomics outlines the complexity of breast cancer molecular subtypes. <em>SCIENTIFIC REPORTS</em>, AUG 30 2017. <a class="reference external" href="https://doi.org/10.1038/s41598-017-10493-w">[DOI]</a>.</p></li>
<li id="id13778"><p>Neeraj Sinha, Maria Suarez-Diez, Evert M. van Schothorst, Jaap Keijer, Vitor A. P. Martins dos Santos, and Guido J. E. J. Hooiveld. Predicting the murine enterocyte metabolic response to diets that differ in lipid and carbohydrate composition. <em>SCIENTIFIC REPORTS</em>, AUG 18 2017. <a class="reference external" href="https://doi.org/10.1038/s41598-017-07350-1">[DOI]</a>.</p></li>
<li id="id13779"><p>Konstantinos Vavitsas, Emil Ostergaard Rue, Lara Kristin Stefansdottir, Thiyagarajan Gnanasekaran, Andreas Blennow, Christoph Crocoll, Steinn Gudmundsson, and Poul Erik Jensen. Responses of &lt;i&gt;synechocystis&lt;/i&gt; sp pcc 6803 to heterologous biosynthetic pathways. <em>MICROBIAL CELL FACTORIES</em>, AUG 15 2017. <a class="reference external" href="https://doi.org/10.1186/s12934-017-0757-y">[DOI]</a>.</p></li>
<li id="id13780"><p>Lisa Katharina Blass, Christian Weyler, and Elmar Heinzle. Network design and analysis for multi-enzyme biocatalysis. <em>BMC BIOINFORMATICS</em>, AUG 10 2017. <a class="reference external" href="https://doi.org/10.1186/s12859-017-1773-y">[DOI]</a>.</p></li>
<li id="id13781"><p>D. Joseph Sexton and Martin Schuster. Nutrient limitation determines the fitness of cheaters in bacterial siderophore cooperation. <em>NATURE COMMUNICATIONS</em>, AUG 10 2017. <a class="reference external" href="https://doi.org/10.1038/s41467-017-00222-2">[DOI]</a>.</p></li>
<li id="id13782"><p>Simon Menendez-Bravo, Julian Paganini, Claudio Avignone-Rossa, Hugo Gramajo, and Ana Arabolaza. Identification of fadab complexes involved in fatty acid β-oxidation in &lt;i&gt;streptomyces coelicolor&lt;/i&gt; and construction of a triacylglycerol overproducing strain. <em>FRONTIERS IN MICROBIOLOGY</em>, AUG 2 2017. <a class="reference external" href="https://doi.org/10.3389/fmicb.2017.01428">[DOI]</a>.</p></li>
<li id="id13783"><p>Nana Y. D. Ankrah, Junbo Luan, and Angela E. Douglas. Cooperative metabolism in a three-partner insect-bacterial symbiosis revealed by metabolic modeling. <em>JOURNAL OF BACTERIOLOGY</em>, AUG 2017. <a class="reference external" href="https://doi.org/10.1128/JB.00872-16">[DOI]</a>.</p></li>
<li id="id13784"><p>Maike K. Aurich, Ronan M. T. Fleming, and Ines Thiele. A systems approach reveals distinct metabolic strategies among the nci-60 cancer cell lines. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2017. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1005698">[DOI]</a>.</p></li>
<li id="id13785"><p>Maryam Iman, Tabassom Sobati, Yunes Panahi, and Meysam Mobasheri. Systems biology approach to bioremediation of nitroaromatics: constraint-based analysis of 2,4,6-trinitrotoluene biotransformation by &lt;i&gt;escherichia coli&lt;/i&gt;. <em>MOLECULES</em>, AUG 2017. <a class="reference external" href="https://doi.org/10.3390/molecules22081242">[DOI]</a>.</p></li>
<li id="id13786"><p>Alan J. Mueller, Mandy J. Peffers, Carole J. Proctor, and Peter D. Clegg. Systems approaches in osteoarthritis: identifying routes to novel diagnostic and therapeutic strategies. <em>JOURNAL OF ORTHOPAEDIC RESEARCH</em>, 35(8):1573–1588, AUG 2017. <a class="reference external" href="https://doi.org/10.1002/jor.23563">[DOI]</a>.</p></li>
<li id="id13787"><p>A. Pentjuss, E. Stalidzans, J. Liepins, A. Kokina, J. Martynova, P. Zikmanis, I. Mozga, R. Scherbaka, H. Hartman, M. G. Poolman, D. A. Fell, and A. Vigants. Model-based biotechnological potential analysis of &lt;i&gt;kluyveromyces marxianus&lt;/i&gt; central metabolism. <em>JOURNAL OF INDUSTRIAL MICROBIOLOGY &amp; BIOTECHNOLOGY</em>, 44(8):1177–1190, AUG 2017. <a class="reference external" href="https://doi.org/10.1007/s10295-017-1946-8">[DOI]</a>.</p></li>
<li id="id13788"><p>Benjamin J. Sanchez, Cheng Zhang, Avlant Nilsson, Petri-Jaan Lahtvee, Eduard J. Kerkhoven, and Jens Nielsen. Improving the phenotype predictions of a yeast genome-scale metabolic model by incorporating enzymatic constraints. <em>MOLECULAR SYSTEMS BIOLOGY</em>, AUG 2017. <a class="reference external" href="https://doi.org/10.15252/msb.20167411">[DOI]</a>.</p></li>
<li id="id13789"><p>Sarah Schatschneider, Jessica Schneider, Jochen Blom, Fabien Letisse, Karsten Niehaus, Alexander Goesmann, and Frank-Joerg Vorhoelter. Systems and synthetic biology perspective of the versatile plant-pathogenic and polysaccharide-producing bacterium &lt;i&gt;xanthomonas campestris&lt;/i&gt;. <em>MICROBIOLOGY-SGM</em>, 163(8):1117–1144, AUG 2017. <a class="reference external" href="https://doi.org/10.1099/mic.0.000473">[DOI]</a>.</p></li>
<li id="id13790"><p>Maureen A. Carey, Jason A. Papin, and Jennifer L. Guler. Novel &lt;i&gt;plasmodium falciparum&lt;/i&gt; metabolic network reconstruction identifies shifts associated with clinical antimalarial resistance. <em>BMC GENOMICS</em>, JUL 19 2017. <a class="reference external" href="https://doi.org/10.1186/s12864-017-3905-1">[DOI]</a>.</p></li>
<li id="id13791"><p>Chiara Damiani, Marzia Di Filippo, Dario Pescini, Davide Maspero, Riccardo Colombo, and Giancarlo Mauri. Popfba: tackling intratumour heterogeneity with flux balance analysis. <em>BIOINFORMATICS</em>, 33(14):I311–I318, JUL 15 2017. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btx251">[DOI]</a>.</p></li>
<li id="id13792"><p>Cheng Wang, Jiao Liu, Huanhuan Liu, Junhua Wang, and Jianping Wen. A genome-scale dynamic flux balance analysis model of &lt;i&gt;streptomyces tsukubaensis&lt;/i&gt; nrrl18488 to predict the targets for increasing fk506 production. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, 123:45–56, JUL 15 2017. <a class="reference external" href="https://doi.org/10.1016/j.bej.2017.03.017">[DOI]</a>.</p></li>
<li id="id13793"><p>Nicolas Loira, Sebastian Mendoza, Maria Paz Cortes, Natalia Rojas, Dante Travisany, Alex Di Genova, Natalia Gajardo, Nicole Ehrenfeld, and Alejandro Maass. Reconstruction of the microalga nannochloropsis salina genome-scale metabolic model with applications to lipid production. <em>BMC SYSTEMS BIOLOGY</em>, JUL 4 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-017-0441-1">[DOI]</a>.</p></li>
<li id="id13794"><p>Ran Chao, Shekhar Mishra, Tong Si, and Huimin Zhao. Engineering biological systems using automated biofoundries. <em>METABOLIC ENGINEERING</em>, 42:98–108, JUL 2017. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2017.06.003">[DOI]</a>.</p></li>
<li id="id13795"><p>Yu Chen, Yonghong Wang, and Jens Nielsen. Systematic inference of functional phosphorylation events in yeast metabolism. <em>BIOINFORMATICS</em>, 33(13):1995–2001, JUL 1 2017. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btx110">[DOI]</a>.</p></li>
<li id="id13796"><p>Yougan Cheng, Craig J. Thalhauser, Shepard Smithline, Jyotsna Pagidala, Marko Miladinov, Heather E. Vezina, Manish Gupta, Tarek A. Leil, and Brian J. Schmidt. Qsp toolbox: computational implementation of integrated workflow components for deploying multi-scale mechanistic models. <em>AAPS JOURNAL</em>, 19(4):1002–1016, JUL 2017. <a class="reference external" href="https://doi.org/10.1208/s12248-017-0100-x">[DOI]</a>.</p></li>
<li id="id13797"><p>Eun Sung Jung, Hye Min Park, Seung Min Hyun, Jong Cheol Shon, Meiyappan Lakshmanan, Minsoo Noh, Hock Chuan Yeo, Kwang-Hyeon Liu, Dong-Yup Lee, Jae Sung Hwang, and Choong Hwan Lee. Integrative metabolomic analysis reveals diet supplementation with green tea alleviates uvb-damaged mouse skin correlated with ascorbate metabolism and urea cycle. <em>METABOLOMICS</em>, JUL 2017. <a class="reference external" href="https://doi.org/10.1007/s11306-017-1218-7">[DOI]</a>.</p></li>
<li id="id13798"><p>Troy E. Sandberg, Colton J. Lloyd, Bernhard O. Palsson, and Adam M. Feist. Laboratory evolution to alternating substrate environments yields distinct phenotypic and genetic adaptive strategies. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, JUL 2017. <a class="reference external" href="https://doi.org/10.1128/AEM.00410-17">[DOI]</a>.</p></li>
<li id="id13799"><p>Yu Zhang, Jingyi Cai, Xiuling Shang, Bo Wang, Shuwen Liu, Xin Chai, Tianwei Tan, Yun Zhang, and Tingyi Wen. A new genome-scale metabolic model of &lt;i&gt;corynebacterium glutamicum&lt;/i&gt; and its application. <em>BIOTECHNOLOGY FOR BIOFUELS</em>, JUN 30 2017. <a class="reference external" href="https://doi.org/10.1186/s13068-017-0856-3">[DOI]</a>.</p></li>
<li id="id13800"><p>Alejandro Acevedo, Raul Conejeros, and German Aroca. Ethanol production improvement driven by genome-scale metabolic modeling and sensitivity analysis in &lt;i&gt;scheffersomyces stipitis&lt;/i&gt;. <em>PLOS ONE</em>, JUN 28 2017. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0180074">[DOI]</a>.</p></li>
<li id="id13801"><p>Sascha Schauble, Anne-Kristin Stavrum, Mathias Bockwoldt, Pal Puntervoll, and Ines Heiland. Sbmlmod: a python-based web application and web service for efficient data integration and model simulation. <em>BMC BIOINFORMATICS</em>, JUN 24 2017. <a class="reference external" href="https://doi.org/10.1186/s12859-017-1722-9">[DOI]</a>.</p></li>
<li id="id13802"><p>Sami Ben Said and Dani Or. Synthetic microbial ecology: engineering habitats for modular consortia. <em>FRONTIERS IN MICROBIOLOGY</em>, JUN 16 2017. <a class="reference external" href="https://doi.org/10.3389/fmicb.2017.01125">[DOI]</a>.</p></li>
<li id="id13803"><p>Luana Presta, Emanuele Bosi, Leila Mansouri, Lenie Dijkshoorn, Renato Fani, and Marco Fondi. Constraint-based modeling identifies new putative targets to fight colistin-resistant a. &lt;i&gt;baumannii&lt;/i&gt; infections. <em>SCIENTIFIC REPORTS</em>, JUN 16 2017. <a class="reference external" href="https://doi.org/10.1038/s41598-017-03416-2">[DOI]</a>.</p></li>
<li id="id13804"><p>Sebastian Weis, Ana Rita Carlos, Maria Raquel Moita, Sumnima Singh, Birte Blankenhaus, Silvia Cardoso, Rasmus Larsen, Sofia Rebelo, Sascha Schaeuble, Laura Del Barrio, Gilles Mithieux, Fabienne Rajas, Sandro Lindig, Michael Bauer, and Miguel P. Soares. Metabolic adaptation establishes disease tolerance to sepsis. <em>CELL</em>, 169(7):1263+, JUN 15 2017. <a class="reference external" href="https://doi.org/10.1016/j.cell.2017.05.031">[DOI]</a>.</p></li>
<li id="id13805"><p>German A. Preciat Gonzalez, Lemmer R. P. El Assal, Alberto Noronha, Ines Thiele, Hulda S. Haraldsdottir, and Ronan M. T. Fleming. Comparative evaluation of atom mapping algorithms for balanced metabolic reactions: application to recon 3d. <em>JOURNAL OF CHEMINFORMATICS</em>, JUN 14 2017. <a class="reference external" href="https://doi.org/10.1186/s13321-017-0223-1">[DOI]</a>.</p></li>
<li id="id13806"><p>Thierry Chenard, Frederic Guenard, Marie-Claude Vohl, Andre Carpentier, Andre Tchernof, and Rafael J. Najmanovich. Remodeling adipose tissue through in silico modulation of fat storage for the prevention of type 2 diabetes. <em>BMC SYSTEMS BIOLOGY</em>, 11:1–15, JUN 12 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-017-0438-9">[DOI]</a>.</p></li>
<li id="id13807"><p>Hulda S. Haraldsdottir, Ben Cousins, Ines Thiele, Ronan M. T. Fleming, and Santosh Vempala. Chrr: coordinate hit-and-run with rounding for uniform sampling of constraint-based models. <em>BIOINFORMATICS</em>, 33(11):1741–1743, JUN 1 2017. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btx052">[DOI]</a>.</p></li>
<li id="id13808"><p>Sarah B. Hintermayer and Dirk Weuster-Botz. Experimental validation of in silico estimated biomass yields of &lt;i&gt;pseudomonas putida&lt;/i&gt; kt2440. <em>BIOTECHNOLOGY JOURNAL</em>, JUN 2017. <a class="reference external" href="https://doi.org/10.1002/biot.201600720">[DOI]</a>.</p></li>
<li id="id13809"><p>Do Gyun Kim, Wang-Hee Lee, Sung-Won Seo, and Hye-Sun Park. Metabolic reconstruction for developing a tissue-specific model for cattle glycolysis using cobra. <em>JOURNAL OF BIOLOGICAL SYSTEMS</em>, 25(2):231–246, JUN 2017. <a class="reference external" href="https://doi.org/10.1142/S0218339017500127">[DOI]</a>.</p></li>
<li id="id13810"><p>Nicholas S. Kruyer and Pamela Peralta-Yahya. Metabolic engineering strategies to bio-adipic acid production. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 45:136–143, JUN 2017. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2017.03.006">[DOI]</a>.</p></li>
<li id="id13811"><p>Daniel Osorio, Janneth Gonzalez, and Andres Pinzon. Minval: an r package for minimal validation of stoichiometric reactions. <em>R JOURNAL</em>, 9(1):114–123, JUN 2017.</p></li>
<li id="id13812"><p>Petri-Jaan Lahtvee, Benjamin J. Sanchez, Agata Smialowska, Sergo Kasvandik, Ibrahim E. Elsemman, Francesco Gatto, and Jens Nielsen. Absolute quantification of protein and mrna abundances demonstrate variability in gene-specific translation efficiency in yeast. <em>CELL SYSTEMS</em>, 4(5):495+, MAY 24 2017. <a class="reference external" href="https://doi.org/10.1016/j.cels.2017.03.003">[DOI]</a>.</p></li>
<li id="id13813"><p>Kaspar Valgepea, Renato de Souza Pinto Lemgruber, Kieran Meaghan, Robin William Palfreyman, Tanus Abdalla, Bjom Daniel Heijstra, James Bruce Behrendorff, Ryan Tappel, Michael Kopke, Sean Dennis Simpson, Lars Keld Nielsen, and Esteban Marcellin. Maintenance of atp homeostasis triggers metabolic shifts in gas-fermenting acetogens. <em>CELL SYSTEMS</em>, 4(5):505+, MAY 24 2017. <a class="reference external" href="https://doi.org/10.1016/j.cels.2017.04.008">[DOI]</a>.</p></li>
<li id="id13814"><p>Faraaz Noor Khan Yusufi, Meiyappan Lakshmanan, Ying Swan Ho, Bernard Liat Wen Loo, Pramila Ariyaratne, Yuansheng Yang, Say Kong Ng, Tessa Rui Min Tan, Hock Chuan Yeo, Hsueh Lee Lim, Sze Wai Ng, Ai Ping Hiu, Chung Ping Chow, Corrine Wan, Shuwen Chen, Gavin Teo, Gao Song, Ju Xin Chin, Xiaoan Ruan, Ken Wing Kin Sung, Wei-Shou Hu, Miranda Gek Sim Yap, Muriel Bardor, Niranjan Nagarajan, and Dong-Yup Lee. Mammalian systems biotechnology reveals global cellular adaptations in a recombinant cho cell line. <em>CELL SYSTEMS</em>, 4(5):530+, MAY 24 2017. <a class="reference external" href="https://doi.org/10.1016/j.cels.2017.04.009">[DOI]</a>.</p></li>
<li id="id13815"><p>Milan J. A. van Hoek and Roeland M. H. Merks. Emergence of microbial diversity due to cross-feeding interactions in a spatial model of gut microbial metabolism. <em>BMC SYSTEMS BIOLOGY</em>, MAY 16 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-017-0430-4">[DOI]</a>.</p></li>
<li id="id13816"><p>Marco Fondi, Eva Pinatel, Adelfia Tala, Fabrizio Damiano, Clarissa Consolandi, Benedetta Mattorre, Daniela Fico, Mariangela Testini, Giuseppe E. De Benedetto, Luisa Siculella, Gianluca De Bellis, Pietro Alifano, and Clelia Peano. Time-resolved transcriptomics and constraint-based modeling identify system-level metabolic features and overexpression targets to increase spiramycin production in &lt;i&gt;streptomyces ambofaciens&lt;/i&gt;. <em>FRONTIERS IN MICROBIOLOGY</em>, MAY 12 2017. <a class="reference external" href="https://doi.org/10.3389/fmicb.2017.00835">[DOI]</a>.</p></li>
<li id="id13817"><p>Govindprasad Bhutada, Martin Kavscek, Rodrigo Ledesma-Amaro, Stephane Thomas, Gerald N. Rechberger, Jean-Marc Nicaud, and Klaus Natter. Sugar versus fat: elimination of glycogen storage improves lipid accumulation in &lt;i&gt;yarrowia lipolytica&lt;/i&gt;. <em>FEMS YEAST RESEARCH</em>, MAY 2017. <a class="reference external" href="https://doi.org/10.1093/femsyr/fox020">[DOI]</a>.</p></li>
<li id="id13818"><p>Laurent Heirendt, Ines Thiele, and Ronan M. T. Fleming. Distributedfba.jl: high-level, high-performance flux balance analysis in julia. <em>BIOINFORMATICS</em>, 33(9):1421–1423, MAY 1 2017. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btw838">[DOI]</a>.</p></li>
<li id="id13819"><p>M. Ahsanul Islam, Noushin Hadadi, Meric Ataman, Vassily Hatzimanikatis, and Gregory Stephanopoulos. Exploring biochemical pathways for mono-ethylene glycol (meg) synthesis from synthesis gas. <em>METABOLIC ENGINEERING</em>, 41:173–181, MAY 2017. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2017.04.005">[DOI]</a>.</p></li>
<li id="id13820"><p>Jimmy G. Lafontaine Rivera, Matthew K. Theisen, Po-Wei Chen, and James C. Liao. Kinetically accessible yield (kay) for redirection of metabolism to produce exo-metabolites. <em>METABOLIC ENGINEERING</em>, 41:144–151, MAY 2017. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2017.03.011">[DOI]</a>.</p></li>
<li id="id13821"><p>Mahesh Sharma, Naeem Shaikh, Shailendra Yadav, Sushma Singh, and Prabha Garg. A systematic reconstruction and constraint-based analysis of &lt;i&gt;leishmania donovani&lt;/i&gt; metabolic network: identification of potential antileishmanial drug targets. <em>MOLECULAR BIOSYSTEMS</em>, 13(5):955–969, MAY 1 2017. <a class="reference external" href="https://doi.org/10.1039/c6mb00823b">[DOI]</a>.</p></li>
<li id="id13822"><p>Kaspar Valgepea, Kim Q. Loi, James B. Behrendorff, Renato de S. P. Lemgruber, Manuel Plan, Mark P. Hodson, Michael Koepke, Lars K. Nielsen, and Esteban Marcellin. Arginine deiminase pathway provides atp and boosts growth of the gas-fermenting acetogen &lt;i&gt;clostridium autoethanogenum&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 41:202–211, MAY 2017. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2017.04.007">[DOI]</a>.</p></li>
<li id="id13823"><p>Deepanwita Banerjee, Dharmeshkumar Parmar, Nivedita Bhattacharya, Avinash D. Ghanate, Venkateswarlu Panchagnula, and Anu Raghunathan. A scalable metabolite supplementation strategy against antibiotic resistant pathogen &lt;i&gt;chromobacterium violaceum&lt;/i&gt; induced by nad&lt;sup&gt;+&lt;/sup&gt;/nadh&lt;sup&gt;+&lt;/sup&gt; imbalance. <em>BMC SYSTEMS BIOLOGY</em>, APR 26 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-017-0427-z">[DOI]</a>.</p></li>
<li id="id13824"><p>Michael A. Rowland, Ahmed Abdelzaher, Preetam Ghosh, and Michael L. Mayo. Crosstalk and the dynamical modularity of feed-forward loops in transcriptional regulatory networks. <em>BIOPHYSICAL JOURNAL</em>, 112(8):1539–1550, APR 25 2017. <a class="reference external" href="https://doi.org/10.1016/j.bpj.2017.02.044">[DOI]</a>.</p></li>
<li id="id13825"><p>Abraham A. Labena, Yuan-Nong Ye, Chuan Dong, Fa-Z Zhang, and Feng-Biao Guo. Sser: species specific essential reactions database. <em>BMC SYSTEMS BIOLOGY</em>, APR 19 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-017-0426-0">[DOI]</a>.</p></li>
<li id="id13826"><p>Aida Kalantari, Tao Chen, Boyang Ji, Ivan A. Stancik, Vaishnavi Ravikumar, Damjan Franjevic, Claire Saulou-Berion, Anne Goelzer, and Ivan Mijakovic. Conversion of glycerol to 3-hydroxypropanoic acid by genetically engineered &lt;i&gt;bacillus subtilis&lt;/i&gt;. <em>FRONTIERS IN MICROBIOLOGY</em>, APR 18 2017. <a class="reference external" href="https://doi.org/10.3389/fmicb.2017.00638">[DOI]</a>.</p></li>
<li id="id13827"><p>Cecilio Valadez-Cano, Roberto Olivares-Hernandez, Osbaldo Resendis-Antonio, Alexander DeLuna, and Luis Delaye. Natural selection drove metabolic specialization of the chromatophore in paulinella chromatophora. <em>BMC EVOLUTIONARY BIOLOGY</em>, APR 14 2017. <a class="reference external" href="https://doi.org/10.1186/s12862-017-0947-6">[DOI]</a>.</p></li>
<li id="id13828"><p>Aarash Bordbar, James T. Yurkovich, Giuseppe Paglia, Ottar Rolfsson, Olafur E. Sigurjonsson, and Bernhard O. Palsson. Elucidating dynamic metabolic physiology through network integration of quantitative time-course metabolomics. <em>SCIENTIFIC REPORTS</em>, APR 7 2017. <a class="reference external" href="https://doi.org/10.1038/srep46249">[DOI]</a>.</p></li>
<li id="id13829"><p>Chao Ye, Nan Xu, Chuan Dong, Yuannong Ye, Xuan Zou, Xiulai Chen, Fengbiao Guo, and Liming Liu. Imgmd: a platform for the integration and standardisation of &lt;i&gt;in silico&lt;/i&gt; microbial genome-scale metabolic models. <em>SCIENTIFIC REPORTS</em>, APR 7 2017. <a class="reference external" href="https://doi.org/10.1038/s41598-017-00820-6">[DOI]</a>.</p></li>
<li id="id13830"><p>Alfredo Braunstein, Anna Paola Muntoni, and Andrea Pagnani. An analytic approximation of the feasible space of metabolic networks. <em>NATURE COMMUNICATIONS</em>, APR 6 2017. <a class="reference external" href="https://doi.org/10.1038/ncomms14915">[DOI]</a>.</p></li>
<li id="id13831"><p>Garrett W. Birkel, Amit Ghosh, Vinay S. Kumar, Daniel Weaver, David Ando, Tyler W. H. Backman, Adam P. Arkin, Jay D. Keasling, and Hector Garcia Martin. The jbei quantitative metabolic modeling library (jqmm): a python library for modeling microbial metabolism. <em>BMC BIOINFORMATICS</em>, APR 5 2017. <a class="reference external" href="https://doi.org/10.1186/s12859-017-1615-y">[DOI]</a>.</p></li>
<li id="id13832"><p>Jun Feng, Jing Yang, Xiaorong Li, Meijin Guo, Bochu Wang, Shang-tian Yang, and Xiang Zou. Reconstruction of a genome-scale metabolic model and &lt;i&gt;in silico&lt;/i&gt; analysis of the polymalic acid producer &lt;i&gt;aureobasidium pullulans&lt;/i&gt; cctcc m2012223. <em>GENE</em>, 607:1–8, APR 5 2017. <a class="reference external" href="https://doi.org/10.1016/j.gene.2016.12.034">[DOI]</a>.</p></li>
<li id="id13833"><p>Evan Appleton, Curtis Madsen, Nicholas Roehner, and Douglas Densmore. Design automation in synthetic biology. <em>COLD SPRING HARBOR PERSPECTIVES IN BIOLOGY</em>, APR 2017. <a class="reference external" href="https://doi.org/10.1101/cshperspect.a023978">[DOI]</a>.</p></li>
<li id="id13834"><p>Helena Mendes-Soares and Nicholas Chia. Community metabolic modeling approaches to understanding the gut microbiome: bridging biochemistry and ecology. <em>FREE RADICAL BIOLOGY AND MEDICINE</em>, 105(SI):102–109, APR 2017. <a class="reference external" href="https://doi.org/10.1016/j.freeradbiomed.2016.12.017">[DOI]</a>.</p></li>
<li id="id13835"><p>Sjoerd Opdam, Anne Richelle, Benjamin Kellman, Shanzhong Li, Daniel C. Zielinski, and Nathan E. Lewis. A systematic evaluation of methods for tailoring genome-scale metabolic models. <em>CELL SYSTEMS</em>, 4(3):318–329, MAR 22 2017. <a class="reference external" href="https://doi.org/10.1016/j.cels.2017.01.010">[DOI]</a>.</p></li>
<li id="id13836"><p>Michael MacGillivray, Amy Ko, Emily Gruber, Miranda Sawyer, Eivind Almaas, and Allen Holder. Robust analysis of fluxes in genome-scale metabolic pathways. <em>SCIENTIFIC REPORTS</em>, MAR 21 2017. <a class="reference external" href="https://doi.org/10.1038/s41598-017-00170-3">[DOI]</a>.</p></li>
<li id="id13837"><p>Jennifer A. Bartell, Anna S. Blazier, Phillip Yen, Juliane C. Thogersen, Lars Jelsbak, Joanna B. Goldberg, and Jason A. Papin. Reconstruction of the metabolic network of &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt; to interrogate virulence factor synthesis. <em>NATURE COMMUNICATIONS</em>, MAR 7 2017. <a class="reference external" href="https://doi.org/10.1038/ncomms14631">[DOI]</a>.</p></li>
<li id="id13838"><p>Balaji Balagurunathan, Vishist Kumar Jain, Crystal Jing Ying Tear, Chan Yuen Lim, and Hua Zhao. In silico design of anaerobic growth-coupled product formation in &lt;i&gt;escherichia coli&lt;/i&gt;: experimental validation using a simple polyol, glycerol. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 40(3):361–372, MAR 2017. <a class="reference external" href="https://doi.org/10.1007/s00449-016-1703-9">[DOI]</a>.</p></li>
<li id="id13839"><p>Keith Dufault-Thompson, Huahua Jian, Ruixue Cheng, Jiefu Li, Fengping Wang, and Ying Zhang. A genome-scale model of &lt;i&gt;shewanella piezotolerans&lt;/i&gt; simulates mechanisms of metabolic diversity and energy conservation. <em>MSYSTEMS</em>, MAR-APR 2017. <a class="reference external" href="https://doi.org/10.1128/mSystems.00165-16">[DOI]</a>.</p></li>
<li id="id13840"><p>Kalyani Korla and Nagasuma Chandra. A systems perspective of signalling networks in host-pathogen interactions. <em>JOURNAL OF THE INDIAN INSTITUTE OF SCIENCE</em>, 97(1):41–57, MAR 2017. <a class="reference external" href="https://doi.org/10.1007/s41745-016-0017-x">[DOI]</a>.</p></li>
<li id="id13841"><p>Hongzhong Lu, Weiqiang Cao, Liming Ouyang, Jianye Xia, Mingzhi Huang, Ju Chu, Yingping Zhuang, Siliang Zhang, and Henk Noorman. Comprehensive reconstruction and in silico analysis of &lt;i&gt;aspergillus niger&lt;/i&gt; genome-scale metabolic network model that accounts for 1210 orfs. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 114(3):685–695, MAR 2017. <a class="reference external" href="https://doi.org/10.1002/bit.26195">[DOI]</a>.</p></li>
<li id="id13842"><p>Fredrick M. Mobegi, Aldert Zomer, Marien I. de Jonge, and Sacha A. F. T. van Hijum. Advances and perspectives in computational prediction of microbial gene essentiality. <em>BRIEFINGS IN FUNCTIONAL GENOMICS</em>, 16(2):70–79, MAR 2017. <a class="reference external" href="https://doi.org/10.1093/bfgp/elv063">[DOI]</a>.</p></li>
<li id="id13843"><p>Yi Yang, Xiao-Pan Hu, and Bin-Guang Ma. Construction and simulation of the &lt;i&gt;bradyrhizobium diazoefficiens&lt;/i&gt; usda110 metabolic network: a comparison between free-living and symbiotic states. <em>MOLECULAR BIOSYSTEMS</em>, 13(3):607–620, MAR 1 2017. <a class="reference external" href="https://doi.org/10.1039/c6mb00553e">[DOI]</a>.</p></li>
<li id="id13844"><p>Francisco Saitua, Paulina Torres, Jose Ricardo Perez-Correa, and Eduardo Agosin. Dynamic genome-scale metabolic modeling of the yeast &lt;i&gt;pichia pastoris&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, FEB 21 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-017-0408-2">[DOI]</a>.</p></li>
<li id="id13845"><p>Sara Saheb Kashaf, Claudio Angione, and Pietro Lio. Making life difficult for clostridium difficile: augmenting the pathogen's metabolic model with transcriptomic and codon usage data for better therapeutic target characterization. <em>BMC SYSTEMS BIOLOGY</em>, FEB 16 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-017-0395-3">[DOI]</a>.</p></li>
<li id="id13846"><p>Alberto Noronha, Anna Drofn Danielsdottir, Piotr Gawron, Freyr Johannsson, Soffia Jonsdottir, Sindri Jarlsson, Jon Petur Gunnarsson, Sigurdur Brynjolfsson, Reinhard Schneider, Ines Thiele, and Ronan M. T. Fleming. Reconmap: an interactive visualization of human metabolism. <em>BIOINFORMATICS</em>, 33(4):605–607, FEB 15 2017. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btw667">[DOI]</a>.</p></li>
<li id="id13847"><p>Zhaleh Hosseini and Sayed-Amir Marashi. Discovering missing reactions of metabolic networks by using gene co-expression data. <em>SCIENTIFIC REPORTS</em>, FEB 2 2017. <a class="reference external" href="https://doi.org/10.1038/srep41774">[DOI]</a>.</p></li>
<li id="id13848"><p>James A. Cotton, Sasisekhar Bennuru, Alexandra Grote, Bhavana Harsha, Alan Tracey, Robin Beech, Stephen R. Doyle, Matthew Dunn, Julie C. Dunning Hotopp, Nancy Holroyd, Taisei Kikuchi, Olivia Lambert, Amruta Mhashilkar, Prudence Mutowo, Nirvana Nursimulu, Jose M. C. Ribeiro, Matthew B. Rogers, Eleanor Stanley, Lakshmipuram S. Swapna, Isheng J. Tsai, Thomas R. Unnasch, Denis Voronin, John Parkinson, Thomas B. Nutman, Elodie Ghedin, Matthew Berriman, and Sara Lustigman. The genome of &lt;i&gt;onchocerca volvulus&lt;/i&gt;, agent of river blindness. <em>NATURE MICROBIOLOGY</em>, FEB 2017. <a class="reference external" href="https://doi.org/10.1038/nmicrobiol.2016.216">[DOI]</a>.</p></li>
<li id="id13849"><p>Lanqing Dang, Jiao Liu, Cheng Wang, Huanhuan Liu, and Jianping Wen. Enhancement of rapamycin production by metabolic engineering in &lt;i&gt;streptomyces hygroscopicus&lt;/i&gt; based on genome-scale metabolic model. <em>JOURNAL OF INDUSTRIAL MICROBIOLOGY &amp; BIOTECHNOLOGY</em>, 44(2):259–270, FEB 2017. <a class="reference external" href="https://doi.org/10.1007/s10295-016-1880-1">[DOI]</a>.</p></li>
<li id="id13850"><p>Di Liu, Ni Wan, Fuzhong Zhang, Yinjie J. Tang, and Stephen G. Wu. Enhancing fatty acid production in &lt;i&gt;escherichia coli&lt;/i&gt; by &lt;i&gt;vitreoscilla&lt;/i&gt; hemoglobin overexpression. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 114(2):463–467, FEB 2017. <a class="reference external" href="https://doi.org/10.1002/bit.26067">[DOI]</a>.</p></li>
<li id="id13851"><p>Lili Liu, Zijun Zhang, Taotao Sheng, and Ming Chen. Def: an automated dead-end filling approach based on quasi-endosymbiosis. <em>BIOINFORMATICS</em>, 33(3):405–413, FEB 1 2017. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btw604">[DOI]</a>.</p></li>
<li id="id13852"><p>Keisuke Wada, Yoshihiro Toya, Satomi Banno, Katsunori Yoshikawa, Fumio Matsuda, and Hiroshi Shimizu. &lt;sup&gt;13&lt;/sup&gt;c-metabolic flux analysis for mevalonate-producing strain of &lt;i&gt;escherichia coli&lt;/i&gt;. <em>JOURNAL OF BIOSCIENCE AND BIOENGINEERING</em>, 123(2):177–182, FEB 2017. <a class="reference external" href="https://doi.org/10.1016/j.jbiosc.2016.08.001">[DOI]</a>.</p></li>
<li id="id13853"><p>Andreas Wagner. The white-knight hypothesis, or does the environment limit innovations? <em>TRENDS IN ECOLOGY &amp; EVOLUTION</em>, 32(2):131–140, FEB 2017. <a class="reference external" href="https://doi.org/10.1016/j.tree.2016.10.017">[DOI]</a>.</p></li>
<li id="id13854"><p>Jason M. Whitham, Mark J. Schulte, Benjamin G. Bobay, Jose M. Bruno-Barcena, Mari S. Chinn, Michael C. Flickinger, Joel J. Pawlak, and Amy M. Grunden. Characterization of &lt;i&gt;clostridium ljungdahlii&lt;/i&gt; ota1: a non-autotrophic hyper ethanol-producing strain. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 101(4):1615–1630, FEB 2017. <a class="reference external" href="https://doi.org/10.1007/s00253-016-7978-6">[DOI]</a>.</p></li>
<li id="id13855"><p>Thomas J. Mueller, Justin L. Ungerer, Himadri B. Pakrasi, and Costas D. Maranas. Identifying the metabolic differences of a fast-growth phenotype in &lt;i&gt;synechococcus&lt;/i&gt; utex 2973. <em>SCIENTIFIC REPORTS</em>, JAN 31 2017. <a class="reference external" href="https://doi.org/10.1038/srep41569">[DOI]</a>.</p></li>
<li id="id13856"><p>Joseph J. Gardner and Nanette R. Boyle. The use of genome-scale metabolic network reconstruction to predict fluxes and equilibrium composition of n-fixing versus c-fixing cells in a diazotrophic cyanobacterium, &lt;i&gt;trichodesmium erythraeum&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, JAN 19 2017. <a class="reference external" href="https://doi.org/10.1186/s12918-016-0383-z">[DOI]</a>.</p></li>
<li id="id13857"><p>Ding Ma, Laurence Yang, Ronan M. T. Fleming, Ines Thiele, Bernhard O. Palsson, and Michael A. Saunders. Reliable and efficient solution of genome-scale models of metabolism and macromolecular expression. <em>SCIENTIFIC REPORTS</em>, JAN 18 2017. <a class="reference external" href="https://doi.org/10.1038/srep40863">[DOI]</a>.</p></li>
<li id="id13858"><p>Qianqian Yuan, Teng Huang, Peishun Li, Tong Hao, Feiran Li, Hongwu Ma, Zhiwen Wang, Xueming Zhao, Tao Chen, and Igor Goryanin. Pathway-consensus approach to metabolic network reconstruction for &lt;i&gt;pseudomonas putida&lt;/i&gt; kt2440 by systematic comparison of published models. <em>PLOS ONE</em>, JAN 13 2017. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0169437">[DOI]</a>.</p></li>
<li id="id13859"><p>Aleksandr Demidenko, Ilya R. Akberdin, Marco Allemann, Eric E. Allen, and Marina G. Kalyuzhnaya. Fatty acid biosynthesis pathways in &lt;i&gt;methylomicrobium buryatense&lt;/i&gt; &lt;i&gt;5g&lt;/i&gt;(&lt;i&gt;b1&lt;/i&gt;). <em>FRONTIERS IN MICROBIOLOGY</em>, JAN 10 2017. <a class="reference external" href="https://doi.org/10.3389/fmicb.2016.02167">[DOI]</a>.</p></li>
<li id="id13860"><p>Jozef Anne, Anastassios Economou, and Kristel Bernaerts. Protein secretion in gram-positive bacteria: from multiple pathways to biotechnology. In F Bagnoli and R Rappuoli, editors, <em>PROTEIN AND SUGAR EXPORT AND ASSEMBLY IN GRAM-POSITIVE BACTERIA</em>, volume 404 of Current Topics in Microbiology and Immunology, pages 267–308. 2017. <a class="reference external" href="https://doi.org/10.1007/82\_2016\_49">[DOI]</a>.</p></li>
<li id="id13861"><p>Arvand Asghari, Sayed-Amir Marashi, and Naser Ansari-Pour. A sperm-specific proteome-scale metabolic network model identifies non-glycolytic genes for energy deficiency in asthenozoospermia. <em>SYSTEMS BIOLOGY IN REPRODUCTIVE MEDICINE</em>, 63(2):100–112, 2017. <a class="reference external" href="https://doi.org/10.1080/19396368.2016.1263367">[DOI]</a>.</p></li>
<li id="id13862"><p>Pablo Carbonell, Oriol Lopez, Alexander Amberg, Manuel Pastor, and Ferran Sanz. Hepatotoxicity prediction by systems biology modeling of disturbed metabolic pathways using gene expression data. <em>ALTEX-ALTERNATIVES TO ANIMAL EXPERIMENTATION</em>, 34(2):219–234, 2017. <a class="reference external" href="https://doi.org/10.14573/altex.1602071">[DOI]</a>.</p></li>
<li id="id13863"><p>Anirikh Chakrabarti, Mathieu Membrez, Delphine Morin-Rivron, Jay Siddharth, Chieh Jason Chou, Hugues Henry, Stephen Bruce, Sylviane Metairon, Frederic Raymond, Bertrand Betrisey, Carole Loyer, Scott J. Parkinson, and Mojgan Masoodi. Transcriptomics-driven lipidomics (tdl) identifies the microbiome-regulated targets of ileal lipid metabolism. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, 2017. <a class="reference external" href="https://doi.org/10.1038/s41540-017-0033-0">[DOI]</a>.</p></li>
<li id="id13864"><p>Sanjeev Dahal, Suresh Poudel, and R. Adam Thompson. Genome-scale modeling of thermophilic microorganisms. In I Nookaew, editor, <em>NETWORK BIOLOGY</em>, volume 160 of Advances in Biochemical Engineering-Biotechnology, pages 103–119. 2017. <a class="reference external" href="https://doi.org/10.1007/10\_2016\_45">[DOI]</a>.</p></li>
<li id="id13865"><p>Jacqueline E. Gonzalez, Christopher P. Long, and Maciek R. Antoniewicz. Comprehensive analysis of glucose and xylose metabolism in &lt;i&gt;escherichia coli&lt;/i&gt; under aerobic and anaerobic conditions by &lt;sup&gt;13&lt;/sup&gt;c metabolic flux analysis. <em>METABOLIC ENGINEERING</em>, 39:9–18, JAN 2017. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2016.11.003">[DOI]</a>.</p></li>
<li id="id13866"><p>Amornpan Klanchui, Nachon Raethong, Peerada Prommeenate, Wanwipa Vongsangnak, and Asawin Meechai. Cyanobacterial biofuels: strategies and developments on network and modeling. In I Nookaew, editor, <em>NETWORK BIOLOGY</em>, volume 160 of Advances in Biochemical Engineering-Biotechnology, pages 75–102. 2017. <a class="reference external" href="https://doi.org/10.1007/10\_2016\_42">[DOI]</a>.</p></li>
<li id="id13867"><p>Jennifer Levering, Christopher L. Dupont, Andrew E. Allen, Bernhard O. Palsson, and Karsten Zengler. Integrated regulatory and metabolic networks of the marine diatom &lt;i&gt;phaeodactylum tricornutum&lt;/i&gt; predict the response to rising co&lt;sub&gt;2&lt;/sub&gt; levels. <em>MSYSTEMS</em>, JAN-FEB 2017. <a class="reference external" href="https://doi.org/10.1128/mSystems.00142-16">[DOI]</a>.</p></li>
<li id="id13868"><p>Stefania Magnusdottir, Almut Heinken, Laura Kutt, Dmitry A. Ravcheev, Eugen Bauer, Alberto Noronha, Kacy Greenhalgh, Christian Jager, Joanna Baginska, Paul Wilmes, Ronan M. T. Fleming, and Ines Thiele. Generation of genome-scale metabolic reconstructions for 773 members of the human gut microbiota. <em>NATURE BIOTECHNOLOGY</em>, 35(1):81–89, JAN 2017. <a class="reference external" href="https://doi.org/10.1038/nbt.3703">[DOI]</a>.</p></li>
<li id="id13869"><p>David Malatinszky, Ralf Steuer, and Patrik R. Jones. A comprehensively curated genome-scale two-cell model for the heterocystous cyanobacterium &lt;i&gt;anabaena&lt;/i&gt; sp pcc 7120. <em>PLANT PHYSIOLOGY</em>, 173(1):509–523, JAN 2017. <a class="reference external" href="https://doi.org/10.1104/pp.16.01487">[DOI]</a>.</p></li>
<li id="id13870"><p>Bashir Sajo Mienda. Genome-scale metabolic models as platforms for strain design and biological discovery. <em>JOURNAL OF BIOMOLECULAR STRUCTURE &amp; DYNAMICS</em>, 35(9):1863–1873, 2017. <a class="reference external" href="https://doi.org/10.1080/07391102.2016.1197153">[DOI]</a>.</p></li>
<li id="id13871"><p>Ildiko Miklossy, Zsolt Bodor, Reka Sinkler, Kalman Csongor Orban, Szabolcs Lanyi, and Beata Albert. &lt;i&gt;in silico&lt;/i&gt; and &lt;i&gt;in vivo&lt;/i&gt; stability analysis of a heterologous biosynthetic pathway for 1,4-butanediol production in metabolically engineered &lt;i&gt;e-coli&lt;/i&gt;. <em>JOURNAL OF BIOMOLECULAR STRUCTURE &amp; DYNAMICS</em>, 35(9):1874–1889, 2017. <a class="reference external" href="https://doi.org/10.1080/07391102.2016.1198721">[DOI]</a>.</p></li>
<li id="id13872"><p>Nicholas Ribaudo, Xianhua Li, Brett Davis, Thomas K. Wood, and Zuyi (Jacky) Huang. A genome-scale modeling approach to quantify biofilm component growth of &lt;i&gt;salmonella typhimurium&lt;/i&gt;. <em>JOURNAL OF FOOD SCIENCE</em>, 82(1):154–166, JAN 2017. <a class="reference external" href="https://doi.org/10.1111/1750-3841.13565">[DOI]</a>.</p></li>
<li id="id13873"><p>Borja Sanchez, Susana Delgado, Aitor Blanco-Miguez, Analia Lourenco, Miguel Gueimonde, and Abelardo Margolles. Probiotics, gut microbiota, and their influence on host health and disease. <em>MOLECULAR NUTRITION &amp; FOOD RESEARCH</em>, JAN 2017. <a class="reference external" href="https://doi.org/10.1002/mnfr.201600240">[DOI]</a>.</p></li>
<li id="id13874"><p>F. Sannino, M. Giuliani, U. Salvatore, G. A. Apuzzo, D. de Pascale, R. Fani, M. Fondi, G. Marino, M. L. Tutino, and E. Parrilli. A novel synthetic medium and expression system for subzero growth and recombinant protein production in &lt;i&gt;pseudoalteromonas haloplanktis&lt;/i&gt; tac125. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 101(2):725–734, JAN 2017. <a class="reference external" href="https://doi.org/10.1007/s00253-016-7942-5">[DOI]</a>.</p></li>
<li id="id13875"><p>ShengShee Thor, Joseph R. Peterson, and Zaida Luthey-Schulten. Genome-scale metabolic modeling of archaea lends insight into diversity of metabolic function. <em>ARCHAEA-AN INTERNATIONAL MICROBIOLOGICAL JOURNAL</em>, 2017. <a class="reference external" href="https://doi.org/10.1155/2017/9763848">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-8" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-8" name="0-8" role="tabpanel" tabindex="0"><div class="docutils container" id="id14561">
<ol class="arabic simple" start="1">
<li id="id15696"><p>Jared T. Broddrick, Benjamin E. Rubin, David G. Welkie, Niu Du, Nathan Mih, Spencer Diamond, Jenny J. Lee, Susan S. Golden, and Bernhard O. Palsson. Unique attributes of cyanobacterial metabolism revealed by improved genome-scale metabolic modeling and essential gene analysis. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 113(51):E8344–E8353, DEC 20 2016. <a class="reference external" href="https://doi.org/10.1073/pnas.1613446113">[DOI]</a>.</p></li>
<li id="id15697"><p>Jasper J. Koehorst, Jesse C. J. van Dam, Ruben G. A. van Heck, Edoardo Saccenti, Vitor A. P. Martins dos Santos, Maria Suarez-Diez, and Peter J. Schaap. Comparison of 432 &lt;i&gt;pseudomonas&lt;/i&gt; strains through integration of genomic, functional, metabolic and expression data. <em>SCIENTIFIC REPORTS</em>, DEC 6 2016. <a class="reference external" href="https://doi.org/10.1038/srep38699">[DOI]</a>.</p></li>
<li id="id15698"><p>Weiqi Fu, Amphun Chaiboonchoe, Basel Khraiwesh, David R. Nelson, Dina Al-Khairy, Alexandra Mystikou, Amnah Alzahmi, and Kourosh Salehi-Ashtiani. Algal cell factories: approaches, applications, and potentials. <em>MARINE DRUGS</em>, DEC 2016. <a class="reference external" href="https://doi.org/10.3390/md14120225">[DOI]</a>.</p></li>
<li id="id15699"><p>Kristopher A. Hunt, Ryan deM. Jennings, William P. Inskeep, and Ross P. Carlson. Stoichiometric modelling of assimilatory and dissimilatory biomass utilisation in a microbial community. <em>ENVIRONMENTAL MICROBIOLOGY</em>, 18(12):4946–4960, DEC 2016. <a class="reference external" href="https://doi.org/10.1111/1462-2920.13444">[DOI]</a>.</p></li>
<li id="id15700"><p>Xianhua Li, Yanhong Liu, Qian Jia, Virginia LaMacchia, Kathryn O'Donoghue, and Zuyi Huang. A systems biology approach to investigate the antimicrobial activity of oleuropein. <em>JOURNAL OF INDUSTRIAL MICROBIOLOGY &amp; BIOTECHNOLOGY</em>, 43(12):1705–1717, DEC 2016. <a class="reference external" href="https://doi.org/10.1007/s10295-016-1841-8">[DOI]</a>.</p></li>
<li id="id15701"><p>V. B. Panichkin, V. A. Livshits, I. V. Biryukova, and S. V. Mashko. Metabolic engineering of &lt;i&gt;escherichia coli&lt;/i&gt; for l-tryptophan production. <em>APPLIED BIOCHEMISTRY AND MICROBIOLOGY</em>, 52(9):783–809, DEC 2016. <a class="reference external" href="https://doi.org/10.1134/S0003683816090052">[DOI]</a>.</p></li>
<li id="id15702"><p>Matthew A. Richards, Thomas J. Lie, Juan Zhang, Stephen W. Ragsdale, John A. Leigh, and Nathan D. Price. Exploring hydrogenotrophic methanogenesis: a genome scale metabolic reconstruction of &lt;i&gt;methanococcus&lt;/i&gt; &lt;i&gt;maripaludis&lt;/i&gt;. <em>JOURNAL OF BACTERIOLOGY</em>, 198(24):3379–3390, DEC 2016. <a class="reference external" href="https://doi.org/10.1128/JB.00571-16">[DOI]</a>.</p></li>
<li id="id15703"><p>Marco Fondi, Emanuele Bosi, Luana Presta, Diletta Natoli, and Renato Fani. Modelling microbial metabolic rewiring during growth in a complex medium. <em>BMC GENOMICS</em>, NOV 24 2016. <a class="reference external" href="https://doi.org/10.1186/s12864-016-3311-0">[DOI]</a>.</p></li>
<li id="id15704"><p>Hooman Hefzi, Kok Siong Ang, Michael Hanscho, Aarash Bordbar, David Ruckerbauer, Meiyappan Lakshmanan, Camila A. Orellana, Deniz Baycin-Hizal, Yingxiang Huang, Daniel Ley, Veronica S. Martinez, Sarantos Kyriakopoulos, Natalia E. Jimenez, Daniel C. Zielinski, Lake-Ee Quek, Tune Wulff, Johnny Arnsdorf, Shangzhong Li, Jae Seong Lee, Giuseppe Paglia, Nicolas Loira, Philipp N. Spahn, Lasse E. Pedersen, Jahir M. Gutierrez, Zachary A. King, Anne Mathilde Lund, Harish Nagarajan, Alex Thomas, Alyaa M. Abdel-Haleem, Juergen Zanghellini, Helene F. Kildegaard, Bjorn G. Voldborg, Ziomara P. Gerdtzen, Michael J. Betenbaugh, Bernhard O. Palsson, Mikael R. Andersen, Lars K. Nielsen, Nicole Borth, Dong-Yup Lee, and Nathan E. Lewis. A consensus genome-scale reconstruction of chinese hamster ovary cell metabolism. <em>CELL SYSTEMS</em>, 3(5):434+, NOV 23 2016. <a class="reference external" href="https://doi.org/10.1016/j.cels.2016.10.020">[DOI]</a>.</p></li>
<li id="id15705"><p>Sandra Castillo, Dorothee Barth, Mikko Arvas, Tiina M. Pakula, Esa Pitkanen, Peter Blomberg, Tuulikki Seppanen-Laakso, Heli Nygren, Dhinakaran Sivasiddarthan, Merja Penttila, and Merja Oja. Whole-genome metabolic model of &lt;i&gt;trichoderma reesei&lt;/i&gt; built by comparative reconstruction. <em>BIOTECHNOLOGY FOR BIOFUELS</em>, NOV 21 2016. <a class="reference external" href="https://doi.org/10.1186/s13068-016-0665-0">[DOI]</a>.</p></li>
<li id="id15706"><p>Ronan M. T. Fleming, Nikos Vlassis, Ines Thiele, and Michael A. Saunders. Conditions for duality between fluxes and concentrations in biochemical networks. <em>JOURNAL OF THEORETICAL BIOLOGY</em>, 409:1–10, NOV 21 2016. <a class="reference external" href="https://doi.org/10.1016/j.jtbi.2016.06.033">[DOI]</a>.</p></li>
<li id="id15707"><p>Jin Chen and Michael A. Henson. &lt;i&gt;in silico&lt;/i&gt; metabolic engineering of &lt;i&gt;clostridium ljungdahlii&lt;/i&gt; for synthesis gas fermentation. <em>METABOLIC ENGINEERING</em>, 38:389–400, NOV 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2016.10.002">[DOI]</a>.</p></li>
<li id="id15708"><p>Hulda S. Haraldsdottir and Ronan M. T. Fleming. Identification of conserved moieties in metabolic networks by graph theoretical analysis of atom transition networks. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1004999">[DOI]</a>.</p></li>
<li id="id15709"><p>Anna Lechner, Elizabeth Brunk, and Jay D. Keasling. The need for integrated approaches in metabolic engineering. <em>COLD SPRING HARBOR PERSPECTIVES IN BIOLOGY</em>, NOV 2016. <a class="reference external" href="https://doi.org/10.1101/cshperspect.a023903">[DOI]</a>.</p></li>
<li id="id15710"><p>Thomas Pfau, Maria Pires Pacheco, and Thomas Sauter. Towards improved genome-scale metabolic network reconstructions: unification, transcript specificity and beyond. <em>BRIEFINGS IN BIOINFORMATICS</em>, 17(6):1060–1069, NOV 2016. <a class="reference external" href="https://doi.org/10.1093/bib/bbv100">[DOI]</a>.</p></li>
<li id="id15711"><p>Janne Wallenius, Hannu Maaheimo, and Tero Eerikainen. Carbon 13-metabolic flux analysis derived constraint-based metabolic modelling of &lt;i&gt;clostridium acetobutylicum&lt;/i&gt; in stressed chemostat conditions. <em>BIORESOURCE TECHNOLOGY</em>, 219:378–386, NOV 2016. <a class="reference external" href="https://doi.org/10.1016/j.biortech.2016.07.137">[DOI]</a>.</p></li>
<li id="id15712"><p>Zhihao Wang, Siu Hung Joshua Chan, Suresh Sudarsan, Lars M. Blank, Peter Ruhdal Jensen, and Christian Solem. Elucidation of the regulatory role of the fructose operon reveals a novel target for enhancing the nadph supply in &lt;i&gt;corynebacterium glutamicum&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 38:344–357, NOV 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2016.08.004">[DOI]</a>.</p></li>
<li id="id15713"><p>Lukasz P. Zielinski, Anthony C. Smith, Alexander G. Smith, and Alan J. Robinson. Metabolic flexibility of mitochondrial respiratory chain disorders predicted by computer modelling. <em>MITOCHONDRION</em>, 31:45–55, NOV 2016. <a class="reference external" href="https://doi.org/10.1016/j.mito.2016.09.003">[DOI]</a>.</p></li>
<li id="id15714"><p>Sayed-Rzgar Hosseini and Andreas Wagner. The potential for non-adaptive origins of evolutionary innovations in central carbon metabolism. <em>BMC SYSTEMS BIOLOGY</em>, OCT 21 2016. <a class="reference external" href="https://doi.org/10.1186/s12918-016-0343-7">[DOI]</a>.</p></li>
<li id="id15715"><p>Eugeni Belda, Ruben G. A. van Heck, Maria Jose Lopez-Sanchez, Stephane Cruveiller, Valerie Barbe, Claire Fraser, Hans-Peter Klenk, Joern Petersen, Anne Morgat, Pablo I. Nikel, David Vallenet, Zoe Rouy, Agnieszka Sekowska, Vitor A. P. Martins dos Santos, Victor de Lorenzo, Antoine Danchin, and Claudine Medigue. The revisited genome of &lt;i&gt;pseudomonas putida&lt;/i&gt; kt2440 enlightens its value as a robust metabolic &lt;i&gt;chassis&lt;/i&gt;. <em>ENVIRONMENTAL MICROBIOLOGY</em>, 18(10, SI):3403–3424, OCT 2016. <a class="reference external" href="https://doi.org/10.1111/1462-2920.13230">[DOI]</a>.</p></li>
<li id="id15716"><p>Daniel Machado, Markus J. Herrgard, and Isabel Rocha. Stoichiometric representation of gene-protein-reaction associations leverages constraint-based analysis from reaction to gene-level phenotype prediction. <em>PLOS COMPUTATIONAL BIOLOGY</em>, OCT 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1005140">[DOI]</a>.</p></li>
<li id="id15717"><p>Kevin Schwahn, Anika Kuken, Daniel J. Kliebenstein, Alisdair R. Fernie, and Zoran Nikoloski. Observability of plant metabolic networks is reflected in the correlation of metabolic profiles. <em>PLANT PHYSIOLOGY</em>, 172(2):1324–1333, OCT 2016. <a class="reference external" href="https://doi.org/10.1104/pp.16.00900">[DOI]</a>.</p></li>
<li id="id15718"><p>Fangzhou Shen, Jian Li, Ying Zhu, and Zhuo Wang. Systematic investigation of metabolic reprogramming in different cancers based on tissue-specific metabolic models. <em>JOURNAL OF BIOINFORMATICS AND COMPUTATIONAL BIOLOGY</em>, OCT 2016. 27th International Conference on Genome Informatics (GIW) - Systems Biology, Fudan Univ, Shanghai, PEOPLES R CHINA, OCT 03-05, 2016. <a class="reference external" href="https://doi.org/10.1142/S0219720016440017">[DOI]</a>.</p></li>
<li id="id15719"><p>Laurence Yang, Ding Ma, Ali Ebrahim, Colton J. Lloyd, Michael A. Saunders, and Bernhard O. Palsson. Solveme: fast and reliable solution of nonlinear me models. <em>BMC BIOINFORMATICS</em>, SEP 22 2016. <a class="reference external" href="https://doi.org/10.1186/s12859-016-1240-1">[DOI]</a>.</p></li>
<li id="id15720"><p>R. Adam Thompson, Sanjeev Dahal, Sergio Garcia, Intawat Nookaew, and Cong T. Trinh. Exploring complex cellular phenotypes and model-guided strain design with a novel genome-scale metabolic model of &lt;i&gt;clostridium thermocellum&lt;/i&gt; dsm 1313 implementing an adjustable cellulosome. <em>BIOTECHNOLOGY FOR BIOFUELS</em>, SEP 6 2016. <a class="reference external" href="https://doi.org/10.1186/s13068-016-0607-x">[DOI]</a>.</p></li>
<li id="id15721"><p>Helena Mendes-Soares, Michael Mundy, Luis Mendes Soares, and Nicholas Chia. Mminte: an application for predicting metabolic interactions among the microbial species in a community. <em>BMC BIOINFORMATICS</em>, SEP 2 2016. <a class="reference external" href="https://doi.org/10.1186/s12859-016-1230-3">[DOI]</a>.</p></li>
<li id="id15722"><p>Peter D. Karp, Mario Latendresse, Suzanne M. Paley, Markus Krummenacker, Quang D. Ong, Richard Billington, Anamika Kothari, Daniel Weaver, Thomas Lee, Pallavi Subhraveti, Aaron Spaulding, Carol Fulcher, Ingrid M. Keseler, and Ron Caspi. Pathway tools version 19.0 update: software for pathway/genome informatics and systems biology. <em>BRIEFINGS IN BIOINFORMATICS</em>, 17(5):877–890, SEP 2016. <a class="reference external" href="https://doi.org/10.1093/bib/bbv079">[DOI]</a>.</p></li>
<li id="id15723"><p>Christopher P. Long, Jacqueline E. Gonzalez, Nicholas R. Sandoval, and Maciek R. Antoniewicz. Characterization of physiological responses to 22 gene knockouts in &lt;i&gt;escherichia coli&lt;/i&gt; central carbon metabolism. <em>METABOLIC ENGINEERING</em>, 37:102–113, SEP 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2016.05.006">[DOI]</a>.</p></li>
<li id="id15724"><p>Georgios Savoglidis, Aline Xavier da Silveira dos Santos, Isabelle Riezman, Paolo Angelino, Howard Riezman, and Vassily Hatzimanikatis. A method for analysis and design of metabolism using metabolomics data and kinetic models: application on lipidomics using a novel kinetic model of sphingolipid metabolism. <em>METABOLIC ENGINEERING</em>, 37:46–62, SEP 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2016.04.002">[DOI]</a>.</p></li>
<li id="id15725"><p>R. P. Vivek-Ananth and Areejit Samal. Advances in the integration of transcriptional regulatory information into genome-scale metabolic models. <em>BIOSYSTEMS</em>, 147:1–10, SEP 2016. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2016.06.001">[DOI]</a>.</p></li>
<li id="id15726"><p>Cristal Zuniga, Chien-Ting Li, Tyler Huelsman, Jennifer Levering, Daniel C. Zielinski, Brian O. McConnell, Christopher P. Long, Eric P. Knoshaug, Michael T. Guarnieri, Maciek R. Antoniewicz, Michael J. Betenbaugh, and Karsten Zengler. Genome-scale metabolic model for the green alga &lt;i&gt;chlorella vulgaris&lt;/i&gt; utex 395 accurately predicts phenotypes under autotrophic, heterotrophic, and mixotrophic growth conditions. <em>PLANT PHYSIOLOGY</em>, 172(1):589–602, SEP 2016. <a class="reference external" href="https://doi.org/10.1104/pp.16.00593">[DOI]</a>.</p></li>
<li id="id15727"><p>Meghana Chitale, Ishita K. Khan, and Daisuke Kihara. Missing gene identification using functional coherence scores. <em>SCIENTIFIC REPORTS</em>, AUG 24 2016. <a class="reference external" href="https://doi.org/10.1038/srep31725">[DOI]</a>.</p></li>
<li id="id15728"><p>Tobias Grosskopf, Jessika Consuegra, Joel Gaffe, John C. Willison, Richard E. Lenski, Orkun S. Soyer, and Dominique Schneider. Metabolic modelling in a dynamic evolutionary framework predicts adaptive diversification of bacteria in a long-term evolution experiment. <em>BMC EVOLUTIONARY BIOLOGY</em>, AUG 20 2016. <a class="reference external" href="https://doi.org/10.1186/s12862-016-0733-x">[DOI]</a>.</p></li>
<li id="id15729"><p>Xudong Wu and Guohui Li. Prevalent accumulation of non-optimal codons through somatic mutations in human cancers. <em>PLOS ONE</em>, AUG 11 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0160463">[DOI]</a>.</p></li>
<li id="id15730"><p>Janaka N. Edirisinghe, Pamela Weisenhorn, Neal Conrad, Fangfang Xia, Ross Overbeek, Rick L. Stevens, and Christopher S. Henry. Modeling central metabolism and energy biosynthesis across microbial life. <em>BMC GENOMICS</em>, AUG 8 2016. <a class="reference external" href="https://doi.org/10.1186/s12864-016-2887-8">[DOI]</a>.</p></li>
<li id="id15731"><p>Maike K. Aurich, Ronan M. T. Fleming, and Ines Thiele. Metabotools: a comprehensive toolbox for analysis of genome-scale metabolic models. <em>FRONTIERS IN PHYSIOLOGY</em>, AUG 3 2016. <a class="reference external" href="https://doi.org/10.3389/fphys.2016.00327">[DOI]</a>.</p></li>
<li id="id15732"><p>M. Cuperlovic-Culf and A. S. Culf. Applied metabolomics in drug discovery. <em>EXPERT OPINION ON DRUG DISCOVERY</em>, 11(8):759–770, AUG 2016. <a class="reference external" href="https://doi.org/10.1080/17460441.2016.1195365">[DOI]</a>.</p></li>
<li id="id15733"><p>Daniel Hartleb, Florian Jarre, and Martin J. Lercher. Improved metabolic models for &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; and &lt;i&gt;mycoplasma genitalium&lt;/i&gt; from globalfit, an algorithm that simultaneously matches growth and non-growth data sets. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1005036">[DOI]</a>.</p></li>
<li id="id15734"><p>John I. Hendry, Charulata B. Prasannan, Aditi Joshi, Santanu Dasgupta, and Pramod P. Wangikar. Metabolic model of &lt;i&gt;synechococcus&lt;/i&gt; sp pcc 7002: prediction of flux distribution and network modification for enhanced biofuel production. <em>BIORESOURCE TECHNOLOGY</em>, 213:190–197, AUG 2016. International Conference on New Horizons in Biotechnology (NHBT-2015), Trivandrum, INDIA, NOV 22-25, 2015. <a class="reference external" href="https://doi.org/10.1016/j.biortech.2016.02.128">[DOI]</a>.</p></li>
<li id="id15735"><p>Ankita Juneja, Frank W. R. Chaplen, and Ganti S. Murthy. Genome scale metabolic reconstruction of &lt;i&gt;chlorella&lt;/i&gt; &lt;i&gt;variabilis&lt;/i&gt; for exploring its metabolic potential for biofuels. <em>BIORESOURCE TECHNOLOGY</em>, 213:103–110, AUG 2016. International Conference on New Horizons in Biotechnology (NHBT-2015), Trivandrum, INDIA, NOV 22-25, 2015. <a class="reference external" href="https://doi.org/10.1016/j.biortech.2016.02.118">[DOI]</a>.</p></li>
<li id="id15736"><p>Eliza J. R. Peterson, Shuyi Ma, David R. Sherman, and Nitin S. Baliga. Network analysis identifies rv0324 and rv0880 as regulators of bedaquiline tolerance in &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt;. <em>NATURE MICROBIOLOGY</em>, AUG 2016. <a class="reference external" href="https://doi.org/10.1038/NMICROBIOL.2016.78">[DOI]</a>.</p></li>
<li id="id15737"><p>Pedro A. Saa and Lars K. Nielsen. Ll-achrb: a scalable algorithm for sampling the feasible solution space of metabolic networks. <em>BIOINFORMATICS</em>, 32(15):2330–2337, AUG 1 2016. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btw132">[DOI]</a>.</p></li>
<li id="id15738"><p>Ruben G. A. van Heck, Mathias Ganter, Vitor A. P. Martins dos Santos, and Joerg Stelling. Efficient reconstruction of predictive consensus metabolic network models. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1005085">[DOI]</a>.</p></li>
<li id="id15739"><p>Xiaoyan Yang, Qianqian Yuan, Yangyang Zheng, Hongwu Ma, Tao Chen, and Xueming Zhao. An engineered non-oxidative glycolysis pathway for acetone production in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOTECHNOLOGY LETTERS</em>, 38(8):1359–1365, AUG 2016. <a class="reference external" href="https://doi.org/10.1007/s10529-016-2115-2">[DOI]</a>.</p></li>
<li id="id15740"><p>Paula Jouhten, Tomasz Boruta, Sergej Andrejev, Filipa Pereira, Isabel Rocha, and Kiran Raosaheb Patil. Yeast metabolic chassis designs for diverse biotechnological products. <em>SCIENTIFIC REPORTS</em>, JUL 19 2016. <a class="reference external" href="https://doi.org/10.1038/srep29694">[DOI]</a>.</p></li>
<li id="id15741"><p>George C. diCenzo, Alice Checcucci, Marco Bazzicalupo, Alessio Mengoni, Carlo Viti, Lukasz Dziewit, Turlough M. Finan, Marco Galardini, and Marco Fondi. Metabolic modelling reveals the specialization of secondary replicons for niche adaptation in &lt;i&gt;sinorhizobium meliloti&lt;/i&gt;. <em>NATURE COMMUNICATIONS</em>, JUL 2016. <a class="reference external" href="https://doi.org/10.1038/ncomms12219">[DOI]</a>.</p></li>
<li id="id15742"><p>Xingxing Jian, Shengguo Zhou, Cheng Zhang, and Qiang Hua. &lt;i&gt;in silico&lt;/i&gt; identification of gene amplification targets based on analysis of production and growth coupling. <em>BIOSYSTEMS</em>, 145:1–8, JUL 2016. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2016.05.002">[DOI]</a>.</p></li>
<li id="id15743"><p>Neil Swainston, Kieran Smallbone, Hooman Hefzi, Paul D. Dobson, Judy Brewer, Michael Hanscho, Daniel C. Zielinski, Kok Siong Ang, Natalie J. Gardiner, Jahir M. Gutierrez, Sarantos Kyriakopoulos, Meiyappan Lakshmanan, Shangzhong Li, Joanne K. Liu, Veronica S. Martinez, Camila A. Orellana, Lake-Ee Quek, Alex Thomas, Juergen Zanghellini, Nicole Borth, Dong-Yup Lee, Lars K. Nielsen, Douglas B. Kell, Nathan E. Lewis, and Pedro Mendes. Recon 2.2: from reconstruction to model of human metabolism. <em>METABOLOMICS</em>, JUL 2016. <a class="reference external" href="https://doi.org/10.1007/s11306-016-1051-4">[DOI]</a>.</p></li>
<li id="id15744"><p>Luis Tobalina, Jon Pey, and Francisco J. Planes. Direct calculation of minimal cut sets involving a specific reaction knock-out. <em>BIOINFORMATICS</em>, 32(13):2001–2007, JUL 1 2016. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btw072">[DOI]</a>.</p></li>
<li id="id15745"><p>Xi Wang, Xiaochao Xiong, Na Sa, Sanja Roje, and Shulin Chen. Metabolic engineering of enhanced glycerol-3-phosphate synthesis to increase lipid production in &lt;i&gt;synechocystis&lt;/i&gt; sp pcc 6803. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 100(13):6091–6101, JUL 2016. <a class="reference external" href="https://doi.org/10.1007/s00253-016-7521-9">[DOI]</a>.</p></li>
<li id="id15746"><p>Daniel A. Cuevas, Janaka Edirisinghe, Chris S. Henry, Ross Overbeek, Taylor G. O'Connell, and Robert A. Edwards. From dna to fba: how to build your own genome-scale metabolic model. <em>FRONTIERS IN MICROBIOLOGY</em>, JUN 17 2016. <a class="reference external" href="https://doi.org/10.3389/fmicb.2016.00907">[DOI]</a>.</p></li>
<li id="id15747"><p>Bin Du, Daniel C. Zielinski, Erol S. Kavvas, Andreas Draeger, Justin Tan, Zhen Zhang, Kayla E. Ruggiero, Garri A. Arzumanyan, and Bernhard O. Palsson. Evaluation of rate law approximations in bottom-up kinetic models of metabolism. <em>BMC SYSTEMS BIOLOGY</em>, JUN 6 2016. <a class="reference external" href="https://doi.org/10.1186/s12918-016-0283-2">[DOI]</a>.</p></li>
<li id="id15748"><p>Kumari Sonal Choudhary, Neha Rohatgi, Skarphedinn Halldorsson, Eirikur Briem, Thorarinn Gudjonsson, Steinn Gudmundsson, and Ottar Rolfsson. Egfr signal-network reconstruction demonstrates metabolic crosstalk in emt. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUN 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1004924">[DOI]</a>.</p></li>
<li id="id15749"><p>Marzia Di Filippo, Riccardo Colombo, Chiara Damiani, Dario Pescini, Daniela Gaglio, Marco Vanoni, Lilia Alberghina, and Giancarlo Mauri. Zooming-in on cancer metabolic rewiring with tissue specific constraint-based models. <em>COMPUTATIONAL BIOLOGY AND CHEMISTRY</em>, 62:60–69, JUN 2016. <a class="reference external" href="https://doi.org/10.1016/j.compbiolchem.2016.03.002">[DOI]</a>.</p></li>
<li id="id15750"><p>Matteo Mori, Terence Hwa, Olivier C. Martin, Andrea De Martino, and Enzo Marinari. Constrained allocation flux balance analysis. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUN 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1004913">[DOI]</a>.</p></li>
<li id="id15751"><p>Wanwipa Vongsangnak, Amornpan Klanchui, Iyarest Tawornsamretkit, Witthawin Tatiyaborwornchai, Kobkul Laoteng, and Asawin Meechai. Genome-scale metabolic modeling of &lt;i&gt;mucor circinelloides&lt;/i&gt; and comparative analysis with other oleaginous species. <em>GENE</em>, 583(2):121–129, JUN 1 2016. <a class="reference external" href="https://doi.org/10.1016/j.gene.2016.02.028">[DOI]</a>.</p></li>
<li id="id15752"><p>Silvio Waschina, Glen D'Souza, Christian Kost, and Christoph Kaleta. Metabolic network architecture and carbon source determine metabolite production costs. <em>FEBS JOURNAL</em>, 283(11):2149–2163, JUN 2016. <a class="reference external" href="https://doi.org/10.1111/febs.13727">[DOI]</a>.</p></li>
<li id="id15753"><p>Ying Li, Xi Wang, Xizhen Ge, and Pingfang Tian. High production of 3-hydroxypropionic acid in &lt;i&gt;klebsiella pneumoniae&lt;/i&gt; by systematic optimization of glycerol metabolism. <em>SCIENTIFIC REPORTS</em>, MAY 27 2016. <a class="reference external" href="https://doi.org/10.1038/srep26932">[DOI]</a>.</p></li>
<li id="id15754"><p>Juliane Gebauer, Christoph Gentsch, Johannes Mansfeld, Kathrin Schmeisser, Silvio Waschina, Susanne Brandes, Lukas Klimmasch, Nicola Zamboni, Kim Zarse, Stefan Schuster, Michael Ristow, Sascha Schaeuble, and Christoph Kaleta. A genome-scale database and reconstruction of &lt;i&gt;caenorhabditis elegans&lt;/i&gt; metabolism. <em>CELL SYSTEMS</em>, 2(5):312–322, MAY 25 2016. <a class="reference external" href="https://doi.org/10.1016/j.cels.2016.04.017">[DOI]</a>.</p></li>
<li id="id15755"><p>Mariana G. Ferrarini, Franciele M. Siqueira, Scheila G. Mucha, Tony L. Palama, Elodie Jobard, Benedicte Elena-Herrmann, Ana T. R. Vasconcelos, Florence Tardy, Irene S. Schrank, Arnaldo Zaha, and Marie-France Sagot. Insights on the virulence of swine respiratory tract mycoplasmas through genome-scale metabolic modeling. <em>BMC GENOMICS</em>, MAY 13 2016. <a class="reference external" href="https://doi.org/10.1186/s12864-016-2644-z">[DOI]</a>.</p></li>
<li id="id15756"><p>Jennifer Levering, Jared Broddrick, Christopher L. Dupont, Graham Peers, Karen Beeri, Joshua Mayers, Alessandra A. Gallina, Andrew E. Allen, Bernhard O. Palsson, and Karsten Zengler. Genome-scale model reveals metabolic basis of biomass partitioning in a model diatom. <em>PLOS ONE</em>, MAY 6 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0155038">[DOI]</a>.</p></li>
<li id="id15757"><p>Luis Tobalina, Jon Pey, Alberto Rezola, and Francisco J. Planes. Assessment of fba based gene essentiality analysis in cancer with a fast context-specific network reconstruction method. <em>PLOS ONE</em>, MAY 4 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0154583">[DOI]</a>.</p></li>
<li id="id15758"><p>Stefano Andreozzi, Anirikh Chakrabarti, Keng Cher Soh, Anthony Burgard, Tae Hoon Yang, Stephen Van Dien, Ljubisa Miskovic, and Vassily Hatzimanikatis. Identification of metabolic engineering targets for the enhancement of 1,4-butanediol production in recombinant &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; using large-scale kinetic models. <em>METABOLIC ENGINEERING</em>, 35:148–159, MAY 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2016.01.009">[DOI]</a>.</p></li>
<li id="id15759"><p>Wei-Chin Ho and Jianzhi Zhang. Adaptive genetic robustness of &lt;i&gt;escherichia coli&lt;/i&gt; metabolic fluxes. <em>MOLECULAR BIOLOGY AND EVOLUTION</em>, 33(5):1164–1176, MAY 2016. <a class="reference external" href="https://doi.org/10.1093/molbev/msw002">[DOI]</a>.</p></li>
<li id="id15760"><p>Manon Morin, Delphine Ropers, Fabien Letisse, Sandrine Laguerre, Jean-Charles Portais, Muriel Cocaign-Bousquet, and Brice Enjalbert. The post-transcriptional regulatory system csr controls the balance of metabolic pools in upper glycolysis of &lt;i&gt;escherichia coli&lt;/i&gt;. <em>MOLECULAR MICROBIOLOGY</em>, 100(4):686–700, MAY 2016. <a class="reference external" href="https://doi.org/10.1111/mmi.13343">[DOI]</a>.</p></li>
<li id="id15761"><p>Octavio Perez-Garcia, Kartik Chandran, Silas G. Villas-Boas, and Naresh Singhal. Assessment of nitric oxide (no) redox reactions contribution to nitrous oxide (n&lt;sub&gt;2&lt;/sub&gt;o) formation during nitrification using a multispecies metabolic network model. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 113(5):1124–1136, MAY 2016. <a class="reference external" href="https://doi.org/10.1002/bit.25880">[DOI]</a>.</p></li>
<li id="id15762"><p>Semidan Robaina-Estevez and Zoran Nikoloski. Metabolic network constrains gene regulation of c&lt;sub&gt;4&lt;/sub&gt; photosynthesis: the case of maize. <em>PLANT AND CELL PHYSIOLOGY</em>, 57(5):933–943, MAY 2016. <a class="reference external" href="https://doi.org/10.1093/pcp/pcw034">[DOI]</a>.</p></li>
<li id="id15763"><p>Xiaolin Zhang, Christopher J. Tervo, and Jennifer L. Reed. Metabolic assessment of e. coli as a biofactory for commercial products. <em>METABOLIC ENGINEERING</em>, 35:64–74, MAY 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2016.01.007">[DOI]</a>.</p></li>
<li id="id15764"><p>Huili Yuan, C. Y. Maurice Cheung, Peter A. J. Hilbers, and Natal A. W. van Riel. Flux balance analysis of plant metabolism: the effect of biomass composition and model structure on model predictions. <em>FRONTIERS IN PLANT SCIENCE</em>, APR 26 2016. <a class="reference external" href="https://doi.org/10.3389/fpls.2016.00537">[DOI]</a>.</p></li>
<li id="id15765"><p>Emrah Ozcan and Tunahan Cakir. Reconstructed metabolic network models predict flux-level metabolic reprogramming in glioblastoma. <em>FRONTIERS IN NEUROSCIENCE</em>, APR 18 2016. <a class="reference external" href="https://doi.org/10.3389/fnins.2016.00156">[DOI]</a>.</p></li>
<li id="id15766"><p>Aditya M. Kunjapur, Jason C. Hyun, and Kristala L. J. Prather. Deregulation of &lt;i&gt;s&lt;/i&gt;-adenosylmethionine biosynthesis and regeneration improves methylation in the &lt;i&gt;e-coli&lt;/i&gt; de novo vanillin biosynthesis pathway. <em>MICROBIAL CELL FACTORIES</em>, APR 11 2016. <a class="reference external" href="https://doi.org/10.1186/s12934-016-0459-x">[DOI]</a>.</p></li>
<li id="id15767"><p>Douglas McCloskey, Jamey D. Young, Sibei Xu, Bernhard O. Palsson, and Adam M. Feist. Modeling method for increased precision and scope of directly measurable fluxes at a genome-scale. <em>ANALYTICAL CHEMISTRY</em>, 88(7):3844–3852, APR 5 2016. <a class="reference external" href="https://doi.org/10.1021/acs.analchem.5b04914">[DOI]</a>.</p></li>
<li id="id15768"><p>Thibault Nidelet, Pascale Brial, Carole Camarasa, and Sylvie Dequin. Diversity of flux distribution in central carbon metabolism of &lt;i&gt;s&lt;/i&gt;. &lt;i&gt;cerevisiae&lt;/i&gt; strains from diverse environments. <em>MICROBIAL CELL FACTORIES</em>, APR 5 2016. <a class="reference external" href="https://doi.org/10.1186/s12934-016-0456-0">[DOI]</a>.</p></li>
<li id="id15769"><p>Aarash Bordbar, Par I. Johansson, Giuseppe Paglia, Scott J. Harrison, Kristine Wichuk, Manuela Magnusdottir, Soley Valgeirsdottir, Mikkel Gybel-Brask, Sisse R. Ostrowski, Sirus Palsson, Ottar Rolfsson, Olafur E. Sigurjonsson, Morten B. Hansen, Sveinn Gudmundsson, and Bernhard O. Palsson. Identified metabolic signature for assessing red blood cell unit quality is associated with endothelial damage markers and clinical outcomes. <em>TRANSFUSION</em>, 56(4):852–862, APR 2016. <a class="reference external" href="https://doi.org/10.1111/trf.13460">[DOI]</a>.</p></li>
<li id="id15770"><p>Hai-Lin Meng, Zhi-Qiang Xiong, Shu-Jie Song, Jianfeng Wang, and Yong Wang. Construction of polyketide overproducing &lt;i&gt;escherichia coli&lt;/i&gt; strains via synthetic antisense rnas based on in silico fluxome analysis and comparative transcriptome analysis. <em>BIOTECHNOLOGY JOURNAL</em>, 11(4):530–541, APR 2016. <a class="reference external" href="https://doi.org/10.1002/biot.201500351">[DOI]</a>.</p></li>
<li id="id15771"><p>Salma Sohrabi-Jahromi, Sayed-Amir Marashi, and Shiva Kalantari. A kidney-specific genome-scale metabolic network model for analyzing focal segmental glomerulosclerosis. <em>MAMMALIAN GENOME</em>, 27(3-4):158–167, APR 2016. <a class="reference external" href="https://doi.org/10.1007/s00335-016-9622-2">[DOI]</a>.</p></li>
<li id="id15772"><p>Yuqi Zhao, Yanjie Wang, Lei Zou, and Jingfei Huang. Reconstruction and applications of consensus yeast metabolic network based on rna sequencing. <em>FEBS OPEN BIO</em>, 6(4):264–275, APR 2016. <a class="reference external" href="https://doi.org/10.1002/2211-5463.12033">[DOI]</a>.</p></li>
<li id="id15773"><p>Bhanwar Lal Puniya, Deepika Kulshreshtha, Inna Mittal, Ahmed Mobeen, and Srinivasan Ramachandran. Integration of metabolic modeling with gene co-expression reveals transcriptionally programmed reactions explaining robustness in &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt;. <em>SCIENTIFIC REPORTS</em>, MAR 22 2016. <a class="reference external" href="https://doi.org/10.1038/srep23440">[DOI]</a>.</p></li>
<li id="id15774"><p>Eli Bogart and Christopher R. Myers. Multiscale metabolic modeling of c4 plants: connecting nonlinear genome-scale models to leaf-scale metabolism in developing maize leaves. <em>PLOS ONE</em>, MAR 18 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0151722">[DOI]</a>.</p></li>
<li id="id15775"><p>Matthew B. Biggs and Jason A. Papin. Metabolic network-guided binning of metagenomic sequence fragments. <em>BIOINFORMATICS</em>, 32(6):867–874, MAR 15 2016. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btv671">[DOI]</a>.</p></li>
<li id="id15776"><p>Elizabeth Brunk, Nathan Mih, Jonathan Monk, Zhen Zhang, Edward J. O'Brien, Spencer E. Bliven, Ke Chen, Roger L. Chang, Philip E. Bourne, and Bernhard O. Palsson. Systems biology of the structural proteome. <em>BMC SYSTEMS BIOLOGY</em>, MAR 11 2016. <a class="reference external" href="https://doi.org/10.1186/s12918-016-0271-6">[DOI]</a>.</p></li>
<li id="id15777"><p>Mohammad Tauqeer Alam, Aleksej Zelezniak, Michael Mulleder, Pavel Shliaha, Roland Schwarz, Floriana Capuano, Jakob Vowinckel, Elahe Radmaneshfar, Antje Krueger, Enrica Calvani, Steve Michel, Stefan Boerno, Stefan Christen, Kiran Raosaheb Patil, Bernd Timmermann, Kathryn S. Lilley, and Markus Ralser. The metabolic background is a global player in &lt;i&gt;saccharomyces&lt;/i&gt; gene expression epistasis. <em>NATURE MICROBIOLOGY</em>, MAR 2016. <a class="reference external" href="https://doi.org/10.1038/NMICROBIOL.2015.30">[DOI]</a>.</p></li>
<li id="id15778"><p>Lisa Maria Dersch, Veronique Beckers, and Christoph Wittmann. Green pathways: metabolic network analysis of plant systems. <em>METABOLIC ENGINEERING</em>, 34:1–24, MAR 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2015.12.001">[DOI]</a>.</p></li>
<li id="id15779"><p>Minsuk Kim, Jeong Sang Yi, Meiyappan Lakshmanan, Dong-Yup Lee, and Byung-Gee Kim. Transcriptomics-based strain optimization tool for designing secondary metabolite overproducing strains of &lt;i&gt;streptomyces coelicolor&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 113(3):651–660, MAR 2016. <a class="reference external" href="https://doi.org/10.1002/bit.25830">[DOI]</a>.</p></li>
<li id="id15780"><p>Andre Schultz and Amina A. Qutub. Reconstruction of tissue-specific metabolic networks using corda. <em>PLOS COMPUTATIONAL BIOLOGY</em>, MAR 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1004808">[DOI]</a>.</p></li>
<li id="id15781"><p>Roberto Gallardo, Alejandro Acevedo, Julian Quintero, Ivan Paredes, Raul Conejeros, and German Aroca. In silico analysis of &lt;i&gt;clostridium acetobutylicum&lt;/i&gt; atcc 824 metabolic response to an external electron supply. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 39(2):295–305, FEB 2016. <a class="reference external" href="https://doi.org/10.1007/s00449-015-1513-5">[DOI]</a>.</p></li>
<li id="id15782"><p>Diego A. Salazar, Alexander Rodriguez-Lopez, Angelica Herreno, Hector Barbosa, Juliana Herrera, Andrea Ardila, George E. Barreto, Janneth Gonzalez, and Carlos J. Almeciga-Diaz. Systems biology study of mucopolysaccharidosis using a human metabolic reconstruction network. <em>MOLECULAR GENETICS AND METABOLISM</em>, 117(2):129–139, FEB 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymgme.2015.08.001">[DOI]</a>.</p></li>
<li id="id15783"><p>John T. Sauls, Dongyang Li, and Suckjoon Jun. Adder and a coarse-grained approach to cell size homeostasis in bacteria. <em>CURRENT OPINION IN CELL BIOLOGY</em>, 38:38–44, FEB 2016. <a class="reference external" href="https://doi.org/10.1016/j.ceb.2016.02.004">[DOI]</a>.</p></li>
<li id="id15784"><p>Hai Shi and Joerg Schwender. Mathematical models of plant metabolism. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 37:143–152, FEB 2016. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2015.10.008">[DOI]</a>.</p></li>
<li id="id15785"><p>Tomas Zavrel, Henning Knoop, Ralf Steuer, Patrik R. Jones, Jan Cerveny, and Martin Trtilek. A quantitative evaluation of ethylene production in the recombinant cyanobacterium &lt;i&gt;synechocystis&lt;/i&gt; sp pcc 6803 harboring the ethylene-forming enzyme by membrane inlet mass spectrometry. <em>BIORESOURCE TECHNOLOGY</em>, 202:142–151, FEB 2016. <a class="reference external" href="https://doi.org/10.1016/j.biortech.2015.11.062">[DOI]</a>.</p></li>
<li id="id15786"><p>Elias Bjornson, Jan Boren, and Adil Mardinoglu. Personalized cardiovascular disease prediction and treatment-a review of existing strategies and novel systems medicine tools. <em>FRONTIERS IN PHYSIOLOGY</em>, JAN 26 2016. <a class="reference external" href="https://doi.org/10.3389/fphys.2016.00002">[DOI]</a>.</p></li>
<li id="id15787"><p>Marius Tomas-Gamisans, Pau Ferrer, and Joan Albiol. Integration and validation of the genome-scale metabolic models of &lt;i&gt;pichia pastoris&lt;/i&gt;: a comprehensive update of protein glycosylation pathways, lipid and energy metabolism. <em>PLOS ONE</em>, JAN 26 2016. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0148031">[DOI]</a>.</p></li>
<li id="id15788"><p>Maria P. Pacheco, Thomas Pfau, and Thomas Sauter. Benchmarking procedures for high-throughput context specific reconstruction algorithms. <em>FRONTIERS IN PHYSIOLOGY</em>, JAN 22 2016. <a class="reference external" href="https://doi.org/10.3389/fphys.2015.00410">[DOI]</a>.</p></li>
<li id="id15789"><p>Hadi Nazem-Bokaee, Saratram Gopalakrishnan, James G. Ferry, Thomas K. Wood, and Costas D. Maranas. Assessing methanotrophy and carbon fixation for biofuel production by &lt;i&gt;methanosarcina acetivorans&lt;/i&gt;. <em>MICROBIAL CELL FACTORIES</em>, JAN 17 2016. <a class="reference external" href="https://doi.org/10.1186/s12934-015-0404-4">[DOI]</a>.</p></li>
<li id="id15790"><p>Jie Mei, Nan Xu, Chao Ye, Liming Liu, and Jianrong Wu. Reconstruction and analysis of a genome-scale metabolic network of &lt;i&gt;corynebacterium glutamicum&lt;/i&gt; s9114. <em>GENE</em>, 575(2, 3):615–622, JAN 10 2016. <a class="reference external" href="https://doi.org/10.1016/j.gene.2015.09.038">[DOI]</a>.</p></li>
<li id="id15791"><p>Zachary A. King, Justin Lu, Andreas Draeger, Philip Miller, Stephen Federowicz, Joshua A. Lerman, Ali Ebrahim, Bernhard O. Palsson, and Nathan E. Lewis. Bigg models: a platform for integrating, standardizing and sharing genome-scale models. <em>NUCLEIC ACIDS RESEARCH</em>, 44(D1):D515–D522, JAN 4 2016. <a class="reference external" href="https://doi.org/10.1093/nar/gkv1049">[DOI]</a>.</p></li>
<li id="id15792"><p>Susanne Alff-Tuomala, Laura Salusjarvi, Dorothee Barth, Merja Oja, Merja Penttila, Juha-Pekka Pitkanen, Laura Ruohonen, and Paula Jouhten. Xylose-induced dynamic effects on metabolism and gene expression in engineered &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; in anaerobic glucose-xylose cultures. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 100(2):969–985, JAN 2016. <a class="reference external" href="https://doi.org/10.1007/s00253-015-7038-7">[DOI]</a>.</p></li>
<li id="id15793"><p>Marouen Ben Guebila and Ines Thiele. Model-based dietary optimization for late-stage, levodopa-treated, parkinson's disease patients. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, 2016. <a class="reference external" href="https://doi.org/10.1038/npjsba.2016.13">[DOI]</a>.</p></li>
<li id="id15794"><p>Amphun Chaiboonchoe, Lila Ghamsari, Bushra Dohai, Patrick Ng, Basel Khraiwesh, Ashish Jaiswal, Kenan Jijakli, Joseph Koussa, David R. Nelson, Hong Cai, Xinping Yang, Roger L. Chang, Jason Papin, Haiyuan Yu, Santhanam Balaji, and Kourosh Salehi-Ashtiani. Systems level analysis of the &lt;i&gt;chlamydomonas&lt;/i&gt; &lt;i&gt;reinhardtii&lt;/i&gt; metabolic network reveals variability in evolutionary co-conservation. <em>MOLECULAR BIOSYSTEMS</em>, 12(8):2394–2407, 2016. <a class="reference external" href="https://doi.org/10.1039/c6mb00237d">[DOI]</a>.</p></li>
<li id="id15795"><p>Neha Chaudhary, Kristin Tondel, Rakesh Bhatnagar, Vitor A. P. Martins dos Santos, and Jacek Puchalka. Characterizing the optimal flux space of genomescale metabolic reconstructions through modified latin-hypercube sampling. <em>MOLECULAR BIOSYSTEMS</em>, 12(3):994–1005, 2016. <a class="reference external" href="https://doi.org/10.1039/c5mb00457h">[DOI]</a>.</p></li>
<li id="id15796"><p>Max Conway, Claudio Angione, and Pietro Lio. Iterative multi level calibration of metabolic networks. <em>CURRENT BIOINFORMATICS</em>, 11(1):93–105, 2016. <a class="reference external" href="https://doi.org/10.2174/1574893611666151203222505">[DOI]</a>.</p></li>
<li id="id15797"><p>Atefeh Taherian Fard, Sriganesh Srihari, Jessica C. Mar, and Mark A. Ragan. Not just a colourful metaphor: modelling the landscape of cellular development using hopfield networks. <em>NPJ SYSTEMS BIOLOGY AND APPLICATIONS</em>, 2016. <a class="reference external" href="https://doi.org/10.1038/npjsba.2016.1">[DOI]</a>.</p></li>
<li id="id15798"><p>Feifei Han, Gonghua Li, Shaoxing Dai, and Jingfei Huang. Genome-wide metabolic model to improve understanding of cd4&lt;sup&gt;+&lt;/sup&gt; t cell metabolism, immunometabolism and application in drug design. <em>MOLECULAR BIOSYSTEMS</em>, 12(2):431–443, 2016. <a class="reference external" href="https://doi.org/10.1039/c5mb00480b">[DOI]</a>.</p></li>
<li id="id15799"><p>Oveis Jamialahmadi, Ehsan Motamedian, and Sameereh Hashemi-Najafabadi. Bikegg: a cobra toolbox extension for bridging the bigg and kegg databases. <em>MOLECULAR BIOSYSTEMS</em>, 12(11):3459–3466, 2016. <a class="reference external" href="https://doi.org/10.1039/c6mb00532b">[DOI]</a>.</p></li>
<li id="id15800"><p>Regiane Kawasaki, Rafael A. Barauna, Artur Silva, Marta S. P. Carepo, Rui Oliveira, Rodolfo Marques, Rommel T. J. Ramos, and Andmaria P. C. Schneider. Reconstruction of the fatty acid biosynthetic pathway of &lt;i&gt;exiguobacterium&lt;/i&gt; &lt;i&gt;antarcticum&lt;/i&gt; b7 based on genomic and bibliomic data. <em>BIOMED RESEARCH INTERNATIONAL</em>, 2016. <a class="reference external" href="https://doi.org/10.1155/2016/7863706">[DOI]</a>.</p></li>
<li id="id15801"><p>Lin Liu, Fangzhou Shen, Changpeng Xin, and Zhuo Wang. Multi-scale modeling of &lt;i&gt;arabidopsis thaliana&lt;/i&gt; response to different co&lt;sub&gt;2&lt;/sub&gt; conditions: from gene expression to metabolic flux. <em>JOURNAL OF INTEGRATIVE PLANT BIOLOGY</em>, 58(1):2–11, JAN 2016. <a class="reference external" href="https://doi.org/10.1111/jipb.12370">[DOI]</a>.</p></li>
<li id="id15802"><p>Esteban Marcellin, James B. Behrendorff, Shilpa Nagaraju, Sashini DeTissera, Simon Segovia, Robin W. Palfreyman, James Daniell, Cuauhtemoc Licona-Cassani, Lake-ee Quek, Robert Speight, Mark P. Hodson, Sean D. Simpson, Wayne P. Mitchell, Michael Kopke, and Lars K. Nielsen. Low carbon fuels and commodity chemicals from waste gases - systematic approach to understand energy metabolism in a model acetogen. <em>GREEN CHEMISTRY</em>, 18(10):3020–3028, 2016. <a class="reference external" href="https://doi.org/10.1039/c5gc02708j">[DOI]</a>.</p></li>
<li id="id15803"><p>Tijana Marinkovic and Matej Oresic. Modeling strategies to study metabolic pathways in progression to type 1 diabetes - challenges and opportunities. <em>ARCHIVES OF BIOCHEMISTRY AND BIOPHYSICS</em>, 589(SI):131–137, JAN 1 2016. <a class="reference external" href="https://doi.org/10.1016/j.abb.2015.08.011">[DOI]</a>.</p></li>
<li id="id15804"><p>Reza Mohammadi, Jalil Fallah-Mehrabadi, Gholamreza Bidkhori, Javad Zahiri, Mohammad Javad Niroomand, and Ali Masoudi-Nejad. A systems biology approach to reconcile metabolic network models with application to &lt;i&gt;synechocystis&lt;/i&gt; sp pcc 6803 for biofuel production. <em>MOLECULAR BIOSYSTEMS</em>, 12(8):2552–2561, 2016. <a class="reference external" href="https://doi.org/10.1039/c6mb00119j">[DOI]</a>.</p></li>
<li id="id15805"><p>E. Motamedian, M. Saeidi, and S. A. Shojaosadati. Reconstruction of a charge balanced genome-scale metabolic model to study the energy-uncoupled growth of &lt;i&gt;zymomonas&lt;/i&gt; &lt;i&gt;mobilis&lt;/i&gt; zm1. <em>MOLECULAR BIOSYSTEMS</em>, 12(4):1241–1249, 2016. <a class="reference external" href="https://doi.org/10.1039/c5mb00588d">[DOI]</a>.</p></li>
<li id="id15806"><p>Matthew Moura, Justin Finkle, Sarah Stainbrook, Jennifer Greene, Linda J. Broadbelt, and Keith E. J. Tyo. Evaluating enzymatic synthesis of small molecule drugs. <em>METABOLIC ENGINEERING</em>, 33:138–147, JAN 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2015.11.006">[DOI]</a>.</p></li>
<li id="id15807"><p>Suthat Phaiphinit, Sittiporn Pattaradilokrat, Chidchanok Lursinsap, and Kitipom Plaimas. In silico multiple-targets identification for heme detoxification in the human malaria parasite &lt;i&gt;plasmodium falciparum&lt;/i&gt;. <em>INFECTION GENETICS AND EVOLUTION</em>, 37:237–244, JAN 2016. <a class="reference external" href="https://doi.org/10.1016/j.meegid.2015.11.025">[DOI]</a>.</p></li>
<li id="id15808"><p>Philipp N. Spahn, Anders H. Hansen, Henning G. Hansen, Johnny Arnsdorf, Helene F. Kildegaard, and Nathan E. Lewis. A markov chain model for n-linked protein glycosylation - towards a low-parameter tool for model-driven glycoengineering. <em>METABOLIC ENGINEERING</em>, 33:52–66, JAN 2016. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2015.10.007">[DOI]</a>.</p></li>
<li id="id15809"><p>H. S. Toogood, S. Tait, A. Jervis, A. Ni Cheallaigh, L. Humphreys, E. Takano, J. M. Gardiner, and N. S. Scrutton. Natural product biosynthesis in &lt;i&gt;escherichia coli&lt;/i&gt;: mentha monoterpenoids. In SE OConnor, editor, <em>SYNTHETIC BIOLOGY AND METABOLIC ENGINEERING IN PLANTS AND MICROBES, PT A: METABOLISM IN MICROBES</em>, volume 575 of Methods in Enzymology, pages 247–270. 2016. <a class="reference external" href="https://doi.org/10.1016/bs.mie.2016.02.020">[DOI]</a>.</p></li>
<li id="id15810"><p>Bin Wang, Qianji Ning, Tong Hao, Ailing Yu, and Jinsheng Sun. Reconstruction and analysis of a genome-scale metabolic model for &lt;i&gt;eriocheir sinensis&lt;/i&gt; eyestalks. <em>MOLECULAR BIOSYSTEMS</em>, 12(1):246–252, 2016. <a class="reference external" href="https://doi.org/10.1039/c5mb00571j">[DOI]</a>.</p></li>
<li id="id15811"><p>Michael Weiner, Julia Troendle, Christoph Albermann, Georg A. Sprenger, and Dirk Weuster-Botz. Perturbation experiments: approaches for metabolic pathway analysis in bioreactors. In J Bao, Q Ye, and JJ Zhong, editors, <em>BIOREACTOR ENGINEERING RESEARCH AND INDUSTRIAL APPLICATIONS II</em>, volume 152 of Advances in Biochemical Engineering-Biotechnology, pages 91–136. 2016. <a class="reference external" href="https://doi.org/10.1007/10\_2015\_326">[DOI]</a>.</p></li>
<li id="id15812"><p>Kamlesh K. Yadav, Khader Shameer, Ben Readhead, Shalini S. Yadav, Li Li, Andrew Kasarskis, Ashutosh K. Tewari, and Joel T. Dudley. Systems medicine approaches to improving understanding, treatment, and clinical management of neuroendocrine prostate cancer. <em>CURRENT PHARMACEUTICAL DESIGN</em>, 22(34):5234–5248, 2016. <a class="reference external" href="https://doi.org/10.2174/1381612822666160513145924">[DOI]</a>.</p></li>
<li id="id15813"><p>James T. Yurkovich and Bernhard O. Palsson. Solving puzzles with missing pieces: the power of systems biology. <em>PROCEEDINGS OF THE IEEE</em>, 104(1, SI):2–7, JAN 2016. <a class="reference external" href="https://doi.org/10.1109/JPROC.2015.2505338">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-9" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-9" name="0-9" role="tabpanel" tabindex="0"><div class="docutils container" id="id16381">
<ol class="arabic simple" start="1">
<li id="id17634"><p>David E. Ruckerbauer, Christian Jungreuthmayer, and Juergen Zanghellini. Predicting genetic engineering targets with elementary flux mode analysis: a review of four current methods. <em>NEW BIOTECHNOLOGY</em>, 32(6):534–546, DEC 25 2015. 16th European Congress on Biotechnology, Edinburgh, SCOTLAND, JUL 13-16, 2014. <a class="reference external" href="https://doi.org/10.1016/j.nbt.2015.03.017">[DOI]</a>.</p></li>
<li id="id17635"><p>Lian He, Stephen G. Wu, Ni Wan, Adrienne C. Reding, and Yinjie J. Tang. Simulating cyanobacterial phenotypes by integrating flux balance analysis, kinetics, and a light distribution function. <em>MICROBIAL CELL FACTORIES</em>, DEC 24 2015. <a class="reference external" href="https://doi.org/10.1186/s12934-015-0396-0">[DOI]</a>.</p></li>
<li id="id17636"><p>Lucas Marmiesse, Remi Peyraud, and Ludovic Cottret. Flexflux: combining metabolic flux and regulatory network analyses. <em>BMC SYSTEMS BIOLOGY</em>, DEC 15 2015. <a class="reference external" href="https://doi.org/10.1186/s12918-015-0238-z">[DOI]</a>.</p></li>
<li id="id17637"><p>Miguel Ponce-de-Leon, Jorge Calle-Espinosa, Juli Pereto, and Francisco Montero. Consistency analysis of genome-scale models of bacterial metabolism: a &lt;i&gt;metamodel&lt;/i&gt; approach. <em>PLOS ONE</em>, DEC 2 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0143626">[DOI]</a>.</p></li>
<li id="id17638"><p>Michael A. Henson. Genome-scale modelling of microbial metabolism with temporal and spatial resolution. <em>BIOCHEMICAL SOCIETY TRANSACTIONS</em>, 43(6):1164–1171, DEC 2015. <a class="reference external" href="https://doi.org/10.1042/BST20150146">[DOI]</a>.</p></li>
<li id="id17639"><p>Stanislav Kopriva, Alexander Calderwood, Silke C. Weckopp, and Anna Koprivova. Plant sulfur and big data. <em>PLANT SCIENCE</em>, 241:1–10, DEC 2015. <a class="reference external" href="https://doi.org/10.1016/j.plantsci.2015.09.014">[DOI]</a>.</p></li>
<li id="id17640"><p>Meiyappan Lakshmanan, Sun-Hyung Lim, Bijayalaxmi Mohanty, Jae Kwang Kim, Sun-Hwa Ha, and Dong-Yup Lee. Unraveling the light-specific metabolic and regulatory signatures of rice through combined in silico modeling and multiomics analysis. <em>PLANT PHYSIOLOGY</em>, 169(4):3002–3020, DEC 2015. <a class="reference external" href="https://doi.org/10.1104/pp.15.01379">[DOI]</a>.</p></li>
<li id="id17641"><p>Sheng Ren, Anna A. Hinzman, Emily L. Kang, Rhonda D. Szczesniak, and Long Jason Lu. Computational and statistical analysis of metabolomics data. <em>METABOLOMICS</em>, 11(6):1492–1513, DEC 2015. <a class="reference external" href="https://doi.org/10.1007/s11306-015-0823-6">[DOI]</a>.</p></li>
<li id="id17642"><p>Eugen Bauer, Cedric Christian Laczny, Stefania Magnusdottir, Paul Wilmes, and Ines Thiele. Phenotypic differentiation of gastrointestinal microbes is reflected in their encoded metabolic repertoires. <em>MICROBIOME</em>, NOV 30 2015. <a class="reference external" href="https://doi.org/10.1186/s40168-015-0121-6">[DOI]</a>.</p></li>
<li id="id17643"><p>Nima Abedpour and Markus Kollmann. Resource constrained flux balance analysis predicts selective pressure on the global structure of metabolic networks. <em>BMC SYSTEMS BIOLOGY</em>, NOV 23 2015. <a class="reference external" href="https://doi.org/10.1186/s12918-015-0232-5">[DOI]</a>.</p></li>
<li id="id17644"><p>Zhenquan Lin, Yan Zhang, Qianqian Yuan, Qiaojie Liu, Yifan Li, Zhiwen Wang, Hongwu Ma, Tao Chen, and Xueming Zhao. Metabolic engineering of &lt;i&gt;escherichia coli&lt;/i&gt; for poly(3-hydroxybutyrate) production via threonine bypass. <em>MICROBIAL CELL FACTORIES</em>, NOV 20 2015. <a class="reference external" href="https://doi.org/10.1186/s12934-015-0369-3">[DOI]</a>.</p></li>
<li id="id17645"><p>Bashir Sajo Mienda and Mohd Shahir Shamsir. &lt;i&gt;in silico&lt;/i&gt; deletion of &lt;i&gt;ptsg&lt;/i&gt; gene in &lt;i&gt;escherichia coli&lt;/i&gt; genome-scale model predicts increased succinate production from glycerol. <em>JOURNAL OF BIOMOLECULAR STRUCTURE &amp; DYNAMICS</em>, 33(11):2380–2389, NOV 2 2015. <a class="reference external" href="https://doi.org/10.1080/07391102.2015.1036461">[DOI]</a>.</p></li>
<li id="id17646"><p>C. A. Contador, V. Rodriguez, B. A. Andrews, and J. A. Asenjo. Genome-scale reconstruction of &lt;i&gt;salinispora tropica&lt;/i&gt; cnb-440 metabolism to study strain-specific adaptation. <em>ANTONIE VAN LEEUWENHOEK INTERNATIONAL JOURNAL OF GENERAL AND MOLECULAR MICROBIOLOGY</em>, 108(5):1075–1090, NOV 2015. <a class="reference external" href="https://doi.org/10.1007/s10482-015-0561-9">[DOI]</a>.</p></li>
<li id="id17647"><p>Benjamin D. Heavner and Nathan D. Price. Comparative analysis of yeast metabolic network models highlights progress, opportunities for metabolic reconstruction. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1004530">[DOI]</a>.</p></li>
<li id="id17648"><p>Jerome Izard, Cindy D. C. Gomez Balderas, Delphine Ropers, Stephan Lacour, Xiaohu Song, Yifan Yang, Ariel B. Lindner, Johannes Geiselmann, and Hidde de Jong. A synthetic growth switch based on controlled expression of rna polymerase. <em>MOLECULAR SYSTEMS BIOLOGY</em>, NOV 2015. <a class="reference external" href="https://doi.org/10.15252/msb.20156382">[DOI]</a>.</p></li>
<li id="id17649"><p>Na-Rae Lee, Ji-Yeong Yun, Sun-Mee Lee, and Jin-Byung Park. Cyclohexanone-induced stress metabolism of &lt;i&gt;escherichia coli&lt;/i&gt; and &lt;i&gt;corynebacterium glutamicum&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 20(6):1088–1098, NOV 2015. <a class="reference external" href="https://doi.org/10.1007/s12257-015-0607-x">[DOI]</a>.</p></li>
<li id="id17650"><p>Shuyi Ma, Kyle J. Minch, Tige R. Rustad, Samuel Hobbs, Suk-Lin Zhou, David R. Sherman, and Nathan D. Price. Integrated modeling of gene regulatory and metabolic networks in &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt;. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1004543">[DOI]</a>.</p></li>
<li id="id17651"><p>Aarthi Ravikrishnan and Karthik Raman. Critical assessment of genome-scale metabolic networks: the need for a unified standard. <em>BRIEFINGS IN BIOINFORMATICS</em>, 16(6):1057–1068, NOV 2015. <a class="reference external" href="https://doi.org/10.1093/bib/bbv003">[DOI]</a>.</p></li>
<li id="id17652"><p>John J. Varga, Mariette Barbier, Xavier Mulet, Piotr Bielecki, Jennifer A. Bartell, Joshua P. Owings, Inmaculada Martinez-Ramos, Lauren E. Hittle, Michael R. Davis, Jr., F. Heath Damron, George W. Liechti, Jacek Puchalka, Vitor A. P. Martins dos Santos, Robert K. Ernst, Jason A. Papin, Sebastian Alberti, Antonio Oliver, and Joanna B. Goldberg. Genotypic and phenotypic analyses of a &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt; chronic bronchiectasis isolate reveal differences from cystic fibrosis and laboratory strains. <em>BMC GENOMICS</em>, OCT 30 2015. <a class="reference external" href="https://doi.org/10.1186/s12864-015-2069-0">[DOI]</a>.</p></li>
<li id="id17653"><p>Aarash Bordbar, Douglas McCloskey, Daniel C. Zielinski, Nikolaus Sonnenschein, Neema Jamshidi, and Bernhard O. Palsson. Personalized whole-cell kinetic models of metabolism for discovery in genomics and pharmacodynamics. <em>CELL SYSTEMS</em>, 1(4):283–292, OCT 28 2015. <a class="reference external" href="https://doi.org/10.1016/j.cels.2015.10.003">[DOI]</a>.</p></li>
<li id="id17654"><p>Luca Gerosa, Bart R. B. Haverkorn van Rijsewijk, Dimitris Christodoulou, Karl Kochanowski, Thomas S. B. Schmidt, Elad Noor, and Uwe Sauer. Pseudo-transition analysis identifies the key regulators of dynamic metabolic adaptations from steady-state data. <em>CELL SYSTEMS</em>, 1(4):270–282, OCT 28 2015. <a class="reference external" href="https://doi.org/10.1016/j.cels.2015.09.008">[DOI]</a>.</p></li>
<li id="id17655"><p>Martin Kavscek, Govindprasad Bhutada, Tobias Madl, and Klaus Natter. Optimization of lipid production with a genome-scale model of &lt;i&gt;yarrowia lipolytica&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, OCT 26 2015. <a class="reference external" href="https://doi.org/10.1186/s12918-015-0217-4">[DOI]</a>.</p></li>
<li id="id17656"><p>Claudio Angione and Pietro Lio. Predictive analytics of environmental adaptability in multi-omic network models. <em>SCIENTIFIC REPORTS</em>, OCT 20 2015. <a class="reference external" href="https://doi.org/10.1038/srep15147">[DOI]</a>.</p></li>
<li id="id17657"><p>Josefine Anfelt, Danuta Kaczmarzyk, Kiyan Shabestary, Bjorn Renberg, Johan Rockberg, Jens Nielsen, Mathias Uhlen, and Elton P. Hudson. Genetic and nutrient modulation of acetyl-coa levels in &lt;i&gt;synechocystis&lt;/i&gt; for &lt;i&gt;n&lt;/i&gt;-butanol production. <em>MICROBIAL CELL FACTORIES</em>, OCT 16 2015. <a class="reference external" href="https://doi.org/10.1186/s12934-015-0355-9">[DOI]</a>.</p></li>
<li id="id17658"><p>Chao Ye, Weihua Qiao, Xiaobin Yu, Xiaojun Ji, He Huang, Jackie L. Collier, and Liming Liu. Reconstruction and analysis of the genome-scale metabolic model of &lt;i&gt;schizochytrium limacinum&lt;/i&gt; sr21 for docosahexaenoic acid production. <em>BMC GENOMICS</em>, OCT 16 2015. <a class="reference external" href="https://doi.org/10.1186/s12864-015-2042-y">[DOI]</a>.</p></li>
<li id="id17659"><p>Ahmad A. Mannan, Yoshihiro Toya, Kazuyuki Shimizu, Johnjoe McFadden, Andrzej M. Kierzek, and Andrea Rocco. Integrating kinetic model of &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; with genome scale metabolic fluxes overcomes its open system problem and reveals bistability in central metabolism. <em>PLOS ONE</em>, OCT 15 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0139507">[DOI]</a>.</p></li>
<li id="id17660"><p>Aditya Pratapa, Shankar Balachandran, and Karthik Raman. Fast-sl: an efficient algorithm to identify synthetic lethal sets in metabolic networks. <em>BIOINFORMATICS</em>, 31(20):3299–3305, OCT 15 2015. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btv352">[DOI]</a>.</p></li>
<li id="id17661"><p>Erik M. Quandt, Jimmy Gollihar, Zachary D. Blount, Andrew D. Ellington, George Georgiou, and Jeffrey E. Barrick. Fine-tuning citrate synthase flux potentiates and refines metabolic innovation in the lenski evolution experiment. <em>ELIFE</em>, OCT 14 2015. <a class="reference external" href="https://doi.org/10.7554/eLife.09696">[DOI]</a>.</p></li>
<li id="id17662"><p>Maria A. Zamora, Andres Pinzon, Maria M. Zambrano, Silvia Restrepo, Linda J. Broadbelte, Matthew Moura, Johana Husserl Orjuela, and Andres F. Gonzalez Barrios. A comparison between functional frequency and metabolic flows framed by biogeochemical cycles in metagenomes: the case of “el coquito” hot spring located at colombia's national nevados park. <em>ECOLOGICAL MODELLING</em>, 313:259–265, OCT 10 2015. <a class="reference external" href="https://doi.org/10.1016/j.ecolmodel.2015.06.041">[DOI]</a>.</p></li>
<li id="id17663"><p>Neema Jamshidi and Anu Raghunathan. Cell scale host-pathogen modeling: another branch in the evolution of constraint-based methods. <em>FRONTIERS IN MICROBIOLOGY</em>, OCT 6 2015. <a class="reference external" href="https://doi.org/10.3389/fmicb.2015.01032">[DOI]</a>.</p></li>
<li id="id17664"><p>Goldy De Bhowmick, Lokanand Koduru, and Ramkrishna Sen. Metabolic pathway engineering towards enhancing microalgal lipid biosynthesis for biofuel application-a review. <em>RENEWABLE &amp; SUSTAINABLE ENERGY REVIEWS</em>, 50:1239–1253, OCT 2015. <a class="reference external" href="https://doi.org/10.1016/j.rser.2015.04.131">[DOI]</a>.</p></li>
<li id="id17665"><p>Ali Ebrahim, Eivind Almaas, Eugen Bauer, Aarash Bordbar, Anthony P. Burgard, Roger L. Chang, Andreas Draeger, Iman Famili, Adam M. Feist, Ronan M. T. Fleming, Stephen S. Fong, Vassily Hatzimanikatis, Markus J. Herrgard, Allen Holder, Michael Hucka, Daniel Hyduke, Neema Jamshidi, Sang Yup Lee, Nicolas Le Novere, Joshua A. Lerman, Nathan E. Lewis, Ding Ma, Radhakrishnan Mahadevan, Costas Maranas, Harish Nagarajan, Ali Navid, Jens Nielsen, Lars K. Nielsen, Juan Nogales, Alberto Noronha, Csaba Pal, Bernhard O. Palsson, Jason A. Papin, Kiran R. Patil, Nathan D. Price, Jennifer L. Reed, Michael Saunders, Ryan S. Senger, Nikolaus Sonnenschein, Yuekai Sun, and Ines Thiele. Do genome-scale models need exact solvers or clearer standards? <em>MOLECULAR SYSTEMS BIOLOGY</em>, OCT 2015. <a class="reference external" href="https://doi.org/10.15252/msb.20156157">[DOI]</a>.</p></li>
<li id="id17666"><p>Wout Megchelenbrink, Rotem Katzir, Xiaowen Lu, Eytan Ruppin, and Richard A. Notebaart. Synthetic dosage lethality in the human metabolic network is highly predictive of tumor growth and cancer patient survival. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 112(39):12217–12222, SEP 29 2015. <a class="reference external" href="https://doi.org/10.1073/pnas.1508573112">[DOI]</a>.</p></li>
<li id="id17667"><p>Abhishek Subramanian, Jitesh Jhawar, and Ram Rup Sarkar. Dissecting &lt;i&gt;leishmania infantum&lt;/i&gt; energy metabolism - a systems perspective. <em>PLOS ONE</em>, SEP 14 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0137976">[DOI]</a>.</p></li>
<li id="id17668"><p>Leanne S. Whitmore and Ping Ye. Dissecting germ cell metabolism through network modeling. <em>PLOS ONE</em>, SEP 14 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0137607">[DOI]</a>.</p></li>
<li id="id17669"><p>Matthew B. Biggs, Gregory L. Medlock, Glynis L. Kolling, and Jason A. Papin. Metabolic network modeling of microbial communities. <em>WILEY INTERDISCIPLINARY REVIEWS-SYSTEMS BIOLOGY AND MEDICINE</em>, 7(5):317–334, SEP-OCT 2015. <a class="reference external" href="https://doi.org/10.1002/wsbm.1308">[DOI]</a>.</p></li>
<li id="id17670"><p>Ruggero La Rosa, Juan Nogales, and Fernando Rojo. The crc/crcz-crcy global regulatory system helps the integration of gluconeogenic and glycolytic metabolism in &lt;i&gt;pseudomonas putida&lt;/i&gt;. <em>ENVIRONMENTAL MICROBIOLOGY</em>, 17(9):3362–3378, SEP 2015. <a class="reference external" href="https://doi.org/10.1111/1462-2920.12812">[DOI]</a>.</p></li>
<li id="id17671"><p>Hector Garcia Martin, Vinay Satish Kumar, Daniel Weaver, Amit Ghosh, Victor Chubukov, Aindrila Mukhopadhyay, Adam Arkin, and Jay D. Keasling. A method to constrain genome-scale models with &lt;sup&gt;13&lt;/sup&gt;c labeling data. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1004363">[DOI]</a>.</p></li>
<li id="id17672"><p>Arne C. Reimers. Obstructions to sampling qualitative properties. <em>PLOS ONE</em>, AUG 19 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0135636">[DOI]</a>.</p></li>
<li id="id17673"><p>Mohammad Tajparast and Dominic Frigon. Genome-scale metabolic model of &lt;i&gt;rhodococcus jostii&lt;/i&gt; rha1 (imt1174) to study the accumulation of storage compounds during nitrogen-limited condition. <em>BMC SYSTEMS BIOLOGY</em>, AUG 7 2015. <a class="reference external" href="https://doi.org/10.1186/s12918-015-0190-y">[DOI]</a>.</p></li>
<li id="id17674"><p>Franziska Erlmeier, Annette Feuchtinger, Daniela Borgmann, Martina Rudelius, Michael Autenrieth, Axel Karl Walch, and Gregor Weirich. Supremacy of modern morphometry in typing renal oncocytoma and malignant look-alikes. <em>HISTOCHEMISTRY AND CELL BIOLOGY</em>, 144(2):147–156, AUG 2015. <a class="reference external" href="https://doi.org/10.1007/s00418-015-1324-4">[DOI]</a>.</p></li>
<li id="id17675"><p>H. Fouladiha, S. -A. Marashi, and M. A. Shokrgozar. Reconstruction and validation of a constraint-based metabolic network model for bone marrow-derived mesenchymal stem cells. <em>CELL PROLIFERATION</em>, 48(4):475–485, AUG 2015. <a class="reference external" href="https://doi.org/10.1111/cpr.12197">[DOI]</a>.</p></li>
<li id="id17676"><p>Sayed-Rzgar Hosseini, Aditya Barve, and Andreas Wagner. Exhaustive analysis of a genotype space comprising 10&lt;sup&gt;15&lt;/sup&gt; central carbon metabolisms reveals an organization conducive to metabolic innovation. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1004329">[DOI]</a>.</p></li>
<li id="id17677"><p>Haythem Latif, Merve Sahin, Janna Tarasova, Yekaterina Tarasova, Vasiliy A. Portnoy, Juan Nogales, and Karsten Zengler. Adaptive evolution of &lt;i&gt;thermotoga maritima&lt;/i&gt; reveals plasticity of the abc transporter network. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, 81(16):5477–5485, AUG 2015. <a class="reference external" href="https://doi.org/10.1128/AEM.01365-15">[DOI]</a>.</p></li>
<li id="id17678"><p>Sayed-Amir Marashi and Zhaleh Hosseini. On correlated reaction sets and coupled reaction sets in metabolic networks. <em>JOURNAL OF BIOINFORMATICS AND COMPUTATIONAL BIOLOGY</em>, AUG 2015. <a class="reference external" href="https://doi.org/10.1142/S0219720015710031">[DOI]</a>.</p></li>
<li id="id17679"><p>Abdul Hakim Mohamed Salleh, Mohd Saberi Mohamad, Safaai Deris, Sigeru Omatu, Florentino Fdez-Riverola, and Juan Manuel Corchado. Gene knockout identification for metabolite production improvement using a hybrid of genetic ant colony optimization and flux balance analysis. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 20(4):685–693, AUG 2015. <a class="reference external" href="https://doi.org/10.1007/s12257-015-0276-9">[DOI]</a>.</p></li>
<li id="id17680"><p>Elias W. Krumholz and Igor G. L. Libourel. Sequence-based network completion reveals the integrality of missing reactions in metabolic networks. <em>JOURNAL OF BIOLOGICAL CHEMISTRY</em>, 290(31):19197–19207, JUL 31 2015. <a class="reference external" href="https://doi.org/10.1074/jbc.M114.634121">[DOI]</a>.</p></li>
<li id="id17681"><p>Cheng Zhang, Boyang Ji, Adil Mardinoglu, Jens Nielsen, and Qiang Hua. Logical transformation of genome-scale metabolic models for gene level applications and analysis. <em>BIOINFORMATICS</em>, 31(14):2324–2331, JUL 15 2015. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btv134">[DOI]</a>.</p></li>
<li id="id17682"><p>Jahir M. Gutierrez and Nathan E. Lewis. Optimizing eukaryotic cell hosts for protein production through systems biotechnology and genome-scale modeling. <em>BIOTECHNOLOGY JOURNAL</em>, 10(7, SI):939–949, JUL 2015. <a class="reference external" href="https://doi.org/10.1002/biot.201400647">[DOI]</a>.</p></li>
<li id="id17683"><p>Joshua J. Hamilton, Montserrat Calixto Contreras, and Jennifer L. Reed. Thermodynamics and h&lt;sub&gt;2&lt;/sub&gt; transfer in a methanogenic, syntrophic community. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUL 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1004364">[DOI]</a>.</p></li>
<li id="id17684"><p>Almut Heinken and Ines Thiele. Systems biology of host-microbe metabolomics. <em>WILEY INTERDISCIPLINARY REVIEWS-SYSTEMS BIOLOGY AND MEDICINE</em>, 7(4):195–219, JUL-AUG 2015. <a class="reference external" href="https://doi.org/10.1002/wsbm.1301">[DOI]</a>.</p></li>
<li id="id17685"><p>Devin H. Currie, Babu Raman, Christopher M. Gowen, Timothy J. Tschaplinski, Miriam L. Land, Steven D. Brown, Sean F. Covalla, Dawn M. Klingeman, Zamin K. Yang, Nancy L. Engle, Courtney M. Johnson, Miguel Rodriguez, A. Joe Shaw, William R. Kenealy, Lee R. Lynd, Stephen S. Fong, Jonathan R. Mielenz, Brian H. Davison, David A. Hogsett, and Christopher D. Herring. Genome-scale resources for &lt;i&gt;thermoanaerobacterium saccharolyticum&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, JUN 26 2015. <a class="reference external" href="https://doi.org/10.1186/s12918-015-0159-x">[DOI]</a>.</p></li>
<li id="id17686"><p>Yali Wang, Nan Xu, Chao Ye, Liming Liu, Zhongping Shi, and Jing Wu. Reconstruction and &lt;i&gt;in silico&lt;/i&gt; analysis of an &lt;i&gt;actinoplanes&lt;/i&gt; sp se50/110 genome-scale metabolic model for acarbose production. <em>FRONTIERS IN MICROBIOLOGY</em>, JUN 25 2015. <a class="reference external" href="https://doi.org/10.3389/fmicb.2015.00632">[DOI]</a>.</p></li>
<li id="id17687"><p>Zahra Azimzadeh Irani, Amir Maghsoudi, Seyed Abbas Shojaosadati, and Ehsan Motamedian. Development and &lt;i&gt;in silico&lt;/i&gt; analysis of a new nitrogen-limited feeding strategy for fed-batch cultures of &lt;i&gt;pichia pastoris&lt;/i&gt; based on a simple ph-control system. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, 98:1–9, JUN 15 2015. <a class="reference external" href="https://doi.org/10.1016/j.bej.2015.02.016">[DOI]</a>.</p></li>
<li id="id17688"><p>Haiqin Chen, Guangfei Hao, Lei Wang, Hongchao Wang, Zhennan Gu, Liming Liu, Hao Zhang, Wei Chen, and Yong Q. Chen. Identification of a critical determinant that enables efficient fatty acid synthesis in oleaginous fungi. <em>SCIENTIFIC REPORTS</em>, JUN 10 2015. <a class="reference external" href="https://doi.org/10.1038/srep11247">[DOI]</a>.</p></li>
<li id="id17689"><p>Maike K. Aurich, Giuseppe Paglia, Ottar Rolfsson, Sigrun Hrafnsdottir, Manuela Magnusdottir, Magdalena M. Stefaniak, Bernhard O. Palsson, Ronan M. T. Fleming, and Ines Thiele. Prediction of intracellular metabolic states from extracellular metabolomic data. <em>METABOLOMICS</em>, 11(3):603–619, JUN 2015. <a class="reference external" href="https://doi.org/10.1007/s11306-014-0721-3">[DOI]</a>.</p></li>
<li id="id17690"><p>Julian Brandl and Mikael R. Andersen. Current state of genome-scale modeling in filamentous fungi. <em>BIOTECHNOLOGY LETTERS</em>, 37(6):1131–1139, JUN 2015. <a class="reference external" href="https://doi.org/10.1007/s10529-015-1782-8">[DOI]</a>.</p></li>
<li id="id17691"><p>Andrew L. Damiani, Q. Peter He, Thomas W. Jeffries, and Jin Wang. Comprehensive evaluation of two genome-scale metabolic network models for &lt;i&gt;scheffersomyces stipitis&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 112(6):1250–1262, JUN 2015. <a class="reference external" href="https://doi.org/10.1002/bit.25535">[DOI]</a>.</p></li>
<li id="id17692"><p>David Hala, Lene H. Petersen, Dalma Martinovic, and Duane B. Huggett. &lt;i&gt;in silico&lt;/i&gt; analysis of perturbed steroidogenesis and gonad growth in fathead minnows (&lt;i&gt;p&lt;/i&gt;. &lt;i&gt;promelas&lt;/i&gt;) exposed to 17α-ethynylestradiol. <em>SYSTEMS BIOLOGY IN REPRODUCTIVE MEDICINE</em>, 61(3):122–138, JUN 2015. <a class="reference external" href="https://doi.org/10.3109/19396368.2015.1035817">[DOI]</a>.</p></li>
<li id="id17693"><p>Almut Heinken and Ines Thiele. Anoxic conditions promote species-specific mutualism between gut microbes &lt;i&gt;in silico&lt;/i&gt;. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, 81(12):4049–4061, JUN 2015. <a class="reference external" href="https://doi.org/10.1128/AEM.00101-15">[DOI]</a>.</p></li>
<li id="id17694"><p>Arnau Montagud, Daniel Gamermann, Pedro Fernandez de Cordoba, and Javier F. Urchueguia. &lt;i&gt;synechocystis&lt;/i&gt; sp pcc6803 metabolic models for the enhanced production of hydrogen. <em>CRITICAL REVIEWS IN BIOTECHNOLOGY</em>, 35(2):184–198, JUN 2015. <a class="reference external" href="https://doi.org/10.3109/07388551.2013.829799">[DOI]</a>.</p></li>
<li id="id17695"><p>Ottar Rolfsson and Bernhard O. Palsson. Decoding the jargon of bottom-up metabolic systems biology. <em>BIOESSAYS</em>, 37(6):588–591, JUN 2015. <a class="reference external" href="https://doi.org/10.1002/bies.201400187">[DOI]</a>.</p></li>
<li id="id17696"><p>Daniel C. Zielinski, Fabian V. Filipp, Aarash Bordbar, Kasper Jensen, Jeffrey W. Smith, Markus J. Herrgard, Monica L. Mo, and Bernhard O. Palsson. Pharmacogenomic and clinical data link non-pharmacokinetic metabolic dysregulation to drug side effect pathogenesis. <em>NATURE COMMUNICATIONS</em>, JUN 2015. <a class="reference external" href="https://doi.org/10.1038/ncomms8101">[DOI]</a>.</p></li>
<li id="id17697"><p>Nicholas D. Gold, Christopher M. Gowen, Francois-Xavier Lussier, Sarat C. Cautha, Radhakrishnan Mahadevan, and Vincent J. J. Martin. Metabolic engineering of a tyrosine-overproducing yeast platform using targeted metabolomics. <em>MICROBIAL CELL FACTORIES</em>, MAY 28 2015. <a class="reference external" href="https://doi.org/10.1186/s12934-015-0252-2">[DOI]</a>.</p></li>
<li id="id17698"><p>Thomas Bayer, Sofia Milker, Thomas Wiesinger, Florian Rudroff, and Marko. D. Mihovilovic. Designer microorganisms for optimized redox cascade reactions - challenges and future perspectives. <em>ADVANCED SYNTHESIS &amp; CATALYSIS</em>, 357(8, SI):1587–1618, MAY 26 2015. <a class="reference external" href="https://doi.org/10.1002/adsc.201500202">[DOI]</a>.</p></li>
<li id="id17699"><p>Edward J. O'Brien, Jonathan M. Monk, and Bernhard O. Palsson. Using genome-scale models to predict biological capabilities. <em>CELL</em>, 161(5):971–987, MAY 21 2015. <a class="reference external" href="https://doi.org/10.1016/j.cell.2015.05.019">[DOI]</a>.</p></li>
<li id="id17700"><p>Kristin J. Adolfsen and Mark P. Brynildsen. Futile cycling increases sensitivity toward oxidative stress in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 29:26–35, MAY 2015. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2015.02.006">[DOI]</a>.</p></li>
<li id="id17701"><p>Justin Tan, Cristal Zuniga, and Karsten Zengler. Unraveling interactions in microbial communities - from co-cultures to microbiomes. <em>JOURNAL OF MICROBIOLOGY</em>, 53(5):295–305, MAY 2015. <a class="reference external" href="https://doi.org/10.1007/s12275-015-5060-1">[DOI]</a>.</p></li>
<li id="id17702"><p>Daniele De Martino, Matteo Mori, and Valerio Parisi. Uniform sampling of steady states in metabolic networks: heterogeneous scales and rounding. <em>PLOS ONE</em>, APR 7 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0122670">[DOI]</a>.</p></li>
<li id="id17703"><p>Javier Ferrer Valenzuela, Luis Andres Pinuer, Apolinaria Garcia Cancino, and Rodrigo Borquez Yanez. Metabolic fluxes in lactic acid bacteria-a review. <em>FOOD BIOTECHNOLOGY</em>, 29(2):185–217, APR 3 2015. <a class="reference external" href="https://doi.org/10.1080/08905436.2015.1027913">[DOI]</a>.</p></li>
<li id="id17704"><p>Gabriele Fontanarosa, Giulia Menichetti, Enrico Giampieri, Gastone Castellani, Giovanni Martinelli, and Daniel Remondini. Network approaches for analysis and modeling of the human metabolism. <em>JOURNAL OF MECHANICS IN MEDICINE AND BIOLOGY</em>, APR 2015. 19th International Conference on Mechanics in Medicine and Biology, Bologna, ITALY, SEP 03-05, 2014. <a class="reference external" href="https://doi.org/10.1142/S0219519415400266">[DOI]</a>.</p></li>
<li id="id17705"><p>Nicolas Loira, Anna Zhukova, and David James Sherman. Pantograph: a template-based method for genome-scale metabolic model reconstruction. <em>JOURNAL OF BIOINFORMATICS AND COMPUTATIONAL BIOLOGY</em>, APR 2015. <a class="reference external" href="https://doi.org/10.1142/S0219720015500067">[DOI]</a>.</p></li>
<li id="id17706"><p>Anand K. Gavai, Farahaniza Supandi, Hannes Hettling, Paul Murrell, Jack A. M. Leunissen, and Johannes H. G. M. van Beek. Using bioconductor package biggr for metabolic flux estimation based on gene expression changes in brain. <em>PLOS ONE</em>, MAR 25 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0119016">[DOI]</a>.</p></li>
<li id="id17707"><p>Chao Shi, Jian Yin, Zhe Liu, Jian-Xin Wu, Qi Zhao, Jian Ren, and Nan Yao. A systematic simulation of the effect of salicylic acid on sphingolipid metabolism. <em>FRONTIERS IN PLANT SCIENCE</em>, MAR 25 2015. <a class="reference external" href="https://doi.org/10.3389/fpls.2015.00186">[DOI]</a>.</p></li>
<li id="id17708"><p>Natalie J. Stanford, Pierre Millard, and Neil Swainston. Robokod: microbial strain design for (over)production of target compounds. <em>FRONTIERS IN CELL AND DEVELOPMENTAL BIOLOGY</em>, MAR 24 2015. <a class="reference external" href="https://doi.org/10.3389/fcell.2015.00017">[DOI]</a>.</p></li>
<li id="id17709"><p>John A. Cole, Lars Kohler, Jamila Hedhli, and Zaida Luthey-Schulten. Spatially-resolved metabolic cooperativity within dense bacterial colonies. <em>BMC SYSTEMS BIOLOGY</em>, MAR 18 2015. <a class="reference external" href="https://doi.org/10.1186/s12918-015-0155-1">[DOI]</a>.</p></li>
<li id="id17710"><p>Anna-Lena Hillje, Elisabeth Beckmann, Maria A. S. Pavlou, Christian Jaeger, Maria P. Pacheco, Thomas Sauter, Jens C. Schwamborn, and Lars Lewejohann. The neural stem cell fate determinant trim32 regulates complex behavioral traits. <em>FRONTIERS IN CELLULAR NEUROSCIENCE</em>, MAR 18 2015. <a class="reference external" href="https://doi.org/10.3389/fncel.2015.00075">[DOI]</a>.</p></li>
<li id="id17711"><p>A. Folch-Fortuny, M. Tortajada, J. M. Prats-Montalban, F. Llaneras, J. Pico, and A. Ferrer. Mcr-als on metabolic networks: obtaining more meaningful pathways. <em>CHEMOMETRICS AND INTELLIGENT LABORATORY SYSTEMS</em>, 142:293–303, MAR 15 2015. 3rd European Conference on Process Analytics and Control Technology, Barcelona, SPAIN, MAY 06-09, 2014. <a class="reference external" href="https://doi.org/10.1016/j.chemolab.2014.10.004">[DOI]</a>.</p></li>
<li id="id17712"><p>Marco Fondi, Isabel Maida, Elena Perrin, Alessandra Mellera, Stefano Mocali, Ermenegilda Parrilli, Maria Luisa Tutino, Pietro Lio, and Renato Fani. Genome-scale metabolic reconstruction and constraint-based modelling of the antarctic bacterium pseudoalteromonas haloplanktis tac125. <em>ENVIRONMENTAL MICROBIOLOGY</em>, 17(3):751–766, MAR 2015. <a class="reference external" href="https://doi.org/10.1111/1462-2920.12513">[DOI]</a>.</p></li>
<li id="id17713"><p>Byoungjin Kim, Won Jun Kim, Dong In Kim, and Sang Yup Lee. Applications of genome-scale metabolic network model in metabolic engineering. <em>JOURNAL OF INDUSTRIAL MICROBIOLOGY &amp; BIOTECHNOLOGY</em>, 42(3, SI):339–348, MAR 2015. <a class="reference external" href="https://doi.org/10.1007/s10295-014-1554-9">[DOI]</a>.</p></li>
<li id="id17714"><p>Ehsan Motamedian, Ghazaleh Ghavami, and Soroush Sardari. Investigation on metabolism of cisplatin resistant ovarian cancer using a genome scale metabolic model and microarray data. <em>IRANIAN JOURNAL OF BASIC MEDICAL SCIENCES</em>, 18(3):267–276, MAR 2015.</p></li>
<li id="id17715"><p>Karthik Sekar and Keith E. J. Tyo. Regulatory effects on central carbon metabolism from poly-3-hydroxybutryate synthesis. <em>METABOLIC ENGINEERING</em>, 28:180–189, MAR 2015. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2015.01.003">[DOI]</a>.</p></li>
<li id="id17716"><p>Evangelos Simeonidis and Nathan D. Price. Genome-scale modeling for metabolic engineering. <em>JOURNAL OF INDUSTRIAL MICROBIOLOGY &amp; BIOTECHNOLOGY</em>, 42(3, SI):327–338, MAR 2015. <a class="reference external" href="https://doi.org/10.1007/s10295-014-1576-3">[DOI]</a>.</p></li>
<li id="id17717"><p>Michael Binns, Pedro de Atauri, Anestis Vlysidis, Marta Cascante, and Constantinos Theodoropoulos. Sampling with poling-based flux balance analysis: optimal versus sub-optimal flux space analysis of &lt;i&gt;actinobacillus succinogenes&lt;/i&gt;. <em>BMC BIOINFORMATICS</em>, FEB 18 2015. <a class="reference external" href="https://doi.org/10.1186/s12859-015-0476-5">[DOI]</a>.</p></li>
<li id="id17718"><p>Eduard J. Kerkhoven, Petri-Jaan Lahtvee, and Jens Nielsen. Applications of computational modeling in metabolic engineering of yeast. <em>FEMS YEAST RESEARCH</em>, FEB 2015. <a class="reference external" href="https://doi.org/10.1111/1567-1364.12199">[DOI]</a>.</p></li>
<li id="id17719"><p>Hae Woo Lee, Andrew Christie, Jason A. Starkey, Erik K. Read, and Seongkyu Yoon. Intracellular metabolic flux analysis of cho cells supplemented with wheat hydrolysates for improved mab production and cell-growth. <em>JOURNAL OF CHEMICAL TECHNOLOGY AND BIOTECHNOLOGY</em>, 90(2, SI):291–302, FEB 2015. <a class="reference external" href="https://doi.org/10.1002/jctb.4523">[DOI]</a>.</p></li>
<li id="id17720"><p>Yanfeng Liu, Hyun-dong Shin, Jianghua Li, and Long Liu. Toward metabolic engineering in the context of system biology and synthetic biology: advances and prospects. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 99(3):1109–1118, FEB 2015. <a class="reference external" href="https://doi.org/10.1007/s00253-014-6298-y">[DOI]</a>.</p></li>
<li id="id17721"><p>Ali Masoudi-Nejad and Yazdan Asgari. Metabolic cancer biology: structural-based analysis of cancer as a metabolic disease, new sights and opportunities for disease treatment. <em>SEMINARS IN CANCER BIOLOGY</em>, 30:21–29, FEB 2015. <a class="reference external" href="https://doi.org/10.1016/j.semcancer.2014.01.007">[DOI]</a>.</p></li>
<li id="id17722"><p>Cheng Wang, Zhi-Luo Deng, Zhi-Ming Xie, Xin-Yi Chu, Ji-Wei Chang, De-Xin Kong, Bao-Ju Li, Hong-Yu Zhang, and Ling-Ling Chen. Construction of a genome-scale metabolic network of the plant pathogen &lt;i&gt;pectobacterium&lt;/i&gt; &lt;i&gt;carotovorum&lt;/i&gt; provides new strategies for bactericide discovery. <em>FEBS LETTERS</em>, 589(3):285–294, JAN 30 2015. <a class="reference external" href="https://doi.org/10.1016/j.febslet.2014.12.010">[DOI]</a>.</p></li>
<li id="id17723"><p>Sanu Shameer, Flora J. Logan-Klumpler, Florence Vinson, Ludovic Cottret, Benjamin Merlet, Fiona Achcar, Michael Boshart, Matthew Berriman, Rainer Breitling, Fredrric Bringaud, Peter Butikofer, Amy M. Cattanach, Bridget Bannerman-Chukualim, Darren J. Creek, Kathryn Crouch, Harry P. de Koning, Hubert Denise, Charles Ebikeme, Alan H. Fairlamb, Michael A. J. Ferguson, Michael L. Ginger, Christiane Hertz-Fowler, Eduard J. Kerkhoven, Pascal Maeser, Paul A. M. Michels, Archana Nayak, David W. Nes, Derek P. Nolan, Christian Olsen, Fatima Silva-Franco, Terry K. Smith, Martin C. Taylor, Aloysius G. M. Tielens, Michael D. Urbaniak, Jaap J. van Hellemond, Isabel M. Vincent, Shane R. Wilkinson, Susan Wyllie, Fred R. Opperdoes, Michael P. Barrett, and Fabien Jourdan. Trypanocyc: a community-led biochemical pathways database for &lt;i&gt;trypanosoma brucei&lt;/i&gt;. <em>NUCLEIC ACIDS RESEARCH</em>, 43(D1):D637–D644, JAN 28 2015. <a class="reference external" href="https://doi.org/10.1093/nar/gku944">[DOI]</a>.</p></li>
<li id="id17724"><p>Brandon Barker, Lin Xu, and Zhenglong Gu. Dynamic epistasis under varying environmental perturbations. <em>PLOS ONE</em>, JAN 27 2015. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0114911">[DOI]</a>.</p></li>
<li id="id17725"><p>Cristiana Gomes de Oliveira Dal'Molin, Lake-Ee Quek, Pedro A. Saa, and Lars K. Nielsen. A multi-tissue genome-scale metabolic modeling framework for the analysis of whole plant systems. <em>FRONTIERS IN PLANT SCIENCE</em>, JAN 22 2015. <a class="reference external" href="https://doi.org/10.3389/fpls.2015.00004">[DOI]</a>.</p></li>
<li id="id17726"><p>Josh Quartermana, Soo Rin Kim, Pan-Jun Kim, and Yong-Su Jin. Enhanced hexose fermentation by &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; through integration of stoichiometric modeling and genetic screening. <em>JOURNAL OF BIOTECHNOLOGY</em>, 194:48–57, JAN 20 2015. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2014.11.017">[DOI]</a>.</p></li>
<li id="id17727"><p>German Plata, Christopher S. Henry, and Dennis Vitkup. Long-term phenotypic evolution of bacteria. <em>NATURE</em>, 517(7534):369–U498, JAN 15 2015. <a class="reference external" href="https://doi.org/10.1038/nature13827">[DOI]</a>.</p></li>
<li id="id17728"><p>Raphael Voges, Stephanie Corsten, Wolfgang Wiechert, and Stephan Noack. Absolute quantification of &lt;i&gt;corynebacterium glutamicum&lt;/i&gt; glycolytic and anaplerotic enzymes by qconcat. <em>JOURNAL OF PROTEOMICS</em>, 113:366–377, JAN 15 2015. <a class="reference external" href="https://doi.org/10.1016/j.jprot.2014.10.008">[DOI]</a>.</p></li>
<li id="id17729"><p>Chao Ye, Nan Xu, Haiqin Chen, Yong Q. Chen, Wei Chen, and Liming Liu. Reconstruction and analysis of a genome-scale metabolic model of the oleaginous fungus &lt;i&gt;mortierella alpina&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, JAN 13 2015. <a class="reference external" href="https://doi.org/10.1186/s12918-014-0137-8">[DOI]</a>.</p></li>
<li id="id17730"><p>Nan Xu, Jie Liu, Lianzhong Ai, and Liming Liu. Reconstruction and analysis of the genome-scale metabolic model of &lt;i&gt;lactobacillus casei&lt;/i&gt; lc2w. <em>GENE</em>, 554(2):140–147, JAN 10 2015. <a class="reference external" href="https://doi.org/10.1016/j.gene.2014.10.034">[DOI]</a>.</p></li>
<li id="id17731"><p>Parizad Babaei, Sayed-Amir Marashi, and Sedigheh Asad. Genome-scale reconstruction of the metabolic network in &lt;i&gt;pseudomonas stutzeri&lt;/i&gt; a1501. <em>MOLECULAR BIOSYSTEMS</em>, 11(11):3022–3032, 2015. <a class="reference external" href="https://doi.org/10.1039/c5mb00086f">[DOI]</a>.</p></li>
<li id="id17732"><p>Joris Beld, D. John Lee, and Michael D. Burkart. Fatty acid biosynthesis revisited: structure elucidation and metabolic engineering. <em>MOLECULAR BIOSYSTEMS</em>, 11(1):38–59, JAN 2015. <a class="reference external" href="https://doi.org/10.1039/c4mb00443d">[DOI]</a>.</p></li>
<li id="id17733"><p>Irina Borodina, Kanchana R. Kildegaard, Niels B. Jensen, Thomas H. Blicher, Jerome Maury, Svetlana Sherstyk, Konstantin Schneider, Pedro Lamosa, Markus J. Herrgard, Inger Rosenstand, Fredrik Oberg, Jochen Forster, and Jens Nielsen. Establishing a synthetic pathway for high-level production of 3-hydroxypropionic acid in &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; via β-alanine. <em>METABOLIC ENGINEERING</em>, 27:57–64, JAN 2015. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2014.10.003">[DOI]</a>.</p></li>
<li id="id17734"><p>Pratish Gawand, Fatumina Said Abukar, Naveen Venayak, Siavash Partow, Adilson E. Motter, and Radhakrishnan Mahadevan. Sub-optimal phenotypes of double-knockout mutants of &lt;i&gt;escherichia coli&lt;/i&gt; depend on the order of gene deletions. <em>INTEGRATIVE BIOLOGY</em>, 7(8):930–939, 2015. <a class="reference external" href="https://doi.org/10.1039/c5ib00096c">[DOI]</a>.</p></li>
<li id="id17735"><p>Almut Heinken and Ines Thiele. Systematic prediction of health-relevant human-microbial co-metabolism through a computational framework. <em>GUT MICROBES</em>, 6(2):120–130, 2015. <a class="reference external" href="https://doi.org/10.1080/19490976.2015.1023494">[DOI]</a>.</p></li>
<li id="id17736"><p>M. Ahsanul Islam, Karsten Zengler, Elizabeth A. Edwards, Radhakrishnan Mahadevan, and Gregory Stephanopoulos. Investigating &lt;i&gt;moorella&lt;/i&gt; &lt;i&gt;thermoacetica&lt;/i&gt; metabolism with a genome-scale constraint-based metabolic model. <em>INTEGRATIVE BIOLOGY</em>, 7(8):869–882, 2015. <a class="reference external" href="https://doi.org/10.1039/c5ib00095e">[DOI]</a>.</p></li>
<li id="id17737"><p>Henning Knoop and Ralf Steuer. A computational analysis of stoichiometric constraints and trade-offs in cyanobacterial biofuel production. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, 2015. <a class="reference external" href="https://doi.org/10.3389/fbioe.2015.00047">[DOI]</a>.</p></li>
<li id="id17738"><p>Yi Li, Guojian Zhang, and Blaine A. Pfeifer. Current and emerging options for taxol production. In J Schrader and J Bohlmann, editors, <em>BIOTECHNOLOGY OF ISOPRENOIDS</em>, volume 148 of Advances in Biochemical Engineering-Biotechnology, pages 405–425. 2015. <a class="reference external" href="https://doi.org/10.1007/10\_2014\_292">[DOI]</a>.</p></li>
<li id="id17739"><p>Longfei Mao, Averina Nicolae, Miguel A. P. Oliveira, Feng He, Siham Hachi, and Ronan M. T. Fleming. A constraint-based modelling approach to metabolic dysfunction in parkinson's disease. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 13:484–491, 2015. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2015.08.002">[DOI]</a>.</p></li>
<li id="id17740"><p>Andras Pasztor, Pauli Kallio, David Malatinszky, M. Kalim Akhtar, and Patrik R. Jones. A synthetic o&lt;sub&gt;2&lt;/sub&gt;-tolerant butanol pathway exploiting native fatty acid biosynthesis in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 112(1):120–128, JAN 2015. <a class="reference external" href="https://doi.org/10.1002/bit.25324">[DOI]</a>.</p></li>
<li id="id17741"><p>Christopher J. Petzold, Leanne Jade G. Chan, Melissa Nhan, and Paul D. Adams. Analytics for metabolic engineering. <em>FRONTIERS IN BIOENGINEERING AND BIOTECHNOLOGY</em>, 2015. <a class="reference external" href="https://doi.org/10.3389/fbioe.2015.00135">[DOI]</a>.</p></li>
<li id="id17742"><p>Jae Yong Ryu, Hyun Uk Kim, and Sang Yup Lee. Reconstruction of genome-scale human metabolic models using omics data. <em>INTEGRATIVE BIOLOGY</em>, 7(8):859–868, 2015. <a class="reference external" href="https://doi.org/10.1039/c5ib00002e">[DOI]</a>.</p></li>
<li id="id17743"><p>Swagatika Sahoo, Hulda S. Haraldsdottir, Ronan M. T. Fleming, and Ines Thiele. Modeling the effects of commonly used drugs on human metabolism. <em>FEBS JOURNAL</em>, 282(2):297–317, JAN 2015. <a class="reference external" href="https://doi.org/10.1111/febs.13128">[DOI]</a>.</p></li>
<li id="id17744"><p>Nadine Castelhano Santos, Maria Olivia Pereira, and Analia Lourenco. Pathogenicity phenomena in three model systems: from network mining to emerging system-level properties. <em>BRIEFINGS IN BIOINFORMATICS</em>, 16(1):169–182, JAN 2015. <a class="reference external" href="https://doi.org/10.1093/bib/bbt071">[DOI]</a>.</p></li>
<li id="id17745"><p>Takeyuki Tamura, Wei Lu, and Tatsuya Akutsu. Computational methods for modification of metabolic networks. <em>COMPUTATIONAL AND STRUCTURAL BIOTECHNOLOGY JOURNAL</em>, 13:376–381, 2015. <a class="reference external" href="https://doi.org/10.1016/j.csbj.2015.05.004">[DOI]</a>.</p></li>
<li id="id17746"><p>Katja Tummler, Clemens Kuhn, and Edda Klipp. Dynamic metabolic models in context: biomass backtracking. <em>INTEGRATIVE BIOLOGY</em>, 7(8):940–951, 2015. <a class="reference external" href="https://doi.org/10.1039/c5ib00050e">[DOI]</a>.</p></li>
<li id="id17747"><p>Zhaobin Xu, Sabina Islam, Thomas K. Wood, and Zuyi Huang. An integrated modeling and experimental approach to study the influence of environmental nutrients on biofilm formation of &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt;. <em>BIOMED RESEARCH INTERNATIONAL</em>, 2015. <a class="reference external" href="https://doi.org/10.1155/2015/506782">[DOI]</a>.</p></li>
<li id="id17748"><p>Daichi Yukihira, Yoshinori Fujimura, Hiroyuki Wariishi, and Daisuke Miura. Bacterial metabolism in immediate response to nutritional perturbation with temporal and network view of metabolites. <em>MOLECULAR BIOSYSTEMS</em>, 11(9):2473–2482, 2015. <a class="reference external" href="https://doi.org/10.1039/c5mb00182j">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-10" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-10" name="0-10" role="tabpanel" tabindex="0"><div class="docutils container" id="id18201">
<ol class="arabic simple" start="1">
<li id="id19569"><p>Alberto Noronha, Paulo Vilaca, and Miguel Rocha. An integrated network visualization framework towards metabolic engineering applications. <em>BMC BIOINFORMATICS</em>, DEC 30 2014. <a class="reference external" href="https://doi.org/10.1186/s12859-014-0420-0">[DOI]</a>.</p></li>
<li id="id19570"><p>Alfered Fernandez-Castane, Tamas Feher, Pablo Carbonell, Cyrille Pauthenier, and Jean-Loup Faulon. Computer-aided design for metabolic engineering. <em>JOURNAL OF BIOTECHNOLOGY</em>, 192(B):302–313, DEC 20 2014. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2014.03.029">[DOI]</a>.</p></li>
<li id="id19571"><p>Jordan O. Hay, Hai Shi, Nicolas Heinzel, Inga Hebbelmann, Hardy Rolletschek, and Jorg Schwender. Integration of a constraint-based metabolic model of &lt;i&gt;brassica napus&lt;/i&gt; developing seeds with &lt;sup&gt;13&lt;/sup&gt;c-metabolic flux analysis. <em>FRONTIERS IN PLANT SCIENCE</em>, DEC 19 2014. <a class="reference external" href="https://doi.org/10.3389/fpls.2014.00724">[DOI]</a>.</p></li>
<li id="id19572"><p>Jose A. Gomez, Kai Hoffner, and Paul I. Barton. Dfbalab: a fast and reliable matlab code for dynamic flux balance analysis. <em>BMC BIOINFORMATICS</em>, DEC 18 2014. <a class="reference external" href="https://doi.org/10.1186/s12859-014-0409-8">[DOI]</a>.</p></li>
<li id="id19573"><p>Viswanadham Sridhara, Austin G. Meyer, Piyush Rai, Jeffrey E. Barrick, Pradeep Ravikumar, Daniel Segre, and Claus O. Wilke. Predicting growth conditions from internal metabolic fluxes in an &lt;i&gt;in-silico&lt;/i&gt; model of &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt;. <em>PLOS ONE</em>, DEC 12 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0114608">[DOI]</a>.</p></li>
<li id="id19574"><p>Pankaj Narang, Shawez Khan, Anmol Jaywant Hemrom, Andrew Michael Lynn, and Open Source Drug Discovery. Metanet - a web-accessible interactive platform for biological metabolic network analysis. <em>BMC SYSTEMS BIOLOGY</em>, DEC 5 2014. <a class="reference external" href="https://doi.org/10.1186/s12918-014-0130-2">[DOI]</a>.</p></li>
<li id="id19575"><p>Rafael Carreira, Pedro Evangelista, Paulo Maia, Paulo Vilaca, Marcellinus Pont, Jean-Francois Tomb, Isabel Rocha, and Miguel Rocha. Cbfa: phenotype prediction integrating metabolic models with constraints derived from experimental data. <em>BMC SYSTEMS BIOLOGY</em>, DEC 3 2014. <a class="reference external" href="https://doi.org/10.1186/s12918-014-0123-1">[DOI]</a>.</p></li>
<li id="id19576"><p>Daniela Carolina Florez Parra, Vanessa Ramirez Tovar, and Andres Fernando Gonzalez Barrios. Elucidating the role of metabolites as quorum sensing signals using phase plane analysis: the case of indole in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>JOURNAL OF BIOLOGICAL SYSTEMS</em>, 22(4):523–531, DEC 2014. <a class="reference external" href="https://doi.org/10.1142/S0218339014500193">[DOI]</a>.</p></li>
<li id="id19577"><p>K. Olavarria, J. De Ingeniis, D. C. Zielinski, M. Fuentealba, R. Munoz, D. McCloskey, A. M. Feist, and R. Cabrera. Metabolic impact of an nadh-producing glucose-6-phosphate dehydrogenase in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>MICROBIOLOGY-SGM</em>, 160(12):2780–2793, DEC 2014. <a class="reference external" href="https://doi.org/10.1099/mic.0.082180-0">[DOI]</a>.</p></li>
<li id="id19578"><p>Lisha K. Parambil and Debasis Sarkar. Probing the bioethanol production potential of &lt;i&gt;scheffersomyces&lt;/i&gt; (&lt;i&gt;pichia&lt;/i&gt;) &lt;i&gt;stipitis&lt;/i&gt; using validated genome-scale model. <em>BIOTECHNOLOGY LETTERS</em>, 36(12):2443–2451, DEC 2014. <a class="reference external" href="https://doi.org/10.1007/s10529-014-1629-8">[DOI]</a>.</p></li>
<li id="id19579"><p>Rienk A. Rienksma, Maria Suarez-Diez, Lucie Spina, Peter J. Schaap, and Vitor A. P. Martins dos Santos. Systems-level modeling of mycobacterial metabolism for the identification of new (multi-)drug targets. <em>SEMINARS IN IMMUNOLOGY</em>, 26(6):610–622, DEC 2014. <a class="reference external" href="https://doi.org/10.1016/j.smim.2014.09.013">[DOI]</a>.</p></li>
<li id="id19580"><p>Erin R. Shellman, Yu Chen, Xiaoxia Lin, Charles F. Burant, and Santiago Schnell. Metabolic network motifs can provide novel insights into evolution: the evolutionary origin of eukaryotic organelles as a case study. <em>COMPUTATIONAL BIOLOGY AND CHEMISTRY</em>, 53(B):242–250, DEC 2014. <a class="reference external" href="https://doi.org/10.1016/j.compbiolchem.2014.09.006">[DOI]</a>.</p></li>
<li id="id19581"><p>Keesha E. Erickson, Ryan T. Gill, and Anushree Chatterjee. Constrictor: constraint modification provides insight into design of biochemical networks. <em>PLOS ONE</em>, NOV 25 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0113820">[DOI]</a>.</p></li>
<li id="id19582"><p>Edward T. Chouchani, Victoria R. Pell, Edoardo Gaude, Dunja Aksentijevic, Stephanie Y. Sundier, Ellen L. Robb, Angela Logan, Sergiy M. Nadtochiy, Emily N. J. Ord, Anthony C. Smith, Filmon Eyassu, Rachel Shirley, Chou-Hui Hu, Anna J. Dare, Andrew M. James, Sebastian Rogatti, Richard C. Hartley, Simon Eaton, Ana S. H. Costa, Paul S. Brookes, Sean M. Davidson, Michael R. Duchen, Kourosh Saeb-Parsy, Michael J. Shattock, Alan J. Robinson, Lorraine M. Work, Christian Frezza, Thomas Krieg, and Michael P. Murphy. Ischaemic accumulation of succinate controls reperfusion injury through mitochondrial ros. <em>NATURE</em>, 515(7527):431+, NOV 20 2014. <a class="reference external" href="https://doi.org/10.1038/nature13909">[DOI]</a>.</p></li>
<li id="id19583"><p>Zhiqiang Sun, Hailin Meng, Jing Li, Jianfeng Wang, Qian Li, Yong Wang, and Yansheng Zhang. Identification of novel knockout targets for improving terpenoids biosynthesis in &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;. <em>PLOS ONE</em>, NOV 11 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0112615">[DOI]</a>.</p></li>
<li id="id19584"><p>Kevin J. Tsai and Chuan-Hsiung Chang. Diagnostics for stochastic genome-scale modeling via model slicing and debugging. <em>PLOS ONE</em>, NOV 4 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0110380">[DOI]</a>.</p></li>
<li id="id19585"><p>Ali Salehzadeh-Yazdi, Yazdan Asgari, Ali Akbar Saboury, and Ali Masoudi-Nejad. Computational analysis of reciprocal association of metabolism and epigenetics in the budding yeast: a genome-scale metabolic model (gsmm) approach. <em>PLOS ONE</em>, NOV 3 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0111686">[DOI]</a>.</p></li>
<li id="id19586"><p>Maria Victoria Aguilar-Pontes, Ronald P. de Vries, and Miaomiao Zhou. (post-)genomics approaches in fungal research. <em>BRIEFINGS IN FUNCTIONAL GENOMICS</em>, 13(6, SI):424–439, NOV 2014. <a class="reference external" href="https://doi.org/10.1093/bfgp/elu028">[DOI]</a>.</p></li>
<li id="id19587"><p>Longfei Mao and Wynand S. Verwoerd. Computational comparison of mediated current generation capacity of &lt;i&gt;chlamydomonas reinhardtii&lt;/i&gt; in photosynthetic and respiratory growth modes. <em>JOURNAL OF BIOSCIENCE AND BIOENGINEERING</em>, 118(5):565–574, NOV 2014. <a class="reference external" href="https://doi.org/10.1016/j.jbiosc.2014.04.021">[DOI]</a>.</p></li>
<li id="id19588"><p>Ryan S. Senger, Jiun Y. Yen, and Stephen S. Fong. A review of genome-scale metabolic flux modeling of anaerobiosis in biotechnology. <em>CURRENT OPINION IN CHEMICAL ENGINEERING</em>, 6:33–42, NOV 2014. <a class="reference external" href="https://doi.org/10.1016/j.coche.2014.08.003">[DOI]</a>.</p></li>
<li id="id19589"><p>Pei-Jung Wu, Wu-Hsiung Wu, Tzu-Chi Chen, Kuan-Ting Lin, Jin-Mei Lai, Chi-Ying F. Huang, and Feng-Sheng Wang. Reconstruction and analysis of a signal transduction network using hela cell protein-protein interaction data. <em>JOURNAL OF THE TAIWAN INSTITUTE OF CHEMICAL ENGINEERS</em>, 45(6):2835–2842, NOV 2014. <a class="reference external" href="https://doi.org/10.1016/j.jtice.2014.07.006">[DOI]</a>.</p></li>
<li id="id19590"><p>Mathieu Larocque, Thierry Chenard, and Rafael Najmanovich. A curated &lt;i&gt;c-difficile&lt;/i&gt; strain 630 metabolic network: prediction of essential targets and inhibitors. <em>BMC SYSTEMS BIOLOGY</em>, OCT 15 2014. <a class="reference external" href="https://doi.org/10.1186/s12918-014-0117-z">[DOI]</a>.</p></li>
<li id="id19591"><p>Gilles Vieira, Marc Carnicer, Jean-Charles Portais, and Stephanie Heux. Findpath: a matlab solution for &lt;i&gt;in silico&lt;/i&gt; design of synthetic metabolic pathways. <em>BIOINFORMATICS</em>, 30(20):2986–2988, OCT 15 2014. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btu422">[DOI]</a>.</p></li>
<li id="id19592"><p>Kambiz Baghalian, Mohammad-Reza Hajirezaei, and Falk Schreiber. Plant metabolic modeling: achieving new insight into metabolism and metabolic engineering. <em>PLANT CELL</em>, 26(10):3847–3866, OCT 2014. <a class="reference external" href="https://doi.org/10.1105/tpc.114.130328">[DOI]</a>.</p></li>
<li id="id19593"><p>Matthew N. Benedict, Michael B. Mundy, Christopher S. Henry, Nicholas Chia, and Nathan D. Price. Likelihood-based gene annotations for gap filling and quality assessment in genome-scale metabolic models. <em>PLOS COMPUTATIONAL BIOLOGY</em>, OCT 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003882">[DOI]</a>.</p></li>
<li id="id19594"><p>John Blazeck, Jarrett Miller, Anny Pan, Jon Gengler, Clinton Holden, Mariam Jamoussi, and Hal S. Alper. Metabolic engineering of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; for itaconic acid production. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 98(19):8155–8164, OCT 2014. <a class="reference external" href="https://doi.org/10.1007/s00253-014-5895-0">[DOI]</a>.</p></li>
<li id="id19595"><p>Leonid Chindelevitch, Jason Trigg, Aviv Regev, and Bonnie Berger. An exact arithmetic toolbox for a consistent and reproducible structural analysis of metabolic network models. <em>NATURE COMMUNICATIONS</em>, OCT 2014. <a class="reference external" href="https://doi.org/10.1038/ncomms5893">[DOI]</a>.</p></li>
<li id="id19596"><p>Michael A. Henson and Timothy J. Hanly. Dynamic flux balance analysis for synthetic microbial communities. <em>IET SYSTEMS BIOLOGY</em>, 8(5):214–229, OCT 2014. <a class="reference external" href="https://doi.org/10.1049/iet-syb.2013.0021">[DOI]</a>.</p></li>
<li id="id19597"><p>Nadeera Jayasinghe, Ashley Franks, Kelly P. Nevin, and Radhakrishnan Mahadevan. Metabolic modeling of spatial heterogeneity of biofilms in microbial fuel cells reveals substrate limitations in electrical current generation. <em>BIOTECHNOLOGY JOURNAL</em>, 9(10):1350–1361, OCT 2014. <a class="reference external" href="https://doi.org/10.1002/biot.201400068">[DOI]</a>.</p></li>
<li id="id19598"><p>Joonhoon Kim and Jennifer L. Reed. Refining metabolic models and accounting for regulatory effects. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 29:34–38, OCT 2014. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2014.02.009">[DOI]</a>.</p></li>
<li id="id19599"><p>Semidan Robaina Estevez and Zoran Nikoloski. Generalized framework for context-specific metabolic model extraction methods. <em>FRONTIERS IN PLANT SCIENCE</em>, SEP 19 2014. <a class="reference external" href="https://doi.org/10.3389/fpls.2014.00491">[DOI]</a>.</p></li>
<li id="id19600"><p>Md Bahadur Badsha, Ryo Tsuboi, and Hiroyuki Kurata. Complementary elementary modes for fast and efficient analysis of metabolic networks. <em>BIOCHEMICAL ENGINEERING JOURNAL</em>, 90:121–130, SEP 15 2014. <a class="reference external" href="https://doi.org/10.1016/j.bej.2014.05.022">[DOI]</a>.</p></li>
<li id="id19601"><p>Marianna Taffi, Nicola Paoletti, Claudio Angione, Sandra Pucciarelli, Mauro Marini, and Pietro Lio. Bioremediation in marine ecosystems: a computational study combining ecological modeling and flux balance analysis. <em>FRONTIERS IN GENETICS</em>, SEP 12 2014. <a class="reference external" href="https://doi.org/10.3389/fgene.2014.00319">[DOI]</a>.</p></li>
<li id="id19602"><p>Miguel A. Campodonico, Barbara A. Andrews, Juan A. Asenjo, Bernhard O. Palsson, and Adam M. Feist. Generation of an atlas for commodity chemical production in &lt;i&gt;escherichia coli&lt;/i&gt; and a novel pathway prediction algorithm, gem-path. <em>METABOLIC ENGINEERING</em>, 25:140–158, SEP 2014. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2014.07.009">[DOI]</a>.</p></li>
<li id="id19603"><p>Javier Cardenas and Nancy A. Da Silva. Metabolic engineering of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; for the production of triacetic acid lactone. <em>METABOLIC ENGINEERING</em>, 25:194–203, SEP 2014. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2014.07.008">[DOI]</a>.</p></li>
<li id="id19604"><p>Almut Heinken, M. Tanweer Khan, Giuseppe Paglia, Dmitry A. Rodionov, Hermie J. M. Harmsen, and Ines Thiele. Functional metabolic map of &lt;i&gt;faecalibacterium prausnitzii&lt;/i&gt;, a beneficial human gut microbe. <em>JOURNAL OF BACTERIOLOGY</em>, 196(18):3289–3302, SEP 2014. <a class="reference external" href="https://doi.org/10.1128/JB.01780-14">[DOI]</a>.</p></li>
<li id="id19605"><p>Minsuk Kim, Jeong Sang Yi, Joonwon Kim, Ji-Nu Kim, Min Woo Kim, and Byung-Gee Kim. Reconstruction of a high-quality metabolic model enables the identification of gene overexpression targets for enhanced antibiotic production in &lt;i&gt;streptomyces coelicolor&lt;/i&gt; a3(2). <em>BIOTECHNOLOGY JOURNAL</em>, 9(9, SI):1185–1194, SEP 2014. <a class="reference external" href="https://doi.org/10.1002/biot.201300539">[DOI]</a>.</p></li>
<li id="id19606"><p>K. P. Lisha and Debasis Sarkar. &lt;i&gt;in silico&lt;/i&gt; analysis of bioethanol production from glucose/xylose mixtures during fed-batch fermentation of co-culture and mono-culture systems. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 19(5):879–891, SEP 2014. <a class="reference external" href="https://doi.org/10.1007/s12257-014-0320-1">[DOI]</a>.</p></li>
<li id="id19607"><p>Hojung Nam, Miguel Campodonico, Aarash Bordbar, Daniel R. Hyduke, Sangwoo Kim, Daniel C. Zielinski, and Bernhard O. Palsson. A systems approach to predict oncometabolites via context-specific genome-scale metabolic networks. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003837">[DOI]</a>.</p></li>
<li id="id19608"><p>Octavio Perez-Garcia, Silas G. Villas-Boas, Simon Swift, Kartik Chandran, and Naresh Singhal. Clarifying the regulation of no/n&lt;sub&gt;2&lt;/sub&gt;o production in &lt;i&gt;nitrosomonas europaea&lt;/i&gt; during anoxic-oxic transition via flux balance analysis of a metabolic network model. <em>WATER RESEARCH</em>, 60:267–277, SEP 1 2014. <a class="reference external" href="https://doi.org/10.1016/j.watres.2014.04.049">[DOI]</a>.</p></li>
<li id="id19609"><p>Benjamin J. Sanchez, Jose R. Perez-Correa, and Eduardo Agosin. Construction of robust dynamic genome-scale metabolic model structures of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; through iterative re-parameterization. <em>METABOLIC ENGINEERING</em>, 25:159–173, SEP 2014. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2014.07.004">[DOI]</a>.</p></li>
<li id="id19610"><p>Anders G. Sandstrom, Henrik Almqvist, Diogo Portugal-Nunes, Dario Neves, Gunnar Liden, and Marie F. Gorwa-Grauslund. &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;: a potential host for carboxylic acid production from lignocellulosic feedstock? <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 98(17):7299–7318, SEP 2014. <a class="reference external" href="https://doi.org/10.1007/s00253-014-5866-5">[DOI]</a>.</p></li>
<li id="id19611"><p>Ines Thiele, Nikos Vlassis, and Ronan M. T. Fleming. Fastgapfill: efficient gap filling in metabolic networks. <em>BIOINFORMATICS</em>, 30(17):2529–2531, SEP 1 2014. 13th European Conference on Computational Biology (ECCB), Strasbourg, FRANCE, SEP 07-10, 2014. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btu321">[DOI]</a>.</p></li>
<li id="id19612"><p>Hong Yang, Elias W. Krumholz, Evan D. Brutinel, Nagendra P. Palani, Michael J. Sadowsky, Andrew M. Odlyzko, Jeffrey A. Gralnick, and Igor G. L. Libourel. Genome-scale metabolic network validation of &lt;i&gt;shewanella oneidensis&lt;/i&gt; using transposon insertion frequency analysis. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003848">[DOI]</a>.</p></li>
<li id="id19613"><p>Lake-Ee Quek, Stefanie Dietmair, Michael Hanscho, Veronica S. Martinez, Nicole Borth, and Lars K. Nielsen. Reducing recon 2 for steady-state flux analysis of hek cell culture. <em>JOURNAL OF BIOTECHNOLOGY</em>, 184:172–178, AUG 20 2014. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2014.05.021">[DOI]</a>.</p></li>
<li id="id19614"><p>Rafael S. Costa, Andre Verissimo, and Susana Vinga. &lt;i&gt;kimosys&lt;/i&gt;: a web-based repository of experimental data for kinetic models of biological systems. <em>BMC SYSTEMS BIOLOGY</em>, AUG 13 2014. <a class="reference external" href="https://doi.org/10.1186/s12918-014-0085-3">[DOI]</a>.</p></li>
<li id="id19615"><p>Chao Ye, Wei Zou, Nan Xu, and Liming Liu. Metabolic model reconstruction and analysis of an artificial microbial ecosystem for vitamin c production. <em>JOURNAL OF BIOTECHNOLOGY</em>, 182:61–67, JUL 20 2014. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2014.04.027">[DOI]</a>.</p></li>
<li id="id19616"><p>Amit Kumar, Thomas Harrelson, Nathan E. Lewis, Emily J. Gallagher, Derek LeRoith, Joseph Shiloach, and Michael J. Betenbaugh. Multi-tissue computational modeling analyzes pathophysiology of type 2 diabetes in mkr mice. <em>PLOS ONE</em>, JUL 16 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0102319">[DOI]</a>.</p></li>
<li id="id19617"><p>Andrea De Martino, Daniele De Martino, Roberto Mulet, and Andrea Pagnani. Identifying all moiety conservation laws in genome-scale metabolic networks. <em>PLOS ONE</em>, JUL 2 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0100750">[DOI]</a>.</p></li>
<li id="id19618"><p>Zsolt Bodor, Andrea (Iuhasz) Fazakas, Erika Kovacs, Szabolcs Lanyi, and Beata Albert. Systems biology and metabolic engineering for obtaining &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; mutants capable to produce succinate from renewable resources. <em>ROMANIAN BIOTECHNOLOGICAL LETTERS</em>, 19(4):9625–9636, JUL-AUG 2014.</p></li>
<li id="id19619"><p>Aarash Bordbar, Harish Nagarajan, Nathan E. Lewis, Haythem Latif, Ali Ebrahim, Stephen Federowicz, Jan Schellenberger, and Bernhard O. Palsson. Minimal metabolic pathway structure is consistent with associated biomolecular interactions. <em>MOLECULAR SYSTEMS BIOLOGY</em>, JUL 2014. <a class="reference external" href="https://doi.org/10.15252/msb.20145243">[DOI]</a>.</p></li>
<li id="id19620"><p>Javier Carrera, Raissa Estrela, Jing Luo, Navneet Rai, Athanasios Tsoukalas, and Ilias Tagkopoulos. An integrative, multi-scale, genome-wide model reveals the phenotypic landscape of &lt;i&gt;escherichia coli&lt;/i&gt;. <em>MOLECULAR SYSTEMS BIOLOGY</em>, JUL 2014. <a class="reference external" href="https://doi.org/10.15252/msb.20145108">[DOI]</a>.</p></li>
<li id="id19621"><p>Omer Eilam, Raphy Zarecki, Matthew Oberhardt, Luke K. Ursell, Martin Kupiec, Rob Knight, Uri Gophna, and Eytan Ruppin. Glycan degradation (glyder) analysis predicts mammalian gut microbiota abundance and host diet-specific adaptations. <em>MBIO</em>, JUL-AUG 2014. <a class="reference external" href="https://doi.org/10.1128/mBio.01526-14">[DOI]</a>.</p></li>
<li id="id19622"><p>Lu Shi Jing, Farah Fathiah Muzaffar Shah, Mohd Saberi Mohamad, Nur Laily Hamran, Abdul Hakim Mohamed Salleh, Safaai Deris, and Hany Alashwal. Database and tools for metabolic network analysis. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 19(4):568–585, JUL 2014. <a class="reference external" href="https://doi.org/10.1007/s12257-014-0172-8">[DOI]</a>.</p></li>
<li id="id19623"><p>Zachary A. King and Adam M. Feist. Optimal cofactor swapping can increase the theoretical yield for chemical production in &lt;i&gt;escherichia coli&lt;/i&gt; and &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 24:117–128, JUL 2014. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2014.05.009">[DOI]</a>.</p></li>
<li id="id19624"><p>Stephen Sarria, Betty Wong, Hector Garcia Martin, Jay D. Keasling, and Pamela Peralta-Yahya. Microbial synthesis of pinene. <em>ACS SYNTHETIC BIOLOGY</em>, 3(7):466–475, JUL 2014. <a class="reference external" href="https://doi.org/10.1021/sb4001382">[DOI]</a>.</p></li>
<li id="id19625"><p>Samuel M. D. Seaver, Svetlana Gerdes, Oceane Frelin, Claudia Lerma-Ortiz, Louis M. T. Bradbury, Remi Zallot, Ghulam Hasnain, Thomas D. Niehaus, Basma El Yacoubi, Shiran Pasternak, Robert Olson, Gordon Pusch, Ross Overbeek, Rick Stevens, Valerie de Crecy-Lagard, Doreen Ware, Andrew D. Hanson, and Christopher S. Henry. High-throughput comparison, functional annotation, and metabolic modeling of plant genomes using the plantseed resource. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 111(26):9645–9650, JUL 1 2014. <a class="reference external" href="https://doi.org/10.1073/pnas.1401329111">[DOI]</a>.</p></li>
<li id="id19626"><p>Michael Weiner, Julia Troendle, Christoph Albermann, Georg A. Sprenger, and Dirk Weuster-Botz. Improvement of constraint-based flux estimation during l-phenylalanine production with escherichia coli using targeted knock-out mutants. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 111(7):1406–1416, JUL 2014. <a class="reference external" href="https://doi.org/10.1002/bit.25195">[DOI]</a>.</p></li>
<li id="id19627"><p>Daniel S. Weaver, Ingrid M. Keseler, Amanda Mackie, Ian T. Paulsen, and Peter D. Karp. A genome-scale metabolic flux model of &lt;i&gt;escherichia coli&lt;/i&gt; k-12 derived from the ecocyc database. <em>BMC SYSTEMS BIOLOGY</em>, JUN 30 2014. <a class="reference external" href="https://doi.org/10.1186/1752-0509-8-79">[DOI]</a>.</p></li>
<li id="id19628"><p>Mario Latendresse. Efficiently gap-filling reaction networks. <em>BMC BIOINFORMATICS</em>, JUN 28 2014. <a class="reference external" href="https://doi.org/10.1186/1471-2105-15-225">[DOI]</a>.</p></li>
<li id="id19629"><p>Neha Rohatgi, Tine Kragh Nielsen, Sara Petersen Bjorn, Ivar Axelsson, Giuseppe Paglia, Bjorn Gunnar Voldborg, Bernhard O. Palsson, and Ottar Rolfsson. Biochemical characterization of human gluconokinase and the proposed metabolic impact of gluconic acid as determined by constraint based metabolic network analysis. <em>PLOS ONE</em>, JUN 4 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0098760">[DOI]</a>.</p></li>
<li id="id19630"><p>Zsolt Bodor, Andrea Fazakas (Iuhasz), Erika Kovacs, Szabolcs Lanyi, and Beata Abraham. Biotechnological production of succinic acid from glycerol; the role of co-substrates. <em>STUDIA UNIVERSITATIS BABES-BOLYAI CHEMIA</em>, 59(2):33–50, JUN 2014.</p></li>
<li id="id19631"><p>Oscar Dias, Rui Pereira, Andreas K. Gombert, Eugenio C. Ferreira, and Isabel Rocha. Iod907, the first genome-scale metabolic model for the milk yeast &lt;i&gt;kluyveromyces lactis&lt;/i&gt;. <em>BIOTECHNOLOGY JOURNAL</em>, 9(6, SI):776–790, JUN 2014. <a class="reference external" href="https://doi.org/10.1002/biot.201300242">[DOI]</a>.</p></li>
<li id="id19632"><p>Robert Nilsson, Fredric Bauer, Sennai Mesfun, Christian Hulteberg, Joakim Lundgren, Sune Wannstrom, Ulrika Rova, and Kris Arvid Berglund. Techno-economics of carbon preserving butanol production using a combined fermentative and catalytic approach. <em>BIORESOURCE TECHNOLOGY</em>, 161:263–269, JUN 2014. <a class="reference external" href="https://doi.org/10.1016/j.biortech.2014.03.055">[DOI]</a>.</p></li>
<li id="id19633"><p>J. M. Gonzalez-Martinez, A. Folch-Fortuny, F. Llaneras, M. Tortajada, J. Pico, and A. Ferrer. Metabolic flux understanding of &lt;i&gt;pichia pastoris&lt;/i&gt; grown on heterogenous culture media. <em>CHEMOMETRICS AND INTELLIGENT LABORATORY SYSTEMS</em>, 134:89–99, MAY 15 2014. <a class="reference external" href="https://doi.org/10.1016/j.chemolab.2014.02.003">[DOI]</a>.</p></li>
<li id="id19634"><p>Maurice H. T. Ling and Chueh Loo Poh. A predictor for predicting &lt;i&gt;escherichia coli&lt;/i&gt; transcriptome and the effects of gene perturbations. <em>BMC BIOINFORMATICS</em>, MAY 13 2014. <a class="reference external" href="https://doi.org/10.1186/1471-2105-15-140">[DOI]</a>.</p></li>
<li id="id19635"><p>Irina Borodina and Jens Nielsen. Advances in metabolic engineering of yeast saccharomyces cerevisiae for production of chemicals. <em>BIOTECHNOLOGY JOURNAL</em>, 9(5, SI):609–620, MAY 2014. <a class="reference external" href="https://doi.org/10.1002/biot.201300445">[DOI]</a>.</p></li>
<li id="id19636"><p>Victor Chubukov, Luca Gerosa, Karl Kochanowski, and Uwe Sauer. Coordination of microbial metabolism. <em>NATURE REVIEWS MICROBIOLOGY</em>, 12(5):327–340, MAY 2014. <a class="reference external" href="https://doi.org/10.1038/nrmicro3238">[DOI]</a>.</p></li>
<li id="id19637"><p>Angela Cintolesi, James M. Clomburg, and Ramon Gonzalez. &lt;i&gt;in silico&lt;/i&gt; assessment of the metabolic capabilities of an engineered functional reversal of the β-oxidation cycle for the synthesis of longer-chain (c ≥ 4) products. <em>METABOLIC ENGINEERING</em>, 23:100–115, MAY 2014. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2014.02.011">[DOI]</a>.</p></li>
<li id="id19638"><p>Paul A. Jensen and Jason A. Papin. Metdraw: automated visualization of genome-scale metabolic network reconstructions and high-throughput data. <em>BIOINFORMATICS</em>, 30(9):1327–1328, MAY 1 2014. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btt758">[DOI]</a>.</p></li>
<li id="id19639"><p>Na-Rae Lee, Meiyappan Lakshmanan, Shilpi Aggarwal, Ji-Won Song, Iftekhar A. Karimi, Dong-Yup Lee, and Jin-Byung Park. Genome-scale metabolic network reconstruction and &lt;i&gt;in silico flux&lt;/i&gt; analysis of the thermophilic bacterium &lt;i&gt;thermus thermophilus&lt;/i&gt; hb27. <em>MICROBIAL CELL FACTORIES</em>, APR 28 2014. <a class="reference external" href="https://doi.org/10.1186/1475-2859-13-61">[DOI]</a>.</p></li>
<li id="id19640"><p>Ming Wu and Christina Chan. Prediction of therapeutic microrna based on the human metabolic network. <em>BIOINFORMATICS</em>, 30(8):1163–1171, APR 15 2014. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btt751">[DOI]</a>.</p></li>
<li id="id19641"><p>Shubo Li, Xiang Gao, Nan Xu, Liming Liu, and Jian Chen. Enhancement of acetoin production in &lt;i&gt;candida glabrata&lt;/i&gt; by &lt;i&gt;in silico&lt;/i&gt;-aided metabolic engineering. <em>MICROBIAL CELL FACTORIES</em>, APR 13 2014. <a class="reference external" href="https://doi.org/10.1186/1475-2859-13-55">[DOI]</a>.</p></li>
<li id="id19642"><p>Ibrahim E. El-Semman, Fredrik H. Karlsson, Saeed Shoaie, Intawat Nookaew, Taysir H. Soliman, and Jens Nielsen. Genome-scale metabolic reconstructions of &lt;i&gt;bifidobacterium adolescentis&lt;/i&gt; l2-32 and &lt;i&gt;faecalibacterium prausnitzii&lt;/i&gt; a2-165 and their interaction. <em>BMC SYSTEMS BIOLOGY</em>, APR 3 2014. <a class="reference external" href="https://doi.org/10.1186/1752-0509-8-41">[DOI]</a>.</p></li>
<li id="id19643"><p>Stephen Federowicz, Donghyuk Kim, Ali Ebrahim, Joshua Lerman, Harish Nagarajan, Byung-kwan Cho, Karsten Zengler, and Bernhard Palsson. Determining the control circuitry of redox metabolism at the genome-scale. <em>PLOS GENETICS</em>, APR 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pgen.1004264">[DOI]</a>.</p></li>
<li id="id19644"><p>Adam M. Feist, Harish Nagarajan, Amelia-Elena Rotaru, Pier-Luc Tremblay, Tian Zhang, Kelly P. Nevin, Derek R. Lovley, and Karsten Zengler. Constraint-based modeling of carbon fixation and the energetics of electron transfer in &lt;i&gt;geobacter&lt;/i&gt; &lt;i&gt;metallireducens&lt;/i&gt;. <em>PLOS COMPUTATIONAL BIOLOGY</em>, APR 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003575">[DOI]</a>.</p></li>
<li id="id19645"><p>Bjoern H. Junker. Flux analysis in plant metabolic networks: increasing throughput and coverage. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 26:183–188, APR 2014. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2014.01.016">[DOI]</a>.</p></li>
<li id="id19646"><p>K. P. Lisha and Debasis Sarkar. Dynamic flux balance analysis of batch fermentation: effect of genetic manipulations on ethanol production. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 37(4):617–627, APR 2014. <a class="reference external" href="https://doi.org/10.1007/s00449-013-1027-y">[DOI]</a>.</p></li>
<li id="id19647"><p>Daniel Machado and Markus Herrgard. Systematic evaluation of methods for integration of transcriptomic data into constraint-based models of metabolism. <em>PLOS COMPUTATIONAL BIOLOGY</em>, APR 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003580">[DOI]</a>.</p></li>
<li id="id19648"><p>John T. Sauls and Joerg M. Buescher. Assimilating genome-scale metabolic reconstructions with modelborgifier. <em>BIOINFORMATICS</em>, 30(7):1036–1038, APR 1 2014. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btt747">[DOI]</a>.</p></li>
<li id="id19649"><p>Xiaoyang Wang, Chuanbo Zhang, Meiling Wang, and Wenyu Lu. Genome-scale metabolic network reconstruction of &lt;i&gt;saccharopolyspora spinosa&lt;/i&gt; for spinosad production improvement. <em>MICROBIAL CELL FACTORIES</em>, MAR 15 2014. <a class="reference external" href="https://doi.org/10.1186/1475-2859-13-41">[DOI]</a>.</p></li>
<li id="id19650"><p>Swagatika Sahoo, Maike K. Aurich, Jon J. Jonsson, and Ines Thiele. Membrane transporters in a human genonne-scale metabolic knowledgebase and their implications for disease. <em>FRONTIERS IN PHYSIOLOGY</em>, MAR 11 2014. <a class="reference external" href="https://doi.org/10.3389/fphys.2014.00091">[DOI]</a>.</p></li>
<li id="id19651"><p>Shubo Li, Nan Xu, Liming Liu, and Jian Chen. Engineering of carboligase activity reaction in &lt;i&gt;candida glabrata&lt;/i&gt; for acetoin production. <em>METABOLIC ENGINEERING</em>, 22:32–39, MAR 2014. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2013.12.005">[DOI]</a>.</p></li>
<li id="id19652"><p>Timo R. Maarleveld, Joost Boele, Frank J. Bruggeman, and Bas Teusink. A data integration and visualization resource for the metabolic network of &lt;i&gt;synechocystis&lt;/i&gt; sp pcc 6803. <em>PLANT PHYSIOLOGY</em>, 164(3):1111–1121, MAR 2014. <a class="reference external" href="https://doi.org/10.1104/pp.113.224394">[DOI]</a>.</p></li>
<li id="id19653"><p>Xinsen Wu, Xiaoyang Wang, and Wenyu Lu. Genome-scale reconstruction of a metabolic network for &lt;i&gt;gluconobacter oxydans&lt;/i&gt; 621h. <em>BIOSYSTEMS</em>, 117:10–14, MAR 2014. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2014.01.001">[DOI]</a>.</p></li>
<li id="id19654"><p>Anna Kubacka, Maria Suarez Diez, David Rojo, Rafael Bargiela, Sergio Ciordia, Ines Zapico, Juan P. Albar, Coral Barbas, Vitor A. P. Martins dos Santos, Marcos Fernandez-Garcia, and Manuel Ferrer. Understanding the antimicrobial mechanism of tio&lt;sub&gt;2&lt;/sub&gt;-based nanocomposite films in a pathogenic bacterium. <em>SCIENTIFIC REPORTS</em>, FEB 19 2014. <a class="reference external" href="https://doi.org/10.1038/srep04134">[DOI]</a>.</p></li>
<li id="id19655"><p>Longfei Mao and Wynand S. Verwoerd. Orca: a cobra toolbox extension for model-driven discovery and analysis. <em>BIOINFORMATICS</em>, 30(4):584–585, FEB 15 2014. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btt723">[DOI]</a>.</p></li>
<li id="id19656"><p>Erno Lindfors, Paula Jouhten, Merja Oja, Eija Rintala, Matej Oresic, and Merja Penttila. Integration of transcription and flux data reveals molecular paths associated with differences in oxygen-dependent phenotypes of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, FEB 14 2014. <a class="reference external" href="https://doi.org/10.1186/1752-0509-8-16">[DOI]</a>.</p></li>
<li id="id19657"><p>Uldis Kalnenieks, Agris Pentjuss, Reinis Rutkis, Egils Stalidzans, and David A. Fell. Modeling of &lt;i&gt;zymomonas mobilis&lt;/i&gt; central metabolism for novel metabolic engineering strategies. <em>FRONTIERS IN MICROBIOLOGY</em>, FEB 5 2014. <a class="reference external" href="https://doi.org/10.3389/fmicb.2014.00042">[DOI]</a>.</p></li>
<li id="id19658"><p>Achuthanunni Chokkathukalam, Dong-Hyun Kim, Michael P. Barrett, Rainer Breitling, and Darren J. Creek. Stable isotope- labeling studies in metabolomics: new insights into structure and dynamics of metabolic networks. <em>BIOANALYSIS</em>, 6(4):511–524, FEB 2014. <a class="reference external" href="https://doi.org/10.4155/bio.13.348">[DOI]</a>.</p></li>
<li id="id19659"><p>Simon Unthan, Alexander Gruenberger, Jan van Ooyen, Jochem Gaetgens, Johanna Heinrich, Nicole Paczia, Wolfgang Wiechert, Dietrich Kohlheyer, and Stephan Noack. Beyond growth rate 0.6: what drives &lt;i&gt;corynebacterium glutamicum&lt;/i&gt; to higher growth rates in defined medium. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 111(2):359–371, FEB 2014. <a class="reference external" href="https://doi.org/10.1002/bit.25103">[DOI]</a>.</p></li>
<li id="id19660"><p>Xu Zi-Xiang, Zheng Ping, and Sun Ji-Bin. Reconstruction of whole cell network and design of cell factory. <em>PROGRESS IN BIOCHEMISTRY AND BIOPHYSICS</em>, 41(2):105–114, FEB 2014. <a class="reference external" href="https://doi.org/10.3724/SP.J.1206.2012.00530">[DOI]</a>.</p></li>
<li id="id19661"><p>Alejandro Acevedo, German Aroca, and Raul Conejeros. Genome-scale nad(h/&lt;sup&gt;+&lt;/sup&gt;) availability patterns as a differentiating feature between &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; and &lt;i&gt;scheffersomyces stipitis&lt;/i&gt; in relation to fermentative metabolism. <em>PLOS ONE</em>, JAN 29 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0087494">[DOI]</a>.</p></li>
<li id="id19662"><p>Matthew N. Benedict, James R. Henriksen, William W. Metcalf, Rachel J. Whitaker, and Nathan D. Price. Itep: an integrated toolkit for exploration of microbial pan-genomes. <em>BMC GENOMICS</em>, JAN 3 2014. <a class="reference external" href="https://doi.org/10.1186/1471-2164-15-8">[DOI]</a>.</p></li>
<li id="id19663"><p>Parizad Babaei, Tahereh Ghasemi-Kahrizsangi, and Sayed-Amir Marashi. Modeling the differences in biochemical capabilities of &lt;i&gt;pseudomonas&lt;/i&gt; species by flux balance analysis: how good are genome-scale metabolic networks at predicting the differences? <em>SCIENTIFIC WORLD JOURNAL</em>, 2014. <a class="reference external" href="https://doi.org/10.1155/2014/416289">[DOI]</a>.</p></li>
<li id="id19664"><p>Jennifer A. Bartell, Phillip Yen, John J. Varga, Joanna B. Goldberg, and Jason A. Papin. Comparative metabolic systems analysis of pathogenic &lt;i&gt;burkholderia&lt;/i&gt;. <em>JOURNAL OF BACTERIOLOGY</em>, 196(2):210–226, JAN 2014. <a class="reference external" href="https://doi.org/10.1128/JB.00997-13">[DOI]</a>.</p></li>
<li id="id19665"><p>Thomas Dandekar, Astrid Fieselmann, Saman Majeed, and Zeeshan Ahmed. Software applications toward quantitative metabolic flux analysis and modeling. <em>BRIEFINGS IN BIOINFORMATICS</em>, 15(1):91–107, JAN 2014. <a class="reference external" href="https://doi.org/10.1093/bib/bbs065">[DOI]</a>.</p></li>
<li id="id19666"><p>Igor Marin de Mas, Esther Aguilar, Anusha Jayaraman, Ibrahim H. Polat, Alfonso Martin-Bernabe, Rohit Bharat, Carles Foguet, Enric Mila, Balazs Papp, Josep J. Centelles, and Marta Cascante. Cancer cell metabolism as new targets for novel designed therapies. <em>FUTURE MEDICINAL CHEMISTRY</em>, 6(16):1791–1810, 2014. <a class="reference external" href="https://doi.org/10.4155/FMC.14.119">[DOI]</a>.</p></li>
<li id="id19667"><p>Xin Fang, Jaques Reifman, and Anders Wallqvist. Modeling metabolism and stage-specific growth of &lt;i&gt;plasmodium falciparum&lt;/i&gt; hb3 during the intraerythrocytic developmental cycle. <em>MOLECULAR BIOSYSTEMS</em>, 10(10):2526–2537, 2014. <a class="reference external" href="https://doi.org/10.1039/c4mb00115j">[DOI]</a>.</p></li>
<li id="id19668"><p>Yue-Dong Gao, Yuqi Zhao, and Jingfei Huang. Metabolic modeling of common &lt;i&gt;escherichia coli&lt;/i&gt; strains in human gut microbiome. <em>BIOMED RESEARCH INTERNATIONAL</em>, 2014. <a class="reference external" href="https://doi.org/10.1155/2014/694967">[DOI]</a>.</p></li>
<li id="id19669"><p>Alexander Golberg, Edward Vitkin, Gregory Linshiz, Sabaa Ahmad Khan, Nathan J. Hillson, Zohar Yakhini, and Martin L. Yarmush. Proposed design of distributed macroalgal biorefineries: thermodynamics, bioconversion technology, and sustainability implications for developing economies. <em>BIOFUELS BIOPRODUCTS &amp; BIOREFINING-BIOFPR</em>, 8(1):67–82, JAN 2014. <a class="reference external" href="https://doi.org/10.1002/bbb.1438">[DOI]</a>.</p></li>
<li id="id19670"><p>Mahdieh Hadi and Sayed-Amir Marashi. Reconstruction of a generic metabolic network model of cancer cells. <em>MOLECULAR BIOSYSTEMS</em>, 10(11):3014–3021, 2014. <a class="reference external" href="https://doi.org/10.1039/c4mb00300d">[DOI]</a>.</p></li>
<li id="id19671"><p>Joshua J. Hamilton and Jennifer L. Reed. Software platforms to facilitate reconstructing genome-scale metabolic networks. <em>ENVIRONMENTAL MICROBIOLOGY</em>, 16(1, SI):49–59, JAN 2014. <a class="reference external" href="https://doi.org/10.1111/1462-2920.12312">[DOI]</a>.</p></li>
<li id="id19672"><p>Chintan Jagdishchandra Joshi and Ashok Prasad. Epistatic interactions among metabolic genes depend upon environmental conditions. <em>MOLECULAR BIOSYSTEMS</em>, 10(10):2578–2589, 2014. <a class="reference external" href="https://doi.org/10.1039/c4mb00181h">[DOI]</a>.</p></li>
<li id="id19673"><p>Joseph Koussa, Amphun Chaiboonchoe, and Kourosh Salehi-Ashtiani. Computational approaches for microalgal biofuel optimization: a review. <em>BIOMED RESEARCH INTERNATIONAL</em>, 2014. <a class="reference external" href="https://doi.org/10.1155/2014/649453">[DOI]</a>.</p></li>
<li id="id19674"><p>Meiyappan Lakshmanan, Bijayalaxmi Mohanty, Sun-Hyung Lim, Sun-Hwa Ha, and Dong-Yup Lee. Metabolic and transcriptional regulatory mechanisms underlying the anoxic adaptation of rice coleoptile. <em>AOB PLANTS</em>, 2014. <a class="reference external" href="https://doi.org/10.1093/aobpla/plu026">[DOI]</a>.</p></li>
<li id="id19675"><p>Meiyappan Lakshmanan, Geoffrey Koh, Bevan K. S. Chung, and Dong-Yup Lee. Software applications for flux balance analysis. <em>BRIEFINGS IN BIOINFORMATICS</em>, 15(1):108–122, JAN 2014. <a class="reference external" href="https://doi.org/10.1093/bib/bbs069">[DOI]</a>.</p></li>
<li id="id19676"><p>Meng Liang, Andrew Damiani, Q. Peter He, and Jin Wang. Elucidating xylose metabolism of &lt;i&gt;scheffersomyces&lt;/i&gt; stipitis for lignocellulosic ethanol production. <em>ACS SUSTAINABLE CHEMISTRY &amp; ENGINEERING</em>, 2(1, SI):38–48, JAN 2014. <a class="reference external" href="https://doi.org/10.1021/sc400265g">[DOI]</a>.</p></li>
<li id="id19677"><p>Tige R. Rustad, Kyle J. Minch, Shuyi Ma, Jessica K. Winkler, Samuel Hobbs, Mark Hickey, William Brabant, Serdar Turkarslan, Nathan D. Price, Nitin S. Baliga, and David R. Sherman. Mapping and manipulating the &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; transcriptome using a transcription factor overexpression-derived regulatory network. <em>GENOME BIOLOGY</em>, 2014. <a class="reference external" href="https://doi.org/10.1186/s13059-014-0502-3">[DOI]</a>.</p></li>
<li id="id19678"><p>Sarah Schatschneider, Claudia Huber, Heiko Neuweger, Tony Francis Watt, Alfred Puehler, Wolfgang Eisenreich, Christoph Wittmann, Karsten Niehaus, and Frank-Joerg Vorhoelter. Metabolic flux pattern of glucose utilization by &lt;i&gt;xanthomonas campestris&lt;/i&gt; pv. campestris: prevalent role of the entner-doudoroff pathway and minor fluxes through the pentose phosphate pathway and glycolysis. <em>MOLECULAR BIOSYSTEMS</em>, 10(10):2663–2676, 2014. <a class="reference external" href="https://doi.org/10.1039/c4mb00198b">[DOI]</a>.</p></li>
<li id="id19679"><p>Benjamin VanderSluis, David C. Hess, Colin Pesyna, Elias W. Krumholz, Tahin Syed, Balazs Szappanos, Corey Nislow, Balazs Papp, Olga G. Troyanskaya, Chad L. Myers, and Amy A. Caudy. Broad metabolic sensitivity profiling of a prototrophic yeast deletion collection. <em>GENOME BIOLOGY</em>, 2014. <a class="reference external" href="https://doi.org/10.1186/gb-2014-15-4-r64">[DOI]</a>.</p></li>
<li id="id19680"><p>Nikos Vlassis, Maria Pires Pacheco, and Thomas Sauter. Fast reconstruction of compact context-specific metabolic network models. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JAN 2014. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003424">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-11" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-11" name="0-11" role="tabpanel" tabindex="0"><div class="docutils container" id="id20021">
<ol class="arabic simple" start="1">
<li id="id21501"><p>Zixiang Xu, Ping Zheng, Jibin Sun, and Yanhe Ma. Reacknock: identifying reaction deletion strategies for microbial strain optimization based on genome-scale metabolic network. <em>PLOS ONE</em>, DEC 11 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0072150">[DOI]</a>.</p></li>
<li id="id21502"><p>Soma Ghosh, Priyanka Baloni, Sumanta Mukherjee, Praveen Anand, and Nagasuma Chandra. A multi-level multi-scale approach to study essential genes in &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, DEC 5 2013. <a class="reference external" href="https://doi.org/10.1186/1752-0509-7-132">[DOI]</a>.</p></li>
<li id="id21503"><p>Sriram Chandrasekaran and Nathan D. Price. Metabolic constraint-based refinement of transcriptional regulatory networks. <em>PLOS COMPUTATIONAL BIOLOGY</em>, DEC 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003370">[DOI]</a>.</p></li>
<li id="id21504"><p>Douglas B. Kell. Finding novel pharmaceuticals in the systems biology era using multiple effective drug targets, phenotypic screening and knowledge of transporters: where drug discovery went wrong and how to fix it. <em>FEBS JOURNAL</em>, 280(23):5957–5980, DEC 2013. <a class="reference external" href="https://doi.org/10.1111/febs.12268">[DOI]</a>.</p></li>
<li id="id21505"><p>Shayan Tabe-Bordbar and Sayed-Amir Marashi. Finding elementary flux modes in metabolic networks based on flux balance analysis and flux coupling analysis: application to the analysis of &lt;i&gt;escherichia coli&lt;/i&gt; metabolism. <em>BIOTECHNOLOGY LETTERS</em>, 35(12):2039–2044, DEC 2013. <a class="reference external" href="https://doi.org/10.1007/s10529-013-1328-x">[DOI]</a>.</p></li>
<li id="id21506"><p>Giuseppe Facchetti and Claudio Altafini. Partial inhibition and bilevel optimization in flux balance analysis. <em>BMC BIOINFORMATICS</em>, NOV 29 2013. <a class="reference external" href="https://doi.org/10.1186/1471-2105-14-344">[DOI]</a>.</p></li>
<li id="id21507"><p>Harish Nagarajan, Merve Sahin, Juan Nogales, Haythem Latif, Derek R. Lovley, Ali Ebrahim, and Karsten Zengler. Characterizing acetogenic metabolism using a genome-scale metabolic reconstruction of &lt;i&gt;clostridium ljungdahlii&lt;/i&gt;. <em>MICROBIAL CELL FACTORIES</em>, NOV 25 2013. <a class="reference external" href="https://doi.org/10.1186/1475-2859-12-118">[DOI]</a>.</p></li>
<li id="id21508"><p>Mona Yousofshahi, Ehsan Ullah, Russell Stern, and Soha Hassoun. Mc&lt;sup&gt;3&lt;/sup&gt;: a steady-state model and constraint consistency checker for biochemical networks. <em>BMC SYSTEMS BIOLOGY</em>, NOV 21 2013. <a class="reference external" href="https://doi.org/10.1186/1752-0509-7-129">[DOI]</a>.</p></li>
<li id="id21509"><p>Nathan Crook and Hal S. Alper. Model-based design of synthetic, biological systems. <em>CHEMICAL ENGINEERING SCIENCE</em>, 103:2–11, NOV 15 2013. <a class="reference external" href="https://doi.org/10.1016/j.ces.2012.12.022">[DOI]</a>.</p></li>
<li id="id21510"><p>Gabriel Gelius-Dietrich, Abdelmoneim Amer Desouki, Claus Jonathan Fritzemeier, and Martin J. Lercher. Sybil - efficient constraint-based modelling in r. <em>BMC SYSTEMS BIOLOGY</em>, NOV 13 2013. <a class="reference external" href="https://doi.org/10.1186/1752-0509-7-125">[DOI]</a>.</p></li>
<li id="id21511"><p>Finja Buechel, Sandra Saliger, Andreas Draeger, Stephanie Hoffmann, Clemens Wrzodek, Andreas Zell, and Philipp J. Kahle. Parkinson's disease: dopaminergic nerve cell model is consistent with experimental finding of increased extracellular transport of α-synuclein. <em>BMC NEUROSCIENCE</em>, NOV 6 2013. <a class="reference external" href="https://doi.org/10.1186/1471-2202-14-136">[DOI]</a>.</p></li>
<li id="id21512"><p>Finja Buechel, Nicolas Rodriguez, Neil Swainston, Clemens Wrzodek, Tobias Czauderna, Roland Keller, Florian Mittag, Michael Schubert, Mihai Glont, Martin Golebiewski, Martijn van Iersel, Sarah Keating, Matthias Rall, Michael Wybrow, Henning Hermjakob, Michael Hucka, Douglas B. Kell, Wolfgang Mueller, Pedro Mendes, Andreas Zell, Claudine Chaouiya, Julio Saez-Rodriguez, Falk Schreiber, Camille Laibe, Andreas Draeger, and Nicolas Le Novere. Path2models: large-scale generation of computational models from biochemical pathway maps. <em>BMC SYSTEMS BIOLOGY</em>, NOV 1 2013. <a class="reference external" href="https://doi.org/10.1186/1752-0509-7-116">[DOI]</a>.</p></li>
<li id="id21513"><p>Tat-Ming Lo, Wei Suong Teo, Hua Ling, Binbin Chen, Aram Kang, and Matthew Wook Chang. Microbial engineering strategies to improve cell viability for biochemical production. <em>BIOTECHNOLOGY ADVANCES</em>, 31(6, SI):903–914, NOV 1 2013. <a class="reference external" href="https://doi.org/10.1016/j.biotechadv.2013.02.001">[DOI]</a>.</p></li>
<li id="id21514"><p>Ivan Lule, Pieter-Jan D'Huys, Lieve Van Mellaert, Jozef Anne, Kristel Bernaerts, and Jan Van Impe. Metabolic impact assessment for heterologous protein production in &lt;i&gt;streptomyces lividans&lt;/i&gt; based on genome-scale metabolic network modeling. <em>MATHEMATICAL BIOSCIENCES</em>, 246(1):113–121, NOV 2013. <a class="reference external" href="https://doi.org/10.1016/j.mbs.2013.08.006">[DOI]</a>.</p></li>
<li id="id21515"><p>Longfei Mao and Wynand S. Verwoerd. Exploration and comparison of inborn capacity of aerobic and anaerobic metabolisms of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; for microbial electrical current production. <em>BIOENGINEERED</em>, 4(6):420–430, NOV-DEC 2013. <a class="reference external" href="https://doi.org/10.4161/bioe.26222">[DOI]</a>.</p></li>
<li id="id21516"><p>Harish Nagarajan, Mallory Embree, Amelia-Elena Rotaru, Pravin M. Shrestha, Adam M. Feist, Bernhard O. Palsson, Derek R. Lovley, and Karsten Zengler. Characterization and modelling of interspecies electron transfer mechanisms and microbial community dynamics of a syntrophic association. <em>NATURE COMMUNICATIONS</em>, NOV 2013. <a class="reference external" href="https://doi.org/10.1038/ncomms3809">[DOI]</a>.</p></li>
<li id="id21517"><p>Carl Song, Melissa A. Chiasson, Nirvana Nursimulu, Stacy S. Hung, James Wasmuth, Michael E. Grigg, and John Parkinson. Metabolic reconstruction identifies strain-specific regulation of virulence in &lt;i&gt;toxoplasma gondii&lt;/i&gt;. <em>MOLECULAR SYSTEMS BIOLOGY</em>, NOV 2013. <a class="reference external" href="https://doi.org/10.1038/msb.2013.62">[DOI]</a>.</p></li>
<li id="id21518"><p>Miguel Ponce-de-Leon, Francisco Montero, and Juli Pereto. Solving gap metabolites and blocked reactions in genome-scale models: application to the metabolic network of &lt;i&gt;blattabacterium cuenoti&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, OCT 31 2013. <a class="reference external" href="https://doi.org/10.1186/1752-0509-7-114">[DOI]</a>.</p></li>
<li id="id21519"><p>Matthew B. Biggs and Jason A. Papin. Novel multiscale modeling tool applied to &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt; biofilm formation. <em>PLOS ONE</em>, OCT 17 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0078011">[DOI]</a>.</p></li>
<li id="id21520"><p>Roger L. Chang, Lei Xie, Philip E. Bourne, and Bernhard O. Palsson. Antibacterial mechanisms identified through structural systems pharmacology. <em>BMC SYSTEMS BIOLOGY</em>, OCT 10 2013. <a class="reference external" href="https://doi.org/10.1186/1752-0509-7-102">[DOI]</a>.</p></li>
<li id="id21521"><p>Nicolas A. L. Flahaut, Anne Wiersma, Bert van de Bunt, Dirk E. Martens, Peter J. Schaap, Lolke Sijtsma, Vitor A. Martins dos Santos, and Willem M. de Vos. Genome-scale metabolic model for &lt;i&gt;lactococcus lactis&lt;/i&gt; mg1363 and its application to the analysis of flavor formation. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 97(19):8729–8739, OCT 2013. <a class="reference external" href="https://doi.org/10.1007/s00253-013-5140-2">[DOI]</a>.</p></li>
<li id="id21522"><p>Longfei Mao and Wynand S. Verwoerd. Genome-scale stoichiometry analysis to elucidate the innate capability of the cyanobacterium &lt;i&gt;synechocystis&lt;/i&gt; for electricity generation. <em>JOURNAL OF INDUSTRIAL MICROBIOLOGY &amp; BIOTECHNOLOGY</em>, 40(10):1161–1180, OCT 2013. <a class="reference external" href="https://doi.org/10.1007/s10295-013-1308-0">[DOI]</a>.</p></li>
<li id="id21523"><p>Koyel Mitra, Anne-Ruxandra Carvunis, Sanath Kumar Ramesh, and Trey Ideker. Integrative approaches for finding modular structure in biological networks. <em>NATURE REVIEWS GENETICS</em>, 14(10):719–732, OCT 2013. <a class="reference external" href="https://doi.org/10.1038/nrg3552">[DOI]</a>.</p></li>
<li id="id21524"><p>Yi Wang, Xiangzhen Li, Caroline B. Milne, Holger Janssen, Weiyin Lin, Gloria Phan, Huiying Hu, Yong-Su Jin, Nathan D. Price, and Hans P. Blaschek. Development of a gene knockout system using mobile group ii introns (targetron) and genetic disruption of acid production pathways in &lt;i&gt;clostridium beijerinckii&lt;/i&gt;. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, 79(19):5853–5863, OCT 2013. <a class="reference external" href="https://doi.org/10.1128/AEM.00971-13">[DOI]</a>.</p></li>
<li id="id21525"><p>Wei Wu, Neema Jamshidi, Satish A. Eraly, Henry C. Liu, Kevin T. Bush, Bernhard O. Palsson, and Sanjay K. Nigam. Multispecific drug transporter &lt;i&gt;slc22a8&lt;/i&gt; (&lt;i&gt;oat3&lt;/i&gt;) regulates multiple metabolic and signaling pathways. <em>DRUG METABOLISM AND DISPOSITION</em>, 41(10):1825–1834, OCT 2013. <a class="reference external" href="https://doi.org/10.1124/dmd.113.052647">[DOI]</a>.</p></li>
<li id="id21526"><p>Jing Meng, Zixiang Xu, Jing Guo, Yunxia Yue, and Xiao Sun. Analysis of enhanced current-generating mechanism of &lt;i&gt;geobacter&lt;/i&gt; &lt;i&gt;sulfurreducens&lt;/i&gt; strain via model-driven metabolism simulation. <em>PLOS ONE</em>, SEP 13 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0073907">[DOI]</a>.</p></li>
<li id="id21527"><p>Anirikh Chakrabarti, Ljubisa Miskovic, Keng Cher Soh, and Vassily Hatzimanikatis. Towards kinetic modeling of genome-scale metabolic networks without sacrificing stoichiometric, thermodynamic and physiological constraints. <em>BIOTECHNOLOGY JOURNAL</em>, 8(9, SI):1043–U105, SEP 2013. <a class="reference external" href="https://doi.org/10.1002/biot.201300091">[DOI]</a>.</p></li>
<li id="id21528"><p>Matthew C. Gonnerman, Matthew N. Benedict, Adam M. Feist, William W. Metcalf, and Nathan D. Price. Genomically and biochemically accurate metabolic reconstruction of &lt;i&gt;methanosarcina barkeri&lt;/i&gt; fusaro, img746. <em>BIOTECHNOLOGY JOURNAL</em>, 8(9, SI):1070–U129, SEP 2013. <a class="reference external" href="https://doi.org/10.1002/biot.201200266">[DOI]</a>.</p></li>
<li id="id21529"><p>Kai Zhuang, Bhavik R. Bakshi, and Markus J. Herrgard. Multi-scale modeling for sustainable chemical production. <em>BIOTECHNOLOGY JOURNAL</em>, 8(9, SI):973+, SEP 2013. <a class="reference external" href="https://doi.org/10.1002/biot.201200272">[DOI]</a>.</p></li>
<li id="id21530"><p>Piyush Labhsetwar, John Andrew Cole, Elijah Roberts, Nathan D. Price, and Zaida A. Luthey-Schulten. Heterogeneity in protein expression induces metabolic variability in a modeled &lt;i&gt;escherichia coli&lt;/i&gt; population. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 110(34):14006–14011, AUG 20 2013. <a class="reference external" href="https://doi.org/10.1073/pnas.1222569110">[DOI]</a>.</p></li>
<li id="id21531"><p>Sarah Schatschneider, Marcus Persicke, Steven Alexander Watt, Gerd Hublik, Alfred Puehler, Karsten Niehaus, and Frank-Joerg Vorhoelter. Establishment, &lt;i&gt;in silico&lt;/i&gt; analysis, and experimental verification of a large-scale metabolic network of the xanthan producing &lt;i&gt;xanthomonas campestris&lt;/i&gt; pv. campestris strain b100. <em>JOURNAL OF BIOTECHNOLOGY</em>, 167(2, SI):123–134, AUG 20 2013. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2013.01.023">[DOI]</a>.</p></li>
<li id="id21532"><p>Benjamin D. Heavner, Kieran Smallbone, Nathan D. Price, and Larry P. Walker. Version 6 of the consensus yeast metabolic network refines biochemical coverage and improves model performance. <em>DATABASE-THE JOURNAL OF BIOLOGICAL DATABASES AND CURATION</em>, AUG 9 2013. <a class="reference external" href="https://doi.org/10.1093/database/bat059">[DOI]</a>.</p></li>
<li id="id21533"><p>Ali Ebrahim, Joshua A. Lerman, Bernhard O. Palsson, and Daniel R. Hyduke. Cobrapy: constraints-based reconstruction and analysis for python. <em>BMC SYSTEMS BIOLOGY</em>, AUG 8 2013. <a class="reference external" href="https://doi.org/10.1186/1752-0509-7-74">[DOI]</a>.</p></li>
<li id="id21534"><p>Ashwini Kumar Sharma and Rainer Koenig. Metabolic network modeling approaches for investigating the “hungry cancer”. <em>SEMINARS IN CANCER BIOLOGY</em>, 23(4):227–234, AUG 2013. <a class="reference external" href="https://doi.org/10.1016/j.semcancer.2013.05.001">[DOI]</a>.</p></li>
<li id="id21535"><p>Janne Wallenius, Matti Viikila, Shrikant Survase, Heikki Ojamo, and Tero Eerikainen. Constraint-based genome-scale metabolic modeling of &lt;i&gt;clostridium acetobutylicum&lt;/i&gt; behavior in an immobilized column. <em>BIORESOURCE TECHNOLOGY</em>, 142:603–610, AUG 2013. <a class="reference external" href="https://doi.org/10.1016/j.biortech.2013.05.085">[DOI]</a>.</p></li>
<li id="id21536"><p>Yuekai Sun, Ronan M. T. Fleming, Ines Thiele, and Michael A. Saunders. Robust flux balance analysis of multiscale biochemical reaction networks. <em>BMC BIOINFORMATICS</em>, JUL 30 2013. <a class="reference external" href="https://doi.org/10.1186/1471-2105-14-240">[DOI]</a>.</p></li>
<li id="id21537"><p>Ashish Misra, Matthew F. Conway, Joseph Johnnie, Tabish M. Qureshi, Bao Lige, Anne M. Derrick, Eddy C. Agbo, and Ganesh Sriram. Metabolic analyses elucidate non-trivial gene targets for amplifying dihydroartemisinic acid production in yeast. <em>FRONTIERS IN MICROBIOLOGY</em>, JUL 26 2013. <a class="reference external" href="https://doi.org/10.3389/fmicb.2013.00200">[DOI]</a>.</p></li>
<li id="id21538"><p>Francisco G. Vital-Lopez, Anders Wallqvist, and Jaques Reifman. Bridging the gap between gene expression and metabolic phenotype via kinetic models. <em>BMC SYSTEMS BIOLOGY</em>, JUL 22 2013. <a class="reference external" href="https://doi.org/10.1186/1752-0509-7-63">[DOI]</a>.</p></li>
<li id="id21539"><p>Jonathan M. Dreyfuss, Jeremy D. Zucker, Heather M. Hood, Linda R. Ocasio, Matthew S. Sachs, and James E. Galagan. Reconstruction and validation of a genome-scale metabolic model for the filamentous fungus &lt;i&gt;neurospora crassa&lt;/i&gt; using farm. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUL 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003126">[DOI]</a>.</p></li>
<li id="id21540"><p>Alison K. Hottes, Peter L. Freddolino, Anupama Khare, Zachary N. Donnell, Julia C. Liu, and Saeed Tavazoie. Bacterial adaptation through loss of function. <em>PLOS GENETICS</em>, JUL 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pgen.1003617">[DOI]</a>.</p></li>
<li id="id21541"><p>Hyun Ju Kim, Bo Kyeng Hou, Sung Gun Lee, Joong Su Kim, Dong-Woo Lee, and Sang Jun Lee. Genome-wide analysis of redox reactions reveals metabolic engineering targets for d-lactate overproduction in &lt;i&gt;escherichia&lt;/i&gt; &lt;i&gt;coli&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 18:44–52, JUL 2013. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2013.03.004">[DOI]</a>.</p></li>
<li id="id21542"><p>Elad Noor, Hulda S. Haraldsdottir, Ron Milo, and Ronan M. T. Fleming. Consistent estimation of gibbs energy using component contributions. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUL 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003098">[DOI]</a>.</p></li>
<li id="id21543"><p>Swagatika Sahoo and Ines Thiele. Predicting the impact of diet and enzymopathies on human small intestinal epithelial cells. <em>HUMAN MOLECULAR GENETICS</em>, 22(13):2705–2722, JUL 1 2013. <a class="reference external" href="https://doi.org/10.1093/hmg/ddt119">[DOI]</a>.</p></li>
<li id="id21544"><p>Henning Knoop, Marianne Gruendel, Yvonne Zilliges, Robert Lehmann, Sabrina Hoffmann, Wolfgang Lockau, and Ralf Steuer. Flux balance analysis of cyanobacterial metabolism: the metabolic network of synechocystis sp pcc 6803. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUN 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1003081">[DOI]</a>.</p></li>
<li id="id21545"><p>Namrata Tomar and Rajat K. De. Comparing methods for metabolic network analysis and an application to metabolic engineering. <em>GENE</em>, 521(1):1–14, MAY 25 2013. <a class="reference external" href="https://doi.org/10.1016/j.gene.2013.03.017">[DOI]</a>.</p></li>
<li id="id21546"><p>Di Huang, Shanshan Li, Menglei Xia, Jianping Wen, and Xiaoqiang Jia. Genome-scale metabolic network guided engineering of &lt;i&gt;streptomyces tsukubaensis&lt;/i&gt; for fk506 production improvement. <em>MICROBIAL CELL FACTORIES</em>, MAY 24 2013. <a class="reference external" href="https://doi.org/10.1186/1475-2859-12-52">[DOI]</a>.</p></li>
<li id="id21547"><p>Sidhartha Chaudhury, Mohamed Diwan M. Abdulhameed, Narender Singh, Gregory J. Tawa, Patrik M. D'haeseleer, Adam T. Zemla, Ali Navid, Carol E. Zhou, Matthew C. Franklin, Jonah Cheung, Michael J. Rudolph, James Love, John F. Graf, David A. Rozak, Jennifer L. Dankmeyer, Kei Amemiya, Simon Daefler, and Anders Wallqvist. Rapid countermeasure discovery against &lt;i&gt;francisella tularensis&lt;/i&gt; based on a metabolic network reconstruction. <em>PLOS ONE</em>, MAY 21 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0063369">[DOI]</a>.</p></li>
<li id="id21548"><p>Agris Pentjuss, Ilona Odzina, Andrejs Kostromins, David A. Fell, Egils Stalidzans, and Uldis Kalnenieks. Biotechnological potential of respiring &lt;i&gt;zymomonas mobilis&lt;/i&gt;: a stoichiometric analysis of its central metabolism. <em>JOURNAL OF BIOTECHNOLOGY</em>, 165(1):1–10, MAY 10 2013. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2013.02.014">[DOI]</a>.</p></li>
<li id="id21549"><p>Debjit Ray and Ping Ye. Characterization of the metabolic requirements in yeast meiosis. <em>PLOS ONE</em>, MAY 8 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0063707">[DOI]</a>.</p></li>
<li id="id21550"><p>Nicole L. Fong, Joshua A. Lerman, Irene Lam, Bernhard O. Palsson, and Pep Charusanti. Reconciling a salmonella enterica metabolic model with experimental data confirms that overexpression of the glyoxylate shunt can rescue a lethal ppc deletion mutant. <em>FEMS MICROBIOLOGY LETTERS</em>, 342(1):62–69, MAY 2013. <a class="reference external" href="https://doi.org/10.1111/1574-6968.12109">[DOI]</a>.</p></li>
<li id="id21551"><p>Ines Thiele, Neil Swainston, Ronan M. T. Fleming, Andreas Hoppe, Swagatika Sahoo, Maike K. Aurich, Hulda Haraldsdottir, Monica L. Mo, Ottar Rolfsson, Miranda D. Stobbe, Stefan G. Thorleifsson, Rasmus Agren, Christian Boelling, Sergio Bordel, Arvind K. Chavali, Paul Dobson, Warwick B. Dunn, Lukas Endler, David Hala, Michael Hucka, Duncan Hull, Daniel Jameson, Neema Jamshidi, Jon J. Jonsson, Nick Juty, Sarah Keating, Intawat Nookaew, Nicolas Le Novere, Naglis Malys, Alexander Mazein, Jason A. Papin, Nathan D. Price, Evgeni Selkov, Sr., Martin I. Sigurdsson, Evangelos Simeonidis, Nikolaus Sonnenschein, Kieran Smallbone, Anatoly Sorokin, Johannes H. G. M. van Beek, Dieter Weichart, Igor Goryanin, Jens Nielsen, Hans V. Westerhoff, Douglas B. Kell, Pedro Mendes, and Bernhard O. Palsson. A community-driven global reconstruction of human metabolism. <em>NATURE BIOTECHNOLOGY</em>, 31(5):419+, MAY 2013. <a class="reference external" href="https://doi.org/10.1038/nbt.2488">[DOI]</a>.</p></li>
<li id="id21552"><p>Jiun Y. Yen, Hadi Nazem-Bokaee, Benjamin G. Freedman, Ahmad I. M. Athamneh, and Ryan S. Senger. Deriving metabolic engineering strategies from genome-scale modeling with flux ratio constraints. <em>BIOTECHNOLOGY JOURNAL</em>, 8(5, SI):581–594, MAY 2013. <a class="reference external" href="https://doi.org/10.1002/biot.201200234">[DOI]</a>.</p></li>
<li id="id21553"><p>Wei Zou, Maoda Zhou, Liming Liu, and Jian Chen. Reconstruction and analysis of the industrial strain &lt;i&gt;bacillus megaterium&lt;/i&gt; wsh002 genome-scale &lt;i&gt;in silico&lt;/i&gt; metabolic model. <em>JOURNAL OF BIOTECHNOLOGY</em>, 164(4):503–509, APR 15 2013. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2013.01.019">[DOI]</a>.</p></li>
<li id="id21554"><p>Cristiana Gomes de Oliveira Dal'Molin and Lars Keld Nielsen. Plant genome-scale metabolic reconstruction and modelling. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 24(2):271–277, APR 2013. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2012.08.007">[DOI]</a>.</p></li>
<li id="id21555"><p>Arne C. Mueller and Alexander Bockmayr. Fast thermodynamically constrained flux variability analysis. <em>BIOINFORMATICS</em>, 29(7):903–909, APR 1 2013. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btt059">[DOI]</a>.</p></li>
<li id="id21556"><p>Yu Qiu, Harish Nagarajan, Mallory Embree, Wendy Shieu, Elisa Abate, Katy Juarez, Byung-Kwan Cho, James G. Elkins, Kelly P. Nevin, Christian L. Barrett, Derek R. Lovley, Bernhard O. Palsson, and Karsten Zengler. Characterizing the interplay between multiple levels of organization within bacterial sigma factor regulatory networks. <em>NATURE COMMUNICATIONS</em>, APR 2013. <a class="reference external" href="https://doi.org/10.1038/ncomms2743">[DOI]</a>.</p></li>
<li id="id21557"><p>Benjamin Steeb, Beatrice Claudi, Neil A. Burton, Petra Tienz, Alexander Schmidt, Hesso Farhan, Alain Maze, and Dirk Bumann. Parallel exploitation of diverse host nutrients enhances &lt;i&gt;salmonella&lt;/i&gt; virulence. <em>PLOS PATHOGENS</em>, APR 2013. <a class="reference external" href="https://doi.org/10.1371/journal.ppat.1003301">[DOI]</a>.</p></li>
<li id="id21558"><p>Judith A. H. Wodke, Jacek Puchalka, Maria Lluch-Senar, Josep Marcos, Eva Yus, Miguel Godinho, Ricardo Gutierrez-Gallego, Vitor A. P. Martins dos Santos, Luis Serrano, Edda Klipp, and Tobias Maier. Dissecting the energy metabolism in &lt;i&gt;mycoplasma pneumoniae&lt;/i&gt; through genome-scale metabolic modeling. <em>MOLECULAR SYSTEMS BIOLOGY</em>, APR 2013. <a class="reference external" href="https://doi.org/10.1038/msb.2013.6">[DOI]</a>.</p></li>
<li id="id21559"><p>Keiko Yonekura-Sakakibara, Atsushi Fukushima, and Kazuki Saito. Transcriptome data modeling for targeted plant metabolic engineering. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 24(2):285–290, APR 2013. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2012.10.018">[DOI]</a>.</p></li>
<li id="id21560"><p>Mathias Ganter, Thomas Bernard, Sebastien Moretti, Joerg Stelling, and Marco Pagni. Metanetx.org: a website and repository for accessing, analysing and manipulating metabolic networks. <em>BIOINFORMATICS</em>, 29(6):815–816, MAR 15 2013. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btt036">[DOI]</a>.</p></li>
<li id="id21561"><p>Rasmus Agren, Liming Liu, Saeed Shoaie, Wanwipa Vongsangnak, Intawat Nookaew, and Jens Nielsen. The raven toolbox and its use for generating a genome-scale metabolic model for &lt;i&gt;penicillium chrysogenum&lt;/i&gt;. <em>PLOS COMPUTATIONAL BIOLOGY</em>, MAR 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1002980">[DOI]</a>.</p></li>
<li id="id21562"><p>Miguel Angel Medina. Systems biology for molecular life sciences and its impact in biomedicine. <em>CELLULAR AND MOLECULAR LIFE SCIENCES</em>, 70(6):1035–1053, MAR 2013. <a class="reference external" href="https://doi.org/10.1007/s00018-012-1109-z">[DOI]</a>.</p></li>
<li id="id21563"><p>Bevan Kai-Sheng Chung, Meiyappan Lakshmanan, Maximilian Klement, Chi Bun Ching, and Dong-Yup Lee. Metabolic reconstruction and flux analysis of industrial &lt;i&gt;pichia&lt;/i&gt; yeasts. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 97(5):1865–1873, MAR 2013. <a class="reference external" href="https://doi.org/10.1007/s00253-013-4702-7">[DOI]</a>.</p></li>
<li id="id21564"><p>Douglas B. Kell, Paul D. Dobson, Elizabeth Bilsland, and Stephen G. Oliver. The promiscuous binding of pharmaceutical drugs and their transporter-mediated uptake into cells: what we (need to) know and how we can do so. <em>DRUG DISCOVERY TODAY</em>, 18(5-6):218–239, MAR 2013. <a class="reference external" href="https://doi.org/10.1016/j.drudis.2012.11.008">[DOI]</a>.</p></li>
<li id="id21565"><p>Sumanta Mukherjee, Awanti Sambarey, Karyala Prashanthi, and Nagasuma Chandra. Current trends in modeling hostpathogen interactions. <em>WILEY INTERDISCIPLINARY REVIEWS-DATA MINING AND KNOWLEDGE DISCOVERY</em>, 3(2):109–128, MAR-APR 2013. <a class="reference external" href="https://doi.org/10.1002/widm.1085">[DOI]</a>.</p></li>
<li id="id21566"><p>Zhaobin Xu, Xin Fang, Thomas K. Wood, and Zuyi Jacky Huang. A systems-level approach for investigating &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt; biofilm formation. <em>PLOS ONE</em>, FEB 22 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0057050">[DOI]</a>.</p></li>
<li id="id21567"><p>Brian J. Schmidt, Jason A. Papin, and Cynthia J. Musante. Mechanistic systems modeling to guide drug discovery and development. <em>DRUG DISCOVERY TODAY</em>, 18(3-4):116–127, FEB 2013. <a class="reference external" href="https://doi.org/10.1016/j.drudis.2012.09.003">[DOI]</a>.</p></li>
<li id="id21568"><p>Ines Thiele, Almut Heinken, and Ronan M. T. Fleming. A systems biology approach to studying the role of microbes in human health. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 24(1):4–12, FEB 2013. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2012.10.001">[DOI]</a>.</p></li>
<li id="id21569"><p>Zixiang Xu, Xiao Sun, and Jibin Sun. Construction and analysis of the model of energy metabolism in &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt;. <em>PLOS ONE</em>, JAN 30 2013. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0055137">[DOI]</a>.</p></li>
<li id="id21570"><p>Ottar Rolfsson, Giuseppe Paglia, Manuela Magnusdottir, Bernhard O. Palsson, and Ines Thiele. Inferring the metabolism of human orphan metabolites from their metabolic network context affirms human gluconokinase activity. <em>BIOCHEMICAL JOURNAL</em>, 449(2):427–435, JAN 15 2013. <a class="reference external" href="https://doi.org/10.1042/BJ20120980">[DOI]</a>.</p></li>
<li id="id21571"><p>Kathleen A. Curran, Johnm. Leavitt, AshtyS. Karim, and Hal S. Alper. Metabolic engineering of muconic acid production in &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 15:55–66, JAN 2013. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2012.10.003">[DOI]</a>.</p></li>
<li id="id21572"><p>Tong Hao, Binbin Han, Hongwu Ma, Jing Fu, Hui Wang, Zhiwen Wang, Bincai Tang, Tao Chen, and Xueming Zhao. &lt;i&gt;in silico&lt;/i&gt; metabolic engineering of &lt;i&gt;bacillus subtilis&lt;/i&gt; for improved production of riboflavin, egl-237, (&lt;i&gt;r,r&lt;/i&gt;)-2,3-butanediol and isobutanol. <em>MOLECULAR BIOSYSTEMS</em>, 9(8):2034–2044, 2013. <a class="reference external" href="https://doi.org/10.1039/c3mb25568a">[DOI]</a>.</p></li>
<li id="id21573"><p>Colin R. Harwood, Susanne Pohl, Wendy Smith, and Anil Wipat. &lt;i&gt;bacillus subtilis&lt;/i&gt;: model gram-positive synthetic biology chassis. In C Harwood and A Wipat, editors, <em>MICROBIAL SYNTHETIC BIOLOGY</em>, volume 40 of Methods in Microbiology, pages 87–117. 2013. <a class="reference external" href="https://doi.org/10.1016/B978-0-12-417029-2.00004-2">[DOI]</a>.</p></li>
<li id="id21574"><p>Almut Heinken, Swagatika Sahoo, Ronan M. T. Fleming, and Ines Thiele. Systems-level characterization of a host-microbe metabolic symbiosis in the mammalian gut. <em>GUT MICROBES</em>, 4(1):28–40, 2013. <a class="reference external" href="https://doi.org/10.4161/gmic.22370">[DOI]</a>.</p></li>
<li id="id21575"><p>Claudia E. Hernandez Patino, Gustavo Jaime-Munoz, and Osbaldo Resendis-Antonio. Systems biology of cancer: moving toward the integrative study of the metabolic alterations in cancer cells. <em>FRONTIERS IN PHYSIOLOGY</em>, 2013. <a class="reference external" href="https://doi.org/10.3389/fphys.2012.00481">[DOI]</a>.</p></li>
<li id="id21576"><p>Young-Mo Kim, Brian J. Schmidt, Afshan S. Kidwai, Marcus B. Jones, Brooke L. Deatherage Kaiser, Heather M. Brewer, Hugh D. Mitchell, Bernhard O. Palsson, Jason E. McDermott, Fred Heffron, Richard D. Smith, Scott N. Peterson, Charles Ansong, Daniel R. Hyduke, Thomas O. Metz, and Joshua N. Adkins. &lt;i&gt;salmonella&lt;/i&gt; modulates metabolism during growth under conditions that induce expression of virulence genes. <em>MOLECULAR BIOSYSTEMS</em>, 9(6):1522–1534, 2013. <a class="reference external" href="https://doi.org/10.1039/c3mb25598k">[DOI]</a>.</p></li>
<li id="id21577"><p>Steffen Krauser, Christian Weyler, Lisa Katharina Blass, and Elmar Heinzle. Directed multistep biocatalysis using tailored permeabilized cells. In AP Zeng, editor, <em>FUNDAMENTALS AND APPLICATION OF NEW BIOPRODUCTION SYSTEMS</em>, volume 137 of Advances in Biochemical Engineering-Biotechnology, pages 185–234. 2013. <a class="reference external" href="https://doi.org/10.1007/10\_2013\_240">[DOI]</a>.</p></li>
<li id="id21578"><p>Jie Liu, Qian Gao, Nan Xu, and Liming Liu. Genome-scale reconstruction and &lt;i&gt;in silico&lt;/i&gt; analysis of &lt;i&gt;aspergillus terreus&lt;/i&gt; metabolism. <em>MOLECULAR BIOSYSTEMS</em>, 9(7):1939–1948, 2013. <a class="reference external" href="https://doi.org/10.1039/c3mb70090a">[DOI]</a>.</p></li>
<li id="id21579"><p>Longfei Mao and Wynand S. Verwoerd. Model-driven elucidation of the inherent capacity of geobacter sulfurreducens for electricity generation. <em>JOURNAL OF BIOLOGICAL ENGINEERING</em>, 2013. <a class="reference external" href="https://doi.org/10.1186/1754-1611-7-14">[DOI]</a>.</p></li>
<li id="id21580"><p>Erin R. Shellman, Charles F. Burant, and Santiago Schnell. Network motifs provide signatures that characterize metabolism. <em>MOLECULAR BIOSYSTEMS</em>, 9(3):352–360, 2013. <a class="reference external" href="https://doi.org/10.1039/c2mb25346a">[DOI]</a>.</p></li>
<li id="id21581"><p>Tie Shen, Bin Rui, Hong Zhou, Ximing Zhang, Yin Yi, Han Wen, Haoran Zheng, Jihui Wu, and Yunyu Shi. Metabolic flux ratio analysis and multi-objective optimization revealed a globally conserved and coordinated metabolic response of &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; to paraquat-induced oxidative stress. <em>MOLECULAR BIOSYSTEMS</em>, 9(1):121–132, 2013. <a class="reference external" href="https://doi.org/10.1039/c2mb25285f">[DOI]</a>.</p></li>
<li id="id21582"><p>Mohammad Tajparast and Dominic Frigon. Genome-scale metabolic modeling to provide insight into the production of storage compounds during feast-famine cycles of activated sludge. <em>WATER SCIENCE AND TECHNOLOGY</em>, 67(3):469–476, 2013. <a class="reference external" href="https://doi.org/10.2166/wst.2012.569">[DOI]</a>.</p></li>
<li id="id21583"><p>Wei Tong, Zhen Chen, Zhe Cao, Quanhui Wang, Jiyuan Zhang, Xue Bai, Rong Wang, and Siqi Liu. Robustness analysis of a constraint-based metabolic model links cell growth and proteomics of &lt;i&gt;thermoanaerobacter tengcongensis&lt;/i&gt; under temperature perturbation. <em>MOLECULAR BIOSYSTEMS</em>, 9(4):713–722, 2013. <a class="reference external" href="https://doi.org/10.1039/c3mb25278g">[DOI]</a>.</p></li>
<li id="id21584"><p>Chuan Xu, Lili Liu, Zhao Zhang, Danfeng Jin, Juanping Qiu, and Ming Chen. Genome-scale metabolic model in guiding metabolic engineering of microbial improvement. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 97(2):519–539, JAN 2013. <a class="reference external" href="https://doi.org/10.1007/s00253-012-4543-9">[DOI]</a>.</p></li>
<li id="id21585"><p>Nan Xu, Liming Liu, Wei Zou, Jie Liu, Qiang Hua, and Jian Chen. Reconstruction and analysis of the genome-scale metabolic network of &lt;i&gt;candida glabrata&lt;/i&gt;. <em>MOLECULAR BIOSYSTEMS</em>, 9(2):205–216, 2013. <a class="reference external" href="https://doi.org/10.1039/c2mb25311a">[DOI]</a>.</p></li>
<li id="id21586"><p>Ai-Di Zhang, Shao-Xing Dai, and Jing-Fei Huang. Reconstruction and analysis of human kidney-specific metabolic network based on omics data. <em>BIOMED RESEARCH INTERNATIONAL</em>, 2013. <a class="reference external" href="https://doi.org/10.1155/2013/187509">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-12" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-12" name="0-12" role="tabpanel" tabindex="0"><div class="docutils container" id="id21841">
<ol class="arabic simple" start="1">
<li id="id23407"><p>Porntip Chiewchankaset, Yanee Srimarut, Amornpan Klanchui, Peter Kurdi, Vethachai Plengvidhya, and Asawin Meechai. Systematic identification of &lt;i&gt;lactobacillus plantarum&lt;/i&gt; auxotrophs for fermented nham using genome-scale metabolic model. <em>JOURNAL OF BIOTECHNOLOGY</em>, 162(2-3):327–335, DEC 31 2012. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2012.08.019">[DOI]</a>.</p></li>
<li id="id23408"><p>Guoqiang Xu, Wei Zou, Xiulai Chen, Nan Xu, Liming Liu, and Jian Chen. Fumaric acid production in &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; by &lt;i&gt;in silico&lt;/i&gt; aided metabolic engineering. <em>PLOS ONE</em>, DEC 26 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0052086">[DOI]</a>.</p></li>
<li id="id23409"><p>Piotr Zakrzewski, Marnix H. Medema, Albert Gevorgyan, Andrzej M. Kierzek, Rainer Breitling, and Eriko Takano. Multimeteval: comparative and multi-objective analysis of genome-scale metabolic models. <em>PLOS ONE</em>, DEC 14 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0051511">[DOI]</a>.</p></li>
<li id="id23410"><p>Yuliang Wang, James A. Eddy, and Nathan D. Price. Reconstruction of genome-scale metabolic models for 126 human tissues using mcadre. <em>BMC SYSTEMS BIOLOGY</em>, DEC 13 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-153">[DOI]</a>.</p></li>
<li id="id23411"><p>Chuanli Wang, Longyun Guo, Yixue Li, and Zhuo Wang. Systematic comparison of c3 and c4 plants based on metabolic network analysis. <em>BMC SYSTEMS BIOLOGY</em>, DEC 12 2012. 23rd International Conference on Genome Informatics (GIW), Tainan, TAIWAN, DEC 12-14, 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-S2-S9">[DOI]</a>.</p></li>
<li id="id23412"><p>Pengcheng Pan and Qiang Hua. Reconstruction and &lt;i&gt;in silico&lt;/i&gt; analysis of metabolic network for an oleaginous yeast, &lt;i&gt;yarrowia lipolytica&lt;/i&gt;. <em>PLOS ONE</em>, DEC 7 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0051535">[DOI]</a>.</p></li>
<li id="id23413"><p>Maike K. Aurich and Ines Thiele. Contextualization procedure and modeling of monocyte specific tlr signaling. <em>PLOS ONE</em>, DEC 6 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0049978">[DOI]</a>.</p></li>
<li id="id23414"><p>Ali Navid and Eivind Almaas. Genome-level transcription data of &lt;i&gt;yersinia pestis&lt;/i&gt; analyzed with a new metabolic constraint-based approach. <em>BMC SYSTEMS BIOLOGY</em>, DEC 6 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-150">[DOI]</a>.</p></li>
<li id="id23415"><p>Jari Kamarainen, Henning Knoop, Natalie J. Stanford, Fernando Guerrero, M. Kalim Akhtar, Eva-Mari Aro, Ralf Steuer, and Patrik R. Jones. Physiological tolerance and stoichiometric potential of cyanobacteria for hydrocarbon fuel production. <em>JOURNAL OF BIOTECHNOLOGY</em>, 162(1, SI):67–74, NOV 30 2012. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2012.07.193">[DOI]</a>.</p></li>
<li id="id23416"><p>Livnat Jerby, Lior Wolf, Carsten Denkert, Gideon Y. Stein, Mika Hilvo, Matej Oresic, Tamar Geiger, and Eytan Ruppin. Metabolic associations of reduced proliferation and oxidative stress in advanced breast cancer. <em>CANCER RESEARCH</em>, 72(22):5712–5720, NOV 15 2012. <a class="reference external" href="https://doi.org/10.1158/0008-5472.CAN-12-2215">[DOI]</a>.</p></li>
<li id="id23417"><p>Michael Bekaert. Reconstruction of &lt;i&gt;danio rerio&lt;/i&gt; metabolic model accounting for subcellular compartmentalisation. <em>PLOS ONE</em>, NOV 14 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0049903">[DOI]</a>.</p></li>
<li id="id23418"><p>Elhanan Borenstein. Computational systems biology and &lt;i&gt;in silico&lt;/i&gt; modeling of the human microbiome. <em>BRIEFINGS IN BIOINFORMATICS</em>, 13(6, SI):769–780, NOV 2012. <a class="reference external" href="https://doi.org/10.1093/bib/bbs022">[DOI]</a>.</p></li>
<li id="id23419"><p>Ana Rita Brochado, Sergej Andrejev, Costas D. Maranas, and Kiran R. Patil. Impact of stoichiometry representation on simulation of genotype-phenotype relationships in metabolic networks. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1002758">[DOI]</a>.</p></li>
<li id="id23420"><p>Ali R. Zomorrodi, Patrick F. Suthers, Sridhar Ranganathan, and Costas D. Maranas. Mathematical optimization applications in metabolic networks. <em>METABOLIC ENGINEERING</em>, 14(6):672–686, NOV 2012. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2012.09.005">[DOI]</a>.</p></li>
<li id="id23421"><p>P. Setoodeh, A. Jahanmiri, and R. Eslamloueyan. Hybrid neural modeling framework for simulation and optimization of diauxie-involved fed-batch fermentative succinate production. <em>CHEMICAL ENGINEERING SCIENCE</em>, 81:57–76, OCT 22 2012. <a class="reference external" href="https://doi.org/10.1016/j.ces.2012.06.031">[DOI]</a>.</p></li>
<li id="id23422"><p>Matthias Bujara and Sven Panke. In silico assessment of cell-free systems. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 109(10):2620–2629, OCT 2012. <a class="reference external" href="https://doi.org/10.1002/bit.24534">[DOI]</a>.</p></li>
<li id="id23423"><p>Ines Thiele, Ronan M. T. Fleming, Richard Que, Aarash Bordbar, Dinh Diep, and Bernhard O. Palsson. Multiscale modeling of metabolism and macromolecular synthesis in &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt; and its application to the evolution of codon usage. <em>PLOS ONE</em>, SEP 28 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0045635">[DOI]</a>.</p></li>
<li id="id23424"><p>Ting Liu, Wei Zou, Liming Liu, and Jian Chen. A constraint-based model of &lt;i&gt;scheffersomyces stipitis&lt;/i&gt; for improved ethanol production. <em>BIOTECHNOLOGY FOR BIOFUELS</em>, SEP 21 2012. <a class="reference external" href="https://doi.org/10.1186/1754-6834-5-72">[DOI]</a>.</p></li>
<li id="id23425"><p>Pieter-Jan D'Huys, Ivan Lule, Dominique Vercammen, Jozef Anne, Jan F. Van Impe, and Kristel Bernaerts. Genome-scale metabolic flux analysis of &lt;i&gt;streptomyces lividans&lt;/i&gt; growing on a complex medium. <em>JOURNAL OF BIOTECHNOLOGY</em>, 161(1):1–13, SEP 15 2012. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2012.04.010">[DOI]</a>.</p></li>
<li id="id23426"><p>Wei Zou, Liming Liu, Jing Zhang, Haoru Yang, Maoda Zhou, Qiang Hua, and Jian Chen. Reconstruction and analysis of a genome-scale metabolic model of the vitamin c producing industrial strain &lt;i&gt;ketogulonicigenium vulgare&lt;/i&gt; wsh-001. <em>JOURNAL OF BIOTECHNOLOGY</em>, 161(1):42–48, SEP 15 2012. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2012.05.015">[DOI]</a>.</p></li>
<li id="id23427"><p>Xin Fang, Anders Wallqvist, and Jaques Reifman. Modeling phenotypic metabolic adaptations of &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; h37rv under hypoxia. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1002688">[DOI]</a>.</p></li>
<li id="id23428"><p>Paula Jouhten, Marilyn Wiebe, and Merja Penttila. Dynamic flux balance analysis of the metabolism of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; during the shift from fully respirative or respirofermentative metabolic states to anaerobiosis. <em>FEBS JOURNAL</em>, 279(18):3338–3354, SEP 2012. <a class="reference external" href="https://doi.org/10.1111/j.1742-4658.2012.08649.x">[DOI]</a>.</p></li>
<li id="id23429"><p>Anja Karlstaedt, Daniela Fliegner, Georgios Kararigas, Hugo Sanchez Ruderisch, Vera Regitz-Zagrosek, and Hermann-Georg Holzhuetter. Cardionet: a human metabolic network suited for the study of cardiomyocyte metabolism. <em>BMC SYSTEMS BIOLOGY</em>, AUG 29 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-114">[DOI]</a>.</p></li>
<li id="id23430"><p>Steven M. Kelk, Brett G. Olivier, Leen Stougie, and Frank J. Bruggeman. Optimal flux spaces of genome-scale stoichiometric models are determined by a few subnetworks. <em>SCIENTIFIC REPORTS</em>, AUG 15 2012. <a class="reference external" href="https://doi.org/10.1038/srep00580">[DOI]</a>.</p></li>
<li id="id23431"><p>Carlos Eduardo Garcia Sanchez, Cesar Augusto Vargas Garcia, and Rodrigo Gonzalo Torres Saez. Predictive potential of flux balance analysis of &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; using as optimization function combinations of cell compartmental objectives. <em>PLOS ONE</em>, AUG 9 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0043006">[DOI]</a>.</p></li>
<li id="id23432"><p>Liam G. Fearnley and Lars K. Nielsen. Pathlogic-s: a scalable boolean framework for modelling cellular signalling. <em>PLOS ONE</em>, AUG 7 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0041977">[DOI]</a>.</p></li>
<li id="id23433"><p>Sandy J. Macdonald, George G. Lin, Calum W. Russell, Gavin H. Thomas, and Angela E. Douglas. The central role of the host cell in symbiotic nitrogen metabolism. <em>PROCEEDINGS OF THE ROYAL SOCIETY B-BIOLOGICAL SCIENCES</em>, 279(1740):2965–2973, AUG 7 2012. <a class="reference external" href="https://doi.org/10.1098/rspb.2012.0414">[DOI]</a>.</p></li>
<li id="id23434"><p>Xueyang Feng, You Xu, Yixin Chen, and Yinjie J. Tang. Microbesflux: a web platform for drafting metabolic models from the kegg database. <em>BMC SYSTEMS BIOLOGY</em>, AUG 2 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-94">[DOI]</a>.</p></li>
<li id="id23435"><p>Eva Collakova, Jiun Y. Yen, and Ryan S. Senger. Are we ready for genome-scale modeling in plants? <em>PLANT SCIENCE</em>, 191:53–70, AUG 2012. <a class="reference external" href="https://doi.org/10.1016/j.plantsci.2012.04.010">[DOI]</a>.</p></li>
<li id="id23436"><p>Bouke de Jong, Verena Siewers, and Jens Nielsen. Systems biology of yeast: enabling technology for development of cell factories for production of advanced biofuels. <em>CURRENT OPINION IN BIOTECHNOLOGY</em>, 23(4):624–630, AUG 2012. <a class="reference external" href="https://doi.org/10.1016/j.copbio.2011.11.021">[DOI]</a>.</p></li>
<li id="id23437"><p>Andrejs Kostromins and Egils Stalidzans. Paint4net: cobra toolbox extension for visualization of stoichiometric models of metabolism. <em>BIOSYSTEMS</em>, 109(2):233–239, AUG 2012. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2012.03.002">[DOI]</a>.</p></li>
<li id="id23438"><p>Joo Sang Lee, Takashi Nishikawa, and Adilson E. Motter. Why optimal states recruit fewer reactions in metabolic networks. <em>DISCRETE AND CONTINUOUS DYNAMICAL SYSTEMS</em>, 32(8, SI):2937–2950, AUG 2012. <a class="reference external" href="https://doi.org/10.3934/dcds.2012.32.2937">[DOI]</a>.</p></li>
<li id="id23439"><p>Jennifer L. Reed. Shrinking the metabolic solution space using experimental datasets. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1002662">[DOI]</a>.</p></li>
<li id="id23440"><p>Khuram Shahzad and Juan J. Loor. Application of top-down and bottom-up systems approaches in ruminant physiology and metabolism. <em>CURRENT GENOMICS</em>, 13(5):379–394, AUG 2012. <a class="reference external" href="https://doi.org/10.2174/138920212801619269">[DOI]</a>.</p></li>
<li id="id23441"><p>Somedutta Barat, Benjamin Steeb, Alain Maze, and Dirk Bumann. Extensive in vivo resilience of persistent &lt;i&gt;salmonella&lt;/i&gt;. <em>PLOS ONE</em>, JUL 24 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0042007">[DOI]</a>.</p></li>
<li id="id23442"><p>Nuno Carinhas, Rui Oliveira, Paula Marques Alves, Manuel J. T. Carrondo, and Ana P. Teixeira. Systems biotechnology of animal cells: the road to prediction. <em>TRENDS IN BIOTECHNOLOGY</em>, 30(7):377–385, JUL 2012. <a class="reference external" href="https://doi.org/10.1016/j.tibtech.2012.03.004">[DOI]</a>.</p></li>
<li id="id23443"><p>Yu-Chieh Liao, Ming-Hsin Tsai, Feng-Chi Chen, and Chao A. Hsiung. Gemsirv: a software platform for genome-scale metabolic model simulation, reconstruction and visualization. <em>BIOINFORMATICS</em>, 28(13):1752–1758, JUL 1 2012. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/bts267">[DOI]</a>.</p></li>
<li id="id23444"><p>Francisco G. Vital-Lopez, Vesna Memisevic, and Bhaskar Dutta. Tutorial on biological networks. <em>WILEY INTERDISCIPLINARY REVIEWS-DATA MINING AND KNOWLEDGE DISCOVERY</em>, 2(4):298–325, JUL-AUG 2012. <a class="reference external" href="https://doi.org/10.1002/widm.1061">[DOI]</a>.</p></li>
<li id="id23445"><p>Lin Xu, Brandon Barker, and Zhenglong Gu. Dynamic epistasis for different alleles of the same gene. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 109(26):10420–10425, JUN 26 2012. <a class="reference external" href="https://doi.org/10.1073/pnas.1121507109">[DOI]</a>.</p></li>
<li id="id23446"><p>Dave Lee, Kieran Smallbone, Warwick B. Dunn, Ettore Murabito, Catherine L. Winder, Douglas B. Kell, Pedro Mendes, and Neil Swainston. Improving metabolic flux predictions using absolute gene expression data. <em>BMC SYSTEMS BIOLOGY</em>, JUN 19 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-73">[DOI]</a>.</p></li>
<li id="id23447"><p>Amornpan Klanchui, Chiraphan Khannapho, Atchara Phodee, Supapon Cheevadhanarak, and Asawin Meechai. &lt;i&gt;i&lt;/i&gt;ak692: a genome-scale &lt;i&gt;metabolic model of spirulina platensis c1&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, JUN 15 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-71">[DOI]</a>.</p></li>
<li id="id23448"><p>Benjamin D. Heavner, Kieran Smallbone, Brandon Barker, Pedro Mendes, and Larry P. Walker. Yeast 5-an expanded reconstruction of the &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; metabolic network. <em>BMC SYSTEMS BIOLOGY</em>, JUN 4 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-55">[DOI]</a>.</p></li>
<li id="id23449"><p>Judith M. Fontana, Elizabeth Alexander, and Mirella Salvatore. Translational research in infectious disease: current paradigms and challenges ahead. <em>TRANSLATIONAL RESEARCH</em>, 159(6):430–453, JUN 2012. <a class="reference external" href="https://doi.org/10.1016/j.trsl.2011.12.009">[DOI]</a>.</p></li>
<li id="id23450"><p>Chiam Yu Ng, Moo-Young Jung, Jinwon Lee, and Min-Kyu Oh. Production of 2,3-butanediol in &lt;i&gt;saccharomyces&lt;/i&gt; &lt;i&gt;cerevisiae&lt;/i&gt; by &lt;i&gt;in silico&lt;/i&gt; aided metabolic engineering. <em>MICROBIAL CELL FACTORIES</em>, MAY 28 2012. <a class="reference external" href="https://doi.org/10.1186/1475-2859-11-68">[DOI]</a>.</p></li>
<li id="id23451"><p>Basti Bergdahl, Dominik Heer, Uwe Sauer, Barbel Hahn-Hagerdal, and Ed W. J. van Niel. Dynamic metabolomics differentiates between carbon and energy starvation in recombinant &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt; fermenting xylose. <em>BIOTECHNOLOGY FOR BIOFUELS</em>, MAY 15 2012. <a class="reference external" href="https://doi.org/10.1186/1754-6834-5-34">[DOI]</a>.</p></li>
<li id="id23452"><p>Aaron Brandes, Desmond S. Lun, Kuhn Ip, Jeremy Zucker, Caroline Colijn, Brian Weiner, and James E. Galagan. Inferring carbon sources from gene expression profiles using metabolic flux models. <em>PLOS ONE</em>, MAY 14 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0036947">[DOI]</a>.</p></li>
<li id="id23453"><p>Michael J. McAnulty, Jiun Y. Yen, Benjamin G. Freedman, and Ryan S. Senger. Genome-scale modeling using flux ratio constraints to enable metabolic engineering of clostridial metabolism &lt;i&gt;in silico&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, MAY 14 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-42">[DOI]</a>.</p></li>
<li id="id23454"><p>Nicolas Loira, Thierry Dulermo, Jean-Marc Nicaud, and David James Sherman. A genome-scale metabolic model of the lipid-accumulating yeast &lt;i&gt;yarrowia lipolytica&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, MAY 4 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-35">[DOI]</a>.</p></li>
<li id="id23455"><p>Wilbert B. Copeland, Bryan A. Bartley, Deepak Chandran, Michal Galdzicki, Kyung H. Kim, Sean C. Sleight, Costas D. Maranas, and Herbert M. Sauro. Computational tools for metabolic engineering. <em>METABOLIC ENGINEERING</em>, 14(3):270–280, MAY 2012. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2012.03.001">[DOI]</a>.</p></li>
<li id="id23456"><p>David Hala, Lene H. Petersen, Dalma Martinovic, and Duane B. Huggett. Constraints-based stoichiometric analysis of hypoxic stress on steroidogenesis in fathead minnows, &lt;i&gt;pimephales promelas&lt;/i&gt;. <em>JOURNAL OF EXPERIMENTAL BIOLOGY</em>, 215(10):1753–1765, MAY 2012. <a class="reference external" href="https://doi.org/10.1242/jeb.066027">[DOI]</a>.</p></li>
<li id="id23457"><p>Jeffrey D. Orth and Bernhard O. Palsson. Gap-filling analysis of the &lt;i&gt;i&lt;/i&gt;jo1366 &lt;i&gt;escherichia coli&lt;/i&gt; metabolic network reconstruction for discovery of metabolic functions. <em>BMC SYSTEMS BIOLOGY</em>, MAY 1 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-30">[DOI]</a>.</p></li>
<li id="id23458"><p>Abdelhalim Larhlimi, Laszlo David, Joachim Selbig, and Alexander Bockmayr. F2c2: a fast tool for the computation of flux coupling in genome-scale metabolic networks. <em>BMC BIOINFORMATICS</em>, APR 23 2012. <a class="reference external" href="https://doi.org/10.1186/1471-2105-13-57">[DOI]</a>.</p></li>
<li id="id23459"><p>H. S. Haraldsdottir, I. Thiele, and R. M. T. Fleming. Quantitative assignment of reaction directionality in a multicompartmental human metabolic reconstruction. <em>BIOPHYSICAL JOURNAL</em>, 102(8):1703–1711, APR 18 2012. <a class="reference external" href="https://doi.org/10.1016/j.bpj.2012.02.032">[DOI]</a>.</p></li>
<li id="id23460"><p>Gunnar Sigurdsson, Ronan M. T. Fleming, Almut Heinken, and Ines Thiele. A systems biology approach to drug targets in &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt; biofilm. <em>PLOS ONE</em>, APR 16 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0034337">[DOI]</a>.</p></li>
<li id="id23461"><p>Rachael Hageman Blair, Daniel J. Kliebenstein, and Gary A. Churchill. What can causal networks tell us about metabolic pathways? <em>PLOS COMPUTATIONAL BIOLOGY</em>, APR 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1002458">[DOI]</a>.</p></li>
<li id="id23462"><p>Milan J. A. van Hoek and Roeland M. H. Merks. Redox balance is key to explaining full &lt;i&gt;vs&lt;/i&gt;. partial switching to low-yield metabolism. <em>BMC SYSTEMS BIOLOGY</em>, MAR 24 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-22">[DOI]</a>.</p></li>
<li id="id23463"><p>Christine Klier. Use of an uncertainty analysis for genome-scale models as a prediction tool for microbial growth processes in subsurface environments. <em>ENVIRONMENTAL SCIENCE &amp; TECHNOLOGY</em>, 46(5):2790–2798, MAR 6 2012. <a class="reference external" href="https://doi.org/10.1021/es203461u">[DOI]</a>.</p></li>
<li id="id23464"><p>Brett A. Boghigian, John Armando, Daniel Salas, and Blaine A. Pfeifer. Computational identification of gene over-expression targets for metabolic engineering of taxadiene production. <em>APPLIED MICROBIOLOGY AND BIOTECHNOLOGY</em>, 93(5):2063–2073, MAR 2012. <a class="reference external" href="https://doi.org/10.1007/s00253-011-3725-1">[DOI]</a>.</p></li>
<li id="id23465"><p>Arvind K. Chavali, Kevin M. D'Auria, Erik L. Hewlett, Richard D. Pearson, and Jason A. Papin. A metabolic network approach for the identification and prioritization of antimicrobial drug targets. <em>TRENDS IN MICROBIOLOGY</em>, 20(3):113–123, MAR 2012. <a class="reference external" href="https://doi.org/10.1016/j.tim.2011.12.004">[DOI]</a>.</p></li>
<li id="id23466"><p>Xuewen Chen and Yair Shachar-Hill. Insights into metabolic efficiency from flux analysis. <em>JOURNAL OF EXPERIMENTAL BOTANY</em>, 63(6, SI):2343–2351, MAR 2012. <a class="reference external" href="https://doi.org/10.1093/jxb/ers057">[DOI]</a>.</p></li>
<li id="id23467"><p>De-Wu Ding and Long Ying. Revisiting modules (and centers) in &lt;i&gt;staphylococcus aureus&lt;/i&gt; metabolic network with link clustering. <em>JOURNAL OF BIOLOGICAL SYSTEMS</em>, 20(1):57–66, MAR 2012. <a class="reference external" href="https://doi.org/10.1142/S021833901150032X">[DOI]</a>.</p></li>
<li id="id23468"><p>Alisdair R. Fernie and Mark Stitt. On the discordance of metabolomics with proteomics and transcriptomics: coping with increasing complexity in logic, chemistry, and network interactions. <em>PLANT PHYSIOLOGY</em>, 158(3):1139–1145, MAR 2012. <a class="reference external" href="https://doi.org/10.1104/pp.112.193235">[DOI]</a>.</p></li>
<li id="id23469"><p>Il-Kwon Kim, Antonio Roldao, Verena Siewers, and Jens Nielsen. A systems-level approach for metabolic engineering of yeast cell factories. <em>FEMS YEAST RESEARCH</em>, 12(2):228–248, MAR 2012. <a class="reference external" href="https://doi.org/10.1111/j.1567-1364.2011.00779.x">[DOI]</a>.</p></li>
<li id="id23470"><p>Elias W. Krumholz, Hong Yang, Pamela Weisenhorn, Christopher S. Henry, and Igor G. L. Libourel. Genome-wide metabolic network reconstruction of the picoalga &lt;i&gt;ostreococcus&lt;/i&gt;. <em>JOURNAL OF EXPERIMENTAL BOTANY</em>, 63(6, SI):2353–2362, MAR 2012. <a class="reference external" href="https://doi.org/10.1093/jxb/err407">[DOI]</a>.</p></li>
<li id="id23471"><p>Marnix H. Medema, Renske van Raaphorst, Eriko Takano, and Rainer Breitling. Computational tools for the synthetic design of biochemical pathways. <em>NATURE REVIEWS MICROBIOLOGY</em>, 10(3):191–202, MAR 2012. <a class="reference external" href="https://doi.org/10.1038/nrmicro2717">[DOI]</a>.</p></li>
<li id="id23472"><p>Sarah A. Munro, Leila Choe, Stephen H. Zinder, Kelvin H. Lee, and Larry P. Walker. Proteomic and physiological experiments to test thermotoga neapolitana constraint-based model hypotheses of carbon source utilization. <em>BIOTECHNOLOGY PROGRESS</em>, 28(2):312–318, MAR-APR 2012. <a class="reference external" href="https://doi.org/10.1002/btpr.735">[DOI]</a>.</p></li>
<li id="id23473"><p>Ralf Steuer, Henning Knoop, and Rainer Machne. Modelling cyanobacteria: from metabolism to integrative models of phototrophic growth. <em>JOURNAL OF EXPERIMENTAL BOTANY</em>, 63(6, SI):2259–2274, MAR 2012. <a class="reference external" href="https://doi.org/10.1093/jxb/ers018">[DOI]</a>.</p></li>
<li id="id23474"><p>Balaji Balagurunathan, Sudhakar Jonnalagadda, Lily Tan, and Rajagopalan Srinivasan. Reconstruction and analysis of a genome-scale metabolic model for &lt;i&gt;scheffersomyces stipitis&lt;/i&gt;. <em>MICROBIAL CELL FACTORIES</em>, FEB 23 2012. <a class="reference external" href="https://doi.org/10.1186/1475-2859-11-27">[DOI]</a>.</p></li>
<li id="id23475"><p>Juan Nogales, Steinn Gudmundsson, Eric M. Knight, Bernhard O. Palsson, and Ines Thiele. Detailing the optimality of photosynthesis in cyanobacteria through systems biology analysis. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 109(7):2678–2683, FEB 14 2012. <a class="reference external" href="https://doi.org/10.1073/pnas.1117907109">[DOI]</a>.</p></li>
<li id="id23476"><p>Matthew N. Benedict, Matthew C. Gonnerman, William W. Metcalf, and Nathan D. Price. Genome-scale metabolic reconstruction and hypothesis testing in the methanogenic archaeon &lt;i&gt;methanosarcina acetivorans&lt;/i&gt; c2a. <em>JOURNAL OF BACTERIOLOGY</em>, 194(4):855–865, FEB 2012. <a class="reference external" href="https://doi.org/10.1128/JB.06040-11">[DOI]</a>.</p></li>
<li id="id23477"><p>A. Bordbar and B. O. Palsson. Using the reconstructed genome-scale human metabolic network to study physiology and pathology. <em>JOURNAL OF INTERNAL MEDICINE</em>, 271(2):131–141, FEB 2012. <a class="reference external" href="https://doi.org/10.1111/j.1365-2796.2011.02494.x">[DOI]</a>.</p></li>
<li id="id23478"><p>Mario Latendresse, Markus Krummenacker, Miles Trupp, and Peter D. Karp. Construction and completion of flux balance models from pathway databases. <em>BIOINFORMATICS</em>, 28(3):388–396, FEB 1 2012. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btr681">[DOI]</a>.</p></li>
<li id="id23479"><p>Liram Vardi, Eytan Ruppin, and Roded Sharan. A linearized constraint-based approach for modeling signaling networks. <em>JOURNAL OF COMPUTATIONAL BIOLOGY</em>, 19(2, SI):232–240, FEB 2012. Joint RECOMB Conference on Systems Biology, Regulatory Genomics/DREAM Challenges, Barcelona, SPAIN, OCT 14-19, 2011. <a class="reference external" href="https://doi.org/10.1089/cmb.2011.0277">[DOI]</a>.</p></li>
<li id="id23480"><p>Hansheng Zhao, Mao Li, Kechi Fang, Wenfeng Chen, and Jing Wang. &lt;i&gt;in silico&lt;/i&gt; insights into the symbiotic nitrogen fixation in &lt;i&gt;sinorhizobium meliloti&lt;/i&gt; via metabolic reconstruction. <em>PLOS ONE</em>, FEB 1 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0031287">[DOI]</a>.</p></li>
<li id="id23481"><p>Joost Boele, Brett G. Olivier, and Bas Teusink. Fame, the flux analysis and modeling environment. <em>BMC SYSTEMS BIOLOGY</em>, JAN 30 2012. <a class="reference external" href="https://doi.org/10.1186/1752-0509-6-8">[DOI]</a>.</p></li>
<li id="id23482"><p>Eugeni Belda, Francisco J. Silva, Juli Pereto, and Andres Moya. Metabolic networks of &lt;i&gt;sodalis glossinidius&lt;/i&gt;: a systems biology approach to reductive evolution. <em>PLOS ONE</em>, JAN 24 2012. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0030652">[DOI]</a>.</p></li>
<li id="id23483"><p>Carmen Maria Gonzalez-Domenech, Eugeni Belda, Rafael Patino-Navarrete, Andres Moya, Juli Pereto, and Amparo Latorre. Metabolic stasis in an ancient symbiosis: genome-scale metabolic networks from two &lt;i&gt;blattabacterium cuenoti&lt;/i&gt; strains, primary endosymbionts of cockroaches. <em>BMC MICROBIOLOGY</em>, JAN 18 2012. <a class="reference external" href="https://doi.org/10.1186/1471-2180-12-S1-S5">[DOI]</a>.</p></li>
<li id="id23484"><p>Aline Metris, Susan George, and Jozsef Baranyi. Modelling osmotic stress by flux balance analysis at the genomic scale. <em>INTERNATIONAL JOURNAL OF FOOD MICROBIOLOGY</em>, 152(3, SI):123–128, JAN 16 2012. 22nd Symposium of the International Committee on Food Microbiology and Hygiene (ICFMH) on Microbial Behaviour in the Food Chain (Food Micro), Copenhagen, DENMARK, AUG 30-SEP 03, 2010. <a class="reference external" href="https://doi.org/10.1016/j.ijfoodmicro.2011.06.016">[DOI]</a>.</p></li>
<li id="id23485"><p>R. M. T. Fleming, C. M. Maes, M. A. Saunders, Y. Ye, and B. O. Palsson. A variational principle for computing nonequilibrium fluxes and potentials in genome-scale biochemical networks. <em>JOURNAL OF THEORETICAL BIOLOGY</em>, 292:71–77, JAN 7 2012. <a class="reference external" href="https://doi.org/10.1016/j.jtbi.2011.09.029">[DOI]</a>.</p></li>
<li id="id23486"><p>Rajeev Surendran Assary, Linda J. Broadbelt, and Larry A. Curtiss. Bronsted-evans-polanyi relationships for c-c bond forming and c-c bond breaking reactions in thiamine-catalyzed decarboxylation of 2-keto acids using density functional theory. <em>JOURNAL OF MOLECULAR MODELING</em>, 18(1):145–150, JAN 2012. <a class="reference external" href="https://doi.org/10.1007/s00894-011-1062-z">[DOI]</a>.</p></li>
<li id="id23487"><p>Thomas Dandekar, Astrid Fieselmann, Jasmin Popp, and Michael Hensel. &lt;i&gt;salmonella enterica&lt;/i&gt;: a surprisingly well-adapted intracellular lifestyle. <em>FRONTIERS IN MICROBIOLOGY</em>, 2012. <a class="reference external" href="https://doi.org/10.3389/fmicb.2012.00164">[DOI]</a>.</p></li>
<li id="id23488"><p>Ziomara P. Gerdtzen. Modeling metabolic networks for mammalian cell systems: general considerations, modeling strategies, and available tools. In WS Hu and AP Zeng, editors, <em>GENOMICS AND SYSTEMS BIOLOGY OF MAMMALIAN CELL CULTURE</em>, volume 127 of Advances in Biochemical Engineering-Biotechnology, pages 71–108. 2012. <a class="reference external" href="https://doi.org/10.1007/10\_2011\_120">[DOI]</a>.</p></li>
<li id="id23489"><p>Joonhoon Kim and Jennifer L. Reed. Relatch: relative optimality in metabolic networks explains robust metabolic and regulatory responses to perturbations. <em>GENOME BIOLOGY</em>, 2012. <a class="reference external" href="https://doi.org/10.1186/gb-2012-13-9-r78">[DOI]</a>.</p></li>
<li id="id23490"><p>Jing Ma, Xun Zhang, Choong Yong Ung, Yu Zong Chen, and Baowen Li. Metabolic network analysis revealed distinct routes of deletion effects between essential and non-essential genes. <em>MOLECULAR BIOSYSTEMS</em>, 8(4):1179–1186, 2012. <a class="reference external" href="https://doi.org/10.1039/c2mb05376d">[DOI]</a>.</p></li>
<li id="id23491"><p>Swagatika Sahoo, Leifur Franzson, Jon J. Jonsson, and Ines Thiele. A compendium of inborn errors of metabolism mapped onto the human metabolic network. <em>MOLECULAR BIOSYSTEMS</em>, 8(10):2545–2558, 2012. <a class="reference external" href="https://doi.org/10.1039/c2mb25075f">[DOI]</a>.</p></li>
<li id="id23492"><p>Andreas Wagner. Metabolic networks and their evolution. In OS Soyer, editor, <em>EVOLUTIONARY SYSTEMS BIOLOGY</em>, volume 751 of Advances in Experimental Medicine and Biology, pages 29–52. 2012. <a class="reference external" href="https://doi.org/10.1007/978-1-4614-3567-9\_2">[DOI]</a>.</p></li>
<li id="id23493"><p>Lei Xie, Li Xie, Sarah L. Kinnings, and Philip E. Bourne. Novel computational approaches to polypharmacology as a means to define responses to individual drugs. In PA Insel, SG Amara, and TF Blaschke, editors, <em>ANNUAL REVIEW OF PHARMACOLOGY AND TOXICOLOGY, VOL 52</em>, volume 52 of Annual Review of Pharmacology and Toxicology, pages 361+. 2012. <a class="reference external" href="https://doi.org/10.1146/annurev-pharmtox-010611-134630">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-13" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-13" name="0-13" role="tabpanel" tabindex="0"><div class="docutils container" id="id23661">
<ol class="arabic simple" start="1">
<li id="id25314"><p>Rajeev S. Assary and Linda J. Broadbelt. 2-keto acids to branched-chain alcohols as biofuels: application of reaction network analysis and high-level quantum chemical methods to understand thermodynamic landscapes. <em>COMPUTATIONAL AND THEORETICAL CHEMISTRY</em>, 978(1-3):160–165, DEC 30 2011. <a class="reference external" href="https://doi.org/10.1016/j.comptc.2011.10.009">[DOI]</a>.</p></li>
<li id="id25315"><p>Cristiana Gomes de Oliveira Dal'Molin, Lake-Ee Quek, Robin W. Palfreyman, and Lars K. Nielsen. Algagem - a genome-scale metabolic reconstruction of algae based on the &lt;i&gt;chlamydomonas reinhardtii&lt;/i&gt; genome. <em>BMC GENOMICS</em>, DEC 22 2011. 6th International Conference of the Brazilian-Association-for-Bioinformatics-and-Computational-Biology (X-meeting), Ouro Preto, BRAZIL, NOV 15-18, 2010. <a class="reference external" href="https://doi.org/10.1186/1471-2164-12-S4-S5">[DOI]</a>.</p></li>
<li id="id25316"><p>Neema Jamshidi, Franklin J. Miller, Jess Mandel, Timothy Evans, and Michael D. Kuo. Individualized therapy of hht driven by network analysis of metabolomic profiles. <em>BMC SYSTEMS BIOLOGY</em>, DEC 20 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-200">[DOI]</a>.</p></li>
<li id="id25317"><p>Yuqi Zhao and Jingfei Huang. Reconstruction and analysis of human heart-specific metabolic network based on transcriptome and proteome data. <em>BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS</em>, 415(3):450–454, NOV 25 2011. <a class="reference external" href="https://doi.org/10.1016/j.bbrc.2011.10.090">[DOI]</a>.</p></li>
<li id="id25318"><p>Amit Ghosh, Huimin Zhao, and Nathan D. Price. Genome-scale consequences of cofactor balancing in engineered pentose utilization pathways in &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;. <em>PLOS ONE</em>, NOV 4 2011. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0027316">[DOI]</a>.</p></li>
<li id="id25319"><p>Juha T. Leppavuori, Michael M. Domach, and Lorenz T. Biegler. Parameter estimation in batch bioreactor simulation using metabolic models: sequential solution with direct sensitivities. <em>INDUSTRIAL &amp; ENGINEERING CHEMISTRY RESEARCH</em>, 50(21):12080–12091, NOV 2 2011. <a class="reference external" href="https://doi.org/10.1021/ie201020g">[DOI]</a>.</p></li>
<li id="id25320"><p>David J. Baumler, Roman G. Peplinski, Jennifer L. Reed, Jeremy D. Glasner, and Nicole T. Perna. The evolution of metabolic networks of &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, NOV 1 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-182">[DOI]</a>.</p></li>
<li id="id25321"><p>R. Lencastre Fernandes, M. Nierychlo, L. Lundin, A. E. Pedersen, P. E. Puentes Tellez, A. Dutta, M. Carlquist, A. Bolic, D. Schapper, A. C. Brunetti, S. Helmark, A. -L Heins, A. D. Jensen, I. Nopens, K. Rottwitt, N. Szita, J. D. van Elsas, P. H. Nielsen, J. Martinussen, S. J. Sorensen, A. E. Lantz, and K. V. Gernaey. Experimental methods and modeling techniques for description of cell population heterogeneity. <em>BIOTECHNOLOGY ADVANCES</em>, 29(6):575–599, NOV-DEC 2011. <a class="reference external" href="https://doi.org/10.1016/j.biotechadv.2011.03.007">[DOI]</a>.</p></li>
<li id="id25322"><p>Aline Metris, Mark Reuter, Duncan J. H. Gaskin, Jozsef Baranyi, and Arnoud H. M. van Vliet. &lt;i&gt;in vivo&lt;/i&gt; and &lt;i&gt;in silico&lt;/i&gt; determination of essential genes of &lt;i&gt;campylobacter jejuni&lt;/i&gt;. <em>BMC GENOMICS</em>, NOV 1 2011. <a class="reference external" href="https://doi.org/10.1186/1471-2164-12-535">[DOI]</a>.</p></li>
<li id="id25323"><p>Ali Navid. Applications of system-level models of metabolism for analysis of bacterial physiology and identification of new drug targets. <em>BRIEFINGS IN FUNCTIONAL GENOMICS</em>, 10(6, SI):354–364, NOV 2011. <a class="reference external" href="https://doi.org/10.1093/bfgp/elr034">[DOI]</a>.</p></li>
<li id="id25324"><p>Aarash Bordbar, Adam M. Feist, Renata Usaite-Black, Joseph Woodcock, Bernhard O. Palsson, and Iman Famili. A multi-tissue type genome-scale metabolic network for analysis of whole-body systems physiology. <em>BMC SYSTEMS BIOLOGY</em>, OCT 31 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-180">[DOI]</a>.</p></li>
<li id="id25325"><p>Eleftheria Tzamali, Panayiota Poirazi, Ioannis G. Tollis, and Martin Reczko. A computational exploration of bacterial metabolic diversity identifying metabolic interactions and growth-efficient strain communities. <em>BMC SYSTEMS BIOLOGY</em>, OCT 18 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-167">[DOI]</a>.</p></li>
<li id="id25326"><p>Pep Charusanti, Sadhana Chauhan, Kathleen McAteer, Joshua A. Lerman, Daniel R. Hyduke, Vladimir L. Motin, Charles Ansong, Joshua N. Adkins, and Bernhard O. Palsson. An experimentally-supported genome-scale metabolic network reconstruction for &lt;i&gt;yersinia pestis&lt;/i&gt; co92. <em>BMC SYSTEMS BIOLOGY</em>, OCT 13 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-163">[DOI]</a>.</p></li>
<li id="id25327"><p>Aditya V. Pandit and Radhakrishnan Mahadevan. &lt;i&gt;in silico&lt;/i&gt; characterization of microbial electrosynthesis for metabolic engineering of biochemicals. <em>MICROBIAL CELL FACTORIES</em>, OCT 3 2011. <a class="reference external" href="https://doi.org/10.1186/1475-2859-10-76">[DOI]</a>.</p></li>
<li id="id25328"><p>Christopher S. Henry, Ross Overbeek, Fangfang Xia, Aaron A. Best, Elizabeth Glass, Jack Gilbert, Peter Larsen, Rob Edwards, Terry Disz, Folker Meyer, Veronika Vonstein, Matthew DeJongh, Daniela Bartels, Narayan Desai, Mark D'Souza, Scott Devoid, Kevin P. Keegan, Robert Olson, Andreas Wilke, Jared Wilkening, and Rick L. Stevens. Connecting genotype to phenotype in the era of high-throughput sequencing. <em>BIOCHIMICA ET BIOPHYSICA ACTA-GENERAL SUBJECTS</em>, 1810(10, SI):967–977, OCT 2011. <a class="reference external" href="https://doi.org/10.1016/j.bbagen.2011.03.010">[DOI]</a>.</p></li>
<li id="id25329"><p>Ehsan Motamedian and Fereshteh Naeimpoor. Prediction of proton exchange and bacterial growth on various substrates using constraint-based modeling approach. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 16(5):875–884, OCT 2011. <a class="reference external" href="https://doi.org/10.1007/s12257-011-0115-6">[DOI]</a>.</p></li>
<li id="id25330"><p>Jeffrey D. Orth, Tom M. Conrad, Jessica Na, Joshua A. Lerman, Hojung Nam, Adam M. Feist, and Bernhard O. Palsson. A comprehensive genome-scale reconstruction of &lt;i&gt;escherichia coli&lt;/i&gt; metabolism-2011. <em>MOLECULAR SYSTEMS BIOLOGY</em>, OCT 2011. <a class="reference external" href="https://doi.org/10.1038/msb.2011.65">[DOI]</a>.</p></li>
<li id="id25331"><p>Ottar Rolfsson, Bernhard O. Palsson, and Ines Thiele. The human metabolic reconstruction recon 1 directs hypotheses of novel human metabolic functions. <em>BMC SYSTEMS BIOLOGY</em>, OCT 1 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-155">[DOI]</a>.</p></li>
<li id="id25332"><p>Paul A. Jensen, Kyla A. Lutz, and Jason A. Papin. Tiger: toolbox for integrating genome-scale metabolic models, expression data, and transcriptional regulatory networks. <em>BMC SYSTEMS BIOLOGY</em>, SEP 23 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-147">[DOI]</a>.</p></li>
<li id="id25333"><p>Marija Cvijovic, Sergio Bordel, and Jens Nielsen. Mathematical models of cell factories: moving towards the core of industrial biotechnology. <em>MICROBIAL BIOTECHNOLOGY</em>, 4(5):572–584, SEP 2011. <a class="reference external" href="https://doi.org/10.1111/j.1751-7915.2010.00233.x">[DOI]</a>.</p></li>
<li id="id25334"><p>Jan Schellenberger, Richard Que, Ronan M. T. Fleming, Ines Thiele, Jeffrey D. Orth, Adam M. Feist, Daniel C. Zielinski, Aarash Bordbar, Nathan E. Lewis, Sorena Rahmanian, Joseph Kang, Daniel R. Hyduke, and Bernhard O. Palsson. Quantitative prediction of cellular metabolism with constraint-based models: the cobra toolbox v2.0. <em>NATURE PROTOCOLS</em>, 6(9):1290–1307, SEP 2011. <a class="reference external" href="https://doi.org/10.1038/nprot.2011.308">[DOI]</a>.</p></li>
<li id="id25335"><p>Caroline B. Milne, James A. Eddy, Ravali Raju, Soroush Ardekani, Pan-Jun Kim, Ryan S. Senger, Yong-Su Jin, Hans P. Blaschek, and Nathan D. Price. Metabolic network reconstruction and genome-scale model of butanol-producing strain &lt;i&gt;clostridium beijerinckii&lt;/i&gt; ncimb 8052. <em>BMC SYSTEMS BIOLOGY</em>, AUG 16 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-130">[DOI]</a>.</p></li>
<li id="id25336"><p>Siu Hung Joshua Chan and Ping Ji. Decomposing flux distributions into elementary flux modes in genome-scale metabolic networks. <em>BIOINFORMATICS</em>, 27(16):2256–2262, AUG 15 2011. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btr367">[DOI]</a>.</p></li>
<li id="id25337"><p>Pablo Carbonell, Anne-Gaelle Planson, Davide Fichera, and Jean-Loup Faulon. A retrosynthetic biology approach to metabolic pathway design for therapeutic production. <em>BMC SYSTEMS BIOLOGY</em>, AUG 5 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-122">[DOI]</a>.</p></li>
<li id="id25338"><p>Roger L. Chang, Lila Ghamsari, Ani Manichaikul, Erik F. Y. Hom, Santhanam Balaji, Weiqi Fu, Yun Shen, Tong Hao, Bernhard O. Palsson, Kourosh Salehi-Ashtiani, and Jason A. Papin. Metabolic network reconstruction of &lt;i&gt;chlamydomonas&lt;/i&gt; offers insight into light-driven algal metabolism. <em>MOLECULAR SYSTEMS BIOLOGY</em>, AUG 2011. <a class="reference external" href="https://doi.org/10.1038/msb.2011.52">[DOI]</a>.</p></li>
<li id="id25339"><p>Karoline Faust, Didier Croes, and Jacques van Helden. Prediction of metabolic pathways from genome-scale metabolic networks. <em>BIOSYSTEMS</em>, 105(2, SI):109–121, AUG 2011. Workshop on Integration of OMICS Datasets into Metabolic Pathway Analysis (IOMPA 2010), Edinburgh, SCOTLAND, OCT 15, 2010. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2011.05.004">[DOI]</a>.</p></li>
<li id="id25340"><p>Jordan Hay and Joerg Schwender. Computational analysis of storage synthesis in developing &lt;i&gt;brassica napus&lt;/i&gt; l. (oilseed rape) embryos: flux variability analysis in relation to &lt;sup&gt;13&lt;/sup&gt;c metabolic flux analysis. <em>PLANT JOURNAL</em>, 67(3):513–525, AUG 2011. <a class="reference external" href="https://doi.org/10.1111/j.1365-313X.2011.04611.x">[DOI]</a>.</p></li>
<li id="id25341"><p>Jordan Hay and Joerg Schwender. Metabolic network reconstruction and flux variability analysis of storage synthesis in developing oilseed rape (&lt;i&gt;brassica napus&lt;/i&gt; l.) embryos. <em>PLANT JOURNAL</em>, 67(3):526–541, AUG 2011. <a class="reference external" href="https://doi.org/10.1111/j.1365-313X.2011.04613.x">[DOI]</a>.</p></li>
<li id="id25342"><p>Steffen Klamt and Axel von Kamp. An application programming interface for &lt;i&gt;cellnetanalyzer&lt;/i&gt;. <em>BIOSYSTEMS</em>, 105(2, SI):162–168, AUG 2011. Workshop on Integration of OMICS Datasets into Metabolic Pathway Analysis (IOMPA 2010), Edinburgh, SCOTLAND, OCT 15, 2010. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2011.02.002">[DOI]</a>.</p></li>
<li id="id25343"><p>Yoshihiro Toya, Nobuaki Kono, Kazuharu Arakawa, and Masaru Tomita. Metabolic flux analysis and visualization. <em>JOURNAL OF PROTEOME RESEARCH</em>, 10(8):3313–3323, AUG 2011. <a class="reference external" href="https://doi.org/10.1021/pr2002885">[DOI]</a>.</p></li>
<li id="id25344"><p>Daniela Xavier, Sara Vazquez, Clara Higuera, Federico Moran, and Francisco Montero. Tools-4-metatool (t4m): online suite of web-tools to process stoichiometric network analysis data from metatool. <em>BIOSYSTEMS</em>, 105(2, SI):169–172, AUG 2011. Workshop on Integration of OMICS Datasets into Metabolic Pathway Analysis (IOMPA 2010), Edinburgh, SCOTLAND, OCT 15, 2010. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2011.04.004">[DOI]</a>.</p></li>
<li id="id25345"><p>Osbaldo Resendis-Antonio, Magdalena Hernandez, Emmanuel Salazar, Sandra Contreras, Gabriel Martinez Batallar, Yolanda Mora, and Sergio Encarnacion. Systems biology of bacterial nitrogen fixation: high-throughput technology and its integrative description with constraint-based modeling. <em>BMC SYSTEMS BIOLOGY</em>, JUL 29 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-120">[DOI]</a>.</p></li>
<li id="id25346"><p>Mohammad Tauqeer Alam, Marnix H. Medema, Eriko Takano, and Rainer Breitling. Comparative genome-scale metabolic modeling of actinomycetes: the topology of essential core metabolism. <em>FEBS LETTERS</em>, 585(14):2389–2394, JUL 21 2011. <a class="reference external" href="https://doi.org/10.1016/j.febslet.2011.06.014">[DOI]</a>.</p></li>
<li id="id25347"><p>Aarash Bordbar, Neema Jamshidi, and Bernhard O. Palsson. Iab-rbc-283: a proteomically derived knowledge-base of erythrocyte metabolism that can be used to simulate its physiological and patho-physiological states. <em>BMC SYSTEMS BIOLOGY</em>, JUL 12 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-110">[DOI]</a>.</p></li>
<li id="id25348"><p>Stephan Noack, Katharina Noeh, Matthias Moch, Marco Oldiges, and Wolfgang Wiechert. Stationary versus non-stationary &lt;sup&gt;13&lt;/sup&gt;c-mfa: a comparison using a consistent dataset. <em>JOURNAL OF BIOTECHNOLOGY</em>, 154(2-3):179–190, JUL 10 2011. <a class="reference external" href="https://doi.org/10.1016/j.jbiotec.2010.07.008">[DOI]</a>.</p></li>
<li id="id25349"><p>Tom M. Conrad, Nathan E. Lewis, and Bernhard O. Palsson. Microbial laboratory evolution in the era of genome-scale science. <em>MOLECULAR SYSTEMS BIOLOGY</em>, JUL 2011. <a class="reference external" href="https://doi.org/10.1038/msb.2011.42">[DOI]</a>.</p></li>
<li id="id25350"><p>J. Bernadette Moore and Mark E. Weeks. Proteomics and systems biology: current and future applications in the nutritional sciences. <em>ADVANCES IN NUTRITION</em>, 2(4):355–364, JUL 2011. <a class="reference external" href="https://doi.org/10.3945/an.111.000554">[DOI]</a>.</p></li>
<li id="id25351"><p>Eleftherios Pilalis, Aristotelis Chatziioannou, Brigitte Thomasset, and Fragiskos Kolisis. An in silico compartmentalized metabolic model of &lt;i&gt;brassica napus&lt;/i&gt; enables the systemic study of regulatory aspects of plant central metabolism. <em>BIOTECHNOLOGY AND BIOENGINEERING</em>, 108(7):1673–1682, JUL 2011. <a class="reference external" href="https://doi.org/10.1002/bit.23107">[DOI]</a>.</p></li>
<li id="id25352"><p>Mona Yousofshahi, Kyongbum Lee, and Soha Hassoun. Probabilistic pathway construction. <em>METABOLIC ENGINEERING</em>, 13(4):435–444, JUL 2011. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2011.01.006">[DOI]</a>.</p></li>
<li id="id25353"><p>Anthony C. Smith and Alan J. Robinson. A metabolic model of the mitochondrion and its use in modelling diseases of the tricarboxylic acid cycle. <em>BMC SYSTEMS BIOLOGY</em>, JUN 29 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-102">[DOI]</a>.</p></li>
<li id="id25354"><p>Hailin Meng, Zhiguo Lu, Yong Wang, Xiaoning Wang, and Siliang Zhang. In silico improvement of heterologous biosynthesis of erythromycin precursor 6-deoxyerythronolide b in escherichia coli. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 16(3):445–456, JUN 2011. <a class="reference external" href="https://doi.org/10.1007/s12257-010-0321-7">[DOI]</a>.</p></li>
<li id="id25355"><p>Wolfram Weckwerth. Unpredictability of metabolism-the key role of metabolomics science in combination with next-generation genome sequencing. <em>ANALYTICAL AND BIOANALYTICAL CHEMISTRY</em>, 400(7):1967–1978, JUN 2011. <a class="reference external" href="https://doi.org/10.1007/s00216-011-4948-9">[DOI]</a>.</p></li>
<li id="id25356"><p>Suvi Santala, Elena Efimova, Virpi Kivinen, Antti Larjo, Tommi Aho, Matti Karp, and Ville Santala. Improved triacylglycerol production in &lt;i&gt;acinetobacter baylyi&lt;/i&gt; adp1 by metabolic engineering. <em>MICROBIAL CELL FACTORIES</em>, MAY 18 2011. <a class="reference external" href="https://doi.org/10.1186/1475-2859-10-36">[DOI]</a>.</p></li>
<li id="id25357"><p>Guillaume Cogne, Marco Ruegen, Alexander Bockmayr, Mariana Titica, Claude-Gilles Dussap, Jean-Francois Cornet, and Jack Legrand. A model-based method for investigating bioenergetic processes in autotrophically growing eukaryotic microalgae: application to the green algae &lt;i&gt;chlamydomonas reinhardtii&lt;/i&gt;. <em>BIOTECHNOLOGY PROGRESS</em>, 27(3):631–640, MAY-JUN 2011. <a class="reference external" href="https://doi.org/10.1002/btpr.596">[DOI]</a>.</p></li>
<li id="id25358"><p>Andrew L. Hopkins, G. Richard Bickerton, Ian M. Carruthers, Stephen K. Boyer, Harvey Rubin, and John P. Overington. Rapid analysis of pharmacology for infectious diseases. <em>CURRENT TOPICS IN MEDICINAL CHEMISTRY</em>, 11(10):1292–1300, MAY 2011.</p></li>
<li id="id25359"><p>S. J. MacDonald, G. H. Thomas, and A. E. Douglas. Genetic and metabolic determinants of nutritional phenotype in an insect-bacterial symbiosis. <em>MOLECULAR ECOLOGY</em>, 20(10):2073–2084, MAY 2011. <a class="reference external" href="https://doi.org/10.1111/j.1365-294X.2011.05031.x">[DOI]</a>.</p></li>
<li id="id25360"><p>Zhi Wang and Jianzhi Zhang. Impact of gene expression noise on organismal fitness and the efficacy of natural selection. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 108(16):E67–E76, APR 19 2011. <a class="reference external" href="https://doi.org/10.1073/pnas.1100059108">[DOI]</a>.</p></li>
<li id="id25361"><p>Yu-Chieh Liao, Tzu-Wen Huang, Feng-Chi Chen, Pep Charusanti, Jay S. J. Hong, Hwan-You Chang, Shih-Feng Tsai, Bernhard O. Palsson, and Chao A. Hsiung. An experimentally validated genome-scale metabolic reconstruction of &lt;i&gt;klebsiella pneumoniae&lt;/i&gt; mgh 78578, &lt;i&gt;i&lt;/i&gt;yl1228. <em>JOURNAL OF BACTERIOLOGY</em>, 193(7):1710–1717, APR 2011. <a class="reference external" href="https://doi.org/10.1128/JB.01218-10">[DOI]</a>.</p></li>
<li id="id25362"><p>Rajeev S. Assary and Linda J. Broadbelt. Computational screening of novel thiamine-catalyzed decarboxylation reactions of 2-keto acids. <em>BIOPROCESS AND BIOSYSTEMS ENGINEERING</em>, 34(3):375–388, MAR 2011. <a class="reference external" href="https://doi.org/10.1007/s00449-010-0481-z">[DOI]</a>.</p></li>
<li id="id25363"><p>Hailin Meng, Yong Wang, Qiang Hua, Siliang Zhang, and Xiaoning Wang. &lt;i&gt;in silico&lt;/i&gt; analysis and experimental improvement of taxadiene heterologous biosynthesis in &lt;i&gt;escherichia coli&lt;/i&gt;. <em>BIOTECHNOLOGY AND BIOPROCESS ENGINEERING</em>, 16(2):205–215, MAR-APR 2011. <a class="reference external" href="https://doi.org/10.1007/s12257-010-0329-z">[DOI]</a>.</p></li>
<li id="id25364"><p>Matthew A. Oberhardt, Jacek Puchalka, Vitor A. P. Martins dos Santos, and Jason A. Papin. Reconciliation of genome-scale metabolic reconstructions for comparative systems analysis. <em>PLOS COMPUTATIONAL BIOLOGY</em>, MAR 2011. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1001116">[DOI]</a>.</p></li>
<li id="id25365"><p>Paulo Vilaca, Isabel Rocha, and Miguel Rocha. A computational tool for the simulation and optimization of microbial strains accounting integrated metabolic/regulatory information. <em>BIOSYSTEMS</em>, 103(3):435–441, MAR 2011. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2010.11.012">[DOI]</a>.</p></li>
<li id="id25366"><p>Yanping Xi, Yi-Ping Phoebe Chen, Chen Qian, and Fei Wang. Comparative study of computational methods to detect the correlated reaction sets in biochemical networks. <em>BRIEFINGS IN BIOINFORMATICS</em>, 12(2):132–150, MAR 2011. <a class="reference external" href="https://doi.org/10.1093/bib/bbp068">[DOI]</a>.</p></li>
<li id="id25367"><p>Oksana Sorokina, Florence Corellou, David Dauvillee, Anatoly Sorokin, Igor Goryanin, Steven Ball, Francois-Yves Bouget, and Andrew J. Millar. Microarray data can predict diurnal changes of starch content in the picoalga &lt;i&gt;ostreococcus&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, FEB 26 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-36">[DOI]</a>.</p></li>
<li id="id25368"><p>Sean P. Cornelius, Joo Sang Lee, and Adilson E. Motter. Dispensability of &lt;i&gt;escherichia coli&lt;/i&gt;'&lt;i&gt;s&lt;/i&gt; latent pathways. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 108(8):3124–3129, FEB 22 2011. <a class="reference external" href="https://doi.org/10.1073/pnas.1009772108">[DOI]</a>.</p></li>
<li id="id25369"><p>Stephan Pabinger, Robert Rader, Rasmus Agren, Jens Nielsen, and Zlatko Trajanoski. Memosys: bioinformatics platform for genome-scale metabolic models. <em>BMC SYSTEMS BIOLOGY</em>, JAN 31 2011. <a class="reference external" href="https://doi.org/10.1186/1752-0509-5-20">[DOI]</a>.</p></li>
<li id="id25370"><p>Andreas Hoppe, Sabrina Hoffmann, Andreas Gerasch, Christoph Gille, and Hermann-Georg Holzhuetter. Fasimu: flexible software for flux-balance computation series in large metabolic networks. <em>BMC BIOINFORMATICS</em>, JAN 22 2011. <a class="reference external" href="https://doi.org/10.1186/1471-2105-12-28">[DOI]</a>.</p></li>
<li id="id25371"><p>Colin T. Archer, Jihyun F. Kim, Haeyoung Jeong, Jin Hwan Park, Claudia E. Vickers, Sang Yup Lee, and Lars K. Nielsen. The genome sequence of &lt;i&gt;e. coli&lt;/i&gt; w (atcc 9637): comparative genome analysis and an improved genome-scale reconstruction of &lt;i&gt;e. coli&lt;/i&gt;. <em>BMC GENOMICS</em>, JAN 6 2011. <a class="reference external" href="https://doi.org/10.1186/1471-2164-12-9">[DOI]</a>.</p></li>
<li id="id25372"><p>Xuewen Chen, Ana P. Alonso, Doug K. Allen, Jennifer L. Reed, and Yair Shachar-Hill. Synergy between &lt;sup&gt;13&lt;/sup&gt;c-metabolic flux analysis and flux balance analysis for understanding metabolic adaption to anaerobiosis in &lt;i&gt;e&lt;/i&gt;. &lt;i&gt;coli&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 13(1):38–48, JAN 2011. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2010.11.004">[DOI]</a>.</p></li>
<li id="id25373"><p>Reza Eslamloueyan and Payam Setoodeh. Optimization of fed-batch recombinant yeast fermentation for ethanol production using a reduced dynamic flux balance model based on artificial neural networks. <em>CHEMICAL ENGINEERING COMMUNICATIONS</em>, 198(11):1309–1338, 2011. <a class="reference external" href="https://doi.org/10.1080/00986445.2011.560512">[DOI]</a>.</p></li>
<li id="id25374"><p>Xin Fang, Anders Wallqvist, and Jaques Reifman. Modeling synergistic drug inhibition of &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; growth in murine macrophages. <em>MOLECULAR BIOSYSTEMS</em>, 7(9):2622–2636, 2011. <a class="reference external" href="https://doi.org/10.1039/c1mb05106g">[DOI]</a>.</p></li>
<li id="id25375"><p>Ronan M. T. Fleming and Ines Thiele. Von bertalanffy 1.0: a cobra toolbox extension to thermodynamically constrain metabolic models. <em>BIOINFORMATICS</em>, 27(1):142–143, JAN 2011. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btq607">[DOI]</a>.</p></li>
<li id="id25376"><p>Charles R. Haggart, Jennifer A. Bartell, Jeffrey J. Saucerman, and Jason A. Papin. Whole-genome metabolic network reconstruction and constraint-based modeling. In D Jameson, M Verma, and HV Westerhoff, editors, <em>METHODS IN ENZYMOLOGY, VOL 500: METHODS IN SYSTEMS BIOLOGY</em>, volume 500 of Methods in Enzymology, pages 411–433. 2011. <a class="reference external" href="https://doi.org/10.1016/B978-0-12-385118-5.00021-9">[DOI]</a>.</p></li>
<li id="id25377"><p>Tom A. Mendum, Jane Newcombe, Ahmad A. Mannan, Andrzej M. Kierzek, and Johnjoe McFadden. Interrogation of global mutagenesis data with a genome scale model of &lt;i&gt;neisseria meningitidis&lt;/i&gt; to assess gene fitness &lt;i&gt;in vitro&lt;/i&gt; and in sera. <em>GENOME BIOLOGY</em>, 2011. <a class="reference external" href="https://doi.org/10.1186/gb-2011-12-12-r127">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-14" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-14" name="0-14" role="tabpanel" tabindex="0"><div class="docutils container" id="id25481">
<ol class="arabic simple" start="1">
<li id="id27198"><p>Sergey Kozhenkov, Yulia Dubinina, Mayya Sedova, Amarnath Gupta, Julia Ponomarenko, and Michael Baitaluk. Biologicalnetworks 2.0-an integrative view of genome biology data. <em>BMC BIOINFORMATICS</em>, DEC 29 2010. <a class="reference external" href="https://doi.org/10.1186/1471-2105-11-610">[DOI]</a>.</p></li>
<li id="id27199"><p>Xin Fang, Anders Wallqvist, and Jaques Reifman. Development and analysis of an &lt;i&gt;in&lt;/i&gt; &lt;i&gt;vivo&lt;/i&gt;-compatible metabolic network of &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, NOV 23 2010. <a class="reference external" href="https://doi.org/10.1186/1752-0509-4-160">[DOI]</a>.</p></li>
<li id="id27200"><p>Sarah L. Kinnings, Li Xie, Kingston H. Fung, Richard M. Jackson, Lei Xie, and Philip E. Bourne. The &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; drugome and its polypharmacological implications. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2010. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1000976">[DOI]</a>.</p></li>
<li id="id27201"><p>Niels Klitgord and Daniel Segre. Environments that induce synthetic microbial ecosystems. <em>PLOS COMPUTATIONAL BIOLOGY</em>, NOV 2010. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1001002">[DOI]</a>.</p></li>
<li id="id27202"><p>Paul D. Dobson, Kieran Smallbone, Daniel Jameson, Evangelos Simeonidis, Karin Lanthaler, Pinar Pir, Chuan Lu, Neil Swainston, Warwick B. Dunn, Paul Fisher, Duncan Hull, Marie Brown, Olusegun Oshota, Natalie J. Stanford, Douglas B. Kell, Ross D. King, Stephen G. Oliver, Robert D. Stevens, and Pedro Mendes. Further developments towards a genome-scale metabolic model of yeast. <em>BMC SYSTEMS BIOLOGY</em>, OCT 28 2010. <a class="reference external" href="https://doi.org/10.1186/1752-0509-4-145">[DOI]</a>.</p></li>
<li id="id27203"><p>Martin I. Sigurdsson, Neema Jamshidi, Eirikur Steingrimsson, Ines Thiele, and Bernhard O. Palsson. A detailed genome-wide reconstruction of mouse metabolism based on human recon 1. <em>BMC SYSTEMS BIOLOGY</em>, OCT 19 2010. <a class="reference external" href="https://doi.org/10.1186/1752-0509-4-140">[DOI]</a>.</p></li>
<li id="id27204"><p>Sriram Chandrasekaran and Nathan D. Price. Probabilistic integrative modeling of genome-scale metabolic and regulatory networks in &lt;i&gt;escherichia&lt;/i&gt; &lt;i&gt;coli&lt;/i&gt; and &lt;i&gt;mycobacterium&lt;/i&gt; &lt;i&gt;tuberculosis&lt;/i&gt;. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 107(41):17845–17850, OCT 12 2010. <a class="reference external" href="https://doi.org/10.1073/pnas.1005139107">[DOI]</a>.</p></li>
<li id="id27205"><p>Limin Li, Xiaobo Zhou, Wai-Ki Ching, and Ping Wang. Predicting enzyme targets for cancer drugs by profiling human metabolic reactions in nci-60 cell lines. <em>BMC BIOINFORMATICS</em>, OCT 8 2010. <a class="reference external" href="https://doi.org/10.1186/1471-2105-11-501">[DOI]</a>.</p></li>
<li id="id27206"><p>Aarash Bordbar, Nathan E. Lewis, Jan Schellenberger, Bernhard O. Palsson, and Neema Jamshidi. Insight into human alveolar macrophage and &lt;i&gt;m&lt;/i&gt;. &lt;i&gt;tuberculosis&lt;/i&gt; interactions via metabolic reconstructions. <em>MOLECULAR SYSTEMS BIOLOGY</em>, OCT 2010. <a class="reference external" href="https://doi.org/10.1038/msb.2010.68">[DOI]</a>.</p></li>
<li id="id27207"><p>Matthew A. Oberhardt, Joanna B. Goldberg, Michael Hogardt, and Jason A. Papin. Metabolic network analysis of &lt;i&gt;pseudomonas aeruginosa&lt;/i&gt; during chronic cystic fibrosis lung infection. <em>JOURNAL OF BACTERIOLOGY</em>, 192(20):5534–5548, OCT 2010. <a class="reference external" href="https://doi.org/10.1128/JB.00900-10">[DOI]</a>.</p></li>
<li id="id27208"><p>Han Min Woo, Stephan Noack, Gerd M. Seibold, Sabine Willbold, Bernhard J. Eikmanns, and Michael Bott. Link between phosphate starvation and glycogen metabolism in &lt;i&gt;corynebacterium glutamicum&lt;/i&gt;, revealed by metabolomics. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, 76(20):6910–6919, OCT 2010. <a class="reference external" href="https://doi.org/10.1128/AEM.01375-10">[DOI]</a>.</p></li>
<li id="id27209"><p>Steinn Gudmundsson and Ines Thiele. Computationally efficient flux variability analysis. <em>BMC BIOINFORMATICS</em>, SEP 29 2010. <a class="reference external" href="https://doi.org/10.1186/1471-2105-11-489">[DOI]</a>.</p></li>
<li id="id27210"><p>Roger L. Chang, Li Xie, Lei Xie, Philip E. Bourne, and Bernhard O. Palsson. Drug off-target effects predicted using structural analysis in the context of a metabolic network model. <em>PLOS COMPUTATIONAL BIOLOGY</em>, SEP 2010. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1000938">[DOI]</a>.</p></li>
<li id="id27211"><p>Thomas Forth, Glenn A. McConkey, and David R. Westhead. Metnetmaker: a free and open-source tool for the creation of novel metabolic networks in sbml format. <em>BIOINFORMATICS</em>, 26(18):2352–2353, SEP 2010. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btq425">[DOI]</a>.</p></li>
<li id="id27212"><p>D. Hala, A. Amin, A. Mikler, and D. B. Huggett. A constraint-based stoichiometric model of the steroidogenic network of zebrafish (&lt;i&gt;danio&lt;/i&gt; &lt;i&gt;rerio&lt;/i&gt;). <em>JOURNAL OF BIOLOGICAL SYSTEMS</em>, 18(3):669–685, SEP 2010. <a class="reference external" href="https://doi.org/10.1142/S0218339010003469">[DOI]</a>.</p></li>
<li id="id27213"><p>Henning Knoop, Yvonne Zilliges, Wolfgang Lockau, and Ralf Steuer. The metabolic network of &lt;i&gt;synechocystis&lt;/i&gt; sp. pcc 6803: systemic properties of autotrophic growth. <em>PLANT PHYSIOLOGY</em>, 154(1):410–422, SEP 2010. <a class="reference external" href="https://doi.org/10.1104/pp.110.157198">[DOI]</a>.</p></li>
<li id="id27214"><p>M. Kschischo. A gentle introduction to the thermodynamics of biochemical stoichiometric networks in steady state. <em>EUROPEAN PHYSICAL JOURNAL-SPECIAL TOPICS</em>, 187(1):255–274, SEP 2010. <a class="reference external" href="https://doi.org/10.1140/epjst/e2010-01290-3">[DOI]</a>.</p></li>
<li id="id27215"><p>German Plata, Tzu-Lin Hsiao, Kellen L. Olszewski, Manuel Llinas, and Dennis Vitkup. Reconstruction and flux-balance analysis of the &lt;i&gt;plasmodium falciparum&lt;/i&gt; metabolic network. <em>MOLECULAR SYSTEMS BIOLOGY</em>, SEP 2010. <a class="reference external" href="https://doi.org/10.1038/msb.2010.60">[DOI]</a>.</p></li>
<li id="id27216"><p>Osbaldo Resendis-Antonio, Alberto Checa, and Sergio Encarnacion. Modeling core metabolism in cancer cells: surveying the topology underlying the warburg effect. <em>PLOS ONE</em>, AUG 25 2010. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0012383">[DOI]</a>.</p></li>
<li id="id27217"><p>Anu Raghunathan, Sookil Shin, and Simon Daefler. Systems approach to investigating host-pathogen interactions in infections with the biothreat agent &lt;i&gt;francisella&lt;/i&gt;. constraints-based model of &lt;i&gt;francisella tularensis&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, AUG 23 2010. <a class="reference external" href="https://doi.org/10.1186/1752-0509-4-118">[DOI]</a>.</p></li>
<li id="id27218"><p>Michael D. Barton, Daniela Delneri, Stephen G. Oliver, Magnus Rattray, and Casey M. Bergman. Evolutionary systems biology of amino acid biosynthetic cost in yeast. <em>PLOS ONE</em>, AUG 17 2010. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0011935">[DOI]</a>.</p></li>
<li id="id27219"><p>Dirk Koschuetzki, Bjoern H. Junker, Joerg Schwender, and Falk Schreiber. Structural analysis of metabolic networks based on flux centrality. <em>JOURNAL OF THEORETICAL BIOLOGY</em>, 265(3):261–269, AUG 7 2010. <a class="reference external" href="https://doi.org/10.1016/j.jtbi.2010.05.009">[DOI]</a>.</p></li>
<li id="id27220"><p>Chen Qi, Wang Zhuo, and Wei DongQing. Progress in the applications of flux analysis of metabolic networks. <em>CHINESE SCIENCE BULLETIN</em>, 55(22):2315–2322, AUG 2010. <a class="reference external" href="https://doi.org/10.1007/s11434-010-3022-x">[DOI]</a>.</p></li>
<li id="id27221"><p>M. Ahsanul Islam, Elizabeth A. Edwards, and Radhakrishnan Mahadevan. Characterizing the metabolism of &lt;i&gt;dehalococcoides&lt;/i&gt; with a constraint-based model. <em>PLOS COMPUTATIONAL BIOLOGY</em>, AUG 2010. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1000887">[DOI]</a>.</p></li>
<li id="id27222"><p>Balaji Veeramani and Joel S. Bader. Predicting functional associations from metabolism using bi-partite network algorithms. <em>BMC SYSTEMS BIOLOGY</em>, JUL 14 2010. <a class="reference external" href="https://doi.org/10.1186/1752-0509-4-95">[DOI]</a>.</p></li>
<li id="id27223"><p>John Blazeck and Hal Alper. Systems metabolic engineering: genome-scale models and beyond. <em>BIOTECHNOLOGY JOURNAL</em>, 5(7):647–659, JUL 2010. <a class="reference external" href="https://doi.org/10.1002/biot.200900247">[DOI]</a>.</p></li>
<li id="id27224"><p>Sergio Bordel, Rasmus Agren, and Jens Nielsen. Sampling the solution space in genome-scale metabolic networks reveals transcriptional regulation in key enzymes. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUL 2010. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1000859">[DOI]</a>.</p></li>
<li id="id27225"><p>Marija Cvijovic, Roberto Olivares-Hernandez, Rasmus Agren, Niklas Dahr, Wanwipa Vongsangnak, Intawat Nookaew, Kiran Raosaheb Patil, and Jens Nielsen. Biomet toolbox: genome-wide analysis of metabolism. <em>NUCLEIC ACIDS RESEARCH</em>, 38(2):W144–W149, JUL 2010. <a class="reference external" href="https://doi.org/10.1093/nar/gkq404">[DOI]</a>.</p></li>
<li id="id27226"><p>Fahimeh Salimi, Kai Zhuang, and Radhakrishnan Mahadevan. Genome-scale metabolic modeling of a clostridial co-culture for consolidated bioprocessing. <em>BIOTECHNOLOGY JOURNAL</em>, 5(7):726–738, JUL 2010. <a class="reference external" href="https://doi.org/10.1002/biot.201000159">[DOI]</a>.</p></li>
<li id="id27227"><p>Brian J. Schmidt, Xiefan Lin-Schmidt, Austin Chamberlin, Kourosh Salehi-Ashtiani, and Jason A. Papin. Metabolic systems analysis to advance algal biotechnology. <em>BIOTECHNOLOGY JOURNAL</em>, 5(7):660–670, JUL 2010. <a class="reference external" href="https://doi.org/10.1002/biot.201000129">[DOI]</a>.</p></li>
<li id="id27228"><p>Liming Liu, Rasmus Agren, Sergio Bordel, and Jens Nielsen. Use of genome-scale metabolic models for understanding microbial physiology. <em>FEBS LETTERS</em>, 584(12, SI):2556–2564, JUN 18 2010. <a class="reference external" href="https://doi.org/10.1016/j.febslet.2010.04.052">[DOI]</a>.</p></li>
<li id="id27229"><p>Keren Yizhak, Tomer Benyamini, Wolfram Liebermeister, Eytan Ruppin, and Tomer Shlomi. Integrating quantitative proteomics and metabolomics with a genome-scale metabolic network model. <em>BIOINFORMATICS</em>, 26(12):i255–i260, JUN 15 2010. 18th Annual International Conference on Intelligent Systems for Molecular Biology (ISMB), Boston, MA, JUL 11-13, 2010. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btq183">[DOI]</a>.</p></li>
<li id="id27230"><p>Adam M. Feist, Daniel C. Zielinski, Jeffrey D. Orth, Jan Schellenberger, Markus J. Herrgard, and Bernhard O. Palsson. Model-driven evaluation of the production potential for growth-coupled products of &lt;i&gt;escherichia coli&lt;/i&gt;. <em>METABOLIC ENGINEERING</em>, 12(3):173–186, MAY 2010. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2009.10.003">[DOI]</a>.</p></li>
<li id="id27231"><p>Jan Schellenberger, Junyoung O. Park, Tom M. Conrad, and Bernhard O. Palsson. Bigg: a biochemical genetic and genomic knowledgebase of large scale metabolic reconstructions. <em>BMC BIOINFORMATICS</em>, APR 29 2010. <a class="reference external" href="https://doi.org/10.1186/1471-2105-11-213">[DOI]</a>.</p></li>
<li id="id27232"><p>Isabel Rocha, Paulo Maia, Pedro Evangelista, Paulo Vilaca, Simao Soares, Jose P. Pinto, Jens Nielsen, Kiran R. Patil, Eugenio C. Ferreira, and Miguel Rocha. Optflux: an open-source software platform for &lt;i&gt;in silico&lt;/i&gt; metabolic engineering. <em>BMC SYSTEMS BIOLOGY</em>, APR 19 2010. <a class="reference external" href="https://doi.org/10.1186/1752-0509-4-45">[DOI]</a>.</p></li>
<li id="id27233"><p>Sven Dietz and Sven Panke. Microbial systems engineering: first successes and the way ahead. <em>BIOESSAYS</em>, 32(4, SI):356–362, APR 2010. <a class="reference external" href="https://doi.org/10.1002/bies.200900174">[DOI]</a>.</p></li>
<li id="id27234"><p>Mohammad T. Alam, Maria E. Merlo, David A. Hodgson, Elizabeth M. H. Wellington, Eriko Takano, Rainer Breitling, and STREAM Consortium. Metabolic modeling and analysis of the metabolic switch in &lt;i&gt;streptomyces coelicolor&lt;/i&gt;. <em>BMC GENOMICS</em>, MAR 26 2010. <a class="reference external" href="https://doi.org/10.1186/1471-2164-11-202">[DOI]</a>.</p></li>
<li id="id27235"><p>J. S. Ramsey, S. J. MacDonald, G. Jander, A. Nakabachi, G. H. Thomas, and A. E. Douglas. Genomic evidence for complementary purine metabolism in the pea aphid, &lt;i&gt;acyrthosiphon pisum&lt;/i&gt;, and its symbiotic bacterium &lt;i&gt;buchnera aphidicola&lt;/i&gt;. <em>INSECT MOLECULAR BIOLOGY</em>, 19(2):241–248, MAR 10 2010. <a class="reference external" href="https://doi.org/10.1111/j.1365-2583.2009.00945.x">[DOI]</a>.</p></li>
<li id="id27236"><p>Vicente Acuna, Alberto Marchetti-Spaccamela, Marie-France Sagot, and Leen Stougie. A note on the complexity of finding and enumerating elementary modes. <em>BIOSYSTEMS</em>, 99(3):210–214, MAR 2010. <a class="reference external" href="https://doi.org/10.1016/j.biosystems.2009.11.004">[DOI]</a>.</p></li>
<li id="id27237"><p>Brett A. Boghigian, Gargi Seth, Robert Kiss, and Blaine A. Pfeifer. Metabolic flux analysis and pharmaceutical production. <em>METABOLIC ENGINEERING</em>, 12(2):81–95, MAR 2010. <a class="reference external" href="https://doi.org/10.1016/j.ymben.2009.10.004">[DOI]</a>.</p></li>
<li id="id27238"><p>Jeffrey D. Orth, Ines Thiele, and Bernhard O. Palsson. What is flux balance analysis? <em>NATURE BIOTECHNOLOGY</em>, 28(3):245–248, MAR 2010. <a class="reference external" href="https://doi.org/10.1038/nbt.1614">[DOI]</a>.</p></li>
<li id="id27239"><p>Jenni Heino, Daniela Calvetti, and Erkki Somersalo. Metabolica: a statistical research tool for analyzing metabolic networks. <em>COMPUTER METHODS AND PROGRAMS IN BIOMEDICINE</em>, 97(2):151–167, FEB 2010. <a class="reference external" href="https://doi.org/10.1016/j.cmpb.2009.07.007">[DOI]</a>.</p></li>
<li id="id27240"><p>Brett A. Boghigian, Kyongbum Lee, and Blaine A. Pfeifer. Computational analysis of phenotypic space in heterologous polyketide biosynthesis-applications to &lt;i&gt;escherichia coli&lt;/i&gt;, &lt;i&gt;bacillus subtilis&lt;/i&gt;, and &lt;i&gt;saccharomyces cerevisiae&lt;/i&gt;. <em>JOURNAL OF THEORETICAL BIOLOGY</em>, 262(2):197–207, JAN 21 2010. <a class="reference external" href="https://doi.org/10.1016/j.jtbi.2009.10.006">[DOI]</a>.</p></li>
<li id="id27241"><p>Karthikeyan Srinivasan and Radhakrishnan Mahadevan. Characterization of proton production and consumption associated with microbial metabolism. <em>BMC BIOTECHNOLOGY</em>, JAN 20 2010. <a class="reference external" href="https://doi.org/10.1186/1472-6750-10-2">[DOI]</a>.</p></li>
<li id="id27242"><p>Xueyang Feng, Lawrence Page, Jacob Rubens, Lauren Chircus, Peter Colletti, Himadri B. Pakrasi, and Yinjie J. Tang. Bridging the gap between fluxomics and industrial biotechnology. <em>JOURNAL OF BIOMEDICINE AND BIOTECHNOLOGY</em>, 2010. <a class="reference external" href="https://doi.org/10.1155/2010/460717">[DOI]</a>.</p></li>
<li id="id27243"><p>Hannes Link, Bernd Anselment, and Dirk Weuster-Botz. Rapid media transition: an experimental approach for steady state analysis of metabolic pathways. <em>BIOTECHNOLOGY PROGRESS</em>, 26(1):1–10, JAN-FEB 2010. <a class="reference external" href="https://doi.org/10.1002/btpr.290">[DOI]</a>.</p></li>
<li id="id27244"><p>Marnix H. Medema, Axel Trefzer, Andriy Kovalchuk, Marco van den Berg, Ulrike Muller, Wilbert Heijne, Liang Wu, Mohammad T. Alam, Catherine M. Ronning, William C. Nierman, Roel A. L. Bovenberg, Rainer Breitling, and Eriko Takano. The sequence of a 1.8-mb bacterial linear plasmid reveals a rich evolutionary reservoir of secondary metabolic pathways. <em>GENOME BIOLOGY AND EVOLUTION</em>, 2:212–224, 2010. <a class="reference external" href="https://doi.org/10.1093/gbe/evq013">[DOI]</a>.</p></li>
<li id="id27245"><p>Ines Thiele and Bernhard O. Palsson. A protocol for generating a high-quality genome-scale metabolic reconstruction. <em>NATURE PROTOCOLS</em>, 5(1):93–121, 2010. <a class="reference external" href="https://doi.org/10.1038/nprot.2009.203">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-15" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-15" name="0-15" role="tabpanel" tabindex="0"><div class="docutils container" id="id27301">
<ol class="arabic simple" start="1">
<li id="id29066"><p>Bernhard Palsson. Metabolic systems biology. <em>FEBS LETTERS</em>, 583(24, SI):3900–3904, DEC 17 2009. 146th Nobel Symposium on Systems Biology, Drobak, NORWAY, JUN 25-27, 2009. <a class="reference external" href="https://doi.org/10.1016/j.febslet.2009.09.031">[DOI]</a>.</p></li>
<li id="id29067"><p>Alfonso Perez-Escudero, Marta Rivera-Alba, and Gonzalo G. de Polavieja. Structure of deviations from optimality in biological systems. <em>PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA</em>, 106(48):20544–20549, DEC 1 2009. <a class="reference external" href="https://doi.org/10.1073/pnas.0905336106">[DOI]</a>.</p></li>
<li id="id29068"><p>P. B. Warren, S. M. Duarte Queiros, and J. L. Jones. Flux networks in metabolic graphs. <em>PHYSICAL BIOLOGY</em>, DEC 2009. <a class="reference external" href="https://doi.org/10.1088/1478-3975/6/4/046006">[DOI]</a>.</p></li>
<li id="id29069"><p>Tae-Sung Jung, Hock Chuan Yeo, Satty G. Reddy, Wan-Sup Cho, and Dong-Yup Lee. Webcoli: an interactive and asynchronous web application for &lt;i&gt;in silico&lt;/i&gt; design and analysis of genome-scale &lt;i&gt;e&lt;/i&gt;.&lt;i&gt;coli&lt;/i&gt; model. <em>BIOINFORMATICS</em>, 25(21):2850–2852, NOV 1 2009. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btp496">[DOI]</a>.</p></li>
<li id="id29070"><p>Matthew A. Oberhardt, Bernhard O. Palsson, and Jason A. Papin. Applications of genome-scale metabolic reconstructions. <em>MOLECULAR SYSTEMS BIOLOGY</em>, NOV 2009. <a class="reference external" href="https://doi.org/10.1038/msb.2009.77">[DOI]</a>.</p></li>
<li id="id29071"><p>Milan J. A. van Hoek and Paulien Hogeweg. Metabolic adaptation after whole genome duplication. <em>MOLECULAR BIOLOGY AND EVOLUTION</em>, 26(11):2441–2453, NOV 2009. <a class="reference external" href="https://doi.org/10.1093/molbev/msp160">[DOI]</a>.</p></li>
<li id="id29072"><p>Eva Grafahrend-Belau, Christian Klukas, Bjoern H. Junker, and Falk Schreiber. Fba-simvis: interactive visualization of constraint-based metabolic models. <em>BIOINFORMATICS</em>, 25(20):2755–2757, OCT 15 2009. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btp408">[DOI]</a>.</p></li>
<li id="id29073"><p>Xin Fang, Anders Wallqvist, and Jaques Reifman. A systems biology framework for modeling metabolic enzyme inhibition of &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, SEP 15 2009. <a class="reference external" href="https://doi.org/10.1186/1752-0509-3-92">[DOI]</a>.</p></li>
<li id="id29074"><p>Jacob D. Feala, Laurence Coquin, Dan Zhou, Gabriel G. Haddad, Giovanni Paternostro, and Andrew D. McCulloch. Metabolism as means for hypoxia adaptation: metabolic profiling and flux balance analysis. <em>BMC SYSTEMS BIOLOGY</em>, SEP 9 2009. <a class="reference external" href="https://doi.org/10.1186/1752-0509-3-91">[DOI]</a>.</p></li>
<li id="id29075"><p>Dewu Ding, Xiaoqing He, Kezhong Lu, Pu Wu, and Haisheng Huang. Application of constraint-based methods in staphylococcus aureus metabolic network. <em>RIVISTA DI BIOLOGIA-BIOLOGY FORUM</em>, 102(3):385–397, SEP-DEC 2009.</p></li>
<li id="id29076"><p>Kenji Nakahigashi, Yoshihiro Toya, Nobuyoshi Ishii, Tomoyoshi Soga, Miki Hasegawa, Hisami Watanabe, Yuki Takai, Masayuki Honma, Hirotada Mori, and Masaru Tomita. Systematic phenome analysis of &lt;i&gt;escherichia coli&lt;/i&gt; multiple-knockout mutants reveals hidden reactions in central carbon metabolism. <em>MOLECULAR SYSTEMS BIOLOGY</em>, SEP 2009. <a class="reference external" href="https://doi.org/10.1038/msb.2009.65">[DOI]</a>.</p></li>
<li id="id29077"><p>F. Le Fevre, S. Smidtas, C. Combe, M. Durot, Florence d'Alche-Buc, and V. Schachter. Cycsim-an online tool for exploring and experimenting with genome-scale metabolic models. <em>BIOINFORMATICS</em>, 25(15):1987–1988, AUG 1 2009. <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btp268">[DOI]</a>.</p></li>
<li id="id29078"><p>Ani Manichaikul, Lila Ghamsari, Erik F. Y. Hom, Chenwei Lin, Ryan R. Murray, Roger L. Chang, S. Balaji, Tong Hao, Yun Shen, Arvind K. Chavali, Ines Thiele, Xinping Yang, Changyu Fan, Elizabeth Mello, David E. Hill, Marc Vidal, Kourosh Salehi-Ashtiani, and Jason A. Papin. Metabolic network analysis integrated with transcript verification for sequenced genomes. <em>NATURE METHODS</em>, 6(8):589–U53, AUG 2009. <a class="reference external" href="https://doi.org/10.1038/NMETH.1348">[DOI]</a>.</p></li>
<li id="id29079"><p>Karthik Raman and Nagasuma Chandra. Flux balance analysis of biological systems: applications and challenges. <em>BRIEFINGS IN BIOINFORMATICS</em>, 10(4):435–449, JUL 2009. <a class="reference external" href="https://doi.org/10.1093/bib/bbp011">[DOI]</a>.</p></li>
<li id="id29080"><p>Maikel P. H. Verouden, Richard A. Notebaart, Johan A. Westerhuis, Mariet J. van der Werf, Bas Teusink, and Age K. Smilde. Multi-way analysis of flux distributions across multiple conditions. <em>JOURNAL OF CHEMOMETRICS</em>, 23(7-8, SI):406–420, JUL-AUG 2009. <a class="reference external" href="https://doi.org/10.1002/cem.1238">[DOI]</a>.</p></li>
<li id="id29081"><p>Bas Teusink, Anne Wiersma, Leo Jacobs, Richard A. Notebaart, and Eddy J. Smid. Understanding the adaptive growth strategy of &lt;i&gt;lactobacillus plantarum&lt;/i&gt; by &lt;i&gt;in silico&lt;/i&gt; optimisation. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JUN 2009. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1000410">[DOI]</a>.</p></li>
<li id="id29082"><p>Segun Fatumo, Kitiporn Plaimas, Jan-Philipp Mallm, Gunnar Schramm, Ezekiel Adebiyi, Marcus Oswald, Roland Eils, and Rainer Koenig. Estimating novel potential drug targets of &lt;i&gt;plasmodium falciparum&lt;/i&gt; by analysing the metabolic network of knock-out strains &lt;i&gt;in silico&lt;/i&gt;. <em>INFECTION GENETICS AND EVOLUTION</em>, 9(3):351–358, MAY 2009. Conference on Bioinformatics of African Pathogens and Vectors, Nairobi, KENYA, MAY 28-JUN 02, 2007. <a class="reference external" href="https://doi.org/10.1016/j.meegid.2008.01.007">[DOI]</a>.</p></li>
<li id="id29083"><p>Monica L. Mo, Bernhard O. Palsson, and Markus J. Herrgard. Connecting extracellular metabolomic measurements to intracellular flux states in yeast. <em>BMC SYSTEMS BIOLOGY</em>, MAR 25 2009. <a class="reference external" href="https://doi.org/10.1186/1752-0509-3-37">[DOI]</a>.</p></li>
<li id="id29084"><p>Osbaldo Resendis-Antonio. Filling kinetic gaps: dynamic modeling of metabolism where detailed kinetic information is lacking. <em>PLOS ONE</em>, MAR 23 2009. <a class="reference external" href="https://doi.org/10.1371/journal.pone.0004967">[DOI]</a>.</p></li>
<li id="id29085"><p>Christian L. Barrett, Markus J. Herrgard, and Bernhard Palsson. Decomposing complex reaction networks using random sampling, principal component analysis and basis rotation. <em>BMC SYSTEMS BIOLOGY</em>, MAR 6 2009. <a class="reference external" href="https://doi.org/10.1186/1752-0509-3-30">[DOI]</a>.</p></li>
<li id="id29086"><p>Aneil Mallavarapu, Matthew Thomson, Benjamin Ullian, and Jeremy Gunawardena. Programming with models: modularity and abstraction provide powerful capabilities for systems biology. <em>JOURNAL OF THE ROYAL SOCIETY INTERFACE</em>, 6(32):257–270, MAR 6 2009. <a class="reference external" href="https://doi.org/10.1098/rsif.2008.0205">[DOI]</a>.</p></li>
<li id="id29087"><p>Jan Schellenberger and Bernhard O. Palsson. Use of randomized sampling for analysis of metabolic networks. <em>JOURNAL OF BIOLOGICAL CHEMISTRY</em>, 284(9):5457–5461, FEB 27 2009. <a class="reference external" href="https://doi.org/10.1074/jbc.R800048200">[DOI]</a>.</p></li>
<li id="id29088"><p>Gavin H. Thomas, Jeremy Zucker, Sandy J. Macdonald, Anatoly Sorokin, Igor Goryanin, and Angela E. Douglas. A fragile metabolic network adapted for cooperation in the symbiotic bacterium &lt;i&gt;buchnera aphidicola&lt;/i&gt;. <em>BMC SYSTEMS BIOLOGY</em>, FEB 21 2009. <a class="reference external" href="https://doi.org/10.1186/1752-0509-3-24">[DOI]</a>.</p></li>
<li id="id29089"><p>Adam M. Feist, Markus J. Herrgard, Ines Thiele, Jennie L. Reed, and Bernhard O. Palsson. Reconstruction of biochemical networks in microorganisms. <em>NATURE REVIEWS MICROBIOLOGY</em>, 7(2):129–143, FEB 2009. <a class="reference external" href="https://doi.org/10.1038/nrmicro1949">[DOI]</a>.</p></li>
<li id="id29090"><p>Fan Li, Ines Thiele, Neema Jamshidi, and Bernhard O. Palsson. Identification of potential pathway mediation targets in toll-like receptor signaling. <em>PLOS COMPUTATIONAL BIOLOGY</em>, FEB 2009. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1000292">[DOI]</a>.</p></li>
<li id="id29091"><p>Balaji Veeramani and Joel S. Bader. Metabolic flux correlations, genetic interactions, and disease. <em>JOURNAL OF COMPUTATIONAL BIOLOGY</em>, 16(2):291–302, FEB 2009. <a class="reference external" href="https://doi.org/10.1089/cmb.2008.14TT">[DOI]</a>.</p></li>
<li id="id29092"><p>Chenggang Yu, Nela Zavaijevski, Valmik Desai, and Jaques Reifman. Genome-wide enzyme annotation with precision control: catalytic families [catfam] databases. <em>PROTEINS-STRUCTURE FUNCTION AND BIOINFORMATICS</em>, 74(2):449–460, FEB 1 2009. <a class="reference external" href="https://doi.org/10.1002/prot.22167">[DOI]</a>.</p></li>
<li id="id29093"><p>Yanping Xi, Yi-Ping Phoebe Chen, Ming Cao, Weirong Wang, and Fei Wang. Analysis on relationship between extreme pathways and correlated reaction sets. <em>BMC BIOINFORMATICS</em>, JAN 30 2009. 9th Asia Pacific Bioinformatics Conference, Tsinghua Univ, Beijing, PEOPLES R CHINA, JAN 13-16, 2009. <a class="reference external" href="https://doi.org/10.1186/1471-2105-10-S1-S58">[DOI]</a>.</p></li>
<li id="id29094"><p>Zixiang Xu, Xiao Sun, and Shihai Yu. Genome-scale analysis to the impact of gene deletion on the metabolism of &lt;i&gt;e-coli&lt;/i&gt;: constraint-based simulation approach. <em>BMC BIOINFORMATICS</em>, JAN 30 2009. 9th Asia Pacific Bioinformatics Conference, Tsinghua Univ, Beijing, PEOPLES R CHINA, JAN 13-16, 2009. <a class="reference external" href="https://doi.org/10.1186/1471-2105-10-S1-S62">[DOI]</a>.</p></li>
<li id="id29095"><p>Maxime Durot, Pierre-Yves Bourguignon, and Vincent Schachter. Genome-scale models of bacterial metabolism: reconstruction and applications. <em>FEMS MICROBIOLOGY REVIEWS</em>, 33(1):164–190, JAN 2009. <a class="reference external" href="https://doi.org/10.1111/j.1574-6976.2008.00146.x">[DOI]</a>.</p></li>
<li id="id29096"><p>Eva Grafahrend-Belau, Falk Schreiber, Dirk Koschuetzki, and Bjoern H. Junker. Flux balance analysis of barley seeds: a computational approach to study systemic properties of central metabolism. <em>PLANT PHYSIOLOGY</em>, 149(1):585–598, JAN 2009. <a class="reference external" href="https://doi.org/10.1104/pp.108.129635">[DOI]</a>.</p></li>
<li id="id29097"><p>Varun Mazumdar, Evan S. Snitkin, Salomon Amar, and Daniel Segre. Metabolic network model of a human oral pathogen. <em>JOURNAL OF BACTERIOLOGY</em>, 191(1):74–90, JAN 2009. <a class="reference external" href="https://doi.org/10.1128/JB.01123-08">[DOI]</a>.</p></li>
<li id="id29098"><p>Monica L. Mo and Bernhard O. Palsson. Understanding human metabolic physiology: a genome-to-systems approach. <em>TRENDS IN BIOTECHNOLOGY</em>, 27(1):37–44, JAN 2009. <a class="reference external" href="https://doi.org/10.1016/j.tibtech.2008.09.007">[DOI]</a>.</p></li>
<li id="id29099"><p>Karthik Raman, Rohit Vashisht, and Nagasuma Chandra. Strategies for efficient disruption of metabolism in &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; from network analysis. <em>MOLECULAR BIOSYSTEMS</em>, 5(12):1740–1751, 2009. <a class="reference external" href="https://doi.org/10.1039/b905817f">[DOI]</a>.</p></li>
<li id="id29100"><p>Martin I. Sigurdsson, Neema Jamshidi, Jon J. Jonsson, and Bernhard O. Palsson. Genome-scale network analysis of imprinted human metabolic genes. <em>EPIGENETICS</em>, 4(1):43–46, JAN 2009. <a class="reference external" href="https://doi.org/10.4161/epi.4.1.7603">[DOI]</a>.</p></li>
<li id="id29101"><p>Ralf Steuer and Bjoern H. Junker. Computational models of metabolism: stability and regulation in metabolic networks. In SA Rice, editor, <em>ADVANCES IN CHEMICAL PHYSICS, VOL 142</em>, volume 142 of Advances in Chemical Physics, pages 105–251. 2009. <a class="reference external" href="https://doi.org/10.1002/9780470475935.ch3">[DOI]</a>.</p></li>
<li id="id29102"><p>Zhi Wang and Jianzhi Zhang. Abundant indispensable redundancies in cellular metabolic networks. <em>GENOME BIOLOGY AND EVOLUTION</em>, 1:23–33, 2009. <a class="reference external" href="https://doi.org/10.1093/gbe/evp002">[DOI]</a>.</p></li>
<li id="id29103"><p>Zhi Wang and Jianzhi Zhang. Why is the correlation between gene importance and gene evolutionary rate so weak? <em>PLOS GENETICS</em>, JAN 2009. <a class="reference external" href="https://doi.org/10.1371/journal.pgen.1000329">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-16" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-16" name="0-16" role="tabpanel" tabindex="0"><div class="docutils container" id="id29121">
<ol class="arabic simple" start="1">
<li id="id30924"><p>Karthik Raman, Kalidas Yeturu, and Nagasuma Chandra. Targettb: a target identification pipeline for &lt;i&gt;mycobacterium tuberculosis&lt;/i&gt; through an interactome, reactome and genome-scale structural analysis. <em>BMC SYSTEMS BIOLOGY</em>, DEC 19 2008. <a class="reference external" href="https://doi.org/10.1186/1752-0509-2-109">[DOI]</a>.</p></li>
<li id="id30925"><p>Laurence Coquin, Jacob D. Feala, Andrew D. McCulloch, and Giovanni Paternostro. Metabolomic and flux-balance analysis of age-related decline of hypoxia tolerance in &lt;i&gt;drosophila&lt;/i&gt; muscle tissue. <em>MOLECULAR SYSTEMS BIOLOGY</em>, DEC 2008. <a class="reference external" href="https://doi.org/10.1038/msb.2008.71">[DOI]</a>.</p></li>
<li id="id30926"><p>Vasiliy A. Portnoy, Markus J. Herrgard, and Bernhard O. Palsson. Aerobic fermentation of d-glucose by an evolved cytochrome oxidase-deficient &lt;i&gt;escherichia coli&lt;/i&gt; strain. <em>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</em>, 74(24):7561–7569, DEC 2008. <a class="reference external" href="https://doi.org/10.1128/AEM.00880-08">[DOI]</a>.</p></li>
<li id="id30927"><p>Dirk Bumann. Has nature already identified all useful antibacterial targets? <em>CURRENT OPINION IN MICROBIOLOGY</em>, 11(5):387–392, OCT 2008. <a class="reference external" href="https://doi.org/10.1016/j.mib.2008.08.002">[DOI]</a>.</p></li>
<li id="id30928"><p>Patrik R. Jones. Improving fermentative biomass-derived h&lt;sub&gt;2&lt;/sub&gt;-production by engineering microbial metabolism. <em>INTERNATIONAL JOURNAL OF HYDROGEN ENERGY</em>, 33(19, SI):5122–5130, OCT 2008. 2nd Asian Bio-Hydrogen Symposium, Daejeon, SOUTH KOREA, NOV, 2007. <a class="reference external" href="https://doi.org/10.1016/j.ijhydene.2008.05.004">[DOI]</a>.</p></li>
<li id="id30929"><p>Juan Nogales, Bernhard O. Palsson, and Ines Thiele. A genome-scale metabolic reconstruction of &lt;i&gt;pseudomonas putida&lt;/i&gt; kt2440: &lt;i&gt;i&lt;/i&gt;jn746 as a cell factory. <em>BMC SYSTEMS BIOLOGY</em>, SEP 16 2008. <a class="reference external" href="https://doi.org/10.1186/1752-0509-2-79">[DOI]</a>.</p></li>
<li id="id30930"><p>Kitiporn Plaimas, Jan-Phillip Mallm, Marcus Oswald, Fabian Svara, Victor Sourjik, Roland Eils, and Rainer Koenig. Machine learning based analyses on metabolic networks supports high-throughput knockout screens. <em>BMC SYSTEMS BIOLOGY</em>, JUL 24 2008. <a class="reference external" href="https://doi.org/10.1186/1752-0509-2-67">[DOI]</a>.</p></li>
<li id="id30931"><p>Jeremiah Wright and Andreas Wagner. The systems biology research tool: evolvable open-source software. <em>BMC SYSTEMS BIOLOGY</em>, JUN 29 2008. <a class="reference external" href="https://doi.org/10.1186/1752-0509-2-55">[DOI]</a>.</p></li>
<li id="id30932"><p>Adam M. Feist and Bernhard O. Palsson. The growing scope of applications of genome-scale metabolic reconstructions using &lt;i&gt;escherichia coli&lt;/i&gt;. <em>NATURE BIOTECHNOLOGY</em>, 26(6):659–667, JUN 2008. <a class="reference external" href="https://doi.org/10.1038/nbt1401">[DOI]</a>.</p></li>
<li id="id30933"><p>Rui Alves, Ester Vilaprinyo, and Albert Sorribas. Integrating bioinformatics and computational biology:: perspectives and possibilities for &lt;i&gt;in silico&lt;/i&gt; network reconstruction in molecular systems biology. <em>CURRENT BIOINFORMATICS</em>, 3(2):98–129, MAY 2008. <a class="reference external" href="https://doi.org/10.2174/157489308784340694">[DOI]</a>.</p></li>
<li id="id30934"><p>Scott A. Becker and Bernhard O. Palsson. Context-specific metabolic networks are consistent with experiments. <em>PLOS COMPUTATIONAL BIOLOGY</em>, MAY 2008. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1000082">[DOI]</a>.</p></li>
<li id="id30935"><p>Scott A. Becker and Bernhard O. Palsson. Three factors underlying incorrect in silico predictions of essential metabolic genes. <em>BMC SYSTEMS BIOLOGY</em>, FEB 4 2008. <a class="reference external" href="https://doi.org/10.1186/1752-0509-2-14">[DOI]</a>.</p></li>
<li id="id30936"><p>Richard A. Notebaart, Bas Teusink, Roland J. Siezen, and Balazs Papp. Co-regulation of metabolic genes is better explained by flux coupling than by network distance. <em>PLOS COMPUTATIONAL BIOLOGY</em>, JAN 2008. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.0040026">[DOI]</a>.</p></li>
<li id="id30937"><p>Sakiko Okumoto, Hitomi Takanaga, and Wolf B. Frommer. Quantitative imaging for discovery and assembly of the metabo-regulome. <em>NEW PHYTOLOGIST</em>, 180(2):271–295, 2008. <a class="reference external" href="https://doi.org/10.1111/j.1469-8137.2008.02611.x">[DOI]</a>.</p></li>
<li id="id30938"><p>Evan S. Snitkin, Aimee M. Dudley, Daniel M. Janse, Kaisheen Wong, George M. Church, and Daniel Segre. Model-driven analysis of experimentally determined growth phenotypes for 465 yeast gene deletion mutants under 16 different conditions. <em>GENOME BIOLOGY</em>, 2008. <a class="reference external" href="https://doi.org/10.1186/gb-2008-9-9-r140">[DOI]</a>.</p></li>
</ol>
</div>
</div><div aria-labelledby="tab-0-0-17" class="sphinx-tabs-panel" hidden="true" id="panel-0-0-17" name="0-17" role="tabpanel" tabindex="0"><div class="docutils container" id="id30941">
<ol class="arabic simple" start="1">
<li id="id32759"><p>Adam M. Feist, Christopher S. Henry, Jennifer L. Reed, Markus Krummenacker, Andrew R. Joyce, Peter D. Karp, Linda J. Broadbelt, Vassily Hatzimanikatis, and Bernhard O. Palsson. A genome-scale metabolic reconstruction for &lt;i&gt;escherichia coli&lt;/i&gt; k-12 mg1655 that accounts for 1260 orfs and thermodynamic information. <em>MOLECULAR SYSTEMS BIOLOGY</em>, JUN 2007. <a class="reference external" href="https://doi.org/10.1038/msb4100155">[DOI]</a>.</p></li>
<li id="id32760"><p>Monica L. Mo, Neema Jamshidi, and Bernhard O. Palsson. A genome-scale, constraint-based approach to systems biology of human metabolism. <em>MOLECULAR BIOSYSTEMS</em>, 3(9):598–603, 2007. <a class="reference external" href="https://doi.org/10.1039/b705597h">[DOI]</a>.</p></li>
</ol>
</div>
</div></div>
</section>
</section>


           </div>
           <div class="articleComments">
            
           </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
      
        <a href="contact.html" class="btn btn-default" title="Contact" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2017-2021, The COBRA Toolbox developers.

    </p>
  </div>
  Built with Sphinx using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  

  

    <script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'./',
            VERSION:'2.0.0',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true,
            SOURCELINK_SUFFIX: '.txt'
        };
    </script>
    <script type="text/javascript" src="_static/documentation_options.js"></script>
    <script type="text/javascript" src="_static/jquery.js"></script>
    <script type="text/javascript" src="_static/underscore.js"></script>
    <script type="text/javascript" src="_static/_sphinx_javascript_frameworks_compat.js"></script>
    <script type="text/javascript" src="_static/doctools.js"></script>
    <script type="text/javascript" src="_static/sphinx_highlight.js"></script>
    <script type="text/javascript" src="_static/tabs.js"></script>

  


  
  
    <script type="text/javascript" src="_static/js/theme.js"></script>
  

  
  
  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.StickyNav.enable();
      });
  </script>
   

  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-TRCMZL1FKK"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    gtag('config', 'G-TRCMZL1FKK');
  </script>
    
</body>
</html>